```{r, echo = F}
load("data/ground_truth_2D.RData")
load("data/SN180_ACI.RData")
load("data/ac.RData")
load("data/ground_truth_3d.RData")
```

# Materials and Methods {#mat-met}

## Materials {#mat}

### Chemicals {#mat-chem}

---------------------------------------------------------------------------------------------
                     Chemical             Company               cat.-no.
----------------------------- --------------------------------- -----------------------------
  Agarose                                   Roth                          6351.2
  
  Agar-Agar                                 Roth                          5210.3

  Ampicillin                                Roth                          K029.2

  ATP                                       Epicentre                     E311K
  
  Blocking Reagent                          Roche                         11096176001

  BCIP                                      Fermentas                     R0822

  CaCl\textsubscript{2}                     Roth                          886.1

  Calyculin                                 Sigma                         208851

  DAPI                                      Sigma                         D9542

  DIG RNA Mix                               Roche                         11277073910

  DMSO                                      Roth                          4720.2

  EtBr                                      Roth                          2218.3

  EtOH                                      Roth                          9065.3

  Formaldehyde                              Roth                          7398.1

  Formamide                                 Roth                          P040.1

  Glycerol                                  Roth                          3783.2

  IPTG                                      Thermo                        R1171

  KCl                                       Roth                          P017.1

  Low melting point Agarose                 Roth                          A9539

  Maleic Acid                               Roth                          3810.3

  MgSO\textsubscript{4}                     Roth                          T888.2

  MeOH                                      Roth                          CP43.3

  MgCl\textsubscript{2}                     Roth                          2189.1

  NaCl                                      Roth                          9265.2

  NaHCO\textsubscript{3}                    Roth                          855.1

  NaOH                                      Roth                          6771.3

  NGS                                       Sigma                         C6767

  _p_-Formaldehyd                           Sigma                         P6148

  Phenol Red                                Sigma                         P0290

  Propan-2-ol                               VWR                           20842330

  Proteinase K                              Roth                          7528.4

  PTU                                       Sigma                         P7629

  Rockout                                   Sigma                         555553

  SSC                                       Roth                          1232.1

  SU5402                                    CALBIOCHEM                    572630

  Torula RNA                                Sigma                         R6625

  Tricaine                                  Sigma                         A5040

  Tris Base                                 Roth                          4855.2

  Triton-X100                               Roth                          3051.2

  Trizol                                    Ambion                        15596018

  Tween20                                   Sigma                         P1379
----------------------------- --------------------------------- -----------------------------
Table: (\#tab:met-chem) Chemicals

### Solutions {#mat-sol}

---------------------------------------------------------------------------------------------
                 **Solution**     Company                       cat.-no.   
----------------------------- --------------------------------- -----------------------------
Cut Smart Buffer                NEB                                       B7204S

Generuler 100 bp                Thermo                                    SM0241

Generuler 1kb                   Thermo                                    SM0311
----------------------------- --------------------------------- -----------------------------
Table: (\#tab:mat-sol) Solutions

### Antibodies {#mat-anitb}

---------------------------------------------------------------------------------------------
         **Antibody**     Company / Provider                concentration     cat.-no.   
--------------------- ----------------------------------- ------------------- ---------------
Anti-Digoxigenin        Roche                                 1:200            11093274910

Anti-GFP                Torrey Pines                          1:200            -

Anti-TAZ (rabbit)       Cell Signaling Technology             1:200            D24E4

Anti-ZO1                Zymed                                 1:200            33-9100

Alexa Fluor488          Invitrogen                            1:500            710369

Alexa Fluor555          Invitrogen                            1:500            Z25005
--------------------- ----------------------------------- ------------------- ---------------
Table: (\#tab:mat-antib) Antibodies

### Enzymes {#mat-enz}

---------------------------------------------------------------------------------------------
                   **Enzyme**     Company                       cat.-no.   
----------------------------- --------------------------------- -----------------------------
BtsCI                           NEB                                         R0647  

DdeI                            NEB                                         R0175    

HaeIII                          NEB                                         R0108   

MnlII                           NEB                                         R0163  

NlaIII                          NEB                                         R0125   

NsiI-HF                         NEB                                         R3127  

Phusion Polymerase              NEB                                         M0530L

Pronase                         Sigma                                       P5147

Ribolock                        Thermo                                      EO0381

RNase A                         Quiagen                                     1006657

RNase H                         NEB                                         M0297L

SP6 RNA Polymerase              Thermo                                      EP0131

T4 Ligase                       NEB                                         M0202T

T7 RNA Polymerase               Thermo                                      EP0111

Taq DNA Polymerase              Invitrogen                                  10342-020 

Taq DNA Polymerase              VWR                                         733-1301
----------------------------- --------------------------------- -----------------------------
Table: (\#tab:mat-enz) Enzymes

### Molecular Biology Kits {#mat-mobikits}

---------------------------------------------------------------------------------------------
                                           **Kit**    Company                 cat.-no.   
-------------------------------------------------- -------------------------- ---------------
EdU Click-iT                                        Invitrogen                  MP 10083

mMessage mMachine Sp6 Polymerase                    Invitrogen                  AM1340

PCR & Gel Clean-Up                                  Sigma                       NA1020

pGEM-T TA Cloning                                   Promega                     A3600

Superscript III cDNA Synthesis                      Thermo                      18080051

Wizard SV Gel and PCR Clean-Up                      Promega                     A9282
-------------------------------------------------- -------------------------- ---------------
Table: (\#tab:mat-mobikits) Molecular Biology Kits

### Buffers {#mat-buff}

---------------------------------------------------------------------------------------------
**Buffer**                 
---------------------- ----------------------------------------------------------------------
Blocking Reagent (BR)   2% BR in maleic buffer + 5% serum

E3                      [@Westerfield2007]

Hybridization buffer    50% Formamide + 25 % 20x SSC + 50mg/mL Heparine + mQ

Maleic buffer           250 mM maleic acid + 5M NaCl + 10% 0.1% Tween-20 + mQ

NTMT                    5 M NaCl + 1 M MgCl\textsubscript{2} + 1 M Tris pH 9.5 + 10% Tween

PBS                     2.7 mM KCl + 12 mM HPO\textsubscript{4}\textsuperscript{2-} 

PBST                    PBS + 0.1% Tween20

PBDT                    PBS + 1% BSA + 1% DMSO + 0.3% Triton

PFA                     4% paraformaldehyde in PBS

P1                      50 mM Tris-HCl pH 8.0 + 10 mM EDTA pH 8.0 + 100 µg/mL RNAse

P2                      1M NaOH + 10 % (w/v) SDS

P3                      3M KOAc pH 5.5

TNT                     50 mM Tris-HCl pH 8.0 + 100 mM NaCl + 0.1% Tween-20 
---------------------- ----------------------------------------------------------------------
Table: (\#tab:mat-buff) Buffers

### Zebrafish lines {#mat-lines}

-----------------------------------------------------------------------------------------------
  **Allele**    **name**                   **zfin**             
------------ ----------------------------- ----------------------------------------------------
zf106Tg       _cldnb:lyn-gfp_              [_Tg(-8.0cldnb:LY-EGFP)_](//zfin.org/ZDB-ALT-060919-2)

fu13Tg        _cxcr4b_(BAC):_H2BRFP_       _TgBAC(cxcr4b:Hsa.HIST1H2BJ-RFP)_

nns8Tg        _atoh1a:Tom_                 [_Tg(atoh1a:dTomato)nns8_](//zfin.org/ZDB-FISH-150901-21622)

fu50          _shroom3_                    -

m1274Tg       _hsp70:shr3v1FL-taqRFP_      [_Tg1(hsp70l:shroom3-TagRFP)_](//zfin.org/ZDB-FISH-150901-25907)
------------ ----------------------------- ----------------------------------------------------
Table: (\#tab:mat-lines) Zebrafish lines

### ISH probes {#mat-probes}

---------------------------------------------------------------------------------------------
**Probe**  **Sequence**                
---------- ----------------------------------------------------------------------------------
atoh1a      see [@Lecaudey2008a]

deltaD      see [@Haddon1998]
---------- ----------------------------------------------------------------------------------
Table: (\#tab:mat-probes) ISH probes

### Morpholinos {#mat-mos}

---------------------------------------------------------------------------------------------
**Probe**  **Sequence**                   **Concentration** 
---------- ------------------------------ ---------------------------------------------------
MoAtoh1a    see [@Lecaudey2008a; @Ma2009]  0.4 ng/mM

p53         see [@Robu2007]                2 ng/mM
---------- ------------------------------ ---------------------------------------------------
Table: (\#tab:mat-mos) Morpholinos

### Hardware {#mat-hrdwr}

#### Mounting Stamp {#mat-stamp}

An stl file for 3D printing can be found at [github.com/KleinhansDa/3DModels](https://github.com/KleinhansDa/3DModels)

---------------------------------------------------------------------------------------------
                                     **Component**      Company               cat.-no.
-------------------------------------------------- -------------------------- ---------------
  $\mu$ dish                                            Ibidi                   81,218–200
                      
  Stamp                                                 -                       -
                      
  Preparation needles                                   VWR                     USBE5470
                      
  Pasteur Pipettes                                      Roth                    4518
  
  Rubber / Silicone bulb                                VWR                     612-2327
  
  Microtubes 2 mL                                       Sarstedt                2691
  
  Heating block                                         PeqLab                  HX2
  
  Microwave oven                                        Severin                 MW7849
  
  Stereo microscope                                     Leica                   M165FC
  
  Transmitted Light Base                                Leica                   MDG36
  
  Countersunk screw DIN7991, 8 × 20 mm                  Dresselhaus (Hornbach)  7662389
  
  Superglue                                             UHU                     509141
-------------------------------------------------- -------------------------- ---------------
Table: (\#tab:mat-mount) Materials for production of a standardized mounting stamp

#### Spinning Disc Microscopy {#mat-SD}

---------------------------------------------------------------------------------------------
       **Component**    Company         Product               Specs    
-------------------- ---------------- ----------------------- -------------------------------
 Microscope            Nikon            Eclipse Ti-E            fully motorized 
 
 PFS                   Nikon            Perfect focus system    Z repositioning 
 
 XY-table              Merzhaeuser      XY motorized table      1 $\mu$m accuracy  
 
 Piezo                                  Piezo Z-table           300 $\mu$m scan range
 
 SD system             Yokogawa         CSU-W1                  50 $\mu$m pattern
 
 Laser                                  Laser Combiner          see table \@ref(tab:lasers)

 FRAPPA                Revolution       FRAPPA                  -
 
 Borealis              Borealis         Borealis                flat field correction 
 
 sCMOS                 Andor            ZYLA PLUS               4.2Mpix; 82%QE
 
 Immersion             Merzhaeuser      Liquid Dispenser        -
-------------------- ---------------- ----------------------- -------------------------------
Table: (\#tab:SDcomp) Spinning Disc system components

------------------------------------------------------------------------------------
                       **Lasers**        Type      Power
--------------------------------- ---------------- ---------------------------------
405 nm                              diode             100 mW  

445 nm                              diode             80 mW  

488 nm                              DPSS              100 mW  

561 nm                              DPSS              100 mW 

640 nm                              diode             100 mW  
--------------------------------- ---------------- ---------------------------------
Table: (\#tab:lasers) Available lasers

---------------------------------------------------------------------------
 **Objective**  Company   Type         Immersion   N.A.  working distance
-------------- --------- ------------ ----------- ------ ------------------
 10x            Nikon    CFI APO        air        0.45   4.00 mm 
 
 20X            Nikon    CFI APO        water      0.95   0.95 mm  
 
 40X            Nikon    CFI APO        water      1.15   0.60 mm  
 
 60X            Nikon    CFI APO        water      1.20   0.30 mm            
-------------- --------- ------------ ----------- ------ ------------------
Table: (\#tab:objectives) Available objectives

#### Workstation {#mat-work}

Statistical computation and image analysis were done on a Fujitsu Siemens (FS) Workstation CELSIUSM740 with the following hardware components...

-------------------------------------------------------------------------------
   **Component**      Company         Product          Specs
---------------- ----------------- ------------------- ------------------------
CPU                 Intel           Xeon E5-1660v4      3.2 GHz, 20MB, 8cores  

RAM                 Fujitsu         -                   4x16GB DDR4-2400 

Graphics            NVIDIA          Quadro M4000        8GB RAM 
---------------- ----------------- ------------------- ------------------------
Table: (\#tab:PCcomp) Workstation hardware components

### Software {#mat-sftwr}

----------------------------------------------------------------------------
                  **Software**   Version      web                    
------------------------------ -------------- ------------------------------ 
 Imagej FIJI                    1.48            https://fiji.sc/
 
 R                              3.6.1           https://cran.r-project.org/
 
 RStudio                        1.0.153         https://www.rstudio.com/
 
 Ubuntu                         17.1            https://www.ubuntu.com/
 
 Windows 10 Pro                 10.0.16299
 
 Total Commander                9.0             http://ghisler.com/
------------------------------ -------------- ------------------------------
Table: (\#tab:sftwr) Used software

## Methods {#met}

### Data Strategy and Analysis

Due to the history of Developmental Biology and the complexity of biological processes _per se_, the field heavily relies on image data. Since the advent of electronic imaging techniques^[e.g. photomultipliers or charge-coupled devices] scientific image data can be processed and analyzed _in silico_. To make use of the possibility of 

1. live imaging, which (as compared to fixation techniques) conserves the cellular integrity and morphology while also offers the possibility of recording time-lapses 
2. the optically clear specimen and 
3. high throughput image analysis and state-of-the-art data science using algorithmic implementations, 

the three following points were paid special attention to.

#### Sample Preparation

For fluorescence microscopy zebrafish embryos are usually immersed in a 1$\%$ solution of _low melting-point agarose_ (LMPA) and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount^[the process of embedding the samples in agarose] eight to ten embryos _per_ dish. To make use of the high number of offspring, a single zebrafish female may lay, which rapidly leads to a sample number of more than 300, a new sample preparation technique was designed that allows for (1) a four - to five - time increase in samples per dish (2) a facilitated navigation _via_ a grid-like orientation through the samples and (3) an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope. For details, see Materials and Methods section \@ref(mount-met) and Kleinhans _et al._, 2019[@Kleinhans2019].

#### Imaging

Technically, speed and sensitivity are most important for live imaging. Considering these two parameters a light-sheet [@Keller2010c] fluorescence microscope (LSFM) would be the best fit. However, LSFMs also have several limitations. First, due to the sample preparation methods available the number of samples that can be imaged at a time is highly restricted. Second, for subcellular resolution a high magnification is required, which is limited by working distances and third - for optimal image analysis a high Signal-to-Noise ratio (SNR) and numerical aperture (N.A.) is preferable. Therefore a spinning disc [@Graf2005] system was chosen for most of the imaging. 
The system makes use of (1) an extra-large field of view (FOV) ideal for large specimen, (2) the possibility of a high degree of automation with state-of-the-art software and (3) a water dispensing system for long-term water immersion imaging. For details about the system see Materials section \@ref(mat-SD).

#### Data handling

After data acquisition and pre-processing, the image data was transferred from the microscope system to the labs main workstation. To uniquely identify each file and have them appear in a structured manner, a file-naming system was established after the following structure

\makebox[\linewidth]{$[stage]\_[group]\_[id]\_[date]$}
\newline

Where _stage_ would _e.g._ be 32hpf, _group_ would be a genotype or drug treatment, _id_ would be a positional identifier on an imaging dish like B1P01^[Where B stands for a batch, that is if multiple dishes were imaged and P stands for the position within a single batch] and _date_ would be a date in the form of YYMMDD.

#### Image and Data Analysis

In order to be as objective and as high throughput as possible, almost all of the analyses performed for this study was solved either algorithmically or using convolutional neural networks (CNNs). Furthermore, to meet the terms and conditions of _open science_^[“movement to make scientific research […] and its dissemination accessible to all levels of an inquiring society” – Wikipedia/en/Open_science] standards, all pipelines were implemented in open source software frameworks such as _Fiji is just image J_ (FIJI) and R. For further information about training datasets, algorithms and versions used see Materials section \@ref(mat-sftwr).

### Zebrafish {#Zeb-met}

#### Husbandry

Zebraﬁsh husbandry was maintained at the University of Frankfurt am Main. All legal procedures were followed while handling and maintaining zebraﬁsh husbandry. All zebraﬁsh lines used in and generated for this study are listed in Materials section \@ref(mat-lines).

#### Handling and rearing

In the afternoon preceding the embryo collection, 1 male and female were set up in crossing cages, physically separated by a transparent separator. Next day, before noon, separators were removed allowing fertilization. Fertilized eggs were then collected, sorted and reared in the well-defined culture medium E3 (Kimmel et.al. 1995, section \@ref(mat-buff)) at 25$^\circ$, 28.5$^\circ$, or 30$^\circ$ C depending on the experimental condition required.

To grow larvae to the adulthood, they were transferred to the system on day 5. Till day 12, larvae were fed Vinegar Eels, Paramecia, and Caviar powder. After the 12 th day, water supply was started and fish were fed Brine Shrimp, Artemia, Paramecia and Vinegar Eels. Adult fish (>1 Month) were fed Artemia and the dry flakes.

#### Zebrafish fin clips

Adult fish were anesthetized with buffered Tricaine (1X, see section \@ref(mat-buff)) until loss of motion. About 1/3 of the caudal fin was cut with a sterile scalpel in a sterile Petri Dish. Immediately the cut fin was transferred to 100 $\mu$L of 50 mM NaOH. Fish were returned to system water and kept in 1L system water in single tanks with 200 $\mu$L of 0.01$\%$ Methylene Blue.

#### Adult Genotyping

The clipped fins were digested for 1 h at 95$^\circ$ C and neutralized subsequently with 10 $\mu$L of 1M Tris-HCl of (pH 9).

#### Embryo Genotyping

Single fixed/live embryos were denatured at 95$^\circ$ C in 20 $\mu$L of 50 mM NaOH for 1 hour and neutralized by adding 2.5 $\mu$L of 1 M Tris-HCl (pH 9).

#### Zebrafish Euthanasia

Adult zebrafish were euthanised by an overdose of Tricaine in ice cold water so as to sacrifice them by hypothermia.

#### Fixation

Embryos and dechorionated larvae were fixed in 2 ml of 4$\%$ PFA in 1X PBS overnight at 4$^\circ$ C.

### Wet lab {#Wet-met}

#### Sample preparation {#sampleprep}

For samples **older than 24 hpf**, embryos were grown in 1X PTU till desired stage and treated with 150 $\mu$L per 10 mL of 0.1 mg/mL Pronase for ~30 min.. Choria were removed by gentle pipetting with a 2 mL plastic pasteur pipette. To replace the Pronase solution with fresh E3, embryos were immobilized by anesthesia and collected in the center of the dish by gentle rotational movement. Then the medium was poured away by collecting the embryos at the corner bottom of the dish while taking care not to loose any. After, the dish was filled with fresh E3. This process was repeated three times.

For samples **younger than 18s**, embryos were treated with 150 $\mu$L per 10 mL of 0.1 mg/mL Pronase directly. Choria were removed the same way as for > 24 hpf embryos but when pouring away the Pronase solution, the dish was simultaneously and very carefully filled with fresh E3 again. Since younger embryos are more fragile and to avoid damage, they must be kept in solution constantly.

Fixation started at the desired stage in 4$\%$ PFA in 0.1$\%$ PBST in 2 mL tubes at 4$^\circ$C o.n.. The next day, samples were rinsed 3 times for ~5 min. in PBST and passed through a MeOH series of 25$\%$ $\rightarrow$ 50$\%$ $\rightarrow$ 75$\%$ $\rightarrow$ 100$\%$ MeOH/PBST (V/V)). For permanent storage, samples were stored in 100$\%$ MeOH at -20$^\circ$C.

#### In Situ Hybridization {#ISH-met}

Samples were prepared after the method described in section \@ref(sampleprep).

##### 1. Permeabilisation & Probe Hybridization

**Permeabilisation** $\rightarrow$ without shaking \newline

Samples were rehydrated in an inverse MeOH series of 75$\%$ $\rightarrow$ 50$\%$ $\rightarrow$ 25$\%$ $\rightarrow$ 0$\%$ PBST and washed again for 5 min. two times in pure PBST. Finally, samples were digested in 10 $\mu$g/mL Proteinase K according to table \@ref(tab:met-protk).

```{r met-protk}
read.delim("figures/materials/protkdig.txt") %>%
knitr::kable(booktabs = T, caption = "Proteinase K digestion", align = "c",
             col.names = c("Stage", "0.6 s", "7 s", "18 s", "24 hpf", "32 hpf", "36 hpf", "42 hpf", "48 hpf", "72 hpf")) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position'))
```
    
Samples were rinsed again two times in PBST and post-fixated in 4$\%$ PFA at 4$^\circ$C for > 30 min. Samples were washed again for 5 min. three times in PBST.

**Probe Hybridisation** $\rightarrow$ all steps at 60$^\circ$C, except stated differently \newline

Samples were pre-hybridized in 350 $\mu$L of hybridization buffer (section \@ref(mat-buff)) for 1 - 8 h. Just before detection probe treatment, the probe was denatured at 80$^\circ$C in 1:200 of hybridization buffer. Subsequently, hybridization buffer was taken off the samples and replaced with the heated probe. Finally, samples were incubated o.n. at a desired temperature (around 65$^\circ$C).

##### 2. Probe removal & Antibody incubation

The next day the probe got collected and stored at -20$^\circ$C for re-use. Washing took place at the same temperature as hybridization (from step 1) To keep solutions at temperature a Thermo-Block was used. For the washing series the samples were first washed one time for 20 min. in hybridization buffer, then two times for 30 min. in 50$\%$ Formamide and one time for 20 min. in 25$\%$ Formamide. Then two times for 15 min. in 2X SSCT and two times for 30 min. in 0.2X SSCT. Finally, one time for 5 min. in TNT.

To reduce noise and increase specific signal strength, the samples were treated with blocking solution (section \@ref(mat-buff)) for 1 - 8 h in 350 $\mu$L of 2% BR/TNT at room temperature (RT). Afterwards the samples were incubated in 100 $\mu$L Anti-Digoxigenin diluted in NTMT buffer (1/4000 (V/V)) in 2$\%$ BR/TNT for 2 h at RT or o.n. at 4$^\circ$C.

##### 3. Probe detection

First, the samples were washed six times for ~20 min. (or one wash o.n.) in TNT and two times for ~ 5min. in NTMT. After washing, color staining was performed with 4.5 NBT $\mu$L + 3.5 $\mu$L BCIP per mL NTMT in the dark and at RT without shaking (in a drawer) for 2 - 8 h, regularly checking the progression of the reaction. As soon as an appropriate degree of color intensity on the target site was achieved (up to two days), the samples were again washed three times in PBST. 

Afterwards the samples were either prepared for immunostaining or imaging. For permanent storage samples were kept in 50$\%$ Glycerol at 4$^\circ$C.

#### Immuno staining {#immuno-met}

Samples were prepared after the method described in section \@ref(sampleprep).

First, samples were blocked in 2$\%$ Goat Serum / PBDT (V/V) for 30 min.. For protein target site detection, a **primary antibody** (150 $\mu$L of 2% NGS / PBDT (V/V)) was incubated for ~2 h at RT or o.n. at 4$^\circ$C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times. To stain the now bound primary antibody, a **secondary antibody** (150 $\mu$L of 2% NGS / PBDT (V/V)) was incubated for 2 h at RT or o.n. at 4$^\circ$C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times.

#### Mounting {#mount-met}

For live microscopy zebrafish embryos are usually immersed in a 1$\%$ solution of low melting-point Agarose (LMPA) solution and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount eight to ten embryos per dish. 

To take advantage of the high number of offspring a single zebrafish female may lay, a new sample preparation technique was designed that allows for 

1. a four to five times increase in samples per dish 
2. a facilitated navigation _via_ a grid-like orientation through the samples and 
3. an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope.

A detailed protocol can be found under section \@ref(res-mount)

### Dry lab {#comp-met}

#### Image J macros

Three IJ macros have been developed to facilitate image analysis and make results more reproducible. Each of them is specifically designed for input of LL and pLLP images of the _cldnb:lyn-gfp_ transgenic line.

Hence, they are called **_anaLLzr_** ...

- **2D** - analysis of Z-projected images of the LL at end of migration
- **2DT** - analysis of Z-projected images of the LL during migration
- **3D** - analysis of 3D image stacks of the pLLP at a given timepoint

Since their development was an integral part of my PhD work, the description of the macros can be found in the results part in section section \@ref(res-met).

#### Proliferation Analysis{#prolif}

The basic principle is based on work done by Laguerre _et al._, 2009[@Laguerre2009a]. For registration of mitotic events an IJ manual tracking tool was used that allows to track an image feature through a stack of images creating tracks as it progresses through volume / time ('MTrackJ'[@Meijering2012]).

For mounting the embryo, the procedure described in section \@ref(mount-met) was used. Nuclei were visualized in a _TgBAC(cxcr4b:H2B-RFP)_ transgenic line. After Z-projection of volumetric timelapses, mitotic events were tracked in each CC and the pLLP. Afterwards the data was exported as one table per embryo and processed by counting mitoses per pLLP / CC / total CC mitoses. Figure \@ref(fig:mitodatapoints) shows an exemplary track for the data analyzed.

(ref:mitodatapoints) Tracking of mitotic events. T1-T3 show consequetive timepoints of a single nucleus.

```{r mitodatapoints, out.width = '60%', fig.cap = "(ref:mitodatapoints)", fig.scap = "Tracking of mitotic events", fig.pos = 'H'}
knitr::include_graphics("figures/materials/prol/Prolif.png")
```

#### Apical Index{#ACI}

##### Rationale

The earliest attempt found for indexing AC can be found in a study published by Lee _et al_[@Lee2009] where they were interested in the 'apical index' (A.I.) of bottle cells during _X.laevis_ gastrulation (figure \@ref(fig:ACLee) Lee). Another example for measuring AC is the _apical constriction index_ (A.C.I., figure \@ref(fig:ACLee) Harding) for the cells of the _D.rerio_ lateral line primordium (pLLP), which can be found in a study from 2012 where it was shown that FGFr-Ras-MAPK signaling is required for Rock2a localization and AC [@Harding2012; @Harding2013].

(ref:ACLee) A.I. indeces in the literature. **Lee** A.I. of _X.leavis_ bottle cells measured in 2D. **Harding** A.I. of _D.r._ pLLP cells measured in 3D.

```{r ACLee, out.width = '75%', fig.cap = "(ref:ACLee)", fig.scap = "A.I. indeces in the literature", fig.pos = 'H'}
knitr::include_graphics("figures/materials/models/ai.png")
```

\noindent In these two publications, the way they measure A.I. [@Lee2009] and ACI [@Harding2013] respectively, does not differ and is the ratio of lateral height over apical width. 

\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{apical\;width\;[\mu m]}\]

\noindent We found two principal weaknesses of applying this ratio to the cells of the pLLP. First, it does not respect the independence of lateral height to AC. Second, it does not differentiate between constriction along the _anterio-posterior_ (AP) or the _dorso-ventral_ (DV) axis. Third, it actually represents the A.I. rather than the apical _constriction_ index.

###### Parameter definition {#ACI-param}

To obtain a precise and biologically meaningful way to quantify AC, first a couple of definitions had to be made.

```{definition, name = "AC is independent of orientation", echo = TRUE}
In a 3D space a cell can have any orientation and still be apically constricted. Therefore, before measuring one should make sure orientation between embryos is aligned and also consider taking measurements along two different directions. Since apical constriction is not necessarily isotropic, it is important to consider constriction along 2 perpendicular axes (AP and DV axis of the embryo/pLLP).
```

```{definition, name = "AC is independent of lateral height", echo = TRUE}
Lateral height can be described as the distance of the two farthest points on the surface area of a cell. Two cells with different lateral heights can be equally apically constricted.
```

```{definition, name = "AC is independent of cell volume", echo = TRUE}
The volume of a cell represents its size. A large cell can be equally constricted as a small cell.
```

###### Adaption for variation in lateral height {#ACI-lat}

To test different A.I. conditions, an apically constricted cell can be approximated by modeling a tetrahedron. For example, shrinking or enlarging a cell symmetrically should not affect the A.I.. As described by Harding(2014)[@Harding2013], the _apical width_ of a cell is measured first by manual 3-D reconstruction, second manual re-orientation, and third by going 1 $\mu$m from the apical tip into the cell (from now on referred to as $\Delta$ap, \@ref(fig:ACICells)B). Finally, _apical width_ is the total width of the 2D object in the respective volume.

If $\Delta$ap is a constant, the A.I. in a symmetrically enlarged cell increases from e.g. ~15 to ~23, since _apical width_ stays the same but lateral height increases (compare figure \@ref(fig:ACICells)A to A'). On the contrary, if $\Delta$ap is adjusted relative to a cells lateral height, e.g. by percentage, the A.I. in a symmetrically enlarged cell stays the same (compare figure \@ref(fig:ACICells)A to A'').

(ref:ACICells) Different ways to quantify the apical index. **A-A’’** A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta is proportional to the lateral height. **B** Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2-D area as seen at $\Delta$ap of 1 or 2.5 $\mu$m.

```{r ACICells, out.width = '85%', fig.cap = "(ref:ACICells)", fig.scap = "Different ways to quantify the apical index", fig.pos = 'H'}
knitr::include_graphics(path = "figures/summary/aci_fig-01.png")
```

Therefore the measurement for apical width has to be relative to lateral height.

\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{relative\;apical\;width\;[\mu m]}\]

###### Adaption for tissue polarization {#ACI-pol}

Organs develop in a 3-D space and are polarized along each axis. AC usually describes a 2-D morphogenetic movement towards a center along the X-Y axes. However, the constriction movements along X and Y might be independent of one another. This could mean that they happen at different speeds, or that one is absent. As a result, the tissue would look less radially (figure \@ref(fig:cellpol)B) constricted, but more constricted along one particular axis (anisotropic). In order to separate those two AC dimensions, the A.I. can be calculated for the _anterio-posterior_ and for the _dorso-ventral_ axis (figure \@ref(fig:cellpol), horizontal vs. vertical).

(ref:cellpol) Schematic AC along the A-P and D-V axis. **A** shows a A-P and D-V constricted cluster of cells. **B** shows a D-V constricted cluster of cells.

```{r cellpol, out.width = '75%', fig.cap = "(ref:cellpol)", fig.scap = "Schematic anisotropic AC", fig.pos = 'H'}
knitr::include_graphics("figures/materials/models/ACI_Cells_pol.png")
```

\noindent By fitting an ellipsoid to the area taken at $\mathrm{\Delta}$ap, one will obtain the following parameters (figure \@ref(fig:ellipse)).

1. Length of Major axis
    + indicates the level of constriction along the less constricted axis
2. Length of Minor axis
    + indicates the level of constriction along the most constricted axis
3. Angle of Major from 0$^\circ$
    + indicates the orientation of the long, less-constricted axis: If the angle is close to 0$^\circ$, the long axis of the apical area is parallel to the AP axis (the cell is constricted along the DV axis). If the angle is close to 90$^\circ$, the long axis of the apical area is parallel to the DV axis (the cell is constricted along the AP axis).

(ref:ellipse) Scheme of ellipsoid measures. **A** shows the major axis as apical width and the minor axis as apical height. **B** shows the angular displacement from the horizon in steps of 30$^\circ$.

```{r ellipse, out.width = '75%', fig.cap = "(ref:ellipse)", fig.scap = "Scheme of ellipsoid measures", fig.pos = 'H'}
knitr::include_graphics("figures/materials/models/ellipse.png")
```

###### Measurement definition {#aci-theorem}

The two dimensions of A.I. indices can therefore be defined as the following ratios...

```{definition, name = "A.I. Major", echo = TRUE}
$$\mathbf{A.I._{Major}} = \frac{lateral\;height\;[\mu m]}{ellipsoid\;major\;axis\;at\;relative\;\Delta ap\;[\mu m]}$$
```

```{definition, name = "A.I. Minor", echo = TRUE}
$$\mathbf{A.I._{Minor}} = \frac{lateral\;height\;[\mu m]}{ellipsoid\;minor\;axis\;at\;relative\;\Delta ap\;[\mu m]}$$
```

```{definition, name = "Angle Major", echo = TRUE}
$$\mathbf{Angle_{Major}} = \measuredangle = \mathrm{\Delta}\;from\;horizon\;[0-90^\circ]$$
```

##### Measurements {#ACI-Dis}

As a proof of principle of the definitions stated in the previous section we compare our results to previously published results from Harding _et al._ [@Harding2012] who, as a control, measured apical constriction in embryos treated with an FGF inhibitor (SU5402) and their DMSO controls.   

###### Single Cell measurements {#ACI-singlecell}

Each geometric object has a centroid coordinate in X and Y (and Z) which is represented as the mean of all X or Y coordinates within the object. In figure \@ref(fig:ACICells), centroid coordinates in X and Y are used to plot the cells as points in the X-Y plane. Additionally, each point is colored for the A.I. value (high values are dark red - red, middle values are green, low values are cyan - blue).

(ref:ACIMinor) A.I.\textsubscript{Major / Minor} single cell measurements

```{r aci-single-cell-measures-graph, out.width = '95%', fig.cap = "(ref:ACIMinor)", fig.pos = 'H'}
load("data/SN180_ACI.RData")

# row bind dmso and su data
  SN180 <- rbind(SN180_DMSO, SN180_SU)
  
# calculate aci
  
  SN180 <-
    SN180 %>%
    select(
      CXN,
      CYU = CY..unit.,
      GT,
      feret = Feret..unit.,
      apFMax,
      apFMin
    ) %>%
    mutate(
      ACI_Major = feret / apFMax,
      ACI_Minor = feret / apFMin
    )

# pivotize  
  SN180_long <-
    SN180 %>% 
    as_tibble() %>% 
    pivot_longer(
      cols = c(ACI_Minor, ACI_Major)
      ) %>%
    mutate(
      name =
      case_when(
        name == 'ACI_Minor' ~ 'A.I.[Minor]',
        name == 'ACI_Major' ~ 'A.I.[Major]'
        )
    ) %>%
    #group_by(GT, name) %>%
    filter(
      value < quantile(.$value, .95),
      !CXN < quantile(.$CXN, .01) & !CXN > quantile(.$CXN, .99)
    ) 

# summaries

SN180_summary <-
  SN180_long %>% 
  group_by(GT, name) %>% 
  summarize(
    min_x = min(CXN, na.rm = T),
    max_ac = max(value, na.rm = T),
    mean_ac = mean(value)
    )

# graph

ggplot(SN180_long, aes(CXN, CYU)) +
  geom_vline(
    data = SN180_summary,
    aes(xintercept = min_x, fill = "L.E."), 
    linetype = 2, size = .5, show.legend = TRUE) +
  geom_point(aes(colour = value), size = 1.5, shape = 16) +
  labs(title = "", 
       caption = 'L.E. = leading edge',
       x = "X centroid normalized to leading edge [µm]", 
       y = "Y centroid [µm]") +
  #scale_colour_gradientn(colours = rev(rainbow(5))) +
  scale_colour_gradientn(colours = jet.colors(5)) +
  facet_grid(name~GT, labeller = label_parsed) +
  scale_y_continuous(limits = c(-10, 55)) +
  coord_fixed() +
  mythemeLIGHT_bottom() +
    theme(
      legend.title = element_blank(),
      strip.background = element_blank(),
      strip.text = element_text(size = rel(1), face = 'bold'),
      axis.title.x = element_text(size = rel(1)),
      axis.title.y = element_text(size = rel(1)),
      axis.text.x = element_text(size = rel(1)),
      axis.text.y = element_text(size = rel(1))
      )
```

```{r aci-sum-stats, include = FALSE}
load("data/SN180_ACI.RData")

# row bind dmso and su data
  SN180 <- rbind(SN180_DMSO, SN180_SU)
  
# calculate aci
  
  SN180 <-
    SN180 %>%
    select(
      CXN,
      CYU = CY..unit.,
      GT,
      feret = Feret..unit.,
      apFMax,
      apFMin
    ) %>%
    mutate(
      ACI_Major = feret / apFMax,
      ACI_Minor = feret / apFMin
    )

SN180_summary <-
SN180 %>% 
  as_tibble() %>% 
  pivot_longer(
    cols = c(ACI_Minor, ACI_Major)
    ) %>%
  mutate(
    name =
    case_when(
      name == 'ACI_Minor' ~ 'ACI[Minor]',
      name == 'ACI_Major' ~ 'ACI[Major]'
      )
  ) %>%
  group_by(GT, name) %>% 
  summarize(
    min_x = min(CXN, na.rm = T),
    max_ac = max(value, na.rm = T),
    mean_ac = mean(value)
    )
```

Harding _et al._ [@Harding2013] were using a constant $\mathrm{\Delta}$ap to measure the apical width, which we have shown to be incorrect in certain cases. In their study they found that certain mean A.C.I. values in the DMSO go as high as 15 (figure \@ref(fig:HardingACI)), which might be related to this (see figure \@ref(fig:ACICells)). By measuring apical width at a relative $\mathrm{\Delta}$ap and taking into account all pLLP cells of the two exemplary pLLPs shown in figure \@ref(fig:ACICells), we measure a mean difference in the Major of `r round(SN180_summary %>% filter(GT == 'DMSO', name == 'ACI[Major]') %>% pull(mean_ac) - SN180_summary %>% filter(GT == 'SU5402', name == 'ACI[Major]') %>% pull(mean_ac), 2)` and `r round(SN180_summary %>% filter(GT == 'DMSO', name == 'ACI[Minor]') %>% pull(mean_ac) - SN180_summary %>% filter(GT == 'SU5402', name == 'ACI[Minor]') %>% pull(mean_ac), 2)` in the Minor.

(ref:HardingACI) A.I. indices by Harding et al. **E-G'** 3-D reconstructions of the highlighted cell. **H** A.C.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of _hsp70:dn-Ras_. (n = 180 cells / N = 6 embryos).

```{r HardingACI, out.width='60%', fig.cap = "(ref:HardingACI)", fig.scap = "A.I. indices by Harding et al.", fig.pos = 'H'}
knitr::include_graphics("figures/materials/models/harding.png")
```

###### Angle densities {#ACI-Angledens}

To check whether there is a bias in orientation of the apical width, the angle measurements \@ref(fig:ellipse) can be shown as a function of density along X (figure \@ref(fig:angletoACI)A).

Interestingly the results indicate that there is less of a difference for the Major~Angle~ at angles bigger than 15-20$^\circ$. This would mean that the apical surface of the cells in SU5402 treated embryos is more strongly oriented along the horizontal _antero-posterior_ axis.

###### ACI magnitude at different Angles {#ACI-mag}

Now, to get an idea of the magnitude of constriction relative to the orientation of the cell (angle to the horizontal), the A.I.~Major/Minor~ can be shown as a function of the Major~Angle~ (figure \@ref(fig:angletoACI)B-B').

\noindent Since AC is a 3-D morphogenetic process and since cells in a wild type pLLP are mostly radially organized, it does make sense to try to look at AC from more than just one perspective. Here we propose to separate the A.I. into an _antero-posterior_ and a _dorso-ventral_ dimension.

1. While for the control (DMSO treated) embryo the distribution of the cells Major Angles seem to be mostly uniform, for the SU5402 treated embryo there is an accumulation of lower Major Angles. This means that cells in SU5402 treated embryos are more oriented along the horizontal (anterior - posterior) axis.

2. Interestingly there does not seem to be much of a difference in A.I.~Major~ (figure \@ref(fig:angletoACI)B), which can also be shown by the mean values which are at `r round(mean(SN180_DMSO$ACI_V_F20), digits = 1)` for the DMSO control and at `r round(mean(SN180_SU$ACI_V_F20), digits = 1)` for the SU5402 treated condition. 

3. For the A.I.~Minor~ (figure \@ref(fig:angletoACI)B') the means are `r round(mean(SN180_DMSO$ACI_H_F20), digits = 1)` for the DMSO control and `r round(mean(SN180_SU$ACI_H_F20), digits = 1)` for SU5402. The base constriction for both, DMSO and SU5402 is at around 3.6, however there is a peak at around 40 - 60$^\circ$ in the DMSO control where cells are most constricted having a maximum A.I. at `r round(max(SN180_DMSO$ACI_H_F20), digits = 1)`. This indicates that for the Minor, cells in that range of angles are more constricted than cells oriented in different directions.

(ref:angletoACI) A.I.~Major~ / A.I.~Minor~ over Major~Angle~

```{r angletoACI, out.width = '49%', fig.width = 5, fig.height = 2.9, fig.pos = "H", fig.cap = "(ref:angletoACI)", fig.keep = 'all', fig.show = 'hold'}

# row bind dmso and su data
  SN180 <- rbind(SN180_DMSO, SN180_SU)
  
# calculate aci
  
  SN180 <-
    SN180 %>%
    select(
      CXN,
      CYU = CY..unit.,
      GT,
      feret = Feret..unit.,
      apFMax,
      apFMin,
      AI_Angle = apAngle
    ) %>%
    mutate(
      ACI_Major = feret / apFMax,
      ACI_Minor = feret / apFMin
    )

# Major Angle KDE

ggplot(SN180, aes(AI_Angle)) +
  stat_density(aes(group = GT), geom = "area", bw = 3, size = 1.5, position = "identity", fill = "black", alpha = .1) +
  stat_density(aes(colour = GT), geom = "line", bw = 3, size = 1.5, position = "identity", span = .5) +
  scale_x_continuous(breaks = seq(0, 90, 15)) +
  labs(
    title = "",
    caption = 'curve = kernel density estimate',
    x = "Major Angle", 
    y = "Density") +
  coord_cartesian(
    xlim = c(5, 82), 
    ylim = c(0.002, 0.035)
    ) +
  annotate("text", x = 5, y = 0.033, label = "A", size = 7) +
  mythemeLIGHT_bottom() + 
    theme(
      strip.background = element_blank(),
      legend.title = element_blank(),
      legend.position = c(0.85, 0.85),
      axis.title = element_text(size = rel(1.2)),
      axis.text = element_text(size = rel(1.0))
      )

# Major Angle over ACI Major and Minor

ann_text <- 
  data.frame(
    value = c(6.2, 6.2),
    Angle = c(8, 8),
    lab = c('B','B\''), 
    A.I. = factor(c("A.I.[Minor]", "A.I.[Major]"))
    )

SN180_long <-
  SN180 %>%
  select(
    Angle = AI_Angle,
    ACI_Major,
    ACI_Minor,
    GT
  ) %>%
  rename(
    `A.I.[Major]` = ACI_Major,
    `A.I.[Minor]` = ACI_Minor
  ) %>%
  pivot_longer(
    c(`A.I.[Major]`, `A.I.[Minor]`), names_to = 'A.I.'
  )

ggplot(SN180_long, aes(Angle, value)) +
  stat_smooth(aes(colour = GT), geom = "line", se = FALSE, method = "loess", span = .5, size = 1.5) +
  scale_x_continuous(breaks = seq(0, 90, 25)) +
  labs(
    title = "",
    caption = 'curve = local polynomial regression',
    x = "Major Angle", 
    y = "A.I.") +
  scale_x_continuous(limits = c(0, 90), breaks = seq(15, 75, 15)) +
  scale_y_continuous(breaks = seq(2, 7, 1)) +
  coord_cartesian(xlim = c(5, 82)) +
  geom_text(data = ann_text, label = c('B\'', 'B'), size = 7, face = 'bold') +
  facet_grid(.~A.I., labeller = label_parsed) +
  mythemeLIGHT_bottom() + 
    theme(
      strip.background = element_blank(),
      legend.title = element_blank(),
      legend.position = c(0.375, 0.85),
      strip.text = element_text(size = rel(1.4), face = 'bold'),
      axis.title = element_text(size = rel(1.2)),
      axis.text = element_text(size = rel(1.0))
      )
```

## Ground Truth {#mat-GrTrDat}

Analyzing images and extracting quantitative measurements can be a tedious task, especially when the analysis becomes more complex. Fortunately, there are ways to automate image analysis by using either machine learning approaches or by tailoring hand-crafted algorithms in an image analysis software tool like e.g. ImageJ[@Schindelin2012]. 
The main advantages of doing so are to...

-	be independent of confirmation bias
-	make the analysis more robust against oversight
-	increase the statistical power by increasing the number of data points[@Button2013]
-	increase effect size by increasing the measurement accuracy[@Button2013]

However, to ensure the measurements taken by a tailored or trained algorithm are meaningful, they must be compared to a _ground truth_ dataset which again describes a general measure of algorithmic quality performance[@Krig2014].

### Cluster Analysis

The _anaLLzR2D_ algorithm was designed for semi-automatic cell cluster detection in the _cldnb:lyn-gfp_ transgenic line and optional nuclei counting in a second DAPI labeled channel within the Regions of Interest (ROIs) derived from the cell cluster detection.

####  Design

To assess the quality of the _anaLLzR2D_ algorithm for nuclei detection the ground truth was designed as follows.

##### Model

-	each Cell Cluster (CC) consists of a number of objects (cells)
-	each object is part of the respective CC and defines one cell entity
-	each object is determined via a fluorescent nucleus label
- embryos are mounted within a 3D mold (section \@ref(mount-met)) to reduce noise

---------------------------------- ----------------------------------
                          XY scale 0.32 px/$\mu$m
 
                         Z-spacing 4 $\mu$m
 
                            Camera Rolera
---------------------------------- ----------------------------------
Table: (\#tab:imgcond3DGrT) Cluster Analysis Model

\pagebreak

##### Training & test data

The training set consists of three randomly picked wild type pLLs. For each the algorithm was run with standard parameters. Cell cluster ROIs and nuclei multi-point labels were edited manually. To test the algorithm, it is run at different nuclei detection thresholds on the same image data.

### Morphometric analysis

The _anallzr3D_ algorithm was designed for fully automated, single cell volume segmentation in the _cldnb:lyn-gfp_ transgenic line. In addition to 3D cellular metrics, it offers Apical Constriction measurement of each cell.

#### Design

To assess the quality of the anaLLzr3D algorithm the ground truth was modeled as follows.

##### Model

-	each pLLP consists of a number of objects (cells)
-	each object is part of the pLLP and defines one cell entity
-	cell boundaries are determined via the transgene _cldnb:lyn-gfp_ +/+
-	embryos are imaged live to conserve signal and membrane integrity
- embryos are mounted within a 3D mold for improved 3D alignment

```{r model3DGrT}
tab <- read.delim("tables/ground_truth/anallzrmodel.txt")
knitr::kable(tab, booktabs = T, caption = "anaLLzr3D Model", align = c("r", "l"), escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  column_spec(1, width = "4cm")
```

##### Training & test data

The training set consists of three randomly picked wildtype pLLPs. For each the algorithm is run with no filters (X, Y, Z border objects, size) and a minimum segmentation threshold. Afterwards the segmentation result is corrected manually for over- and under-segmentation and objects that are not part of the pLLP.

To test the algorithm it is run at different segmentation thresholds on the same image data.

## Image Data Sets {#mat-datasets}

Summaries of Image datasets. _Pairs_ describe the number of parent pairs I harvested eggs from. _Stage_ describes the time I waited for the parent pairs to mate and lay eggs. Since pair #1 might have laid their eggs earlier than pair #2, those batches would be some time apart in their _staging_. _Stamp_ describes the version of the stamp I used, where the main difference between version 4 and 5 are more wells added and some minor upgrades in well-design.

```{r imgdatcc}
tab <- read.delim("tables/img_data/imgdat_cc.txt")
knitr::kable(tab, booktabs = T, caption = "Cell Cluster dataset", escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  collapse_rows(columns = 1, latex_hline = "none", valign = "top") %>%
  column_spec(1, width = "2cm") %>%
  column_spec(2, width = "2.5cm")
```

```{r imgdatai}
tab <- read.delim("tables/img_data/imgdat_ai.txt")
knitr::kable(tab, booktabs = T, caption = "A.I. dataset", escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  collapse_rows(columns = 1, latex_hline = "none", valign = "top") %>%
  column_spec(1, width = "2cm") %>%
  column_spec(2, width = "2.5cm") 
``` 

```{r imgdatprol}
tab <- read.delim("tables/img_data/imgdat_prol.txt")
knitr::kable(tab, booktabs = T, caption = "Proliferation dataset", escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  collapse_rows(columns = 1, latex_hline = "none", valign = "top") %>%
  column_spec(1, width = "2cm") %>%
  column_spec(2, width = "2.5cm")
```

```{r imgdatdet}
tab <- read.delim("tables/img_data/imgdat_det.txt")
knitr::kable(tab, booktabs = T, caption = "Detection dataset", escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  collapse_rows(columns = 1, latex_hline = "none", valign = "top") %>%
  column_spec(1, width = "2cm") %>%
  column_spec(2, width = "2.5cm")
```

```{r imgdatato}
tab <- read.delim("tables/img_data/imgdat_ato.txt")
knitr::kable(tab, booktabs = T, caption = "Atoh1a dataset", escape = F, col.names = NULL) %>%
  kable_styling(full_width = T, latex_options = c('striped', 'hold_position')) %>%
  collapse_rows(columns = 1, latex_hline = "none", valign = "top") %>%
  column_spec(1, width = "2cm") %>%
  column_spec(2, width = "2.5cm")
```
