<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="Chapter 1 Introduction | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.13.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="list-of-abbreviations.html"/>
<link rel="next" href="2-mat-met.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="list-of-abbreviations.html"><a href="list-of-abbreviations.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#morphogenesis"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#objective-and-open-questions"><i class="fa fa-check"></i><b>1.3</b> Objective and Open Questions</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-Intro.html"><a href="1-Intro.html#motivation"><i class="fa fa-check"></i><b>1.3.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-Intro.html"><a href="1-Intro.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>1.4</b> Data Strategy and Analysis</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-Intro.html"><a href="1-Intro.html#sample-preparation"><i class="fa fa-check"></i><b>1.4.1</b> Sample Preparation</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-Intro.html"><a href="1-Intro.html#imaging"><i class="fa fa-check"></i><b>1.4.2</b> Imaging</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-Intro.html"><a href="1-Intro.html#data-handling"><i class="fa fa-check"></i><b>1.4.3</b> Data handling</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-Intro.html"><a href="1-Intro.html#image-and-data-analysis"><i class="fa fa-check"></i><b>1.4.4</b> Image and Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.9</b> Hardware</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.10</b> Software</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.1.11</b> Ground Truth Data</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.1</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.2</b> Wetlab Methods</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.3</b> Computational Methods</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.3</b> Image Data Sets</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cc-data"><i class="fa fa-check"></i><b>2.3.1</b> Cell-Cluster</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#aci-data"><i class="fa fa-check"></i><b>2.3.2</b> Apical Index</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-mat-met.html"><a href="2-mat-met.html#tl-data"><i class="fa fa-check"></i><b>2.3.3</b> Proliferation</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-mat-met.html"><a href="2-mat-met.html#detect-data"><i class="fa fa-check"></i><b>2.3.4</b> Detection</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-ato-set"><i class="fa fa-check"></i><b>2.3.5</b> Atoh1a</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.1</b> Lateral Line Morphometrics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#dataset"><i class="fa fa-check"></i><b>3.1.1</b> Dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#res-ccounts"><i class="fa fa-check"></i><b>3.1.2</b> Number and Position of Cell Clusters</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#res-llmorph"><i class="fa fa-check"></i><b>3.1.3</b> Cell Count and Area of CCs</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2</b> Proliferation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#dataset-1"><i class="fa fa-check"></i><b>3.2.1</b> Dataset</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#res-prolpllp"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Mitoses</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.3</b> Rosette Formation and Cluster Deposition</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-res.html"><a href="3-res.html#dataset-2"><i class="fa fa-check"></i><b>3.3.1</b> Dataset</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-res.html"><a href="3-res.html#cluster-deposition"><i class="fa fa-check"></i><b>3.3.2</b> Cluster Deposition</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-res.html"><a href="3-res.html#registration-1"><i class="fa fa-check"></i><b>3.3.3</b> Registration</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-res.html"><a href="3-res.html#res-rdtdet"><i class="fa fa-check"></i><b>3.3.4</b> Rosette Detection</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-res.html"><a href="3-res.html#correlations"><i class="fa fa-check"></i><b>3.3.5</b> Correlations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-res.html"><a href="3-res.html#apical-constriction-1"><i class="fa fa-check"></i><b>3.4</b> Apical Constriction</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-res.html"><a href="3-res.html#dataset-3"><i class="fa fa-check"></i><b>3.4.1</b> Dataset</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-res.html"><a href="3-res.html#cell-shape-and-arrangement"><i class="fa fa-check"></i><b>3.4.2</b> Cell Shape and Arrangement</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-res.html"><a href="3-res.html#single-cell-metrics"><i class="fa fa-check"></i><b>3.4.3</b> Single Cell Metrics</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.5</b> Haircell Specification</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-res.html"><a href="3-res.html#dataset-4"><i class="fa fa-check"></i><b>3.5.1</b> Dataset</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-res.html"><a href="3-res.html#atoh1a-in-the-ll"><i class="fa fa-check"></i><b>3.5.2</b> Atoh1a in the LL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-res.html"><a href="3-res.html#rescue-experiments"><i class="fa fa-check"></i><b>3.6</b> Rescue Experiments</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-res.html"><a href="3-res.html#res-shrmresc"><i class="fa fa-check"></i><b>3.6.1</b> Shroom3 Ectopic Expression</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-res.html"><a href="3-res.html#morphogen-rescue"><i class="fa fa-check"></i><b>3.6.2</b> Morphogen Rescue</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-res.html"><a href="3-res.html#res-rockresc"><i class="fa fa-check"></i><b>3.6.3</b> Mechanic Rescue</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-res.html"><a href="3-res.html#res-tempresc"><i class="fa fa-check"></i><b>3.6.4</b> Thermodynamic Rescue</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-res.html"><a href="3-res.html#atoh1a-knockdown"><i class="fa fa-check"></i><b>3.6.5</b> atoh1a knockdown</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#adult-phenotype"><i class="fa fa-check"></i><b>4.1</b> Adult Phenotype</a></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#lateral-line"><i class="fa fa-check"></i><b>4.2</b> Lateral Line</a></li>
<li class="chapter" data-level="4.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.3</b> Rosettes</a></li>
<li class="chapter" data-level="4.4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#hair-cells"><i class="fa fa-check"></i><b>4.4</b> Hair Cells</a></li>
<li class="chapter" data-level="4.5" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-2"><i class="fa fa-check"></i><b>4.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.6" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.6</b> Model</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.1</b> Proliferation</a></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection"><i class="fa fa-check"></i><b>5.2</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.3" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification-1"><i class="fa fa-check"></i><b>5.3</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.4" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.4</b> Single cell reconstruction and AC measurement</a></li>
<li class="chapter" data-level="5.5" data-path="5-discussion.html"><a href="5-discussion.html#future-perspectives"><i class="fa fa-check"></i><b>5.5</b> Future perspectives</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>

<div id="development" class="section level2">
<h2><span class="header-section-number">1.1</span> Development</h2>
<p>In biology differentiation describes the continuous process of cells dividing and adapting to their ever-changing environment during advancing development, whereby adopting specialized function. Physically a cell represents an open system, which is defined as a unit system able for external interactions. Such a system is therefore not self-dependent and self-sustained, but its current conformation is determined by external interactions. To illustrate this, the cell can the thought of as a marble rolling down a furrowed landscape (figure <a href="1-Intro.html#fig:wadd">1.1</a>). In this landscape a hill would be a high energy-, a valley a low energy state. The path the marble will take is determined by the furrows in the landscape, since it would always prefer a valley. However, this landscape is not a static structure but is a one that changes at every instant of time.</p>
<div class="figure" style="text-align: center"><span id="fig:wadd"></span>
<img src="figures/intro/waddington.jpg" alt="Waddington's Classical Epigenetic Landscape"  />
<p class="caption">
Figure 1.1: Waddington’s Classical Epigenetic Landscape
</p>
</div>
<p>For developmental biology, which as a scientific discipline originates from embryology, the central interest is to understand how inanimate matter can form such complex structures we see in living matter at different levels of organismal hierarchy while ensuring a robust developmental plan. Using the marble analogy, studying developmental biology can be thought of tracking multiple cells as they roll down the valley while observing and testing how interactions between them might change their fate. The basic questions arising from this interest are</p>
<ul>
<li>How do tissues arise from a population of cells?</li>
<li>How do organs form from tissues?</li>
<li>Why do organs form at their particular location?</li>
<li>How do migrating cells know whether they reached their destination?</li>
<li>How is growth controlled and how do body axes form?</li>
</ul>
<div id="cell-types" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Cell Types</h3>
<p>In animals there are two basic types of cells.</p>
<ol style="list-style-type: decimal">
<li>Epithelial cells, which can form strong bonds between each other and thereby are able to exert forces upon each other to achieve complex architectures.</li>
<li>Mesenchymal cells, which do not bond with each other and are more independent.</li>
</ol>
<p>This however describes only the extremes on a continuous scale. A cell is not a binary system but can show characteristics of both extremities, e.g. during Epithelial to Mesenchymal transition (EMT), a bidirectional process whereby epithelial cells can gain migratory and invasive properties and <em>vice versa</em>.</p>
<p>A cells identity on this continuum is determined by</p>
<ul>
<li>The cells genome, which reflects its repertoire of molecular machinery (proteins) and therefore determines its competence to react to internal and external cues.</li>
<li>The cells micro-environment, which has a <em>physical</em> (forces, energy) and a <em>chemical</em> (signaling molecules, diluents) dimension. A change in the latter usually brings about a reaction in the cell which becomes evident both in expression of genes, which represents the programmatic adaption, and morphologically, which represents the functional adaption.</li>
<li>The cells shape and incorporation into a tissue, which may modulate the cells affection to its micro-environment (e.g. certain regions of the cell can be more or less exposed, or it can be more or less tightly packed, tuning its susceptibility to forces and signals).</li>
</ul>
<p>With rising numbers of cells and differentiated tissue a shape and body axes begin to emerge that for the earliest developmental stages is highly similar across certain phyla and only begins to diversify at later developmental stages (figure <a href="1-Intro.html#fig:heak">1.2</a>), which reflects our evolutionary ancestry. This principle was first formulated by Ernst Haeckel as the biogenetic rule <span class="citation">(<em>1</em>)</span>, which states that ontogenesis (individual development) recapitulates <em>phylogenesis</em> (development of phylum)<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>.</p>

<div class="figure" style="text-align: center"><span id="fig:heak"></span>
<img src="figures/intro/haekel.jpg" alt="Ontogenesis across species (George Romanes, 1892)" width="0.75\textwidth" />
<p class="caption">
Figure 1.2: Ontogenesis across species (George Romanes, 1892)
</p>
</div>
</div>
<div id="morphogenesis" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Morphogenesis</h3>
<p>Morphogenesis, from the Greek morphê (shape) and genesis (creation)</p>
<blockquote>
<p>“Beginning of the shape”</p>
</blockquote>
<p>For objects that fulfill a purpose, many times (if not always) their forms (or shapes) are an expression of their function<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>. Analyzing the shape of an object can give one information about its function. It is therefore an important feature many different sciences, e.g. when correlating an organism’s size to the climate it lives or when correlating intelligence to the size of an animal’s brain. During an organism’s development diversification of shape on a cellular level is an important step in breaking symmetry of daughter cells. In order to form a tissue or an organ with a variety of specialized cells, it is important for the single cell to have information about where it is located, what its neighbors are doing, how densely it is packed and what the chemical composition of its surrounding is. This is accomplished by being in constant feedback with its neighbor cells and sensation of its environment. After the information got processed, the cell will react by adjusting the levels of proteins expressed to undergo proliferation or develop specialized structures like cilia or axons.</p>
<p>For a cell, its present shape impacts its further specification by impacting how well it perceives specific signals and the magnitude of signals received and transmitted. E.g. it has been shown that during lateral inhibition <span class="citation">(<em>4</em>)</span>. Shape therefore sets the general framework for cell-cell interaction and follows a</p>

<p>scale hierarchy, where each scale’s output again feeds back to the others (figure <a href="1-Intro.html#fig:feedb">1.3</a>). E.g. it has been shown that simple changes in cell geometry affect fundamental processes such as cell growth, death, or direction of cell divisions <span class="citation">(<em>5</em>–<em>7</em>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:feedb"></span>
<img src="figures/intro/feedback.png" alt="Form and function feedback loops" width="0.75\textwidth" />
<p class="caption">
Figure 1.3: Form and function feedback loops
</p>
</div>
<div id="apical-constriction" class="section level4">
<h4><span class="header-section-number">1.1.2.1</span> Apical Constriction</h4>
<p>A defining feature of Epithelial cells is they have a basal- (bottom) to apical (top) polarity. At the basal site, the cell is in contact with the substrate like the extracellular matrix (ECM), apically the cell forms tight connections to its neighbouring cells – a region called the apical junctional complex (AJC). The AJC encompasses three types of junctions: Adherent junctions (AJ) at the zonula adherens (ZA), tight junctions (TJ) and desmosomes. Around the ZA dense cables of actomyosin are found that, analog to muscle sarcomeres, are able to contract upon RHO‑associated protein kinase (Rock) mediated phosphorylation of the motor protein non-muscle myosin II (NMII) <span class="citation">(<em>8</em>)</span>.</p>
<p>Apical constriction (AC) is a single cell morphogenetic process manifest by an active apical narrowing, making the cell appear bottle or wedge shaped. It is usually coordinated by multiple cells within an epithelial layer that raise forces necessary to deform a tissue.</p>
<div class="figure" style="text-align: center"><span id="fig:arrange"></span>
<img src="figures/intro/arrangements.png" alt="Multi-cellular Arrangements" width="0.75\textwidth" />
<p class="caption">
Figure 1.4: Multi-cellular Arrangements
</p>
</div>
<p>Developmental processes that involve AC are…</p>
<ul>
<li>Tissue folding and tube formation</li>
<li>Single cell ingression and EMT</li>
<li>Gastrulation</li>
<li>Healing and sealing of embryonic tissue</li>
</ul>
<p>Epithelial rosettes are intermediate structures of radially organized cells within an epithelial tissue whose vertices interface a common center. While the mechanisms of cytoskeletal rearrangements seem to be well conserved, the extracellular cues that lead to rosette formation are less well understood and more diverse.
At least two architectural distinct types of rosettes exist, depending on the tissues polarization (figure <a href="1-Intro.html#fig:constr">1.5</a>). First, in a planar polarized tissue, several cells converge at a central apico-basal (A-B) line with no shrinkage of the apical surface to form a cylindrical structure. Such rosettes are usually observed during tissue elongation and rather short-lived. In a second scenario, cells converge to a central apical point through AC. This type of rosette is more long-lived and, usually does not resolve but already represents a morphologically pre-mature state of the organ to be formed <span class="citation">(<em>9</em>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:constr"></span>
<img src="figures/intro/constriction.png" alt="Modes of Constriction" width="0.75\textwidth" />
<p class="caption">
Figure 1.5: Modes of Constriction
</p>
</div>
</div>
</div>
<div id="cell-communication" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Cell communication</h3>
<p>To communicate with each other, cells have developed a variety of intercellular communication systems and a complex network of intracellular signal transduction pathways. Some information transfer depends on direct cell-to-cell contact, others rely on freely diffusible ligands that can be sensed by other cells. Each signaling pathway consists of some key regulators of a ligand – receptor pair that determines their main function. In the following, three pathways are introduced that play major roles in embryonic development.</p>
<div id="wnt-signal-transduction" class="section level4">
<h4><span class="header-section-number">1.1.3.1</span> WNT Signal transduction</h4>
<p>The word ‘WNT’ is a compound word of <em>Wingless</em> and <em>Int-1</em>, both of which are important genes during development of <em>Drosophila melanogaster</em> (commonly known as fruit fly), where WNT signaling was first studied. WNT is evolutionary conserved with 15 different receptors and co-receptors and plays a major role during embryonic axis formation, body segmentation, organogenesis and stem cell proliferation. Aberrant WNT signaling is involved in diseases like colon cancer, melanoma and neurogenerative diseases <span class="citation">(<em>10</em>)</span>.</p>
<p>Besides this WNT signaling is sub-divided in a canonical (<span class="math inline">\(\beta\)</span>-catenin<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> dependent) and a non-canonical (<span class="math inline">\(\beta\)</span>-catenin independent) branch. In canonical signaling, WNT ligand binds, together with co-receptor <em>Lipoprotein Receptor-related Protein</em> (LRP), to the receptor <em>Frizzled</em> (Frz) – jointly activating protein <em>Dishevelled</em> (Dsh). Dsh again inhibits a protein complex usually degrading <span class="math inline">\(\beta\)</span>-catenin, leading to an enrichment in the cytoplasm and the nucleus. Within the nucleus, <span class="math inline">\(\beta\)</span>-catenin forms a complex with Lef to activate specific target genes <span class="citation">(<em>10</em>)</span>.</p>
</div>
<div id="fibroblast-growth-factor" class="section level4">
<h4><span class="header-section-number">1.1.3.2</span> Fibroblast Growth Factor</h4>
<p>Key roles of <em>Fibroblast growth factor</em> (Fgf) signaling is mesoderm<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> patterning in the early embryo, regulation of angiogenesis and wound repair. On a cellular level it is also an important regulator of proliferation and differentiation. The mammalian Fgf family is comprised of 18 ligands and four highly conserved transmembrane tyrosine kinases named Fgf receptor 1-4 (Fgfr1-4). Aberrant Fgf signaling is <em>e.g.</em> associated with tumor growth <span class="citation">(<em>11</em>)</span>.</p>
<p>Upon ligand binding Fgfr dimerizes and undergoes a conformational shift activating the intracellular kinase<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> domain. Subsequent trans-phosphorylation of tyrosine kinase domains serve as docking sites for adaptor proteins. Activated Fgfr then phosphorylates Fgfr substrate 2 (FRS2), recruiting adaptor protein <em>Son of Sevenless</em> (SoS) and <em>Growth factor Receptor bound 2</em> (GRb2) to set on a cascade of kinase dependent signal transduction eventually leading to activation of target genes <span class="citation">(<em>11</em>)</span>.</p>
<p>Furthermore, it was shown that rosette formation is an important morphological feature for a vesicle to form on top of the rosette, which acts as a locally enriched source of Fgf signalling <span class="citation">(<em>12</em>)</span>.</p>
</div>
<div id="intro-notch" class="section level4">
<h4><span class="header-section-number">1.1.3.3</span> Lateral Inhibition</h4>
<p>In neurobiology the term <em>lateral inhibition</em> describes the process of an excited neuron reducing the activity of its neighbors. The same principle however can be found in other types of cells too where one cell signals its neighbor cell(s) <em>not</em> to do something. term has therefore been adapted by developmental biologists to describe processes in which Notch signaling takes place <span class="citation">(<em>13</em>, <em>14</em>)</span>.</p>
<p>Notch signaling is a highly conserved in evolution and controls various developmental and homeostatic processes that involve patterning, such as sensory hair cell formation, branched arterial networks or organ morphogenesis. Notch signaling consists of four components: (1) The extracellular, membrane bound Notch receptor (2) Notch ligands (3) the Notch intracellular domain (NICD) (4) and the <span class="math inline">\(\gamma\)</span>-secretase. Upon ligand activation of NICD and cleavage and release of NICD, NICD enters the nucleus and together with DNA-binding proteins and co-factors initiates expression of target genes. In contrast to other signaling pathways</p>
<ol style="list-style-type: decimal">
<li>there are no intermediates between membrane signaling and nucleus and therefore no amplification or dampening of the signal occurs</li>
<li>signaling requires direct contact between cells, which makes Notch signaling particularly biased by features of cellular morphology and tissue organization <span class="citation">(<em>13</em>)</span>.</li>
</ol>
<p>Since Notch signaling occurs at sites where cells are in contact, the signal generated is proportional to the contact area. Additionally, the strength of the signal increases further where cells are tightly opposed, such as sites of apical constriction <span class="citation">(<em>4</em>, <em>13</em>, <em>15</em>, <em>16</em>)</span>.</p>
</div>
</div>
</div>
<div id="model-organism-and-system" class="section level2">
<h2><span class="header-section-number">1.2</span> Model Organism and System</h2>
<p>To discover biological phenomena, biologists use a variety of non-human model organism species. While each model organism has their advantages and disadvantages, the choice for a particular model depends on the scientific question.</p>
<p>To study embryonic development the fresh water fish <em>Danio rerio</em> (also known as eng: <em>zebrafish</em> or <em>ger:</em> zebrabärbling (figure<a href="1-Intro.html#fig:zebra">1.6</a>)) has become an important model organism over the recent years. <em>D.rerio</em> is a diploid organism with a fully sequenced genome (of the human genes 71.4% have at least one <em>D.rerio</em> ortholog, 47% have a one-to-one ortholog <span class="citation">(<em>17</em>)</span>. It has a relatively short alternation of generations (12-16 weeks), a regularly large number of embryos (100 / week / female) and is relatively undemanding in terms of space for breeding and time for maturation. Furthermore, it offers well established methods for mutagenesis, screening and generation of transgenic lines. Since its embryos are naturally transparent and develop <em>ex utero</em>, it’s an ideal system for microscopic examination using molecular dyes and tags to visualize <em>inter</em> and <em>intra</em> cellular components even deep within the tissue (e.g. cell nuclei or cell membrane fluorescent tags). Together with advances in imaging techniques, this also allows for high throughput, high resolution, long-term <em>in vivo</em> imaging. Especially the combination of organ or tissue specific fluorescent makers like Green- or Red fluorescent protein (G- or RFP) and observation of pre-larval stages, where the embryo is still transparent, offers enormous possibilities and to address interesting and long standing open questions.</p>
<p>In nature zebrafish can be found in the shallow waters of the Indian and Pakistan Ganges inflows. It exhibits an oval body shape and can reach a length of up to 5 cm in adulthood. While females are usually more silverish, for males the back is brownish and the belly yellow-whit. Laterally it exhibits its name-giving dark-blue iridescent stripes with silver in between.</p>

<div class="figure" style="text-align: center"><span id="fig:zebra"></span>
<img src="figures/intro/CM_fish.png" alt="Model organism Danio rerio (aquarell by Christine Molenda)" width="0.50\textwidth" />
<p class="caption">
Figure 1.6: Model organism <em>Danio rerio</em> (aquarell by Christine Molenda)
</p>
</div>
<div id="developmental-stages" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Developmental Stages</h3>
<p>A single female may lay up to 100 fertilized eggs. Each Zygote<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a> then undergoes the first zygotic cell cycle (<strong>up to 0.75 h</strong>). The following two to seven cell cycles (period: <em>Cleavage</em>, <strong>up to 2.25 h</strong>) occur directed and synchronous every ~15 min. Cells in this stage are called <em>blastomeres</em> and are incompletely undercut the <em>blastodisc</em> (the collective of blastomeres) and remain interconnected by cytoplasmic bridges. The <em>Blastula</em> period (up to 5.25 h) is determined by an increasing a-synchronicity between the cells, flattening of the blastodisc and lengthening of the cell cycle. This period is also marked by the onset of <em>epiboly</em><a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a> <span class="citation">(<em>18</em>)</span>.</p>
<p>After Blastula and <strong>up to 10 h</strong> the <em>Gastrula</em> period takes place, followed by the <em>Segmentation</em> period (<strong>up to 24 h</strong>). Both of which are depicted in more detail in figure <a href="1-Intro.html#fig:stages">1.7</a>. At even later stages the embryo starts to elongate posteriorly, grow in size and develop organs until it first active muscle are present and it starts to swim (figure <a href="1-Intro.html#fig:stages">1.7</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:stages"></span>
<img src="figures/intro/stages.png" alt="Zebrafish embryonic development. Microscopic images are from a time-lapse where 24 embryos were imaged simultaneously in brightfield and at 488 nm Z-Stacks. Representation shows contrast enhanced EDFs. Bottom row (30-48 hpf) stages are handmade drawings from live embryos made during the first week of my PhD." width=".95\textwidth" />
<p class="caption">
Figure 1.7: Zebrafish embryonic development. Microscopic images are from a time-lapse where 24 embryos were imaged simultaneously in brightfield and at 488 nm Z-Stacks. Representation shows contrast enhanced EDFs. Bottom row (30-48 hpf) stages are handmade drawings from live embryos made during the first week of my PhD.
</p>
</div>
</div>
<div id="the-lateral-line-system" class="section level3">
<h3><span class="header-section-number">1.2.2</span> The Lateral Line System</h3>
<p>The lateral line (LL) system is a mechano-sensory organ that is common to all teleost fish. It enables the animal to sense water movements and therefore to orient itself, and to detect prey and predators. Fully developed, the lateral line system is comprised of hundreds of neuromasts positioned in an orderly pattern all over the animal’s body (figure <a href="1-Intro.html#fig:llsystem">1.8</a>A). Its functional subunits are the neuromasts (NM) (figure <a href="1-Intro.html#fig:llsystem">1.8</a>B-C) that, when fully developed, consist of hair-, support- and mantle cells. To sense water movements, each NM projects kinocilia out of the skin that, upon water induced deflection, create action potentials that are transduced via afferent and efferent fibers <span class="citation">(<em>19</em>, <em>20</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:llsystem"></span>
<img src="figures/intro/ll_system.jpg" alt="The lateral line system. A modified after (Ghysen et al., 2012) and A.Bergs, 2016 (student presentation at AK Lecaudey). Development of the later line system at embryonic, larval and juvenile stage. B Schematic showing a crossection and organization of a single neuromast. C-C’ SEM images of a single, pre-mature (3 dpf) neuromast." width=".95\textwidth" />
<p class="caption">
Figure 1.8: The lateral line system. <strong>A</strong> modified after (Ghysen et al., 2012) and A.Bergs, 2016 (student presentation at AK Lecaudey). Development of the later line system at embryonic, larval and juvenile stage. <strong>B</strong> Schematic showing a crossection and organization of a single neuromast. <strong>C-C’</strong> SEM images of a single, pre-mature (3 dpf) neuromast.
</p>
</div>
<p>Each NM is first deposited as a premature cluster of about 30 cells from a migrating cell-aggregate called the posterior lateral line primordium (pLLP) <a href="1-Intro.html#fig:llgfp">1.9</a>A. The pLLP delaminates from the pLL placode, caudal to the otic vesicle (figure <a href="1-Intro.html#fig:llgfp">1.9</a>B) at around 20 <em>hours post fertilization</em> (hpf) as a group of ~100 cells. After formation of the leading region with cells undergoing EMT, it starts migrating along a chemokine gradient positioned at the horizontal myoseptum to the tip of the tail (figure <a href="1-Intro.html#fig:pllp">1.10</a>) <span class="citation">(<em>19</em>, <em>20</em>)</span>. To ensure the development of a functional organ, several fundamental biological processes like cell migration, morphogenesis, proliferation and cell polarization need to be integrated into the pLLP.
An important breakthrough in LL research has been the development of a transgenic line expressing a membrane tethered GFP fusion protein (lyn-GFP) that is expressed under the LL specific promotor of <em>cldnb</em> (claudin b<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a>) <span class="citation">(<em>21</em>)</span>, which allowed for a much more detailed view and to observe lateral line development <em>in vivo</em>. An example of the fluorescence signal visible at ~60 hpf can be seen in figure <a href="1-Intro.html#fig:llgfp">1.9</a>B.</p>

<div class="figure" style="text-align: center"><span id="fig:llgfp"></span>
<img src="figures/intro/ll_gfp.jpg" alt="Neuromast deposition and pattern A Scheme showing NM deposition over three timepoints (10 min. interval). Dotted lines are time-tracks of rosettes, which become more concentrated over time. Bottom arrow indicates regions of rosette formation, maturation and deposition within the pLLP (scale bar = 20 µm; 20X WI; ~20 Z-planes; 2.5 µm spacing. MaxIP. Colors inverted.) B Scheme showing the lateral line at end of migration (~60 hpf) and other parts visible through the cldnb:lyn-gfp transgene (as documented through zfin.org) (air objective + 1.5X tube lens; four tiles; ~20 Z-planes; 5 µm spacing. MaxIP. Colors inverted)." width=".95\textwidth" />
<p class="caption">
Figure 1.9: Neuromast deposition and pattern <strong>A</strong> Scheme showing NM deposition over three timepoints (10 min. interval). Dotted lines are time-tracks of rosettes, which become more concentrated over time. Bottom arrow indicates regions of rosette formation, maturation and deposition within the pLLP (scale bar = 20 µm; 20X WI; ~20 Z-planes; 2.5 µm spacing. MaxIP. Colors inverted.) <strong>B</strong> Scheme showing the lateral line at end of migration (~60 hpf) and other parts visible through the cldnb:lyn-gfp transgene (as documented through zfin.org) (air objective + 1.5X tube lens; four tiles; ~20 Z-planes; 5 µm spacing. MaxIP. Colors inverted).
</p>
</div>
</div>
<div id="posterior-lateral-line-primordium" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Posterior Lateral Line Primordium</h3>
<p>The pLLP is about 100-150 <span class="math inline">\(\mu\)</span>m in length (depending on deposition cycle) over which it exhibits a diverse surface topology and cellular morphology. The current model assumes that the <em>caudal</em> (also <em>posterior</em>), more mesenchymal cells, are leading the path of migration, while the <em>cranial</em> (also <em>anterior</em>), more epithelial cells, are trailing. Towards the leading region the cells are more flat, towards the trailing region the cells become more columnar and increasingly radially organized into formations called epithelial rosettes (figure <a href="1-Intro.html#fig:pllp">1.10</a>). During migration the pLLP typically contains 2-3 <em>rosettes</em> (~25-30 cells each), while the most trailing one will eventually be deposited to further mature to a functional NM <span class="citation">(<em>19</em>, <em>22</em>)</span>. Since every deposition comes with a loss of cells in the pLLP, cells need to be regenerated. While one study concludes a spatial heterogeneity in distribution of proliferative cells <span class="citation">(<em>23</em>)</span>, another one suggests a higher proliferative rate near the leading region <span class="citation">(<em>24</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:pllp"></span>
<img src="figures/intro/pllp.png" alt="Summary of signaling gradients within the pLLP. Cross-section of the lateral line primordium. (red = trailing region, yellow = leading region; embryonic axes indicated by arrows. Signaling gradients indicated by black-white gradients, sorted by hierarchy.)" width=".75\textwidth" />
<p class="caption">
Figure 1.10: Summary of signaling gradients within the pLLP. Cross-section of the lateral line primordium. (red = trailing region, yellow = leading region; embryonic axes indicated by arrows. Signaling gradients indicated by black-white gradients, sorted by hierarchy.)
</p>
</div>
<div id="rosette-formation" class="section level4">
<h4><span class="header-section-number">1.2.3.1</span> Rosette formation</h4>
<p>The onset of morphological and functional changes is determined by signaling of Fgf, which is mostly active in the trailing region. Causal for expression of Fgf is a signaling center of WNT in the leading region, which promotes expression of Fgf ligands <em>Fgf-3</em> and <em>-10</em> <span class="citation">(<em>25</em>)</span>. Those ligands then diffuse to the trailing domain where binding through <em>Fgf receptor 1</em> (Fgfr1) triggers a signaling cascade through which the cells become more columnar, apically constricted and eventually re-organize into epithelial rosettes <span class="citation">(<em>24</em>, <em>26</em>, <em>27</em>)</span>. Concurrently, cells of the Wnt signaling center themselves are not competent to Fgf signaling, which is achieved through expression of <em>Sef</em>, an intracellular antagonist to Fgfr1 signaling <span class="citation">(<em>28</em>)</span>. In a more recent publication <span class="citation">(<em>29</em>)</span> it was shown that Shroom3 plays a key role in mediating AC and rosette formation in the pLLP.</p>
</div>
<div id="hair-cell-specification" class="section level4">
<h4><span class="header-section-number">1.2.3.2</span> Hair cell specification</h4>
<p>Downstream of Fgf lies the expression of transcription factor (TF) atoh1a, which gives cells the potential to become sensory hair cells <span class="citation">(<em>24</em>)</span>. The current model suggests that Fgf initiates expression of <em>atoh1a</em> and <em>deltaA</em>, where DeltaA activates Notch in neighboring cells to inhibit expression of <em>atoh1a</em> in those. Atoh1a in turn suppresses competence for Fgf and initiates expression of <em>atoh1b</em> and <em>deltaD</em>. While the latter acts synergistic with DeltaA, Atoh1b again drives expression of <em>atoh1a</em> <span class="citation">(<em>30</em>)</span>. Once a prospective hair cell is specified it will itself become a source for Fgf-10. By this process adjacent cells are laterally inhibited and determined <em>support cells</em> (figure <a href="1-Intro.html#fig:llsystem">1.8</a>B), still capable of receiving Fgf signals.</p>
<p>Just before the most trailing rosette is deposited, its prospective hair cell will undergo a final division to form a doublet of sensory hair cells that will orient orthogonal to planar polarity and establish mirror symmetry in maturing neuromasts <span class="citation">(<em>31</em>, <em>32</em>)</span>.</p>
</div>
</div>
<div id="shroom3-in-the-pllp" class="section level3">
<h3><span class="header-section-number">1.2.4</span> Shroom3 in the pLLP</h3>
<p>The Shroom protein family is conserved through evolution (supplementary figure) and involved in contraction of the actomyosin network (e.g. during AC), which has been confirmed in several studies <em>e.g.</em> investigating e.g. epithelial planar remodelling, neural tube morphogenesis, <em>Xenopus</em> bottle cells and epithelial invagination <span class="citation">(<em>33</em>–<em>38</em>)</span>. Furthermore, Shroom might be a novel candidate involved in heterotaxy <span class="citation">(<em>39</em>)</span>.</p>
<div id="intro-shroom" class="section level4">
<h4><span class="header-section-number">1.2.4.1</span> Recent research</h4>
<p>Shroom proteins have three characteristic domains (1) a <em>PDZ</em> domain close to the N-terminus to interact with other proteins with PDZ domains and (2) two <em>Apx/Shroom</em> domains (ASD1 and 2), the latter being close to the proteins C-terminus. The Shroom domains may interact with proteins containing a <em>Shroom binding domain</em> (SBD) such as Rock to help and facilitate phosphorylation of NMII (figure <a href="1-Intro.html#fig:shrminteract">1.11</a>) <span class="citation">(<em>40</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:shrminteract"></span>
<img src="figures/intro/shrm3_interaction.png" alt="Shroom3 functional domains and mode of operation. Sequence of events numbered from 1-4. Approximate scale jumps indicated at arrows." width=".95\textwidth" />
<p class="caption">
Figure 1.11: Shroom3 functional domains and mode of operation. Sequence of events numbered from 1-4. Approximate scale jumps indicated at arrows.
</p>
</div>
<p>While in all of the above-mentioned studies Shroom3 was the focus of interest, in zebrafish and most other model organisms there are four paralogs. A previous study done on Shroom3 morphants in <em>D.rerio</em> <span class="citation">(<em>29</em>)</span> indicates that Shroom3 is also necessary for AC and rosette formation in the migrating pLLP. In summary they were able to show and that</p>
<ol style="list-style-type: decimal">
<li>Shroom3 is indeed expressed in the pLLP from stages 24 – 48 hpf which was confirmed via <em>in situ</em> hybridization (ISH)</li>
<li><em>shroom3</em> is expressed downstream of Fgf signalling, which was confirmed via treatment with an Fgfr1 inhibiting drug. Subsequent ISH with the Probe used before showed that the signal was gone in Morpholino (MO) injected embryos. The drug used has been used before (SU5402) in the same context <span class="citation">(<em>24</em>, <em>27</em>)</span>. In these it was found that SU5402 has the ability to prevent rosette formation, but not to dismantle already established ones.</li>
<li>Shroom3 localizes to rosette centers, which was accomplished by inserting a genetic construct where a <em>shroom3-tagRFP</em> fusion protein is under control of a heat-shock<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a> promotor into a <em>cldnb:lyn-GFP</em> transgenic line. This way they were able to ectopically express <em>shroom3-tRFP</em> and confirm its localization by confocal fluorescence microscopy (figure <a href="1-Intro.html#fig:shrmernst">1.12</a>A).</li>
<li>Rosette formation is impaired, which was confirmed quantitatively by using a specifically trained <em>rosette detector</em> <span class="citation">(<em>41</em>)</span> to count and weight single rosettes. As a metric to compare whole pLLPs the sum of weights as in <span class="math inline">\(\sum_{i=1}^{i-n}{weighted\ detections}\)</span> was used (figure <a href="1-Intro.html#fig:shrmernst">1.12</a>B-B’).</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:shrmernst"></span>
<img src="figures/intro/shrm_ernst.png" alt="Shroom3 in rosette formation (adapted from Ernst et al., 2012) A composite MaxIPs of membrane label and fusion protein showing the localization of Shroom within the pLLP B uninjected control and shroom3 MO injected MaxIPs B’ Heat-maps of rosette detector score." width=".95\textwidth" />
<p class="caption">
Figure 1.12: Shroom3 in rosette formation (adapted from Ernst et al., 2012) <strong>A</strong> composite MaxIPs of membrane label and fusion protein showing the localization of Shroom within the pLLP <strong>B</strong> uninjected control and shroom3 MO injected MaxIPs <strong>B’</strong> Heat-maps of rosette detector score.
</p>
</div>
</div>
<div id="current-model" class="section level4">
<h4><span class="header-section-number">1.2.4.2</span> Current Model</h4>
<p>Based on these and previous results, the current model for apical constriction in the pLLP assumes that (1) expression of <em>shroom3</em> is induced by Fgf signaling (2) Shroom3 binds Rock and translocates to the AJC to (3) mediate phosphorylation of NMII which (3) induces contraction of the actin network and AC (figure <a href="1-Intro.html#fig:shrmmodel">1.13</a>). Furthermore, AC is necessary for rosette assembly and subsequent NM deposition. In conclusion the current understanding is that without Shroom3 AC can not take place and rosette formation does not occur, therefore NMs are not deposited.</p>

<div class="figure" style="text-align: center"><span id="fig:shrmmodel"></span>
<img src="figures/intro/shrm_model.png" alt="Shroom3 current model" width=".70\textwidth" />
<p class="caption">
Figure 1.13: Shroom3 current model
</p>
</div>
</div>
</div>
<div id="shroom3-mutants" class="section level3">
<h3><span class="header-section-number">1.2.5</span> <em>shroom3</em> mutants</h3>
<p>The strategy for the mutant was to inhibit interaction with SBD-domain proteins. Furthermore, the mutant should be easily genotyped by amplifying a fragment of the gene via PCR and correlating gel electrophoresis band patterns after a restrictive digest. To achieve this, an eight base pair deletion was introduced in Shroom domain 1 (SD1) using the <em>Transcription activator-like effector nuclease</em> (TALEN) <span class="citation">(<em>42</em>)</span> genome editing system that would prevent interaction with SBD-domain proteins, or even lead to a NULL mutation<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> (figure <a href="1-Intro.html#fig:shrmmut">1.14</a>A-A’). The mutation lead to (1) a shift in the open reading frame and to a pre-mature stop codon and (2) a deletion of a restriction site for a restriction enzyme named NsiI (figure <a href="1-Intro.html#fig:shrmmut">1.14</a>B-B’).</p>
<div id="intro-phen" class="section level4">
<h4><span class="header-section-number">1.2.5.1</span> Phenotype description</h4>
<p>While birth rates follow a distribution of Mendelian inheritance (after genotyping at 3 months of age), mutant adults are more sensitive to mechanical stress and have a shortened lifespan (~6-9 months). In body shape, <em>shroom3</em> mutants are not smaller or posses any other striking phenotype (figure <a href="1-Intro.html#fig:shrmmut">1.14</a>C), however their gill flaps seem to be increased in size, swollen, and not exactly streamlined with the body. This is also evident by an increased frequency of gill flap beating. When looking at the pLLP at different stages (figure <a href="1-Intro.html#fig:shrmmut">1.14</a>D), they exhibit the same phenotype as the MO injected embryos. Here it is noteworthy that a fraction of about 20<span class="math inline">\(\%\)</span> of the pLLPs exhibit a more wildtype (or <em>weak</em>) phenotype, even though these embryos were genotyped as homozygous<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a> mutant. A possible reason for this will also be subject of thesis. Figure <a href="1-Intro.html#fig:shrmmut">1.14</a>E shows exemplary specimen at about 60 hpf where for the mutant, the first thing that catches the eye is the increase in the number of clusters<a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a> deposited. Interestingly, this is rather contradictive to the current model of Shroom3’s function during LL development.</p>

<div class="figure" style="text-align: center"><span id="fig:shrmmut"></span>
<img src="figures/intro/shrm_mutants.png" alt="shroom3 mutant phenotype (A-A’) mutation strategy (A) Talen arms (black blocks) bordering a sequence within SD1 including a restriction site for NsiI (indicated back arrows and grey background) (A’) wildtype and mutant allele with 8 bp deletion (B-B’) Amino acid code and protein schematic with functional domains and stop codon (indicated by asterisk) (C-C’) Adult phenotype with closeup to gill flaps (D) pLLP phenotype for three different stages (columns) and different manifestations (rows) (40X WI objective). Arrows indicate epithelial rosettes. (E) LL phenotype at end of migration (10X air objective). Scale bars = 1 mm. MaxIPs for D-E." width=".95\textwidth" />
<p class="caption">
Figure 1.14: <em>shroom3</em> mutant phenotype <strong>(A-A’)</strong> mutation strategy (A) Talen arms (black blocks) bordering a sequence within SD1 including a restriction site for NsiI (indicated back arrows and grey background) (A’) wildtype and mutant allele with 8 bp deletion <strong>(B-B’)</strong> Amino acid code and protein schematic with functional domains and stop codon (indicated by asterisk) <strong>(C-C’)</strong> Adult phenotype with closeup to gill flaps <strong>(D)</strong> pLLP phenotype for three different stages (columns) and different manifestations (rows) (40X WI objective). Arrows indicate epithelial rosettes. <strong>(E)</strong> LL phenotype at end of migration (10X air objective). Scale bars = 1 mm. MaxIPs for D-E.
</p>
</div>
</div>
</div>
</div>
<div id="objective-and-open-questions" class="section level2">
<h2><span class="header-section-number">1.3</span> Objective and Open Questions</h2>
<p>After having the results of the morpholino induced phenotype, a mutant was generated whose pLLP phenotype closely resembled the morpholino induced one. However, the phenotype at later stages revealed discrepancies with the current model and the role of Shroom3 during LL development.</p>
<div id="motivation" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Motivation</h3>
<p>Due to the pLLP’s relative simplicity (~100 cells), excellent accessibility for advanced light-microscopes (~1 cell layer beneath the skin) and a broad and well-established palette of available methods, it promises an <em>in toto</em> understanding and complete model of factors leading to its development. Having access to such a model may be very beneficial to other fields such as organ regeneration, cancer- and clinical research.</p>
<p>In this scenario, the advantages of working with a knockout- as compared to a knockdown<a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a> mutant are (1) an equal amount of gene expression across all samples (2) no side effects due to injection procedure or solvents in drug treatments (3) the ability to observe the effects of the mutation for very long periods of time.</p>
<p>One aspect about LL development that still remains unclear are the exact factors that lead to cell cluster (CC) deposition. What determines the timing of deposition? When is a rosette mature enough to be deposited? What happens to the cells before and after deposition? How do the cells specify before deposition? What are the morphological features of depositing cells? What influence does morphology have on Neuromast deposition?</p>
<p>Using the Shroom3 mutants as a model system offers great potential to shed new light on organogenesis, the interplay between morphogenesis and cellular specification, epithelial rosette formation and neuromast deposition.</p>
</div>
</div>
<div id="data-strategy-and-analysis" class="section level2">
<h2><span class="header-section-number">1.4</span> Data Strategy and Analysis</h2>
<p>Due to the history of Developmental Biology and the complexity of biological processes <em>per se</em>, the field heavily relies on image data. Since the advent of electronic imaging techniques<a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a> scientific image data can be processed and analyzed <em>in silico</em>. To make use of the possibility of</p>
<ol style="list-style-type: decimal">
<li>live imaging, which (as compared to fixation techniques) conserves the cellular integrity and morphology while also offers the possibility of recording time-lapses</li>
<li>the optically clear specimen and</li>
<li>high throughput image analysis and state-of-the-art data science using algorithmic implementations,</li>
</ol>
<p>the three following points were paid special attention to.</p>
<div id="sample-preparation" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Sample Preparation</h3>
<p>For fluorescence microscopy zebrafish embryos are usually immersed in a 1% solution of <em>low melting-point agarose</em> (LMPA) solution and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> eight to ten embryos <em>per</em> dish. To make use of the high number of offspring a single zebrafish female may lay, which rapidly leads to a sample number of more than 300, a new sample preparation technique was devised that allows for (1) a four to five times increase in samples per dish (2) a facilitated navigation <em>via</em> a grid-like orientation through the samples and (3) an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope. For details, see Materials and Methods section <a href="2-mat-met.html#mount-met">2.2.2.3</a> and Kleinhans <em>et al.</em>, 2019.</p>
</div>
<div id="imaging" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Imaging</h3>
<p>Technically, speed and sensitivity are most important for live imaging. Considering these two parameters a light-sheet <span class="citation">(<em>43</em>)</span> fluorescence microscope (LSFM) would be the best fit. However, LSFMs also have several limitations. First, due to the sample preparation methods available the number of samples that can be imaged at a time is highly restricted. Second, for subcellular resolution a high magnification is required, which is limited by working distances and third - for optimal image analysis a high Signal-to-Noise ratio (SNR) and numerical aperture (N.A.) is preferable. Therefore a spinning disc <span class="citation">(<em>44</em>)</span> system was chosen for most of the imaging.
The system makes use of (1) an extra-large field of view (FOV) ideal for large specimen (2) the possibility of a high degree of automation with state-of-the-art software and (3) a water dispension system for long-term water immersion imaging. For details about the system see Materials and Methods section <a href="2-mat-met.html#mat-SD">2.1.9.2</a>.</p>
</div>
<div id="data-handling" class="section level3">
<h3><span class="header-section-number">1.4.3</span> Data handling</h3>
<p>Subsequently after data acquisition and pre-processing the image data was transferred from the microscope system to the labs main workstation. To uniquely identify each file and have them appear in a structured manner, a file-naming system was established after the following structure</p>

<p>Where <em>stage</em> would e.g. be 32hpf, <em>group</em> would be a genotype or drug treatment, <em>id</em> would be a positional identifier on an imaging dish like B1P01<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> and <em>date</em> would be a date in the form of YYMMDD.</p>
</div>
<div id="image-and-data-analysis" class="section level3">
<h3><span class="header-section-number">1.4.4</span> Image and Data Analysis</h3>
<p>In order to be as objective an as high throughput as possible, almost all of the analyses performed for this study was solved using either algorithmically or state-of-the-art convolutional neural networks (CNNs). Furthermore, to meet the terms and conditions of <em>open science</em><a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a> standards, all pipelines were implemented in open source software frameworks such as <em>Fiji is just image J</em> (FIJI) and R. For further information about training datasets, algorithms and versions used see Materials and Methods section <a href="2-mat-met.html#mat-sftwr">2.1.10</a>.</p>

</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="2">
<li id="fn2"><p>While the biogenetic got refuted in its core (there is no ‘complete’ phylogenesis in ontogenesis). It still can’t be neglected that embryogenesis even of evolutionary distant species shows remarkable similarities<span class="citation">(<em>2</em>)</span><a href="1-Intro.html#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>Even though the expression Form follows function <span class="citation">(<em>3</em>)</span> is usually found in design and architecture, it formulates the general idea that any objects form is (or in design should) be shaped by the requirements to it.<a href="1-Intro.html#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>catenins are regulators of cell-cell adhesion and gene transcription<a href="1-Intro.html#fnref4" class="footnote-back">↩</a></p></li>
<li id="fn5"><p>one of the earliest differentiating layer of cells. Cells of the mesoderm will e.g. form the musculature.<a href="1-Intro.html#fnref5" class="footnote-back">↩</a></p></li>
<li id="fn6"><p>enzymes that transfer energy to specific substrates<a href="1-Intro.html#fnref6" class="footnote-back">↩</a></p></li>
<li id="fn7"><p>first diploid cell after fertilization between two germ cells<a href="1-Intro.html#fnref7" class="footnote-back">↩</a></p></li>
<li id="fn8"><p>cells in late blastula start to dome, while a monolayer of the domes circumfence begins to wrap around the Yolk<a href="1-Intro.html#fnref8" class="footnote-back">↩</a></p></li>
<li id="fn9"><p>construct name: <em>Tg(-8.0cldnb:lynGFP)</em>; ZFIN ID: ZDB-TGCONSTRCT-070117-15<a href="1-Intro.html#fnref9" class="footnote-back">↩</a></p></li>
<li id="fn10"><p>heat shock proteins are <em>Escherichia coli</em> enzymes that assist protein folding and are heat activated<a href="1-Intro.html#fnref10" class="footnote-back">↩</a></p></li>
<li id="fn11"><p>mutation that leads to a complete loss of function<a href="1-Intro.html#fnref11" class="footnote-back">↩</a></p></li>
<li id="fn12"><p>when both alleles are the same<a href="1-Intro.html#fnref12" class="footnote-back">↩</a></p></li>
<li id="fn13"><p>forth on Neuromasts will be called ‘cell clusters<a href="1-Intro.html#fnref13" class="footnote-back">↩</a></p></li>
<li id="fn14"><p>permanent versus transient mutation<a href="1-Intro.html#fnref14" class="footnote-back">↩</a></p></li>
<li id="fn15"><p>e.g. photomultipliers or charge-coupled devices<a href="1-Intro.html#fnref15" class="footnote-back">↩</a></p></li>
<li id="fn16"><p>the process of embedding the samples in agarose<a href="1-Intro.html#fnref16" class="footnote-back">↩</a></p></li>
<li id="fn17"><p>Where B stands for a batch, that is if multiple dishes were imaged and P stands for the position within a single batch<a href="1-Intro.html#fnref17" class="footnote-back">↩</a></p></li>
<li id="fn18"><p>“movement to make scientific research […] and its dissemination accessible to all levels of an inquiring society” – Wikipedia/en/Open_science<a href="1-Intro.html#fnref18" class="footnote-back">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="list-of-abbreviations.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-mat-met.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
