<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Genetic and Mechanic Signaling in Organ Development</title>
  <meta name="description" content="Chapter 1 Introduction | Genetic and Mechanic Signaling in Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Genetic and Mechanic Signaling in Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Genetic and Mechanic Signaling in Organ Development" />
  
  
  

<meta name="author" content="David Simon Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="summary.html"/>
<link rel="next" href="2-mat-met.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { font-weight: bold; } /* Alert */
code span.an { font-style: italic; } /* Annotation */
code span.cf { font-weight: bold; } /* ControlFlow */
code span.co { font-style: italic; } /* Comment */
code span.cv { font-style: italic; } /* CommentVar */
code span.do { font-style: italic; } /* Documentation */
code span.dt { text-decoration: underline; } /* DataType */
code span.er { font-weight: bold; } /* Error */
code span.in { font-style: italic; } /* Information */
code span.kw { font-weight: bold; } /* Keyword */
code span.pp { font-weight: bold; } /* Preprocessor */
code span.wa { font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#about"><i class="fa fa-check"></i>About</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#foreword"><i class="fa fa-check"></i>Foreword</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abbreviations"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a>
<ul>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#einleitung"><i class="fa fa-check"></i>Einleitung</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#zielsetzung"><i class="fa fa-check"></i>Zielsetzung</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#ergebnisse"><i class="fa fa-check"></i>Ergebnisse</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#intro-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#cell-signaling"><i class="fa fa-check"></i><b>1.1.2</b> Cell Signaling</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#intro-morph"><i class="fa fa-check"></i><b>1.1.3</b> Morphogenesis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#open-questions-and-motivation"><i class="fa fa-check"></i><b>1.3</b> Open Questions and Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mos"><i class="fa fa-check"></i><b>2.1.9</b> Morpholinos</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.10</b> Hardware</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.11</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Data Strategy and Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.2</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.3</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.4</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.3</b> Ground Truth</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cluster-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Cluster Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#morphometric-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Morphometric analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.4</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#res-met"><i class="fa fa-check"></i><b>3.1</b> Quantitative imaging and image analysis</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#res-mount"><i class="fa fa-check"></i><b>3.1.1</b> A 3D-printed stamp to standardize sample mounting and semi-automatize imaging</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#mat-anallzr2d"><i class="fa fa-check"></i><b>3.1.2</b> anaLLzr2D - Automated 2D neuromast analysis and nuclei count</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#mat-anallzr2dt"><i class="fa fa-check"></i><b>3.1.3</b> anaLLzr2DT - Automated 2D pLLP migration and neuromast deposition analysis through time</a></li>
<li class="chapter" data-level="3.1.4" data-path="3-res.html"><a href="3-res.html#mat-anallzr3d"><i class="fa fa-check"></i><b>3.1.4</b> anaLLzr3D - Automated 3D single cell segmentation and A.I. analysis in the pLLP</a></li>
<li class="chapter" data-level="3.1.5" data-path="3-res.html"><a href="3-res.html#CNN"><i class="fa fa-check"></i><b>3.1.5</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#intro-phen"><i class="fa fa-check"></i><b>3.2.1</b> Phenotype description</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.3</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.4</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#apical-constriction-2"><i class="fa fa-check"></i><b>3.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.6</b> Haircell Specification</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-discussion.html"><a href="4-discussion.html"><i class="fa fa-check"></i><b>4</b> Discussion</a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-discussion.html"><a href="4-discussion.html#methods"><i class="fa fa-check"></i><b>4.1</b> Methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="4-discussion.html"><a href="4-discussion.html#standardized-mounting"><i class="fa fa-check"></i><b>4.1.1</b> Standardized mounting</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-discussion.html"><a href="4-discussion.html#automated-image-analysis"><i class="fa fa-check"></i><b>4.1.2</b> Automated image analysis</a></li>
<li class="chapter" data-level="4.1.3" data-path="4-discussion.html"><a href="4-discussion.html#rosette-detection-1"><i class="fa fa-check"></i><b>4.1.3</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-discussion.html"><a href="4-discussion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-discussion.html"><a href="4-discussion.html#lateral-line"><i class="fa fa-check"></i><b>4.2.1</b> Lateral Line</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-discussion.html"><a href="4-discussion.html#rosette-formation-1"><i class="fa fa-check"></i><b>4.2.2</b> Rosette Formation</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-discussion.html"><a href="4-discussion.html#apical-constriction-3"><i class="fa fa-check"></i><b>4.2.3</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-discussion.html"><a href="4-discussion.html#hair-cell-specification"><i class="fa fa-check"></i><b>4.2.4</b> Hair Cell specification</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-discussion.html"><a href="4-discussion.html#model-2"><i class="fa fa-check"></i><b>4.2.5</b> Model</a></li>
</ul></li>
</ul></li>
<!--<li class="chapter" data-level="" data-path="appendix-appendix.html"><a href="appendix-appendix.html"><i class="fa fa-check"></i>(Appendix) Appendix</a></li>-->
<li class="chapter" data-level="" data-path="supplement.html"><a href="supplement.html"><i class="fa fa-check"></i>Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Genetic and Mechanic Signaling in Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Intro" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<div id="development" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Development</h2>
<p><strong>Developmental Biology</strong> studies the continuous process of cells adapting to their ever-changing environment from the fertilized egg to a full organism. For an organism to develop, cells need to proliferate (multiply) and become different from each other. To achieve this, <em>genes</em> are utilized that are expressed differently at certain times during development and types of cells. Those proteins again are used for geometrical construction or facilitation of chemical reactions. To illustrate this, the cell can be thought of as a marble rolling down a furrowed landscape (figure <a href="1-Intro.html#fig:wadd">1.1</a>). Physically a cell represents an open system, which is defined as a unit system able for external interactions. Such a system is therefore not self-dependent and self-sustained, but its current conformation is determined by external interactions. In this landscape a hill would be a high energy-, a valley a low energy state. The path the marble will take is determined by the furrows in the landscape, since it would always prefer a valley. However, this landscape is not a <em>static</em> structure but one that changes at every instant of time.</p>
<div class="figure" style="text-align: center"><span id="fig:wadd"></span>
<img src="figures/intro/waddington.jpg" alt="Waddington's Classical Epigenetic Landscape" width="50%" />
<p class="caption">
Figure 1.1: Waddington’s Classical Epigenetic Landscape
</p>
</div>
<p>As cells differentiate they acquire a certain identity - controlled by the genes they express and the proteins they produce - that again influences cell ‘behavior’ and its phenotype which includes cell division, migration and cell shape changes. This allows cells to assemble into tissues, which themselves assemble into organs. For developmental biology, which as a scientific discipline originates from embryology, the central interest is to understand how initially inanimate matter (like single atoms and molecules) is able to organize itself to such complex structures we see in living matter at different levels of organismal hierarchy while ensuring a robust developmental plan. Using the marble analogy, studying developmental biology can be thought of as tracking multiple cells as they roll down the valley while observing and testing how interactions between them might change their fate. The basic questions arising from this interest are</p>
<ul>
<li>How do tissues arise from a population of cells?</li>
<li>How do organs form from tissues?</li>
<li>Why do organs form at their particular location?</li>
<li>How do migrating cells know whether they reached their destination?</li>
<li>How is growth controlled and how do body axes form?</li>
</ul>
<div id="intro-types" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Cell Types</h3>
<p>In animals there are two basic types of cells.</p>
<ol style="list-style-type: decimal">
<li>Epithelial cells, which can form strong adhesions between each other and thereby are able to exert forces upon each other to achieve complex architectures.</li>
<li>Mesenchymal cells, which do not adhere as strong with each other and are more independent.</li>
</ol>
<p>This however describes only the extremes on a continuous scale. A cell is not a binary system but can show characteristics of both extremities, <em>e.g.</em> during Epithelial to Mesenchymal transition (EMT), a bidirectional process whereby epithelial cells can gain migratory and invasive properties and <em>vice versa</em>.</p>
<p>A cells identity on this continuum is determined by</p>
<ul>
<li>its gene expression profile, which reflects its repertoire of molecular machinery (proteins) and therefore determines its competence to react to internal and external cues.</li>
<li>its micro-environment, which has a <em>physical</em> (forces, energy) and a <em>chemical</em> (signaling molecules, diluents) dimension. A change in the latter usually brings about a reaction in the cell that becomes evident both in expression of genes and morphology.</li>
<li>its shape and incorporation into a tissue, which may modulate how a cell reacts to its micro-environment (<em>e.g.</em> certain regions of the cell can be more or less exposed, or it can be more or less tightly packed, tuning its susceptibility to forces and signals).</li>
</ul>
<p>With rising numbers of specialized cells assembling into tissue, a shape and body axes begin to emerge that for the earliest developmental stages is highly similar across certain phyla and only begins to diversify at later developmental stages, which reflects our evolutionary ancestry.</p>
</div>
<div id="cell-signaling" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Cell Signaling</h3>
<p>To communicate with each other, cells have developed a variety of intercellular communication systems and a complex network of intracellular signal transduction pathways. Some information transfer depends on direct cell-to-cell contact, others rely on freely diffusible ligands that can be sensed by other cells. Each signaling pathway consists of a ligand – receptor pair that determines their main function. In the following, three pathways are introduced that play major roles in embryonic development.</p>
<div id="wnt" class="section level4" number="1.1.2.1">
<h4><span class="header-section-number">1.1.2.1</span> WNT</h4>
<p>The word ‘WNT’ is a compound word of <em>Wingless</em> and <em>Int-1</em>, both of which are important genes during development of <em>Drosophila melanogaster</em> (commonly known as fruit fly), where WNT signaling was first studied. WNT is evolutionary conserved with 15 different receptors and co-receptors and plays a major role during embryonic axis formation, body segmentation, organogenesis and stem cell proliferation. Aberrant WNT signaling is involved in diseases like colon cancer, melanoma and neurodegenerative diseases <span class="citation">(<em>2</em>)</span>.</p>
<p>Beside this, WNT signaling is sub-divided in a canonical (<span class="math inline">\(\beta\)</span>-catenin<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> dependent) and a non-canonical (<span class="math inline">\(\beta\)</span>-catenin independent) branch. In canonical signaling, WNT ligand binds, together with co-receptor <em>Lipoprotein Receptor-related Protein 6</em> (LRP6), to the receptor <em>Frizzled</em> (Frz) – jointly activating protein <em>Dishevelled</em> (Dsh). Dsh again inhibits a protein complex usually degrading <span class="math inline">\(\beta\)</span>-catenin, leading to the stabilization of <span class="math inline">\(\beta\)</span>-catenin in the cytoplasm and the nucleus. Within the nucleus, <span class="math inline">\(\beta\)</span>-catenin forms a complex with LEF/TCF to activate specific target genes <span class="citation">(<em>2</em>)</span>.</p>
</div>
<div id="intro-FGF" class="section level4" number="1.1.2.2">
<h4><span class="header-section-number">1.1.2.2</span> Fibroblast Growth Factor</h4>
<p>Key roles of <em>Fibroblast growth factor</em> (FGF) signaling is mesoderm<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> patterning in the early embryo, regulation of angiogenesis and wound repair. On a cellular level it is also an important regulator of proliferation and differentiation. The mammalian FGF family comprises 18 ligands and four highly conserved transmembrane tyrosine receptors named FGF receptor 1-4 (FGFR1-4). Aberrant FGF signaling is <em>e.g.</em> associated with tumor growth <span class="citation">(<em>3</em>)</span>.</p>
<p>Upon ligand binding FGFR dimerizes and undergoes a conformational shift activating the intracellular kinase<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> domain. Subsequent trans-phosphorylation of tyrosine kinase domains serve as docking sites for adaptor proteins. Activated FGFR then phosphorylates FGFR substrate 2 (FRS2), recruiting adaptor protein <em>Son of Sevenless</em> (SoS) and <em>Growth factor Receptor bound 2</em> (GRb2) to set on a cascade of at least three possible kinase dependent signal transduction pathways eventually leading to activation of target genes involved in the regulation of <em>e.g.</em> proliferation, autophagy and EMT <span class="citation">(<em>3</em>)</span>.</p>
</div>
<div id="intro-notch" class="section level4" number="1.1.2.3">
<h4><span class="header-section-number">1.1.2.3</span> Notch</h4>
<p>In neurobiology the term <em>lateral inhibition</em> describes the process of an excited neuron reducing the activity of its neighbors. The same principle however can be found in other types of cells too where one cell signals its neighbor cell(s) <em>not</em> to acquire a certain fate. For initially equivalent cells to diversify and acquire distinct identities there needs to be a break in symmetry. Often this is accomplished by competition about Notch signaling sources where whichever cell expresses a trait first is able to suppress the same in its neighbors. Once this hierarchy is established, it is maintained in a positive feedback manner <span class="citation">(<em>4</em>, <em>5</em>)</span>.</p>
<p>Notch signaling gives cells the ability to self-organize and controls various developmental and homeostatic processes that involve patterning, such as sensory hair cell (HC) formation, branched arterial networks or organ morphogenesis. Notch signaling consists of four components:
(1) The extracellular domain of the membrane bound Notch receptor (2) Notch ligands (3) the Notch intracellular domain (NICD) (4) and the <span class="math inline">\(\gamma\)</span>-Secretase. Upon ligand activation of Notch through Delta, NICD is cleaved and released through <span class="math inline">\(\gamma\)</span>-Secretase. Subsequently NICD enters the nucleus and together with DNA-binding proteins and co-factors initiates expression of target genes. In contrast to other signaling pathways</p>
<ol style="list-style-type: decimal">
<li>there are no intermediates between membrane signaling and nucleus and therefore no amplification or dampening of the signal occurs</li>
<li>signaling requires direct contact between cells, which makes Notch signaling particularly biased by features of cellular morphology and tissue organization <span class="citation">(<em>4</em>)</span>.</li>
</ol>
<p>Assuming a homogeneous receptor concentration in a cell membrane and since Notch signaling occurs at sites where cells are in contact, the signal generated is proportional to the contact area. Additionally, the strength of the signal increases further where cells are tightly opposed <span class="citation">(<em>4</em>, <em>6</em>–<em>8</em>)</span>.</p>
</div>
</div>
<div id="intro-morph" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Morphogenesis</h3>
<p>Morphogenesis, from the Greek morphê (shape) and genesis (creation)</p>
<blockquote>
<p>“Creation of the shape”</p>
</blockquote>
<p>For objects that fulfill a purpose, their form is an expression of their function<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a>. Hence, analyzing the shape of an object can give information about its function. It is therefore an important feature in different scientific disciplines.</p>
<p>Breaking symmetry of daughter cells often results in one of the daughter cells adopting a different fate, which eventually results in diversification of shape (<em>e.g.</em> muscle cells, neural cells, <em>etc.</em>). In order to form a tissue or an organ with a variety of specialized cells, it is important for the single cell to have information about where it is located, what its neighbors are doing, how densely it is packed and what the chemical composition of its surrounding is. This is accomplished by being in constant feedback with its neighbor cells and sensation of its environment. After the information is processed, the cell reacts by adjusting the levels of proteins expressed to undergo proliferation or develop specialized structures like cilia or axons.</p>
<blockquote>
<p>“For an isolated cell, cell shape reflects a balance between cortical tension and intracellular pressure.” - Y.Pan <em>et al.</em><span class="citation">(<em>10</em>)</span></p>
</blockquote>
<p>For a cell, its present shape impacts its further specification by defining its ability in perceiving specific signals and the magnitude of signals received and transmitted <span class="citation">(<em>8</em>)</span>. Shape therefore sets the general framework for cell-cell interaction and follows a</p>
<p>scale hierarchy, where each scale’s output again feeds back to the others (figure <a href="1-Intro.html#fig:feedb">1.2</a>). <em>E.g.</em> it has been shown that simple changes in cell geometry affect fundamental processes such as cell growth, death, direction of cell divisions and extracellular vesicle cargo <span class="citation">(<em>11</em>–<em>22</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:feedb"></span>
<img src="figures/intro/feedback.png" alt="Form and function feedback loops. On a cellular scale expression of genes may cause a cell to constrict, which again feeds back to the molecular scale (e.g. force transmission) and the tissue scale (e.g. evagination or constriction). All of which standing in a continuous feedback." width="85%" />
<p class="caption">
Figure 1.2: Form and function feedback loops. On a cellular scale expression of genes may cause a cell to constrict, which again feeds back to the molecular scale (<em>e.g.</em> force transmission) and the tissue scale (<em>e.g.</em> evagination or constriction). All of which standing in a continuous feedback.
</p>
</div>
<p>Cell shape changes may occur in two different ways…</p>
<ul>
<li>active: Each cell has a dynamic skeleton that is composed of a complex matrix of interconnected proteins - the cytoskeleton. It is composed by three basic classes of filaments that have different physical properties. <em>Microtubules</em> (the most rigid), <em>Intermediate</em> and <em>micro-filaments</em> (the softest). Motor-proteins like Myosin may adhere to actin micro-filaments and contract them, thereby causing <em>e.g.</em> a narrowing of the cell diameter.</li>
<li>passive: In a tissue cells are, besides their individual cytoskeletal elements, connected at the supracellular level <em>via</em> actin through Adherens junctions. This way, if the tissue becomes deformed at one place, other more distant cells will get deformed as well <span class="citation">(<em>23</em>)</span>.</li>
</ul>
<div id="apical-constriction" class="section level4" number="1.1.3.1">
<h4><span class="header-section-number">1.1.3.1</span> Apical Constriction</h4>
<p>Apical constriction (AC) is a cell morphogenetic process manifesting by an active narrowing of the apical surface, making an epithelial cell appear bottle or wedge shaped instead of cuboidal. It is usually coordinated by multiple cells within an epithelial layer that raise forces necessary to deform a tissue.</p>
<p>A defining feature of epithelial cells is that they have a basal- (bottom) to apical (top) polarity. At the basal side, the cell is in contact with a specialized extracellular matrix (ECM, a thin sheet-like structure) called the basement membrane, apically the cell forms tight connections to its neighboring cells – a region called the apical junctional complex (AJC). The AJC encompasses three types of junctions: Adherent junctions (AJ) or zonula adherens (ZA), tight junctions (TJ) and desmosomes. Around the ZA dense cables of actomyosin are located that, analog to muscle sarcomeres, are able to contract upon activation of RHO‑associated protein kinase (Rock) and Rock-mediated phosphorylation of the motor protein non-muscle myosin II (NMII) <span class="citation">(<em>24</em>)</span>. The exact mechanism of apical constriction may differ in different organism and organs. <em>E.g.</em> it has been shown by Martin <em>et al.</em> <span class="citation">(<em>25</em>)</span> that in <em>Drosophila</em> cells are radially organized and RhoA and ROCK have a medioapical focus with RhoA also present at junctions, while chick cells exhibit a planar cell polarity and RhoGEF and ROCK are localized to junctions.</p>
<p>Developmental processes that involve AC are…</p>
<ul>
<li>Tissue folding and tube formation</li>
<li>Single cell ingression and EMT</li>
<li>Gastrulation</li>
<li>Healing and sealing of embryonic tissue</li>
</ul>
<p>Epithelial rosettes are radially organized cell clusters within an epithelial tissue whose vertices interface a common center similar to a garlic bud or a pie cut into pieces along its center. While the mechanisms of cytoskeletal rearrangements seem to be well conserved, the signals that lead to rosette formation are less well understood and more diverse.
At least two architectural distinct types of rosettes exist, depending on tissue polarization.
First, in a planar polarized tissue, several cells converge at a central apico-basal line where cells go from a square to a more triangular shape to form a cylindrical structure (like a pie cut into <span class="math inline">\(\geq\)</span> 4) (figure <a href="1-Intro.html#fig:constr">1.3</a>B). Such rosettes are usually observed during tissue elongation and are rather short-lived. In a second scenario, cells converge to a central apical point through AC (figure <a href="1-Intro.html#fig:constr">1.3</a>A). This type of rosette is more long-lived and usually does not resolve but already represents a morphologically pre-mature state of the organ to be formed <span class="citation">(<em>26</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:constr"></span>
<img src="figures/intro/constriction.png" alt="Modes of constriction. A Cells converge to a central apical point B Cells converge at a central apico-basal line." width="70%" />
<p class="caption">
Figure 1.3: Modes of constriction. <strong>A</strong> Cells converge to a central apical point <strong>B</strong> Cells converge at a central apico-basal line.
</p>
</div>
</div>
</div>
</div>
<div id="model-organism-and-system" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Model Organism and System</h2>
<p>To study biological phenomena, biologists use a variety of non-human model organisms. While each model organism has its advantages and disadvantages, the choice for a model depends on the scientific question.</p>
<p>To study embryonic development the fresh water fish <em>Danio rerio</em> (also known as eng: <em>zebrafish</em> or <em>ger:</em> Zebrabärbling (figure <a href="1-Intro.html#fig:zebra">1.4</a>) has become an important model organism over the recent years. <em>D. rerio</em> is a diploid organism with a fully sequenced genome (of the human genes 71.4% have at least one <em>D. rerio</em> ortholog, 47% have a one-to-one ortholog <span class="citation">(<em>27</em>)</span>). It has a relatively short alternation of generations (12-16 weeks), a regularly large number of embryos (100 / week &amp; female) and is relatively undemanding in terms of space for breeding. Furthermore, it offers well-established methods for mutagenesis, screening, and generation of transgenic lines. Since its embryos are naturally transparent and develop externally, it is an ideal system for microscopic examination using molecular dyes and tags to visualize <em>inter</em> and <em>intra</em> cellular components even deep within the tissue (<em>e.g.</em> cell nuclei or cell membrane fluorescent tags). Together with advances in imaging techniques, this also allows for high-throughput, high-resolution, long-term <em>in vivo</em> imaging. Especially the expression of fluorescent proteins in a tissue- or organ-specific manner in transparent embryos offers enormous possibilities to address interesting and long-standing open questions.</p>
<p>In nature, zebrafish can be found in the shallow waters of the Indian and Pakistan Ganges inflows. It exhibits an oval body shape and can reach a length of up to 5 cm in adulthood. While females are usually more silverish, males have a brownish back and a yellow-whit belly. Laterally it exhibits its name-giving dark-blue iridescent stripes with silver in between.</p>

<div class="figure" style="text-align: center"><span id="fig:zebra"></span>
<img src="figures/intro/CM_fish.png" alt="Model organism Danio rerio (aquarell by Christine Molenda)" width="50%" />
<p class="caption">
Figure 1.4: Model organism <em>Danio rerio</em> (aquarell by Christine Molenda)
</p>
</div>
<div id="developmental-stages" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Developmental Stages</h3>
<p>A single female may lay 100 eggs per week. Each zygote<a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a> then undergoes the first zygotic cell cycle (<strong>at ~ 0.75 h</strong>). The following two to seven cell cycles (period: <em>Cleavage</em>, <strong>at ~ 2.25 h</strong>) occur directed and synchronous every ~15 min. Cells in this stage are called <em>blastomeres</em>, are incompletely separated from the yolk and remain interconnected by cytoplasmic bridges. The <em>Blastula</em> period (<strong>up to 5.25 h</strong>) is determined by increasingly asynchronous division, flattening of the blastoderm <em>via</em> cell intercalation and lengthening of the cell cycle. This period is also marked by the onset of <em>epiboly</em>, the period when cells in late blastula start to dome while a monolayer of the dome circumfence begins to wrap around the yolk <span class="citation">(<em>28</em>)</span>.</p>
<p>After Blastula and <strong>until 10 h</strong> the <em>Gastrula</em> period takes place, followed by the <em>Segmentation</em> period (<strong>until 24 h</strong>). Both of which are depicted in more detail in figure <a href="1-Intro.html#fig:stages">1.5</a>. At even later stages the embryo starts to elongate posteriorly, grow and develop organs until it first active muscle are present and it starts to swim (figure <a href="1-Intro.html#fig:stages">1.5</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:stages"></span>
<img src="figures/intro/stages.png" alt="Zebrafish embryonic development. Microscopic images are from a time-lapse where 24 embryos were imaged simultaneously in brightfield and at 488 nm Z-Stacks. Representation shows contrast-enhanced EDFs. Bottom row (30-48 hpf) stages are handmade drawings from live embryos made during the first week of my PhD." width="95%" />
<p class="caption">
Figure 1.5: Zebrafish embryonic development. Microscopic images are from a time-lapse where 24 embryos were imaged simultaneously in brightfield and at 488 nm Z-Stacks. Representation shows contrast-enhanced EDFs. Bottom row (30-48 hpf) stages are handmade drawings from live embryos made during the first week of my PhD.
</p>
</div>
</div>
<div id="the-lateral-line-system" class="section level3" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> The Lateral Line System</h3>
<p>The lateral line (LL) system is a mechano-sensory organ that is common to all aquatic vertebrates and evolutionary remnants could even be found in mice <span class="citation">(<em>29</em>)</span>. It enables the animal to sense water movements and therefore to orient itself, and to detect prey and predators. Fully developed, the lateral line system is comprised of hundreds of neuromasts positioned in an orderly pattern all over the animal’s body (figure <a href="1-Intro.html#fig:llsystem">1.6</a>A). Its functional subunits are the neuromasts (NM) (figure <a href="1-Intro.html#fig:llsystem">1.6</a>B-C) that, when fully developed, consist of hair-, support- and mantle cells. To sense water movements, each NM projects kino- and stereo-cilia out of the skin that, upon water-induced deflection, generate action potentials that are transduced <em>via</em> afferent fibers <span class="citation">(<em>30</em>, <em>31</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:llsystem"></span>
<img src="figures/intro/ll_system.jpg" alt="The lateral line system. A modified after (Ghysen et al., 2012) and A.Bergs, 2016 (student presentation at AK Lecaudey). Development of the lateral line system at embryonic, larval and juvenile stage. B Schematic showing a crossection and organization of a single neuromast. C-C’ SEM images of a single, pre-mature (3 dpf) neuromast." width="85%" />
<p class="caption">
Figure 1.6: The lateral line system. <strong>A</strong> modified after (Ghysen <em>et al.</em>, 2012) and A.Bergs, 2016 (student presentation at AK Lecaudey). Development of the lateral line system at embryonic, larval and juvenile stage. <strong>B</strong> Schematic showing a crossection and organization of a single neuromast. <strong>C-C’</strong> SEM images of a single, pre-mature (3 dpf) neuromast.
</p>
</div>
<p>Each NM is first deposited as a premature cluster of about 30 cells from a migrating cell-aggregate called the posterior lateral line primordium (pLLP) (figure <a href="1-Intro.html#fig:llgfp">1.7</a>A). The pLLP delaminates from the pLL placode, caudal to the otic vesicle (figure <a href="1-Intro.html#fig:llgfp">1.7</a>B) at around 20 <em>hours post fertilization</em> (hpf) as a group of ~100 cells. After formation of a mesenchymal-like leading region, it starts migrating along a chemokine gradient positioned at the horizontal myoseptum to the tip of the tail <span class="citation">(<em>30</em>, <em>31</em>)</span>. To ensure the development of a functional organ, several fundamental biological processes like cell migration, morphogenesis, proliferation and cell polarization need to be integrated into the pLLP.
An important breakthrough in LL research has been the development of a transgenic line expressing a membrane-tethered GFP fusion protein (lyn-GFP) that is expressed under the LL specific promotor of <em>cldnb</em> (claudin b<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a>) <span class="citation">(<em>32</em>)</span>, which allows for a much more detailed view and to observe lateral line development <em>in vivo</em>. An example of the fluorescence signal visible at ~60 hpf can be seen in figure <a href="1-Intro.html#fig:llgfp">1.7</a>B.</p>

<div class="figure" style="text-align: center"><span id="fig:llgfp"></span>
<img src="figures/intro/ll_gfp.jpg" alt="Neuromast deposition and pattern. A Scheme showing NM deposition over three timepoints (10 min. interval). Dotted lines are time-tracks of rosettes, which become more concentrated over time. Bottom arrow indicates regions of rosette formation, maturation and deposition within the pLLP (scale bar, 20 \(\mu\)m; 20* WI; ~20 Z-planes; 2.5 \(\mu\)m spacing. MaxIP. Colors inverted.) B Scheme showing the lateral line at end of migration (~60 hpf) and other parts visible through the cldnb:lyn-gfp transgene (as documented through zfin.org) (10X air objective + 1.5X tube lens; four tiles; ~20 Z-planes; 5 \(\mu\)m spacing. MaxIP. Colors inverted)." width="95%" />
<p class="caption">
Figure 1.7: Neuromast deposition and pattern. <strong>A</strong> Scheme showing NM deposition over three timepoints (10 min. interval). Dotted lines are time-tracks of rosettes, which become more concentrated over time. Bottom arrow indicates regions of rosette formation, maturation and deposition within the pLLP (scale bar, 20 <span class="math inline">\(\mu\)</span>m; 20* WI; ~20 Z-planes; 2.5 <span class="math inline">\(\mu\)</span>m spacing. MaxIP. Colors inverted.) <strong>B</strong> Scheme showing the lateral line at end of migration (~60 hpf) and other parts visible through the <em>cldnb:lyn-gfp</em> transgene (as documented through zfin.org) (10X air objective + 1.5X tube lens; four tiles; ~20 Z-planes; 5 <span class="math inline">\(\mu\)</span>m spacing. MaxIP. Colors inverted).
</p>
</div>
</div>
<div id="posterior-lateral-line-primordium" class="section level3" number="1.2.3">
<h3><span class="header-section-number">1.2.3</span> Posterior Lateral Line Primordium</h3>
<p>The pLLP is about 100-150 <span class="math inline">\(\mu\)</span>m in length (depending on deposition cycle) over which it exhibits a diverse surface topology and cellular morphology. Previous research found that the <em>caudal</em> (also <em>posterior</em>), more mesenchymal cells, are leading the path of migration, while the <em>cranial</em> (also <em>anterior</em>), more epithelial cells, are trailing. Towards the leading region the cells are more flat, towards the trailing region the cells become more columnar and increasingly radially organized into formations called epithelial rosettes. During migration the pLLP typically contains 2-3 <em>rosettes</em> (~25-30 cells each), while the most trailing one will eventually be deposited to further mature to a functional NM <span class="citation">(<em>30</em>, <em>33</em>)</span>. Every deposition comes with a loss of cells in the pLLP, but this loss is partially compensated by proliferation during migration. While one study concludes a general spatial heterogeneity in distribution of proliferative cells <span class="citation">(<em>34</em>)</span>, another one suggests a higher proliferative rate specifically near the leading region <span class="citation">(<em>35</em>)</span>.</p>
<div id="rosette-formation" class="section level4" number="1.2.3.1">
<h4><span class="header-section-number">1.2.3.1</span> Rosette formation</h4>
<p>The onset of morphological and functional changes is determined by signaling of FGF, which is mostly active in the trailing region. Causal for expression of FGF is a signaling center of WNT in the leading region, which promotes expression of FGF ligands <em>Fgf-3</em> and <em>-10</em> <span class="citation">(<em>36</em>)</span>. Those ligands then diffuse to the trailing domain where binding through <em>FGF receptor 1</em> (Fgfr1) triggers a signaling cascade through which the cells become more columnar, apically constricted and eventually re-organize into epithelial rosettes <span class="citation">(<em>35</em>, <em>37</em>, <em>38</em>)</span>. Concurrently, cells of the WNT signaling center themselves are not competent to FGF signaling, which is achieved through expression of <em>Sef</em>, an intracellular antagonist of Fgfr1 signaling <span class="citation">(<em>39</em>)</span>.</p>
<p>It was shown that rosette formation in the lateral line primordium (LLP) is an important morphological feature for a lumen to form on top of the rosette which, filled with FGF, acts as a locally enriched source of FGF signaling <span class="citation">(<em>40</em>)</span>.</p>
</div>
<div id="intro-atoh" class="section level4" number="1.2.3.2">
<h4><span class="header-section-number">1.2.3.2</span> Hair cell specification</h4>
<p>Downstream of FGF lies the expression of the transcription factor (TF) Atoh1a, which gives cells the potential to become sensory HCs <span class="citation">(<em>35</em>)</span>.</p>
<p>The current model suggests that:</p>
<ol style="list-style-type: decimal">
<li>FGF initiates expression of <em>atoh1a</em> and <em>deltaA</em>, where DeltaA activates Notch in neighboring cells to inhibit expression of <em>atoh1a</em> in those.</li>
<li>Atoh1a in turn suppresses competence for FGF and initiates expression of <em>atoh1b</em> and <em>deltaD</em> which act synergistically with DeltaA in the center cell of the forming rosette.</li>
<li>While the latter acts synergistic with DeltaA, Atoh1b in turn drives expression of <em>atoh1a</em> <span class="citation">(<em>41</em>)</span>.</li>
<li>Once a prospective HC is specified it will itself become a source for Fgf-10. By this process adjacent cells are laterally inhibited and determined as <em>support cells</em> (figure <a href="1-Intro.html#fig:llsystem">1.6</a>B), still capable of receiving FGF signals <em>via</em> Fgfr1.</li>
</ol>
<p>Just before the most trailing rosette is deposited, its prospective HC will undergo a final division to form a doublet of sensory HCs <span class="citation">(<em>42</em>, <em>43</em>)</span>.</p>
</div>
</div>
<div id="shroom3-in-the-pllp" class="section level3" number="1.2.4">
<h3><span class="header-section-number">1.2.4</span> Shroom3 in the pLLP</h3>
<p>The Shroom protein family is mostly conserved through animal evolution (figure <a href="supplement.html#fig:suppshrmort">4.3</a>) and involved in contraction of the actomyosin network (<em>e.g.</em> during AC), which has been confirmed in several studies investigating <em>e.g.</em> epithelial planar remodeling, neural tube morphogenesis and <em>Xenopus</em> bottle cells <span class="citation">(<em>44</em>–<em>49</em>)</span>.</p>
<div id="intro-shroom" class="section level4" number="1.2.4.1">
<h4><span class="header-section-number">1.2.4.1</span> Recent research</h4>
<p>Shroom proteins have three characteristic domains (1) a <em>PDZ</em> domain close to the N-terminus that interact with other proteins with PDZ-binding domains and (2) two <em>Apx/Shroom</em> domains (ASD-1 and -2), the latter being close to the proteins C-terminus. The Shroom domains may interact with proteins containing a <em>Shroom binding domain</em> (SBD) such as Rock which promotes phosphorylation of NMII (figure <a href="1-Intro.html#fig:shrminteract">1.8</a>) <span class="citation">(<em>50</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:shrminteract"></span>
<img src="figures/intro/shrm3_interaction.png" alt="Shroom3 functional domains and mode of operation. Sequence of events numbered from 1-4. Approximate scale jumps indicated at arrows." width="95%" />
<p class="caption">
Figure 1.8: Shroom3 functional domains and mode of operation. Sequence of events numbered from 1-4. Approximate scale jumps indicated at arrows.
</p>
</div>
<p>While in all of the above-mentioned studies Shroom3 was the focus of interest, in zebrafish and most other model organisms there are four paralogs. A previous study performed by a colleague done on Shroom3 morphants in <em>D. rerio</em> <span class="citation">(<em>1</em>)</span> indicates that Shroom3 is also necessary for AC and rosette formation in the migrating pLLP. In summary she was able to show that…</p>
<ol style="list-style-type: decimal">
<li><em>shroom3</em> is expressed in the pLLP from stages 24 – 48 hpf.</li>
<li><em>shroom3</em> is expressed downstream of FGF signaling, which was shown <em>via</em> treatment with an Fgfr inhibiting drug.</li>
<li>Shroom3 localizes to rosette centers, which was accomplished by generating a transgenic line expressing a <em>shroom3-tagRFP</em> fusion protein under the control of a heat-shock<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a> promotor (figure <a href="1-Intro.html#fig:shrmernst">1.9</a>A).</li>
<li>Rosette formation is impaired in MO injected embryos, which was shown quantitatively by using a specifically trained <em>rosette detector</em> <span class="citation">(<em>51</em>)</span> to count and weight single rosettes.</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:shrmernst"></span>
<img src="figures/intro/shrm_ernst.png" alt="Shroom3 in rosette formation (adapted from Ernst et al., 2012) A composite MaxIPs of membrane label and fusion protein showing the localization of Shroom within the pLLP B uninjected control and shroom3 MO injected MaxIPs B’ Heat-maps of rosette detector score." width="95%" />
<p class="caption">
Figure 1.9: Shroom3 in rosette formation (adapted from Ernst et al., 2012) <strong>A</strong> composite MaxIPs of membrane label and fusion protein showing the localization of Shroom within the pLLP <strong>B</strong> uninjected control and shroom3 MO injected MaxIPs <strong>B’</strong> Heat-maps of rosette detector score.
</p>
</div>
</div>
<div id="current-model" class="section level4" number="1.2.4.2">
<h4><span class="header-section-number">1.2.4.2</span> Current Model</h4>
<p>Based on these and previous results, the current model for apical constriction in the pLLP assumes that (1) expression of <em>shroom3</em> is induced by FGF signaling (2) Shroom3 binds Rock and translocates it to the AJC to (3) mediate phosphorylation of NMII which (4) induces contraction of the actin network and AC (figure <a href="1-Intro.html#fig:shrmmodel">1.10</a>). Furthermore, AC is necessary for rosette assembly and subsequent NM deposition. In conclusion the current understanding is that without Shroom3 AC and rosette formation are impaired, leading to a defect in NMs and NM deposition.</p>

<div class="figure" style="text-align: center"><span id="fig:shrmmodel"></span>
<img src="figures/intro/shrm_model.png" alt="Shroom3 current model" width="50%" />
<p class="caption">
Figure 1.10: Shroom3 current model
</p>
</div>
</div>
</div>
<div id="shroom3-mutants" class="section level3" number="1.2.5">
<h3><span class="header-section-number">1.2.5</span> <em>shroom3</em> mutants</h3>
<p>Morpholino injection allows transient gene knockdown in various species which has been broadly used over recent years. Although they have been and are still a useful tool for the zebrafish community, there are a number of limitations.</p>
<p>Those include…</p>
<ul>
<li>they need to be injected for each experiment. Even if the person injecting is well trained, there is always some loss in embryos. Furthermore, injection leads to a delay in development</li>
<li>the induction of non-specific phenotypes due to activation of, among others, p53-mediated cell death.</li>
<li>the degradation of the morpholino, which allows to analyze the phenotype over a limited period of time only (3 to 5 days after injection)</li>
</ul>
<p>Because of these limitations, a former colleague in the lab had generated a mutant once it became technically possible using Transcription activator-like effector nuclease (TALEN) to confirm and further study the role of Shroom3 during morphogenesis. An 8 bp deletion in the SD2 domain was isolated and maintained as a stable line. This mutation leads to a premature STOP codon disrupting the SD2 domain, thereby inhibiting Shroom3’s function.</p>
<p>While birth rates follow a distribution of Mendelian inheritance (after genotyping at 3 months of age), <em>homozygous</em> mutant adults have a shortened lifespan (~6-9 months). Shroom3 mutants are morphologically similar to their siblings, however their gill flaps seem to be increased in size, swollen, and not exactly streamlined with the body. This is also evident by an increased frequency of gill flap beating. Like the MO injected embryos, in a first qualitative analysis of the phenotype the pLLP exhibits a noticeable defect in rosette assembly. To our surprise however, the number of NMs deposited at the end of migration was significantly increased in the mutants as compared to the controls.</p>
</div>
</div>
<div id="open-questions-and-motivation" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Open Questions and Motivation</h2>
<p>The main objectives of my thesis were…</p>
<ol style="list-style-type: decimal">
<li>to characterize the newly generated <em>shroom3</em> mutant phenotype <em>via</em> quantitative imaging methods</li>
<li>to develop a methodology for an automated, more precise, higher throughput and less invasive quantification of the pLLP and LL phenotypes</li>
<li>to use the mutant phenotype to better understand the relationship between rosette assembly and NM deposition</li>
<li>to analyze the feedback between morphological changes (apical constriction and mediated rosette assembly) and cell fate specification in the pLLP.</li>
</ol>

</div>
</div>
<div class="footnotes">
<hr />
<ol start="4">
<li id="fn4"><p>catenins are regulators of cell-cell adhesion and gene transcription<a href="1-Intro.html#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>one of the earliest differentiating layer of cells. Cells of the mesoderm will <em>e.g.</em> form the musculature.<a href="1-Intro.html#fnref5" class="footnote-back">↩︎</a></p></li>
<li id="fn6"><p>enzymes that transfer energy to specific substrates<a href="1-Intro.html#fnref6" class="footnote-back">↩︎</a></p></li>
<li id="fn7"><p>Even though the expression Form follows function <span class="citation">(<em>9</em>)</span> is usually found in design and architecture, it formulates the general idea that any objects form is (or in design should) be shaped by the requirements to it.<a href="1-Intro.html#fnref7" class="footnote-back">↩︎</a></p></li>
<li id="fn8"><p>first diploid cell after fertilization<a href="1-Intro.html#fnref8" class="footnote-back">↩︎</a></p></li>
<li id="fn9"><p>construct name: <em>Tg(-8.0cldnb:lynGFP)</em>; ZFIN ID: ZDB-TGCONSTRCT-070117-15<a href="1-Intro.html#fnref9" class="footnote-back">↩︎</a></p></li>
<li id="fn10"><p>heat shock proteins are enzymes that assist protein folding whose expression is heat activated<a href="1-Intro.html#fnref10" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="summary.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-mat-met.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis_dsk.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
