<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-Intro.html"/>
<link rel="next" href="3-res.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { font-weight: bold; } /* Alert */
code span.an { font-style: italic; } /* Annotation */
code span.cf { font-weight: bold; } /* ControlFlow */
code span.co { font-style: italic; } /* Comment */
code span.cv { font-style: italic; } /* CommentVar */
code span.do { font-style: italic; } /* Documentation */
code span.dt { text-decoration: underline; } /* DataType */
code span.er { font-weight: bold; } /* Error */
code span.in { font-style: italic; } /* Information */
code span.kw { font-weight: bold; } /* Keyword */
code span.pp { font-weight: bold; } /* Preprocessor */
code span.wa { font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Index</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#print-version"><i class="fa fa-check"></i>Print version</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#electronic-version"><i class="fa fa-check"></i>Electronic version</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="list-of-publications.html"><a href="list-of-publications.html"><i class="fa fa-check"></i>List of publications</a></li>
<li class="chapter" data-level="" data-path="list-of-abbreviations.html"><a href="list-of-abbreviations.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#intro-morph"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#open-questions-and-motivation"><i class="fa fa-check"></i><b>1.3</b> Open Questions and Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probe list</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mos"><i class="fa fa-check"></i><b>2.1.9</b> ISH probe list</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.10</b> Hardware</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.11</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Data Strategy and Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.2</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.3</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.4</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.3</b> Ground Truth</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cluster-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Cluster Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#morphometric-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Morphometric analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.4</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#methods"><i class="fa fa-check"></i><b>3.1</b> Methods</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#standardized-mounting"><i class="fa fa-check"></i><b>3.1.1</b> Standardized mounting</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#anallzr2d---cc-segmentation"><i class="fa fa-check"></i><b>3.1.2</b> anaLLzr2D - CC segmentation</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#anallzr3d---pllp-segmentation"><i class="fa fa-check"></i><b>3.1.3</b> anaLLzr3D - pLLP segmentation</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#intro-phen"><i class="fa fa-check"></i><b>3.2.1</b> Phenotype description</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.3</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.4</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#apical-constriction-3"><i class="fa fa-check"></i><b>3.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.6</b> Haircell Specification</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#methods-1"><i class="fa fa-check"></i><b>4.1</b> Methods</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#mounting"><i class="fa fa-check"></i><b>4.1.1</b> Mounting</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.2.1</b> Rosettes</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-4"><i class="fa fa-check"></i><b>4.2.2</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.2.3</b> Model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#methods-2"><i class="fa fa-check"></i><b>5.1</b> Methods</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-discussion.html"><a href="5-discussion.html#mounting-1"><i class="fa fa-check"></i><b>5.1.1</b> Mounting</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#shroom3-2"><i class="fa fa-check"></i><b>5.2</b> Shroom3</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-discussion.html"><a href="5-discussion.html#lateral-line"><i class="fa fa-check"></i><b>5.2.1</b> Lateral Line</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.2.2</b> Proliferation</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection-1"><i class="fa fa-check"></i><b>5.2.3</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification"><i class="fa fa-check"></i><b>5.2.4</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.2.5</b> Single cell reconstruction and AC measurement</a></li>
</ul></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mat-met" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="mat" class="section level2">
<h2><span class="header-section-number">2.1</span> Materials</h2>
<div id="mat-chem" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Chemicals</h3>
<table>
<caption><span id="tab:mat-chem">Table 2.1: </span> Chemicals</caption>
<colgroup>
<col width="28%" />
<col width="54%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Chemical</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Agarose</td>
<td align="center">Roth</td>
<td align="left">6351.2</td>
</tr>
<tr class="even">
<td align="right">Agar-Agar</td>
<td align="center">Roth</td>
<td align="left">5210.3</td>
</tr>
<tr class="odd">
<td align="right">Ampicillin</td>
<td align="center">Roth</td>
<td align="left">K029.2</td>
</tr>
<tr class="even">
<td align="right">ATP</td>
<td align="center">Epicentre</td>
<td align="left">E311K</td>
</tr>
<tr class="odd">
<td align="right">Blocking Reagent</td>
<td align="center">Roche</td>
<td align="left">11096176001</td>
</tr>
<tr class="even">
<td align="right">BCIP</td>
<td align="center">Fermentas</td>
<td align="left">R0822</td>
</tr>
<tr class="odd">
<td align="right">CaCl</td>
<td align="center">Roth</td>
<td align="left">886.1</td>
</tr>
<tr class="even">
<td align="right">Calyculin</td>
<td align="center">Sigma</td>
<td align="left">208851</td>
</tr>
<tr class="odd">
<td align="right">DAPI</td>
<td align="center">Sigma</td>
<td align="left">D9542</td>
</tr>
<tr class="even">
<td align="right">DIG RNA Mix</td>
<td align="center">Roche</td>
<td align="left">11277073910</td>
</tr>
<tr class="odd">
<td align="right">DMSO</td>
<td align="center">Roth</td>
<td align="left">4720.2</td>
</tr>
<tr class="even">
<td align="right">EtBr</td>
<td align="center">Roth</td>
<td align="left">2218.3</td>
</tr>
<tr class="odd">
<td align="right">EtOH</td>
<td align="center">Roth</td>
<td align="left">9065.3</td>
</tr>
<tr class="even">
<td align="right">Formaldehyde</td>
<td align="center">Roth</td>
<td align="left">7398.1</td>
</tr>
<tr class="odd">
<td align="right">Formamide</td>
<td align="center">Roth</td>
<td align="left">P040.1</td>
</tr>
<tr class="even">
<td align="right">Glycerol</td>
<td align="center">Roth</td>
<td align="left">3783.2</td>
</tr>
<tr class="odd">
<td align="right">IPTG</td>
<td align="center">Thermo</td>
<td align="left">R1171</td>
</tr>
<tr class="even">
<td align="right">KCl</td>
<td align="center">Roth</td>
<td align="left">P017.1</td>
</tr>
<tr class="odd">
<td align="right">Low melting point Agarose</td>
<td align="center">Roth</td>
<td align="left">A9539</td>
</tr>
<tr class="even">
<td align="right">Maleic Acid</td>
<td align="center">Roth</td>
<td align="left">3810.3</td>
</tr>
<tr class="odd">
<td align="right">MeOH</td>
<td align="center">Roth</td>
<td align="left">CP43.3</td>
</tr>
<tr class="even">
<td align="right">MgCl</td>
<td align="center">Roth</td>
<td align="left">2189.1</td>
</tr>
<tr class="odd">
<td align="right">NaCl</td>
<td align="center">Roth</td>
<td align="left">9265.2</td>
</tr>
<tr class="even">
<td align="right">NaHCO</td>
<td align="center">Roth</td>
<td align="left">855.1</td>
</tr>
<tr class="odd">
<td align="right">NaOH</td>
<td align="center">Roth</td>
<td align="left">6771.3</td>
</tr>
<tr class="even">
<td align="right">NGS</td>
<td align="center">Sigma</td>
<td align="left">C6767</td>
</tr>
<tr class="odd">
<td align="right"><em>p</em>-Formaldehyd</td>
<td align="center">Sigma</td>
<td align="left">P6148</td>
</tr>
<tr class="even">
<td align="right">Phenol Red</td>
<td align="center">Sigma</td>
<td align="left">P0290</td>
</tr>
<tr class="odd">
<td align="right">Propan-2-ol</td>
<td align="center">VWR</td>
<td align="left">20842330</td>
</tr>
<tr class="even">
<td align="right">Proteinase K</td>
<td align="center">Roth</td>
<td align="left">7528.4</td>
</tr>
<tr class="odd">
<td align="right">PTU</td>
<td align="center">Sigma</td>
<td align="left">P7629</td>
</tr>
<tr class="even">
<td align="right">Rockout</td>
<td align="center">Sigma</td>
<td align="left">555553</td>
</tr>
<tr class="odd">
<td align="right">SSC</td>
<td align="center">Roth</td>
<td align="left">1232.1</td>
</tr>
<tr class="even">
<td align="right">SU5402</td>
<td align="center">CALBIOCHEM</td>
<td align="left">572630</td>
</tr>
<tr class="odd">
<td align="right">Torula RNA</td>
<td align="center">Sigma</td>
<td align="left">R6625</td>
</tr>
<tr class="even">
<td align="right">Tricaine</td>
<td align="center">Sigma</td>
<td align="left">A5040</td>
</tr>
<tr class="odd">
<td align="right">Tris Base</td>
<td align="center">Roth</td>
<td align="left">4855.2</td>
</tr>
<tr class="even">
<td align="right">Triton-X100</td>
<td align="center">Roth</td>
<td align="left">3051.2</td>
</tr>
<tr class="odd">
<td align="right">Trizol</td>
<td align="center">mbion</td>
<td align="left">15596018</td>
</tr>
<tr class="even">
<td align="right">Tween20</td>
<td align="center">Sigma</td>
<td align="left">P1379</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-sol" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Solutions</h3>
<table>
<caption><span id="tab:mat-sol">Table 2.2: </span> Solutions</caption>
<colgroup>
<col width="23%" />
<col width="59%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Solution</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Cut Smart Buffer</td>
<td align="center">NEB</td>
<td align="left">B7204S</td>
</tr>
<tr class="even">
<td align="right">Generuler 100 bp</td>
<td align="center">Thermo</td>
<td align="left">SM0241</td>
</tr>
<tr class="odd">
<td align="right">Generuler 1kb</td>
<td align="center">Thermo</td>
<td align="left">SM0311</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-anitb" class="section level3">
<h3><span class="header-section-number">2.1.3</span> Antibodies</h3>
<table>
<caption><span id="tab:mat-antib">Table 2.3: </span> Antibodies</caption>
<colgroup>
<col width="23%" />
<col width="38%" />
<col width="21%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Antibody</strong></th>
<th align="center">Company / Provider</th>
<th align="center">concentration</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Anti-Digoxigenin</td>
<td align="center">Roche</td>
<td align="center">1:200</td>
<td align="left">11093274910</td>
</tr>
<tr class="even">
<td align="right">anti-GFP</td>
<td align="center">Torrey Pines</td>
<td align="center">1:200</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">Anti-TAZ (rabbit)</td>
<td align="center">Cell Signaling Technology</td>
<td align="center">1:200</td>
<td align="left">D24E4</td>
</tr>
<tr class="even">
<td align="right">Alexa Fluor488</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">710369</td>
</tr>
<tr class="odd">
<td align="right">Alexa Fluor555</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">Z25005</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-enz" class="section level3">
<h3><span class="header-section-number">2.1.4</span> Enzymes</h3>
<table>
<caption><span id="tab:mat-enz">Table 2.4: </span> Enzymes</caption>
<colgroup>
<col width="27%" />
<col width="55%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Enzyme</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">BtsCI</td>
<td align="center">NEB</td>
<td align="left">R0647</td>
</tr>
<tr class="even">
<td align="right">DdeI</td>
<td align="center">NEB</td>
<td align="left">R0175</td>
</tr>
<tr class="odd">
<td align="right">HaeIII</td>
<td align="center">NEB</td>
<td align="left">R0108</td>
</tr>
<tr class="even">
<td align="right">MnlII</td>
<td align="center">NEB</td>
<td align="left">R0163</td>
</tr>
<tr class="odd">
<td align="right">NlaIII</td>
<td align="center">NEB</td>
<td align="left">R0125</td>
</tr>
<tr class="even">
<td align="right">NsiI-HF</td>
<td align="center">NEB</td>
<td align="left">R3127</td>
</tr>
<tr class="odd">
<td align="right">Phusion Polymerase</td>
<td align="center">NEB</td>
<td align="left">M0530L</td>
</tr>
<tr class="even">
<td align="right">Pronase</td>
<td align="center">Sigma</td>
<td align="left">P5147</td>
</tr>
<tr class="odd">
<td align="right">Ribolock</td>
<td align="center">Thermo</td>
<td align="left">EO0381</td>
</tr>
<tr class="even">
<td align="right">RNase A</td>
<td align="center">Quiagen</td>
<td align="left">1006657</td>
</tr>
<tr class="odd">
<td align="right">RNase H</td>
<td align="center">NEB</td>
<td align="left">M0297L</td>
</tr>
<tr class="even">
<td align="right">SP6 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0131</td>
</tr>
<tr class="odd">
<td align="right">T4 Ligase</td>
<td align="center">NEB</td>
<td align="left">M0202T</td>
</tr>
<tr class="even">
<td align="right">T7 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0111</td>
</tr>
<tr class="odd">
<td align="right">Taq DNA Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">10342-020</td>
</tr>
<tr class="even">
<td align="right">Taq DNA Polymerase</td>
<td align="center">VWR</td>
<td align="left">733-1301</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mobikits" class="section level3">
<h3><span class="header-section-number">2.1.5</span> Molecular Biology Kits</h3>
<table>
<caption><span id="tab:mat-mobikits">Table 2.5: </span> Molecular Biology Kits</caption>
<colgroup>
<col width="54%" />
<col width="28%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Kit</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">EdU Click-iT</td>
<td align="center">Invitrogen</td>
<td align="left">MP 10083</td>
</tr>
<tr class="even">
<td align="right">mMessage mMachine Sp6 Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">AM1340</td>
</tr>
<tr class="odd">
<td align="right">PCR &amp; Gel Clean-Up</td>
<td align="center">Sigma</td>
<td align="left">NA1020</td>
</tr>
<tr class="even">
<td align="right">pGEM-T TA Cloning</td>
<td align="center">Promega</td>
<td align="left">A3600</td>
</tr>
<tr class="odd">
<td align="right">Superscript III cDNA Synthesis</td>
<td align="center">Thermo</td>
<td align="left">18080051</td>
</tr>
<tr class="even">
<td align="right">Wizard SV Gel and PCR Clean-Up</td>
<td align="center">Promega</td>
<td align="left">A9282</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-buff" class="section level3">
<h3><span class="header-section-number">2.1.6</span> Buffers</h3>
<table>
<caption><span id="tab:mat-buff">Table 2.6: </span> Buffers</caption>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Buffer</strong></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Blocking Reagent (BR)</td>
<td>2% BR in maleic buffer + 5% serum</td>
</tr>
<tr class="even">
<td align="left">E3</td>
<td><span class="citation">(<em>47</em>)</span></td>
</tr>
<tr class="odd">
<td align="left">Hybridization buffer</td>
<td>50% Formamide + 25 % 20x SSC + 50mg/mL Heparine + mQ</td>
</tr>
<tr class="even">
<td align="left">Maleic buffer</td>
<td>250 mM maleic acid + 5M NaCl + 10% 0.1% Tween-20 + mQ</td>
</tr>
<tr class="odd">
<td align="left">NTMT</td>
<td>75 mM NaCl + 15 mM MgCl + 15 mM</td>
</tr>
<tr class="even">
<td align="left">PBS</td>
<td>2.7 mM KCl + 12 mM HPO</td>
</tr>
<tr class="odd">
<td align="left">PBST</td>
<td>PBS + 0.1% Tween20</td>
</tr>
<tr class="even">
<td align="left">PBDT</td>
<td>PBS + 1% BSA + 1% DMSO + 0.3% Triton</td>
</tr>
<tr class="odd">
<td align="left">PFA</td>
<td>4% paraformaldehyde in PBS</td>
</tr>
<tr class="even">
<td align="left">P1</td>
<td>50 mM Tris-HCl pH 8.0 + 10 mM EDTA pH 8.0 + 100 µg/mL RNAse</td>
</tr>
<tr class="odd">
<td align="left">P2</td>
<td>1M NaOH + 10 % (w/v) SDS</td>
</tr>
<tr class="even">
<td align="left">P3</td>
<td>3M KOAc pH 5.5</td>
</tr>
<tr class="odd">
<td align="left">TNT</td>
<td>50 mM Tris-HCl pH 8.0 + 100 mM NaCl + 0.1% Tween-20</td>
</tr>
<tr class="even">
<td align="left">Tricaine</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-lines" class="section level3">
<h3><span class="header-section-number">2.1.7</span> Zebrafish lines</h3>
<table>
<caption><span id="tab:mat-lines">Table 2.7: </span> Zebrafish lines</caption>
<colgroup>
<col width="13%" />
<col width="31%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Allele</strong></th>
<th align="center"><strong>name</strong></th>
<th align="left"><strong>zfin</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">zf106Tg</td>
<td align="center"><em>cldnb:lyn-gfp</em></td>
<td align="left"><a href="//zfin.org/ZDB-ALT-060919-2"><em>Tg(-8.0cldnb:LY-EGFP)</em></a></td>
</tr>
<tr class="even">
<td align="right">fu13Tg</td>
<td align="center"><em>cxcr4b</em>(BAC):<em>H2BRFP</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">nns8Tg</td>
<td align="center"><em>atoh1a:Tom</em></td>
<td align="left"><a href="//zfin.org/ZDB-FISH-150901-21622"><em>atoh1a</em></a></td>
</tr>
<tr class="even">
<td align="right">fu50</td>
<td align="center"><em>shroom3</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">m1274Tg</td>
<td align="center"><em>hsp70:shr3v1FL-taqRFP</em></td>
<td align="left">-</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-probes" class="section level3">
<h3><span class="header-section-number">2.1.8</span> ISH probe list</h3>
<table>
<caption><span id="tab:mat-probes">Table 2.8: </span> ISH probes</caption>
<colgroup>
<col width="11%" />
<col width="88%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">atoh1a</td>
<td align="left">see <span class="citation">(<em>32</em>)</span></td>
</tr>
<tr class="even">
<td align="left">deltaD</td>
<td align="left">see <span class="citation">(<em>48</em>)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mos" class="section level3">
<h3><span class="header-section-number">2.1.9</span> ISH probe list</h3>
<table>
<caption><span id="tab:mat-mos">Table 2.9: </span> Morpholinos</caption>
<colgroup>
<col width="11%" />
<col width="32%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
<th align="left"><strong>Concentration</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">MoAtoh1a</td>
<td align="left">see <span class="citation">(<em>32</em>, <em>49</em>)</span></td>
<td align="left">0.4 ng/mM</td>
</tr>
<tr class="even">
<td align="left">p53</td>
<td align="left">see <span class="citation">(<em>50</em>)</span></td>
<td align="left">2 ng/mM</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-hrdwr" class="section level3">
<h3><span class="header-section-number">2.1.10</span> Hardware</h3>
<div id="mat-stamp" class="section level4">
<h4><span class="header-section-number">2.1.10.1</span> Mounting Stamp</h4>
<p>An stl file for 3D printing can be found at <a href="https://github.com/KleinhansDa/3DModels">github.com/KleinhansDa/3DModels</a></p>
</div>
<div id="mat-SD" class="section level4">
<h4><span class="header-section-number">2.1.10.2</span> Spinning Disc Microscopy</h4>
<table>
<caption><span id="tab:SDcomp">Table 2.10: </span> Spinning Disc system components</caption>
<colgroup>
<col width="22%" />
<col width="18%" />
<col width="25%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Microscope</td>
<td align="center">Nikon</td>
<td align="center">Eclipse Ti-E</td>
<td align="left">fully motorized</td>
</tr>
<tr class="even">
<td align="right">PFS</td>
<td align="center">Nikon</td>
<td align="center">Perfect focus system</td>
<td align="left">Z repositioning</td>
</tr>
<tr class="odd">
<td align="right">XY-table</td>
<td align="center">Merzhaeuser</td>
<td align="center">XY motorized table</td>
<td align="left">1 <span class="math inline">\(\mu\)</span>m accuracy</td>
</tr>
<tr class="even">
<td align="right">Piezo</td>
<td align="center"></td>
<td align="center">Piezo Z-table</td>
<td align="left">300 <span class="math inline">\(\mu\)</span>m scan range</td>
</tr>
<tr class="odd">
<td align="right">SD system</td>
<td align="center">Yokogawa</td>
<td align="center">CSU-W1</td>
<td align="left">50 <span class="math inline">\(\mu\)</span>m pattern</td>
</tr>
<tr class="even">
<td align="right">Laser</td>
<td align="center"></td>
<td align="center">Laser Combiner</td>
<td align="left">see table <a href="2-mat-met.html#tab:lasers">2.11</a></td>
</tr>
<tr class="odd">
<td align="right">FRAPPA</td>
<td align="center">Revolution</td>
<td align="center">FRAPPA</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="right">Borealis</td>
<td align="center">Borealis</td>
<td align="center">Borealis</td>
<td align="left">flat field correction</td>
</tr>
<tr class="odd">
<td align="right">sCMOS</td>
<td align="center">Andor</td>
<td align="center">ZYLA PLUS</td>
<td align="left">4.2Mpix; 82%QE</td>
</tr>
<tr class="even">
<td align="right">Immersion</td>
<td align="center">Merzhaeuser</td>
<td align="center">Liquid Dispenser</td>
<td align="left">-</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:lasers">Table 2.11: </span> Available lasers</caption>
<colgroup>
<col width="40%" />
<col width="20%" />
<col width="40%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Lasers</strong></th>
<th align="center">Type</th>
<th align="left">Power</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">405 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">445 nm</td>
<td align="center">diode</td>
<td align="left">80 mW</td>
</tr>
<tr class="odd">
<td align="right">488 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">561 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="odd">
<td align="right">640 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:objectives">Table 2.12: </span> Available objectives</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="17%" />
<col width="15%" />
<col width="9%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Objective</strong></th>
<th align="center">Company</th>
<th align="center">Type</th>
<th align="center">Immersion</th>
<th align="center">N.A.</th>
<th align="left">working distance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">10x</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">air</td>
<td align="center">0.45</td>
<td align="left">4.00 mm</td>
</tr>
<tr class="even">
<td align="right">20X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">0.95</td>
<td align="left">0.95 mm</td>
</tr>
<tr class="odd">
<td align="right">40X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.15</td>
<td align="left">0.60 mm</td>
</tr>
<tr class="even">
<td align="right">60X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.20</td>
<td align="left">0.30 mm</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-work" class="section level4">
<h4><span class="header-section-number">2.1.10.3</span> Workstation</h4>
<p>Statistical computation and image analysis were done on a Fujitsu Siemens (FS) Workstation CELSIUSM740 with the following hardware components…</p>
<table>
<caption><span id="tab:PCcomp">Table 2.13: </span> Workstation hardware components</caption>
<colgroup>
<col width="21%" />
<col width="22%" />
<col width="25%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">CPU</td>
<td align="center">Intel</td>
<td align="center">Xeon E5-1660v4</td>
<td align="left">3.2 GHz, 20MB, 8cores</td>
</tr>
<tr class="even">
<td align="right">RAM</td>
<td align="center">Fujitsu</td>
<td align="center">-</td>
<td align="left">4x16GB DDR4-2400</td>
</tr>
<tr class="odd">
<td align="right">Graphics</td>
<td align="center">NVIDIA</td>
<td align="center">Quadro M4000</td>
<td align="left">8GB RAM</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="mat-sftwr" class="section level3">
<h3><span class="header-section-number">2.1.11</span> Software</h3>
<table>
<caption><span id="tab:sftwr">Table 2.14: </span> Used software</caption>
<colgroup>
<col width="39%" />
<col width="19%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Software</strong></th>
<th align="center">Version</th>
<th align="left">web</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Imagej FIJI</td>
<td align="center">1.48</td>
<td align="left"><a href="https://fiji.sc/" class="uri">https://fiji.sc/</a></td>
</tr>
<tr class="even">
<td align="right">RStudio</td>
<td align="center">1.0.153</td>
<td align="left"><a href="https://www.rstudio.com/" class="uri">https://www.rstudio.com/</a></td>
</tr>
<tr class="odd">
<td align="right">Ubuntu</td>
<td align="center">17.1</td>
<td align="left"><a href="https://www.ubuntu.com/" class="uri">https://www.ubuntu.com/</a></td>
</tr>
<tr class="even">
<td align="right">Windows 10 Pro</td>
<td align="center">10.0.16299</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">Total Commander</td>
<td align="center">9.0</td>
<td align="left"><a href="http://ghisler.com/" class="uri">http://ghisler.com/</a></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="met" class="section level2">
<h2><span class="header-section-number">2.2</span> Methods</h2>
<div id="data-strategy-and-analysis" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Data Strategy and Analysis</h3>
<p>Due to the history of Developmental Biology and the complexity of biological processes <em>per se</em>, the field heavily relies on image data. Since the advent of electronic imaging techniques<a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a> scientific image data can be processed and analyzed <em>in silico</em>. To make use of the possibility of</p>
<ol style="list-style-type: decimal">
<li>live imaging, which (as compared to fixation techniques) conserves the cellular integrity and morphology while also offers the possibility of recording time-lapses</li>
<li>the optically clear specimen and</li>
<li>high throughput image analysis and state-of-the-art data science using algorithmic implementations,</li>
</ol>
<p>the three following points were paid special attention to.</p>
<div id="sample-preparation" class="section level4">
<h4><span class="header-section-number">2.2.1.1</span> Sample Preparation</h4>
<p>For fluorescence microscopy zebrafish embryos are usually immersed in a 1% solution of <em>low melting-point agarose</em> (LMPA) solution and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> eight to ten embryos <em>per</em> dish. To make use of the high number of offspring a single zebrafish female may lay, which rapidly leads to a sample number of more than 300, a new sample preparation technique was devised that allows for (1) a four to five times increase in samples per dish (2) a facilitated navigation <em>via</em> a grid-like orientation through the samples and (3) an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope. For details, see Materials and Methods section <a href="2-mat-met.html#mount-met">2.2.3.5</a> and Kleinhans <em>et al.</em>, 2019<span class="citation">(<em>51</em>)</span>.</p>
</div>
<div id="imaging" class="section level4">
<h4><span class="header-section-number">2.2.1.2</span> Imaging</h4>
<p>Technically, speed and sensitivity are most important for live imaging. Considering these two parameters a light-sheet <span class="citation">(<em>52</em>)</span> fluorescence microscope (LSFM) would be the best fit. However, LSFMs also have several limitations. First, due to the sample preparation methods available the number of samples that can be imaged at a time is highly restricted. Second, for subcellular resolution a high magnification is required, which is limited by working distances and third - for optimal image analysis a high Signal-to-Noise ratio (SNR) and numerical aperture (N.A.) is preferable. Therefore a spinning disc <span class="citation">(<em>53</em>)</span> system was chosen for most of the imaging.
The system makes use of (1) an extra-large field of view (FOV) ideal for large specimen (2) the possibility of a high degree of automation with state-of-the-art software and (3) a water dispension system for long-term water immersion imaging. For details about the system see Materials and Methods section <a href="2-mat-met.html#mat-SD">2.1.10.2</a>.</p>
</div>
<div id="data-handling" class="section level4">
<h4><span class="header-section-number">2.2.1.3</span> Data handling</h4>
<p>Subsequently after data acquisition and pre-processing the image data was transferred from the microscope system to the labs main workstation. To uniquely identify each file and have them appear in a structured manner, a file-naming system was established after the following structure</p>

<p>Where <em>stage</em> would e.g. be 32hpf, <em>group</em> would be a genotype or drug treatment, <em>id</em> would be a positional identifier on an imaging dish like B1P01<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> and <em>date</em> would be a date in the form of YYMMDD.</p>
</div>
<div id="image-and-data-analysis" class="section level4">
<h4><span class="header-section-number">2.2.1.4</span> Image and Data Analysis</h4>
<p>In order to be as objective an as high throughput as possible, almost all of the analyses performed for this study was solved using either algorithmically or state-of-the-art convolutional neural networks (CNNs). Furthermore, to meet the terms and conditions of <em>open science</em><a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a> standards, all pipelines were implemented in open source software frameworks such as <em>Fiji is just image J</em> (FIJI) and R. For further information about training datasets, algorithms and versions used see Materials and Methods section <a href="2-mat-met.html#mat-sftwr">2.1.11</a>.</p>
</div>
</div>
<div id="Zeb-met" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Zebrafish</h3>
<div id="husbandry" class="section level4">
<h4><span class="header-section-number">2.2.2.1</span> Husbandry</h4>
<p>Zebraﬁsh husbandry was maintained at the University of Frankfurt am Main. All legal procedures were followed while handling and maintaining zebraﬁsh husbandry. All zebraﬁsh lines used in and generated for this study are listed in the Appendix A.</p>
</div>
<div id="handling-and-rearing" class="section level4">
<h4><span class="header-section-number">2.2.2.2</span> Handling and rearing</h4>
<p>In the afternoon preceding the embryo collection, 1 male and female were set up in crossing cages, physically separated by a transparent separator. Next day, before noon, separators were removed allowing fertilization. Fertilized eggs were then collected, sorted and reared in the well-defined culture medium E3 (Kimmel et.al. 1995, section <a href="2-mat-met.html#mat-buff">2.1.6</a>) at 25<span class="math inline">\(^\circ\)</span>, 28.5<span class="math inline">\(^\circ\)</span>, or 30<span class="math inline">\(^\circ\)</span> Celsius depending on the experimental condition required.</p>
<p>To grow larvae to the adulthood, they were transferred to the system on day 5. Till day 12, larvae were fed Vinegar Eels, Paramecia, and Caviar powder. After the 12 th day, water supply was started and fish were fed Brine Shrimp, Artemia, Paramecia and Vinegar Eels. Adult fish (&gt;1 Month) were fed Artemia and the dry flakes.</p>
</div>
<div id="zebrafish-fin-clips" class="section level4">
<h4><span class="header-section-number">2.2.2.3</span> Zebrafish fin clips</h4>
<p>Adult ﬁsh were anesthetized with buﬀered Tricaine (1X, see section <a href="2-mat-met.html#mat-buff">2.1.6</a>) until. About 1/3 rd of the caudal ﬁn was cut with a sterile scalpel in a sterile Petri Dish. Immediately the cut ﬁn was transferred to 50 mM NaOH. Fish were returned to system water and kept in 1L system water in single tanks with 200 <span class="math inline">\(\mu\)</span>L of 0.01<span class="math inline">\(\%\)</span> Methylene Blue.</p>
</div>
<div id="adult-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.2.4</span> Adult Genotyping</h4>
<p>The clipped fins were digested for 1 h at 95<span class="math inline">\(^\circ\)</span> Celsius and neutralized subsequently with 1M Tris-HCl of (pH 9).</p>
</div>
<div id="embryo-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.2.5</span> Embryo Genotyping</h4>
<p>Single fixed/live embryos were denatured at 95<span class="math inline">\(^\circ\)</span> Celsius in 50mM NaOH for 1 hour and neutralized by adding 1/10 the volume of 1 M Tris-HCl (pH 8.2).</p>
</div>
<div id="zebrafish-euthanasia" class="section level4">
<h4><span class="header-section-number">2.2.2.6</span> Zebrafish Euthanasia</h4>
<p>Adult zebrafish were anesthetized in tricaine (1X) till stage II-III of anaestezia and put in the ice cold water so as to sacrifice them by hypothermia.</p>
</div>
<div id="fixation" class="section level4">
<h4><span class="header-section-number">2.2.2.7</span> Fixation</h4>
<p>Embryos and dechorionated larvae were fixed in 2 ml of 4<span class="math inline">\(\%\)</span> PFA in 1X PBS overnight at 4<span class="math inline">\(^\circ\)</span> Celsius.</p>
</div>
</div>
<div id="Wet-met" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Wet lab</h3>
<div id="sampleprep" class="section level4">
<h4><span class="header-section-number">2.2.3.1</span> Sample preparation</h4>
<p>For samples <strong>later than 24 hpf</strong>, embryos were grown in 1X PTU till desired stage and treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase for ~30 min.. Choria were removed by gentle pipetting with a 2 mL plastic pasteur pipette. To replace the Pronase solution with fresh E3, embryos were immobilized by anesthesia and collected in the center of the dish by gentle rotational movement. Then the medium was poured away by collecting the embryos at the corner bottom of the dish while taking care not to loose any. After, the dish was filled with fresh E3. This process was repeated three times.</p>
<p>For samples <strong>earlier than 18s</strong>, embryos were treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase directly. Choria were removed the same way as for &gt; 24 hpf embryos but when pouring away the Pronase solution, the dish was simultaneously and very carefully filled with fresh E3 again. Since younger embryos are more fragile and to avoid damage, they must be kept in solution constantly.</p>
<p>Fixation started at the desired stage in 4% PFA in 0.1% PBST at 4<span class="math inline">\(^\circ\)</span>C o.n.. The next day, samples were rinsed 3 times for ~5 min. in PBST and passed through a MeOH series of 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 100<span class="math inline">\(\%\)</span> MeOH/PBST (V/V)). For permanent storage, samples were stored in 100<span class="math inline">\(\%\)</span> MeOH at -20<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="miniprep" class="section level4">
<h4><span class="header-section-number">2.2.3.2</span> Plasmid preparation</h4>
<p>To remove solvent and medium, in a 1.5 or 2 mL tube, the bacteria were centrifuged in 2 mL tubes for 3 min. at 12000 rpm. To solve the pellet, the supernatant was discarded by pouring on a tissue and 200 <span class="math inline">\(\mu\)</span>L of buffer P1 were added followed by thorough vortexing. To lyse the bacteria 400 <span class="math inline">\(\mu\)</span>L of P2 were added, standing still. After a maximum of 5 min. 300 <span class="math inline">\(\mu\)</span>L of P3 were added to neutralize the reaction and bind cellular material, and the tube was inverted five to six times.</p>
<p>To harvest the plasmid DNA the solution was then first centrifuged for 10 min. at 12000 rpm and at RT. After, the supernatant was transferred to a new 2 mL tube. Second, to precipitate the DNA, 700 <span class="math inline">\(\mu\)</span>L of iso-propanol were added and centrifuged for 20 min. at full speed and at 4<span class="math inline">\(^\circ\)</span>C. After, the tube was checked for a opaque pellet and the supernatant was discarded. Third, to wash the pellet, 200 <span class="math inline">\(\mu\)</span>L of 70<span class="math inline">\(%\)</span> EtOH were added and centrifuged for 7 min. at full speed and at 4<span class="math inline">\(^\circ\)</span>C. The supernatant was discarded again. Finally, the pellet was air dryed under the hood and re-suspended in 50 <span class="math inline">\(\mu\)</span>L ddHO.</p>
</div>
<div id="ISH-met" class="section level4">
<h4><span class="header-section-number">2.2.3.3</span> In Situ Hybridization</h4>
<p>Sample preparation as described in section <a href="2-mat-met.html#sampleprep">2.2.3.1</a>.</p>
<div id="permeabilisation-probe-hybridization" class="section level5">
<h5><span class="header-section-number">2.2.3.3.1</span> 1. Permeabilisation &amp; Probe Hybridization</h5>
<p><strong>Permeabilisation</strong> <span class="math inline">\(\rightarrow\)</span> without shaking</p>
<p>Samples were rehydrated in an inverse MeOH series of 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 0<span class="math inline">\(\%\)</span> PBST and washed again for 5 min. two times in pure PBST. Finally, samples were digested in 10 <span class="math inline">\(\mu\)</span>g/mL Proteinase K according to table <a href="2-mat-met.html#tab:met-protk">2.15</a>.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:met-protk">Table 2.15: </span>Proteinase K digestion
</caption>
<thead>
<tr>
<th style="text-align:center;">
Stage
</th>
<th style="text-align:center;">
0.6 s
</th>
<th style="text-align:center;">
7 s
</th>
<th style="text-align:center;">
18 s
</th>
<th style="text-align:center;">
24 hpf
</th>
<th style="text-align:center;">
32 hpf
</th>
<th style="text-align:center;">
36 hpf
</th>
<th style="text-align:center;">
42 hpf
</th>
<th style="text-align:center;">
48 hpf
</th>
<th style="text-align:center;">
72 hpf
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
min.
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
30
</td>
<td style="text-align:center;">
40
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:center;">
60
</td>
<td style="text-align:center;">
60
</td>
</tr>
</tbody>
</table>
<p>Samples were washed again for 5 min. two times in PBST and post-fixated in 4<span class="math inline">\(\%\)</span> PFA at 4<span class="math inline">\(^\circ\)</span>C for &gt; 30 min. Samples were washed again for 5 min. three times in PBST.</p>
<p><strong>Probe Hybridisation</strong> <span class="math inline">\(\rightarrow\)</span> all steps at 60<span class="math inline">\(^\circ\)</span>C, except stated differently</p>
<p>Samples were pre-hybridized in 350 <span class="math inline">\(\mu\)</span>L of hybridization buffer (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h. Just before detection probe treatment, the probe was denatured at 80<span class="math inline">\(^\circ\)</span>C in 1:200 of hybridization buffer. Subsequently, hybridization buffer was taken off the samples and replaced with the heated probe. Finally, samples were incubated o.n. at a desired temperature (around 65<span class="math inline">\(^\circ\)</span>C).</p>
</div>
<div id="probe-removal-antibody-incubation" class="section level5">
<h5><span class="header-section-number">2.2.3.3.2</span> 2. Probe removal &amp; Antibody incubation</h5>
<p>The next day the probe got collected and stored at -20<span class="math inline">\(^\circ\)</span>C for re-use. Washing took place at the same temperature as hybridization (from step 1) To keep solutions at temperature a Thermo-Block was used. For the washing series the samples were first washed one time for 20 min. in hybridization buffer, then two times for 30 min. in 50<span class="math inline">\(\%\)</span> Formamide and one time for 20 min. in 25<span class="math inline">\(\%\)</span> Formamide. Then two times for 15 min. in 2X SSCT and two times for 30 min. in 0.2X SSCT. Finally, one time for 5 min. in TNT.</p>
<p>To reduce noise and increase specific signal strength, the samples were treated with blocking solution (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h in 350 <span class="math inline">\(\mu\)</span>L of 2% BR/TNT at room temperature (RT). Afterwards the samples were incubated in 100 <span class="math inline">\(\mu\)</span>L Anti-Digoxigenin diluted in NTMT buffer (1/4000 (V/V)) in 2<span class="math inline">\(\%\)</span> BR/TNT for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="probe-detection" class="section level5">
<h5><span class="header-section-number">2.2.3.3.3</span> 3. Probe detection</h5>
<p>First, the samples were washed six times for ~20 min. (or one wash o.n.) in TNT and two times for ~ 5min. in NTMT. After washing, color staining was performed with 4.5 NBT <span class="math inline">\(\mu\)</span>L + 3.5 <span class="math inline">\(\mu\)</span>L BCIP per mL NTMT in the dark and at RT without shaking (in a drawer) for 2 - 8 h, regularly checking the progression of the reaction. As soon as an appropriate degree of color intensity on the target site was achieved (up to two days), the samples were again washed three times in PBST.</p>
<p>Afterwards the samples were either prepared for immunostaining or imaging. For permanent storage samples were kept in 50<span class="math inline">\(\%\)</span> Glycerol at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
</div>
<div id="immuno-met" class="section level4">
<h4><span class="header-section-number">2.2.3.4</span> Immuno staining</h4>
<p>Sample preparation as described in section <a href="2-mat-met.html#sampleprep">2.2.3.1</a>.</p>
<p>First, samples were blocked in 2<span class="math inline">\(\%\)</span> Goat Serum / PBDT (V/V) for 30 min.. For protein target site detection, a <strong>primary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for ~2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times. To stain the now bound primary antibody, a <strong>secondary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times.</p>
</div>
<div id="mount-met" class="section level4">
<h4><span class="header-section-number">2.2.3.5</span> Mounting</h4>
<p>For mounting, an improved 3D specimen preparation and well-plate like sample navigation for zebrafish larvae confocal microscopy was developed which offers the following improvements…</p>
<ol style="list-style-type: decimal">
<li><p><strong>To obtain a sufficient amount of data that is closely staged and still comprises &gt; 4 independent groups.</strong> Using this protocol, it is possible to define a custom well plate in your preferred imaging software, which greatly facilitates the imaging procedure and therefore helps to stay focused. However, once the well plate is aligned and the positions are defined it still needs some minor correction in Z.</p></li>
<li><p><strong>To do the imaging live. Most of the time, especially in morphological analyses, live imaging is preferred since any fixation alters cell - and therefore organ morphology.</strong> Live imaging guarantees an undisturbed and unbiased look at the <em>in vivo</em> situation. Using this protocol, the embryonic body axes will be more in line with the XY displacement of the microscope stage which results, depending on the volume of interest, in narrower Z-stacks. This ultimately also leads to a reduction of photo-bleaching, phototoxicity and expenditure of time. The gained time can then be used to image shorter time-intervals in timelapse imaging.</p></li>
<li><p><strong>To reduce background signal and noise.</strong> Instead of transforming and aligning the image data in a post-processing approach, the specimen can be imaged in the right spacial configuration in the first place. This is not only important for 3D analysis, but also for Z projected data. If the specimen body axes are not aligned with the axes of the microscopic table, the image stacks need to be bigger to record the volume of interest. In Z projections, for X Y coordinates out of the volume of interest, this will end up as background signal or noise, complicating quantitative analysis and interpretation. The method therefore provides significant improvement over existing mounting techniques.</p></li>
</ol>
<div id="protocol" class="section level5">
<h5><span class="header-section-number">2.2.3.5.1</span> Protocol</h5>

<div class="figure" style="text-align: center"><span id="fig:stampprod-1"></span>
<img src="files/pubs/klein_2019/006_03.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.1: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-2"></span>
<img src="files/pubs/klein_2019/007.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.2: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-3"></span>
<img src="files/pubs/klein_2019/008.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.3: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-4"></span>
<img src="files/pubs/klein_2019/009.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.4: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-5"></span>
<img src="files/pubs/klein_2019/010.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.5: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
</div>
</div>
</div>
<div id="comp-met" class="section level3">
<h3><span class="header-section-number">2.2.4</span> Dry lab</h3>
<div id="image-j-macros" class="section level4">
<h4><span class="header-section-number">2.2.4.1</span> Image J macros</h4>
<p>Three IJ macros have been developed to facilitate image analysis and make results more reproducible. Each of them is specifically designed for input of LL and pLLP images of the <em>cldnb:lyn-gfp</em> transgenic line.</p>
<p>Hence, they are called <strong>anaLLzr</strong> …</p>
<ul>
<li><strong>2D</strong> - analysis of Z-projected images of the LL at end of migration</li>
<li><strong>2DT</strong> - analysis of Z-projected images of the LL during migration</li>
<li><strong>3D</strong> - analysis of 3D image stacks of the pLLP at a given timepoint</li>
</ul>
<div id="mat-anallzr2d" class="section level5">
<h5><span class="header-section-number">2.2.4.1.1</span> anaLLzr2D</h5>
<p>From the opening dialog (figure <a href="2-mat-met.html#fig:anallzr2ddialog">2.6</a>) the user can choose to <em>count nuclei</em> and / or <em>sort ROIs</em>.</p>
<p>If nuclei count is chosen, the macro expects a dual-channel <em>tiff</em>-file as input.
If ROI sorting is selected, segmented CCs are numbered and sorted from left to right instead of IJ’s native sorting method (from top to bottom).</p>
<p>Slider values are optimized for the the microscope setup described in section <a href="2-mat-met.html#mat-SD">2.1.10.2</a>.</p>
<ul>
<li><em>Membrane label blur</em> controls the detail of pLLP and CC segmentation, where lower values result in more detail</li>
<li><em>Closing filter</em> controls how harsh objects are separated from each other</li>
<li><em>Nuclei label blur</em> controls the details of single nuclei and is evaluated after ground truth data described in section <a href="2-mat-met.html#mat-GrTrDat">2.3</a></li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:anallzr2ddialog"></span>
<img src="figures/materials/macros/anallzr2D_macro.png" alt="anaLLzR2D opening dialog checkboxes Choose to count nuclei and whether ROIs sorting should be applied. sliders Choose filter and blurring levels." width="40%" />
<p class="caption">
Figure 2.6: anaLLzR2D opening dialog <strong>checkboxes</strong> Choose to count nuclei and whether ROIs sorting should be applied. <strong>sliders</strong> Choose filter and blurring levels.
</p>
</div>
<p>Code-snippets describing the main functionality are described in the next couple of sub-sections.</p>
<div id="segmentation" class="section level6">
<h6><span class="header-section-number">2.2.4.1.1.1</span> Segmentation</h6>
<p>The membrane labeled image is segmented based on optimized hard coded values which are based on experience. After, the macro halts for manual correction.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="2-mat-met.html#cb1-1"></a><span class="co"># Background subtraction</span></span>
<span id="cb1-2"><a href="2-mat-met.html#cb1-2"></a>    <span class="kw">run</span>(<span class="st">&quot;Subtract Background...&quot;</span>, <span class="st">&quot;rolling = 50&quot;</span>);</span>
<span id="cb1-3"><a href="2-mat-met.html#cb1-3"></a>    <span class="kw">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>, <span class="st">&quot;operation = Opening element = Disk radius = 20&quot;</span>);</span>
<span id="cb1-4"><a href="2-mat-met.html#cb1-4"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 5 scaled&quot;</span>);</span>
<span id="cb1-5"><a href="2-mat-met.html#cb1-5"></a><span class="co"># Thresholding</span></span>
<span id="cb1-6"><a href="2-mat-met.html#cb1-6"></a>    <span class="kw">setAutoThreshold</span>(<span class="st">&quot;Moments dark&quot;</span>);</span>
<span id="cb1-7"><a href="2-mat-met.html#cb1-7"></a>    <span class="kw">setOption</span>(<span class="st">&quot;BlackBackground&quot;</span>, false);</span>
<span id="cb1-8"><a href="2-mat-met.html#cb1-8"></a>    <span class="kw">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb1-9"><a href="2-mat-met.html#cb1-9"></a><span class="co"># Particle analysis</span></span>
<span id="cb1-10"><a href="2-mat-met.html#cb1-10"></a>    <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size = 250-Infinity exclude add&quot;</span>);</span>
<span id="cb1-11"><a href="2-mat-met.html#cb1-11"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;Show All without labels&quot;</span>);</span>
<span id="cb1-12"><a href="2-mat-met.html#cb1-12"></a>    <span class="kw">run</span>(<span class="st">&quot;Enhance Contrast&quot;</span>, <span class="st">&quot;saturated = 0.35&quot;</span>);</span>
<span id="cb1-13"><a href="2-mat-met.html#cb1-13"></a>    <span class="kw">waitForUser</span>(<span class="st">&quot;Check ROIs, correct if necessary&quot;</span>);</span>
<span id="cb1-14"><a href="2-mat-met.html#cb1-14"></a>    <span class="cf">if</span> (sort) {</span>
<span id="cb1-15"><a href="2-mat-met.html#cb1-15"></a>        <span class="kw">sortROIs</span>();</span>
<span id="cb1-16"><a href="2-mat-met.html#cb1-16"></a>    }</span></code></pre></div>
</div>
<div id="sorting" class="section level6">
<h6><span class="header-section-number">2.2.4.1.1.2</span> Sorting</h6>
<p>To sort manually corrected ROIs from left to right, each ROIs position in calculated relatively to total image width.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="2-mat-met.html#cb2-1"></a><span class="co"># Sort ROIs from left to right</span></span>
<span id="cb2-2"><a href="2-mat-met.html#cb2-2"></a><span class="cf">function</span> <span class="kw">sortROIs</span>() {</span>
<span id="cb2-3"><a href="2-mat-met.html#cb2-3"></a>    <span class="kw">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, <span class="st">&quot;centroid redirect = None decimal = 0&quot;</span>);</span>
<span id="cb2-4"><a href="2-mat-met.html#cb2-4"></a>            <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">0</span> ; j<span class="op">&lt;</span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb2-5"><a href="2-mat-met.html#cb2-5"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-6"><a href="2-mat-met.html#cb2-6"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;measure&quot;</span>);</span>
<span id="cb2-7"><a href="2-mat-met.html#cb2-7"></a>                x =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;X&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb2-8"><a href="2-mat-met.html#cb2-8"></a>                w =<span class="st"> </span><span class="kw">getWidth</span>();</span>
<span id="cb2-9"><a href="2-mat-met.html#cb2-9"></a>                a =<span class="st"> </span>x<span class="op">/</span>w;</span>
<span id="cb2-10"><a href="2-mat-met.html#cb2-10"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, a);</span>
<span id="cb2-11"><a href="2-mat-met.html#cb2-11"></a>                <span class="kw">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-12"><a href="2-mat-met.html#cb2-12"></a>                }</span>
<span id="cb2-13"><a href="2-mat-met.html#cb2-13"></a>        <span class="kw">setBatchMode</span>(false);</span>
<span id="cb2-14"><a href="2-mat-met.html#cb2-14"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;sort&quot;</span>); </span>
<span id="cb2-15"><a href="2-mat-met.html#cb2-15"></a>            <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">0</span> ; j <span class="op">&lt;</span><span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb2-16"><a href="2-mat-met.html#cb2-16"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-17"><a href="2-mat-met.html#cb2-17"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, j);</span>
<span id="cb2-18"><a href="2-mat-met.html#cb2-18"></a>                <span class="kw">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-19"><a href="2-mat-met.html#cb2-19"></a>                }   </span>
<span id="cb2-20"><a href="2-mat-met.html#cb2-20"></a>            <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">0</span> ; j <span class="op">&lt;</span><span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb2-21"><a href="2-mat-met.html#cb2-21"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-22"><a href="2-mat-met.html#cb2-22"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, j<span class="op">+</span><span class="dv">1</span>);</span>
<span id="cb2-23"><a href="2-mat-met.html#cb2-23"></a>                <span class="kw">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-24"><a href="2-mat-met.html#cb2-24"></a>                }</span>
<span id="cb2-25"><a href="2-mat-met.html#cb2-25"></a>}</span></code></pre></div>
</div>
<div id="count-nuclei" class="section level6">
<h6><span class="header-section-number">2.2.4.1.1.3</span> Count nuclei</h6>
<p>After smoothing the nuclei signal in the DAPI labeled channel, maxima are detected only within each ROI.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="2-mat-met.html#cb3-1"></a><span class="co"># count nuclei within segmented cell clusters</span></span>
<span id="cb3-2"><a href="2-mat-met.html#cb3-2"></a><span class="co"># steps performed on DAPI channel</span></span>
<span id="cb3-3"><a href="2-mat-met.html#cb3-3"></a><span class="co"># count nuclei</span></span>
<span id="cb3-4"><a href="2-mat-met.html#cb3-4"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 0.6 scaled&quot;</span>);</span>
<span id="cb3-5"><a href="2-mat-met.html#cb3-5"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;open&quot;</span>, datdir <span class="op">+</span><span class="st"> </span>filename <span class="op">+</span><span class="st"> &quot;_ROIset.zip&quot;</span>);</span>
<span id="cb3-6"><a href="2-mat-met.html#cb3-6"></a>    rcount =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb3-7"><a href="2-mat-met.html#cb3-7"></a><span class="co"># for each ROI</span></span>
<span id="cb3-8"><a href="2-mat-met.html#cb3-8"></a>    <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">0</span> ; j <span class="op">&lt;</span><span class="st"> </span>rcount; j<span class="op">++</span>) {</span>
<span id="cb3-9"><a href="2-mat-met.html#cb3-9"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;open&quot;</span>, datdir <span class="op">+</span><span class="st"> </span>filename <span class="op">+</span><span class="st"> &quot;_ROIset.zip&quot;</span>);</span>
<span id="cb3-10"><a href="2-mat-met.html#cb3-10"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb3-11"><a href="2-mat-met.html#cb3-11"></a>            <span class="kw">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>);</span>
<span id="cb3-12"><a href="2-mat-met.html#cb3-12"></a>            <span class="kw">run</span>(<span class="st">&quot;Enhance Contrast&quot;</span>, <span class="st">&quot;saturated = 0.35&quot;</span>);</span>
<span id="cb3-13"><a href="2-mat-met.html#cb3-13"></a>            <span class="kw">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = 0 output = [Point Selection]&quot;</span>);</span>
<span id="cb3-14"><a href="2-mat-met.html#cb3-14"></a>            <span class="kw">run</span>(<span class="st">&quot;Capture Image&quot;</span>);</span>
<span id="cb3-15"><a href="2-mat-met.html#cb3-15"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb3-16"><a href="2-mat-met.html#cb3-16"></a>        <span class="kw">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = 0 output = Count&quot;</span>);</span>
<span id="cb3-17"><a href="2-mat-met.html#cb3-17"></a>        NC =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Count&quot;</span>);</span>
<span id="cb3-18"><a href="2-mat-met.html#cb3-18"></a>        <span class="kw">setResult</span>(<span class="st">&quot;Nuclei&quot;</span>, j, NC);</span>
<span id="cb3-19"><a href="2-mat-met.html#cb3-19"></a>        <span class="kw">setResult</span>(<span class="st">&quot;Pos&quot;</span>, j, j <span class="op">+</span><span class="st"> </span><span class="dv">1</span>);</span>
<span id="cb3-20"><a href="2-mat-met.html#cb3-20"></a>    }</span></code></pre></div>
</div>
</div>
<div id="mat-anallzr2dt" class="section level5">
<h5><span class="header-section-number">2.2.4.1.2</span> anaLLzR2DT</h5>
<p>Upon macro execution the opening dialog is presented which is divided in three sections (figure <a href="2-mat-met.html#fig:anallzr2dtdialog">2.7</a>A).</p>
<p>In the <strong>processing</strong> section the user may choose which modules are executed.</p>
<ul>
<li><em>Segmentation</em> controls whether the images are segmented before measurement. If de-seleted the user has to provide segmentation masks seperately.</li>
<li><em>Include Cell Clusters</em> controls whether Cell Clusters should be included in the analysis. If de-selected, only the pLLP will be considered.</li>
<li><em>Registration</em> controls whether the pLLP should be captured in time and space and saved in a separate stack.</li>
<li><em>Multichannel</em> controls whether on a second channel summary statistics from each ROI at each timepoint should be taken.
<ul>
<li>measurements taken are the mean, standard deviation, minimum and maximum intensity</li>
</ul></li>
</ul>
<p>In the <strong>options</strong> section the user may choose whether the macro should be run in <em>headless mode</em> (without showing every single action), whether the input images are timeseries and whether all other windows should be closed upon start of processing.</p>
<p>After confirmation, the user has to enter a date of experiment as an identifier in a second dialog. Furthermore the user is presented the images physical properties pre-filled where the idea here is just a review since this is a major source of mistakes. Finally, a third dialog is presented to the user giving an approximate duration and basic instructions.</p>

<div class="figure" style="text-align: center"><span id="fig:anallzr2dtdialog"></span>
<img src="figures/materials/macros/anallzr2DT_macro.png" alt="anaLLzR2DT opening dialog A Opening dialog: Main functionality B Opening Dialog: metadata setup options C Time approximation dialog and basic instructions." width="95%" />
<p class="caption">
Figure 2.7: anaLLzR2DT opening dialog <strong>A</strong> Opening dialog: Main functionality <strong>B</strong> Opening Dialog: metadata setup options <strong>C</strong> Time approximation dialog and basic instructions.
</p>
</div>
<p>Code-snippets describing the main functionality are described in the next couple of sub-sections.</p>
<div id="registration" class="section level6">
<h6><span class="header-section-number">2.2.4.1.2.1</span> Registration</h6>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="2-mat-met.html#cb4-1"></a><span class="co">#   Subtract background</span></span>
<span id="cb4-2"><a href="2-mat-met.html#cb4-2"></a>        <span class="kw">run</span>(<span class="st">&quot;Z Project...&quot;</span>, <span class="st">&quot;projection=[Average Intensity]&quot;</span>);</span>
<span id="cb4-3"><a href="2-mat-met.html#cb4-3"></a>        ZPAVG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-4"><a href="2-mat-met.html#cb4-4"></a>        <span class="cf">if</span> (reg) {</span>
<span id="cb4-5"><a href="2-mat-met.html#cb4-5"></a>            <span class="kw">print</span>(<span class="st">&quot;Calculating registration parameters...&quot;</span>);</span>
<span id="cb4-6"><a href="2-mat-met.html#cb4-6"></a>            <span class="kw">selectWindow</span>(ORG);</span>
<span id="cb4-7"><a href="2-mat-met.html#cb4-7"></a>            <span class="kw">setSlice</span>(n);</span>
<span id="cb4-8"><a href="2-mat-met.html#cb4-8"></a>            <span class="kw">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>); </span>
<span id="cb4-9"><a href="2-mat-met.html#cb4-9"></a>            DORG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-10"><a href="2-mat-met.html#cb4-10"></a>            <span class="kw">imageCalculator</span>(<span class="st">&quot;Subtract create&quot;</span>, DORG, ZPAVG);</span>
<span id="cb4-11"><a href="2-mat-met.html#cb4-11"></a>            IC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-12"><a href="2-mat-met.html#cb4-12"></a>            <span class="kw">close</span>(DORG);</span>
<span id="cb4-13"><a href="2-mat-met.html#cb4-13"></a>            <span class="kw">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>, <span class="st">&quot;operation=Closing element=Disk radius=15&quot;</span>);</span>
<span id="cb4-14"><a href="2-mat-met.html#cb4-14"></a>            REG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-15"><a href="2-mat-met.html#cb4-15"></a>            <span class="kw">close</span>(IC);</span>
<span id="cb4-16"><a href="2-mat-met.html#cb4-16"></a>            <span class="kw">close</span>(IC);</span>
<span id="cb4-17"><a href="2-mat-met.html#cb4-17"></a>            <span class="kw">close</span>(DORG);</span>
<span id="cb4-18"><a href="2-mat-met.html#cb4-18"></a>        <span class="co">#   Thresholding </span></span>
<span id="cb4-19"><a href="2-mat-met.html#cb4-19"></a>            <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma=6 scaled&quot;</span>); </span>
<span id="cb4-20"><a href="2-mat-met.html#cb4-20"></a>            REG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-21"><a href="2-mat-met.html#cb4-21"></a>            <span class="kw">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>);</span>
<span id="cb4-22"><a href="2-mat-met.html#cb4-22"></a>            REGD =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-23"><a href="2-mat-met.html#cb4-23"></a>            <span class="kw">selectWindow</span>(REG);</span>
<span id="cb4-24"><a href="2-mat-met.html#cb4-24"></a>            <span class="kw">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>, <span class="st">&quot;saturated=0.3 normalize&quot;</span>);</span>
<span id="cb4-25"><a href="2-mat-met.html#cb4-25"></a>            <span class="kw">run</span>(<span class="st">&quot;8-bit&quot;</span>);</span>
<span id="cb4-26"><a href="2-mat-met.html#cb4-26"></a>            <span class="kw">selectWindow</span>(REG);</span>
<span id="cb4-27"><a href="2-mat-met.html#cb4-27"></a>            <span class="kw">setAutoThreshold</span>(<span class="st">&quot;MaxEntropy  dark&quot;</span>);</span>
<span id="cb4-28"><a href="2-mat-met.html#cb4-28"></a>            <span class="kw">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb4-29"><a href="2-mat-met.html#cb4-29"></a>            REG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-30"><a href="2-mat-met.html#cb4-30"></a></span>
<span id="cb4-31"><a href="2-mat-met.html#cb4-31"></a>    <span class="co"># ############################################## ANALYSIS ###########################################</span></span>
<span id="cb4-32"><a href="2-mat-met.html#cb4-32"></a>    </span>
<span id="cb4-33"><a href="2-mat-met.html#cb4-33"></a>        <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size=150-10000 include exclude add&quot;</span>);</span>
<span id="cb4-34"><a href="2-mat-met.html#cb4-34"></a>        rmcount =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="op">-</span><span class="dv">1</span>;</span>
<span id="cb4-35"><a href="2-mat-met.html#cb4-35"></a>        <span class="kw">print</span>(<span class="st">&quot;rois: &quot;</span><span class="op">+</span>rmcount);</span>
<span id="cb4-36"><a href="2-mat-met.html#cb4-36"></a>        </span>
<span id="cb4-37"><a href="2-mat-met.html#cb4-37"></a>    <span class="co"># ############################################## ANGLE ###########################################</span></span>
<span id="cb4-38"><a href="2-mat-met.html#cb4-38"></a>    </span>
<span id="cb4-39"><a href="2-mat-met.html#cb4-39"></a>        <span class="cf">if</span>(<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb4-40"><a href="2-mat-met.html#cb4-40"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb4-41"><a href="2-mat-met.html#cb4-41"></a>            List.setMeasurements;</span>
<span id="cb4-42"><a href="2-mat-met.html#cb4-42"></a>            Angle =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;FeretAngle&quot;</span>);</span>
<span id="cb4-43"><a href="2-mat-met.html#cb4-43"></a>            <span class="kw">print</span>(<span class="st">&quot;Angle: &quot;</span><span class="op">+</span>Angle);</span>
<span id="cb4-44"><a href="2-mat-met.html#cb4-44"></a>            <span class="cf">if</span> (Angle <span class="op">&lt;</span><span class="st"> </span><span class="dv">0</span>) {Angle =<span class="st"> </span>Angle<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb4-45"><a href="2-mat-met.html#cb4-45"></a>            <span class="cf">if</span> (Angle <span class="op">&gt;</span><span class="st"> </span><span class="dv">90</span>) {Angle =<span class="st"> </span>(<span class="dv">180</span><span class="op">-</span>Angle)<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb4-46"><a href="2-mat-met.html#cb4-46"></a>        } <span class="cf">else</span> {</span>
<span id="cb4-47"><a href="2-mat-met.html#cb4-47"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb4-48"><a href="2-mat-met.html#cb4-48"></a>            List.setMeasurements;</span>
<span id="cb4-49"><a href="2-mat-met.html#cb4-49"></a>            X1Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-50"><a href="2-mat-met.html#cb4-50"></a>            Y1Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-51"><a href="2-mat-met.html#cb4-51"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb4-52"><a href="2-mat-met.html#cb4-52"></a>            List.setMeasurements;</span>
<span id="cb4-53"><a href="2-mat-met.html#cb4-53"></a>            X2Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-54"><a href="2-mat-met.html#cb4-54"></a>            Y2Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-55"><a href="2-mat-met.html#cb4-55"></a>            <span class="kw">makeLine</span>(X1Line, Y1Line, X2Line, Y2Line);</span>
<span id="cb4-56"><a href="2-mat-met.html#cb4-56"></a>            List.setMeasurements;</span>
<span id="cb4-57"><a href="2-mat-met.html#cb4-57"></a>            Angle =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb4-58"><a href="2-mat-met.html#cb4-58"></a>            <span class="cf">if</span> (Angle <span class="op">&lt;</span><span class="st"> </span><span class="dv">0</span>) {Angle =<span class="st"> </span>Angle<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb4-59"><a href="2-mat-met.html#cb4-59"></a>            <span class="cf">if</span> (Angle <span class="op">&gt;</span><span class="st"> </span><span class="dv">90</span>) {Angle =<span class="st"> </span>(<span class="dv">180</span><span class="op">-</span>Angle)<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb4-60"><a href="2-mat-met.html#cb4-60"></a>            }</span>
<span id="cb4-61"><a href="2-mat-met.html#cb4-61"></a>            <span class="kw">print</span>(<span class="st">&quot;Angle: &quot;</span><span class="op">+</span>Angle);</span>
<span id="cb4-62"><a href="2-mat-met.html#cb4-62"></a>            <span class="kw">selectWindow</span>(ZPAVG);</span>
<span id="cb4-63"><a href="2-mat-met.html#cb4-63"></a>            <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=&quot;</span><span class="op">+</span>Angle<span class="op">+</span><span class="st">&quot; grid=1 interpolation=Bilinear&quot;</span>);</span>
<span id="cb4-64"><a href="2-mat-met.html#cb4-64"></a>            ZPAVG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-65"><a href="2-mat-met.html#cb4-65"></a>            <span class="kw">selectWindow</span>(REG);</span>
<span id="cb4-66"><a href="2-mat-met.html#cb4-66"></a>            <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=&quot;</span><span class="op">+</span>Angle<span class="op">+</span><span class="st">&quot; grid=1 interpolation=Bilinear&quot;</span>);</span>
<span id="cb4-67"><a href="2-mat-met.html#cb4-67"></a>            <span class="kw">run</span>(<span class="st">&quot;Make Binary&quot;</span>);</span>
<span id="cb4-68"><a href="2-mat-met.html#cb4-68"></a>            REG =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-69"><a href="2-mat-met.html#cb4-69"></a>        </span>
<span id="cb4-70"><a href="2-mat-met.html#cb4-70"></a>        <span class="co"># ############################################ CROPPING ######################################### </span></span>
<span id="cb4-71"><a href="2-mat-met.html#cb4-71"></a></span>
<span id="cb4-72"><a href="2-mat-met.html#cb4-72"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb4-73"><a href="2-mat-met.html#cb4-73"></a>            <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size=150-10000 include add&quot;</span>);</span>
<span id="cb4-74"><a href="2-mat-met.html#cb4-74"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>); <span class="co"># oder doch rmcount?</span></span>
<span id="cb4-75"><a href="2-mat-met.html#cb4-75"></a>            List.setMeasurements;</span>
<span id="cb4-76"><a href="2-mat-met.html#cb4-76"></a>            XRect =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-77"><a href="2-mat-met.html#cb4-77"></a>            YRect =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-78"><a href="2-mat-met.html#cb4-78"></a>            <span class="kw">selectWindow</span>(REG);</span>
<span id="cb4-79"><a href="2-mat-met.html#cb4-79"></a>            <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb4-80"><a href="2-mat-met.html#cb4-80"></a>            List.setMeasurements;</span>
<span id="cb4-81"><a href="2-mat-met.html#cb4-81"></a>            height =<span class="st"> </span><span class="dv">120</span><span class="op">/</span>sizeX; <span class="co"># change height of rect here</span></span>
<span id="cb4-82"><a href="2-mat-met.html#cb4-82"></a>            <span class="kw">toUnscaled</span>(YRect);</span>
<span id="cb4-83"><a href="2-mat-met.html#cb4-83"></a>            YRect =<span class="st"> </span>YRect<span class="op">-</span>(height<span class="op">/</span><span class="dv">2</span>);</span>
<span id="cb4-84"><a href="2-mat-met.html#cb4-84"></a>            <span class="kw">print</span>(<span class="st">&quot;YRectcor: &quot;</span><span class="op">+</span>YRect);</span>
<span id="cb4-85"><a href="2-mat-met.html#cb4-85"></a>            <span class="kw">close</span>(REG);</span>
<span id="cb4-86"><a href="2-mat-met.html#cb4-86"></a>            <span class="kw">close</span>(REGD);</span>
<span id="cb4-87"><a href="2-mat-met.html#cb4-87"></a>        } </span>
<span id="cb4-88"><a href="2-mat-met.html#cb4-88"></a></span>
<span id="cb4-89"><a href="2-mat-met.html#cb4-89"></a>    <span class="co"># ############################################ REGISTER ###########################################</span></span>
<span id="cb4-90"><a href="2-mat-met.html#cb4-90"></a>    </span>
<span id="cb4-91"><a href="2-mat-met.html#cb4-91"></a>        <span class="kw">resetMinAndMax</span>();</span>
<span id="cb4-92"><a href="2-mat-met.html#cb4-92"></a>        <span class="cf">if</span> (dual) {</span>
<span id="cb4-93"><a href="2-mat-met.html#cb4-93"></a>        <span class="co">#   C1</span></span>
<span id="cb4-94"><a href="2-mat-met.html#cb4-94"></a>            <span class="kw">selectWindow</span>(ORG);</span>
<span id="cb4-95"><a href="2-mat-met.html#cb4-95"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-96"><a href="2-mat-met.html#cb4-96"></a>                <span class="kw">print</span>(<span class="st">&quot;  Registering &quot;</span><span class="op">+</span>embryoID<span class="op">+</span><span class="st">&quot;...&quot;</span>);</span>
<span id="cb4-97"><a href="2-mat-met.html#cb4-97"></a>                <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=&quot;</span><span class="op">+</span>Angle<span class="op">+</span><span class="st">&quot; grid=1 interpolation=Bilinear stack&quot;</span>);</span>
<span id="cb4-98"><a href="2-mat-met.html#cb4-98"></a>                <span class="kw">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-99"><a href="2-mat-met.html#cb4-99"></a>                <span class="kw">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-100"><a href="2-mat-met.html#cb4-100"></a>            }</span>
<span id="cb4-101"><a href="2-mat-met.html#cb4-101"></a>            <span class="kw">print</span>(<span class="st">&quot;    Saving: &quot;</span><span class="op">+</span>embryoID<span class="op">+</span><span class="st">&quot;_RC_C1&quot;</span>);</span>
<span id="cb4-102"><a href="2-mat-met.html#cb4-102"></a>            <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, rcdirc1 <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;_RC.tif&quot;</span>);</span>
<span id="cb4-103"><a href="2-mat-met.html#cb4-103"></a>        <span class="co">#   C2</span></span>
<span id="cb4-104"><a href="2-mat-met.html#cb4-104"></a>            <span class="kw">open</span>(dualdir<span class="op">+</span>dualdirlist[q]);</span>
<span id="cb4-105"><a href="2-mat-met.html#cb4-105"></a>            dualname =<span class="st"> </span><span class="kw">replace</span>(dualdirlist[q], <span class="st">&quot;.tif&quot;</span>, <span class="st">&quot;&quot;</span>); </span>
<span id="cb4-106"><a href="2-mat-met.html#cb4-106"></a>            ORGC2 =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb4-107"><a href="2-mat-met.html#cb4-107"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-108"><a href="2-mat-met.html#cb4-108"></a>                <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=&quot;</span><span class="op">+</span>Angle<span class="op">+</span><span class="st">&quot; grid=1 interpolation=Bilinear stack&quot;</span>);</span>
<span id="cb4-109"><a href="2-mat-met.html#cb4-109"></a>                <span class="kw">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-110"><a href="2-mat-met.html#cb4-110"></a>                <span class="kw">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-111"><a href="2-mat-met.html#cb4-111"></a>            }</span>
<span id="cb4-112"><a href="2-mat-met.html#cb4-112"></a>            <span class="kw">print</span>(<span class="st">&quot;    Saving: &quot;</span><span class="op">+</span>embryoID<span class="op">+</span><span class="st">&quot;_RC_C2&quot;</span>);</span>
<span id="cb4-113"><a href="2-mat-met.html#cb4-113"></a>            <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, rcdirc2 <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;_RC.tif&quot;</span>);</span>
<span id="cb4-114"><a href="2-mat-met.html#cb4-114"></a>            <span class="kw">close</span>();</span>
<span id="cb4-115"><a href="2-mat-met.html#cb4-115"></a>        } <span class="cf">else</span> {</span>
<span id="cb4-116"><a href="2-mat-met.html#cb4-116"></a>            <span class="kw">selectWindow</span>(ORG);</span>
<span id="cb4-117"><a href="2-mat-met.html#cb4-117"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-118"><a href="2-mat-met.html#cb4-118"></a>                <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=&quot;</span><span class="op">+</span>Angle<span class="op">+</span><span class="st">&quot; grid=1 interpolation=Bilinear stack&quot;</span>);</span>
<span id="cb4-119"><a href="2-mat-met.html#cb4-119"></a>                <span class="kw">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-120"><a href="2-mat-met.html#cb4-120"></a>                <span class="kw">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-121"><a href="2-mat-met.html#cb4-121"></a>                <span class="kw">print</span>(<span class="st">&quot;    Saving: &quot;</span><span class="op">+</span>orgname<span class="op">+</span><span class="st">&quot;_RC&quot;</span>);</span>
<span id="cb4-122"><a href="2-mat-met.html#cb4-122"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, rcdir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;_RC.tif&quot;</span>);</span>
<span id="cb4-123"><a href="2-mat-met.html#cb4-123"></a>            } <span class="cf">else</span> {</span>
<span id="cb4-124"><a href="2-mat-met.html#cb4-124"></a>                <span class="kw">print</span>(<span class="st">&quot;    Saving: &quot;</span><span class="op">+</span>orgname);</span>
<span id="cb4-125"><a href="2-mat-met.html#cb4-125"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, rcdir <span class="op">+</span><span class="st"> </span>orgname);    </span>
<span id="cb4-126"><a href="2-mat-met.html#cb4-126"></a>            }</span>
<span id="cb4-127"><a href="2-mat-met.html#cb4-127"></a>        }</span>
<span id="cb4-128"><a href="2-mat-met.html#cb4-128"></a>        ORG =<span class="st"> </span><span class="kw">getTitle</span>();  <span class="co"># Name ORG changed by saving</span></span>
<span id="cb4-129"><a href="2-mat-met.html#cb4-129"></a>    <span class="co"># crop ZPAVG for image calc</span></span>
<span id="cb4-130"><a href="2-mat-met.html#cb4-130"></a>        <span class="kw">selectWindow</span>(ZPAVG);</span>
<span id="cb4-131"><a href="2-mat-met.html#cb4-131"></a>        <span class="cf">if</span> (reg) {</span>
<span id="cb4-132"><a href="2-mat-met.html#cb4-132"></a>        <span class="kw">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-133"><a href="2-mat-met.html#cb4-133"></a>        <span class="kw">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-134"><a href="2-mat-met.html#cb4-134"></a>        }</span></code></pre></div>
</div>
<div id="segmentation-1" class="section level6">
<h6><span class="header-section-number">2.2.4.1.2.2</span> Segmentation</h6>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="2-mat-met.html#cb5-1"></a><span class="co">#  #################################### Background correction ####################################</span></span>
<span id="cb5-2"><a href="2-mat-met.html#cb5-2"></a>    </span>
<span id="cb5-3"><a href="2-mat-met.html#cb5-3"></a>    <span class="kw">print</span>(<span class="st">&quot;  Segmenting &quot;</span><span class="op">+</span>embryoID<span class="op">+</span><span class="st">&quot;_RC...&quot;</span>);</span>
<span id="cb5-4"><a href="2-mat-met.html#cb5-4"></a>        <span class="kw">selectWindow</span>(ORG);</span>
<span id="cb5-5"><a href="2-mat-met.html#cb5-5"></a>        <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb5-6"><a href="2-mat-met.html#cb5-6"></a>        <span class="kw">imageCalculator</span>(<span class="st">&quot;Subtract create stack&quot;</span>, ORG, ZPAVG);</span>
<span id="cb5-7"><a href="2-mat-met.html#cb5-7"></a>        IC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb5-8"><a href="2-mat-met.html#cb5-8"></a>        <span class="kw">close</span>(ZPAVG);</span>
<span id="cb5-9"><a href="2-mat-met.html#cb5-9"></a>    <span class="kw">selectWindow</span>(IC);</span>
<span id="cb5-10"><a href="2-mat-met.html#cb5-10"></a>        <span class="kw">print</span>(<span class="st">&quot;Bleach correction...&quot;</span>);</span>
<span id="cb5-11"><a href="2-mat-met.html#cb5-11"></a>        <span class="kw">run</span>(<span class="st">&quot;Bleach Correction&quot;</span>, <span class="st">&quot;correction=[Simple Ratio] background=0&quot;</span>);</span>
<span id="cb5-12"><a href="2-mat-met.html#cb5-12"></a>        BC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb5-13"><a href="2-mat-met.html#cb5-13"></a>        <span class="kw">close</span>(IC);</span>
<span id="cb5-14"><a href="2-mat-met.html#cb5-14"></a>        <span class="kw">selectWindow</span>(BC);</span>
<span id="cb5-15"><a href="2-mat-met.html#cb5-15"></a>        nslbc =<span class="st"> </span><span class="kw">nSlices</span>();</span>
<span id="cb5-16"><a href="2-mat-met.html#cb5-16"></a>        <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">1</span>; j <span class="op">&lt;</span><span class="st"> </span>nslbc; j<span class="op">++</span>) {</span>
<span id="cb5-17"><a href="2-mat-met.html#cb5-17"></a>            <span class="kw">selectWindow</span>(BC);</span>
<span id="cb5-18"><a href="2-mat-met.html#cb5-18"></a>            <span class="kw">setSlice</span>(j);</span>
<span id="cb5-19"><a href="2-mat-met.html#cb5-19"></a>            <span class="kw">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>, <span class="st">&quot;operation=Closing element=Disk radius=15&quot;</span>);</span>
<span id="cb5-20"><a href="2-mat-met.html#cb5-20"></a>        }</span>
<span id="cb5-21"><a href="2-mat-met.html#cb5-21"></a>        <span class="kw">close</span>(BC);</span>
<span id="cb5-22"><a href="2-mat-met.html#cb5-22"></a>        <span class="kw">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;name=&quot;</span><span class="op">+</span><span class="st"> </span>ORG <span class="op">+</span><span class="st">&quot; title = [] use&quot;</span>);</span>
<span id="cb5-23"><a href="2-mat-met.html#cb5-23"></a>        MC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb5-24"><a href="2-mat-met.html#cb5-24"></a>        </span>
<span id="cb5-25"><a href="2-mat-met.html#cb5-25"></a>    <span class="co">#   ############################################ SEGMENTATION #######################################</span></span>
<span id="cb5-26"><a href="2-mat-met.html#cb5-26"></a>    </span>
<span id="cb5-27"><a href="2-mat-met.html#cb5-27"></a>        <span class="kw">selectWindow</span>(MC);</span>
<span id="cb5-28"><a href="2-mat-met.html#cb5-28"></a>        <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma=5.5 scaled stack&quot;</span>);</span>
<span id="cb5-29"><a href="2-mat-met.html#cb5-29"></a>    <span class="co">#   Normalize saturated pixels</span></span>
<span id="cb5-30"><a href="2-mat-met.html#cb5-30"></a>        <span class="kw">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>, <span class="st">&quot;saturated=0.5 normalize process_all&quot;</span>);</span>
<span id="cb5-31"><a href="2-mat-met.html#cb5-31"></a>        <span class="kw">setSlice</span>(n);</span>
<span id="cb5-32"><a href="2-mat-met.html#cb5-32"></a>        <span class="kw">resetThreshold</span>();</span>
<span id="cb5-33"><a href="2-mat-met.html#cb5-33"></a>        <span class="kw">setAutoThreshold</span>(<span class="st">&quot;MaxEntropy dark&quot;</span>);</span>
<span id="cb5-34"><a href="2-mat-met.html#cb5-34"></a>        <span class="kw">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>, <span class="st">&quot;method=MaxEntropy background=Dark black&quot;</span>);</span>
<span id="cb5-35"><a href="2-mat-met.html#cb5-35"></a>        <span class="kw">run</span>(<span class="st">&quot;Invert LUT&quot;</span>);</span>
<span id="cb5-36"><a href="2-mat-met.html#cb5-36"></a>        <span class="kw">run</span>(<span class="st">&quot;Fill Holes&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb5-37"><a href="2-mat-met.html#cb5-37"></a>        <span class="kw">run</span>(<span class="st">&quot;Options...&quot;</span>, <span class="st">&quot;iterations=2 count=1 pad do=Erode stack&quot;</span>);</span>
<span id="cb5-38"><a href="2-mat-met.html#cb5-38"></a>        <span class="kw">run</span>(<span class="st">&quot;Options...&quot;</span>, <span class="st">&quot;iterations=2 count=1 pad do=Open stack&quot;</span>);</span>
<span id="cb5-39"><a href="2-mat-met.html#cb5-39"></a>        <span class="kw">run</span>(<span class="st">&quot;Options...&quot;</span>, <span class="st">&quot;iterations=1 count=1 pad do=Dilate stack&quot;</span>);</span>
<span id="cb5-40"><a href="2-mat-met.html#cb5-40"></a>        <span class="kw">print</span>(<span class="st">&quot;    Saving binary&quot;</span>);</span>
<span id="cb5-41"><a href="2-mat-met.html#cb5-41"></a>        <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, bindir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;_RC_bin.tif&quot;</span>);</span>
<span id="cb5-42"><a href="2-mat-met.html#cb5-42"></a>        MC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb5-43"><a href="2-mat-met.html#cb5-43"></a>        <span class="kw">close</span>(MC); <span class="co"># close BIN</span></span>
<span id="cb5-44"><a href="2-mat-met.html#cb5-44"></a>        T2 =<span class="st"> </span><span class="kw">getTime</span>();</span>
<span id="cb5-45"><a href="2-mat-met.html#cb5-45"></a>        Tdiff =<span class="st"> </span>T2<span class="op">-</span>T1;</span>
<span id="cb5-46"><a href="2-mat-met.html#cb5-46"></a>        <span class="kw">print</span>(<span class="st">&quot;  Duration: &quot;</span><span class="op">+</span>Tdiff<span class="op">+</span><span class="st">&quot;ms&quot;</span>);</span>
<span id="cb5-47"><a href="2-mat-met.html#cb5-47"></a>    <span class="er">}</span></span>
<span id="cb5-48"><a href="2-mat-met.html#cb5-48"></a>    month =<span class="st"> </span>month<span class="op">+</span><span class="dv">1</span>;</span>
<span id="cb5-49"><a href="2-mat-met.html#cb5-49"></a>    <span class="kw">selectWindow</span>(<span class="st">&quot;Log&quot;</span>);  <span class="op">/</span><span class="er">/</span>select Log<span class="op">-</span>window </span>
<span id="cb5-50"><a href="2-mat-met.html#cb5-50"></a>    <span class="kw">saveAs</span>(<span class="st">&quot;Text&quot;</span>, par <span class="op">+</span><span class="st"> </span>File.separator <span class="op">+</span><span class="st"> &quot;Log_v&quot;</span> <span class="op">+</span><span class="st"> </span>version <span class="op">+</span><span class="st"> &quot;_&quot;</span> <span class="op">+</span><span class="st"> </span>year<span class="op">+</span><span class="st">&quot;0&quot;</span><span class="op">+</span>month<span class="op">+</span>dayOfMonth<span class="op">+</span><span class="st">&quot;.txt&quot;</span>); </span>
<span id="cb5-51"><a href="2-mat-met.html#cb5-51"></a>    <span class="kw">closelog</span>();</span>
<span id="cb5-52"><a href="2-mat-met.html#cb5-52"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb5-53"><a href="2-mat-met.html#cb5-53"></a>    <span class="kw">cleanup</span>();</span>
<span id="cb5-54"><a href="2-mat-met.html#cb5-54"></a>    <span class="kw">setBatchMode</span>(false);</span>
<span id="cb5-55"><a href="2-mat-met.html#cb5-55"></a>    <span class="kw">wait</span>(<span class="dv">500</span>);</span>
<span id="cb5-56"><a href="2-mat-met.html#cb5-56"></a>    <span class="kw">call</span>(<span class="st">&quot;bar.Utils.revealFile&quot;</span>, bindir);</span>
<span id="cb5-57"><a href="2-mat-met.html#cb5-57"></a>    <span class="kw">waitForUser</span>(<span class="st">&quot;Check Segmentations&quot;</span>);</span>
<span id="cb5-58"><a href="2-mat-met.html#cb5-58"></a>    <span class="er">}</span> </span>
<span id="cb5-59"><a href="2-mat-met.html#cb5-59"></a>    <span class="cf">if</span> (pre) {</span>
<span id="cb5-60"><a href="2-mat-met.html#cb5-60"></a>        <span class="kw">close</span>(ORG); </span>
<span id="cb5-61"><a href="2-mat-met.html#cb5-61"></a>    }</span>
<span id="cb5-62"><a href="2-mat-met.html#cb5-62"></a>    <span class="cf">if</span> (dual) {</span>
<span id="cb5-63"><a href="2-mat-met.html#cb5-63"></a>        rcdirc1list =<span class="st"> </span><span class="kw">getFileList</span>(rcdirc1);</span>
<span id="cb5-64"><a href="2-mat-met.html#cb5-64"></a>        rcdirc2list =<span class="st"> </span><span class="kw">getFileList</span>(rcdirc2);</span>
<span id="cb5-65"><a href="2-mat-met.html#cb5-65"></a>    } <span class="cf">else</span> {</span>
<span id="cb5-66"><a href="2-mat-met.html#cb5-66"></a>        rcdirlist =<span class="st"> </span><span class="kw">getFileList</span>(rcdir);</span>
<span id="cb5-67"><a href="2-mat-met.html#cb5-67"></a>    }</span>
<span id="cb5-68"><a href="2-mat-met.html#cb5-68"></a>    bindirlist =<span class="st"> </span><span class="kw">getFileList</span>(bindir);</span></code></pre></div>
</div>
<div id="analysis" class="section level6">
<h6><span class="header-section-number">2.2.4.1.2.3</span> Analysis</h6>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="2-mat-met.html#cb6-1"></a><span class="cf">for</span> (<span class="dt">b =</span> <span class="dv">0</span>; b <span class="op">&lt;</span><span class="st"> </span>orgdirlist.length; b<span class="op">++</span>) {</span>
<span id="cb6-2"><a href="2-mat-met.html#cb6-2"></a><span class="co">#   get genotypes and embryoIDs from arrays</span></span>
<span id="cb6-3"><a href="2-mat-met.html#cb6-3"></a>        type =<span class="st"> </span>types[b];</span>
<span id="cb6-4"><a href="2-mat-met.html#cb6-4"></a>        embryoID =<span class="st"> </span>embryoIDs[b];</span>
<span id="cb6-5"><a href="2-mat-met.html#cb6-5"></a>        orgname =<span class="st"> </span><span class="kw">replace</span>(orgdirlist[b], <span class="st">&quot;.tif&quot;</span>, <span class="st">&quot;&quot;</span>);</span>
<span id="cb6-6"><a href="2-mat-met.html#cb6-6"></a><span class="co">#   create directories for single embryos to save results</span></span>
<span id="cb6-7"><a href="2-mat-met.html#cb6-7"></a>        embryodir =<span class="st"> </span>output <span class="op">+</span><span class="st"> </span>File.separator <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> </span>File.separator;</span>
<span id="cb6-8"><a href="2-mat-met.html#cb6-8"></a>        <span class="kw">File.makeDirectory</span>(embryodir);</span>
<span id="cb6-9"><a href="2-mat-met.html#cb6-9"></a><span class="co">#   print StatsLog descriptors</span></span>
<span id="cb6-10"><a href="2-mat-met.html#cb6-10"></a>        <span class="cf">if</span> (b<span class="op">==</span><span class="dv">0</span>) {</span>
<span id="cb6-11"><a href="2-mat-met.html#cb6-11"></a>            <span class="kw">StatsLogInfo</span>();</span>
<span id="cb6-12"><a href="2-mat-met.html#cb6-12"></a>            }</span>
<span id="cb6-13"><a href="2-mat-met.html#cb6-13"></a><span class="co">#   open and define binary</span></span>
<span id="cb6-14"><a href="2-mat-met.html#cb6-14"></a>        <span class="kw">open</span>(bindir<span class="op">+</span>bindirlist[b]);</span>
<span id="cb6-15"><a href="2-mat-met.html#cb6-15"></a>          BIN =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb6-16"><a href="2-mat-met.html#cb6-16"></a><span class="co"># open and define orginal </span></span>
<span id="cb6-17"><a href="2-mat-met.html#cb6-17"></a>        <span class="cf">if</span> (dual) {</span>
<span id="cb6-18"><a href="2-mat-met.html#cb6-18"></a>            <span class="kw">open</span>(rcdirc1<span class="op">+</span>rcdirc1list[b]);</span>
<span id="cb6-19"><a href="2-mat-met.html#cb6-19"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-20"><a href="2-mat-met.html#cb6-20"></a>            <span class="kw">open</span>(rcdir<span class="op">+</span>rcdirlist[b]);</span>
<span id="cb6-21"><a href="2-mat-met.html#cb6-21"></a>        }</span>
<span id="cb6-22"><a href="2-mat-met.html#cb6-22"></a>        RC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb6-23"><a href="2-mat-met.html#cb6-23"></a>        dotIndex =<span class="st"> </span><span class="kw">indexOf</span>(RC, <span class="st">&quot;.&quot;</span>);</span>
<span id="cb6-24"><a href="2-mat-met.html#cb6-24"></a>        title =<span class="st"> </span><span class="kw">substring</span>(RC, <span class="dv">0</span>, dotIndex);</span>
<span id="cb6-25"><a href="2-mat-met.html#cb6-25"></a>    </span>
<span id="cb6-26"><a href="2-mat-met.html#cb6-26"></a><span class="co"># #############################  ENTER 2nd LOOP TO INCREMENT OVER EACH SLICE OF THE TIME-SERIES ##############################</span></span>
<span id="cb6-27"><a href="2-mat-met.html#cb6-27"></a></span>
<span id="cb6-28"><a href="2-mat-met.html#cb6-28"></a>    <span class="kw">selectWindow</span>(BIN);   <span class="co"># select binary</span></span>
<span id="cb6-29"><a href="2-mat-met.html#cb6-29"></a>    pangles =<span class="st"> </span><span class="kw">newArray</span>(<span class="kw">nSlices</span>()<span class="op">+</span><span class="dv">1</span>);</span>
<span id="cb6-30"><a href="2-mat-met.html#cb6-30"></a>        <span class="cf">for</span> (<span class="dt">i=</span><span class="dv">1</span> ; i<span class="op">&lt;=</span><span class="kw">nSlices</span>(); i<span class="op">++</span>) {</span>
<span id="cb6-31"><a href="2-mat-met.html#cb6-31"></a>            s =<span class="st"> </span><span class="kw">nSlices</span>();</span>
<span id="cb6-32"><a href="2-mat-met.html#cb6-32"></a>            <span class="kw">setSlice</span>(i);</span>
<span id="cb6-33"><a href="2-mat-met.html#cb6-33"></a>            <span class="cf">if</span> (ccs) {</span>
<span id="cb6-34"><a href="2-mat-met.html#cb6-34"></a>                <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size=150-10000 include add&quot;</span>);</span>
<span id="cb6-35"><a href="2-mat-met.html#cb6-35"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-36"><a href="2-mat-met.html#cb6-36"></a>                <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size=750-10000 include add&quot;</span>);</span>
<span id="cb6-37"><a href="2-mat-met.html#cb6-37"></a>            }</span>
<span id="cb6-38"><a href="2-mat-met.html#cb6-38"></a>        <span class="co">#   Loop though ROI List</span></span>
<span id="cb6-39"><a href="2-mat-met.html#cb6-39"></a>            <span class="cf">for</span> (<span class="dt">j=</span><span class="dv">0</span> ; j<span class="op">&lt;</span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb6-40"><a href="2-mat-met.html#cb6-40"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb6-41"><a href="2-mat-met.html#cb6-41"></a>                <span class="kw">run</span>(<span class="st">&quot;Set Scale...&quot;</span>, <span class="st">&quot;distance=1 known=0.00005 pixel=1 unit=micron&quot;</span>);</span>
<span id="cb6-42"><a href="2-mat-met.html#cb6-42"></a>                List.setMeasurements;</span>
<span id="cb6-43"><a href="2-mat-met.html#cb6-43"></a>                x =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb6-44"><a href="2-mat-met.html#cb6-44"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, x);</span>
<span id="cb6-45"><a href="2-mat-met.html#cb6-45"></a>            }</span>
<span id="cb6-46"><a href="2-mat-met.html#cb6-46"></a>            <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels=1 slices=1 frames=[s] unit=micron pixel_width=[xs] pixel_height=[ys] voxel_depth=[zs] frame=[time] global&quot;</span>);</span>
<span id="cb6-47"><a href="2-mat-met.html#cb6-47"></a>        <span class="co">#   Sort ROIs and select last one</span></span>
<span id="cb6-48"><a href="2-mat-met.html#cb6-48"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;Sort&quot;</span>);</span>
<span id="cb6-49"><a href="2-mat-met.html#cb6-49"></a>            <span class="cf">for</span> (<span class="dt">j=</span><span class="dv">0</span> ; j<span class="op">&lt;</span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb6-50"><a href="2-mat-met.html#cb6-50"></a>                ccn =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="op">+</span>j;</span>
<span id="cb6-51"><a href="2-mat-met.html#cb6-51"></a>                <span class="cf">if</span> (ccn <span class="op">==</span><span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>)) {</span>
<span id="cb6-52"><a href="2-mat-met.html#cb6-52"></a>                    ccn =<span class="st"> &quot;prim&quot;</span>;</span>
<span id="cb6-53"><a href="2-mat-met.html#cb6-53"></a>                    roiselect =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="op">-</span><span class="dv">1</span>;</span>
<span id="cb6-54"><a href="2-mat-met.html#cb6-54"></a>                } <span class="cf">else</span> {</span>
<span id="cb6-55"><a href="2-mat-met.html#cb6-55"></a>                    ccn =<span class="st"> &quot;CC&quot;</span><span class="op">+</span>j;</span>
<span id="cb6-56"><a href="2-mat-met.html#cb6-56"></a>                    roiselect =<span class="st"> </span>j<span class="dv">-1</span>;</span>
<span id="cb6-57"><a href="2-mat-met.html#cb6-57"></a>                }</span>
<span id="cb6-58"><a href="2-mat-met.html#cb6-58"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, roiselect);</span>
<span id="cb6-59"><a href="2-mat-met.html#cb6-59"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, ccn);</span>
<span id="cb6-60"><a href="2-mat-met.html#cb6-60"></a>            }</span>
<span id="cb6-61"><a href="2-mat-met.html#cb6-61"></a>            rmc =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb6-62"><a href="2-mat-met.html#cb6-62"></a>            m =<span class="st"> </span>rmc<span class="dv">-1</span>;</span>
<span id="cb6-63"><a href="2-mat-met.html#cb6-63"></a>            <span class="kw">selectWindow</span>(RC);</span>
<span id="cb6-64"><a href="2-mat-met.html#cb6-64"></a>  <span class="co">#   Prim registration</span></span>
<span id="cb6-65"><a href="2-mat-met.html#cb6-65"></a>            <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-66"><a href="2-mat-met.html#cb6-66"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-67"><a href="2-mat-met.html#cb6-67"></a>            sln =<span class="st"> </span><span class="kw">getSliceNumber</span>();</span>
<span id="cb6-68"><a href="2-mat-met.html#cb6-68"></a>            <span class="kw">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge=6&quot;</span>);</span>
<span id="cb6-69"><a href="2-mat-met.html#cb6-69"></a>            <span class="kw">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb6-70"><a href="2-mat-met.html#cb6-70"></a>            <span class="kw">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot;use&quot;</span>);</span>
<span id="cb6-71"><a href="2-mat-met.html#cb6-71"></a>            <span class="kw">rename</span>(sln);</span>
<span id="cb6-72"><a href="2-mat-met.html#cb6-72"></a>            <span class="kw">resetMinAndMax</span>();</span>
<span id="cb6-73"><a href="2-mat-met.html#cb6-73"></a>    <span class="co">#   Rotate</span></span>
<span id="cb6-74"><a href="2-mat-met.html#cb6-74"></a>            List.setMeasurements;</span>
<span id="cb6-75"><a href="2-mat-met.html#cb6-75"></a>            A =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb6-76"><a href="2-mat-met.html#cb6-76"></a>            <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-77"><a href="2-mat-met.html#cb6-77"></a>            <span class="cf">if</span> (A <span class="op">&lt;</span><span class="st"> </span><span class="dv">10</span>) {</span>
<span id="cb6-78"><a href="2-mat-met.html#cb6-78"></a>                A =<span class="st"> </span>A;</span>
<span id="cb6-79"><a href="2-mat-met.html#cb6-79"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-80"><a href="2-mat-met.html#cb6-80"></a>                A =<span class="st"> </span><span class="dv">180</span><span class="op">-</span>A;</span>
<span id="cb6-81"><a href="2-mat-met.html#cb6-81"></a>                A =<span class="st"> </span>A<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);</span>
<span id="cb6-82"><a href="2-mat-met.html#cb6-82"></a>            }</span>
<span id="cb6-83"><a href="2-mat-met.html#cb6-83"></a>            pangles[i] =<span class="st"> </span>A;</span>
<span id="cb6-84"><a href="2-mat-met.html#cb6-84"></a>            <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=[A] grid=1 interpolation=Bilinear slice&quot;</span>);</span>
<span id="cb6-85"><a href="2-mat-met.html#cb6-85"></a>            <span class="kw">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>);</span>
<span id="cb6-86"><a href="2-mat-met.html#cb6-86"></a>            <span class="kw">selectWindow</span>(RC); <span class="co"># select &amp; deselect to remove selected ROIs</span></span>
<span id="cb6-87"><a href="2-mat-met.html#cb6-87"></a>            <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-88"><a href="2-mat-met.html#cb6-88"></a>    <span class="co">#   Measure and save segmented Mask ROI</span></span>
<span id="cb6-89"><a href="2-mat-met.html#cb6-89"></a>            pLLProis =<span class="st"> </span>embryodir <span class="op">+</span><span class="st"> </span>File.separator <span class="op">+</span><span class="st"> &quot;ROIs&quot;</span> <span class="op">+</span><span class="st"> </span>File.separator;</span>
<span id="cb6-90"><a href="2-mat-met.html#cb6-90"></a>            <span class="kw">File.makeDirectory</span>(pLLProis);</span>
<span id="cb6-91"><a href="2-mat-met.html#cb6-91"></a>            pLLPxy =<span class="st"> </span>embryodir <span class="op">+</span><span class="st"> </span>File.separator <span class="op">+</span><span class="st"> &quot;ROIsXY&quot;</span> <span class="op">+</span><span class="st"> </span>File.separator;</span>
<span id="cb6-92"><a href="2-mat-met.html#cb6-92"></a>            <span class="kw">File.makeDirectory</span>(pLLPxy);</span>
<span id="cb6-93"><a href="2-mat-met.html#cb6-93"></a>            <span class="kw">selectWindow</span>(BIN);</span>
<span id="cb6-94"><a href="2-mat-met.html#cb6-94"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;show none&quot;</span>); <span class="co"># supress roimanager popping up</span></span>
<span id="cb6-95"><a href="2-mat-met.html#cb6-95"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-96"><a href="2-mat-met.html#cb6-96"></a>        <span class="co">#   Save ROIs and XY coordinates</span></span>
<span id="cb6-97"><a href="2-mat-met.html#cb6-97"></a>            <span class="cf">if</span> (i<span class="op">&lt;</span><span class="dv">10</span>) {</span>
<span id="cb6-98"><a href="2-mat-met.html#cb6-98"></a>                slice =<span class="st"> </span><span class="kw">d2s</span>(<span class="dv">0</span>,<span class="dv">0</span>) <span class="op">+</span><span class="st"> </span><span class="kw">d2s</span>(i,<span class="dv">0</span>);</span>
<span id="cb6-99"><a href="2-mat-met.html#cb6-99"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;save&quot;</span>, pLLProis <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>slice <span class="op">+</span><span class="st"> &quot;.zip&quot;</span>);</span>
<span id="cb6-100"><a href="2-mat-met.html#cb6-100"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;XY Coordinates&quot;</span>, pLLPxy <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>slice <span class="op">+</span><span class="st"> &quot;.txt&quot;</span>);</span>
<span id="cb6-101"><a href="2-mat-met.html#cb6-101"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-102"><a href="2-mat-met.html#cb6-102"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;save&quot;</span>, pLLProis <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>i <span class="op">+</span><span class="st"> &quot;.zip&quot;</span>);</span>
<span id="cb6-103"><a href="2-mat-met.html#cb6-103"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;XY Coordinates&quot;</span>, pLLPxy <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>i <span class="op">+</span><span class="st"> &quot;.txt&quot;</span>);</span>
<span id="cb6-104"><a href="2-mat-met.html#cb6-104"></a>            }</span>
<span id="cb6-105"><a href="2-mat-met.html#cb6-105"></a>    <span class="co">#   Measure</span></span>
<span id="cb6-106"><a href="2-mat-met.html#cb6-106"></a>            <span class="kw">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, <span class="st">&quot;area centroid bounding fit shape feret&#39;s stack redirect=None decimal=2&quot;</span>);</span>
<span id="cb6-107"><a href="2-mat-met.html#cb6-107"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;measure&quot;</span>);</span>
<span id="cb6-108"><a href="2-mat-met.html#cb6-108"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-109"><a href="2-mat-met.html#cb6-109"></a>            <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-110"><a href="2-mat-met.html#cb6-110"></a>    <span class="co">#  Calculate additional variables based on measurements</span></span>
<span id="cb6-111"><a href="2-mat-met.html#cb6-111"></a>            n =<span class="st"> </span><span class="kw">nResults</span>();</span>
<span id="cb6-112"><a href="2-mat-met.html#cb6-112"></a>            r =<span class="st"> </span>n<span class="dv">-1</span>;  <span class="co"># actual RowNumber</span></span>
<span id="cb6-113"><a href="2-mat-met.html#cb6-113"></a>            r2 =<span class="st"> </span>n<span class="dv">-2</span>; <span class="co"># RowNumber -1</span></span>
<span id="cb6-114"><a href="2-mat-met.html#cb6-114"></a>            <span class="cf">if</span> (i <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {  <span class="co"># get X &amp; Y coordinates, keep X0 and Y0 for normalization</span></span>
<span id="cb6-115"><a href="2-mat-met.html#cb6-115"></a>                X0 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb6-116"><a href="2-mat-met.html#cb6-116"></a>                Y0 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb6-117"><a href="2-mat-met.html#cb6-117"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-118"><a href="2-mat-met.html#cb6-118"></a>                X1 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;X&quot;</span>, r2);</span>
<span id="cb6-119"><a href="2-mat-met.html#cb6-119"></a>                X2 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;X&quot;</span>, r);</span>
<span id="cb6-120"><a href="2-mat-met.html#cb6-120"></a>                Y1 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Y&quot;</span>, r2);</span>
<span id="cb6-121"><a href="2-mat-met.html#cb6-121"></a>                Y2 =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Y&quot;</span>, r); </span>
<span id="cb6-122"><a href="2-mat-met.html#cb6-122"></a>            }</span>
<span id="cb6-123"><a href="2-mat-met.html#cb6-123"></a>    <span class="co">#   Width of bounding rectangle</span></span>
<span id="cb6-124"><a href="2-mat-met.html#cb6-124"></a>            W =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Width&quot;</span>);</span>
<span id="cb6-125"><a href="2-mat-met.html#cb6-125"></a>    <span class="co">#   Calculations (XN = normalized X; LE = Leading Edge)</span></span>
<span id="cb6-126"><a href="2-mat-met.html#cb6-126"></a>    <span class="co">#   Euclidian Distance of X + normalized to offspring &#39;zero&#39;</span></span>
<span id="cb6-127"><a href="2-mat-met.html#cb6-127"></a>            <span class="cf">if</span> (i <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb6-128"><a href="2-mat-met.html#cb6-128"></a>                XED =<span class="st"> </span><span class="dv">0</span>;</span>
<span id="cb6-129"><a href="2-mat-met.html#cb6-129"></a>                XN =<span class="st"> </span><span class="dv">0</span>;</span>
<span id="cb6-130"><a href="2-mat-met.html#cb6-130"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-131"><a href="2-mat-met.html#cb6-131"></a>                XED =<span class="st"> </span><span class="kw">sqrt</span>((X2<span class="op">-</span>X1)<span class="op">*</span>(X2<span class="op">-</span>X1)<span class="op">+</span>(Y2<span class="op">-</span>Y1)<span class="op">*</span>(Y2<span class="op">-</span>Y1));</span>
<span id="cb6-132"><a href="2-mat-met.html#cb6-132"></a>                XN =<span class="st"> </span>(X2 <span class="op">-</span><span class="st"> </span>X0) <span class="op">+</span><span class="st"> </span>XED;</span>
<span id="cb6-133"><a href="2-mat-met.html#cb6-133"></a>            }</span>
<span id="cb6-134"><a href="2-mat-met.html#cb6-134"></a>            LE =<span class="st"> </span>XN <span class="op">+</span><span class="st"> </span>(W<span class="op">/</span><span class="dv">2</span>); <span class="co"># Leading Edge </span></span>
<span id="cb6-135"><a href="2-mat-met.html#cb6-135"></a>            T =<span class="st"> </span>time <span class="op">*</span><span class="st"> </span>r; <span class="co"># Time interval</span></span>
<span id="cb6-136"><a href="2-mat-met.html#cb6-136"></a>            <span class="kw">setResult</span>(<span class="st">&quot;embryo&quot;</span>, r, orgname); <span class="co"># set Results</span></span>
<span id="cb6-137"><a href="2-mat-met.html#cb6-137"></a>            <span class="kw">setResult</span>(<span class="st">&quot;group&quot;</span>, r, type);</span>
<span id="cb6-138"><a href="2-mat-met.html#cb6-138"></a>            <span class="kw">setResult</span>(<span class="st">&quot;time&quot;</span>, r, T);</span>
<span id="cb6-139"><a href="2-mat-met.html#cb6-139"></a>            <span class="kw">setResult</span>(<span class="st">&quot;deg&quot;</span>, r, A);</span>
<span id="cb6-140"><a href="2-mat-met.html#cb6-140"></a>            <span class="kw">setResult</span>(<span class="st">&quot;X_ED&quot;</span>, r, XED);</span>
<span id="cb6-141"><a href="2-mat-met.html#cb6-141"></a>            <span class="kw">setResult</span>(<span class="st">&quot;X_N&quot;</span>, r, XN);</span>
<span id="cb6-142"><a href="2-mat-met.html#cb6-142"></a>            <span class="kw">setResult</span>(<span class="st">&quot;LE&quot;</span>, r, LE);</span>
<span id="cb6-143"><a href="2-mat-met.html#cb6-143"></a>            <span class="kw">updateResults</span>();</span>
<span id="cb6-144"><a href="2-mat-met.html#cb6-144"></a>    <span class="co">#   Velocitiy LE (LE1 = LE @ timepoint 1; LEN = normalized value of LE, LENV = Velocity of the normalized value of LE)</span></span>
<span id="cb6-145"><a href="2-mat-met.html#cb6-145"></a>            <span class="cf">if</span> (i <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb6-146"><a href="2-mat-met.html#cb6-146"></a>                LE1 =<span class="st"> </span>LE; <span class="co"># LE1 will be the same for all further timepoints</span></span>
<span id="cb6-147"><a href="2-mat-met.html#cb6-147"></a>                LEN =<span class="st"> </span><span class="dv">0</span>; <span class="co">#</span></span>
<span id="cb6-148"><a href="2-mat-met.html#cb6-148"></a>                LENED =<span class="st"> </span><span class="ot">NaN</span>; <span class="co"># &#39;Leading Edge Normalized Euclidian Distance&#39;</span></span>
<span id="cb6-149"><a href="2-mat-met.html#cb6-149"></a>                LEV =<span class="st"> </span><span class="ot">NaN</span>; <span class="co"># For the first timepoint there can be no speed, since there was no coordinate of X and Y before</span></span>
<span id="cb6-150"><a href="2-mat-met.html#cb6-150"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-151"><a href="2-mat-met.html#cb6-151"></a>                LEN =<span class="st"> </span>LE <span class="op">-</span><span class="st"> </span>LE1; <span class="co"># The value of &#39;LE Normalized&#39; to zero </span></span>
<span id="cb6-152"><a href="2-mat-met.html#cb6-152"></a>                LED =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;LE_N&quot;</span>, r2); <span class="co"># LED = The value of LE one row before</span></span>
<span id="cb6-153"><a href="2-mat-met.html#cb6-153"></a>                LENED =<span class="st"> </span><span class="kw">sqrt</span>((LEN<span class="op">-</span>LED)<span class="op">*</span>(LEN<span class="op">-</span>LED)<span class="op">+</span>(Y2<span class="op">-</span>Y1)<span class="op">*</span>(Y2<span class="op">-</span>Y1)); <span class="co"># LENED = LEN - (LEN-LED);</span></span>
<span id="cb6-154"><a href="2-mat-met.html#cb6-154"></a>                LEV =<span class="st"> </span>LENED <span class="op">/</span><span class="st"> </span>time;</span>
<span id="cb6-155"><a href="2-mat-met.html#cb6-155"></a>            }</span>
<span id="cb6-156"><a href="2-mat-met.html#cb6-156"></a>            <span class="kw">setResult</span>(<span class="st">&quot;LE_N&quot;</span>, r, LEN); <span class="co"># setResult Leading Edge Normalized (LE_N)</span></span>
<span id="cb6-157"><a href="2-mat-met.html#cb6-157"></a>            <span class="kw">setResult</span>(<span class="st">&quot;LE_N_ED&quot;</span>, r, LENED); <span class="co"># setResult Leading Edge Normalized Euclidian Distance (LE_N_ED)</span></span>
<span id="cb6-158"><a href="2-mat-met.html#cb6-158"></a>            <span class="kw">setResult</span>(<span class="st">&quot;LE_V&quot;</span>, r, LEV); <span class="co"># setResult Leading Edge Velocity (LE_V)</span></span>
<span id="cb6-159"><a href="2-mat-met.html#cb6-159"></a>            <span class="kw">updateResults</span>();</span>
<span id="cb6-160"><a href="2-mat-met.html#cb6-160"></a>            }</span>
<span id="cb6-161"><a href="2-mat-met.html#cb6-161"></a>            <span class="kw">close</span>(BIN); <span class="co"># could be reduced to close(BIN, ORG); or close (&quot;.tif&quot;);</span></span>
<span id="cb6-162"><a href="2-mat-met.html#cb6-162"></a>            <span class="kw">close</span>(RC);</span>
<span id="cb6-163"><a href="2-mat-met.html#cb6-163"></a>        <span class="co">#   Merge registered prim timepoints</span></span>
<span id="cb6-164"><a href="2-mat-met.html#cb6-164"></a>            <span class="kw">setBatchMode</span>(<span class="st">&quot;exit and display&quot;</span>);</span>
<span id="cb6-165"><a href="2-mat-met.html#cb6-165"></a>            <span class="kw">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;method=[Copy (top-left)] name=Stack title=[] use&quot;</span>);</span>
<span id="cb6-166"><a href="2-mat-met.html#cb6-166"></a>            <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels=1 slices=1 frames=[s] unit=micron pixel_width=[xs] pixel_height=[ys] voxel_depth=[zs] frame=[time] global&quot;</span>); </span>
<span id="cb6-167"><a href="2-mat-met.html#cb6-167"></a>            <span class="kw">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb6-168"><a href="2-mat-met.html#cb6-168"></a>            <span class="cf">if</span> (dual) {</span>
<span id="cb6-169"><a href="2-mat-met.html#cb6-169"></a>            <span class="co">#   save C1</span></span>
<span id="cb6-170"><a href="2-mat-met.html#cb6-170"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, pLLPdir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;-C01.tif&quot;</span>);</span>
<span id="cb6-171"><a href="2-mat-met.html#cb6-171"></a>                <span class="kw">close</span>();</span>
<span id="cb6-172"><a href="2-mat-met.html#cb6-172"></a>            <span class="co">#   open C2</span></span>
<span id="cb6-173"><a href="2-mat-met.html#cb6-173"></a>                <span class="kw">open</span>(rcdirc2<span class="op">+</span>rcdirc2list[b]);</span>
<span id="cb6-174"><a href="2-mat-met.html#cb6-174"></a>                <span class="kw">resetMinAndMax</span>();</span>
<span id="cb6-175"><a href="2-mat-met.html#cb6-175"></a>                RC =<span class="st"> </span><span class="kw">getTitle</span>();</span>
<span id="cb6-176"><a href="2-mat-met.html#cb6-176"></a>                dotIndex =<span class="st"> </span><span class="kw">indexOf</span>(RC, <span class="st">&quot;.&quot;</span>);</span>
<span id="cb6-177"><a href="2-mat-met.html#cb6-177"></a>                title =<span class="st"> </span><span class="kw">substring</span>(RC, <span class="dv">0</span>, dotIndex);</span>
<span id="cb6-178"><a href="2-mat-met.html#cb6-178"></a>                <span class="cf">for</span> (<span class="dt">i=</span><span class="dv">1</span> ; i<span class="op">&lt;=</span><span class="kw">nSlices</span>(); i<span class="op">++</span>) {</span>
<span id="cb6-179"><a href="2-mat-met.html#cb6-179"></a>                    s =<span class="st"> </span>i;</span>
<span id="cb6-180"><a href="2-mat-met.html#cb6-180"></a>                    <span class="kw">setSlice</span>(i);</span>
<span id="cb6-181"><a href="2-mat-met.html#cb6-181"></a>                    <span class="kw">wait</span>(<span class="dv">200</span>);</span>
<span id="cb6-182"><a href="2-mat-met.html#cb6-182"></a>            <span class="co">#   Prim registration</span></span>
<span id="cb6-183"><a href="2-mat-met.html#cb6-183"></a>                    <span class="cf">if</span> (i<span class="op">&lt;</span><span class="dv">10</span>) {</span>
<span id="cb6-184"><a href="2-mat-met.html#cb6-184"></a>                        slice =<span class="st"> </span><span class="kw">d2s</span>(<span class="dv">0</span>,<span class="dv">0</span>) <span class="op">+</span><span class="st"> </span><span class="kw">d2s</span>(i,<span class="dv">0</span>);</span>
<span id="cb6-185"><a href="2-mat-met.html#cb6-185"></a>                        <span class="kw">roiManager</span>(<span class="st">&quot;open&quot;</span>, pLLProis <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>slice <span class="op">+</span><span class="st"> &quot;.zip&quot;</span>);</span>
<span id="cb6-186"><a href="2-mat-met.html#cb6-186"></a>                    } <span class="cf">else</span> {</span>
<span id="cb6-187"><a href="2-mat-met.html#cb6-187"></a>                        <span class="kw">roiManager</span>(<span class="st">&quot;open&quot;</span>, pLLProis <span class="op">+</span><span class="st"> &quot;s&quot;</span> <span class="op">+</span><span class="st"> </span>i <span class="op">+</span><span class="st"> &quot;.zip&quot;</span>);</span>
<span id="cb6-188"><a href="2-mat-met.html#cb6-188"></a>                    }</span>
<span id="cb6-189"><a href="2-mat-met.html#cb6-189"></a>                    rmc =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb6-190"><a href="2-mat-met.html#cb6-190"></a>                    m =<span class="st"> </span>rmc<span class="dv">-1</span>;</span>
<span id="cb6-191"><a href="2-mat-met.html#cb6-191"></a>                    <span class="kw">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-192"><a href="2-mat-met.html#cb6-192"></a>                    <span class="kw">selectWindow</span>(RC);</span>
<span id="cb6-193"><a href="2-mat-met.html#cb6-193"></a>                    sln =<span class="st"> </span><span class="kw">getSliceNumber</span>();</span>
<span id="cb6-194"><a href="2-mat-met.html#cb6-194"></a>                    <span class="kw">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge=6&quot;</span>);</span>
<span id="cb6-195"><a href="2-mat-met.html#cb6-195"></a>                    <span class="kw">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb6-196"><a href="2-mat-met.html#cb6-196"></a>                    <span class="kw">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot;use&quot;</span>);</span>
<span id="cb6-197"><a href="2-mat-met.html#cb6-197"></a>                    <span class="kw">rename</span>(sln);</span>
<span id="cb6-198"><a href="2-mat-met.html#cb6-198"></a>                <span class="co">#   Rotate</span></span>
<span id="cb6-199"><a href="2-mat-met.html#cb6-199"></a>                    A =<span class="st"> </span>pangles[s];</span>
<span id="cb6-200"><a href="2-mat-met.html#cb6-200"></a>                    <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-201"><a href="2-mat-met.html#cb6-201"></a>                    <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle=[A] grid=1 interpolation=Bilinear slice&quot;</span>);</span>
<span id="cb6-202"><a href="2-mat-met.html#cb6-202"></a>                    <span class="kw">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>);</span>
<span id="cb6-203"><a href="2-mat-met.html#cb6-203"></a>                    <span class="kw">selectWindow</span>(RC); <span class="co"># select &amp; deselect to remove selected ROIs</span></span>
<span id="cb6-204"><a href="2-mat-met.html#cb6-204"></a>                    <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-205"><a href="2-mat-met.html#cb6-205"></a>                    <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-206"><a href="2-mat-met.html#cb6-206"></a>                }</span>
<span id="cb6-207"><a href="2-mat-met.html#cb6-207"></a>            <span class="co">#   close and merge</span></span>
<span id="cb6-208"><a href="2-mat-met.html#cb6-208"></a>                <span class="kw">close</span>(RC);</span>
<span id="cb6-209"><a href="2-mat-met.html#cb6-209"></a>                <span class="kw">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;method=[Copy (top-left)] name=Stack title=[] use&quot;</span>);</span>
<span id="cb6-210"><a href="2-mat-met.html#cb6-210"></a>                <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels=1 slices=1 frames=[s] unit=micron pixel_width=[xs] pixel_height=[ys] voxel_depth=[zs] frame=[time] global&quot;</span>); </span>
<span id="cb6-211"><a href="2-mat-met.html#cb6-211"></a>                <span class="kw">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb6-212"><a href="2-mat-met.html#cb6-212"></a>                <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-213"><a href="2-mat-met.html#cb6-213"></a>          <span class="co"># save C2</span></span>
<span id="cb6-214"><a href="2-mat-met.html#cb6-214"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, pLLPdir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;-C02.tif&quot;</span>);</span>
<span id="cb6-215"><a href="2-mat-met.html#cb6-215"></a>                <span class="kw">close</span>();</span>
<span id="cb6-216"><a href="2-mat-met.html#cb6-216"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-217"><a href="2-mat-met.html#cb6-217"></a>                <span class="kw">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, pLLPdir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;.tif&quot;</span>);</span>
<span id="cb6-218"><a href="2-mat-met.html#cb6-218"></a>                <span class="kw">close</span>();</span>
<span id="cb6-219"><a href="2-mat-met.html#cb6-219"></a>            }</span>
<span id="cb6-220"><a href="2-mat-met.html#cb6-220"></a>    <span class="co">#   Save Results Table</span></span>
<span id="cb6-221"><a href="2-mat-met.html#cb6-221"></a>            <span class="kw">run</span>(<span class="st">&quot;Input/Output...&quot;</span>, <span class="st">&quot;jpeg=100 gif=-1 file=.txt use_file copy_column copy_row save_column&quot;</span>);</span>
<span id="cb6-222"><a href="2-mat-met.html#cb6-222"></a>            <span class="kw">saveAs</span>(<span class="st">&quot;results&quot;</span>, embryodir <span class="op">+</span><span class="st"> </span>orgname <span class="op">+</span><span class="st"> &quot;_Results&quot;</span> <span class="op">+</span><span class="st"> &quot;.txt&quot;</span>);</span>
<span id="cb6-223"><a href="2-mat-met.html#cb6-223"></a>        <span class="co"># Calulate Stats and show in Log</span></span>
<span id="cb6-224"><a href="2-mat-met.html#cb6-224"></a>            <span class="kw">StatsLog</span>();</span>
<span id="cb6-225"><a href="2-mat-met.html#cb6-225"></a>            <span class="kw">cleanup</span>();</span></code></pre></div>
</div>
</div>
<div id="mat-anallzr3d" class="section level5">
<h5><span class="header-section-number">2.2.4.1.3</span> anaLLzR3D</h5>
<p>Upon macro execution the opening dialog is presented which is divided in three sections (figure <a href="2-mat-met.html#fig:anallzr3ddialog">2.8</a>A).</p>
<p>In the <strong>processing</strong> section the user has to define the input format as well as in which direction the Z-stack was recorded. Furthermore, the user may choose choose to have the pLLP registered, save intermediate steps for debugging and to have objects segmented without any restrictions and manual ROI correction.</p>
<p>In the <strong>thresholds</strong> section the user may fine tune segmentation and filter levels</p>
<ul>
<li><em>Segmentation</em> controls the segmentation threshold at which the membrane signal is detected, cell volumes are seperated from each other and is evaluated after ground truth data described in section <a href="2-mat-met.html#mat-GrTrDat">2.3</a></li>
<li><em>Min. volume</em> controls the minimum volume, below which objects are discarded</li>
<li><em>Max. volume</em> controls the maximum volume, above which objects are discarded</li>
</ul>
<p>In the <strong>measurements</strong> section the user may choose may choose whether apical constriction measurement should be applied or not.</p>
<p>In case apical constriction measurement was selected the user may choose whether ACI should be measured at an absolute- or relative- distance from the tip. Furthermore, the user has the option to measure from a fit ellipsoid or rectangle.</p>

<div class="figure" style="text-align: center"><span id="fig:anallzr3ddialog"></span>
<img src="figures/materials/macros/anallzr3D_macro.png" alt="anaLLzR3D opening dialog A Opening dialog: Main functionality B Opening Dialog: Apical Constriction options C Log window after startup" width="95%" />
<p class="caption">
Figure 2.8: anaLLzR3D opening dialog <strong>A</strong> Opening dialog: Main functionality <strong>B</strong> Opening Dialog: Apical Constriction options <strong>C</strong> Log window after startup
</p>
</div>
<p>Code-snippets describing the main functionality are described in the next couple of sub-sections.</p>
<div id="registration-1" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.1</span> Registration</h6>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="2-mat-met.html#cb7-1"></a><span class="co"># get registration parameters angle and height</span></span>
<span id="cb7-2"><a href="2-mat-met.html#cb7-2"></a><span class="co"># steps performed on z-projected data</span></span>
<span id="cb7-3"><a href="2-mat-met.html#cb7-3"></a></span>
<span id="cb7-4"><a href="2-mat-met.html#cb7-4"></a><span class="co"># 2D segmentation mask</span></span>
<span id="cb7-5"><a href="2-mat-met.html#cb7-5"></a>  <span class="kw">run</span>(<span class="st">&quot;Z Project...&quot;</span>, <span class="st">&quot;projection=[Max Intensity]&quot;</span>);</span>
<span id="cb7-6"><a href="2-mat-met.html#cb7-6"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 8 scaled&quot;</span>);</span>
<span id="cb7-7"><a href="2-mat-met.html#cb7-7"></a>    <span class="kw">setAutoThreshold</span>(<span class="st">&quot;Minimum dark&quot;</span>);</span>
<span id="cb7-8"><a href="2-mat-met.html#cb7-8"></a>    <span class="kw">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb7-9"><a href="2-mat-met.html#cb7-9"></a>    <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-10"><a href="2-mat-met.html#cb7-10"></a><span class="co"># angle from horizontal midline</span></span>
<span id="cb7-11"><a href="2-mat-met.html#cb7-11"></a>    <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;include add&quot;</span>);</span>
<span id="cb7-12"><a href="2-mat-met.html#cb7-12"></a>    rmcount =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="op">-</span><span class="dv">1</span>;</span>
<span id="cb7-13"><a href="2-mat-met.html#cb7-13"></a>    <span class="cf">if</span>(<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb7-14"><a href="2-mat-met.html#cb7-14"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-15"><a href="2-mat-met.html#cb7-15"></a>        <span class="kw">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb7-16"><a href="2-mat-met.html#cb7-16"></a>        List.setMeasurements;</span>
<span id="cb7-17"><a href="2-mat-met.html#cb7-17"></a>        Angle =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;FeretAngle&quot;</span>);</span>
<span id="cb7-18"><a href="2-mat-met.html#cb7-18"></a>        <span class="cf">if</span> (Angle <span class="op">&lt;</span><span class="st"> </span><span class="dv">0</span>) {Angle =<span class="st"> </span>Angle <span class="op">*</span><span class="st"> </span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb7-19"><a href="2-mat-met.html#cb7-19"></a>        <span class="cf">if</span> (Angle <span class="op">&gt;</span><span class="st"> </span><span class="dv">90</span>) {Angle =<span class="st"> </span>(<span class="dv">180</span> <span class="op">-</span><span class="st"> </span>Angle) <span class="op">*</span><span class="st"> </span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb7-20"><a href="2-mat-met.html#cb7-20"></a>    } <span class="cf">else</span> {</span>
<span id="cb7-21"><a href="2-mat-met.html#cb7-21"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-22"><a href="2-mat-met.html#cb7-22"></a>        <span class="kw">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb7-23"><a href="2-mat-met.html#cb7-23"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;update&quot;</span>);</span>
<span id="cb7-24"><a href="2-mat-met.html#cb7-24"></a>        List.setMeasurements;</span>
<span id="cb7-25"><a href="2-mat-met.html#cb7-25"></a>        X1Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-26"><a href="2-mat-met.html#cb7-26"></a>        Y1Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-27"><a href="2-mat-met.html#cb7-27"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb7-28"><a href="2-mat-met.html#cb7-28"></a>        List.setMeasurements;</span>
<span id="cb7-29"><a href="2-mat-met.html#cb7-29"></a>        X2Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-30"><a href="2-mat-met.html#cb7-30"></a>        Y2Line =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-31"><a href="2-mat-met.html#cb7-31"></a>        <span class="kw">makeLine</span>(X1Line, Y1Line, X2Line, Y2Line); </span>
<span id="cb7-32"><a href="2-mat-met.html#cb7-32"></a>        List.setMeasurements;</span>
<span id="cb7-33"><a href="2-mat-met.html#cb7-33"></a>        Angle =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb7-34"><a href="2-mat-met.html#cb7-34"></a>        <span class="cf">if</span> (Angle <span class="op">&lt;</span><span class="st"> </span><span class="dv">0</span>) {Angle =<span class="st"> </span>Angle<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb7-35"><a href="2-mat-met.html#cb7-35"></a>        <span class="cf">if</span> (Angle <span class="op">&gt;</span><span class="st"> </span><span class="dv">90</span>) {Angle =<span class="st"> </span>(<span class="dv">180</span><span class="op">-</span>Angle)<span class="op">*</span>(<span class="op">-</span><span class="dv">1</span>);}</span>
<span id="cb7-36"><a href="2-mat-met.html#cb7-36"></a>    }</span>
<span id="cb7-37"><a href="2-mat-met.html#cb7-37"></a>    <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-38"><a href="2-mat-met.html#cb7-38"></a>    <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = &quot;</span><span class="op">+</span><span class="st"> </span>Angle <span class="op">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear&quot;</span>);</span>
<span id="cb7-39"><a href="2-mat-met.html#cb7-39"></a><span class="co"># height to crop image to</span></span>
<span id="cb7-40"><a href="2-mat-met.html#cb7-40"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>); <span class="co"># the image was rotated, so we need to get the ROIs again</span></span>
<span id="cb7-41"><a href="2-mat-met.html#cb7-41"></a>    <span class="kw">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-42"><a href="2-mat-met.html#cb7-42"></a>    <span class="kw">run</span>(<span class="st">&quot;Make Binary&quot;</span>);</span>
<span id="cb7-43"><a href="2-mat-met.html#cb7-43"></a>    <span class="kw">run</span>(<span class="st">&quot;Erode&quot;</span>);</span>
<span id="cb7-44"><a href="2-mat-met.html#cb7-44"></a>    <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size = 150-10000 include exclude add&quot;</span>);</span>
<span id="cb7-45"><a href="2-mat-met.html#cb7-45"></a>    rmcount =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">-</span><span class="st"> </span><span class="dv">1</span>;</span>
<span id="cb7-46"><a href="2-mat-met.html#cb7-46"></a>    <span class="cf">if</span>(<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb7-47"><a href="2-mat-met.html#cb7-47"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-48"><a href="2-mat-met.html#cb7-48"></a>    } <span class="cf">else</span> {</span>
<span id="cb7-49"><a href="2-mat-met.html#cb7-49"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb7-50"><a href="2-mat-met.html#cb7-50"></a>    }</span>
<span id="cb7-51"><a href="2-mat-met.html#cb7-51"></a>    List.setMeasurements;</span>
<span id="cb7-52"><a href="2-mat-met.html#cb7-52"></a>    XRect =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-53"><a href="2-mat-met.html#cb7-53"></a>    YRect =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-54"><a href="2-mat-met.html#cb7-54"></a>    <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb7-55"><a href="2-mat-met.html#cb7-55"></a>    Regwidth =<span class="st"> </span>width;</span>
<span id="cb7-56"><a href="2-mat-met.html#cb7-56"></a>    Regheight =<span class="st"> </span><span class="dv">400</span>; <span class="co"># change height of rectangle here</span></span>
<span id="cb7-57"><a href="2-mat-met.html#cb7-57"></a>    <span class="kw">toUnscaled</span>(YRect);</span>
<span id="cb7-58"><a href="2-mat-met.html#cb7-58"></a>    YRect =<span class="st"> </span>YRect <span class="op">-</span><span class="st"> </span>(Regheight<span class="op">/</span><span class="dv">2</span>);</span></code></pre></div>
</div>
<div id="transformation" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.2</span> Transformation</h6>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="2-mat-met.html#cb8-1"></a><span class="co"># pllp registration</span></span>
<span id="cb8-2"><a href="2-mat-met.html#cb8-2"></a><span class="co"># steps performed on 3D data</span></span>
<span id="cb8-3"><a href="2-mat-met.html#cb8-3"></a><span class="co"># register pllp</span></span>
<span id="cb8-4"><a href="2-mat-met.html#cb8-4"></a>    <span class="kw">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = &quot;</span><span class="op">+</span><span class="st"> </span>Angle <span class="op">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear stack&quot;</span>);</span>
<span id="cb8-5"><a href="2-mat-met.html#cb8-5"></a>    <span class="kw">makeRectangle</span>(<span class="dv">0</span>, YRect, Regwidth, Regheight);</span>
<span id="cb8-6"><a href="2-mat-met.html#cb8-6"></a>    <span class="kw">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb8-7"><a href="2-mat-met.html#cb8-7"></a><span class="co"># Create threshold mask to clear signals outside ROI</span></span>
<span id="cb8-8"><a href="2-mat-met.html#cb8-8"></a>    <span class="kw">run</span>(<span class="st">&quot;Normalize Local Contrast&quot;</span>, <span class="st">&quot;block_radius_x = 300 block_radius_y = 20 standard_deviations = 4 stretch&quot;</span>);</span>
<span id="cb8-9"><a href="2-mat-met.html#cb8-9"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 1 scaled&quot;</span>);</span>
<span id="cb8-10"><a href="2-mat-met.html#cb8-10"></a>    <span class="kw">setAutoThreshold</span>(<span class="st">&quot;Otsu dark&quot;</span>);</span>
<span id="cb8-11"><a href="2-mat-met.html#cb8-11"></a>    <span class="kw">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb8-12"><a href="2-mat-met.html#cb8-12"></a><span class="co"># most right roi</span></span>
<span id="cb8-13"><a href="2-mat-met.html#cb8-13"></a>    <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">0</span> ; j <span class="op">&lt;</span><span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="op">++</span>) {</span>
<span id="cb8-14"><a href="2-mat-met.html#cb8-14"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb8-15"><a href="2-mat-met.html#cb8-15"></a>        <span class="kw">run</span>(<span class="st">&quot;Set Scale...&quot;</span>, <span class="st">&quot;distance = 1 known = 0.00005 pixel = 1 unit = micron&quot;</span>);</span>
<span id="cb8-16"><a href="2-mat-met.html#cb8-16"></a>        List.setMeasurements;</span>
<span id="cb8-17"><a href="2-mat-met.html#cb8-17"></a>    x =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb8-18"><a href="2-mat-met.html#cb8-18"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;rename&quot;</span>, x);</span>
<span id="cb8-19"><a href="2-mat-met.html#cb8-19"></a>    }</span>
<span id="cb8-20"><a href="2-mat-met.html#cb8-20"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;Sort&quot;</span>);</span>
<span id="cb8-21"><a href="2-mat-met.html#cb8-21"></a>    <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = 1 frames = 1 unit = micron pixel_width = [sizeX] pixel_height = [sizeY] voxel_depth = [sizeZ]&quot;</span>);</span>
<span id="cb8-22"><a href="2-mat-met.html#cb8-22"></a>    primroi =<span class="st"> </span><span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">-</span><span class="st"> </span><span class="dv">1</span>;</span>
<span id="cb8-23"><a href="2-mat-met.html#cb8-23"></a>    <span class="kw">roiManager</span>(<span class="st">&#39;select&#39;</span>, primroi);</span>
<span id="cb8-24"><a href="2-mat-met.html#cb8-24"></a><span class="co"># enlarge</span></span>
<span id="cb8-25"><a href="2-mat-met.html#cb8-25"></a>    <span class="kw">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge=10&quot;</span>);</span>
<span id="cb8-26"><a href="2-mat-met.html#cb8-26"></a>    <span class="kw">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb8-27"><a href="2-mat-met.html#cb8-27"></a>    <span class="kw">roiManager</span>(<span class="st">&#39;update&#39;</span>);</span></code></pre></div>
</div>
<div id="rosette-detection" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.3</span> Rosette detection</h6>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="2-mat-met.html#cb9-1"></a><span class="co"># rosette detection</span></span>
<span id="cb9-2"><a href="2-mat-met.html#cb9-2"></a><span class="co"># register pllp based on registration parameters</span></span>
<span id="cb9-3"><a href="2-mat-met.html#cb9-3"></a><span class="co">#  [...] registration</span></span>
<span id="cb9-4"><a href="2-mat-met.html#cb9-4"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 4 scaled&quot;</span>);</span>
<span id="cb9-5"><a href="2-mat-met.html#cb9-5"></a><span class="co"># detect points based on individual image max finder threshold</span></span>
<span id="cb9-6"><a href="2-mat-met.html#cb9-6"></a>    List.setMeasurements;</span>
<span id="cb9-7"><a href="2-mat-met.html#cb9-7"></a>    mean =<span class="st"> </span><span class="kw">List.getValue</span>(<span class="st">&quot;Mean&quot;</span>);</span>
<span id="cb9-8"><a href="2-mat-met.html#cb9-8"></a>    pointthresh =<span class="st"> </span>mean<span class="op">/</span><span class="fl">2.5</span>;</span>
<span id="cb9-9"><a href="2-mat-met.html#cb9-9"></a>    pointthresh =<span class="st"> </span><span class="kw">round</span>(pointthresh);</span>
<span id="cb9-10"><a href="2-mat-met.html#cb9-10"></a>    <span class="kw">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = &quot;</span><span class="op">+</span>pointthresh<span class="op">+</span><span class="st">&quot; output = [Point Selection]&quot;</span>);</span>
<span id="cb9-11"><a href="2-mat-met.html#cb9-11"></a>    <span class="kw">run</span>(<span class="st">&quot;Point Tool...&quot;</span>, <span class="st">&quot;type = Dot color = Green size = [Extra Large] label counter = 0&quot;</span>);</span>
<span id="cb9-12"><a href="2-mat-met.html#cb9-12"></a>    <span class="kw">getSelectionCoordinates</span>(xpoints, ypoints);</span>
<span id="cb9-13"><a href="2-mat-met.html#cb9-13"></a>    <span class="kw">roiManager</span>(<span class="st">&quot;Add&quot;</span>);</span>
<span id="cb9-14"><a href="2-mat-met.html#cb9-14"></a><span class="co"># measure intensities along horizontal midline</span></span>
<span id="cb9-15"><a href="2-mat-met.html#cb9-15"></a><span class="co">#   [...] registration</span></span>
<span id="cb9-16"><a href="2-mat-met.html#cb9-16"></a>    Rlx =<span class="st"> </span><span class="kw">lengthOf</span>(xpoints); <span class="co"># collect Arrays</span></span>
<span id="cb9-17"><a href="2-mat-met.html#cb9-17"></a>    RX =<span class="st"> </span><span class="kw">Array.sort</span>(xpoints); <span class="co"># put xpoints in right order</span></span>
<span id="cb9-18"><a href="2-mat-met.html#cb9-18"></a>    RX =<span class="st"> </span><span class="kw">Array.invert</span>(RX);</span>
<span id="cb9-19"><a href="2-mat-met.html#cb9-19"></a><span class="co"># put number of AC into array + print to log</span></span>
<span id="cb9-20"><a href="2-mat-met.html#cb9-20"></a>    ACnum =<span class="st"> </span><span class="kw">newArray</span>(list.length);</span>
<span id="cb9-21"><a href="2-mat-met.html#cb9-21"></a>    ACnum[i] =<span class="st"> </span>Rlx;</span>
<span id="cb9-22"><a href="2-mat-met.html#cb9-22"></a><span class="co"># Fill ypoints with mean values of all y coordinates</span></span>
<span id="cb9-23"><a href="2-mat-met.html#cb9-23"></a>    <span class="kw">Array.getStatistics</span>(ypoints, min, max, mean, stdDev);</span>
<span id="cb9-24"><a href="2-mat-met.html#cb9-24"></a>    meanline =<span class="st"> </span>mean;</span>
<span id="cb9-25"><a href="2-mat-met.html#cb9-25"></a>    <span class="kw">Array.fill</span>(ypoints, meanline);</span>
<span id="cb9-26"><a href="2-mat-met.html#cb9-26"></a>    RY =<span class="st"> </span>ypoints;</span>
<span id="cb9-27"><a href="2-mat-met.html#cb9-27"></a>    <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb9-28"><a href="2-mat-met.html#cb9-28"></a>    <span class="kw">makeLine</span>(<span class="dv">0</span>, meanline, width, meanline, <span class="dv">1</span>);</span>
<span id="cb9-29"><a href="2-mat-met.html#cb9-29"></a>    <span class="kw">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb9-30"><a href="2-mat-met.html#cb9-30"></a>    profile =<span class="st"> </span><span class="kw">getProfile</span>();</span>
<span id="cb9-31"><a href="2-mat-met.html#cb9-31"></a>    <span class="cf">for</span> (<span class="dt">a =</span> <span class="dv">0</span>; a <span class="op">&lt;</span><span class="st"> </span>profile.length; a<span class="op">++</span>) {</span>
<span id="cb9-32"><a href="2-mat-met.html#cb9-32"></a>    <span class="kw">setResult</span>(<span class="st">&quot;Value&quot;</span>, a, profile[a]);</span>
<span id="cb9-33"><a href="2-mat-met.html#cb9-33"></a>        <span class="kw">updateResults</span>();</span>
<span id="cb9-34"><a href="2-mat-met.html#cb9-34"></a>    }</span></code></pre></div>
</div>
<div id="segmentation-2" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.4</span> Segmentation</h6>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="2-mat-met.html#cb10-1"></a><span class="co"># Image segmentation using the MorphoLibJ&#39;s[@Legland2016] _Morpholigical Segmentation_ plugin</span></span>
<span id="cb10-2"><a href="2-mat-met.html#cb10-2"></a><span class="co"># steps performed on 3D data</span></span>
<span id="cb10-3"><a href="2-mat-met.html#cb10-3"></a></span>
<span id="cb10-4"><a href="2-mat-met.html#cb10-4"></a><span class="co"># 3D gaussian blur</span></span>
<span id="cb10-5"><a href="2-mat-met.html#cb10-5"></a>    <span class="kw">run</span>(<span class="st">&quot;Gaussian Blur 3D...&quot;</span>, <span class="st">&quot;x=2 y=2 z=0.5&quot;</span>);</span>
<span id="cb10-6"><a href="2-mat-met.html#cb10-6"></a>    <span class="kw">resetMinAndMax</span>();</span>
<span id="cb10-7"><a href="2-mat-met.html#cb10-7"></a><span class="co"># run segmentation</span></span>
<span id="cb10-8"><a href="2-mat-met.html#cb10-8"></a>    <span class="kw">run</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-9"><a href="2-mat-met.html#cb10-9"></a>    <span class="kw">selectWindow</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-10"><a href="2-mat-met.html#cb10-10"></a>    <span class="kw">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.setInputImageType&quot;</span>, <span class="st">&quot;border&quot;</span>);</span>
<span id="cb10-11"><a href="2-mat-met.html#cb10-11"></a>    <span class="kw">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.segment&quot;</span>, <span class="st">&quot;tolerance=&quot;</span><span class="op">+</span>tol<span class="op">+</span><span class="st">&quot;&quot;</span>, <span class="st">&quot;calculateDams=true&quot;</span>, <span class="st">&quot;connectivity=6&quot;</span>);</span>
<span id="cb10-12"><a href="2-mat-met.html#cb10-12"></a><span class="co"># wait till segmentation is done</span></span>
<span id="cb10-13"><a href="2-mat-met.html#cb10-13"></a>    initTime =<span class="st"> </span><span class="kw">getTime</span>(); </span>
<span id="cb10-14"><a href="2-mat-met.html#cb10-14"></a>  oldTime =<span class="st"> </span>initTime; </span>
<span id="cb10-15"><a href="2-mat-met.html#cb10-15"></a>    <span class="cf">while</span> (<span class="kw">isOpen</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>)) {</span>
<span id="cb10-16"><a href="2-mat-met.html#cb10-16"></a>      elapsedTime =<span class="st"> </span><span class="kw">getTime</span>() <span class="op">-</span><span class="st"> </span>initTime; </span>
<span id="cb10-17"><a href="2-mat-met.html#cb10-17"></a>        newTime =<span class="st"> </span><span class="kw">getTime</span>() <span class="op">-</span><span class="st"> </span>oldTime;</span>
<span id="cb10-18"><a href="2-mat-met.html#cb10-18"></a>        <span class="cf">if</span> (newTime <span class="op">&gt;</span><span class="st"> </span><span class="dv">10000</span>) {</span>
<span id="cb10-19"><a href="2-mat-met.html#cb10-19"></a>        oldTime =<span class="st"> </span><span class="kw">getTime</span>(); </span>
<span id="cb10-20"><a href="2-mat-met.html#cb10-20"></a>        newTime =<span class="st"> </span><span class="dv">0</span>;</span>
<span id="cb10-21"><a href="2-mat-met.html#cb10-21"></a>        loginfo =<span class="st"> </span><span class="kw">getInfo</span>(<span class="st">&quot;log&quot;</span>);</span>
<span id="cb10-22"><a href="2-mat-met.html#cb10-22"></a>            loginfo =<span class="st"> </span><span class="kw">split</span>(loginfo, <span class="st">&quot; &quot;</span>);</span>
<span id="cb10-23"><a href="2-mat-met.html#cb10-23"></a>            loginfo =<span class="st"> </span><span class="kw">Array.reverse</span>(loginfo);</span>
<span id="cb10-24"><a href="2-mat-met.html#cb10-24"></a>            loginfo =<span class="st"> </span><span class="kw">Array.trim</span>(loginfo, <span class="dv">5</span>);</span>
<span id="cb10-25"><a href="2-mat-met.html#cb10-25"></a>            loginfo =<span class="st"> </span><span class="kw">Array.reverse</span>(loginfo);</span>
<span id="cb10-26"><a href="2-mat-met.html#cb10-26"></a>            loginfo =<span class="st"> </span><span class="kw">split</span>(loginfo[<span class="dv">0</span>], <span class="st">&quot;.&quot;</span>);</span>
<span id="cb10-27"><a href="2-mat-met.html#cb10-27"></a>            loginfo =<span class="st"> </span><span class="kw">Array.reverse</span>(loginfo);</span>
<span id="cb10-28"><a href="2-mat-met.html#cb10-28"></a>            loginfo =<span class="st"> </span>loginfo[<span class="dv">0</span>];</span>
<span id="cb10-29"><a href="2-mat-met.html#cb10-29"></a>            <span class="cf">if</span> (loginfo <span class="op">==</span><span class="st"> &quot;</span><span class="ch">\n</span><span class="st">Whole&quot;</span>) {</span>
<span id="cb10-30"><a href="2-mat-met.html#cb10-30"></a>                <span class="kw">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.setDisplayFormat&quot;</span>, <span class="st">&quot;Catchment basins&quot;</span>);</span>
<span id="cb10-31"><a href="2-mat-met.html#cb10-31"></a>                <span class="kw">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.createResultImage&quot;</span>);</span>
<span id="cb10-32"><a href="2-mat-met.html#cb10-32"></a>                <span class="kw">run</span>(<span class="st">&quot;Grays&quot;</span>);</span>
<span id="cb10-33"><a href="2-mat-met.html#cb10-33"></a>                <span class="kw">selectWindow</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-34"><a href="2-mat-met.html#cb10-34"></a>                <span class="kw">close</span>();</span>
<span id="cb10-35"><a href="2-mat-met.html#cb10-35"></a>            }</span>
<span id="cb10-36"><a href="2-mat-met.html#cb10-36"></a>    }</span>
<span id="cb10-37"><a href="2-mat-met.html#cb10-37"></a>    }</span>
<span id="cb10-38"><a href="2-mat-met.html#cb10-38"></a>  <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = &quot;</span><span class="op">+</span><span class="st"> </span>n <span class="op">+</span><span class="st">&quot; frames = 1 unit = microns pixel_width=&quot;</span><span class="op">+</span><span class="st"> </span>sizeX <span class="op">+</span><span class="st">&quot; pixel_height=&quot;</span><span class="op">+</span><span class="st"> </span>sizeY <span class="op">+</span><span class="st"> &quot;voxel_depth =&quot;</span><span class="op">+</span><span class="st"> </span>sizeZ);</span></code></pre></div>
</div>
<div id="filter-and-clearing" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.5</span> Filter and clearing</h6>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="2-mat-met.html#cb11-1"></a><span class="co"># erase objects V &lt; vmin and V &gt; vmax</span></span>
<span id="cb11-2"><a href="2-mat-met.html#cb11-2"></a>  <span class="kw">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy sync distance_between_centers = 10 distance_max_contact = 1.80&quot;</span>);</span>
<span id="cb11-3"><a href="2-mat-met.html#cb11-3"></a>    <span class="kw">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb11-4"><a href="2-mat-met.html#cb11-4"></a>  <span class="kw">selectWindow</span>(OMap);</span>
<span id="cb11-5"><a href="2-mat-met.html#cb11-5"></a>    <span class="kw">Ext.Manager3D_AddImage</span>();</span>
<span id="cb11-6"><a href="2-mat-met.html#cb11-6"></a><span class="co"># get number of objects</span></span>
<span id="cb11-7"><a href="2-mat-met.html#cb11-7"></a>    <span class="kw">Ext.Manager3D_Count</span>(nb);</span>
<span id="cb11-8"><a href="2-mat-met.html#cb11-8"></a>    <span class="kw">Ext.Manager3D_MultiSelect</span>();</span>
<span id="cb11-9"><a href="2-mat-met.html#cb11-9"></a><span class="co"># loop through all the objects and erase by filter settings</span></span>
<span id="cb11-10"><a href="2-mat-met.html#cb11-10"></a>    <span class="cf">for</span>(<span class="dt">k =</span> <span class="dv">0</span>; k <span class="op">&lt;</span><span class="st"> </span>nb; k<span class="op">++</span>) {</span>
<span id="cb11-11"><a href="2-mat-met.html#cb11-11"></a>        <span class="kw">showStatus</span>(<span class="st">&quot;Processing &quot;</span><span class="op">+</span><span class="st"> </span>k <span class="op">+</span><span class="st">&quot;/&quot;</span><span class="op">+</span><span class="st"> </span>nb);</span>
<span id="cb11-12"><a href="2-mat-met.html#cb11-12"></a>        <span class="kw">Ext.Manager3D_Measure3D</span>(k, <span class="st">&quot;Vol&quot;</span>,V);</span>
<span id="cb11-13"><a href="2-mat-met.html#cb11-13"></a>        <span class="cf">if</span> (V <span class="op">&lt;</span><span class="st"> </span>vmin) {</span>
<span id="cb11-14"><a href="2-mat-met.html#cb11-14"></a>            <span class="kw">Ext.Manager3D_Select</span>(k);</span>
<span id="cb11-15"><a href="2-mat-met.html#cb11-15"></a>            <span class="kw">Ext.Manager3D_Erase</span>();</span>
<span id="cb11-16"><a href="2-mat-met.html#cb11-16"></a>            <span class="cf">if</span> (V <span class="op">&gt;</span><span class="st"> </span>vmax) {</span>
<span id="cb11-17"><a href="2-mat-met.html#cb11-17"></a>                <span class="kw">Ext.Manager3D_Select</span>(k);</span>
<span id="cb11-18"><a href="2-mat-met.html#cb11-18"></a>                <span class="kw">Ext.Manager3D_Erase</span>();</span>
<span id="cb11-19"><a href="2-mat-met.html#cb11-19"></a>            }</span>
<span id="cb11-20"><a href="2-mat-met.html#cb11-20"></a>        }</span>
<span id="cb11-21"><a href="2-mat-met.html#cb11-21"></a>    }</span>
<span id="cb11-22"><a href="2-mat-met.html#cb11-22"></a><span class="co"># clean blank slices from bottom and top</span></span>
<span id="cb11-23"><a href="2-mat-met.html#cb11-23"></a>    <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb11-24"><a href="2-mat-met.html#cb11-24"></a>    var done =<span class="st"> </span>false; </span>
<span id="cb11-25"><a href="2-mat-met.html#cb11-25"></a>    <span class="cf">for</span>(<span class="dt">l =</span> <span class="dv">1</span>; l <span class="op">&lt;</span><span class="st"> </span>slices <span class="op">&amp;&amp;!</span>done; l<span class="op">++</span>) {</span>
<span id="cb11-26"><a href="2-mat-met.html#cb11-26"></a>        <span class="kw">setSlice</span>(l);</span>
<span id="cb11-27"><a href="2-mat-met.html#cb11-27"></a>        <span class="kw">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb11-28"><a href="2-mat-met.html#cb11-28"></a>        <span class="cf">if</span>(max <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb11-29"><a href="2-mat-met.html#cb11-29"></a>        amax =<span class="st"> </span>l<span class="dv">-1</span>;</span>
<span id="cb11-30"><a href="2-mat-met.html#cb11-30"></a>            <span class="kw">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="op">+</span><span class="st"> </span>amax <span class="op">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb11-31"><a href="2-mat-met.html#cb11-31"></a>            <span class="kw">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb11-32"><a href="2-mat-met.html#cb11-32"></a>            <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb11-33"><a href="2-mat-met.html#cb11-33"></a>            <span class="cf">for</span>(<span class="dt">l =</span> <span class="dv">1</span>; l <span class="op">&lt;</span><span class="st"> </span>slices <span class="op">&amp;&amp;!</span>done; l<span class="op">++</span>) {</span>
<span id="cb11-34"><a href="2-mat-met.html#cb11-34"></a>                <span class="kw">setSlice</span>(l);</span>
<span id="cb11-35"><a href="2-mat-met.html#cb11-35"></a>                <span class="kw">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb11-36"><a href="2-mat-met.html#cb11-36"></a>                <span class="cf">if</span>(max <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb11-37"><a href="2-mat-met.html#cb11-37"></a>                bmax =<span class="st"> </span>l<span class="dv">-1</span>;</span>
<span id="cb11-38"><a href="2-mat-met.html#cb11-38"></a>                    <span class="kw">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="op">+</span><span class="st"> </span>bmax <span class="op">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb11-39"><a href="2-mat-met.html#cb11-39"></a>                    <span class="kw">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb11-40"><a href="2-mat-met.html#cb11-40"></a>                    done =<span class="st"> </span>true;</span>
<span id="cb11-41"><a href="2-mat-met.html#cb11-41"></a>                    }</span>
<span id="cb11-42"><a href="2-mat-met.html#cb11-42"></a>            }</span>
<span id="cb11-43"><a href="2-mat-met.html#cb11-43"></a>        }</span>
<span id="cb11-44"><a href="2-mat-met.html#cb11-44"></a>    }</span></code></pre></div>
</div>
<div id="apical-constriction-1" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.6</span> Apical Constriction</h6>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="2-mat-met.html#cb12-1"></a><span class="co"># define Z values</span></span>
<span id="cb12-2"><a href="2-mat-met.html#cb12-2"></a></span>
<span id="cb12-3"><a href="2-mat-met.html#cb12-3"></a>    <span class="cf">if</span> (rosAC) {</span>
<span id="cb12-4"><a href="2-mat-met.html#cb12-4"></a>        cellum =<span class="st"> </span>rosum; <span class="co"># I copy pasted the code, so rosette um would be cell um</span></span>
<span id="cb12-5"><a href="2-mat-met.html#cb12-5"></a>        Zslice =<span class="st"> </span>rosum<span class="op">/</span>sizeZ;</span>
<span id="cb12-6"><a href="2-mat-met.html#cb12-6"></a>        <span class="kw">round</span>(Zslice);</span>
<span id="cb12-7"><a href="2-mat-met.html#cb12-7"></a>    }</span>
<span id="cb12-8"><a href="2-mat-met.html#cb12-8"></a></span>
<span id="cb12-9"><a href="2-mat-met.html#cb12-9"></a><span class="co"># cell Constriction</span></span>
<span id="cb12-10"><a href="2-mat-met.html#cb12-10"></a></span>
<span id="cb12-11"><a href="2-mat-met.html#cb12-11"></a>    <span class="cf">if</span> (cellAC) {</span>
<span id="cb12-12"><a href="2-mat-met.html#cb12-12"></a>        aciza =<span class="st"> </span>cellum <span class="op">/</span><span class="st"> </span>sizeZ;</span>
<span id="cb12-13"><a href="2-mat-met.html#cb12-13"></a>        Zslice =<span class="st"> </span>aciza;</span>
<span id="cb12-14"><a href="2-mat-met.html#cb12-14"></a>        <span class="kw">round</span>(Zslice);</span>
<span id="cb12-15"><a href="2-mat-met.html#cb12-15"></a>        <span class="kw">round</span>(aciza);</span>
<span id="cb12-16"><a href="2-mat-met.html#cb12-16"></a>        <span class="cf">if</span> (fixAC<span class="op">||</span>symAC) {</span>
<span id="cb12-17"><a href="2-mat-met.html#cb12-17"></a>    <span class="co"># crop single objects and measure ----------------------</span></span>
<span id="cb12-18"><a href="2-mat-met.html#cb12-18"></a>      <span class="kw">selectWindow</span>(OMap);</span>
<span id="cb12-19"><a href="2-mat-met.html#cb12-19"></a>        <span class="kw">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy sync distance_between_centers = 10 distance_max_contact = 1.80&quot;</span>);</span>
<span id="cb12-20"><a href="2-mat-met.html#cb12-20"></a>        <span class="kw">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb12-21"><a href="2-mat-met.html#cb12-21"></a>        <span class="kw">Ext.Manager3D_AddImage</span>();</span>
<span id="cb12-22"><a href="2-mat-met.html#cb12-22"></a>    <span class="co"># get number of objects</span></span>
<span id="cb12-23"><a href="2-mat-met.html#cb12-23"></a>        <span class="kw">Ext.Manager3D_Count</span>(nb);</span>
<span id="cb12-24"><a href="2-mat-met.html#cb12-24"></a>    <span class="co"># create arrays to fill with measurements</span></span>
<span id="cb12-25"><a href="2-mat-met.html#cb12-25"></a>        objlabelArray =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-26"><a href="2-mat-met.html#cb12-26"></a>        MajorAngle =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-27"><a href="2-mat-met.html#cb12-27"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;ellipsoid&quot;</span>) {</span>
<span id="cb12-28"><a href="2-mat-met.html#cb12-28"></a>          ACIMajor =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-29"><a href="2-mat-met.html#cb12-29"></a>            ACIMinor =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-30"><a href="2-mat-met.html#cb12-30"></a>            Dap =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-31"><a href="2-mat-met.html#cb12-31"></a>        }</span>
<span id="cb12-32"><a href="2-mat-met.html#cb12-32"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;bounding&quot;</span>) {</span>
<span id="cb12-33"><a href="2-mat-met.html#cb12-33"></a>          apbw =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-34"><a href="2-mat-met.html#cb12-34"></a>            apbh =<span class="st"> </span><span class="kw">newArray</span>(nb);</span>
<span id="cb12-35"><a href="2-mat-met.html#cb12-35"></a>        }</span>
<span id="cb12-36"><a href="2-mat-met.html#cb12-36"></a>        <span class="kw">Ext.Manager3D_MultiSelect</span>();</span>
<span id="cb12-37"><a href="2-mat-met.html#cb12-37"></a>        <span class="cf">for</span>(<span class="dt">k =</span> <span class="dv">0</span>; k <span class="op">&lt;</span><span class="st"> </span>nb; k<span class="op">++</span>) {</span>
<span id="cb12-38"><a href="2-mat-met.html#cb12-38"></a>            <span class="cf">if</span> (k <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb12-39"><a href="2-mat-met.html#cb12-39"></a>              <span class="kw">Ext.Manager3D_AddImage</span>();</span>
<span id="cb12-40"><a href="2-mat-met.html#cb12-40"></a>            }</span>
<span id="cb12-41"><a href="2-mat-met.html#cb12-41"></a>        <span class="kw">Ext.Manager3D_GetName</span>(k, obj);</span>
<span id="cb12-42"><a href="2-mat-met.html#cb12-42"></a>        <span class="kw">Ext.Manager3D_Centroid3D</span>(k, cx, cy, cz);</span>
<span id="cb12-43"><a href="2-mat-met.html#cb12-43"></a>    <span class="co"># if dynamic ACI</span></span>
<span id="cb12-44"><a href="2-mat-met.html#cb12-44"></a>        <span class="cf">if</span> (symAC) {</span>
<span id="cb12-45"><a href="2-mat-met.html#cb12-45"></a>          <span class="kw">Ext.Manager3D_Measure3D</span>(k, <span class="st">&quot;Feret&quot;</span>, ferr); </span>
<span id="cb12-46"><a href="2-mat-met.html#cb12-46"></a>            da =<span class="st"> </span>ferr <span class="op">*</span><span class="st"> </span>cellum;</span>
<span id="cb12-47"><a href="2-mat-met.html#cb12-47"></a>            da =<span class="st"> </span><span class="kw">round</span>(da);</span>
<span id="cb12-48"><a href="2-mat-met.html#cb12-48"></a>          <span class="cf">if</span> (da <span class="op">==</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb12-49"><a href="2-mat-met.html#cb12-49"></a>              da =<span class="st"> </span><span class="dv">1</span>;</span>
<span id="cb12-50"><a href="2-mat-met.html#cb12-50"></a>          }</span>
<span id="cb12-51"><a href="2-mat-met.html#cb12-51"></a>        Dap[k] =<span class="st"> </span>da;</span>
<span id="cb12-52"><a href="2-mat-met.html#cb12-52"></a>        }</span>
<span id="cb12-53"><a href="2-mat-met.html#cb12-53"></a>        <span class="kw">toString</span>(obj);</span>
<span id="cb12-54"><a href="2-mat-met.html#cb12-54"></a>        objlabelArray[k] =<span class="st"> </span>obj;</span>
<span id="cb12-55"><a href="2-mat-met.html#cb12-55"></a>    <span class="co"># erase all objects except k</span></span>
<span id="cb12-56"><a href="2-mat-met.html#cb12-56"></a>        <span class="kw">Ext.Manager3D_SelectAll</span>();</span>
<span id="cb12-57"><a href="2-mat-met.html#cb12-57"></a>        <span class="kw">Ext.Manager3D_Select</span>(k);</span>
<span id="cb12-58"><a href="2-mat-met.html#cb12-58"></a>        <span class="kw">Ext.Manager3D_Erase</span>();</span>
<span id="cb12-59"><a href="2-mat-met.html#cb12-59"></a>        <span class="kw">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>, <span class="st">&quot;saturated=0.3 equalize process_all&quot;</span>);</span>
<span id="cb12-60"><a href="2-mat-met.html#cb12-60"></a>        <span class="kw">run</span>(<span class="st">&quot;8-bit&quot;</span>);</span>
<span id="cb12-61"><a href="2-mat-met.html#cb12-61"></a>        <span class="kw">run</span>(<span class="st">&quot;Crop Label&quot;</span>, <span class="st">&quot;label=255 border=5&quot;</span>);</span>
<span id="cb12-62"><a href="2-mat-met.html#cb12-62"></a>    <span class="co"># clear cell stack in Z</span></span>
<span id="cb12-63"><a href="2-mat-met.html#cb12-63"></a>        <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb12-64"><a href="2-mat-met.html#cb12-64"></a>        var done =<span class="st"> </span>false; </span>
<span id="cb12-65"><a href="2-mat-met.html#cb12-65"></a>        <span class="cf">for</span>(<span class="dt">l =</span> <span class="dv">1</span>; l <span class="op">&lt;</span><span class="st"> </span>slices <span class="op">&amp;&amp;!</span>done; l<span class="op">++</span>) {</span>
<span id="cb12-66"><a href="2-mat-met.html#cb12-66"></a>          <span class="kw">setSlice</span>(l);</span>
<span id="cb12-67"><a href="2-mat-met.html#cb12-67"></a>            <span class="kw">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb12-68"><a href="2-mat-met.html#cb12-68"></a>            <span class="cf">if</span>(max <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) { <span class="op">/</span><span class="er">/</span><span class="st"> </span>from apical</span>
<span id="cb12-69"><a href="2-mat-met.html#cb12-69"></a>            smax =<span class="st"> </span>l<span class="dv">-1</span>;</span>
<span id="cb12-70"><a href="2-mat-met.html#cb12-70"></a>            <span class="kw">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first=1 last=&quot;</span><span class="op">+</span>smax<span class="op">+</span><span class="st">&quot; increment=1&quot;</span>);</span>
<span id="cb12-71"><a href="2-mat-met.html#cb12-71"></a>            <span class="kw">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb12-72"><a href="2-mat-met.html#cb12-72"></a>            <span class="kw">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb12-73"><a href="2-mat-met.html#cb12-73"></a>            <span class="cf">for</span>(<span class="dt">l =</span> <span class="dv">1</span>; l <span class="op">&lt;</span><span class="st"> </span>slices <span class="op">&amp;&amp;!</span>done; l<span class="op">++</span>) { <span class="op">/</span><span class="er">/</span><span class="st"> </span>from basal</span>
<span id="cb12-74"><a href="2-mat-met.html#cb12-74"></a>                  <span class="kw">setSlice</span>(l);</span>
<span id="cb12-75"><a href="2-mat-met.html#cb12-75"></a>                    <span class="kw">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb12-76"><a href="2-mat-met.html#cb12-76"></a>                    <span class="cf">if</span>(max <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb12-77"><a href="2-mat-met.html#cb12-77"></a>                      smax =<span class="st"> </span>l<span class="dv">-1</span>;</span>
<span id="cb12-78"><a href="2-mat-met.html#cb12-78"></a>                        <span class="kw">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="op">+</span><span class="st"> </span>smax <span class="op">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb12-79"><a href="2-mat-met.html#cb12-79"></a>                        <span class="kw">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb12-80"><a href="2-mat-met.html#cb12-80"></a>                        done =<span class="st"> </span>true;</span>
<span id="cb12-81"><a href="2-mat-met.html#cb12-81"></a>                    }</span>
<span id="cb12-82"><a href="2-mat-met.html#cb12-82"></a>                }</span>
<span id="cb12-83"><a href="2-mat-met.html#cb12-83"></a>            }</span>
<span id="cb12-84"><a href="2-mat-met.html#cb12-84"></a>        }</span>
<span id="cb12-85"><a href="2-mat-met.html#cb12-85"></a>        naci =<span class="st"> </span><span class="kw">nSlices</span>();</span>
<span id="cb12-86"><a href="2-mat-met.html#cb12-86"></a>        nacimax =<span class="st"> </span>naci<span class="op">/</span><span class="dv">2</span>;</span>
<span id="cb12-87"><a href="2-mat-met.html#cb12-87"></a>        <span class="kw">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = &quot;</span><span class="op">+</span><span class="st"> </span>naci <span class="op">+</span><span class="st">&quot; frames = 1 unit = microns pixel_width = &quot;</span><span class="op">+</span><span class="st"> </span>sizeX <span class="op">+</span><span class="st">&quot; pixel_height = &quot;</span><span class="op">+</span><span class="st"> </span>sizeY <span class="op">+</span><span class="st">&quot; voxel_depth = &quot;</span><span class="op">+</span><span class="st"> </span>sizeZ);</span>
<span id="cb12-88"><a href="2-mat-met.html#cb12-88"></a>        <span class="cf">if</span> (symAC) {</span>
<span id="cb12-89"><a href="2-mat-met.html#cb12-89"></a>            aciza =<span class="st"> </span>da <span class="op">/</span><span class="st"> </span>sizeZ;</span>
<span id="cb12-90"><a href="2-mat-met.html#cb12-90"></a>            db =<span class="st"> </span>naci <span class="op">-</span><span class="st"> </span>da;</span>
<span id="cb12-91"><a href="2-mat-met.html#cb12-91"></a>        }</span>
<span id="cb12-92"><a href="2-mat-met.html#cb12-92"></a>    <span class="co"># if fixed ACI</span></span>
<span id="cb12-93"><a href="2-mat-met.html#cb12-93"></a>        <span class="cf">if</span> (aciza <span class="op">&lt;</span><span class="st"> </span>(nacimax)) {</span>
<span id="cb12-94"><a href="2-mat-met.html#cb12-94"></a>          acizb =<span class="st"> </span>naci <span class="op">-</span><span class="st"> </span>aciza;</span>
<span id="cb12-95"><a href="2-mat-met.html#cb12-95"></a>    <span class="co"># measure apical</span></span>
<span id="cb12-96"><a href="2-mat-met.html#cb12-96"></a>        <span class="kw">run</span>(<span class="st">&quot;Make Binary&quot;</span>, <span class="st">&quot;method = Default background=Default calculate black&quot;</span>);</span>
<span id="cb12-97"><a href="2-mat-met.html#cb12-97"></a>        <span class="kw">setSlice</span>(aciza);</span>
<span id="cb12-98"><a href="2-mat-met.html#cb12-98"></a>        <span class="kw">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, <span class="st">&quot;area centroid bounding fit feret&#39;s redirect = None decimal = 2&quot;</span>);</span>
<span id="cb12-99"><a href="2-mat-met.html#cb12-99"></a>        <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;display slice&quot;</span>);</span>
<span id="cb12-100"><a href="2-mat-met.html#cb12-100"></a>    <span class="co"># get angle</span></span>
<span id="cb12-101"><a href="2-mat-met.html#cb12-101"></a>        MajorAngle[k] =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;FeretAngle&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb12-102"><a href="2-mat-met.html#cb12-102"></a>        <span class="cf">if</span> (MajorAngle[k] <span class="op">&gt;</span><span class="st"> </span><span class="dv">90</span>) {</span>
<span id="cb12-103"><a href="2-mat-met.html#cb12-103"></a>          MajorAngle[k] =<span class="st"> </span><span class="dv">90</span> <span class="op">-</span><span class="st"> </span>(MajorAngle[k] <span class="op">-</span><span class="st"> </span><span class="dv">90</span>);</span>
<span id="cb12-104"><a href="2-mat-met.html#cb12-104"></a>        }</span>
<span id="cb12-105"><a href="2-mat-met.html#cb12-105"></a>    <span class="co"># get max feret / bounding width / area / major</span></span>
<span id="cb12-106"><a href="2-mat-met.html#cb12-106"></a>        resultsArray =<span class="st"> </span><span class="kw">newArray</span>(<span class="kw">nResults</span>());</span>
<span id="cb12-107"><a href="2-mat-met.html#cb12-107"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;ellipsoid&quot;</span>) {</span>
<span id="cb12-108"><a href="2-mat-met.html#cb12-108"></a>            <span class="cf">for</span>(<span class="dt">p =</span> <span class="dv">0</span>; p <span class="op">&lt;</span><span class="st"> </span><span class="kw">nResults</span>(); p<span class="op">++</span>) { </span>
<span id="cb12-109"><a href="2-mat-met.html#cb12-109"></a>                resultsArray[p] =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Major&quot;</span>, p);    </span>
<span id="cb12-110"><a href="2-mat-met.html#cb12-110"></a>        }</span>
<span id="cb12-111"><a href="2-mat-met.html#cb12-111"></a>        total =<span class="st"> </span><span class="dv">0</span>; </span>
<span id="cb12-112"><a href="2-mat-met.html#cb12-112"></a>            <span class="cf">for</span>(<span class="dt">p =</span> <span class="dv">0</span>; p <span class="op">&lt;</span><span class="st"> </span><span class="kw">nResults</span>(); p<span class="op">++</span>) { </span>
<span id="cb12-113"><a href="2-mat-met.html#cb12-113"></a>                total =<span class="st"> </span>total <span class="op">+</span><span class="st"> </span>resultsArray[p]; </span>
<span id="cb12-114"><a href="2-mat-met.html#cb12-114"></a>            }</span>
<span id="cb12-115"><a href="2-mat-met.html#cb12-115"></a>        }</span>
<span id="cb12-116"><a href="2-mat-met.html#cb12-116"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;ellipsoid&quot;</span>) {</span>
<span id="cb12-117"><a href="2-mat-met.html#cb12-117"></a>            ACIMajor[k] =<span class="st"> </span>total;</span>
<span id="cb12-118"><a href="2-mat-met.html#cb12-118"></a>        }</span>
<span id="cb12-119"><a href="2-mat-met.html#cb12-119"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;bounding&quot;</span>) {</span>
<span id="cb12-120"><a href="2-mat-met.html#cb12-120"></a>            apbw[k] =<span class="st"> </span>total;</span>
<span id="cb12-121"><a href="2-mat-met.html#cb12-121"></a>        }</span>
<span id="cb12-122"><a href="2-mat-met.html#cb12-122"></a>    <span class="co"># get min feret / bounding height / minor</span></span>
<span id="cb12-123"><a href="2-mat-met.html#cb12-123"></a>        resultsArray =<span class="st"> </span><span class="kw">newArray</span>(<span class="kw">nResults</span>());</span>
<span id="cb12-124"><a href="2-mat-met.html#cb12-124"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;ellipsoid&quot;</span>) {</span>
<span id="cb12-125"><a href="2-mat-met.html#cb12-125"></a>            <span class="cf">for</span>(<span class="dt">p =</span> <span class="dv">0</span>; p <span class="op">&lt;</span><span class="st"> </span><span class="kw">nResults</span>(); p<span class="op">++</span>) { </span>
<span id="cb12-126"><a href="2-mat-met.html#cb12-126"></a>            resultsArray[p] =<span class="st"> </span><span class="kw">getResult</span>(<span class="st">&quot;Minor&quot;</span>, p); </span>
<span id="cb12-127"><a href="2-mat-met.html#cb12-127"></a>            }</span>
<span id="cb12-128"><a href="2-mat-met.html#cb12-128"></a>            total =<span class="st"> </span><span class="dv">0</span>; </span>
<span id="cb12-129"><a href="2-mat-met.html#cb12-129"></a>            <span class="cf">for</span>(<span class="dt">p =</span> <span class="dv">0</span>; p <span class="op">&lt;</span><span class="st"> </span><span class="kw">nResults</span>(); p<span class="op">++</span>) { </span>
<span id="cb12-130"><a href="2-mat-met.html#cb12-130"></a>                total =<span class="st"> </span>total <span class="op">+</span><span class="st"> </span>resultsArray[p]; </span>
<span id="cb12-131"><a href="2-mat-met.html#cb12-131"></a>            }</span>
<span id="cb12-132"><a href="2-mat-met.html#cb12-132"></a>        }</span>
<span id="cb12-133"><a href="2-mat-met.html#cb12-133"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;ellipsoid&quot;</span>) {</span>
<span id="cb12-134"><a href="2-mat-met.html#cb12-134"></a>            ACIMinor[k] =<span class="st"> </span>total;</span>
<span id="cb12-135"><a href="2-mat-met.html#cb12-135"></a>        }</span>
<span id="cb12-136"><a href="2-mat-met.html#cb12-136"></a>        <span class="cf">if</span> (ACM <span class="op">==</span><span class="st"> &quot;bounding&quot;</span>) {</span>
<span id="cb12-137"><a href="2-mat-met.html#cb12-137"></a>            apbh[k] =<span class="st"> </span>total;</span>
<span id="cb12-138"><a href="2-mat-met.html#cb12-138"></a>        }</span></code></pre></div>
</div>
<div id="single-cell-measurement" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.7</span> Single cell measurement</h6>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="2-mat-met.html#cb13-1"></a>  <span class="kw">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy sync distance_between_centers=10 distance_max_contact=1.80&quot;</span>);</span>
<span id="cb13-2"><a href="2-mat-met.html#cb13-2"></a>  <span class="kw">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb13-3"><a href="2-mat-met.html#cb13-3"></a>  <span class="kw">Ext.Manager3D_AddImage</span>();</span>
<span id="cb13-4"><a href="2-mat-met.html#cb13-4"></a><span class="co"># refresh RoiManager</span></span>
<span id="cb13-5"><a href="2-mat-met.html#cb13-5"></a>  <span class="kw">Ext.Manager3D_DeselectAll</span>();</span>
<span id="cb13-6"><a href="2-mat-met.html#cb13-6"></a>  <span class="kw">Ext.Manager3D_Measure</span>();</span>
<span id="cb13-7"><a href="2-mat-met.html#cb13-7"></a>  <span class="kw">Ext.Manager3D_SaveResult</span>(<span class="st">&quot;M&quot;</span>, datcelldir <span class="op">+</span><span class="st"> </span>name <span class="op">+</span><span class="st"> &quot;.csv&quot;</span>);</span>
<span id="cb13-8"><a href="2-mat-met.html#cb13-8"></a>  <span class="kw">Ext.Manager3D_CloseResult</span>(<span class="st">&quot;M&quot;</span>);</span>
<span id="cb13-9"><a href="2-mat-met.html#cb13-9"></a>  <span class="kw">Ext.Manager3D_Reset</span>();</span>
<span id="cb13-10"><a href="2-mat-met.html#cb13-10"></a>  <span class="kw">Ext.Manager3D_Close</span>();</span></code></pre></div>
</div>
<div id="functions" class="section level6">
<h6><span class="header-section-number">2.2.4.1.3.8</span> Functions</h6>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="2-mat-met.html#cb14-1"></a><span class="co"># sliceclear</span></span>
<span id="cb14-2"><a href="2-mat-met.html#cb14-2"></a>  <span class="cf">function</span> <span class="kw">sliceclear</span>() {</span>
<span id="cb14-3"><a href="2-mat-met.html#cb14-3"></a>        <span class="cf">for</span> (<span class="dt">j =</span> <span class="dv">1</span>; j <span class="op">&lt;=</span><span class="st"> </span>n; j<span class="op">++</span>) {</span>
<span id="cb14-4"><a href="2-mat-met.html#cb14-4"></a>        <span class="kw">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb14-5"><a href="2-mat-met.html#cb14-5"></a>        <span class="kw">setSlice</span>(j);</span>
<span id="cb14-6"><a href="2-mat-met.html#cb14-6"></a>        <span class="kw">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;include add slice&quot;</span>);</span>
<span id="cb14-7"><a href="2-mat-met.html#cb14-7"></a>        <span class="cf">if</span> (<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">==</span><span class="st"> </span><span class="dv">0</span>) {</span>
<span id="cb14-8"><a href="2-mat-met.html#cb14-8"></a>            <span class="kw">setSlice</span>(j);</span>
<span id="cb14-9"><a href="2-mat-met.html#cb14-9"></a>            <span class="kw">makeRectangle</span>(<span class="dv">0</span>, <span class="dv">0</span>, width, height);</span>
<span id="cb14-10"><a href="2-mat-met.html#cb14-10"></a>            <span class="kw">run</span>(<span class="st">&quot;Cut&quot;</span>);</span>
<span id="cb14-11"><a href="2-mat-met.html#cb14-11"></a>        }</span>
<span id="cb14-12"><a href="2-mat-met.html#cb14-12"></a>        <span class="cf">if</span> (<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">==</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb14-13"><a href="2-mat-met.html#cb14-13"></a>            <span class="kw">setSlice</span>(j);</span>
<span id="cb14-14"><a href="2-mat-met.html#cb14-14"></a>            <span class="kw">allroi</span>();</span>
<span id="cb14-15"><a href="2-mat-met.html#cb14-15"></a>            <span class="kw">wait</span>(<span class="dv">200</span>);</span>
<span id="cb14-16"><a href="2-mat-met.html#cb14-16"></a>            <span class="kw">run</span>(<span class="st">&quot;Clear Outside&quot;</span>, <span class="st">&quot;slice&quot;</span>);</span>
<span id="cb14-17"><a href="2-mat-met.html#cb14-17"></a>        }</span>
<span id="cb14-18"><a href="2-mat-met.html#cb14-18"></a>        <span class="cf">if</span> (<span class="kw">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="op">&gt;</span><span class="st"> </span><span class="dv">1</span>) {</span>
<span id="cb14-19"><a href="2-mat-met.html#cb14-19"></a>            <span class="kw">setSlice</span>(j);</span>
<span id="cb14-20"><a href="2-mat-met.html#cb14-20"></a>            <span class="kw">allroi</span>();</span>
<span id="cb14-21"><a href="2-mat-met.html#cb14-21"></a>            <span class="kw">roiManager</span>(<span class="st">&quot;Combine&quot;</span>);</span>
<span id="cb14-22"><a href="2-mat-met.html#cb14-22"></a>            <span class="kw">run</span>(<span class="st">&quot;Clear Outside&quot;</span>, <span class="st">&quot;slice&quot;</span>);</span>
<span id="cb14-23"><a href="2-mat-met.html#cb14-23"></a>        }</span>
<span id="cb14-24"><a href="2-mat-met.html#cb14-24"></a>    }</span>
<span id="cb14-25"><a href="2-mat-met.html#cb14-25"></a>  }</span></code></pre></div>
</div>
</div>
</div>
<div id="CNN" class="section level4">
<h4><span class="header-section-number">2.2.4.2</span> Rosette Detection</h4>
<p>The method used for rosette detection is based on a convolutional neuronal network (CNN) and was modified from the “rosette detector” algorithm previously used in the lab and described in <span class="citation">(<em>44</em>, <em>45</em>)</span>. Since the former method was technically outdated and since we had new data in which we needed to detect rosettes, we updated the former method to a <em>state-of-the-art</em> CNN using Caffe<span class="citation">(<em>54</em>)</span> as a backend. Network configuration and training was done by our collaborators at the institute for Informatics, Albert-Ludwigs-University Freiburg.</p>
<p>The training dataset consitsts of 17 DMSO- and SU5402- treated embryos each. SU5402 is an inhibitor of FGF receptors and embryos treated with these inhibitors show a strongly reduced number of rosettes (<span class="citation">(<em>32</em>)</span>). In order to give the network something to learn, the data had to be labeled manually, which was done in ImageJ by placing multipoint ROIs at the center of the rosettes. The data was then further permutated to artificially increase the amount of training data and make the detection more robust against different kinds of input. Further parameters about the training data is listed in Table <a href="2-mat-met.html#tab:CNNtraining">2.16</a>. One example for each is shown in figure <a href="#fig:CNNtrain"><strong>??</strong></a>.</p>

<p><img src="figures/materials/cnn/CNNtrain.png" width="75%" style="display: block; margin: auto;" /></p>
<p>The main advantages for using a neural network in a task like this are…</p>
<ol style="list-style-type: decimal">
<li>Objectivity
<ul>
<li>A computer model is not biased in a way that it prefers one outcome over the other. It evaluates based on what it was trained to.</li>
<li>Unlike the human brain, once a CNN is trained it is static and does not keep on learning.</li>
</ul></li>
<li>Degree of rosette registration
<ul>
<li>The output data are continous rational numbers (<span class="math inline">\(\mathbb{Q}\)</span>) instead of integers (<span class="math inline">\(\mathbb{Z}\)</span>) which does not only tell if a rosette is there fore not, but also for ‘how much’ (50-100<span class="math inline">\(\%\)</span>) it is there.</li>
</ul></li>
<li>Training is done relatively quick</li>
</ol>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:CNNtraining">Table 2.16: </span>CNN training data
</caption>
<thead>
<tr>
<th style="text-align:center;font-weight: bold;">
group
</th>
<th style="text-align:center;font-weight: bold;">
conc.
</th>
<th style="text-align:center;font-weight: bold;">
nm
</th>
<th style="text-align:center;font-weight: bold;">
intensity….
</th>
<th style="text-align:center;font-weight: bold;">
exposure
</th>
<th style="text-align:center;font-weight: bold;">
z.planes
</th>
<th style="text-align:center;font-weight: bold;">
magnification
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
DMSO
</td>
<td style="text-align:center;width: 1cm; ">
0.1<span class="math inline">\(\%\)</span>
</td>
<td style="text-align:center;width: 2cm; ">
488
</td>
<td style="text-align:center;width: 1.6cm; ">
100
</td>
<td style="text-align:center;width: 1.5cm; ">
100 ms
</td>
<td style="text-align:center;width: 1.4cm; ">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
<tr>
<td style="text-align:center;">
SU5402
</td>
<td style="text-align:center;width: 1cm; ">
10<span class="math inline">\(\mu\)</span>M
</td>
<td style="text-align:center;width: 2cm; ">
488
</td>
<td style="text-align:center;width: 1.6cm; ">
100
</td>
<td style="text-align:center;width: 1.5cm; ">
100 ms
</td>
<td style="text-align:center;width: 1.4cm; ">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
</tbody>
</table>
</div>
<div id="prolif" class="section level4">
<h4><span class="header-section-number">2.2.4.3</span> Proliferation Analysis</h4>
<p>The basic principle is based on work done by Laguerre <em>et al.</em>, 2009<span class="citation">(<em>28</em>)</span>. For registration of mitotic events an IJ manual tracking tool was used was used that allows to track an image feature through a stack of images creating tracks as it progresses through volume / time (‘MTrackJ’<span class="citation">(<em>55</em>)</span>).</p>
<p>For mounting the procedure described in section <a href="2-mat-met.html#mount-met">2.2.3.5</a> was used. Nuclei were visualized in a <em>cxcr4b</em>(BAC):<em>H2BRFP</em> transgenic line. Z-projection of volumetric timelapses, Mitotic events were tracked in each CC and the pLLP. Afterwards the data was exported as one table per embryo and processes by counting mitoses per pLLP / CC / total CC mitoses per embryo.</p>
<p>Figure <a href="#fig:mitodatapoints"><strong>??</strong></a> shows an exemplary track for the data analyzed.</p>

<p><img src="figures/materials/prol/Prolif.png" width="60%" style="display: block; margin: auto;" /></p>
</div>
<div id="ACI" class="section level4">
<h4><span class="header-section-number">2.2.4.4</span> Apical Index</h4>
<div id="rationale" class="section level5">
<h5><span class="header-section-number">2.2.4.4.1</span> Rationale</h5>
<p>The earliest attempt found for indexing AC can be found in a study published by Lee <em>et al</em><span class="citation">(<em>56</em>)</span> where they were interested in the ‘apical index’ (A.I.) of bottle cells during <em>X.laevis</em> gastrulation(figure <a href="#fig:ACLee"><strong>??</strong></a> Lee). Another example for measuring AC is the apical <em>constriction</em> index (ACI, figure <a href="#fig:ACLee"><strong>??</strong></a> Harding) for the cells of the <em>D.rerio</em> lateral line primordium (pLLP), which can be found in a study from 2012 where it was shown that Fgfr-Ras-MAPK signaling is required for AC and Rock2a localization <span class="citation">(<em>57</em>, <em>58</em>)</span>.</p>

<p><img src="figures/materials/models/ai.png" width="75%" style="display: block; margin: auto;" /></p>
<p>In these two publications, the way they measure A.I. <span class="citation">(<em>56</em>)</span> and ACI <span class="citation">(<em>58</em>)</span> respectively, does not differ and is the ratio of lateral height over apical width.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{apical\;width\;[\mu m]}\]</span></p>
<p>We found two principal weaknesses of applying this ratio to the cells of the PLLp. First, it does not respect the independence of lateral height to AC. Second, it does not differentiate between <em>anterio-posterior</em> (AP) and <em>dorso-ventral</em> (DV) tissue polarization. Third, it actually represents the A.I. rather than the apical <em>constriction</em> index.</p>
<div id="ACI-param" class="section level6">
<h6><span class="header-section-number">2.2.4.4.1.1</span> Parameter definition</h6>
<p>To obtain a precise and biologically meaningful quantity for AC, first a couple of definitions had to be made.</p>
<ol style="list-style-type: decimal">
<li>AC is independent of orientation and polarization
<ul>
<li>Cells may be polarized along the AP axis</li>
<li>Cells may also be polarized along the DV axis</li>
</ul></li>
<li>AC is independent of lateral height
<ul>
<li>Lateral height can be described as the distance of the two farthest points on the surface area of a cell</li>
<li>Two cells with different lateral heights can be equally apically constricted</li>
</ul></li>
<li>AC is independent of cell size
<ul>
<li>The size of a cell is its volume</li>
<li>Two cells of different volume can be equally apically constricted</li>
</ul></li>
</ol>
</div>
<div id="ACI-lat" class="section level6">
<h6><span class="header-section-number">2.2.4.4.1.2</span> Adaption for variation in lateral height</h6>
<p>To test different A.I. conditions, an apically constricted cell can be approximated by modeling a tetrahedron. For example, shrinking or enlarging a cell symmetrically should not affect the ACI. As described by Harding(2014)<span class="citation">(<em>58</em>)</span>, the <em>apical width</em> of a cell is measured first by manual 3-D reconstruction, second manual re-orientation, and third by going 1 <span class="math inline">\(\mu\)</span>m from the apical tip into the cell (from now on referred to as <span class="math inline">\(\Delta\)</span>ap, <a href="2-mat-met.html#fig:ACICells">2.9</a>B). Finally, <em>apical width</em> is the total width of the 2-D object in the respective Z-stack plane.</p>
<p>If <span class="math inline">\(\Delta\)</span>ap is a constant, the A.I. in a symmetrically enlarged cell increases from e.g. ~15 to ~23, since <em>apical width</em> stays the same but lateral height increases. On the contrary, if <span class="math inline">\(\Delta\)</span>ap is adjusted relative to a cells lateral height, e.g. by percentage, the A.I. in a symmetrically enlarged cell stays the same (figure <a href="2-mat-met.html#fig:ACICells">2.9</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:ACICells"></span>
<img src="figures/summary/aci_fig-01.png" alt="A.I. model A-A’’ A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta was adjusted relative to the lateral height. B Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2D area as seen at delta ap of 1 or 2.5 \(\mu\)m." width="75%" />
<p class="caption">
Figure 2.9: A.I. model <strong>A-A’’</strong> A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta was adjusted relative to the lateral height. <strong>B</strong> Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2D area as seen at delta ap of 1 or 2.5 <span class="math inline">\(\mu\)</span>m.
</p>
</div>
<p>Therefore the measurement for apical width has to be relative to lateral height.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{relative\;apical\;width\;[\mu m]}\]</span></p>
</div>
<div id="ACI-pol" class="section level6">
<h6><span class="header-section-number">2.2.4.4.1.3</span> Adaption for tissue polarization</h6>
<p>Organs develop in a 3-D space and are polarized along each axis. AC usually describes a 2-D morphogenetic movement towards a center along the X-Y axes. However, the contraction movements along X and Y might be independent of one another. This could mean that they happen at different speeds, or that one is absent. As a result, the tissue would look less radially (figure <a href="2-mat-met.html#fig:cellpol">2.10</a>B) constricted, but more constricted along one particular axis (anisotropic). In order to separate those two AC dimensions, the A.I. can be calculated for the <em>anterio-posterior</em> and for the <em>dorso-ventral</em> axis (figure <a href="2-mat-met.html#fig:cellpol">2.10</a>, horizontal vs. vertical).</p>

<div class="figure" style="text-align: center"><span id="fig:cellpol"></span>
<img src="figures/materials/models/ACI_Cells_pol.png" alt="Schematic AC along the A-P and D-V axis. A shows a A-P and D-V constricted cluster of cells. B shows a D-V constricted cluster of cells." width="75%" />
<p class="caption">
Figure 2.10: Schematic AC along the A-P and D-V axis. <strong>A</strong> shows a A-P and D-V constricted cluster of cells. <strong>B</strong> shows a D-V constricted cluster of cells.
</p>
</div>
<p>By fitting an ellipsoid to the volume area taken at <span class="math inline">\(\mathrm{\Delta}\)</span>ap, one will obtain the following parameters (figure <a href="#fig:ellipse"><strong>??</strong></a>).</p>
<ol style="list-style-type: decimal">
<li>Length of Major axis
<ul>
<li>constituting <em>apical width</em></li>
</ul></li>
<li>Length of Minor axis
<ul>
<li>constituting <em>apical height</em></li>
</ul></li>
<li>Angle of Major from 0<span class="math inline">\(^\circ\)</span>
<ul>
<li>indicative for orientation of <em>lateral height</em>: If the angle is close to 0<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the AP axis (the cell is constricted along the DV axis). If the angle is close to 90<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the DV axis (the cell is constricted along the AP axis).</li>
</ul></li>
</ol>
<p>(ref:ellipse) Scheme of ellipsoid measures. <strong>A</strong> shows the major axis as apical width and the minor axis as apical height. <strong>B</strong> shows the angular displacement from the horizon in steps of 30<span class="math inline">\(^\circ\)</span>.
<img src="figures/materials/models/ellipse.png" width="75%" style="display: block; margin: auto;" /></p>
<p>The two dimensions of AC indices can therefore be described as the following ratios…</p>
<p><span class="math display">\[\mathbf{A.I._{Major}} = \frac{lateral\;height\;[\mu m]}{Major\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{A.I._{Minor}} = \frac{lateral\;height\;[\mu m]}{Minor\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{Angle_{Major}} = \measuredangle = \mathrm{\Delta}\;from\;horizont\;[0-90^\circ]\]</span></p>
</div>
</div>
<div id="ACI-Dis" class="section level5">
<h5><span class="header-section-number">2.2.4.4.2</span> Measurements</h5>
<div id="ACI-singlecell" class="section level6">
<h6><span class="header-section-number">2.2.4.4.2.1</span> Single Cell measurements</h6>
<p>Each geometric object has a centroid coordinate in X and Y (and Z) which is represented as the mean of all X or Y coordinates within the object. Here, the centroid coordinates in X and Y are used to plot the cells as points in the X-Y plane. Additionally, each point is colored for the A.I. value (high values are red - dark red, middle values are green, low values are cyan - blue).</p>
(ref:ACIMinor) A.C.I.~Major / Minor~ single cell measurements
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-4"></span>
<img src="thesis_files/figure-html/unnamed-chunk-4-1.png" alt="(ref:ACIMinor)" width="90%" />
<p class="caption">
Figure 2.11: (ref:ACIMinor)
</p>
</div>
<p>Harding(2013)<span class="citation">(<em>58</em>)</span> were using a constant <span class="math inline">\(\mathrm{\Delta}\)</span>ap to measure the apical width, which we have shown to be incorrect in certain cases. In their study they found that certain mean A.I. values in the DMSO go as high as 15, which might be related to this <a href="2-mat-met.html#fig:ACICells">2.9</a>.
By measuring apical width at a relative <span class="math inline">\(\mathrm{\Delta}\)</span>ap and taking into account all pLLP cells these two exemplary pLLPs show a mean difference in the Major of 1.11 and 0.53 in the Minor (see table @ref(tab:aci_summary)).</p>
<p>(ref:HardingACI) AI indices by Harding et al. <strong>E-G’</strong> 3-D reconstructions of the highlighted cell. <strong>H</strong> A.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of <em>hsp70:dn-Ras</em>. (n = 180 cells / N = 6 embryos).
<img src="figures/materials/models/harding.png" width="60%" style="display: block; margin: auto;" /></p>
</div>
<div id="ACI-Angledens" class="section level6">
<h6><span class="header-section-number">2.2.4.4.2.2</span> Angle densities</h6>
<p>To check whether there is a bias in orientation of the apical width, the angle measurements <a href="#fig:ellipse"><strong>??</strong></a> can be shown as a function of density along X. Here, a binwidth of 2.5 and a grouping factor to color the two conditions ‘DMSO’ and ‘SU5402’ was used.</p>
<p>Interestingly the results indicate that there is less of a difference for the Major<sub>Angle</sub> at angles bigger than 15-20<span class="math inline">\(^\circ\)</span>. This would mean that the <em>apical width</em> in SU5402 treated embryos is more strongly oriented along the horizontal <em>antero-posterior</em> axis.</p>
</div>
<div id="ACI-mag" class="section level6">
<h6><span class="header-section-number">2.2.4.4.2.3</span> ACI magnitude at different Angles</h6>
<p>Now, to get an idea of the magnitude ACI<sub>Major</sub> and ACI<sub>Minor</sub> have at each angle, each can be shown as a function of the Major<sub>Angle</sub> over ACI (while Major<sub>Angle</sub> is the same for both ACIs). Again, the grouping factor ‘GT’ was used to color the two conditions ‘DMSO’ and ‘SU5402’.</p>
<p>Since AC is a 3-D morphogenetic process and since cells in a wild type PLLp are mostly radially organized, it does make sense to try to look at AC from more than just one perspective. Here we propose to separate the ACI into an <em>antero-posterior</em> and a <em>dorso-ventral</em> dimension.</p>
<ol style="list-style-type: decimal">
<li><p>Interestingly there does not seem to be much of a difference in ACI<sub>Minor</sub>, which can also be shown by the mean values which are at 2.6 for the DMSO control and at 2.1 for the SU5402 treated condition. This indicates that there is no significant difference in contraction along the <em>dorso-ventral</em> axis.</p></li>
<li><p>For the ACI<sub>Major</sub> the base constriction for both, DMSO and SU5402 is at around 3.6, however there is a peak at around 40-60° in the DMSO control where cells are most constricted having a maximum ACI at 15.8. This indicates that cells coming from 35.7° are most constricted in the <em>anterior-posterior</em> axis.</p></li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:angletoACI"></span>
<img src="thesis_files/figure-html/angletoACI-1.png" alt="ACIMajor / ACIMinor over MajorAngle" width="49%" /><img src="thesis_files/figure-html/angletoACI-2.png" alt="ACIMajor / ACIMinor over MajorAngle" width="49%" />
<p class="caption">
Figure 2.12: ACI<sub>Major</sub> / ACI<sub>Minor</sub> over Major<sub>Angle</sub>
</p>
</div>
</div>
</div>
</div>
</div>
</div>
<div id="mat-GrTrDat" class="section level2">
<h2><span class="header-section-number">2.3</span> Ground Truth</h2>
<p>Analyzing images and extracting quantitative measurements can be a tedious task, especially when the analysis becomes more complex. Fortunately, there are ways to automate image analysis by using either machine learning approaches or by tailoring hand-crafted algorithms in an image analysis software tool like e.g. ImageJ<span class="citation">(<em>59</em>)</span>.
The main advantages of doing so are to…</p>
<ul>
<li>be independent of confirmation bias</li>
<li>make the analysis more robust against oversight</li>
<li>increase the statistical power by increasing the number of data points<span class="citation">(<em>60</em>)</span></li>
<li>increase effect size by increasing the measurement accuracy<span class="citation">(<em>60</em>)</span></li>
</ul>
<p>However, to ensure the measurements taken by a tailored or trained algorithm are meaningful, they must be compared to a <em>ground truth</em> dataset which again describes a general measure of algorithmic quality performance<span class="citation">(<em>61</em>)</span>.</p>
<div id="cluster-analysis" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Cluster Analysis</h3>
<p>The <em>anaLLzR2D</em> algorithm was designed for semi-automatic cell cluster detection in the <em>cldnb:lyn-gfp</em> transgenic line and optional nuclei counting in a second DAPI labeled channel within the Regions of Interest (ROIs) derived from the cell cluster detection.</p>
<div id="design" class="section level4">
<h4><span class="header-section-number">2.3.1.1</span> Design</h4>
<p>To assess the quality of the <em>anaLLzR2D</em> algorithm for nuclei detection the ground truth was designed as follows.</p>
<div id="model" class="section level5">
<h5><span class="header-section-number">2.3.1.1.1</span> Model</h5>
<ul>
<li>each Cell Cluster (CC) consists of a number of objects (cells)</li>
<li>each object is part of the respective CC and defines one cell entity</li>
<li>each object is determined via a fluorescent nucleus label</li>
<li>embryos are mounted within a 3D mold (section <a href="2-mat-met.html#mount-met">2.2.3.5</a>) to reduce noise</li>
</ul>
<table style="width:97%;">
<caption><span id="tab:imgcond3DGrT">Table 2.17: </span> Cluster Analysis Model</caption>
<colgroup>
<col width="48%" />
<col width="48%" />
</colgroup>
<tbody>
<tr class="odd">
<td align="right">XY scale</td>
<td align="left">0.32 px/<span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="even">
<td align="right">Z-spacing</td>
<td align="left">4 <span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="odd">
<td align="right">Camera</td>
<td align="left">Rolera</td>
</tr>
</tbody>
</table>
</div>
<div id="training-test-data" class="section level5">
<h5><span class="header-section-number">2.3.1.1.2</span> Training &amp; test data</h5>
<p>The training set consists of three randomly picked wild type pLLPs. For each the algorithm was run with standard parameters. Cell cluster ROIs and nuclei multi-point labels were edited manually.</p>
<p>To test the algorithm, it is run at different nuclei detection thresholds on the same image data.</p>
</div>
</div>
</div>
<div id="morphometric-analysis" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Morphometric analysis</h3>
<p>The <em>anallzr3D</em> algorithm was designed for fully automated, single cell volume segmentation in the <em>cldnb:lyn-gfp</em> transgenic line. In addition to 3D cellular metrics, it offers Apical Constriction measurement of each cell.</p>
<div id="design-1" class="section level4">
<h4><span class="header-section-number">2.3.2.1</span> Design</h4>
<p>To assess the quality of the anaLLzr3D algorithm the ground truth was modeled as follows.</p>
<div id="model-1" class="section level5">
<h5><span class="header-section-number">2.3.2.1.1</span> Model</h5>
<ul>
<li>each PLLp consists of a number of objects (cells)</li>
<li>each object is part of the PLLp and defines one cell entity</li>
<li>cell boundaries are determined via the transgene <em>cldnb:lyn-gfp</em> +/+</li>
<li>embryos are imaged live to conserve signal and membrane integrity</li>
<li>embryos are mounted within a 3D mold for improved 3D alignment</li>
</ul>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:model3DGrT">Table 2.18: </span>anaLLzr3D Model
</caption>
<tbody>
<tr>
<td style="text-align:right;width: 4cm; ">
Exposure time
</td>
<td style="text-align:left;">
100 ms
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Laser intensity
</td>
<td style="text-align:left;">
100<span class="math inline">\(\%\)</span> / 9.3 mW
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Objective
</td>
<td style="text-align:left;">
40X; CFI APO LWD WI; N.A. = 1.15, W.D. = 0.60 mm
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Z-spacing
</td>
<td style="text-align:left;">
0.4 - 0.5 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Camera
</td>
<td style="text-align:left;">
sCMOS; 4.2 M.Pix; 82<span class="math inline">\(\%\)</span> Q.E.
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
SD system
</td>
<td style="text-align:left;">
Yokogawa CSU - W1; 50 <span class="math inline">\(\mu\)</span>m pattern
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Piezo
</td>
<td style="text-align:left;">
Piezo Z-table; 300 <span class="math inline">\(\mu\)</span>m scan range
</td>
</tr>
</tbody>
</table>
</div>
<div id="training-test-data-1" class="section level5">
<h5><span class="header-section-number">2.3.2.1.2</span> Training &amp; test data</h5>
<p>The training set consists of three randomly picked wildtype PLLps. For each the algorithm is run with no filters (X, Y, Z border objects, size) and a minimum segmentation threshold. Afterwards the segmentation result is corrected manually for over- and under-segmentation and objects that are not part of the PLLp.</p>
<p>To test the algorithm it is run at different segmentation thresholds on the same image data.</p>
</div>
</div>
</div>
</div>
<div id="mat-datasets" class="section level2">
<h2><span class="header-section-number">2.4</span> Image Data Sets</h2>
<p>Summaries of Image datasets</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatcc">Table 2.19: </span>Cell Cluster dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
60 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="2">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Fixation
</td>
<td style="text-align:left;">
4<span class="math inline">\(\%\)</span> PFA o.N.
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
1<span class="math inline">\(\%\)</span> LMPA
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
25X WI
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 12*X large image
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatai">Table 2.20: </span>AI dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.5<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
stamp v4A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
40X objective
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 1x1; Gain 1; Exposure 100 ms
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (100<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
0.4 <span class="math inline">\(\mu\)</span>m Z-spacing
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatprol">Table 2.21: </span>Proliferation dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; cxcr4b(BAC):H2BRFP +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v4A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="6">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 35 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
651 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 7 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatdet">Table 2.22: </span>Detection dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v5A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="6">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 20 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
3 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 3*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 10 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatato">Table 2.23: </span>Atoh1a dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; atoh1a:Tom +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v5A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="7">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (Int: 20<span class="math inline">\(\%\)</span>; Exp: 25 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
561 nm (Int: 30<span class="math inline">\(\%\)</span>; Exp: 50 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 20 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="footnotes">
<hr />
<ol start="15">
<li id="fn15"><p>e.g. photomultipliers or charge-coupled devices<a href="2-mat-met.html#fnref15" class="footnote-back">↩︎</a></p></li>
<li id="fn16"><p>the process of embedding the samples in agarose<a href="2-mat-met.html#fnref16" class="footnote-back">↩︎</a></p></li>
<li id="fn17"><p>Where B stands for a batch, that is if multiple dishes were imaged and P stands for the position within a single batch<a href="2-mat-met.html#fnref17" class="footnote-back">↩︎</a></p></li>
<li id="fn18"><p>“movement to make scientific research […] and its dissemination accessible to all levels of an inquiring society” – Wikipedia/en/Open_science<a href="2-mat-met.html#fnref18" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-Intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-res.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
