<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-Intro.html"/>
<link rel="next" href="3-res.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Index</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#print-version"><i class="fa fa-check"></i>Print version</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#electronic-version"><i class="fa fa-check"></i>Electronic version</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="list-of-publications.html"><a href="list-of-publications.html"><i class="fa fa-check"></i>List of publications</a></li>
<li class="chapter" data-level="" data-path="list-of-abbreviations.html"><a href="list-of-abbreviations.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#morphogenesis"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#objective-and-open-questions"><i class="fa fa-check"></i><b>1.3</b> Objective and Open Questions</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-Intro.html"><a href="1-Intro.html#motivation"><i class="fa fa-check"></i><b>1.3.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-Intro.html"><a href="1-Intro.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>1.4</b> Data Strategy and Analysis</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-Intro.html"><a href="1-Intro.html#sample-preparation"><i class="fa fa-check"></i><b>1.4.1</b> Sample Preparation</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-Intro.html"><a href="1-Intro.html#imaging"><i class="fa fa-check"></i><b>1.4.2</b> Imaging</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-Intro.html"><a href="1-Intro.html#data-handling"><i class="fa fa-check"></i><b>1.4.3</b> Data handling</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-Intro.html"><a href="1-Intro.html#image-and-data-analysis"><i class="fa fa-check"></i><b>1.4.4</b> Image and Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probe list</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.9</b> Hardware</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.10</b> Software</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.1.11</b> Ground Truth Data</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.1</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.2</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.3</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.3</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#methods"><i class="fa fa-check"></i><b>3.1</b> Methods</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#mounting"><i class="fa fa-check"></i><b>3.1.1</b> Mounting</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#aci-measurement"><i class="fa fa-check"></i><b>3.1.2</b> ACI Measurement</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.1</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.2</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.3</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#apical-constriction-1"><i class="fa fa-check"></i><b>3.2.4</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.5</b> Haircell Specification</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#rescue-experiments"><i class="fa fa-check"></i><b>3.2.6</b> Rescue Experiments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#methods-1"><i class="fa fa-check"></i><b>4.1</b> Methods</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#mounting-1"><i class="fa fa-check"></i><b>4.1.1</b> Mounting</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#adult-phenotype"><i class="fa fa-check"></i><b>4.2.1</b> Adult Phenotype</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#lateral-line"><i class="fa fa-check"></i><b>4.2.2</b> Lateral Line</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.2.3</b> Rosettes</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#hair-cells"><i class="fa fa-check"></i><b>4.2.4</b> Hair Cells</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-2"><i class="fa fa-check"></i><b>4.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.2.6</b> Model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#methods-2"><i class="fa fa-check"></i><b>5.1</b> Methods</a></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#shroom3-2"><i class="fa fa-check"></i><b>5.2</b> Shroom3</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.2.1</b> Proliferation</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection"><i class="fa fa-check"></i><b>5.2.2</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification-1"><i class="fa fa-check"></i><b>5.2.3</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.2.4</b> Single cell reconstruction and AC measurement</a></li>
</ul></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mat-met" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="mat" class="section level2">
<h2><span class="header-section-number">2.1</span> Materials</h2>
<div id="mat-chem" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Chemicals</h3>
<table>
<caption><span id="tab:mat-chem">Table 2.1: </span> Chemicals</caption>
<colgroup>
<col width="28%" />
<col width="54%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Chemical</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Agarose</td>
<td align="center">Roth</td>
<td align="left">6351.2</td>
</tr>
<tr class="even">
<td align="right">Agar-Agar</td>
<td align="center">Roth</td>
<td align="left">5210.3</td>
</tr>
<tr class="odd">
<td align="right">Ampicillin</td>
<td align="center">Roth</td>
<td align="left">K029.2</td>
</tr>
<tr class="even">
<td align="right">ATP</td>
<td align="center">Epicentre</td>
<td align="left">E311K</td>
</tr>
<tr class="odd">
<td align="right">Blocking Reagent</td>
<td align="center">Roche</td>
<td align="left">11096176001</td>
</tr>
<tr class="even">
<td align="right">BCIP</td>
<td align="center">Fermentas</td>
<td align="left">R0822</td>
</tr>
<tr class="odd">
<td align="right">CaCl</td>
<td align="center">Roth</td>
<td align="left">886.1</td>
</tr>
<tr class="even">
<td align="right">Calyculin</td>
<td align="center">Sigma</td>
<td align="left">208851</td>
</tr>
<tr class="odd">
<td align="right">DAPI</td>
<td align="center">Sigma</td>
<td align="left">D9542</td>
</tr>
<tr class="even">
<td align="right">DIG RNA Mix</td>
<td align="center">Roche</td>
<td align="left">11277073910</td>
</tr>
<tr class="odd">
<td align="right">DMSO</td>
<td align="center">Roth</td>
<td align="left">4720.2</td>
</tr>
<tr class="even">
<td align="right">EtBr</td>
<td align="center">Roth</td>
<td align="left">2218.3</td>
</tr>
<tr class="odd">
<td align="right">EtOH</td>
<td align="center">Roth</td>
<td align="left">9065.3</td>
</tr>
<tr class="even">
<td align="right">Formaldehyde</td>
<td align="center">Roth</td>
<td align="left">7398.1</td>
</tr>
<tr class="odd">
<td align="right">Formamide</td>
<td align="center">Roth</td>
<td align="left">P040.1</td>
</tr>
<tr class="even">
<td align="right">Glycerol</td>
<td align="center">Roth</td>
<td align="left">3783.2</td>
</tr>
<tr class="odd">
<td align="right">IPTG</td>
<td align="center">Thermo</td>
<td align="left">R1171</td>
</tr>
<tr class="even">
<td align="right">KCl</td>
<td align="center">Roth</td>
<td align="left">P017.1</td>
</tr>
<tr class="odd">
<td align="right">Low melting point Agarose</td>
<td align="center">Roth</td>
<td align="left">A9539</td>
</tr>
<tr class="even">
<td align="right">Maleic Acid</td>
<td align="center">Roth</td>
<td align="left">3810.3</td>
</tr>
<tr class="odd">
<td align="right">MeOH</td>
<td align="center">Roth</td>
<td align="left">CP43.3</td>
</tr>
<tr class="even">
<td align="right">MgCl</td>
<td align="center">Roth</td>
<td align="left">2189.1</td>
</tr>
<tr class="odd">
<td align="right">NaCl</td>
<td align="center">Roth</td>
<td align="left">9265.2</td>
</tr>
<tr class="even">
<td align="right">NaHCO</td>
<td align="center">Roth</td>
<td align="left">855.1</td>
</tr>
<tr class="odd">
<td align="right">NaOH</td>
<td align="center">Roth</td>
<td align="left">6771.3</td>
</tr>
<tr class="even">
<td align="right">NGS</td>
<td align="center">Sigma</td>
<td align="left">C6767</td>
</tr>
<tr class="odd">
<td align="right"><em>p</em>-Formaldehyd</td>
<td align="center">Sigma</td>
<td align="left">P6148</td>
</tr>
<tr class="even">
<td align="right">Phenol Red</td>
<td align="center">Sigma</td>
<td align="left">P0290</td>
</tr>
<tr class="odd">
<td align="right">Propan-2-ol</td>
<td align="center">VWR</td>
<td align="left">20842330</td>
</tr>
<tr class="even">
<td align="right">Proteinase K</td>
<td align="center">Roth</td>
<td align="left">7528.4</td>
</tr>
<tr class="odd">
<td align="right">PTU</td>
<td align="center">Sigma</td>
<td align="left">P7629</td>
</tr>
<tr class="even">
<td align="right">Rockout</td>
<td align="center">Sigma</td>
<td align="left">555553</td>
</tr>
<tr class="odd">
<td align="right">SSC</td>
<td align="center">Roth</td>
<td align="left">1232.1</td>
</tr>
<tr class="even">
<td align="right">SU5402</td>
<td align="center">CALBIOCHEM</td>
<td align="left">572630</td>
</tr>
<tr class="odd">
<td align="right">Torula RNA</td>
<td align="center">Sigma</td>
<td align="left">R6625</td>
</tr>
<tr class="even">
<td align="right">Tricaine</td>
<td align="center">Sigma</td>
<td align="left">A5040</td>
</tr>
<tr class="odd">
<td align="right">Tris Base</td>
<td align="center">Roth</td>
<td align="left">4855.2</td>
</tr>
<tr class="even">
<td align="right">Triton-X100</td>
<td align="center">Roth</td>
<td align="left">3051.2</td>
</tr>
<tr class="odd">
<td align="right">Trizol</td>
<td align="center">mbion</td>
<td align="left">15596018</td>
</tr>
<tr class="even">
<td align="right">Tween20</td>
<td align="center">Sigma</td>
<td align="left">P1379</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-sol" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Solutions</h3>
<table>
<caption><span id="tab:mat-sol">Table 2.2: </span> Solutions</caption>
<colgroup>
<col width="23%" />
<col width="59%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Solution</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Cut Smart Buffer</td>
<td align="center">NEB</td>
<td align="left">B7204S</td>
</tr>
<tr class="even">
<td align="right">Generuler 100 bp</td>
<td align="center">Thermo</td>
<td align="left">SM0241</td>
</tr>
<tr class="odd">
<td align="right">Generuler 1kb</td>
<td align="center">Thermo</td>
<td align="left">SM0311</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-anitb" class="section level3">
<h3><span class="header-section-number">2.1.3</span> Antibodies</h3>
<table>
<caption><span id="tab:mat-antib">Table 2.3: </span> Antibodies</caption>
<colgroup>
<col width="23%" />
<col width="38%" />
<col width="21%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Antibody</strong></th>
<th align="center">Company / Provider</th>
<th align="center">concentration</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Anti-Digoxigenin</td>
<td align="center">Roche</td>
<td align="center">1:200</td>
<td align="left">11093274910</td>
</tr>
<tr class="even">
<td align="right">anti-GFP</td>
<td align="center">Torrey Pines</td>
<td align="center">1:200</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">Anti-TAZ (rabbit)</td>
<td align="center">Cell Signaling Technology</td>
<td align="center">1:200</td>
<td align="left">D24E4</td>
</tr>
<tr class="even">
<td align="right">Alexa Fluor488</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">710369</td>
</tr>
<tr class="odd">
<td align="right">Alexa Fluor555</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">Z25005</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-enz" class="section level3">
<h3><span class="header-section-number">2.1.4</span> Enzymes</h3>
<table>
<caption><span id="tab:mat-enz">Table 2.4: </span> Enzymes</caption>
<colgroup>
<col width="27%" />
<col width="55%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Enzyme</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">BtsCI</td>
<td align="center">NEB</td>
<td align="left">R0647</td>
</tr>
<tr class="even">
<td align="right">DdeI</td>
<td align="center">NEB</td>
<td align="left">R0175</td>
</tr>
<tr class="odd">
<td align="right">HaeIII</td>
<td align="center">NEB</td>
<td align="left">R0108</td>
</tr>
<tr class="even">
<td align="right">MnlII</td>
<td align="center">NEB</td>
<td align="left">R0163</td>
</tr>
<tr class="odd">
<td align="right">NlaIII</td>
<td align="center">NEB</td>
<td align="left">R0125</td>
</tr>
<tr class="even">
<td align="right">NsiI-HF</td>
<td align="center">NEB</td>
<td align="left">R3127</td>
</tr>
<tr class="odd">
<td align="right">Phusion Polymerase</td>
<td align="center">NEB</td>
<td align="left">M0530L</td>
</tr>
<tr class="even">
<td align="right">Pronase</td>
<td align="center">Sigma</td>
<td align="left">P5147</td>
</tr>
<tr class="odd">
<td align="right">Ribolock</td>
<td align="center">Thermo</td>
<td align="left">EO0381</td>
</tr>
<tr class="even">
<td align="right">RNase A</td>
<td align="center">Quiagen</td>
<td align="left">1006657</td>
</tr>
<tr class="odd">
<td align="right">RNase H</td>
<td align="center">NEB</td>
<td align="left">M0297L</td>
</tr>
<tr class="even">
<td align="right">SP6 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0131</td>
</tr>
<tr class="odd">
<td align="right">T4 Ligase</td>
<td align="center">NEB</td>
<td align="left">M0202T</td>
</tr>
<tr class="even">
<td align="right">T7 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0111</td>
</tr>
<tr class="odd">
<td align="right">Taq DNA Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">10342-020</td>
</tr>
<tr class="even">
<td align="right">Taq DNA Polymerase</td>
<td align="center">VWR</td>
<td align="left">733-1301</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mobikits" class="section level3">
<h3><span class="header-section-number">2.1.5</span> Molecular Biology Kits</h3>
<table>
<caption><span id="tab:mat-mobikits">Table 2.5: </span> Molecular Biology Kits</caption>
<colgroup>
<col width="54%" />
<col width="28%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Kit</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">EdU Click-iT</td>
<td align="center">Invitrogen</td>
<td align="left">MP 10083</td>
</tr>
<tr class="even">
<td align="right">mMessage mMachine Sp6 Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">AM1340</td>
</tr>
<tr class="odd">
<td align="right">PCR &amp; Gel Clean-Up</td>
<td align="center">Sigma</td>
<td align="left">NA1020</td>
</tr>
<tr class="even">
<td align="right">pGEM-T TA Cloning</td>
<td align="center">Promega</td>
<td align="left">A3600</td>
</tr>
<tr class="odd">
<td align="right">Superscript III cDNA Synthesis</td>
<td align="center">Thermo</td>
<td align="left">18080051</td>
</tr>
<tr class="even">
<td align="right">Wizard SV Gel and PCR Clean-Up</td>
<td align="center">Promega</td>
<td align="left">A9282</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-buff" class="section level3">
<h3><span class="header-section-number">2.1.6</span> Buffers</h3>
<table>
<caption><span id="tab:mat-buff">Table 2.6: </span> Buffers</caption>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Buffer</strong></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Blocking Reagent (BR)</td>
<td>2% BR in maleic buffer + 5% serum</td>
</tr>
<tr class="even">
<td align="left">E3</td>
<td><span class="citation">(<em>45</em>)</span></td>
</tr>
<tr class="odd">
<td align="left">Hybridization buffer</td>
<td>50% Formamide + 25 % 20x SSC + 50mg/mL Heparine + mQ</td>
</tr>
<tr class="even">
<td align="left">Maleic buffer</td>
<td>250 mM maleic acid + 5M NaCl + 10% 0.1% Tween-20 + mQ</td>
</tr>
<tr class="odd">
<td align="left">NTMT</td>
<td>75 mM NaCl + 15 mM MgCl + 15 mM</td>
</tr>
<tr class="even">
<td align="left">PBS</td>
<td>2.7 mM KCl + 12 mM HPO</td>
</tr>
<tr class="odd">
<td align="left">PBST</td>
<td>PBS + 0.1% Tween20</td>
</tr>
<tr class="even">
<td align="left">PBDT</td>
<td>PBS + 1% BSA + 1% DMSO + 0.3% Triton</td>
</tr>
<tr class="odd">
<td align="left">PFA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">P1</td>
<td>50 mM Tris-HCl pH 8.0 + 10 mM EDTA pH 8.0 + 100 µg/mL RNAse</td>
</tr>
<tr class="odd">
<td align="left">P2</td>
<td>1M NaOH + 10 % (w/v) SDS</td>
</tr>
<tr class="even">
<td align="left">P3</td>
<td>3M KOAc pH 5.5</td>
</tr>
<tr class="odd">
<td align="left">TNT</td>
<td>50 mM Tris-HCl pH 8.0 + 100 mM NaCl + 0.1% Tween-20</td>
</tr>
<tr class="even">
<td align="left">Tricaine</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-lines" class="section level3">
<h3><span class="header-section-number">2.1.7</span> Zebrafish lines</h3>
<table>
<caption><span id="tab:mat-lines">Table 2.7: </span> Zebrafish lines</caption>
<colgroup>
<col width="13%" />
<col width="31%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Allele</strong></th>
<th align="center"><strong>name</strong></th>
<th align="left"><strong>zfin</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">zf106Tg</td>
<td align="center"><em>cldnb:lyn-gfp</em></td>
<td align="left"><a href="//zfin.org/ZDB-ALT-060919-2"><em>Tg(-8.0cldnb:LY-EGFP)</em></a></td>
</tr>
<tr class="even">
<td align="right">fu13Tg</td>
<td align="center"><em>cxcr4b</em>(BAC):<em>H2BRFP</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">nns8Tg</td>
<td align="center"><em>atoh1a:Tom</em></td>
<td align="left"><a href="//zfin.org/ZDB-FISH-150901-21622"><em>atoh1a</em></a></td>
</tr>
<tr class="even">
<td align="right">fu50</td>
<td align="center"><em>shroom3</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">m1274Tg</td>
<td align="center"><em>hsp70:shr3v1FL-taqRFP</em></td>
<td align="left">-</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-probes" class="section level3">
<h3><span class="header-section-number">2.1.8</span> ISH probe list</h3>
<table>
<caption><span id="tab:mat-probes">Table 2.8: </span> ISH probes</caption>
<colgroup>
<col width="11%" />
<col width="88%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">atoh1a</td>
<td align="left">see <span class="citation">(<em>27</em>)</span></td>
</tr>
<tr class="even">
<td align="left">deltaD</td>
<td align="left">see <span class="citation">(<em>46</em>)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-hrdwr" class="section level3">
<h3><span class="header-section-number">2.1.9</span> Hardware</h3>
<div id="mat-stamp" class="section level4">
<h4><span class="header-section-number">2.1.9.1</span> Mounting Stamp</h4>
<p>An stl file for 3D printing can be found at <a href="https://github.com/KleinhansDa/3DModels">github.com/KleinhansDa/3DModels</a></p>
</div>
<div id="mat-SD" class="section level4">
<h4><span class="header-section-number">2.1.9.2</span> Spinning Disc Microscopy</h4>
<table>
<caption><span id="tab:SDcomp">Table 2.9: </span> Spinning Disc system components</caption>
<colgroup>
<col width="22%" />
<col width="18%" />
<col width="25%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Microscope</td>
<td align="center">Nikon</td>
<td align="center">Eclipse Ti-E</td>
<td align="left">fully motorized</td>
</tr>
<tr class="even">
<td align="right">PFS</td>
<td align="center">Nikon</td>
<td align="center">Perfect focus system</td>
<td align="left">Z repositioning</td>
</tr>
<tr class="odd">
<td align="right">XY-table</td>
<td align="center">Merzhaeuser</td>
<td align="center">XY motorized table</td>
<td align="left">1 <span class="math inline">\(\mu\)</span>m accuracy</td>
</tr>
<tr class="even">
<td align="right">Piezo</td>
<td align="center"></td>
<td align="center">Piezo Z-table</td>
<td align="left">300 <span class="math inline">\(\mu\)</span>m scan range</td>
</tr>
<tr class="odd">
<td align="right">SD system</td>
<td align="center">Yokogawa</td>
<td align="center">CSU-W1</td>
<td align="left">50 <span class="math inline">\(\mu\)</span>m pattern</td>
</tr>
<tr class="even">
<td align="right">Laser</td>
<td align="center"></td>
<td align="center">Laser Combiner</td>
<td align="left">see table <a href="2-mat-met.html#tab:lasers">2.10</a></td>
</tr>
<tr class="odd">
<td align="right">FRAPPA</td>
<td align="center">Revolution</td>
<td align="center">FRAPPA</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="right">Borealis</td>
<td align="center">Borealis</td>
<td align="center">Borealis</td>
<td align="left">flat field correction</td>
</tr>
<tr class="odd">
<td align="right">sCMOS</td>
<td align="center">Andor</td>
<td align="center">ZYLA PLUS</td>
<td align="left">4.2Mpix; 82%QE</td>
</tr>
<tr class="even">
<td align="right">Immersion</td>
<td align="center">Merzhaeuser</td>
<td align="center">Liquid Dispenser</td>
<td align="left">-</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:lasers">Table 2.10: </span> Available lasers</caption>
<colgroup>
<col width="40%" />
<col width="20%" />
<col width="40%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Lasers</strong></th>
<th align="center">Type</th>
<th align="left">Power</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">405 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">445 nm</td>
<td align="center">diode</td>
<td align="left">80 mW</td>
</tr>
<tr class="odd">
<td align="right">488 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">561 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="odd">
<td align="right">640 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:objectives">Table 2.11: </span> Available objectives</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="17%" />
<col width="15%" />
<col width="9%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Objective</strong></th>
<th align="center">Company</th>
<th align="center">Type</th>
<th align="center">Immersion</th>
<th align="center">N.A.</th>
<th align="left">working distance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">10x</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">air</td>
<td align="center">0.45</td>
<td align="left">4.00 mm</td>
</tr>
<tr class="even">
<td align="right">20X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">0.95</td>
<td align="left">0.95 mm</td>
</tr>
<tr class="odd">
<td align="right">40X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.15</td>
<td align="left">0.60 mm</td>
</tr>
<tr class="even">
<td align="right">60X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.20</td>
<td align="left">0.30 mm</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-work" class="section level4">
<h4><span class="header-section-number">2.1.9.3</span> Workstation</h4>
<p>Statistical computation and image analysis were done on a Fujitsu Siemens (FS) Workstation CELSIUSM740 with the following hardware components…</p>
<table>
<caption><span id="tab:PCcomp">Table 2.12: </span> Workstation hardware components</caption>
<colgroup>
<col width="21%" />
<col width="22%" />
<col width="25%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">CPU</td>
<td align="center">Intel</td>
<td align="center">Xeon E5-1660v4</td>
<td align="left">3.2 GHz, 20MB, 8cores</td>
</tr>
<tr class="even">
<td align="right">RAM</td>
<td align="center">Fujitsu</td>
<td align="center">-</td>
<td align="left">4x16GB DDR4-2400</td>
</tr>
<tr class="odd">
<td align="right">Graphics</td>
<td align="center">NVIDIA</td>
<td align="center">Quadro M4000</td>
<td align="left">8GB RAM</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="mat-sftwr" class="section level3">
<h3><span class="header-section-number">2.1.10</span> Software</h3>
<table>
<caption><span id="tab:sftwr">Table 2.13: </span> Used software</caption>
<colgroup>
<col width="39%" />
<col width="19%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Software</strong></th>
<th align="center">Version</th>
<th align="left">web</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Imagej FIJI</td>
<td align="center">1.48</td>
<td align="left">[<a href="https://fiji.sc/" class="uri">https://fiji.sc/</a>]</td>
</tr>
<tr class="even">
<td align="right">RStudio</td>
<td align="center">1.0.153</td>
<td align="left">[<a href="https://www.rstudio.com/" class="uri">https://www.rstudio.com/</a>]</td>
</tr>
<tr class="odd">
<td align="right">Ubuntu</td>
<td align="center">17.1</td>
<td align="left">[<a href="https://www.ubuntu.com/" class="uri">https://www.ubuntu.com/</a>]</td>
</tr>
<tr class="even">
<td align="right">Windows 10 Pro</td>
<td align="center">10.0.16299</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">Total Commander</td>
<td align="center">9.0</td>
<td align="left">[<a href="http://ghisler.com/" class="uri">http://ghisler.com/</a>]</td>
</tr>
</tbody>
</table>
<div id="image-i-macros" class="section level4">
<h4><span class="header-section-number">2.1.10.1</span> Image I macros</h4>
<div id="cluster-analysis" class="section level5">
<h5><span class="header-section-number">2.1.10.1.1</span> Cluster Analysis</h5>
</div>
<div id="morphometric-analysis" class="section level5">
<h5><span class="header-section-number">2.1.10.1.2</span> Morphometric Analysis</h5>
</div>
<div id="timelapse-analysis" class="section level5">
<h5><span class="header-section-number">2.1.10.1.3</span> Timelapse Analysis</h5>
</div>
</div>
</div>
<div id="mat-GrTrDat" class="section level3">
<h3><span class="header-section-number">2.1.11</span> Ground Truth Data</h3>
<p>Analyzing images and extracting quantitative measurements can be a tedious task, especially when the analysis becomes more complex. Fortunately, there are ways to automate image analysis by using either machine learning approaches or by tailoring hand-crafted algorithms in an image analysis software tool like e.g. ImageJ<span class="citation">(<em>47</em>)</span>.
The main advantages of doing so are to…</p>
<ul>
<li>be independent of confirmation bias</li>
<li>make the analysis more robust against oversight</li>
<li>increase the statistical power by increasing the number of data points<span class="citation">(<em>48</em>)</span></li>
<li>increase effect size by increasing the measurement accuracy<span class="citation">(<em>48</em>)</span></li>
</ul>
<p>However, to ensure the measurements taken by a tailored or trained algorithm are meaningful, they must be compared to a <em>ground truth</em> dataset which again describes a general measure of algorithmic quality performance<span class="citation">(<em>49</em>)</span>.</p>
<div id="cluster-analysis-1" class="section level4">
<h4><span class="header-section-number">2.1.11.1</span> Cluster Analysis</h4>
<p>The <em>Cluster Analysis</em> algorithm was designed for semi-automatic cell cluster detection in the <em>cldnb:lyn-gfp</em> transgenic line and optional nuclei counting in a second DAPI labeled channel within the Regions of Interest (ROIs) derived from the cell cluster detection.</p>
<div id="design" class="section level5">
<h5><span class="header-section-number">2.1.11.1.1</span> Design</h5>
<p>To assess the quality of the <em>ClusterAnalysis</em> algorithm for nuclei detection the ground truth was designed as follows.</p>
<div id="model" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.1</span> Model</h6>
<ul>
<li>each Cell Cluster (CC) consists of a number of objects (cells)</li>
<li>each object is part of the respective CC and defines one cell entity</li>
<li>each object is determined via a fluorescent nucleus label</li>
<li>embryos are mounted within a 3D mold (section <a href="2-mat-met.html#mount-met">2.2.2.4</a>) to reduce noise</li>
</ul>
<table style="width:97%;">
<caption><span id="tab:imgcond3DGrT">Table 2.14: </span> Cluster Analysis Model</caption>
<colgroup>
<col width="48%" />
<col width="48%" />
</colgroup>
<tbody>
<tr class="odd">
<td align="right">XY scale</td>
<td align="left">0.32 px/<span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="even">
<td align="right">Z-spacing</td>
<td align="left">4 <span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="odd">
<td align="right">Camera</td>
<td align="left">Rolera</td>
</tr>
</tbody>
</table>
</div>
<div id="training-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.2</span> Training data</h6>
<p>The training set consists of three randomly picked wild type PLLps. For each the algorithm was run with standard parameters. Cell cluster ROIs and nuclei multi point labels were edited manually.</p>
</div>
<div id="test-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.3</span> Test data</h6>
<p>To test the algorithm, it is run at different nuclei detection thresholds on the same image data.</p>
</div>
</div>
<div id="results" class="section level5">
<h5><span class="header-section-number">2.1.11.1.2</span> Results</h5>
<div id="image-analysis" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.1</span> Image Analysis</h6>
<p>For image analysis a custom IJ macro script was developed that segments individual cell clusters based on morphological filtering<span class="citation">(<em>50</em>)</span>, Gaussian blurring and subsequent thresholding. The user is then presented the segmentation ROIs to manually edit them, if necessary. Using those ROIs as masks, the nuclei within them are counted with a 2-D maxima finder. To detect the right amount of nuclei however, it is necessary to evaluate the distance over which the blurring should be applied.</p>
<p>A typical nucleus in the PLLp is about 5 <span class="math inline">\(\mu\)</span>m in diameter. To determine the right blurring value, a range of 4-6 <span class="math inline">\(\mu\)</span>m in steps of 0.5 is tested.</p>
</div>
<div id="macro-modules" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.2</span> Macro modules</h6>
<p>The macro developed consists of the following main modules:</p>
<ol style="list-style-type: decimal">
<li>Background subtraction, morphological filtering, Gaussian blurring &amp; segmentation</li>
<li>Manual correction for cell cluster segmentation masks</li>
<li>ROI indexing from left (1) to right (n)</li>
<li>Nuclei counting within cell cluster masks</li>
<li>Export of results table (Area, Count, …)</li>
</ol>
</div>
<div id="registered-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.3</span> Registered data</h6>
<p>Figure <a href="2-mat-met.html#fig:maxllreg">2.1</a> shows a registered maximum Z projected lateral line used for ground truth evaluation.</p>

<div class="figure" style="text-align: center"><span id="fig:maxllreg"></span>
<img src="figures/materials/ground_truth/clusters/clusters.png" alt="Registration of 2D data cluster segments Registered, MaxIP data with cell cluster segments of the lyn-GFP signal laid upon the DAPI signal. cell labels Magenta dots represent the maxima found within each ROI and hence the nuclei labels." width="90%" />
<p class="caption">
Figure 2.1: Registration of 2D data <strong>cluster segments</strong> Registered, MaxIP data with cell cluster segments of the lyn-GFP signal laid upon the DAPI signal. <strong>cell labels</strong> Magenta dots represent the maxima found within each ROI and hence the nuclei labels.
</p>
</div>
</div>
</div>
<div id="data-analysis" class="section level5">
<h5><span class="header-section-number">2.1.11.1.3</span> Data Analysis</h5>
<div id="nuclei-count" class="section level6">
<h6><span class="header-section-number">2.1.11.1.3.1</span> Nuclei Count</h6>
<p>Comparing the segmented objects count of the Gaussian parameters with the ground truth gives an indication for over- resp. under-segmentation. In figure <a href="2-mat-met.html#fig:GrTratioCC">2.2</a> the relative numbers for each blurring parameter can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). The red area represents over-segmentation, the green under-segmentation.</p>

<div class="figure" style="text-align: center"><span id="fig:GrTratioCC"></span>
<img src="thesis_files/figure-html/GrTratioCC-1.png" alt="Relative difference of segment counts (N=3, Error bars = Std./2, NB = nuclei blurring)" width="65%" />
<p class="caption">
Figure 2.2: Relative difference of segment counts (N=3, Error bars = Std./2, NB = nuclei blurring)
</p>
</div>
<p>To estimate the quality of nuclei detection for each parameter, the ratio of automatically detected and ground truth objects count can be calculated and compared (table <a href="2-mat-met.html#tab:nrcount">2.15</a>). The closer it is to 1, the better.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:nrcount">Table 2.15: </span>Nuclei count ratio
</caption>
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
NB45
</th>
<th style="text-align:left;">
NB50
</th>
<th style="text-align:left;">
NB55
</th>
<th style="text-align:left;">
NB60
</th>
<th style="text-align:left;">
NB70
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Mean_ratio
</td>
<td style="text-align:left;">
1.30
</td>
<td style="text-align:left;">
1.14
</td>
<td style="text-align:left;">
1.04
</td>
<td style="text-align:left;">
0.95
</td>
<td style="text-align:left;">
0.86
</td>
</tr>
<tr>
<td style="text-align:left;">
STD.
</td>
<td style="text-align:left;">
0.23
</td>
<td style="text-align:left;">
0.17
</td>
<td style="text-align:left;">
0.15
</td>
<td style="text-align:left;">
0.14
</td>
<td style="text-align:left;">
0.14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="summary-1" class="section level5">
<h5><span class="header-section-number">2.1.11.1.4</span> Summary</h5>
<p>Maximum performance is achieved at a scaled parameter of 6 <span class="math inline">\(\mu\)</span>m, with a ratio of of 0.95 for the count objects and a standard deviation of 0.14.</p>
</div>
</div>
<div id="anallzr3d" class="section level4">
<h4><span class="header-section-number">2.1.11.2</span> anallzr3D</h4>
<p>The anallzr3D algorithm was designed for fully automated, single cell volume segmentation in the <em>cldnb:lyn-gfp</em> transgenic line. In addition to 3D cellular metrics, it offers Apical Constriction measurement of each cell.</p>
<div id="design-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.1</span> Design</h5>
<p>To assess the quality of the anaLLzr3D algorithm the ground truth was modeled as follows.</p>
<div id="model-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.1</span> Model</h6>
<ul>
<li>each PLLp consists of a number of objects (cells)</li>
<li>each object is part of the PLLp and defines one cell entity</li>
<li>cell boundaries are determined via the transgene <em>cldnb:lyn-gfp</em> +/+</li>
<li>embryos are imaged live to conserve signal and membrane integrity</li>
<li>embryos are mounted within a 3D mold for improved 3D alignment</li>
</ul>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:model3DGrT">Table 2.16: </span>anaLLzr3D Model
</caption>
<tbody>
<tr>
<td style="text-align:right;width: 3cm; ">
Exposure time
</td>
<td style="text-align:left;">
100 ms
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
Laser intensity
</td>
<td style="text-align:left;">
100<span class="math inline">\(\%\)</span> / 9.3 mW
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
Objective
</td>
<td style="text-align:left;">
40X; CFI APO LWD WI; N.A. = 1.15, W.D. = 0.60 mm
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
Z-spacing
</td>
<td style="text-align:left;">
0.4 - 0.5 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
Camera
</td>
<td style="text-align:left;">
sCMOS; 4.2 M.Pix; 82<span class="math inline">\(\%\)</span> Q.E.
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
SD system
</td>
<td style="text-align:left;">
Yokogawa CSU - W1; 50 <span class="math inline">\(\mu\)</span>m pattern
</td>
</tr>
<tr>
<td style="text-align:right;width: 3cm; ">
Piezo
</td>
<td style="text-align:left;">
Piezo Z-table; 300 <span class="math inline">\(\mu\)</span>m scan range
</td>
</tr>
</tbody>
</table>
</div>
<div id="training-data-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.2</span> Training data</h6>
<p>The training set consists of three randomly picked wildtype PLLps. For each the algorithm is run with no filters (X, Y, Z border objects, size) and a minimum segmentation threshold. Afterwards the segmentation result is corrected manually for over- and under-segmentation and objects that are not part of the PLLp.</p>
</div>
<div id="test-data-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.3</span> Test data</h6>
<p>To test the algorithm it is run at different segmentation thresholds on the same image data.</p>
</div>
</div>
<div id="results-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.2</span> Results</h5>
<div id="image-analysis-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.1</span> Image Analysis</h6>
<p>For image analysis a custom IJ macro script was developed that recognizes the cell boundaries via the fluorescence signal emitted by a Lyn Kinase tethered eGFP which expression is controlled by the <em>cxcr4b</em> lateral line specific promotor <span class="citation">(<em>21</em>)</span>. The central IJ tool used to do this is the MorphoLibJ’s<span class="citation">(<em>50</em>)</span> <em>Morpholigical Segmentation</em> plugin. The plugin however requires to choose for a ‘segmentation threshold’ that determines the quality and the quantity of segmented objects. This parameter therefore plays an essential role in the reliability of the analysis results.</p>
</div>
<div id="macro-modules-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.2</span> Macro modules</h6>
<p>The macro developed consists of the following main modules:</p>
<ol style="list-style-type: decimal">
<li>PLLp registration and cropping in X, Y and Z</li>
<li>Image segmentation using the MorphoLibJ’s<span class="citation">(<em>50</em>)</span> <em>Morpholigical Segmentation</em> plugin</li>
<li>Extration of 3-D measurements using the 3D ImageJ Suite’s<span class="citation">(<em>51</em>)</span> <em>3D Manager</em> plugin</li>
<li>Cropping, clearing and measurement of single cell volumes using a custom IJ script</li>
</ol>
</div>
<div id="registration" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.3</span> Registration</h6>
<p>The first module of the macro is PLLp registration in X, Y and cropping in Z. This is accomplished by an initial maximum Z projection and blurring of the image, 2D segmentation using a minimum threshold and lastly by rotating the segment through the angle formed by the long axis of the ellipsoid and the horizon (at 0<span class="math inline">\(^{\circ}\)</span>). After rotation the image is cropped according to the obtained ROI. Additionally, the centers of the most constricting areas are detected via an intensity based dynamic threshold and highlighted as magenta circles in <a href="2-mat-met.html#fig:maxraw">2.3</a>.</p>
(ref:maxraw) Registration of 3D data
<div class="figure" style="text-align: center"><span id="fig:maxraw"></span>
<img src="figures/materials/ground_truth/registration.png" alt="(ref:maxraw)" width="90%" />
<p class="caption">
Figure 2.3: (ref:maxraw)
</p>
</div>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgprop">Table 2.17: </span>Physical image properties (scaling)
</caption>
<tbody>
<tr>
<td style="text-align:right;">
XY resolution
</td>
<td style="text-align:left;">
0.1625 / 0.1625 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;">
Z resolution
</td>
<td style="text-align:left;">
0.4 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;">
<span class="math inline">\(\Delta\)</span>T in Z
</td>
<td style="text-align:left;">
0.5428 s
</td>
</tr>
</tbody>
</table>
</div>
<div id="ground-truth-data" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.4</span> Ground Truth data</h6>
<p>In figure <a href="#fig:stackmem"><strong>??</strong></a> the fluorescence signal of the three PLLps tested is shown in a single central cross-section in the dorso-ventral and the apico-basal axis.</p>
<p>To compare the results between the ground truth segments and the segments obtained from different threshold levels graphically, for a single PLLp the ground truth and threshold level[n] are shown as a composite color image in figure <a href="2-mat-met.html#fig:stackcomp">2.4</a>. By using a green lookup table (LUT) for the ground truth and a magenta LUT for the threshold level[n], one can readily detect overlapping objects (white), over segmentation (magenta) and under segmentation (green).</p>
(ref:stackmem) <em>cldnb:lyn-gfp</em> fluorescence signal in a cross-section of the PLLp (Obj.: 40X APO)
<div class="figure" style="text-align: center"><span id="fig:stackcomp"></span>
<img src="figures/materials/ground_truth/stackmem.png" alt="Graphical comparison of the thresholds tested" width="85%" />
<p class="caption">
Figure 2.4: Graphical comparison of the thresholds tested
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:anallzrvols"></span>
<img src="figures/materials/ground_truth/volumes.png" alt="Graphical comparison of the thresholds tested" width="85%" />
<p class="caption">
Figure 2.5: Graphical comparison of the thresholds tested
</p>
</div>
<p>Volume renderings have been done with IJ’s <a href="%22https://github.com/fiji/Volume_Viewer/releases/tag/Volume_Viewer-2.01.2%22">VolumeViewer</a></p>
</div>
</div>
<div id="data-analysis-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.3</span> Data Analysis</h5>
<div id="cell-count" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.1</span> Cell Count</h6>
<p>In figure <a href="2-mat-met.html#fig:gtratio">2.6</a> the relative numbers for each threshold level can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). Analogous to the graphical inspection, the magenta area represents over segmentation, the green under segmentation. Additionally, table <a href="2-mat-met.html#tab:gtrcellcount">2.18</a> shows the absolute numbers.</p>

<div class="figure" style="text-align: center"><span id="fig:gtratio"></span>
<img src="thesis_files/figure-html/gtratio-1.png" alt="Relative difference of segment counts (N = 3, Errorbars = Std./2)" width="65%" />
<p class="caption">
Figure 2.6: Relative difference of segment counts (N = 3, Errorbars = Std./2)
</p>
</div>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:gtrcellcount">Table 2.18: </span>Cellcount ground truth
</caption>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;">
Group
</td>
<td style="text-align:left;font-weight: bold;">
T01_GrT
</td>
<td style="text-align:left;font-weight: bold;">
T02
</td>
<td style="text-align:left;font-weight: bold;">
T03
</td>
<td style="text-align:left;font-weight: bold;">
T04
</td>
<td style="text-align:left;font-weight: bold;">
T05
</td>
<td style="text-align:left;font-weight: bold;">
T10
</td>
</tr>
<tr>
<td style="text-align:left;">
P05
</td>
<td style="text-align:left;">
146
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
133
</td>
<td style="text-align:left;">
131
</td>
<td style="text-align:left;">
131
</td>
<td style="text-align:left;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
P07
</td>
<td style="text-align:left;">
150
</td>
<td style="text-align:left;">
184
</td>
<td style="text-align:left;">
169
</td>
<td style="text-align:left;">
164
</td>
<td style="text-align:left;">
156
</td>
<td style="text-align:left;">
137
</td>
</tr>
<tr>
<td style="text-align:left;">
P08
</td>
<td style="text-align:left;">
127
</td>
<td style="text-align:left;">
139
</td>
<td style="text-align:left;">
133
</td>
<td style="text-align:left;">
123
</td>
<td style="text-align:left;">
117
</td>
<td style="text-align:left;">
92
</td>
</tr>
</tbody>
</table>
</div>
<div id="cell-morphology" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.2</span> Cell Morphology</h6>
<blockquote>
<p><em>There is always a precision / recall trade-off in detection tasks, e.g., when we set a lower threshold [. . .], we can get a higher recall with a lower precision ([. . .], but meanwhile we also get more false-positives in the results).</em><span class="citation">(<em>41</em>)</span></p>
</blockquote>
<p>Inspecting the cell count unfortunately does not directly tell us how well the cell morphology is conserved at different threshold levels, since at higher threshold levels the cell boundaries are differently determined and eventually not even recognized as such anymore.</p>
<p>The volume of a cell is a very robust morphological feature. Therefore, if its volume does not differ significantly at a given threshold level we consider its morphology to be conserved.</p>
</div>
<div id="sample-distribution" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.3</span> Sample distribution</h6>
<p>In figure <a href="2-mat-met.html#fig:volsecdffull">2.7</a> and <a href="2-mat-met.html#fig:volsecdfzoom">2.8</a> the distribution and amount of the cell volumes across the different threshold levels can be inspected via the ‘empirical cumulative distribution function’ (ECDF). The closer the slopes are to the ground truth, the stronger they are conserved. Figure <a href="2-mat-met.html#fig:volsecdffull">2.7</a> shows the full distribution, where the major differences seem to appear within the 0.4 quantile (red line). Therefore, in figure <a href="2-mat-met.html#fig:volsecdfzoom">2.8</a> only the values within the 0.4 quantile are shown to get a more detailed impression.</p>
(ref:volsecdffull) Cell volumes full distribution (red line = .4 Quantile)
<div class="figure" style="text-align: center"><span id="fig:volsecdffull"></span>
<img src="thesis_files/figure-html/volsecdffull-1.png" alt="(ref:volsecdffull)" width="90%" />
<p class="caption">
Figure 2.7: (ref:volsecdffull)
</p>
</div>
(ref:volsecdfzoom) Cell volumes distribution within .4 Quantile
<div class="figure" style="text-align: center"><span id="fig:volsecdfzoom"></span>
<img src="thesis_files/figure-html/volsecdfzoom-1.png" alt="(ref:volsecdfzoom)" width="90%" />
<p class="caption">
Figure 2.8: (ref:volsecdfzoom)
</p>
</div>
</div>
<div id="confidence-level" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.4</span> Confidence level</h6>
<p>To statistically check how closely related each threshold level sample distribution is compared to the ground truth, a Kolmogorov-Smirnov-Test (ks.test) was performed. The ks.test is a nonparametric test whose null hypothesis is that the both groups compared were sampled from populations with identical populations. Therefore, the closer the p-value is to 1, the more similar the tested sample distribution would be to the ground truth (figure <a href="2-mat-met.html#fig:kstestvol">2.9</a>; 5<span class="math inline">\(\%\)</span> confidence threshold indicated by red line).</p>
<p><span class="math display">\[H_{0} : F_X(x)=F_0(x)\]</span></p>

<div class="figure" style="text-align: center"><span id="fig:kstestvol"></span>
<img src="thesis_files/figure-html/kstestvol-1.png" alt="Volumetric conservation (red line = 5\(\%\) confidence threshold)" width="60%" />
<p class="caption">
Figure 2.9: Volumetric conservation (red line = 5<span class="math inline">\(\%\)</span> confidence threshold)
</p>
</div>
<p>In comparison to the Mann-Whitney test, the ks.test is more sensitive to detect changes in the shape of the distribution than to detect a shift of the median<span class="citation">(<em>52</em>)</span></p>
</div>
</div>
</div>
</div>
</div>
<div id="met" class="section level2">
<h2><span class="header-section-number">2.2</span> Methods</h2>
<div id="Zeb-met" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Zebrafish</h3>
<div id="husbandry" class="section level4">
<h4><span class="header-section-number">2.2.1.1</span> Husbandry</h4>
<p>Zebraﬁsh husbandry was maintained at the University of Frankfurt am Main. All legal procedures were followed while handling and maintaining zebraﬁsh husbandry. All zebraﬁsh lines used in and generated for this study are listed in the Appendix A.</p>
</div>
<div id="handling-and-rearing" class="section level4">
<h4><span class="header-section-number">2.2.1.2</span> Handling and rearing</h4>
<p>In the afternoon preceding the embryo collection, 1 male and female were set up in crossing cages, physically separated by a transparent separator. Next day, before noon, separators were removed allowing fertilization. Fertilized eggs were then collected, sorted and reared in the well-defined culture medium E3 (Kimmel et.al. 1995, section <a href="2-mat-met.html#mat-buff">2.1.6</a>) at 25<span class="math inline">\(^\circ\)</span>, 28.5<span class="math inline">\(^\circ\)</span>, or 30<span class="math inline">\(^\circ\)</span> Celsius depending on the experimental condition required.</p>
<p>To grow larvae to the adulthood, they were transferred to the system on day 5. Till day 12, larvae were fed Vinegar Eels, Paramecia, and Caviar powder. After the 12 th day, water supply was started and fish were fed Brine Shrimp, Artemia, Paramecia and Vinegar Eels. Adult fish (&gt;1 Month) were fed Artemia and the dry flakes.</p>
</div>
<div id="zebrafish-fin-clips" class="section level4">
<h4><span class="header-section-number">2.2.1.3</span> Zebrafish fin clips</h4>
<p>Adult ﬁsh were anesthetized with buﬀered Tricaine (1X, see section <a href="2-mat-met.html#mat-buff">2.1.6</a>) until. About 1/3 rd of the caudal ﬁn was cut with a sterile scalpel in a sterile Petri Dish. Immediately the cut ﬁn was transferred to 50 mM NaOH. Fish were returned to system water and kept in 1L system water in single tanks with 200 <span class="math inline">\(\mu\)</span>L of 0.01<span class="math inline">\(\%\)</span> Methylene Blue.</p>
</div>
<div id="adult-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.1.4</span> Adult Genotyping</h4>
<p>The clipped fins were digested for 1 h at 95<span class="math inline">\(^\circ\)</span> Celsius and neutralized subsequently with 1M Tris-HCl of (pH 9).</p>
</div>
<div id="embryo-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.1.5</span> Embryo Genotyping</h4>
<p>Single fixed/live embryos were denatured at 95<span class="math inline">\(^\circ\)</span> Celsius in 50mM NaOH for 1 hour and neutralized by adding 1/10 the volume of 1 M Tris-HCl (pH 8.2).</p>
</div>
<div id="zebrafish-euthanasia" class="section level4">
<h4><span class="header-section-number">2.2.1.6</span> Zebrafish Euthanasia</h4>
<p>Adult zebrafish were anesthetized in tricaine (1X) till stage II-III of anaestezia and put in the ice cold water so as to sacrifice them by hypothermia.</p>
</div>
<div id="fixation" class="section level4">
<h4><span class="header-section-number">2.2.1.7</span> Fixation</h4>
<p>Embryos and dechorionated larvae were fixed in 2 ml of 4<span class="math inline">\(\%\)</span> PFA in 1X PBS overnight at 4<span class="math inline">\(^\circ\)</span> Celsius.</p>
</div>
</div>
<div id="Wet-met" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Wet lab</h3>
<div id="sampleprep" class="section level4">
<h4><span class="header-section-number">2.2.2.1</span> Sample preparation</h4>
<p>For samples <strong>later than 24 hpf</strong>, embryos were grown in 1X PTU till desired stage and treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase for ~30 min.. Choria were removed by gentle pipetting with a 2 mL plastic pasteur pipette. To replace the Pronase solution with fresh E3, embryos were immobilized by anesthesia and collected in the center of the dish by gentle rotational movement. Then the medium was poured away by collecting the embryos at the corner bottom of the dish while taking care not to loose any. After, the dish was filled with fresh E3. This process was repeated three times.</p>
<p>For samples <strong>earlier than 18s</strong>, embryos were treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase directly. Choria were removed the same way as for &gt; 24 hpf embryos but when pouring away the Pronase solution, the dish was simultaneously and very carefully filled with fresh E3 again. Since younger embryos are more fragile and to avoid damage, they must be kept in solution constantly.</p>
<p>Fixation started at the desired stage in 4% PFA in 0.1% PBST at 4<span class="math inline">\(^\circ\)</span>C o.n.. The next day, samples were rinsed 3 times for ~5 min. in PBST and passed through a MeOH series of 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 100<span class="math inline">\(\%\)</span> MeOH/PBST (V/V)). For permanent storage, samples were stored in 100<span class="math inline">\(\%\)</span> MeOH at -20<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="ISH-met" class="section level4">
<h4><span class="header-section-number">2.2.2.2</span> In Situ Hybridization</h4>
<p>Sample preparation as described in section <a href="2-mat-met.html#sampleprep">2.2.2.1</a>.</p>
<div id="permeabilisation-probe-hybridization" class="section level5">
<h5><span class="header-section-number">2.2.2.2.1</span> 1. Permeabilisation &amp; Probe Hybridization </h5>
<p><strong>Permeabilisation</strong> <span class="math inline">\(\rightarrow\)</span> without shaking</p>
<p>Samples were rehydrated in an inverse MeOH series of 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 0<span class="math inline">\(\%\)</span> PBST and washed again for 5 min. two times in pure PBST. Finally, samples were digested in 10 <span class="math inline">\(\mu\)</span>g/mL Proteinase K according to table <a href="2-mat-met.html#tab:met-protk">2.19</a>.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:met-protk">Table 2.19: </span>Proteinase K digestion
</caption>
<thead>
<tr>
<th style="text-align:center;">
Stage
</th>
<th style="text-align:center;">
0.6 s
</th>
<th style="text-align:center;">
7 s
</th>
<th style="text-align:center;">
18 s
</th>
<th style="text-align:center;">
24 hpf
</th>
<th style="text-align:center;">
32 hpf
</th>
<th style="text-align:center;">
36 hpf
</th>
<th style="text-align:center;">
42 hpf
</th>
<th style="text-align:center;">
48 hpf
</th>
<th style="text-align:center;">
72 hpf
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
min.
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
30
</td>
<td style="text-align:center;">
40
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:center;">
60
</td>
<td style="text-align:center;">
60
</td>
</tr>
</tbody>
</table>
<p>Samples were washed again for 5 min. two times in PBST and post-fixated in 4<span class="math inline">\(\%\)</span> PFA at 4<span class="math inline">\(^\circ\)</span>C for &gt; 30 min. Samples were washed again for 5 min. three times in PBST.</p>
<p><strong>Probe Hybridisation</strong> <span class="math inline">\(\rightarrow\)</span> all steps at 60<span class="math inline">\(^\circ\)</span>C, except stated differently</p>
<p>Samples were pre-hybridized in 350 <span class="math inline">\(\mu\)</span>L of hybridization buffer (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h. Just before detection probe treatment, the probe was denatured at 80<span class="math inline">\(^\circ\)</span>C in 1:200 of hybridization buffer. Subsequently, hybridization buffer was taken off the samples and replaced with the heated probe. Finally, samples were incubated o.n. at a desired temperature (around 65<span class="math inline">\(^\circ\)</span>C).</p>
</div>
<div id="probe-removal-antibody-incubation" class="section level5">
<h5><span class="header-section-number">2.2.2.2.2</span> 2. Probe removal &amp; Antibody incubation</h5>
<p>The next day the probe got collected and stored at -20<span class="math inline">\(^\circ\)</span>C for re-use. Washing took place at the same temperature as hybridization (from step 1) To keep solutions at temperature a Thermo-Block was used. For the washing series the samples were first washed one time for 20 min. in hybridization buffer, then two times for 30 min. in 50<span class="math inline">\(\%\)</span> Formamide and one time for 20 min. in 25<span class="math inline">\(\%\)</span> Formamide. Then two times for 15 min. in 2X SSCT and two times for 30 min. in 0.2X SSCT. Finally, one time for 5 min. in TNT.</p>
<p>To reduce noise and increase specific signal strength, the samples were treated with blocking solution (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h in 350 <span class="math inline">\(\mu\)</span>L of 2% BR/TNT at room temperature (RT). Afterwards the samples were incubated in 100 <span class="math inline">\(\mu\)</span>L Anti-Digoxigenin diluted in NTMT buffer (1/4000 (V/V)) in 2<span class="math inline">\(\%\)</span> BR/TNT for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="probe-detection" class="section level5">
<h5><span class="header-section-number">2.2.2.2.3</span> 3. Probe detection</h5>
<p>First, the samples were washed six times for ~20 min. (or one wash o.n.) in TNT and two times for ~ 5min. in NTMT. After washing, color staining was performed with 4.5 NBT <span class="math inline">\(\mu\)</span>L + 3.5 <span class="math inline">\(\mu\)</span>L BCIP per mL NTMT in the dark and at RT without shaking (in a drawer) for 2 - 8 h, regularly checking the progression of the reaction. As soon as an appropriate degree of color intensity on the target site was achieved (up to two days), the samples were again washed three times in PBST.</p>
<p>Afterwards the samples were either prepared for immunostaining or imaging. For permanent storage samples were kept in 50<span class="math inline">\(\%\)</span> Glycerol at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
</div>
<div id="immuno-met" class="section level4">
<h4><span class="header-section-number">2.2.2.3</span> Immuno staining</h4>
<p>Sample preparation as described in section <a href="2-mat-met.html#sampleprep">2.2.2.1</a>.</p>
<p>First, samples were blocked in 2<span class="math inline">\(\%\)</span> Goat Serum / PBDT (V/V) for 30 min.. For protein target site detection, a <strong>primary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for ~2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times. To stain the now bound primary antibody, a <strong>secondary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times.</p>
</div>
<div id="mount-met" class="section level4">
<h4><span class="header-section-number">2.2.2.4</span> Mounting</h4>
<p>For mounting, an improved 3D specimen preparation and well-plate like sample navigation for zebrafish larvae confocal microscopy was developed which offers the following improvements…</p>
<ol style="list-style-type: decimal">
<li><p><strong>To obtain a sufficient amount of data that is closely staged and still comprises &gt; 4 independent groups.</strong> Using this protocol, it is possible to define a custom well plate (figure <a href="#fig:stampprod"><strong>??</strong></a> C) in your preferred imaging software, which greatly facilitates the imaging procedure and therefore helps to stay focused. However, once the well plate is aligned and the positions are defined it still needs some minor correction in Z.</p></li>
<li><p><strong>To do the imaging live. Most of the time, especially in morphological analyses, live imaging is preferred since any fixation alters cell - and therefore organ morphology.</strong> Live imaging guarantees an undisturbed and unbiased look at the <em>in vivo</em> situation. Using this protocol, the embryonic body axes will be more in line with the XY displacement of the microscope stage which results, depending on the volume of interest, in narrower Z-stacks. This ultimately also leads to a reduction of photo-bleaching, phototoxicity and expenditure of time. The gained time can then be used to image shorter time-intervals in timelapse imaging.</p></li>
<li><p><strong>To reduce background signal and noise.</strong> Instead of transforming and aligning the image data in a post-processing approach, the specimen can be imaged in the right spacial configuration in the first place. This is not only important for 3D analysis, but also for Z projected data. If the specimen body axes are not aligned with the axes of the microscopic table, the image stacks need to be bigger to record the volume of interest. In Z projections, for X Y coordinates out of the volume of interest, this will end up as background signal or noise (figure <a href="#fig:stampprod"><strong>??</strong></a> B), complicating quantitative analysis and interpretation. The method therefore provides significant improvement over existing mounting techniques.</p></li>
</ol>
<div id="protocol" class="section level5">
<h5><span class="header-section-number">2.2.2.4.1</span> Protocol</h5>
<p>First, make sure your stamp is clean of dust or other particles. If necessary clean it with 70% Ethanol and / or pressured air. Of a 1% standard Agarose in E3, apply…</p>
<ul>
<li>600 <span class="math inline">\(\mu\)</span>L in case of a <span class="math inline">\(\varnothing\)</span> 35 mm ibidi dish</li>
<li>2000 <span class="math inline">\(\mu\)</span>L in case of a <span class="math inline">\(\varnothing\)</span> 50 mm mattek dish</li>
</ul>
<p>to the glass or polymer surface and place the stamp on it so that it touches the cover. Place it as central as possible. Excessive agarose not needed for the imprint will fill the surrounding area in the dish and stabilize the structure when removing the stamp again.</p>
<p>Gently rotate the dish so that excess agarose solution will cover the whole dish.</p>
<p>Wait for ~45 min., then use a clean preparation needle and slip it through stamp - polymerized agarose interface. Remove the stamp carefully by lifting it and holding the <span class="math inline">\(\mu\)</span>-Dish in place.</p>

<div class="figure" style="text-align: center"><span id="fig:stampprod-1"></span>
<img src="files/pubs/klein_2019/006_prep.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.10: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-2"></span>
<img src="files/pubs/klein_2019/007.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.11: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-3"></span>
<img src="files/pubs/klein_2019/008.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.12: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:stampprod-4"></span>
<img src="files/pubs/klein_2019/009.jpg" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%" />
<p class="caption">
Figure 2.13: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
</div>
</div>
</div>
<div id="comp-met" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Dry lab</h3>
<div id="CNN" class="section level4">
<h4><span class="header-section-number">2.2.3.1</span> Rosette Detection</h4>
<p>The method used for rosette detection is based on a convolutional neuronal network (CNN) and was modified from the “rosette detector” algorithm previously used in the lab and described in <span class="citation">(<em>29</em>, <em>41</em>)</span>. Since the former method was technically outdated and since we had new data in which we needed to detect rosettes, we updated the former method to a state of the art CNN using Caffe<span class="citation">(<em>53</em>)</span> as a backend. The training was done by our collaborators at the institute for Informatics, Albert-Ludwigs-University Freiburg.</p>
<p>The CNN was trained on 17 DMSO- and SU5402- treated embryos each. SU5402 is an inhibitor of FGF receptors and embryos treated with these inhibitors show a strongly reduced number of rosettes (<span class="citation">(<em>27</em>)</span>). In order to give the network something to learn, the data had to be labeled manually, which was done in ImageJ by placing multipoint ROIs at the center of the rosettes. The data was then further permutated to artificially increase the amount of training data and make the detection more robust against different kinds of input. Further parameters about the training data is listed in Table <a href="2-mat-met.html#tab:CNNtraining">2.20</a>. One example for each is shown in figure <a href="2-mat-met.html#fig:CNNtrain">2.14</a>.</p>

<div class="figure" style="text-align: center"><span id="fig:CNNtrain"></span>
<img src="figures/materials/cnn/CNNtrain.png" alt="Example for rosette detection on the training data. left Maximum Z-projected input data. middle heatmap of scores. blue indicates a low score, red a high one. right score map projected onto the input data." width="75%" />
<p class="caption">
Figure 2.14: Example for rosette detection on the training data. <strong>left</strong> Maximum Z-projected input data. <strong>middle</strong> heatmap of scores. blue indicates a low score, red a high one. <strong>right</strong> score map projected onto the input data.
</p>
</div>
<p>The main advantages for using a neural network in a task like this are…</p>
<ol style="list-style-type: decimal">
<li>Objectivity
<ul>
<li>A computer model is not biased in a way that it prefers one outcome over the other. It evaluates based on what it was trained to.</li>
<li>Unlike the human brain, once a CNN is trained it is static and does not keep on learning.</li>
</ul></li>
<li>Degree of rosette registration
<ul>
<li>The output data are continous rational numbers (<span class="math inline">\(\mathbb{Q}\)</span>) instead of integers (<span class="math inline">\(\mathbb{Z}\)</span>) which does not only tell if a rosette is there fore not, but also for ‘how much’ (50-100<span class="math inline">\(\%\)</span>) it is there.</li>
</ul></li>
<li>Training is done relatively quick</li>
</ol>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:CNNtraining">Table 2.20: </span>CNN training data
</caption>
<thead>
<tr>
<th style="text-align:center;font-weight: bold;">
group
</th>
<th style="text-align:center;font-weight: bold;">
conc.
</th>
<th style="text-align:center;font-weight: bold;">
nm
</th>
<th style="text-align:center;font-weight: bold;">
intensity….
</th>
<th style="text-align:center;font-weight: bold;">
exposure
</th>
<th style="text-align:center;font-weight: bold;">
z.planes
</th>
<th style="text-align:center;font-weight: bold;">
magnification
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
DMSO
</td>
<td style="text-align:center;width: 1cm; ">
0.1<span class="math inline">\(\%\)</span>
</td>
<td style="text-align:center;width: 2cm; ">
488
</td>
<td style="text-align:center;width: 1.5cm; ">
100
</td>
<td style="text-align:center;width: 1.5cm; ">
100 ms
</td>
<td style="text-align:center;width: 1.5cm; ">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
<tr>
<td style="text-align:center;">
SU5402
</td>
<td style="text-align:center;width: 1cm; ">
10<span class="math inline">\(\mu\)</span>M
</td>
<td style="text-align:center;width: 2cm; ">
488
</td>
<td style="text-align:center;width: 1.5cm; ">
100
</td>
<td style="text-align:center;width: 1.5cm; ">
100 ms
</td>
<td style="text-align:center;width: 1.5cm; ">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
</tbody>
</table>
</div>
<div id="prolif" class="section level4">
<h4><span class="header-section-number">2.2.3.2</span> Proliferation Analysis</h4>
<p>The basic principle is based on work done by Laguerre <em>et al.</em>, 2009<span class="citation">(<em>23</em>)</span>. For registration of mitotic events an IJ manual tracking tool was used was used that allows to track an image feature through a stack of images creating tracks as it progresses through volume / time (‘MTrackJ’<span class="citation">(<em>54</em>)</span>).</p>
<p>For mounting the procedure described in section <a href="2-mat-met.html#mount-met">2.2.2.4</a> was used. Nuclei were visualized in a <em>cxcr4b</em>(BAC):<em>H2BRFP</em> transgenic line. Z-projection of volumetric timelapses, Mitotic events were tracked in each CC and the pLLP. Afterwards the data was exported as one table per embryo and processes by counting mitoses per pLLP / CC / total CC mitoses per embryo.</p>
<p>Figure <a href="2-mat-met.html#fig:mitodatapoints">2.15</a> shows an exemplary track for the data analyzed.</p>

<div class="figure" style="text-align: center"><span id="fig:mitodatapoints"></span>
<img src="figures/materials/prol/Prolif.png" alt="Tracking of mitotic events" width="50%" />
<p class="caption">
Figure 2.15: Tracking of mitotic events
</p>
</div>
</div>
<div id="ACI" class="section level4">
<h4><span class="header-section-number">2.2.3.3</span> Apical Index</h4>
<div id="rationale" class="section level5">
<h5><span class="header-section-number">2.2.3.3.1</span> Rationale</h5>
<p>The earliest attempt found for indexing AC can be found in a study published by Lee <em>et al</em><span class="citation">(<em>55</em>)</span> where they were interested in the ‘apical index’ (A.I.) of bottle cells during <em>X.laevis</em> gastrulation(figure <a href="2-mat-met.html#fig:ACLee">2.16</a> Lee). Another example for measuring AC is the apical <em>constriction</em> index (ACI, figure <a href="2-mat-met.html#fig:ACLee">2.16</a> Harding) for the cells of the <em>D.rerio</em> lateral line primordium (pLLP), which can be found in a study from 2012 where it was shown that Fgfr-Ras-MAPK signaling is required for AC and Rock2a localization <span class="citation">(<em>56</em>, <em>57</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:ACLee"></span>
<img src="figures/materials/models/ai.png" alt="A.I. indeces in the literature. Lee A.I. of X.leavis bottle cells measured in 2D. Harding A.I. of D.r. pLLP cells measured in 3D." width="75%" />
<p class="caption">
Figure 2.16: A.I. indeces in the literature. <strong>Lee</strong> A.I. of <em>X.leavis</em> bottle cells measured in 2D. <strong>Harding</strong> A.I. of <em>D.r.</em> pLLP cells measured in 3D.
</p>
</div>
<p>In these two publications, the way they measure A.I. <span class="citation">(<em>55</em>)</span> and ACI <span class="citation">(<em>57</em>)</span> respectively, does not differ and is the ratio of lateral height over apical width.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{apical\;width\;[\mu m]}\]</span></p>
<p>We found two principal weaknesses of applying this ratio to the cells of the PLLp. First, it does not respect the independence of lateral height to AC. Second, it does not differentiate between <em>anterio-posterior</em> (AP) and <em>dorso-ventral</em> (DV) tissue polarization. Third, it actually represents the A.I. rather than the apical <em>constriction</em> index.</p>
<div id="ACI-param" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.1</span> Parameter definition</h6>
<p>To obtain a precise and biologically meaningful quantity for AC, first a couple of definitions had to be made.</p>
<ol style="list-style-type: decimal">
<li>AC is independent of orientation and polarization
<ul>
<li>Cells may be polarized along the AP axis</li>
<li>Cells may also be polarized along the DV axis</li>
</ul></li>
<li>AC is independent of lateral height
<ul>
<li>Lateral height can be described as the distance of the two farthest points on the surface area of a cell</li>
<li>Two cells with different lateral heights can be equally apically constricted</li>
</ul></li>
<li>AC is independent of cell size
<ul>
<li>The size of a cell is its volume</li>
<li>Two cells of different volume can be equally apically constricted</li>
</ul></li>
</ol>
</div>
<div id="ACI-lat" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.2</span> Adaption for variation in lateral height</h6>
<p>To test different A.I. conditions, an apically constricted cell can be approximated by modeling a tetrahedron. For example, shrinking or enlarging a cell symmetrically should not affect the ACI. As described by Harding(2014)<span class="citation">(<em>57</em>)</span>, the <em>apical width</em> of a cell is measured first by manual 3-D reconstruction, second manual re-orientation, and third by going 1 <span class="math inline">\(\mu\)</span>m from the apical tip into the cell (from now on referred to as <span class="math inline">\(\Delta\)</span>ap, <a href="2-mat-met.html#fig:ACICells">2.17</a>B). Finally, <em>apical width</em> is the total width of the 2-D object in the respective Z-stack plane.</p>
<p>If <span class="math inline">\(\Delta\)</span>ap is a constant, the A.I. in a symmetrically enlarged cell increases from e.g. ~15 to ~23, since <em>apical width</em> stays the same but lateral height increases. On the contrary, if <span class="math inline">\(\Delta\)</span>ap is adjusted relative to a cells lateral height, e.g. by percentage, the A.I. in a symmetrically enlarged cell stays the same (figure <a href="2-mat-met.html#fig:ACICells">2.17</a>).</p>
(ref:ACICells) A.I. model <strong>A-A’’</strong> A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta was adjusted relative to the lateral height. <strong>B</strong> Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2D area as seen at delta ap of 1 or 2.5 <span class="math inline">\(\mu\)</span>m.
<div class="figure" style="text-align: center"><span id="fig:ACICells"></span>
<img src="figures/summary/aci_fig-01.png" alt="(ref:ACICells)" width="75%" />
<p class="caption">
Figure 2.17: (ref:ACICells)
</p>
</div>
<p>Therefore the measurement for apical width has to be relative to lateral height.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{relative\;apical\;width\;[\mu m]}\]</span></p>
</div>
<div id="ACI-pol" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.3</span> Adaption for tissue polarization</h6>
<p>Organs develop in a 3-D space and are polarized along each axis. AC usually describes a 2-D morphogenetic movement towards a center along the X-Y axes. However, the contraction movements along X and Y might be independent of one another. This could mean that they happen at different speeds, or that one is absent. As a result, the tissue would look less radially (figure <a href="2-mat-met.html#fig:cellpol">2.18</a>B) constricted, but more constricted along one particular axis (anisotropic). In order to separate those two AC dimensions, the A.I. can be calculated for the <em>anterio-posterior</em> and for the <em>dorso-ventral</em> axis (figure <a href="2-mat-met.html#fig:cellpol">2.18</a>, horizontal vs. vertical).</p>
(ref:cellpol) Schematic AC along the A-P and D-V axis. <strong>A</strong> shows a A-P and D-V constricted cluster of cells. <strong>B</strong> shows a D-V constricted cluster of cells.
<div class="figure" style="text-align: center"><span id="fig:cellpol"></span>
<img src="figures/materials/models/ACI_Cells_pol.png" alt="(ref:cellpol)" width="75%" />
<p class="caption">
Figure 2.18: (ref:cellpol)
</p>
</div>
<p>By fitting an ellipsoid to the volume area taken at <span class="math inline">\(\mathrm{\Delta}\)</span>ap, one will obtain the following parameters (figure <a href="2-mat-met.html#fig:ellipse">2.19</a>).</p>
<ol style="list-style-type: decimal">
<li>Length of Major axis
<ul>
<li>constituting <em>apical width</em></li>
</ul></li>
<li>Length of Minor axis
<ul>
<li>constituting <em>apical height</em></li>
</ul></li>
<li>Angle of Major from 0<span class="math inline">\(^\circ\)</span>
<ul>
<li>indicative for orientation of <em>lateral height</em>: If the angle is close to 0<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the AP axis (the cell is constricted along the DV axis). If the angle is close to 90<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the DV axis (the cell is constricted along the AP axis).</li>
</ul></li>
</ol>
(ref:ellipse) Scheme of ellipsoid measures. <strong>A</strong> shows the major axis as apical width and the minor axis as apical height. <strong>B</strong> shows the angular displacement from the horizon in steps of 30<span class="math inline">\(^\circ\)</span>.
<div class="figure" style="text-align: center"><span id="fig:ellipse"></span>
<img src="figures/materials/models/ellipse.png" alt="(ref:ellipse)" width="75%" />
<p class="caption">
Figure 2.19: (ref:ellipse)
</p>
</div>
<p>The two dimensions of AC indices can therefore be described as the following ratios…</p>
<p><span class="math display">\[\mathbf{A.I._{Major}} = \frac{lateral\;height\;[\mu m]}{Major\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{A.I._{Minor}} = \frac{lateral\;height\;[\mu m]}{Minor\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{Angle_{Major}} = \measuredangle = \mathrm{\Delta}\;from\;horizont\;[0-90^\circ]\]</span></p>
</div>
</div>
<div id="ACI-Dis" class="section level5">
<h5><span class="header-section-number">2.2.3.3.2</span> Measurements</h5>
<div id="ACI-singlecell" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.1</span> Single Cell measurements</h6>
<p>Each geometric object has a centroid coordinate in X and Y (and Z) which is represented as the mean of all X or Y coordinates within the object. Here, the centroid coordinates in X and Y are used to plot the cells as points in the X-Y plane. Additionally, each point is colored for the A.I. value (high values are red - dark red, middle values are green, low values are cyan - blue).</p>
(ref:ACIMinor) A.C.I.~Major / Minor~ single cell measurements
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-5"></span>
<img src="thesis_files/figure-html/unnamed-chunk-5-1.png" alt="(ref:ACIMinor)" width="85%" />
<p class="caption">
Figure 2.20: (ref:ACIMinor)
</p>
</div>
<p>Harding(2013)<span class="citation">(<em>57</em>)</span> were using a constant <span class="math inline">\(\mathrm{\Delta}\)</span>ap to measure the apical width, which we have shown to be incorrect in certain cases. In their study they found that certain mean A.I. values in the DMSO go as high as 15, which might be related to this <a href="2-mat-met.html#fig:ACICells">2.17</a>.
By measuring apical width at a relative <span class="math inline">\(\mathrm{\Delta}\)</span>ap and taking into account all pLLP cells these two exemplary pLLPs show a mean difference in the Major of 1.11 and 0.53 in the Minor (see table @ref(tab:aci_summary)).</p>
(ref:HardingACI) AI indices by Harding et al. <strong>E-G’</strong> 3-D reconstructions of the highlighted cell. <strong>H</strong> A.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of <em>hsp70:dn-Ras</em>. (n = 180 cells / N = 6 embryos).
<div class="figure" style="text-align: center"><span id="fig:HardingACI"></span>
<img src="figures/materials/models/harding.png" alt="(ref:HardingACI)" width="60%" />
<p class="caption">
Figure 2.21: (ref:HardingACI)
</p>
</div>
</div>
<div id="ACI-Angledens" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.2</span> Angle densities</h6>
<p>To check whether there is a bias in orientation of the apical width, the angle measurements <a href="2-mat-met.html#fig:ellipse">2.19</a> can be shown as a function of density along X. Here, a binwidth of 2.5 and a grouping factor to color the two conditions ‘DMSO’ and ‘SU5402’ was used.</p>
<p>Interestingly the results indicate that there is less of a difference for the Major<sub>Angle</sub> at angles bigger than 15-20<span class="math inline">\(^\circ\)</span>. This would mean that the <em>apical width</em> in SU5402 treated embryos is more strongly oriented along the horizontal <em>antero-posterior</em> axis.</p>
</div>
<div id="ACI-mag" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.3</span> ACI magnitude at different Angles</h6>
<p>Now, to get an idea of the magnitude ACI<sub>Major</sub> and ACI<sub>Minor</sub> have at each angle, each can be shown as a function of the Major<sub>Angle</sub> over ACI (while Major<sub>Angle</sub> is the same for both ACIs). Again, the grouping factor ‘GT’ was used to color the two conditions ‘DMSO’ and ‘SU5402’.</p>
<p>Since AC is a 3-D morphogenetic process and since cells in a wild type PLLp are mostly radially organized, it does make sense to try to look at AC from more than just one perspective. Here we propose to separate the ACI into an <em>antero-posterior</em> and a <em>dorso-ventral</em> dimension.</p>
<ol style="list-style-type: decimal">
<li><p>Interestingly there does not seem to be much of a difference in ACI<sub>Minor</sub>, which can also be shown by the mean values which are at 2.6 for the DMSO control and at 2.1 for the SU5402 treated condition. This indicates that there is no significant difference in contraction along the <em>dorso-ventral</em> axis.</p></li>
<li><p>For the ACI<sub>Major</sub> the base constriction for both, DMSO and SU5402 is at around 3.6, however there is a peak at around 40-60° in the DMSO control where cells are most constricted having a maximum ACI at 15.8. This indicates that cells coming from 35.7° are most constricted in the <em>anterior-posterior</em> axis.</p></li>
</ol>
(ref:angletoACI) ACI<sub>Major</sub> / ACI<sub>Minor</sub> over Major<sub>Angle</sub>
<div class="figure" style="text-align: center"><span id="fig:angletoACI"></span>
<img src="thesis_files/figure-html/angletoACI-1.png" alt="(ref:angletoACI)" width="49%" /><img src="thesis_files/figure-html/angletoACI-2.png" alt="(ref:angletoACI)" width="49%" />
<p class="caption">
Figure 2.22: (ref:angletoACI)
</p>
</div>
</div>
</div>
</div>
<div id="timelapse-analysis-1" class="section level4">
<h4><span class="header-section-number">2.2.3.4</span> Timelapse analysis</h4>
<p>ImageJ Macro</p>
</div>
</div>
</div>
<div id="mat-datasets" class="section level2">
<h2><span class="header-section-number">2.3</span> Image Data Sets</h2>
<p>Summaries of Image datasets</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatcc">Table 2.21: </span>Cell Cluster dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
60 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="2">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Fixation
</td>
<td style="text-align:left;">
4<span class="math inline">\(\%\)</span> PFA o.N.
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
1<span class="math inline">\(\%\)</span> LMPA
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
25X WI
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 12*X large image
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatai">Table 2.22: </span>AI dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.5<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
stamp v4A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
40X objective
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 1x1; Gain 1; Exposure 100 ms
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (100<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
0.4 <span class="math inline">\(\mu\)</span>m Z-spacing
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatprol">Table 2.23: </span>Proliferation dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; cxcr4b(BAC):H2BRFP +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v4A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="6">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 35 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
651 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 7 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatdet">Table 2.24: </span>Detection dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v5A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="6">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 20 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
3 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 3*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 10 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatato">Table 2.25: </span>Atoh1a dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; atoh1a:Tom +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
190930 D.S. Kleinhans
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp v5A
</td>
<td style="text-align:left;width: 2.5cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="7">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (Int: 20<span class="math inline">\(\%\)</span>; Exp: 25 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
561 nm (Int: 30<span class="math inline">\(\%\)</span>; Exp: 50 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 20 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-Intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-res.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
