<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Genetic and Mechanic Signaling in Organ Development</title>
  <meta name="description" content="Chapter 2 Materials and Methods | Genetic and Mechanic Signaling in Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Genetic and Mechanic Signaling in Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Genetic and Mechanic Signaling in Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-Intro.html"/>
<link rel="next" href="3-res.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { font-weight: bold; } /* Alert */
code span.an { font-style: italic; } /* Annotation */
code span.cf { font-weight: bold; } /* ControlFlow */
code span.co { font-style: italic; } /* Comment */
code span.cv { font-style: italic; } /* CommentVar */
code span.do { font-style: italic; } /* Documentation */
code span.dt { text-decoration: underline; } /* DataType */
code span.er { font-weight: bold; } /* Error */
code span.in { font-style: italic; } /* Information */
code span.kw { font-weight: bold; } /* Keyword */
code span.pp { font-weight: bold; } /* Preprocessor */
code span.wa { font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html"><i class="fa fa-check"></i>Preliminary Content</a><ul>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#about"><i class="fa fa-check"></i>About</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#foreword"><i class="fa fa-check"></i>Foreword</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abbreviations"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#intro-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#cell-signaling"><i class="fa fa-check"></i><b>1.1.2</b> Cell Signaling</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#intro-morph"><i class="fa fa-check"></i><b>1.1.3</b> Morphogenesis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#open-questions-and-motivation"><i class="fa fa-check"></i><b>1.3</b> Open Questions and Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mos"><i class="fa fa-check"></i><b>2.1.9</b> Morpholinos</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.10</b> Hardware</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.11</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Data Strategy and Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.2</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.3</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.4</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.3</b> Ground Truth</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cluster-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Cluster Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#morphometric-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Morphometric analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.4</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#res-met"><i class="fa fa-check"></i><b>3.1</b> Quantitative imaging and image analysis</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#res-mount"><i class="fa fa-check"></i><b>3.1.1</b> A 3D-printed stamp to standardize sample mounting and semi-automatize imaging</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#mat-anallzr2d"><i class="fa fa-check"></i><b>3.1.2</b> anaLLzr2D - Automated 2D neuromast analysis and nuclei count</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#mat-anallzr2dt"><i class="fa fa-check"></i><b>3.1.3</b> anaLLzr2DT - Automated 2D neuromast analysis and nuclei count through time</a></li>
<li class="chapter" data-level="3.1.4" data-path="3-res.html"><a href="3-res.html#mat-anallzr3d"><i class="fa fa-check"></i><b>3.1.4</b> anaLLzr3D - Automated 3D single cell segmentation and A.I. analysis in the pLLP</a></li>
<li class="chapter" data-level="3.1.5" data-path="3-res.html"><a href="3-res.html#CNN"><i class="fa fa-check"></i><b>3.1.5</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#intro-phen"><i class="fa fa-check"></i><b>3.2.1</b> Phenotype description</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.3</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.4</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#apical-constriction-3"><i class="fa fa-check"></i><b>3.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.6</b> Haircell Specification</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-discussion.html"><a href="4-discussion.html"><i class="fa fa-check"></i><b>4</b> Discussion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-discussion.html"><a href="4-discussion.html#methods"><i class="fa fa-check"></i><b>4.1</b> Methods</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-discussion.html"><a href="4-discussion.html#standardized-mounting"><i class="fa fa-check"></i><b>4.1.1</b> Standardized mounting</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-discussion.html"><a href="4-discussion.html#automated-image-analysis"><i class="fa fa-check"></i><b>4.1.2</b> Automated image analysis</a></li>
<li class="chapter" data-level="4.1.3" data-path="4-discussion.html"><a href="4-discussion.html#rosette-detection-1"><i class="fa fa-check"></i><b>4.1.3</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-discussion.html"><a href="4-discussion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-discussion.html"><a href="4-discussion.html#lateral-line"><i class="fa fa-check"></i><b>4.2.1</b> Lateral Line</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-discussion.html"><a href="4-discussion.html#rosette-formation-1"><i class="fa fa-check"></i><b>4.2.2</b> Rosette Formation</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-discussion.html"><a href="4-discussion.html#apical-constriction-4"><i class="fa fa-check"></i><b>4.2.3</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-discussion.html"><a href="4-discussion.html#hair-cell-specification"><i class="fa fa-check"></i><b>4.2.4</b> Hair Cell specification</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-discussion.html"><a href="4-discussion.html#model-2"><i class="fa fa-check"></i><b>4.2.5</b> Model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="appendix-appendix.html"><a href="appendix-appendix.html"><i class="fa fa-check"></i>(Appendix) Appendix</a></li>
<li class="chapter" data-level="" data-path="supplement.html"><a href="supplement.html"><i class="fa fa-check"></i>Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Genetic and Mechanic Signaling in Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mat-met" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="mat" class="section level2">
<h2><span class="header-section-number">2.1</span> Materials</h2>
<div id="mat-chem" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Chemicals</h3>
<table>
<caption><span id="tab:met-chem">Table 2.1: </span> Chemicals</caption>
<colgroup>
<col width="31%" />
<col width="36%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Chemical</th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Agarose</td>
<td align="center">Roth</td>
<td align="left">6351.2</td>
</tr>
<tr class="even">
<td align="right">Agar-Agar</td>
<td align="center">Roth</td>
<td align="left">5210.3</td>
</tr>
<tr class="odd">
<td align="right">Ampicillin</td>
<td align="center">Roth</td>
<td align="left">K029.2</td>
</tr>
<tr class="even">
<td align="right">ATP</td>
<td align="center">Epicentre</td>
<td align="left">E311K</td>
</tr>
<tr class="odd">
<td align="right">Blocking Reagent</td>
<td align="center">Roche</td>
<td align="left">11096176001</td>
</tr>
<tr class="even">
<td align="right">BCIP</td>
<td align="center">Fermentas</td>
<td align="left">R0822</td>
</tr>
<tr class="odd">
<td align="right">CaCl</td>
<td align="center">Roth</td>
<td align="left">886.1</td>
</tr>
<tr class="even">
<td align="right">Calyculin</td>
<td align="center">Sigma</td>
<td align="left">208851</td>
</tr>
<tr class="odd">
<td align="right">DAPI</td>
<td align="center">Sigma</td>
<td align="left">D9542</td>
</tr>
<tr class="even">
<td align="right">DIG RNA Mix</td>
<td align="center">Roche</td>
<td align="left">11277073910</td>
</tr>
<tr class="odd">
<td align="right">DMSO</td>
<td align="center">Roth</td>
<td align="left">4720.2</td>
</tr>
<tr class="even">
<td align="right">EtBr</td>
<td align="center">Roth</td>
<td align="left">2218.3</td>
</tr>
<tr class="odd">
<td align="right">EtOH</td>
<td align="center">Roth</td>
<td align="left">9065.3</td>
</tr>
<tr class="even">
<td align="right">Formaldehyde</td>
<td align="center">Roth</td>
<td align="left">7398.1</td>
</tr>
<tr class="odd">
<td align="right">Formamide</td>
<td align="center">Roth</td>
<td align="left">P040.1</td>
</tr>
<tr class="even">
<td align="right">Glycerol</td>
<td align="center">Roth</td>
<td align="left">3783.2</td>
</tr>
<tr class="odd">
<td align="right">IPTG</td>
<td align="center">Thermo</td>
<td align="left">R1171</td>
</tr>
<tr class="even">
<td align="right">KCl</td>
<td align="center">Roth</td>
<td align="left">P017.1</td>
</tr>
<tr class="odd">
<td align="right">Low melting point Agarose</td>
<td align="center">Roth</td>
<td align="left">A9539</td>
</tr>
<tr class="even">
<td align="right">Maleic Acid</td>
<td align="center">Roth</td>
<td align="left">3810.3</td>
</tr>
<tr class="odd">
<td align="right">MgSO</td>
<td align="center">Roth</td>
<td align="left">T888.2</td>
</tr>
<tr class="even">
<td align="right">MeOH</td>
<td align="center">Roth</td>
<td align="left">CP43.3</td>
</tr>
<tr class="odd">
<td align="right">MgCl</td>
<td align="center">Roth</td>
<td align="left">2189.1</td>
</tr>
<tr class="even">
<td align="right">NaCl</td>
<td align="center">Roth</td>
<td align="left">9265.2</td>
</tr>
<tr class="odd">
<td align="right">NaHCO</td>
<td align="center">Roth</td>
<td align="left">855.1</td>
</tr>
<tr class="even">
<td align="right">NaOH</td>
<td align="center">Roth</td>
<td align="left">6771.3</td>
</tr>
<tr class="odd">
<td align="right">NGS</td>
<td align="center">Sigma</td>
<td align="left">C6767</td>
</tr>
<tr class="even">
<td align="right"><em>p</em>-Formaldehyd</td>
<td align="center">Sigma</td>
<td align="left">P6148</td>
</tr>
<tr class="odd">
<td align="right">Phenol Red</td>
<td align="center">Sigma</td>
<td align="left">P0290</td>
</tr>
<tr class="even">
<td align="right">Propan-2-ol</td>
<td align="center">VWR</td>
<td align="left">20842330</td>
</tr>
<tr class="odd">
<td align="right">Proteinase K</td>
<td align="center">Roth</td>
<td align="left">7528.4</td>
</tr>
<tr class="even">
<td align="right">PTU</td>
<td align="center">Sigma</td>
<td align="left">P7629</td>
</tr>
<tr class="odd">
<td align="right">Rockout</td>
<td align="center">Sigma</td>
<td align="left">555553</td>
</tr>
<tr class="even">
<td align="right">SSC</td>
<td align="center">Roth</td>
<td align="left">1232.1</td>
</tr>
<tr class="odd">
<td align="right">SU5402</td>
<td align="center">CALBIOCHEM</td>
<td align="left">572630</td>
</tr>
<tr class="even">
<td align="right">Torula RNA</td>
<td align="center">Sigma</td>
<td align="left">R6625</td>
</tr>
<tr class="odd">
<td align="right">Tricaine</td>
<td align="center">Sigma</td>
<td align="left">A5040</td>
</tr>
<tr class="even">
<td align="right">Tris Base</td>
<td align="center">Roth</td>
<td align="left">4855.2</td>
</tr>
<tr class="odd">
<td align="right">Triton-X100</td>
<td align="center">Roth</td>
<td align="left">3051.2</td>
</tr>
<tr class="even">
<td align="right">Trizol</td>
<td align="center">Ambion</td>
<td align="left">15596018</td>
</tr>
<tr class="odd">
<td align="right">Tween20</td>
<td align="center">Sigma</td>
<td align="left">P1379</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-sol" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Solutions</h3>
<table>
<caption><span id="tab:mat-sol">Table 2.2: </span> Solutions</caption>
<colgroup>
<col width="31%" />
<col width="36%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Solution</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Cut Smart Buffer</td>
<td align="center">NEB</td>
<td align="left">B7204S</td>
</tr>
<tr class="even">
<td align="right">Generuler 100 bp</td>
<td align="center">Thermo</td>
<td align="left">SM0241</td>
</tr>
<tr class="odd">
<td align="right">Generuler 1kb</td>
<td align="center">Thermo</td>
<td align="left">SM0311</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-anitb" class="section level3">
<h3><span class="header-section-number">2.1.3</span> Antibodies</h3>
<table>
<caption><span id="tab:mat-antib">Table 2.3: </span> Antibodies</caption>
<colgroup>
<col width="23%" />
<col width="38%" />
<col width="21%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Antibody</strong></th>
<th align="center">Company / Provider</th>
<th align="center">concentration</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Anti-Digoxigenin</td>
<td align="center">Roche</td>
<td align="center">1:200</td>
<td align="left">11093274910</td>
</tr>
<tr class="even">
<td align="right">Anti-GFP</td>
<td align="center">Torrey Pines</td>
<td align="center">1:200</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">Anti-TAZ (rabbit)</td>
<td align="center">Cell Signaling Technology</td>
<td align="center">1:200</td>
<td align="left">D24E4</td>
</tr>
<tr class="even">
<td align="right">Anti-ZO1</td>
<td align="center">Zymed</td>
<td align="center">1:200</td>
<td align="left">33-9100</td>
</tr>
<tr class="odd">
<td align="right">Alexa Fluor488</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">710369</td>
</tr>
<tr class="even">
<td align="right">Alexa Fluor555</td>
<td align="center">Invitrogen</td>
<td align="center">1:500</td>
<td align="left">Z25005</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-enz" class="section level3">
<h3><span class="header-section-number">2.1.4</span> Enzymes</h3>
<table>
<caption><span id="tab:mat-enz">Table 2.4: </span> Enzymes</caption>
<colgroup>
<col width="31%" />
<col width="36%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Enzyme</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">BtsCI</td>
<td align="center">NEB</td>
<td align="left">R0647</td>
</tr>
<tr class="even">
<td align="right">DdeI</td>
<td align="center">NEB</td>
<td align="left">R0175</td>
</tr>
<tr class="odd">
<td align="right">HaeIII</td>
<td align="center">NEB</td>
<td align="left">R0108</td>
</tr>
<tr class="even">
<td align="right">MnlII</td>
<td align="center">NEB</td>
<td align="left">R0163</td>
</tr>
<tr class="odd">
<td align="right">NlaIII</td>
<td align="center">NEB</td>
<td align="left">R0125</td>
</tr>
<tr class="even">
<td align="right">NsiI-HF</td>
<td align="center">NEB</td>
<td align="left">R3127</td>
</tr>
<tr class="odd">
<td align="right">Phusion Polymerase</td>
<td align="center">NEB</td>
<td align="left">M0530L</td>
</tr>
<tr class="even">
<td align="right">Pronase</td>
<td align="center">Sigma</td>
<td align="left">P5147</td>
</tr>
<tr class="odd">
<td align="right">Ribolock</td>
<td align="center">Thermo</td>
<td align="left">EO0381</td>
</tr>
<tr class="even">
<td align="right">RNase A</td>
<td align="center">Quiagen</td>
<td align="left">1006657</td>
</tr>
<tr class="odd">
<td align="right">RNase H</td>
<td align="center">NEB</td>
<td align="left">M0297L</td>
</tr>
<tr class="even">
<td align="right">SP6 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0131</td>
</tr>
<tr class="odd">
<td align="right">T4 Ligase</td>
<td align="center">NEB</td>
<td align="left">M0202T</td>
</tr>
<tr class="even">
<td align="right">T7 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">EP0111</td>
</tr>
<tr class="odd">
<td align="right">Taq DNA Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">10342-020</td>
</tr>
<tr class="even">
<td align="right">Taq DNA Polymerase</td>
<td align="center">VWR</td>
<td align="left">733-1301</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mobikits" class="section level3">
<h3><span class="header-section-number">2.1.5</span> Molecular Biology Kits</h3>
<table>
<caption><span id="tab:mat-mobikits">Table 2.5: </span> Molecular Biology Kits</caption>
<colgroup>
<col width="54%" />
<col width="28%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Kit</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">EdU Click-iT</td>
<td align="center">Invitrogen</td>
<td align="left">MP 10083</td>
</tr>
<tr class="even">
<td align="right">mMessage mMachine Sp6 Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">AM1340</td>
</tr>
<tr class="odd">
<td align="right">PCR &amp; Gel Clean-Up</td>
<td align="center">Sigma</td>
<td align="left">NA1020</td>
</tr>
<tr class="even">
<td align="right">pGEM-T TA Cloning</td>
<td align="center">Promega</td>
<td align="left">A3600</td>
</tr>
<tr class="odd">
<td align="right">Superscript III cDNA Synthesis</td>
<td align="center">Thermo</td>
<td align="left">18080051</td>
</tr>
<tr class="even">
<td align="right">Wizard SV Gel and PCR Clean-Up</td>
<td align="center">Promega</td>
<td align="left">A9282</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-buff" class="section level3">
<h3><span class="header-section-number">2.1.6</span> Buffers</h3>
<table>
<caption><span id="tab:mat-buff">Table 2.6: </span> Buffers</caption>
<colgroup>
<col width="24%" />
<col width="75%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Buffer</strong></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Blocking Reagent (BR)</td>
<td>2% BR in maleic buffer + 5% serum</td>
</tr>
<tr class="even">
<td align="left">E3</td>
<td><span class="citation">(<em>52</em>)</span></td>
</tr>
<tr class="odd">
<td align="left">Hybridization buffer</td>
<td>50% Formamide + 25 % 20x SSC + 50mg/mL Heparine + mQ</td>
</tr>
<tr class="even">
<td align="left">Maleic buffer</td>
<td>250 mM maleic acid + 5M NaCl + 10% 0.1% Tween-20 + mQ</td>
</tr>
<tr class="odd">
<td align="left">NTMT</td>
<td>5 M NaCl + 1 M MgCl + 1 M Tris pH 9.5 + 10% Tween</td>
</tr>
<tr class="even">
<td align="left">PBS</td>
<td>2.7 mM KCl + 12 mM HPO</td>
</tr>
<tr class="odd">
<td align="left">PBST</td>
<td>PBS + 0.1% Tween20</td>
</tr>
<tr class="even">
<td align="left">PBDT</td>
<td>PBS + 1% BSA + 1% DMSO + 0.3% Triton</td>
</tr>
<tr class="odd">
<td align="left">PFA</td>
<td>4% paraformaldehyde in PBS</td>
</tr>
<tr class="even">
<td align="left">P1</td>
<td>50 mM Tris-HCl pH 8.0 + 10 mM EDTA pH 8.0 + 100 µg/mL RNAse</td>
</tr>
<tr class="odd">
<td align="left">P2</td>
<td>1M NaOH + 10 % (w/v) SDS</td>
</tr>
<tr class="even">
<td align="left">P3</td>
<td>3M KOAc pH 5.5</td>
</tr>
<tr class="odd">
<td align="left">TNT</td>
<td>50 mM Tris-HCl pH 8.0 + 100 mM NaCl + 0.1% Tween-20</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-lines" class="section level3">
<h3><span class="header-section-number">2.1.7</span> Zebrafish lines</h3>
<table>
<caption><span id="tab:mat-lines">Table 2.7: </span> Zebrafish lines</caption>
<colgroup>
<col width="13%" />
<col width="31%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Allele</strong></th>
<th align="center"><strong>name</strong></th>
<th align="left"><strong>zfin</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">zf106Tg</td>
<td align="center"><em>cldnb:lyn-gfp</em></td>
<td align="left"><a href="//zfin.org/ZDB-ALT-060919-2"><em>Tg(-8.0cldnb:LY-EGFP)</em></a></td>
</tr>
<tr class="even">
<td align="right">fu13Tg</td>
<td align="center"><em>cxcr4b</em>(BAC):<em>H2BRFP</em></td>
<td align="left"><em>TgBAC(cxcr4b:Hsa.HIST1H2BJ-RFP)</em></td>
</tr>
<tr class="odd">
<td align="right">nns8Tg</td>
<td align="center"><em>atoh1a:Tom</em></td>
<td align="left"><a href="//zfin.org/ZDB-FISH-150901-21622"><em>Tg(atoh1a:dTomato)nns8</em></a></td>
</tr>
<tr class="even">
<td align="right">fu50</td>
<td align="center"><em>shroom3</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">m1274Tg</td>
<td align="center"><em>hsp70:shr3v1FL-taqRFP</em></td>
<td align="left"><a href="//zfin.org/ZDB-FISH-150901-25907"><em>Tg1(hsp70l:shroom3-TagRFP)</em></a></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-probes" class="section level3">
<h3><span class="header-section-number">2.1.8</span> ISH probes</h3>
<table>
<caption><span id="tab:mat-probes">Table 2.8: </span> ISH probes</caption>
<colgroup>
<col width="11%" />
<col width="88%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">atoh1a</td>
<td align="left">see <span class="citation">(<em>38</em>)</span></td>
</tr>
<tr class="even">
<td align="left">deltaD</td>
<td align="left">see <span class="citation">(<em>53</em>)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mos" class="section level3">
<h3><span class="header-section-number">2.1.9</span> Morpholinos</h3>
<table>
<caption><span id="tab:mat-mos">Table 2.9: </span> Morpholinos</caption>
<colgroup>
<col width="11%" />
<col width="32%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
<th align="left"><strong>Concentration</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">MoAtoh1a</td>
<td align="left">see <span class="citation">(<em>38</em>, <em>54</em>)</span></td>
<td align="left">0.4 ng/mM</td>
</tr>
<tr class="even">
<td align="left">p53</td>
<td align="left">see <span class="citation">(<em>55</em>)</span></td>
<td align="left">2 ng/mM</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-hrdwr" class="section level3">
<h3><span class="header-section-number">2.1.10</span> Hardware</h3>
<div id="mat-stamp" class="section level4">
<h4><span class="header-section-number">2.1.10.1</span> Mounting Stamp</h4>
<p>An stl file for 3D printing can be found at <a href="https://github.com/KleinhansDa/3DModels">github.com/KleinhansDa/3DModels</a></p>
<table>
<caption><span id="tab:mat-mount">Table 2.10: </span> Materials for production of a standardized mounting stamp</caption>
<colgroup>
<col width="54%" />
<col width="28%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right"><span class="math inline">\(\mu\)</span> dish</td>
<td align="center">Ibidi</td>
<td align="left">81,218–200</td>
</tr>
<tr class="even">
<td align="right">Stamp</td>
<td align="center">-</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">Preparation needles</td>
<td align="center">VWR</td>
<td align="left">USBE5470</td>
</tr>
<tr class="even">
<td align="right">Pasteur Pipettes</td>
<td align="center">Roth</td>
<td align="left">4518</td>
</tr>
<tr class="odd">
<td align="right">Rubber / Silicone bulb</td>
<td align="center">VWR</td>
<td align="left">612-2327</td>
</tr>
<tr class="even">
<td align="right">Microtubes 2 mL</td>
<td align="center">Sarstedt</td>
<td align="left">2691</td>
</tr>
<tr class="odd">
<td align="right">Heating block</td>
<td align="center">PeqLab</td>
<td align="left">HX2</td>
</tr>
<tr class="even">
<td align="right">Microwave oven</td>
<td align="center">Severin</td>
<td align="left">MW7849</td>
</tr>
<tr class="odd">
<td align="right">Stereo microscope</td>
<td align="center">Leica</td>
<td align="left">M165FC</td>
</tr>
<tr class="even">
<td align="right">Transmitted Light Base</td>
<td align="center">Leica</td>
<td align="left">MDG36</td>
</tr>
<tr class="odd">
<td align="right">Countersunk screw DIN7991, 8 × 20 mm</td>
<td align="center">Dresselhaus (Hornbach)</td>
<td align="left">7662389</td>
</tr>
<tr class="even">
<td align="right">Superglue</td>
<td align="center">UHU</td>
<td align="left">509141</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-SD" class="section level4">
<h4><span class="header-section-number">2.1.10.2</span> Spinning Disc Microscopy</h4>
<table>
<caption><span id="tab:SDcomp">Table 2.11: </span> Spinning Disc system components</caption>
<colgroup>
<col width="22%" />
<col width="18%" />
<col width="25%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Microscope</td>
<td align="center">Nikon</td>
<td align="center">Eclipse Ti-E</td>
<td align="left">fully motorized</td>
</tr>
<tr class="even">
<td align="right">PFS</td>
<td align="center">Nikon</td>
<td align="center">Perfect focus system</td>
<td align="left">Z repositioning</td>
</tr>
<tr class="odd">
<td align="right">XY-table</td>
<td align="center">Merzhaeuser</td>
<td align="center">XY motorized table</td>
<td align="left">1 <span class="math inline">\(\mu\)</span>m accuracy</td>
</tr>
<tr class="even">
<td align="right">Piezo</td>
<td align="center"></td>
<td align="center">Piezo Z-table</td>
<td align="left">300 <span class="math inline">\(\mu\)</span>m scan range</td>
</tr>
<tr class="odd">
<td align="right">SD system</td>
<td align="center">Yokogawa</td>
<td align="center">CSU-W1</td>
<td align="left">50 <span class="math inline">\(\mu\)</span>m pattern</td>
</tr>
<tr class="even">
<td align="right">Laser</td>
<td align="center"></td>
<td align="center">Laser Combiner</td>
<td align="left">see table <a href="2-mat-met.html#tab:lasers">2.12</a></td>
</tr>
<tr class="odd">
<td align="right">FRAPPA</td>
<td align="center">Revolution</td>
<td align="center">FRAPPA</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="right">Borealis</td>
<td align="center">Borealis</td>
<td align="center">Borealis</td>
<td align="left">flat field correction</td>
</tr>
<tr class="odd">
<td align="right">sCMOS</td>
<td align="center">Andor</td>
<td align="center">ZYLA PLUS</td>
<td align="left">4.2Mpix; 82%QE</td>
</tr>
<tr class="even">
<td align="right">Immersion</td>
<td align="center">Merzhaeuser</td>
<td align="center">Liquid Dispenser</td>
<td align="left">-</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:lasers">Table 2.12: </span> Available lasers</caption>
<colgroup>
<col width="40%" />
<col width="20%" />
<col width="40%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Lasers</strong></th>
<th align="center">Type</th>
<th align="left">Power</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">405 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">445 nm</td>
<td align="center">diode</td>
<td align="left">80 mW</td>
</tr>
<tr class="odd">
<td align="right">488 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">561 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="odd">
<td align="right">640 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:objectives">Table 2.13: </span> Available objectives</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="17%" />
<col width="15%" />
<col width="9%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Objective</strong></th>
<th align="center">Company</th>
<th align="center">Type</th>
<th align="center">Immersion</th>
<th align="center">N.A.</th>
<th align="left">working distance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">10x</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">air</td>
<td align="center">0.45</td>
<td align="left">4.00 mm</td>
</tr>
<tr class="even">
<td align="right">20X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">0.95</td>
<td align="left">0.95 mm</td>
</tr>
<tr class="odd">
<td align="right">40X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.15</td>
<td align="left">0.60 mm</td>
</tr>
<tr class="even">
<td align="right">60X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.20</td>
<td align="left">0.30 mm</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-work" class="section level4">
<h4><span class="header-section-number">2.1.10.3</span> Workstation</h4>
<p>Statistical computation and image analysis were done on a Fujitsu Siemens (FS) Workstation CELSIUSM740 with the following hardware components…</p>
<table>
<caption><span id="tab:PCcomp">Table 2.14: </span> Workstation hardware components</caption>
<colgroup>
<col width="21%" />
<col width="22%" />
<col width="25%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">CPU</td>
<td align="center">Intel</td>
<td align="center">Xeon E5-1660v4</td>
<td align="left">3.2 GHz, 20MB, 8cores</td>
</tr>
<tr class="even">
<td align="right">RAM</td>
<td align="center">Fujitsu</td>
<td align="center">-</td>
<td align="left">4x16GB DDR4-2400</td>
</tr>
<tr class="odd">
<td align="right">Graphics</td>
<td align="center">NVIDIA</td>
<td align="center">Quadro M4000</td>
<td align="left">8GB RAM</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="mat-sftwr" class="section level3">
<h3><span class="header-section-number">2.1.11</span> Software</h3>
<table>
<caption><span id="tab:sftwr">Table 2.15: </span> Used software</caption>
<colgroup>
<col width="39%" />
<col width="19%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Software</strong></th>
<th align="center">Version</th>
<th align="left">web</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Imagej FIJI</td>
<td align="center">1.48</td>
<td align="left"><a href="https://fiji.sc/" class="uri">https://fiji.sc/</a></td>
</tr>
<tr class="even">
<td align="right">R</td>
<td align="center">3.6.1</td>
<td align="left"><a href="https://cran.r-project.org/" class="uri">https://cran.r-project.org/</a></td>
</tr>
<tr class="odd">
<td align="right">RStudio</td>
<td align="center">1.0.153</td>
<td align="left"><a href="https://www.rstudio.com/" class="uri">https://www.rstudio.com/</a></td>
</tr>
<tr class="even">
<td align="right">Ubuntu</td>
<td align="center">17.1</td>
<td align="left"><a href="https://www.ubuntu.com/" class="uri">https://www.ubuntu.com/</a></td>
</tr>
<tr class="odd">
<td align="right">Windows 10 Pro</td>
<td align="center">10.0.16299</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="right">Total Commander</td>
<td align="center">9.0</td>
<td align="left"><a href="http://ghisler.com/" class="uri">http://ghisler.com/</a></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="met" class="section level2">
<h2><span class="header-section-number">2.2</span> Methods</h2>
<div id="data-strategy-and-analysis" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Data Strategy and Analysis</h3>
<p>Due to the history of Developmental Biology and the complexity of biological processes <em>per se</em>, the field heavily relies on image data. Since the advent of electronic imaging techniques<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> scientific image data can be processed and analyzed <em>in silico</em>. To make use of the possibility of</p>
<ol style="list-style-type: decimal">
<li>live imaging, which (as compared to fixation techniques) conserves the cellular integrity and morphology while also offers the possibility of recording time-lapses</li>
<li>the optically clear specimen and</li>
<li>high throughput image analysis and state-of-the-art data science using algorithmic implementations,</li>
</ol>
<p>the three following points were paid special attention to.</p>
<div id="sample-preparation" class="section level4">
<h4><span class="header-section-number">2.2.1.1</span> Sample Preparation</h4>
<p>For fluorescence microscopy zebrafish embryos are usually immersed in a 1<span class="math inline">\(\%\)</span> solution of <em>low melting-point agarose</em> (LMPA) and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a> eight to ten embryos <em>per</em> dish. To make use of the high number of offspring, a single zebrafish female may lay, which rapidly leads to a sample number of more than 300, a new sample preparation technique was designed that allows for (1) a four - to five - time increase in samples per dish (2) a facilitated navigation <em>via</em> a grid-like orientation through the samples and (3) an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope. For details, see Materials and Methods section <a href="2-mat-met.html#mount-met">2.2.3.4</a> and Kleinhans <em>et al.</em>, 2019<span class="citation">(<em>56</em>)</span>.</p>
</div>
<div id="imaging" class="section level4">
<h4><span class="header-section-number">2.2.1.2</span> Imaging</h4>
<p>Technically, speed and sensitivity are most important for live imaging. Considering these two parameters a light-sheet <span class="citation">(<em>57</em>)</span> fluorescence microscope (LSFM) would be the best fit. However, LSFMs also have several limitations. First, due to the sample preparation methods available the number of samples that can be imaged at a time is highly restricted. Second, for subcellular resolution a high magnification is required, which is limited by working distances and third - for optimal image analysis a high Signal-to-Noise ratio (SNR) and numerical aperture (N.A.) is preferable. Therefore a spinning disc <span class="citation">(<em>58</em>)</span> system was chosen for most of the imaging.
The system makes use of (1) an extra-large field of view (FOV) ideal for large specimen, (2) the possibility of a high degree of automation with state-of-the-art software and (3) a water dispensing system for long-term water immersion imaging. For details about the system see Materials section <a href="2-mat-met.html#mat-SD">2.1.10.2</a>.</p>
</div>
<div id="data-handling" class="section level4">
<h4><span class="header-section-number">2.2.1.3</span> Data handling</h4>
<p>After data acquisition and pre-processing, the image data was transferred from the microscope system to the labs main workstation. To uniquely identify each file and have them appear in a structured manner, a file-naming system was established after the following structure</p>

<p>Where <em>stage</em> would <em>e.g.</em> be 32hpf, <em>group</em> would be a genotype or drug treatment, <em>id</em> would be a positional identifier on an imaging dish like B1P01<a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a> and <em>date</em> would be a date in the form of YYMMDD.</p>
</div>
<div id="image-and-data-analysis" class="section level4">
<h4><span class="header-section-number">2.2.1.4</span> Image and Data Analysis</h4>
<p>In order to be as objective and as high throughput as possible, almost all of the analyses performed for this study was solved either algorithmically or using convolutional neural networks (CNNs). Furthermore, to meet the terms and conditions of <em>open science</em><a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a> standards, all pipelines were implemented in open source software frameworks such as <em>Fiji is just image J</em> (FIJI) and R. For further information about training datasets, algorithms and versions used see Materials section <a href="2-mat-met.html#mat-sftwr">2.1.11</a>.</p>
</div>
</div>
<div id="Zeb-met" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Zebrafish</h3>
<div id="husbandry" class="section level4">
<h4><span class="header-section-number">2.2.2.1</span> Husbandry</h4>
<p>Zebraﬁsh husbandry was maintained at the University of Frankfurt am Main. All legal procedures were followed while handling and maintaining zebraﬁsh husbandry. All zebraﬁsh lines used in and generated for this study are listed in Materials section <a href="2-mat-met.html#mat-lines">2.1.7</a>.</p>
</div>
<div id="handling-and-rearing" class="section level4">
<h4><span class="header-section-number">2.2.2.2</span> Handling and rearing</h4>
<p>In the afternoon preceding the embryo collection, 1 male and female were set up in crossing cages, physically separated by a transparent separator. Next day, before noon, separators were removed allowing fertilization. Fertilized eggs were then collected, sorted and reared in the well-defined culture medium E3 (Kimmel et.al. 1995, section <a href="2-mat-met.html#mat-buff">2.1.6</a>) at 25<span class="math inline">\(^\circ\)</span>, 28.5<span class="math inline">\(^\circ\)</span>, or 30<span class="math inline">\(^\circ\)</span> C depending on the experimental condition required.</p>
<p>To grow larvae to the adulthood, they were transferred to the system on day 5. Till day 12, larvae were fed Vinegar Eels, Paramecia, and Caviar powder. After the 12 th day, water supply was started and fish were fed Brine Shrimp, Artemia, Paramecia and Vinegar Eels. Adult fish (&gt;1 Month) were fed Artemia and the dry flakes.</p>
</div>
<div id="zebrafish-fin-clips" class="section level4">
<h4><span class="header-section-number">2.2.2.3</span> Zebrafish fin clips</h4>
<p>Adult fish were anesthetized with buffered Tricaine (1X, see section <a href="2-mat-met.html#mat-buff">2.1.6</a>) until loss of motion. About 1/3 of the caudal fin was cut with a sterile scalpel in a sterile Petri Dish. Immediately the cut fin was transferred to 100 <span class="math inline">\(\mu\)</span>L of 50 mM NaOH. Fish were returned to system water and kept in 1L system water in single tanks with 200 <span class="math inline">\(\mu\)</span>L of 0.01<span class="math inline">\(\%\)</span> Methylene Blue.</p>
</div>
<div id="adult-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.2.4</span> Adult Genotyping</h4>
<p>The clipped fins were digested for 1 h at 95<span class="math inline">\(^\circ\)</span> C and neutralized subsequently with 10 <span class="math inline">\(\mu\)</span>L of 1M Tris-HCl of (pH 9).</p>
</div>
<div id="embryo-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.2.5</span> Embryo Genotyping</h4>
<p>Single fixed/live embryos were denatured at 95<span class="math inline">\(^\circ\)</span> C in 20 <span class="math inline">\(\mu\)</span>L of 50 mM NaOH for 1 hour and neutralized by adding 2.5 <span class="math inline">\(\mu\)</span>L of 1 M Tris-HCl (pH 9).</p>
</div>
<div id="zebrafish-euthanasia" class="section level4">
<h4><span class="header-section-number">2.2.2.6</span> Zebrafish Euthanasia</h4>
<p>Adult zebrafish were euthanised by an overdose of Tricaine in ice cold water so as to sacrifice them by hypothermia.</p>
</div>
<div id="fixation" class="section level4">
<h4><span class="header-section-number">2.2.2.7</span> Fixation</h4>
<p>Embryos and dechorionated larvae were fixed in 2 ml of 4<span class="math inline">\(\%\)</span> PFA in 1X PBS overnight at 4<span class="math inline">\(^\circ\)</span> C.</p>
</div>
</div>
<div id="Wet-met" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Wet lab</h3>
<div id="sampleprep" class="section level4">
<h4><span class="header-section-number">2.2.3.1</span> Sample preparation</h4>
<p>For samples <strong>older than 24 hpf</strong>, embryos were grown in 1X PTU till desired stage and treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase for ~30 min.. Choria were removed by gentle pipetting with a 2 mL plastic pasteur pipette. To replace the Pronase solution with fresh E3, embryos were immobilized by anesthesia and collected in the center of the dish by gentle rotational movement. Then the medium was poured away by collecting the embryos at the corner bottom of the dish while taking care not to loose any. After, the dish was filled with fresh E3. This process was repeated three times.</p>
<p>For samples <strong>younger than 18s</strong>, embryos were treated with 150 <span class="math inline">\(\mu\)</span>L per 10 mL of 0.1 mg/mL Pronase directly. Choria were removed the same way as for &gt; 24 hpf embryos but when pouring away the Pronase solution, the dish was simultaneously and very carefully filled with fresh E3 again. Since younger embryos are more fragile and to avoid damage, they must be kept in solution constantly.</p>
<p>Fixation started at the desired stage in 4<span class="math inline">\(\%\)</span> PFA in 0.1<span class="math inline">\(\%\)</span> PBST in 2 mL tubes at 4<span class="math inline">\(^\circ\)</span>C o.n.. The next day, samples were rinsed 3 times for ~5 min. in PBST and passed through a MeOH series of 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 100<span class="math inline">\(\%\)</span> MeOH/PBST (V/V)). For permanent storage, samples were stored in 100<span class="math inline">\(\%\)</span> MeOH at -20<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="ISH-met" class="section level4">
<h4><span class="header-section-number">2.2.3.2</span> In Situ Hybridization</h4>
<p>Samples were prepared after the method described in section <a href="2-mat-met.html#sampleprep">2.2.3.1</a>.</p>
<div id="permeabilisation-probe-hybridization" class="section level5">
<h5><span class="header-section-number">2.2.3.2.1</span> 1. Permeabilisation &amp; Probe Hybridization</h5>
<p><strong>Permeabilisation</strong> <span class="math inline">\(\rightarrow\)</span> without shaking </p>
<p>Samples were rehydrated in an inverse MeOH series of 75<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 50<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 25<span class="math inline">\(\%\)</span> <span class="math inline">\(\rightarrow\)</span> 0<span class="math inline">\(\%\)</span> PBST and washed again for 5 min. two times in pure PBST. Finally, samples were digested in 10 <span class="math inline">\(\mu\)</span>g/mL Proteinase K according to table <a href="2-mat-met.html#tab:met-protk">2.16</a>.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:met-protk">Table 2.16: </span>Proteinase K digestion
</caption>
<thead>
<tr>
<th style="text-align:center;">
Stage
</th>
<th style="text-align:center;">
0.6 s
</th>
<th style="text-align:center;">
7 s
</th>
<th style="text-align:center;">
18 s
</th>
<th style="text-align:center;">
24 hpf
</th>
<th style="text-align:center;">
32 hpf
</th>
<th style="text-align:center;">
36 hpf
</th>
<th style="text-align:center;">
42 hpf
</th>
<th style="text-align:center;">
48 hpf
</th>
<th style="text-align:center;">
72 hpf
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
min.
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
30
</td>
<td style="text-align:center;">
40
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:center;">
60
</td>
<td style="text-align:center;">
60
</td>
</tr>
</tbody>
</table>
<p>Samples were rinsed again two times in PBST and post-fixated in 4<span class="math inline">\(\%\)</span> PFA at 4<span class="math inline">\(^\circ\)</span>C for &gt; 30 min. Samples were washed again for 5 min. three times in PBST.</p>
<p><strong>Probe Hybridisation</strong> <span class="math inline">\(\rightarrow\)</span> all steps at 60<span class="math inline">\(^\circ\)</span>C, except stated differently </p>
<p>Samples were pre-hybridized in 350 <span class="math inline">\(\mu\)</span>L of hybridization buffer (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h. Just before detection probe treatment, the probe was denatured at 80<span class="math inline">\(^\circ\)</span>C in 1:200 of hybridization buffer. Subsequently, hybridization buffer was taken off the samples and replaced with the heated probe. Finally, samples were incubated o.n. at a desired temperature (around 65<span class="math inline">\(^\circ\)</span>C).</p>
</div>
<div id="probe-removal-antibody-incubation" class="section level5">
<h5><span class="header-section-number">2.2.3.2.2</span> 2. Probe removal &amp; Antibody incubation</h5>
<p>The next day the probe got collected and stored at -20<span class="math inline">\(^\circ\)</span>C for re-use. Washing took place at the same temperature as hybridization (from step 1) To keep solutions at temperature a Thermo-Block was used. For the washing series the samples were first washed one time for 20 min. in hybridization buffer, then two times for 30 min. in 50<span class="math inline">\(\%\)</span> Formamide and one time for 20 min. in 25<span class="math inline">\(\%\)</span> Formamide. Then two times for 15 min. in 2X SSCT and two times for 30 min. in 0.2X SSCT. Finally, one time for 5 min. in TNT.</p>
<p>To reduce noise and increase specific signal strength, the samples were treated with blocking solution (section <a href="2-mat-met.html#mat-buff">2.1.6</a>) for 1 - 8 h in 350 <span class="math inline">\(\mu\)</span>L of 2% BR/TNT at room temperature (RT). Afterwards the samples were incubated in 100 <span class="math inline">\(\mu\)</span>L Anti-Digoxigenin diluted in NTMT buffer (1/4000 (V/V)) in 2<span class="math inline">\(\%\)</span> BR/TNT for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="probe-detection" class="section level5">
<h5><span class="header-section-number">2.2.3.2.3</span> 3. Probe detection</h5>
<p>First, the samples were washed six times for ~20 min. (or one wash o.n.) in TNT and two times for ~ 5min. in NTMT. After washing, color staining was performed with 4.5 NBT <span class="math inline">\(\mu\)</span>L + 3.5 <span class="math inline">\(\mu\)</span>L BCIP per mL NTMT in the dark and at RT without shaking (in a drawer) for 2 - 8 h, regularly checking the progression of the reaction. As soon as an appropriate degree of color intensity on the target site was achieved (up to two days), the samples were again washed three times in PBST.</p>
<p>Afterwards the samples were either prepared for immunostaining or imaging. For permanent storage samples were kept in 50<span class="math inline">\(\%\)</span> Glycerol at 4<span class="math inline">\(^\circ\)</span>C.</p>
</div>
</div>
<div id="immuno-met" class="section level4">
<h4><span class="header-section-number">2.2.3.3</span> Immuno staining</h4>
<p>Samples were prepared after the method described in section <a href="2-mat-met.html#sampleprep">2.2.3.1</a>.</p>
<p>First, samples were blocked in 2<span class="math inline">\(\%\)</span> Goat Serum / PBDT (V/V) for 30 min.. For protein target site detection, a <strong>primary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for ~2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times. To stain the now bound primary antibody, a <strong>secondary antibody</strong> (150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)) was incubated for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C. Samples were washed for 2 h in PBDT while changing the solution 5 - 6 times.</p>
</div>
<div id="mount-met" class="section level4">
<h4><span class="header-section-number">2.2.3.4</span> Mounting</h4>
<p>For live microscopy zebrafish embryos are usually immersed in a 1<span class="math inline">\(\%\)</span> solution of low melting-point Agarose (LMPA) solution and then oriented on an optical cover slip manually until the LMPA has solidified. This process allows to mount eight to ten embryos per dish.</p>
<p>To take advantage of the high number of offspring a single zebrafish female may lay, a new sample preparation technique was designed that allows for</p>
<ol style="list-style-type: decimal">
<li>a four to five times increase in samples per dish</li>
<li>a facilitated navigation <em>via</em> a grid-like orientation through the samples and</li>
<li>an improved spatial orientation where the embryos body axes are aligned parallel to the optical Z-sections of the confocal microscope.</li>
</ol>
<p>A detailed protocol can be found under section <a href="3-res.html#res-mount">3.1.1</a></p>
</div>
</div>
<div id="comp-met" class="section level3">
<h3><span class="header-section-number">2.2.4</span> Dry lab</h3>
<div id="image-j-macros" class="section level4">
<h4><span class="header-section-number">2.2.4.1</span> Image J macros</h4>
<p>Three IJ macros have been developed to facilitate image analysis and make results more reproducible. Each of them is specifically designed for input of LL and pLLP images of the <em>cldnb:lyn-gfp</em> transgenic line.</p>
<p>Hence, they are called <strong><em>anaLLzr</em></strong> …</p>
<ul>
<li><strong>2D</strong> - analysis of Z-projected images of the LL at end of migration</li>
<li><strong>2DT</strong> - analysis of Z-projected images of the LL during migration</li>
<li><strong>3D</strong> - analysis of 3D image stacks of the pLLP at a given timepoint</li>
</ul>
<p>Since their development was an integral part of my PhD work, the description of the macros can be found in the results part in section section <a href="3-res.html#res-met">3.1</a>.</p>
</div>
<div id="prolif" class="section level4">
<h4><span class="header-section-number">2.2.4.2</span> Proliferation Analysis</h4>
<p>The basic principle is based on work done by Laguerre <em>et al.</em>, 2009<span class="citation">(<em>34</em>)</span>. For registration of mitotic events an IJ manual tracking tool was used that allows to track an image feature through a stack of images creating tracks as it progresses through volume / time (‘MTrackJ’<span class="citation">(<em>59</em>)</span>).</p>
<p>For mounting the embryo, the procedure described in section <a href="2-mat-met.html#mount-met">2.2.3.4</a> was used. Nuclei were visualized in a <em>TgBAC(cxcr4b:H2B-RFP)</em> transgenic line. After Z-projection of volumetric timelapses, mitotic events were tracked in each CC and the pLLP. Afterwards the data was exported as one table per embryo and processed by counting mitoses per pLLP / CC / total CC mitoses. Figure <a href="2-mat-met.html#fig:mitodatapoints">2.1</a> shows an exemplary track for the data analyzed.</p>

<div class="figure" style="text-align: center"><span id="fig:mitodatapoints"></span>
<img src="figures/materials/prol/Prolif.png" alt="Tracking of mitotic events. T1-T3 show consequetive timepoints of a single nucleus." width="60%" />
<p class="caption">
Figure 2.1: Tracking of mitotic events. T1-T3 show consequetive timepoints of a single nucleus.
</p>
</div>
</div>
<div id="ACI" class="section level4">
<h4><span class="header-section-number">2.2.4.3</span> Apical Index</h4>
<div id="rationale" class="section level5">
<h5><span class="header-section-number">2.2.4.3.1</span> Rationale</h5>
<p>The earliest attempt found for indexing AC can be found in a study published by Lee <em>et al</em><span class="citation">(<em>60</em>)</span> where they were interested in the ‘apical index’ (A.I.) of bottle cells during <em>X.laevis</em> gastrulation (figure <a href="2-mat-met.html#fig:ACLee">2.2</a> Lee). Another example for measuring AC is the <em>apical constriction index</em> (A.C.I., figure <a href="2-mat-met.html#fig:ACLee">2.2</a> Harding) for the cells of the <em>D.rerio</em> lateral line primordium (pLLP), which can be found in a study from 2012 where it was shown that FGFr-Ras-MAPK signaling is required for Rock2a localization and AC <span class="citation">(<em>61</em>, <em>62</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:ACLee"></span>
<img src="figures/materials/models/ai.png" alt="A.I. indeces in the literature. Lee A.I. of X.leavis bottle cells measured in 2D. Harding A.I. of D.r. pLLP cells measured in 3D." width="75%" />
<p class="caption">
Figure 2.2: A.I. indeces in the literature. <strong>Lee</strong> A.I. of <em>X.leavis</em> bottle cells measured in 2D. <strong>Harding</strong> A.I. of <em>D.r.</em> pLLP cells measured in 3D.
</p>
</div>
<p>In these two publications, the way they measure A.I. <span class="citation">(<em>60</em>)</span> and ACI <span class="citation">(<em>62</em>)</span> respectively, does not differ and is the ratio of lateral height over apical width.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{apical\;width\;[\mu m]}\]</span></p>
<p>We found two principal weaknesses of applying this ratio to the cells of the pLLP. First, it does not respect the independence of lateral height to AC. Second, it does not differentiate between constriction along the <em>anterio-posterior</em> (AP) or the <em>dorso-ventral</em> (DV) axis. Third, it actually represents the A.I. rather than the apical <em>constriction</em> index.</p>
<div id="ACI-param" class="section level6">
<h6><span class="header-section-number">2.2.4.3.1.1</span> Parameter definition</h6>
<p>To obtain a precise and biologically meaningful way to quantify AC, first a couple of definitions had to be made.</p>

<div class="definition">
<span id="def:unnamed-chunk-3" class="definition"><strong>Definition 2.1  (AC is independent of orientation)  </strong></span>In a 3D space a cell can have any orientation and still be apically constricted. Therefore, before measuring one should make sure orientation between embryos is aligned and also consider taking measurements along two different directions. Since apical constriction is not necessarily isotropic, it is important to consider constriction along 2 perpendicular axes (AP and DV axis of the embryo/pLLP).
</div>


<div class="definition">
<span id="def:unnamed-chunk-4" class="definition"><strong>Definition 2.2  (AC is independent of lateral height)  </strong></span>Lateral height can be described as the distance of the two farthest points on the surface area of a cell. Two cells with different lateral heights can be equally apically constricted.
</div>


<div class="definition">
<span id="def:unnamed-chunk-5" class="definition"><strong>Definition 2.3  (AC is independent of cell volume)  </strong></span>The volume of a cell represents its size. A large cell can be equally constricted as a small cell.
</div>

</div>
<div id="ACI-lat" class="section level6">
<h6><span class="header-section-number">2.2.4.3.1.2</span> Adaption for variation in lateral height</h6>
<p>To test different A.I. conditions, an apically constricted cell can be approximated by modeling a tetrahedron. For example, shrinking or enlarging a cell symmetrically should not affect the A.I.. As described by Harding(2014)<span class="citation">(<em>62</em>)</span>, the <em>apical width</em> of a cell is measured first by manual 3-D reconstruction, second manual re-orientation, and third by going 1 <span class="math inline">\(\mu\)</span>m from the apical tip into the cell (from now on referred to as <span class="math inline">\(\Delta\)</span>ap, <a href="2-mat-met.html#fig:ACICells">2.3</a>B). Finally, <em>apical width</em> is the total width of the 2D object in the respective volume.</p>
<p>If <span class="math inline">\(\Delta\)</span>ap is a constant, the A.I. in a symmetrically enlarged cell increases from e.g. ~15 to ~23, since <em>apical width</em> stays the same but lateral height increases (compare figure <a href="2-mat-met.html#fig:ACICells">2.3</a>A to A’). On the contrary, if <span class="math inline">\(\Delta\)</span>ap is adjusted relative to a cells lateral height, e.g. by percentage, the A.I. in a symmetrically enlarged cell stays the same (compare figure <a href="2-mat-met.html#fig:ACICells">2.3</a>A to A’’).</p>

<div class="figure" style="text-align: center"><span id="fig:ACICells"></span>
<img src="figures/summary/aci_fig-01.png" alt="Different ways to quantify the apical index. A-A’’ A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta is proportional to the lateral height. B Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2-D area as seen at \(\Delta\)ap of 1 or 2.5 \(\mu\)m." width="85%" />
<p class="caption">
Figure 2.3: Different ways to quantify the apical index. <strong>A-A’’</strong> A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta is proportional to the lateral height. <strong>B</strong> Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2-D area as seen at <span class="math inline">\(\Delta\)</span>ap of 1 or 2.5 <span class="math inline">\(\mu\)</span>m.
</p>
</div>
<p>Therefore the measurement for apical width has to be relative to lateral height.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{relative\;apical\;width\;[\mu m]}\]</span></p>
</div>
<div id="ACI-pol" class="section level6">
<h6><span class="header-section-number">2.2.4.3.1.3</span> Adaption for tissue polarization</h6>
<p>Organs develop in a 3-D space and are polarized along each axis. AC usually describes a 2-D morphogenetic movement towards a center along the X-Y axes. However, the constriction movements along X and Y might be independent of one another. This could mean that they happen at different speeds, or that one is absent. As a result, the tissue would look less radially (figure <a href="2-mat-met.html#fig:cellpol">2.4</a>B) constricted, but more constricted along one particular axis (anisotropic). In order to separate those two AC dimensions, the A.I. can be calculated for the <em>anterio-posterior</em> and for the <em>dorso-ventral</em> axis (figure <a href="2-mat-met.html#fig:cellpol">2.4</a>, horizontal vs. vertical).</p>

<div class="figure" style="text-align: center"><span id="fig:cellpol"></span>
<img src="figures/materials/models/ACI_Cells_pol.png" alt="Schematic AC along the A-P and D-V axis. A shows a A-P and D-V constricted cluster of cells. B shows a D-V constricted cluster of cells." width="75%" />
<p class="caption">
Figure 2.4: Schematic AC along the A-P and D-V axis. <strong>A</strong> shows a A-P and D-V constricted cluster of cells. <strong>B</strong> shows a D-V constricted cluster of cells.
</p>
</div>
<p>By fitting an ellipsoid to the area taken at <span class="math inline">\(\mathrm{\Delta}\)</span>ap, one will obtain the following parameters (figure <a href="2-mat-met.html#fig:ellipse">2.5</a>).</p>
<ol style="list-style-type: decimal">
<li>Length of Major axis
<ul>
<li>indicates the level of constriction along the less constricted axis</li>
</ul></li>
<li>Length of Minor axis
<ul>
<li>indicates the level of constriction along the most constricted axis</li>
</ul></li>
<li>Angle of Major from 0<span class="math inline">\(^\circ\)</span>
<ul>
<li>indicates the orientation of the long, less-constricted axis: If the angle is close to 0<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the AP axis (the cell is constricted along the DV axis). If the angle is close to 90<span class="math inline">\(^\circ\)</span>, the long axis of the apical area is parallel to the DV axis (the cell is constricted along the AP axis).</li>
</ul></li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:ellipse"></span>
<img src="figures/materials/models/ellipse.png" alt="Scheme of ellipsoid measures. A shows the major axis as apical width and the minor axis as apical height. B shows the angular displacement from the horizon in steps of 30\(^\circ\)." width="75%" />
<p class="caption">
Figure 2.5: Scheme of ellipsoid measures. <strong>A</strong> shows the major axis as apical width and the minor axis as apical height. <strong>B</strong> shows the angular displacement from the horizon in steps of 30<span class="math inline">\(^\circ\)</span>.
</p>
</div>
</div>
<div id="aci-theorem" class="section level6">
<h6><span class="header-section-number">2.2.4.3.1.4</span> Measurement definition</h6>
<p>The two dimensions of A.I. indices can therefore be defined as the following ratios…</p>

<div class="definition">
<span id="def:unnamed-chunk-6" class="definition"><strong>Definition 2.4  (A.I. Major)  </strong></span><span class="math display">\[\mathbf{A.I._{Major}} = \frac{lateral\;height\;[\mu m]}{ellipsoid\;major\;axis\;at\;relative\;\Delta ap\;[\mu m]}\]</span>
</div>


<div class="definition">
<span id="def:unnamed-chunk-7" class="definition"><strong>Definition 2.5  (A.I. Minor)  </strong></span><span class="math display">\[\mathbf{A.I._{Minor}} = \frac{lateral\;height\;[\mu m]}{ellipsoid\;minor\;axis\;at\;relative\;\Delta ap\;[\mu m]}\]</span>
</div>


<div class="definition">
<span id="def:unnamed-chunk-8" class="definition"><strong>Definition 2.6  (Angle Major)  </strong></span><span class="math display">\[\mathbf{Angle_{Major}} = \measuredangle = \mathrm{\Delta}\;from\;horizon\;[0-90^\circ]\]</span>
</div>

</div>
</div>
<div id="ACI-Dis" class="section level5">
<h5><span class="header-section-number">2.2.4.3.2</span> Measurements</h5>
<p>As a proof of principle of the definitions stated in the previous section we compare our results to previously published results from Harding <em>et al.</em> <span class="citation">(<em>61</em>)</span> who, as a control, measured apical constriction in embryos treated with an FGF inhibitor (SU5402) and their DMSO controls.</p>
<div id="ACI-singlecell" class="section level6">
<h6><span class="header-section-number">2.2.4.3.2.1</span> Single Cell measurements</h6>
<p>Each geometric object has a centroid coordinate in X and Y (and Z) which is represented as the mean of all X or Y coordinates within the object. In figure <a href="2-mat-met.html#fig:ACICells">2.3</a>, centroid coordinates in X and Y are used to plot the cells as points in the X-Y plane. Additionally, each point is colored for the A.I. value (high values are dark red - red, middle values are green, low values are cyan - blue).</p>

<div class="figure" style="text-align: center"><span id="fig:aci-single-cell-measures-graph"></span>
<img src="thesis_dsk_files/figure-html/aci-single-cell-measures-graph-1.png" alt="A.I. single cell measurements" width="95%" />
<p class="caption">
Figure 2.6: A.I. single cell measurements
</p>
</div>
<p>Harding <em>et al.</em> <span class="citation">(<em>62</em>)</span> were using a constant <span class="math inline">\(\mathrm{\Delta}\)</span>ap to measure the apical width, which we have shown to be incorrect in certain cases. In their study they found that certain mean A.C.I. values in the DMSO go as high as 15 (figure <a href="2-mat-met.html#fig:HardingACI">2.7</a>), which might be related to this (see figure <a href="2-mat-met.html#fig:ACICells">2.3</a>). By measuring apical width at a relative <span class="math inline">\(\mathrm{\Delta}\)</span>ap and taking into account all pLLP cells of the two exemplary pLLPs shown in figure <a href="2-mat-met.html#fig:ACICells">2.3</a>, we measure a mean difference in the Major of 0.53 and 1.11 in the Minor.</p>

<div class="figure" style="text-align: center"><span id="fig:HardingACI"></span>
<img src="figures/materials/models/harding.png" alt="A.I. indices by Harding et al. E-G’ 3-D reconstructions of the highlighted cell. H A.C.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of hsp70:dn-Ras. (n = 180 cells / N = 6 embryos)." width="60%" />
<p class="caption">
Figure 2.7: A.I. indices by Harding et al. <strong>E-G’</strong> 3-D reconstructions of the highlighted cell. <strong>H</strong> A.C.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of <em>hsp70:dn-Ras</em>. (n = 180 cells / N = 6 embryos).
</p>
</div>
</div>
<div id="ACI-Angledens" class="section level6">
<h6><span class="header-section-number">2.2.4.3.2.2</span> Angle densities</h6>
<p>To check whether there is a bias in orientation of the apical width, the angle measurements <a href="2-mat-met.html#fig:ellipse">2.5</a> can be shown as a function of density along X (figure <a href="2-mat-met.html#fig:angletoACI">2.8</a>A).</p>
<p>Interestingly the results indicate that there is less of a difference for the Major<sub>Angle</sub> at angles bigger than 15-20<span class="math inline">\(^\circ\)</span>. This would mean that the apical surface of the cells in SU5402 treated embryos is more strongly oriented along the horizontal <em>antero-posterior</em> axis.</p>
</div>
<div id="ACI-mag" class="section level6">
<h6><span class="header-section-number">2.2.4.3.2.3</span> ACI magnitude at different Angles</h6>
<p>Now, to get an idea of the magnitude of constriction relative to the orientation of the cell (angle to the horizontal), the A.I.<sub>Major/Minor</sub> can be shown as a function of the Major<sub>Angle</sub> (figure <a href="2-mat-met.html#fig:angletoACI">2.8</a>B-B’).</p>
<p>Since AC is a 3-D morphogenetic process and since cells in a wild type pLLP are mostly radially organized, it does make sense to try to look at AC from more than just one perspective. Here we propose to separate the A.I. into an <em>antero-posterior</em> and a <em>dorso-ventral</em> dimension.</p>
<ol style="list-style-type: decimal">
<li><p>While for the control (DMSO treated) embryo the distribution of the cells Major Angles seem to be mostly uniform, for the SU5402 treated embryo there is an accumulation of lower Major Angles. This means that cells in SU5402 treated embryos are more oriented along the horizontal (anterior - posterior) axis.</p></li>
<li><p>Interestingly there does not seem to be much of a difference in A.I.<sub>Major</sub> (figure <a href="2-mat-met.html#fig:angletoACI">2.8</a>B), which can also be shown by the mean values which are at 2.6 for the DMSO control and at 2.1 for the SU5402 treated condition.</p></li>
<li><p>For the A.I.<sub>Minor</sub> (figure <a href="2-mat-met.html#fig:angletoACI">2.8</a>B’) the means are 4.7 for the DMSO control and 3.6 for SU5402. The base constriction for both, DMSO and SU5402 is at around 3.6, however there is a peak at around 40 - 60<span class="math inline">\(^\circ\)</span> in the DMSO control where cells are most constricted having a maximum A.I. at 15.8. This indicates that for the Minor, cells in that range of angles are more constricted than cells oriented in different directions.</p></li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:angletoACI"></span>
<img src="thesis_dsk_files/figure-html/angletoACI-1.png" alt="A.I.Major / A.I.Minor over MajorAngle" width="49%" /><img src="thesis_dsk_files/figure-html/angletoACI-2.png" alt="A.I.Major / A.I.Minor over MajorAngle" width="49%" />
<p class="caption">
Figure 2.8: A.I.<sub>Major</sub> / A.I.<sub>Minor</sub> over Major<sub>Angle</sub>
</p>
</div>
</div>
</div>
</div>
</div>
</div>
<div id="mat-GrTrDat" class="section level2">
<h2><span class="header-section-number">2.3</span> Ground Truth</h2>
<p>Analyzing images and extracting quantitative measurements can be a tedious task, especially when the analysis becomes more complex. Fortunately, there are ways to automate image analysis by using either machine learning approaches or by tailoring hand-crafted algorithms in an image analysis software tool like e.g. ImageJ<span class="citation">(<em>63</em>)</span>.
The main advantages of doing so are to…</p>
<ul>
<li>be independent of confirmation bias</li>
<li>make the analysis more robust against oversight</li>
<li>increase the statistical power by increasing the number of data points<span class="citation">(<em>64</em>)</span></li>
<li>increase effect size by increasing the measurement accuracy<span class="citation">(<em>64</em>)</span></li>
</ul>
<p>However, to ensure the measurements taken by a tailored or trained algorithm are meaningful, they must be compared to a <em>ground truth</em> dataset which again describes a general measure of algorithmic quality performance<span class="citation">(<em>65</em>)</span>.</p>
<div id="cluster-analysis" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Cluster Analysis</h3>
<p>The <em>anaLLzR2D</em> algorithm was designed for semi-automatic cell cluster detection in the <em>cldnb:lyn-gfp</em> transgenic line and optional nuclei counting in a second DAPI labeled channel within the Regions of Interest (ROIs) derived from the cell cluster detection.</p>
<div id="design" class="section level4">
<h4><span class="header-section-number">2.3.1.1</span> Design</h4>
<p>To assess the quality of the <em>anaLLzR2D</em> algorithm for nuclei detection the ground truth was designed as follows.</p>
<div id="model" class="section level5">
<h5><span class="header-section-number">2.3.1.1.1</span> Model</h5>
<ul>
<li>each Cell Cluster (CC) consists of a number of objects (cells)</li>
<li>each object is part of the respective CC and defines one cell entity</li>
<li>each object is determined via a fluorescent nucleus label</li>
<li>embryos are mounted within a 3D mold (section <a href="2-mat-met.html#mount-met">2.2.3.4</a>) to reduce noise</li>
</ul>
<table style="width:97%;">
<caption><span id="tab:imgcond3DGrT">Table 2.17: </span> Cluster Analysis Model</caption>
<colgroup>
<col width="48%" />
<col width="48%" />
</colgroup>
<tbody>
<tr class="odd">
<td align="right">XY scale</td>
<td align="left">0.32 px/<span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="even">
<td align="right">Z-spacing</td>
<td align="left">4 <span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="odd">
<td align="right">Camera</td>
<td align="left">Rolera</td>
</tr>
</tbody>
</table>
<div style="page-break-after: always;"></div>
</div>
<div id="training-test-data" class="section level5">
<h5><span class="header-section-number">2.3.1.1.2</span> Training &amp; test data</h5>
<p>The training set consists of three randomly picked wild type pLLs. For each the algorithm was run with standard parameters. Cell cluster ROIs and nuclei multi-point labels were edited manually. To test the algorithm, it is run at different nuclei detection thresholds on the same image data.</p>
</div>
</div>
</div>
<div id="morphometric-analysis" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Morphometric analysis</h3>
<p>The <em>anallzr3D</em> algorithm was designed for fully automated, single cell volume segmentation in the <em>cldnb:lyn-gfp</em> transgenic line. In addition to 3D cellular metrics, it offers Apical Constriction measurement of each cell.</p>
<div id="design-1" class="section level4">
<h4><span class="header-section-number">2.3.2.1</span> Design</h4>
<p>To assess the quality of the anaLLzr3D algorithm the ground truth was modeled as follows.</p>
<div id="model-1" class="section level5">
<h5><span class="header-section-number">2.3.2.1.1</span> Model</h5>
<ul>
<li>each pLLP consists of a number of objects (cells)</li>
<li>each object is part of the pLLP and defines one cell entity</li>
<li>cell boundaries are determined via the transgene <em>cldnb:lyn-gfp</em> +/+</li>
<li>embryos are imaged live to conserve signal and membrane integrity</li>
<li>embryos are mounted within a 3D mold for improved 3D alignment</li>
</ul>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:model3DGrT">Table 2.18: </span>anaLLzr3D Model
</caption>
<tbody>
<tr>
<td style="text-align:right;width: 4cm; ">
Exposure time
</td>
<td style="text-align:left;">
100 ms
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Laser intensity
</td>
<td style="text-align:left;">
100<span class="math inline">\(\%\)</span> / 9.3 mW
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Objective
</td>
<td style="text-align:left;">
40X; CFI APO LWD WI; N.A. = 1.15, W.D. = 0.60 mm
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Tube lens
</td>
<td style="text-align:left;">
1.0X
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Z-spacing
</td>
<td style="text-align:left;">
0.4 - 0.5 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Camera
</td>
<td style="text-align:left;">
sCMOS; 4.2 M.Pix; 82<span class="math inline">\(\%\)</span> Q.E.
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
SD system
</td>
<td style="text-align:left;">
Yokogawa CSU - W1; 50 <span class="math inline">\(\mu\)</span>m pattern
</td>
</tr>
<tr>
<td style="text-align:right;width: 4cm; ">
Piezo
</td>
<td style="text-align:left;">
Piezo Z-table; 300 <span class="math inline">\(\mu\)</span>m scan range
</td>
</tr>
</tbody>
</table>
</div>
<div id="training-test-data-1" class="section level5">
<h5><span class="header-section-number">2.3.2.1.2</span> Training &amp; test data</h5>
<p>The training set consists of three randomly picked wildtype pLLPs. For each the algorithm is run with no filters (X, Y, Z border objects, size) and a minimum segmentation threshold. Afterwards the segmentation result is corrected manually for over- and under-segmentation and objects that are not part of the pLLP.</p>
<p>To test the algorithm it is run at different segmentation thresholds on the same image data.</p>
</div>
</div>
</div>
</div>
<div id="mat-datasets" class="section level2">
<h2><span class="header-section-number">2.4</span> Image Data Sets</h2>
<p>Summaries of Image datasets. <em>Pairs</em> describe the number of parent pairs I harvested eggs from. <em>Stage</em> describes the time I waited for the parent pairs to mate and lay eggs. Since pair #1 might have laid their eggs earlier than pair #2, those batches would be some time apart in their <em>staging</em>. <em>Stamp</em> describes the version of the stamp I used, where the main difference between version 4 and 5 are more wells added and some minor upgrades in well-design.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatcc">Table 2.19: </span>Cell Cluster dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
60 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="2">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Fixation
</td>
<td style="text-align:left;">
4<span class="math inline">\(\%\)</span> PFA o.N.
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
1<span class="math inline">\(\%\)</span> LMPA
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
25X WI + 1.0x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 12*X large image
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatai">Table 2.20: </span>A.I. dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
Kleinhans <span class="math inline">\(\textit{et al.}\)</span>, 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.5<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
version 4A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
40X objective + 1.0x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 1x1; Gain 1; Exposure 100 ms
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (100<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
0.4 <span class="math inline">\(\mu\)</span>m Z-spacing
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatprol">Table 2.21: </span>Proliferation dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; cxcr4b(BAC):H2BRFP +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
Kleinhans <span class="math inline">\(\textit{et al.}\)</span>, 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
version 4A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="5">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 35 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
651 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 110 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 7 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatdet">Table 2.22: </span>Detection dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
Kleinhans <span class="math inline">\(\textit{et al.}\)</span>, 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
version 5A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="6">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.5x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; Exposure 20 ms; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (25<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
3 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 3*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 10 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgdatato">Table 2.23: </span>Atoh1a dataset
</caption>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
Crossings
</td>
<td style="text-align:left;width: 2.5cm; ">
Pairs
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Transgenes
</td>
<td style="text-align:left;width: 2.5cm; ">
cldnb:lyn-gfp +/?; atoh1a:Tom +/0
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Mutation
</td>
<td style="text-align:left;width: 2.5cm; ">
shroom3
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Staging
</td>
<td style="text-align:left;width: 2.5cm; ">
30 min.
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
Mounting
</td>
<td style="text-align:left;width: 2.5cm; ">
Protocol
</td>
<td style="text-align:left;">
Kleinhans <span class="math inline">\(\textit{et al.}\)</span>, 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Agarose
</td>
<td style="text-align:left;width: 2.5cm; ">
0.3<span class="math inline">\(\%\)</span> LMPA + 20<span class="math inline">\(\%\)</span> Tricaine (V/V<span class="math inline">\(\%\)</span>)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
stamp
</td>
<td style="text-align:left;width: 2.5cm; ">
version 5A
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="7">
Imaging
</td>
<td style="text-align:left;width: 2.5cm; ">
Magnification
</td>
<td style="text-align:left;">
20X + 1.0x zoom
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Camera
</td>
<td style="text-align:left;width: 2.5cm; ">
Binning 2x2; Gain 4; full FOV*150 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
488 nm (Int: 20<span class="math inline">\(\%\)</span>; Exp: 25 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Channels
</td>
<td style="text-align:left;width: 2.5cm; ">
561 nm (Int: 30<span class="math inline">\(\%\)</span>; Exp: 50 ms)
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Positions
</td>
<td style="text-align:left;width: 2.5cm; ">
36
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Z-Stack
</td>
<td style="text-align:left;width: 2.5cm; ">
2.5 <span class="math inline">\(\mu\)</span>m Z-spacing; 100 <span class="math inline">\(\mu\)</span>m stack size; 2*X large image
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
Time
</td>
<td style="text-align:left;width: 2.5cm; ">
20 h / 20 min. interval / start ~ 2 p.m.
</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="footnotes">
<hr />
<ol start="11">
<li id="fn11"><p>e.g. photomultipliers or charge-coupled devices<a href="2-mat-met.html#fnref11" class="footnote-back">↩︎</a></p></li>
<li id="fn12"><p>the process of embedding the samples in agarose<a href="2-mat-met.html#fnref12" class="footnote-back">↩︎</a></p></li>
<li id="fn13"><p>Where B stands for a batch, that is if multiple dishes were imaged and P stands for the position within a single batch<a href="2-mat-met.html#fnref13" class="footnote-back">↩︎</a></p></li>
<li id="fn14"><p>“movement to make scientific research […] and its dissemination accessible to all levels of an inquiring society” – Wikipedia/en/Open_science<a href="2-mat-met.html#fnref14" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-Intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-res.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis_dsk.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
