<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.13.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-Intro.html"/>
<link rel="next" href="3-res.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#morphogenesis"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#objective-and-open-questions"><i class="fa fa-check"></i><b>1.3</b> Objective and Open Questions</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-Intro.html"><a href="1-Intro.html#motivation"><i class="fa fa-check"></i><b>1.3.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-Intro.html"><a href="1-Intro.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>1.4</b> Data Strategy and Analysis</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-Intro.html"><a href="1-Intro.html#sample-preparation"><i class="fa fa-check"></i><b>1.4.1</b> Sample Preparation</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-Intro.html"><a href="1-Intro.html#imaging"><i class="fa fa-check"></i><b>1.4.2</b> Imaging</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-Intro.html"><a href="1-Intro.html#data-handling"><i class="fa fa-check"></i><b>1.4.3</b> Data handling</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-Intro.html"><a href="1-Intro.html#image-and-data-analysis"><i class="fa fa-check"></i><b>1.4.4</b> Image and Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.9</b> Hardware</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.10</b> Software</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.1.11</b> Ground Truth Data</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.1</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.2</b> Wetlab</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.3</b> Computational</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.3</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.1</b> Lateral Line Morphometrics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#dataset"><i class="fa fa-check"></i><b>3.1.1</b> Dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#res-ccounts"><i class="fa fa-check"></i><b>3.1.2</b> Number and Position of Cell Clusters</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#res-llmorph"><i class="fa fa-check"></i><b>3.1.3</b> Cell Count and Area of CCs</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2</b> Proliferation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#dataset-1"><i class="fa fa-check"></i><b>3.2.1</b> Dataset</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#res-prolpllp"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Mitoses</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.3</b> Rosette Formation and Cluster Deposition</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-res.html"><a href="3-res.html#dataset-2"><i class="fa fa-check"></i><b>3.3.1</b> Dataset</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-res.html"><a href="3-res.html#cluster-deposition"><i class="fa fa-check"></i><b>3.3.2</b> Cluster Deposition</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-res.html"><a href="3-res.html#registration-1"><i class="fa fa-check"></i><b>3.3.3</b> Registration</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-res.html"><a href="3-res.html#res-rdtdet"><i class="fa fa-check"></i><b>3.3.4</b> Rosette Detection</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-res.html"><a href="3-res.html#correlations"><i class="fa fa-check"></i><b>3.3.5</b> Correlations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-res.html"><a href="3-res.html#apical-constriction-1"><i class="fa fa-check"></i><b>3.4</b> Apical Constriction</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-res.html"><a href="3-res.html#dataset-3"><i class="fa fa-check"></i><b>3.4.1</b> Dataset</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-res.html"><a href="3-res.html#cell-shape-and-arrangement"><i class="fa fa-check"></i><b>3.4.2</b> Cell Shape and Arrangement</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-res.html"><a href="3-res.html#single-cell-metrics"><i class="fa fa-check"></i><b>3.4.3</b> Single Cell Metrics</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.5</b> Haircell Specification</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-res.html"><a href="3-res.html#dataset-4"><i class="fa fa-check"></i><b>3.5.1</b> Dataset</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-res.html"><a href="3-res.html#atoh1a-in-the-ll"><i class="fa fa-check"></i><b>3.5.2</b> Atoh1a in the LL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-res.html"><a href="3-res.html#rescue-experiments"><i class="fa fa-check"></i><b>3.6</b> Rescue Experiments</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-res.html"><a href="3-res.html#res-shrmresc"><i class="fa fa-check"></i><b>3.6.1</b> Shroom3 Ectopic Expression</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-res.html"><a href="3-res.html#morphogen-rescue"><i class="fa fa-check"></i><b>3.6.2</b> Morphogen Rescue</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-res.html"><a href="3-res.html#res-rockresc"><i class="fa fa-check"></i><b>3.6.3</b> Mechanic Rescue</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-res.html"><a href="3-res.html#res-tempresc"><i class="fa fa-check"></i><b>3.6.4</b> Thermodynamic Rescue</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-res.html"><a href="3-res.html#atoh1a-knockdown"><i class="fa fa-check"></i><b>3.6.5</b> atoh1a knockdown</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#adult-phenotype"><i class="fa fa-check"></i><b>4.1</b> Adult Phenotype</a></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#lateral-line"><i class="fa fa-check"></i><b>4.2</b> Lateral Line</a></li>
<li class="chapter" data-level="4.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.3</b> Rosettes</a></li>
<li class="chapter" data-level="4.4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#hair-cells"><i class="fa fa-check"></i><b>4.4</b> Hair Cells</a></li>
<li class="chapter" data-level="4.5" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-2"><i class="fa fa-check"></i><b>4.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.6" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.6</b> Model</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.1</b> Proliferation</a></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection"><i class="fa fa-check"></i><b>5.2</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.3" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification-1"><i class="fa fa-check"></i><b>5.3</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.4" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.4</b> Single cell reconstruction and AC measurement</a></li>
<li class="chapter" data-level="5.5" data-path="5-discussion.html"><a href="5-discussion.html#future-perspectives"><i class="fa fa-check"></i><b>5.5</b> Future perspectives</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mat-met" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="mat" class="section level2">
<h2><span class="header-section-number">2.1</span> Materials</h2>
<div id="mat-chem" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Chemicals</h3>
<table>
<caption><span id="tab:mat-chem">Table 2.1: </span> Chemicals</caption>
<colgroup>
<col width="28%" />
<col width="54%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Chemical</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Agarose</td>
<td align="center">Roth</td>
<td align="left">6351.2</td>
</tr>
<tr class="even">
<td align="right">Agar-Agar</td>
<td align="center">Roth</td>
<td align="left">5210.3</td>
</tr>
<tr class="odd">
<td align="right">Ampicillin</td>
<td align="center">Roth</td>
<td align="left">K029.2</td>
</tr>
<tr class="even">
<td align="right">ATP</td>
<td align="center">Epicentre</td>
<td align="left">E311K</td>
</tr>
<tr class="odd">
<td align="right">Blocking Reagent</td>
<td align="center">Roche</td>
<td align="left">11096176001</td>
</tr>
<tr class="even">
<td align="right">BCIP</td>
<td align="center">Fermentas</td>
<td align="left">R0822</td>
</tr>
<tr class="odd">
<td align="right">CaCl</td>
<td align="center">Roth</td>
<td align="left">886.1</td>
</tr>
<tr class="even">
<td align="right">Calyculin</td>
<td align="center"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">DIG RNA Mix</td>
<td align="center">Roche</td>
<td align="left">11277073910</td>
</tr>
<tr class="even">
<td align="right">DMSO</td>
<td align="center">Roth</td>
<td align="left">4720.2</td>
</tr>
<tr class="odd">
<td align="right">EtBr</td>
<td align="center">Roth</td>
<td align="left">2218.3</td>
</tr>
<tr class="even">
<td align="right">EtOH</td>
<td align="center">Roth</td>
<td align="left">9065.3</td>
</tr>
<tr class="odd">
<td align="right">Formaldehyde</td>
<td align="center">Roth</td>
<td align="left">7398.1</td>
</tr>
<tr class="even">
<td align="right">Formamide</td>
<td align="center">Roth</td>
<td align="left">P040.1</td>
</tr>
<tr class="odd">
<td align="right">Glycerol</td>
<td align="center">Roth</td>
<td align="left">3783.2</td>
</tr>
<tr class="even">
<td align="right">IPTG</td>
<td align="center">Thermo</td>
<td align="left">R1171</td>
</tr>
<tr class="odd">
<td align="right">KCl</td>
<td align="center">Roth</td>
<td align="left">P017.1</td>
</tr>
<tr class="even">
<td align="right">Low melting point Agarose</td>
<td align="center">Roth</td>
<td align="left">A9539</td>
</tr>
<tr class="odd">
<td align="right">Maleic Acid</td>
<td align="center"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="right">MeOH</td>
<td align="center">Roth</td>
<td align="left">CP43.3</td>
</tr>
<tr class="odd">
<td align="right">MgCl</td>
<td align="center">Roth</td>
<td align="left">2189.1</td>
</tr>
<tr class="even">
<td align="right">NaCl</td>
<td align="center">Roth</td>
<td align="left">9265.2</td>
</tr>
<tr class="odd">
<td align="right">NaHCO</td>
<td align="center">Roth</td>
<td align="left">855.1</td>
</tr>
<tr class="even">
<td align="right">NaOH</td>
<td align="center">Roth</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">NGS</td>
<td align="center">Sigma</td>
<td align="left">C6767</td>
</tr>
<tr class="even">
<td align="right"><em>p</em>-Formaldehyd</td>
<td align="center">Sigma</td>
<td align="left">P6148</td>
</tr>
<tr class="odd">
<td align="right">Phenol Red</td>
<td align="center">Sigma</td>
<td align="left">P0290</td>
</tr>
<tr class="even">
<td align="right">Propan-2-ol</td>
<td align="center">VWR</td>
<td align="left">20842330</td>
</tr>
<tr class="odd">
<td align="right">Proteinase K</td>
<td align="center">Roth</td>
<td align="left">7528.4</td>
</tr>
<tr class="even">
<td align="right">PTU</td>
<td align="center">Sigma</td>
<td align="left">P7629</td>
</tr>
<tr class="odd">
<td align="right">Rockout</td>
<td align="center"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="right">SSC</td>
<td align="center"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">SU5402</td>
<td align="center">CALBIOCHEM</td>
<td align="left">572630</td>
</tr>
<tr class="even">
<td align="right">Torula RNA</td>
<td align="center">Sigma</td>
<td align="left">R6625</td>
</tr>
<tr class="odd">
<td align="right">Tricaine</td>
<td align="center">Fluka</td>
<td align="left">A5040</td>
</tr>
<tr class="even">
<td align="right">Tris Base</td>
<td align="center">Roth</td>
<td align="left">4855.2</td>
</tr>
<tr class="odd">
<td align="right">Triton-X100</td>
<td align="center">Roth</td>
<td align="left">3051.2</td>
</tr>
<tr class="even">
<td align="right">Trizol</td>
<td align="center">Ambion</td>
<td align="left">15596018</td>
</tr>
<tr class="odd">
<td align="right">Tween20</td>
<td align="center">Sigma</td>
<td align="left">P1379</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-sol" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Solutions</h3>
<table>
<caption><span id="tab:mat-sol">Table 2.2: </span> Solutions</caption>
<colgroup>
<col width="23%" />
<col width="59%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Solution</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Cut Smart Buffer</td>
<td align="center">NEB</td>
<td align="left">B7204S</td>
</tr>
<tr class="even">
<td align="right">Generuler 100 bp</td>
<td align="center">Thermo</td>
<td align="left">SM0241</td>
</tr>
<tr class="odd">
<td align="right">Generuler 1kb</td>
<td align="center">Thermo</td>
<td align="left">SM0311</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-anitb" class="section level3">
<h3><span class="header-section-number">2.1.3</span> Antibodies</h3>
<table>
<caption><span id="tab:mat-antib">Table 2.3: </span> Antibodies</caption>
<colgroup>
<col width="23%" />
<col width="59%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Antibody</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Anti-Digoxigenin</td>
<td align="center">Roche</td>
<td align="left">11093274910</td>
</tr>
<tr class="even">
<td align="right">JL8</td>
<td align="center"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">Anti-TAZ</td>
<td align="center"></td>
<td align="left"></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-enz" class="section level3">
<h3><span class="header-section-number">2.1.4</span> Enzymes</h3>
<table>
<caption><span id="tab:mat-enz">Table 2.4: </span> Enzymes</caption>
<colgroup>
<col width="27%" />
<col width="55%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Enzyme</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">BtsCI</td>
<td align="center">NEB</td>
<td align="left">#R0647</td>
</tr>
<tr class="even">
<td align="right">DdeI</td>
<td align="center">NEB</td>
<td align="left">#R0175</td>
</tr>
<tr class="odd">
<td align="right">HaeIII</td>
<td align="center">NEB</td>
<td align="left">#R0108</td>
</tr>
<tr class="even">
<td align="right">MnlII</td>
<td align="center">NEB</td>
<td align="left">#R0163</td>
</tr>
<tr class="odd">
<td align="right">NlaIII Restriction</td>
<td align="center">NEB</td>
<td align="left">#R0125</td>
</tr>
<tr class="even">
<td align="right">NsiI-HF Restriction</td>
<td align="center">NEB</td>
<td align="left">#R3127</td>
</tr>
<tr class="odd">
<td align="right">Phusion Polymerase</td>
<td align="center">NEB</td>
<td align="left">#M0530L</td>
</tr>
<tr class="even">
<td align="right">Ribolock</td>
<td align="center">Thermo</td>
<td align="left">#EO0381</td>
</tr>
<tr class="odd">
<td align="right">RNase A</td>
<td align="center">Quiagen</td>
<td align="left">1006657</td>
</tr>
<tr class="even">
<td align="right">RNase H</td>
<td align="center">NEB</td>
<td align="left">M0297L</td>
</tr>
<tr class="odd">
<td align="right">sp6 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">#EP0131</td>
</tr>
<tr class="even">
<td align="right">T4 Ligase</td>
<td align="center">NEB</td>
<td align="left">#M0202T</td>
</tr>
<tr class="odd">
<td align="right">T7 RNA Polymerase</td>
<td align="center">Thermo</td>
<td align="left">#EP0111</td>
</tr>
<tr class="even">
<td align="right">Taq DNA Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">#10342-020</td>
</tr>
<tr class="odd">
<td align="right">Taq DNA Polymerase</td>
<td align="center">VWR</td>
<td align="left"></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-mobikits" class="section level3">
<h3><span class="header-section-number">2.1.5</span> Molecular Biology Kits</h3>
<table>
<caption><span id="tab:mat-mobikits">Table 2.5: </span> Molecular Biology Kits</caption>
<colgroup>
<col width="54%" />
<col width="28%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Kit</strong></th>
<th align="center">Company</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">mMessage mMachine Sp6 Polymerase</td>
<td align="center">Invitrogen</td>
<td align="left">AM1340</td>
</tr>
<tr class="even">
<td align="right">pGEM-T TA Cloning</td>
<td align="center">Promega</td>
<td align="left">A3600</td>
</tr>
<tr class="odd">
<td align="right">Superscript III cDNA Synthesis</td>
<td align="center">Thermo</td>
<td align="left">18080051</td>
</tr>
<tr class="even">
<td align="right">Wizard SV Gel and PCR Clean-Up</td>
<td align="center">Promega</td>
<td align="left">A9282</td>
</tr>
<tr class="odd">
<td align="right">PCR &amp; Gel Clean-Up</td>
<td align="center">Sigma</td>
<td align="left"></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-buff" class="section level3">
<h3><span class="header-section-number">2.1.6</span> Buffers</h3>
<table>
<caption><span id="tab:mat-buff">Table 2.6: </span> Buffers</caption>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Buffer</strong></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Blocking Reagent (BR)</td>
<td></td>
</tr>
<tr class="even">
<td align="left">Cut Smart Buffer</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">E3</td>
<td></td>
</tr>
<tr class="even">
<td align="left">Hybridization buffer</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">LB Medium / Agar</td>
<td></td>
</tr>
<tr class="even">
<td align="left">NTMT</td>
<td>75 mM NaCl + 15 mM MgCl + 15 mM</td>
</tr>
<tr class="odd">
<td align="left">PBS</td>
<td>2.7 mM KCl + 12 mM HPO</td>
</tr>
<tr class="even">
<td align="left">PBST</td>
<td>PBS + 0.1% Tween20</td>
</tr>
<tr class="odd">
<td align="left">PBDT</td>
<td>PBS + 1% BSA + 1% DMSO + 0.3% Triton</td>
</tr>
<tr class="even">
<td align="left">P1</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">P2</td>
<td></td>
</tr>
<tr class="even">
<td align="left">P3</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">Tricaine</td>
<td></td>
</tr>
<tr class="even">
<td align="left">TNT</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-lines" class="section level3">
<h3><span class="header-section-number">2.1.7</span> Zebrafish lines</h3>
<table>
<caption><span id="tab:mat-lines">Table 2.7: </span> Zebrafish lines</caption>
<colgroup>
<col width="13%" />
<col width="31%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Allele</strong></th>
<th align="center"><strong>name</strong></th>
<th align="left"><strong>zfin</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">zf106Tg</td>
<td align="center"><em>cldnb:lyn-gfp</em></td>
<td align="left"><a href="//zfin.org/ZDB-ALT-060919-2"><em>Tg(-8.0cldnb:LY-EGFP)</em></a></td>
</tr>
<tr class="even">
<td align="right">fu13Tg</td>
<td align="center"><em>cxcr4b</em>(BAC):<em>H2BRFP</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">nns8Tg</td>
<td align="center"><em>atoh1a:Tom</em></td>
<td align="left"><a href="//zfin.org/ZDB-FISH-150901-21622"><em>atoh1a</em></a></td>
</tr>
<tr class="even">
<td align="right">fu50</td>
<td align="center"><em>shroom3</em></td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="right">m1274Tg</td>
<td align="center"><em>hsp70:shr3v1FL-taqRFP</em></td>
<td align="left">-</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-probes" class="section level3">
<h3><span class="header-section-number">2.1.8</span> ISH probes</h3>
<table>
<caption><span id="tab:mat-probes">Table 2.8: </span> ISH probes</caption>
<colgroup>
<col width="11%" />
<col width="88%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Probe</strong></th>
<th align="left"><strong>Sequence</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">atoh1a</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">deltaD</td>
<td align="left">see <span class="citation">(<em>45</em>)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="mat-hrdwr" class="section level3">
<h3><span class="header-section-number">2.1.9</span> Hardware</h3>
<div id="mat-stamp" class="section level4">
<h4><span class="header-section-number">2.1.9.1</span> Mounting Stamp</h4>
<p>The stl file for printing can be found at <a href="https://github.com/KleinhansDa/3DModels">github.com/KleinhansDa/3DModels</a></p>
</div>
<div id="mat-SD" class="section level4">
<h4><span class="header-section-number">2.1.9.2</span> Spinning Disc Microscopy</h4>
<table>
<caption><span id="tab:SDcomp">Table 2.9: </span> Spinning Disc system components</caption>
<colgroup>
<col width="22%" />
<col width="18%" />
<col width="25%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Microscope</td>
<td align="center">Nikon</td>
<td align="center">Eclipse Ti-E</td>
<td align="left">fully motorized</td>
</tr>
<tr class="even">
<td align="right">PFS</td>
<td align="center">Nikon</td>
<td align="center">Perfect focus system</td>
<td align="left">Z repositioning</td>
</tr>
<tr class="odd">
<td align="right">XY-table</td>
<td align="center">Merzhaeuser</td>
<td align="center">XY motorized table</td>
<td align="left">1 <span class="math inline">\(\mu\)</span>m accuracy</td>
</tr>
<tr class="even">
<td align="right">Piezo</td>
<td align="center"></td>
<td align="center">Piezo Z-table</td>
<td align="left">300 <span class="math inline">\(\mu\)</span>m scan range</td>
</tr>
<tr class="odd">
<td align="right">SD system</td>
<td align="center">Yokogawa</td>
<td align="center">CSU-W1</td>
<td align="left">50 <span class="math inline">\(\mu\)</span>m pattern</td>
</tr>
<tr class="even">
<td align="right">Laser</td>
<td align="center"></td>
<td align="center">Laser Combiner</td>
<td align="left">see table <a href="2-mat-met.html#tab:lasers">2.10</a></td>
</tr>
<tr class="odd">
<td align="right">FRAPPA</td>
<td align="center">Revolution</td>
<td align="center">FRAPPA</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="right">Borealis</td>
<td align="center">Borealis</td>
<td align="center">Borealis</td>
<td align="left">flat field correction</td>
</tr>
<tr class="odd">
<td align="right">sCMOS</td>
<td align="center">Andor</td>
<td align="center">ZYLA PLUS</td>
<td align="left">4.2Mpix; 82%QE</td>
</tr>
<tr class="even">
<td align="right">Immersion</td>
<td align="center">Merzhaeuser</td>
<td align="center">Liquid Dispenser</td>
<td align="left">-</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:lasers">Table 2.10: </span> Available lasers</caption>
<colgroup>
<col width="40%" />
<col width="20%" />
<col width="40%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Lasers</strong></th>
<th align="center">Type</th>
<th align="left">Power</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">405 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">445 nm</td>
<td align="center">diode</td>
<td align="left">80 mW</td>
</tr>
<tr class="odd">
<td align="right">488 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="even">
<td align="right">561 nm</td>
<td align="center">DPSS</td>
<td align="left">100 mW</td>
</tr>
<tr class="odd">
<td align="right">640 nm</td>
<td align="center">diode</td>
<td align="left">100 mW</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:objectives">Table 2.11: </span> Available objectives</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="17%" />
<col width="15%" />
<col width="9%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Objective</strong></th>
<th align="center">Company</th>
<th align="center">Type</th>
<th align="center">Immersion</th>
<th align="center">N.A.</th>
<th align="left">working distance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">10x</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">air</td>
<td align="center">0.45</td>
<td align="left">4.00 mm</td>
</tr>
<tr class="even">
<td align="right">20X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">0.95</td>
<td align="left">0.95 mm</td>
</tr>
<tr class="odd">
<td align="right">40X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.15</td>
<td align="left">0.60 mm</td>
</tr>
<tr class="even">
<td align="right">60X</td>
<td align="center">Nikon</td>
<td align="center">CFI APO</td>
<td align="center">water</td>
<td align="center">1.20</td>
<td align="left">0.30 mm</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-work" class="section level4">
<h4><span class="header-section-number">2.1.9.3</span> Workstation</h4>
<p>Statistical computation and image analysis were done on a Fujitsu Siemens (FS) Workstation CELSIUSM740 with the following hardware components…</p>
<table>
<caption><span id="tab:PCcomp">Table 2.12: </span> Workstation hardware components</caption>
<colgroup>
<col width="21%" />
<col width="22%" />
<col width="25%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Component</strong></th>
<th align="center">Company</th>
<th align="center">Product</th>
<th align="left">Specs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">CPU</td>
<td align="center">Intel</td>
<td align="center">Xeon E5-1660v4</td>
<td align="left">3.2 GHz, 20MB, 8cores</td>
</tr>
<tr class="even">
<td align="right">RAM</td>
<td align="center">Fujitsu</td>
<td align="center">-</td>
<td align="left">4x16GB DDR4-2400</td>
</tr>
<tr class="odd">
<td align="right">Graphics</td>
<td align="center">NVIDIA</td>
<td align="center">Quadro M4000</td>
<td align="left">8GB RAM</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="mat-sftwr" class="section level3">
<h3><span class="header-section-number">2.1.10</span> Software</h3>
<table>
<caption><span id="tab:sftwr">Table 2.13: </span> Used software</caption>
<colgroup>
<col width="39%" />
<col width="19%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"><strong>Software</strong></th>
<th align="center">Version</th>
<th align="left">web</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Imagej FIJI</td>
<td align="center">1.48</td>
<td align="left">[<a href="https://fiji.sc/" class="uri">https://fiji.sc/</a>]</td>
</tr>
<tr class="even">
<td align="right">RStudio</td>
<td align="center">1.0.153</td>
<td align="left">[<a href="https://www.rstudio.com/" class="uri">https://www.rstudio.com/</a>]</td>
</tr>
<tr class="odd">
<td align="right">Ubuntu</td>
<td align="center">17.1</td>
<td align="left">[<a href="https://www.ubuntu.com/" class="uri">https://www.ubuntu.com/</a>]</td>
</tr>
<tr class="even">
<td align="right">Windows 10 Pro</td>
<td align="center">10.0.16299</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="right">Total Commander</td>
<td align="center">9.0</td>
<td align="left">[<a href="http://ghisler.com/" class="uri">http://ghisler.com/</a>]</td>
</tr>
</tbody>
</table>
</div>
<div id="mat-GrTrDat" class="section level3">
<h3><span class="header-section-number">2.1.11</span> Ground Truth Data</h3>
<p>Analyzing images and extracting quantitative measurements can be a tedious task, especially when the analysis becomes more complex. Fortunately, there are ways to automate image analysis by using either machine learning approaches or by tailoring hand-crafted algorithms in an image analysis software tool like e.g. ImageJ<span class="citation">(<em>46</em>)</span>.
The main advantages of doing so are…</p>
<ul>
<li>to be independent of confirmation bias</li>
<li>to make the analysis more robust against oversight</li>
<li>to increase the statistical power by increasing the number of data points<span class="citation">(<em>47</em>)</span></li>
<li>to increase effect size by increasing the measurement accuracy<span class="citation">(<em>47</em>)</span></li>
</ul>
<p>However, to reassure the measurements taken by a tailored or trained algorithm are meaningful, they must be compared to a <em>ground truth</em> dataset which again describes a general measure of algorithmic quality performance<span class="citation">(<em>48</em>)</span>.</p>
<div id="d-ground-truth" class="section level4">
<h4><span class="header-section-number">2.1.11.1</span> 2-D Ground Truth</h4>
<div id="ground-truth-design" class="section level5">
<h5><span class="header-section-number">2.1.11.1.1</span> Ground Truth Design</h5>
<p>To assess the quality of the <em>ClusterAnalysis</em> algorithm for nuclei detection the ground truth was designed as follows.</p>
<div id="model" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.1</span> Model</h6>
<ul>
<li>each CellCluster (CC) consists of a number of objects (cells)</li>
<li>each object is part of the respective CC and defines one cell entity</li>
<li>each object is determined via a fluorescent nucleus label</li>
<li>embryos are mounted within a 3D mold to reduce noise</li>
</ul>
<table style="width:97%;">
<caption><span id="tab:imgcond3DGrT">Table 2.14: </span> Cluster Analysis Model</caption>
<colgroup>
<col width="48%" />
<col width="48%" />
</colgroup>
<tbody>
<tr class="odd">
<td align="right">XY scale</td>
<td align="left">0.32 px/<span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="even">
<td align="right">Z-spacing</td>
<td align="left">4 <span class="math inline">\(\mu\)</span>m</td>
</tr>
<tr class="odd">
<td align="right">Camera</td>
<td align="left">Rolera</td>
</tr>
</tbody>
</table>
</div>
<div id="training-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.2</span> Training data</h6>
<p>The training set consists of three randomly picked wild type PLLps. For each the algorithm is run with standard parameters. Cell cluster ROIs and nuclei multi point labels are edited manually.</p>
</div>
<div id="test-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.1.3</span> Test data</h6>
<p>To test the algorithm, it is run at different nuclei detection thresholds on the same image data.</p>
</div>
</div>
<div id="results" class="section level5">
<h5><span class="header-section-number">2.1.11.1.2</span> Results</h5>
<div id="image-analysis" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.1</span> Image Analysis</h6>
<p>For image analysis a custom IJ macro script was developed that segments individual cell clusters based on morphological filtering<span class="citation">(<em>49</em>)</span>, Gaussian blurring and subsequent thresholding. The user is then presented the segmentation ROIs to manually edit them, if necessary. Using those ROIs as masks, the nuclei within them are count based on Gaussian blurring and a 2-D maxima finder. To detect the right amount of nuclei however, it is necessary to evaluate the distance over which the blurring should be applied.</p>
<p>A typical nucleus in the PLLp is about 5 <span class="math inline">\(\mu\)</span>m in diameter. To determine the right blurring value, a range of 4-6 <span class="math inline">\(\mu\)</span>m in steps of 0.5 is tested.</p>
</div>
<div id="macro-modules" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.2</span> Macro modules</h6>
<p>The macro developed consists of the following main modules:</p>
<ol style="list-style-type: decimal">
<li>Background subtraction, morphological filtering, Gaussian blurring &amp; segmentation</li>
<li>Manual correction for cell cluster segmentation masks</li>
<li>ROI indexing from left (1) to right (n)</li>
<li>Nuclei counting within cell cluster masks</li>
<li>Export of results table (Area, Count, …)</li>
</ol>
</div>
<div id="registered-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.3</span> Registered data</h6>
<p>Figure <a href="2-mat-met.html#fig:maxllreg">2.1</a> shows a registered maximum Z projected lateral line used for ground truth evaluation.</p>
(ref:maxllreg) Registered, Maximum Z projected data with cell cluster labels
<div class="figure" style="text-align: center"><span id="fig:maxllreg"></span>
<img src="figure/02-MaMo/GrTr/ClusterAnal/B06SB1P05_montage.png" alt="(ref:maxllreg)" width="90%" />
<p class="caption">
Figure 2.1: (ref:maxllreg)
</p>
</div>
</div>
<div id="ground-truth-data" class="section level6">
<h6><span class="header-section-number">2.1.11.1.2.4</span> Ground Truth data</h6>
<p>Figure <a href="2-mat-met.html#fig:countP05">2.2</a> shows the single registered cell clusters from figure 1 and the manually labeled nuclei within.</p>
(ref:countP05) Nuclei labels
<div class="figure" style="text-align: center"><span id="fig:countP05"></span>
<img src="figure/02-MaMo/GrTr/ClusterAnal/count_P05.png" alt="(ref:countP05)" width="90%" />
<p class="caption">
Figure 2.2: (ref:countP05)
</p>
</div>
</div>
</div>
<div id="data-analysis" class="section level5">
<h5><span class="header-section-number">2.1.11.1.3</span> Data Analysis</h5>
<div id="nuclei-count" class="section level6">
<h6><span class="header-section-number">2.1.11.1.3.1</span> Nuclei Count</h6>
<p>Comparing the segmented objects count of the Gaussian parameters with the ground truth gives an indication for over- resp. under-segmentation. In figure <a href="2-mat-met.html#fig:GrTratioCC">2.3</a> the relative numbers for each parameter can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). The red area represents over-segmentation, the green under-segmentation.</p>

<div class="figure" style="text-align: center"><span id="fig:GrTratioCC"></span>
<img src="thesis_files/figure-html/GrTratioCC-1.png" alt="Relative difference of segment counts (N=3, Errorbars = Std./2)" width="65%" />
<p class="caption">
Figure 2.3: Relative difference of segment counts (N=3, Errorbars = Std./2)
</p>
</div>
<p>To estimate the quality of nuclei detection for each parameter, the ratio of automatically detected and ground truth objects count can be calculated and compared (table <a href="#tab:nrcount"><strong>??</strong></a>). The closer it is to 1, the better.</p>

</div>
</div>
<div id="summary" class="section level5">
<h5><span class="header-section-number">2.1.11.1.4</span> Summary</h5>
<p>Maximum performance is achieved at a scaled parameter of 6 <span class="math inline">\(\mu\)</span>m, with a ratio of of 0.95 for the count objects and a standard deviation of 0.14.</p>
</div>
</div>
<div id="d-ground-truth-1" class="section level4">
<h4><span class="header-section-number">2.1.11.2</span> 3-D Ground Truth</h4>
<div id="ground-truth-design-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.1</span> Ground Truth Design</h5>
<p>To assess the quality of the anaLLzr3D algorithm the ground truth was modeled as follows.</p>
<div id="model-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.1</span> Model</h6>
<ul>
<li>each PLLp consists of a number of objects (cells)</li>
<li>each object is part of the PLLp and defines one cell entity</li>
<li>cell boundaries are determined via the transgene <em>cldnb:lyn-gfp</em> +/+</li>
<li>embryos are imaged live to conserve signal and membrane integrity</li>
<li>embryos are mounted within a 3D mold for improved 3D alignment</li>
</ul>

</div>
<div id="training-data-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.2</span> Training data</h6>
<p>The training set consists of three randomly picked wildtype PLLps. For each the algorithm is run with no filters (X, Y, Z border objects, size) and a minimum segmentation threshold. Afterwards the segmentation result is corrected manually for over- and under-segmentation and objects that are not part of the PLLp.</p>
</div>
<div id="test-data-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.1.3</span> Test data</h6>
<p>To test the algorithm it is run at different segmentation thresholds on the same image data.</p>
</div>
</div>
<div id="results-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.2</span> Results</h5>
<div id="image-analysis-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.1</span> Image Analysis</h6>
<p>For image analysis a custom IJ macro script was developed that recognizes the cell boundaries via the fluorescence signal emitted by a Lyn Kinase tethered eGFP which expression is controlled by the <em>cxcr4b</em> lateral line specific promotor <span class="citation">(<em>21</em>)</span>. The central IJ tool used to do this is the MorphoLibJ’s<span class="citation">(<em>49</em>)</span> <em>Morpholigical Segmentation</em> plugin. The plugin however requires to choose for a ‘segmentation threshold’ that determines the quality and the quantity of segmented objects. This parameter therefore plays an essential role in the truthfulness of the analysis results.</p>
</div>
<div id="macro-modules-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.2</span> Macro modules</h6>
<p>The macro developed consists of the following main modules:</p>
<ol style="list-style-type: decimal">
<li>PLLp registration and cropping in X, Y and Z</li>
<li>Image segmentation using the MorphoLibJ’s<span class="citation">(<em>49</em>)</span> <em>Morpholigical Segmentation</em> plugin</li>
<li>Extration of 3-D measurements using the 3D ImageJ Suite’s<span class="citation">(<em>50</em>)</span> <em>3D Manager</em> plugin</li>
<li>Cropping, clearing and measurement of single cell volumes using a custom IJ script</li>
</ol>
</div>
<div id="registration" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.3</span> Registration</h6>
<p>The first module of the macro is PLLp registration in X, Y and cropping in Z. This is accomplished by an initial maximum Z projection and blurring of the image, 2-D segmentation using a minimum threshold and lastly by measuring the angle the segment is rotated against the horizon (at 0<span class="math inline">\(^{\circ}\)</span>). After rotation the PLLp is then registered in Y and the image is cropped by drawing a rectangle with fixed height.</p>

<div class="figure" style="text-align: center"><span id="fig:maxraw"></span>
<img src="figures/materials/ground_truth/registration.png" alt="Registration of 3D data" width="90%" />
<p class="caption">
Figure 2.4: Registration of 3D data
</p>
</div>

<p>During the registration process, the image is rotated against the horizon and cropped in X and Y. Additionally, the centers of the most constricting areas are detected via an intensity based dynamic threshold and highlighted as magenta circles in <a href="2-mat-met.html#fig:maxraw">2.4</a>.</p>
</div>
<div id="ground-truth-data-1" class="section level6">
<h6><span class="header-section-number">2.1.11.2.2.4</span> Ground Truth data</h6>
<p>Watershed segmentation depends on a continuous signal along the boundary of the object to be segmented. Since the PLLp lies within the zebrafish larvae and since light gets scattered more when traveling through more organic tissues, the fluorescence signal in deeper tissue sections is more disturbed. Still, the PLLp is located just superficial to the skin, so the effect may be minor. In figure <a href="#fig:stackmem"><strong>??</strong></a> the fluorescence signal of the three PLLps tested is shown in a single central cross-section in the dorso-ventral and the apico-basal axis.</p>
<p>To compare the results between the ground truth segments and the segments obtained from different threshold levels graphically, for a single PLLp the ground truth and threshold level[n] are shown as a composite color image in figure <a href="2-mat-met.html#fig:stackcomp">2.5</a>. By using a green lookup table (LUT) for the ground truth and a magenta LUT for the threshold level[n], one can readily detect overlapping objects (white), over segmentation (magenta) and under segmentation (green).</p>

<div class="figure" style="text-align: center"><span id="fig:stackcomp"></span>
<img src="figures/materials/ground_truth/volumes.png" alt="Graphical comparison of the thresholds tested" width="85%" />
<p class="caption">
Figure 2.5: Graphical comparison of the thresholds tested
</p>
</div>
<p>Volume renderings have been done with IJ’s <a href="%22https://github.com/fiji/Volume_Viewer/releases/tag/Volume_Viewer-2.01.2%22">VolumeViewer</a></p>
</div>
</div>
<div id="data-analysis-1" class="section level5">
<h5><span class="header-section-number">2.1.11.2.3</span> Data Analysis</h5>
<div id="cell-count" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.1</span> Cell Count</h6>
<p>In figure <a href="2-mat-met.html#fig:gtratio">2.6</a> the relative numbers for each threshold level can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). Analogous to the graphical inspection, the magenta area represents over segmentation, the green under segmentation. Additionally, table <a href="#tab:gtrcellcount"><strong>??</strong></a> shows the absolute numbers.</p>

<div class="figure" style="text-align: center"><span id="fig:gtratio"></span>
<img src="thesis_files/figure-html/gtratio-1.png" alt="Relative difference of segment counts (N=3, Errorbars = Std./2)" width="65%" />
<p class="caption">
Figure 2.6: Relative difference of segment counts (N=3, Errorbars = Std./2)
</p>
</div>

</div>
<div id="cell-morphology" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.2</span> Cell Morphology</h6>
<blockquote>
<p><em>There is always a precision / recall trade-off in detection tasks, e.g., when we set a lower threshold [. . .], we can get a higher recall with a lower precision ([. . .], but meanwhile we also get more false-positives in the results).</em><span class="citation">(<em>41</em>)</span></p>
</blockquote>
<p>Inspecting the cell count unfortunately does not directly tell us how well the cell morphology is conserved at different threshold levels, since at higher threshold levels the cell boundaries are differently determined and eventually not even recognized as such anymore.</p>
<p>The volume of a cell is a very robust morphological feature. Therefore, if its volume does not differ significantly at a given threshold level we consider its morphology to be conserved.</p>
</div>
<div id="sample-distribution" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.3</span> Sample distribution</h6>
<p>In figure <a href="2-mat-met.html#fig:volsecdffull">2.7</a> and <a href="2-mat-met.html#fig:volsecdfzoom">2.8</a> the distribution and amount of the cell volumes across the different threshold levels can be inspected via the ‘empirical cumulative distribution function’ (ECDF). The closer the slopes are to the ground truth, the stronger they are conserved. Figure <a href="2-mat-met.html#fig:volsecdffull">2.7</a> shows the full distribution, where the major differences seem to appear within the 0.4 quantile (red line). Therefore, in figure <a href="2-mat-met.html#fig:volsecdfzoom">2.8</a> only the values within the 0.4 quantile are shown to get a more detailed impression.</p>
(ref:volsecdffull) ECDF - All cell volumes
<div class="figure" style="text-align: center"><span id="fig:volsecdffull"></span>
<img src="thesis_files/figure-html/volsecdffull-1.png" alt="(ref:volsecdffull)" width="90%" />
<p class="caption">
Figure 2.7: (ref:volsecdffull)
</p>
</div>
(ref:volsecdfzoom) ECDF - Small cell volumes
<div class="figure" style="text-align: center"><span id="fig:volsecdfzoom"></span>
<img src="thesis_files/figure-html/volsecdfzoom-1.png" alt="(ref:volsecdfzoom)" width="90%" />
<p class="caption">
Figure 2.8: (ref:volsecdfzoom)
</p>
</div>
</div>
<div id="confidence-level" class="section level6">
<h6><span class="header-section-number">2.1.11.2.3.4</span> Confidence level</h6>
<p>To statistically check how closely related each threshold value sample distribution is compared to the ground truth, a Kolmogorov-Smirnov-Test (ks.test) was performed. The ks.test is a nonparametric test whose null hypothesis is that the both groups compared were sampled from populations with identical populations. Therefore, the closer the p-value is to 1, the more similar the tested sample distribution would be to the ground truth (figure <a href="2-mat-met.html#fig:kstestvol">2.9</a>; 5% confidence threshold indicated by red line).</p>
<p><span class="math display">\[H_{0} : F_X(x)=F_0(x)\]</span></p>

<div class="figure" style="text-align: center"><span id="fig:kstestvol"></span>
<img src="thesis_files/figure-html/kstestvol-1.png" alt="Volumetric conservation confidence level" width="60%" />
<p class="caption">
Figure 2.9: Volumetric conservation confidence level
</p>
</div>
<p>In comparison to the Mann-Whitney test, the ks.test is more sensitive to detect changes in the shape of the distribution than to detect a shift of the median<span class="citation">(<em>51</em>)</span></p>
</div>
</div>
</div>
</div>
</div>
<div id="met" class="section level2">
<h2><span class="header-section-number">2.2</span> Methods</h2>
<div id="Zeb-met" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Zebrafish</h3>
<div id="husbandry" class="section level4">
<h4><span class="header-section-number">2.2.1.1</span> Husbandry</h4>
<p>The zebrafish husbandry is maintained at the University of Frankfurt am Main. All the legal procedures were followed while handling and maintaining the zebrafish husbandry. All the zebrafish lines used in and generated for this study are listed in the Appendix A.</p>
</div>
<div id="handling-and-rearing" class="section level4">
<h4><span class="header-section-number">2.2.1.2</span> Handling and rearing</h4>
<p>12 h before the embryo collection, males and females are set up in crossing cages filled ~75. Next day, before noon, separators are removed and eggs are laid. The zebrafish eggs are then collected and reared untill the larval stage (Day 5) in the well-defined culture medium E3 (Kimmel et.al. 1995, Table no. 6 in ‘List of Buffers’ section) at 25, 28.5, or 30 degrees celsius, according to the need of experiments.</p>
<p>To grow larvae to the adulthood, they were transferred to the system on day 5. Till day 12, larvae were fed Vinegar Eels, Paramecia, and Caviar powder. After the 12 th day, water supply was started and fish were fed Brine Shrimp, Artemia, Paramecia and Vinegar Eels. Adult fish (&gt;1 Month) were fed Artemia and the dry flakes.</p>
</div>
<div id="zebrafish-fin-clips" class="section level4">
<h4><span class="header-section-number">2.2.1.3</span> Zebrafish fin clips</h4>
<p>Adult fish were anesthetized with buffered Tricaine (1X, see ‘List of Buffers’) until stage II-III of anaestezia. About 1/3 rd of the pectoral fin was cut with a sterile scalpel in a sterile petri plate. Immediately the cut fin was transferred to 50 mM NaOH. Fish were returned to system water and kept in the single tanks with 200 micro liters of 0.01 percent methylen blue, maximum up to 4 days.</p>
</div>
<div id="adult-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.1.4</span> Adult Genotyping</h4>
<p>The clipped fins were digested for 1 h at 95<span class="math inline">\(^\circ\)</span> celsius and neutralized subsequently with 1M Tris-HCl of pH 9.</p>
</div>
<div id="embryo-genotyping" class="section level4">
<h4><span class="header-section-number">2.2.1.5</span> Embryo Genotyping</h4>
<p>Single fixed/live embryos were denatured at 95<span class="math inline">\(^\circ\)</span> Celsius in 50mM NaOH for 1 hour and neutralized by adding 1/10 the volume of 1 M Tris-HCl (pH 8.2).</p>
</div>
<div id="zebrafish-euthanasia" class="section level4">
<h4><span class="header-section-number">2.2.1.6</span> Zebrafish Euthanasia</h4>
<p>Adult zebrafish were anesthetized in tricaine (1X) till stage II-III of anaestezia and put in the ice cold water so as to sacrifice them by hypothermia.</p>
</div>
<div id="fixation" class="section level4">
<h4><span class="header-section-number">2.2.1.7</span> Fixation</h4>
<p>Embryos and dechorionated larvae were fixed in 2 ml of 4<span class="math inline">\(\%\)</span> PFA in 1X PBS overnight at 4<span class="math inline">\(^\circ\)</span> Celsius.</p>
</div>
</div>
<div id="Wet-met" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Wetlab</h3>
<div id="ISH-met" class="section level4">
<h4><span class="header-section-number">2.2.2.1</span> In Situ Hybridization</h4>
<div id="breeding-fixation" class="section level5">
<h5><span class="header-section-number">2.2.2.1.1</span> 0. Breeding &amp; Fixation</h5>
<ul>
<li>Embryos are grown till desired stage
<ul>
<li>from 24 hpf on in 1X PTU</li>
</ul></li>
<li>Dechorionation in 0.1 mg/mL Pronase
<ul>
<li>150 <span class="math inline">\(\mu\)</span>L per 10 mL for ~30 min.</li>
<li>For embryos younger than 18s, keep embryos in solution while washing and gently add fresh E3</li>
</ul></li>
<li>Fixation at desired stage in 4% PFA in 0.1% PBST at 4<span class="math inline">\(^\circ\)</span>C o.n.</li>
<li>Rinse 3x5 min. in PBST</li>
<li>Pass through MeOH series (25% <span class="math inline">\(\rightarrow\)</span> 50% <span class="math inline">\(\rightarrow\)</span> 75% <span class="math inline">\(\rightarrow\)</span> 100% MeOH/PBST (V/V))</li>
<li>Store in 100% MeOH at -20<span class="math inline">\(^\circ\)</span>C for &gt; 2h</li>
</ul>
</div>
<div id="permeabilisation-probe-hybridization" class="section level5">
<h5><span class="header-section-number">2.2.2.1.2</span> 1. Permeabilisation &amp; Probe Hybridization </h5>
<p><strong>Permeabilisation</strong> <span class="math inline">\(\rightarrow\)</span> without shaking</p>
<ul>
<li>Rehydrate embryos (75% <span class="math inline">\(\rightarrow\)</span> 50% <span class="math inline">\(\rightarrow\)</span> 25% <span class="math inline">\(\rightarrow\)</span> 0% PBST)</li>
<li>Wash 2x in PBST</li>
<li>Digest embryos in 10 <span class="math inline">\(\mu\)</span>g/mL Proteinase K according to table</li>
</ul>

<ul>
<li>Wash 2x in PBST</li>
<li>PostFix in 4% PFA at 4<span class="math inline">\(^\circ\)</span>C for &gt; 30 min.</li>
<li>Wash 3x in PBST</li>
</ul>
<p><strong>Probe Hybridisation</strong> <span class="math inline">\(\rightarrow\)</span> all steps at 60<span class="math inline">\(^\circ\)</span>C, except stated differently</p>
<ul>
<li>Prehybridise in 350 <span class="math inline">\(\mu\)</span>L of Hyb buffer for 1 - 8 h</li>
<li>Heat the probe in 200 <span class="math inline">\(\mu\)</span>L of Hyb to 80<span class="math inline">\(^\circ\)</span>C, then replace Hyb buffer with it</li>
<li>Incubate o.n.</li>
</ul>
</div>
<div id="probe-removal-antibody-incubation" class="section level5">
<h5><span class="header-section-number">2.2.2.1.3</span> 2. Probe removal &amp; Antibody incubation</h5>
<ul>
<li>Collect probe <span class="math inline">\(\rightarrow\)</span> For reuse, store at -20<span class="math inline">\(^\circ\)</span>C</li>
<li>Wash at the same temperature as hybridisation <span class="math inline">\(\rightarrow\)</span> use Thermo-Block and waterbath or oven to keep solutions at temperature
<ul>
<li>1 x 20 min. Hyb buffer</li>
<li>2 x 30 min. 50% Formamide</li>
<li>1 x 20 min. 25% Formamide</li>
<li>2 x 15 min. 2X SSCT</li>
<li>2 x 30 min. 0.2X SSCT</li>
<li>1 x 5 min. TNT</li>
</ul></li>
<li>Block for 1 - 8 h in 350 <span class="math inline">\(\mu\)</span>L of 2% BR/TNT at RT</li>
<li>Incubate in 100 <span class="math inline">\(\mu\)</span>L antibody in 2% BR/TNT for 2 h at RT or o.n. at 4<span class="math inline">\(^\circ\)</span>C
<ul>
<li>antibody = Anti-Digoxigenin (1/4000 (V/V))</li>
<li>diluent = NTMT buffer</li>
</ul></li>
</ul>
</div>
<div id="probe-detection" class="section level5">
<h5><span class="header-section-number">2.2.2.1.4</span> 3. Probe detection</h5>
<ul>
<li>Wash 6 x 20 min. in TNT (or one wash o.n.)</li>
<li>Wash 2 x 5 min. in NTMT</li>
<li>Replace with…
<ul>
<li>4.5 NBT <span class="math inline">\(\mu\)</span>L + 3.5 <span class="math inline">\(\mu\)</span>L BCIP per mL NTMT or</li>
<li>150 <span class="math inline">\(\mu\)</span>L BM Purple</li>
</ul></li>
<li>Develop at RT in the dark without shaking <span class="math inline">\(\rightarrow\)</span> 2 - 8 h depending on probe</li>
<li>Wash 3x in PBST</li>
<li>If necessary, proceed to immunostaining</li>
<li>For permanent storage, keep specimen in 50% Glycerol at 4<span class="math inline">\(^\circ\)</span>C</li>
</ul>
</div>
</div>
<div id="immuno-met" class="section level4">
<h4><span class="header-section-number">2.2.2.2</span> Immuno staining</h4>
<ul>
<li>Breeding and Fixation as for ISH</li>
<li>Block for 30 min. in 2% Goat Serum / PBDT (V/V)</li>
<li>Primary Antibody
<ul>
<li>Incubate for 2 h at RT (or o.n. at 4<span class="math inline">\(^\circ\)</span>C) in 150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)</li>
<li>Wash in PBDT for 2 h (change solution ~6x)</li>
</ul></li>
<li>Secondary Antibody
<ul>
<li>Incubate for 2 h at RT (or o.n. at 4<span class="math inline">\(^\circ\)</span>C) in 150 <span class="math inline">\(\mu\)</span>L of 2% NGS / PBDT (V/V)</li>
<li>Wash in PBDT for 2 h (change solution ~6x)</li>
</ul></li>
</ul>
</div>
<div id="mount-met" class="section level4">
<h4><span class="header-section-number">2.2.2.3</span> Mounting</h4>
<p>For mounting, an improved 3D specimen preparation and well-plate like sample navigation for zebrafish larvae confocal microscopy was developed which offers the following improvements…</p>
<ol style="list-style-type: decimal">
<li><strong>To obtain a sufficient amount of data that is closely staged and still comprises &gt; 4 independent groups.</strong> Using this protocol, it is possible to define a custom well plate (figure 5) in your preferred imaging software, which greatly facilitates the imaging procedure and therefore helps to stay focused. However, once the well plate is aligned and the positions are defined it still needs some minor correction in Z.</li>
<li><p><strong>To do the imaging live. Most of the time, especially in morphological analyses, live imaging is preferred since any fixation alters cell and therefore organ morphology.</strong> Live imaging guarantees an undisturbed and unbiased look at the in vivo situation. Using this protocol, the embryonic body axes will be more in line with the movement axes of the microscopic table which results, depending on the volume of interest, in narrower Z-stacks. This ultimately also leads to a reduction of photo-bleaching, phototoxicity and expenditure of time. The won time difference can then be used to image shorter time-intervals in timelapse imaging.</p></li>
<li><p><strong>To have the embryos aligned with the 3D movements of the microscopic table.</strong> Instead of transforming and aligning the image data in a post-processing approach, the specimen can be imaged in the right spacial configuration in the first place. This is not only important for 3D analysis, but also for Z projected data. If the specimen body axes are not aligned with the axes of the microscopic table, the image stacks need to be bigger to record the volume of interest. In the Z projections, this will end up in X Y coordinates not aligned in Z, as noise or false signal (figure 1B). The method therefore provides significant improvement over existing mounting techniques.</p></li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:Mountmicro"></span>
<img src="figure/02-MaMo/Mount/mountmicro.png" alt="Stamp and \(\mu\)-well properties A single micro well in XY and A’ in XZ B mounting without \(\mu\)-well and B’ mounting with \(\mu\)-well C elements and dimensions of the stamp." width="80%" />
<p class="caption">
Figure 2.10: Stamp and <span class="math inline">\(\mu\)</span>-well properties <strong>A</strong> single micro well in XY and <strong>A’</strong> in XZ <strong>B</strong> mounting without <span class="math inline">\(\mu\)</span>-well and <strong>B’</strong> mounting with <span class="math inline">\(\mu\)</span>-well <strong>C</strong> elements and dimensions of the stamp.
</p>
</div>
<div id="preparations" class="section level5">
<h5><span class="header-section-number">2.2.2.3.1</span> Preparations</h5>
<table>
<caption><span id="tab:mounthard">Table 2.15: </span> Mounting hardware</caption>
<colgroup>
<col width="29%" />
<col width="10%" />
<col width="47%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"><strong>item</strong></th>
<th align="center">Company</th>
<th align="center">Description</th>
<th align="left">cat.-no.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><span class="math inline">\(\mu\)</span>-Dish</td>
<td align="center">ibidi</td>
<td align="center">20-35 mm polymer, #1.5, high wall</td>
<td align="left">81218-200</td>
</tr>
<tr class="even">
<td align="center">3D Mould Stamp</td>
<td align="center">IBG3D</td>
<td align="center">20-35 mm</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="center">Preparation needles</td>
<td align="center">VWR</td>
<td align="center">-</td>
<td align="left">USBE5470</td>
</tr>
<tr class="even">
<td align="center">Glass pipettes</td>
<td align="center">Roth</td>
<td align="center">-</td>
<td align="left">4518</td>
</tr>
<tr class="odd">
<td align="center">Silicone bulbs</td>
<td align="center">VWR</td>
<td align="center">-</td>
<td align="left">612-2327</td>
</tr>
<tr class="even">
<td align="center">Microtubes 2 mL</td>
<td align="center">Sarstedt</td>
<td align="center">-</td>
<td align="left">2691</td>
</tr>
<tr class="odd">
<td align="center">Heating block</td>
<td align="center">PeqLab</td>
<td align="center">-</td>
<td align="left">HX2</td>
</tr>
<tr class="even">
<td align="center">Microwave oven</td>
<td align="center">Severin</td>
<td align="center">-</td>
<td align="left">MW7849</td>
</tr>
<tr class="odd">
<td align="center">Stereo microscope</td>
<td align="center">Leica</td>
<td align="center">-</td>
<td align="left">M165FC</td>
</tr>
<tr class="even">
<td align="center">Transmitted Light Base</td>
<td align="center">Leica</td>
<td align="center">-</td>
<td align="left">MDG36</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:mountsol">Table 2.16: </span> Mounting solutions</caption>
<colgroup>
<col width="41%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>solution</strong></th>
<th align="center">Info</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1.0% (m/V) Agarose / E3</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td align="left">0.3% (m/V) LMPA / E3</td>
<td align="center">incl. 20% (v/v) 25X TMS for live imaging</td>
</tr>
<tr class="odd">
<td align="left">0.5% (m/V) LMPA / E3</td>
<td align="center">incl. 20% (v/v) 25X TMS for live imaging</td>
</tr>
</tbody>
</table>
</div>
<div id="protocol" class="section level5">
<h5><span class="header-section-number">2.2.2.3.2</span> Protocol</h5>
<p>First, make sure your stamp is clean of dust or other particles. If necessary clean it with 70% Ethanol and / or pressured air. Of a 1% standard Agarose in E3, apply…</p>
<ul>
<li>600 <span class="math inline">\(\mu\)</span>L in case of a <span class="math inline">\(\varnothing\)</span> 35 mm ibidi dish</li>
<li>2000 <span class="math inline">\(\mu\)</span>L in case of a <span class="math inline">\(\varnothing\)</span> 50 mm mattek dish</li>
</ul>
<p>to the glass or polymer surface and place the stamp on it so that it touches the cover. Place it as central as possible. Excessive agarose not needed for the imprint will fill the surrounding area in the dish and stabilize the structure when removing the stamp again.</p>
<p>Gently rotate the dish so that excess agarose solution will cover the whole dish.</p>
<p>Wait for ~45 min., then use a clean preparation needle and slip it through stamp - polymerized agarose interface. Remove the stamp carefully by lifting it and holding the <span class="math inline">\(\mu\)</span>-Dish in place.</p>

<div class="figure" style="text-align: center"><span id="fig:stampprod"></span>
<img src="figures/materials/mounting/mounting.png" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="80%" />
<p class="caption">
Figure 2.11: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
</div>
</div>
</div>
<div id="comp-met" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Computational</h3>
<div id="CNN" class="section level4">
<h4><span class="header-section-number">2.2.3.1</span> Rosette Detection</h4>
<p>The method used for rosette detection is based on a convolutional neuronal network (CNN) and is similar method to what was used before by Ernst <em>et al.</em>(2012)<span class="citation">(<em>29</em>, <em>41</em>)</span> for the description of the <em>shroom3</em> morpholino injected phenotype. Since the former method was technically deprecated and since we had new data to detect rosettes, we updated the former method to a state of the art CNN using Caffe<span class="citation">(<em>52</em>)</span> as a backend. The training was done by our partners at the institute for informatics, Albert-Ludwigs-University Freiburg.</p>
<p>The CNN was trained on 17 wildtype DMSO control and SU5402 treated (Fgf signaling depleted embryos which do not form rosettes<span class="citation">(<em>27</em>)</span>) embryos each. In order togive the network something to learn, the data had to be labeled manually, which was done in imagej by placing multipoint rois at the center of the rosettes. The data was then further permutated to artificially increase the amount of training data and make the detection more robust against different kinds of input. Further parameters about the training data is listed in Table <a href="#tab:CNNtraining"><strong>??</strong></a>. One example for each is shown in figure <a href="2-mat-met.html#fig:CNNtrain">2.12</a>.</p>

<div class="figure" style="text-align: center"><span id="fig:CNNtrain"></span>
<img src="figure/02-MaMo/CNN/CNNtrain.png" alt="Example for rosette detection on the training data. left Maximum Z-projected input data. middle heatmap of scores. blue indicates a low score, red a high one. right score map projected onto the input data." width="75%" />
<p class="caption">
Figure 2.12: Example for rosette detection on the training data. <strong>left</strong> Maximum Z-projected input data. <strong>middle</strong> heatmap of scores. blue indicates a low score, red a high one. <strong>right</strong> score map projected onto the input data.
</p>
</div>
<p>The main advantages for using a neural network in a task like this are…</p>
<ol style="list-style-type: decimal">
<li>Objectivity
<ul>
<li>Unlike the human brain, one a CNN is trained it is static and does not keep on learning. This makes it possible to have an unbiased quantification of rosette counts.</li>
</ul></li>
<li>Degree of rosette registration
<ul>
<li>The output data are continous rational numbers (<span class="math inline">\(\mathbb{Q}\)</span>) instead of integers (<span class="math inline">\(\mathbb{Z}\)</span>) which does not only tell if a rosette is there fore not, but also for ‘how much’ (50-100%) it is there.</li>
</ul></li>
<li>Training is done relatively quick</li>
</ol>

</div>
<div id="prolif" class="section level4">
<h4><span class="header-section-number">2.2.3.2</span> Proliferation Analysis</h4>
<p>The basic principle is based on work done by Laguerre <em>et al.</em>, 2009<span class="citation">(<em>23</em>)</span>. For manual tracking of mitotic events ‘MTrackJ’<span class="citation">(<em>53</em>)</span> was used…</p>
<ul>
<li>Mitotic events were tracked either in the pLLP or deposited CCs exclusively from Prometa- / Meta-phase till Ana- / Telo-phase</li>
<li>‘Tracks’ and ‘Points’ data was saved as ‘.txt’ files</li>
</ul>
<p>Figure <a href="2-mat-met.html#fig:mitodatapoints">2.13</a> shows an exemplary track for the data analyzed.</p>
(ref:mitodatapoints) Tracking of mitotic events
<div class="figure" style="text-align: center"><span id="fig:mitodatapoints"></span>
<img src="figure/02-MaMo/Prol/Prolif.png" alt="(ref:mitodatapoints)" width="50%" />
<p class="caption">
Figure 2.13: (ref:mitodatapoints)
</p>
</div>
</div>
<div id="ACI" class="section level4">
<h4><span class="header-section-number">2.2.3.3</span> Apical Index</h4>
<div id="rationale" class="section level5">
<h5><span class="header-section-number">2.2.3.3.1</span> Rationale</h5>
<p>The earliest attempt found for indexing AC can be found in a study published by Lee <em>et al</em><span class="citation">(<em>54</em>)</span> where they were interested in the ‘apical index’ (A.I.) of bottle cells during <em>X.laevis</em> gastrulation(figure <a href="2-mat-met.html#fig:ACLee">2.14</a>).</p>
(ref:ACLee) A.I. of <em>X.leavis</em> bottle cells.
<div class="figure" style="text-align: center"><span id="fig:ACLee"></span>
<img src="figure/02-MaMo/ACI/Jen_yi_Lee.png" alt="(ref:ACLee)" width="75%" />
<p class="caption">
Figure 2.14: (ref:ACLee)
</p>
</div>
<p>Another example for measuring AC is the apical <em>constriction</em> index (ACI, figure <a href="2-mat-met.html#fig:ACHard">2.15</a>) for the cells of the <em>D.rerio</em> lateral line primordium (PLLp), which can be found in a study from 2012 where it was shown that Fgfr-Ras-MAPK signaling is required for AC and Rock2a localization6 <span class="citation">(<em>55</em>, <em>56</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:ACHard"></span>
<img src="figure/02-MaMo/ACI/Harding.jpg" alt="Apical constriction index. A 3D reconstructed cells of a CLCLNA control primordium and B of a primordium deficient of Fgf signaling." width="75%" />
<p class="caption">
Figure 2.15: Apical constriction index. <strong>A</strong> 3D reconstructed cells of a CLCLNA control primordium and <strong>B</strong> of a primordium deficient of Fgf signaling.
</p>
</div>
<p>In both cases, the measurements for the A.I.<span class="citation">(<em>54</em>)</span> or ACI<span class="citation">(<em>56</em>)</span> does not differ and describes the ratio for lateral height over apical width <a href="2-mat-met.html#fig:ACICells">2.16</a>.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{apical\;width\;[\mu m]}\]</span></p>
<p>While trying to reproduce these results we found two principal weaknesses of applying this ratio to the cells of the PLLp. First, it does not respect the independence of lateral height to AC. Second, it does not differentiate between <em>anterio-posterior</em> (AP) and <em>dorso-ventral</em> (DV) tissue polarization. Third, it is really more an A.I. than a constriction index.</p>
<div id="ACI-param" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.1</span> Parameter definition</h6>
<p>To obtain a precise and biologically meaningful quantity for AC, first a couple of definitions have to be made.</p>
<ol style="list-style-type: decimal">
<li>AC is independent of orientation and polarization
<ul>
<li>Cells can be polarized along the AP axis</li>
<li>Cells can also be polarized along the DV axis</li>
</ul></li>
<li>AC is independent of lateral height
<ul>
<li>Lateral height can be described as the distance of the two farthest points on the surface area of a cell</li>
<li>Two cells with different lateral heights can be equally apically constricted</li>
</ul></li>
<li>AC is independent of cell size
<ul>
<li>The size of a cell is its volume</li>
<li>Two cells of different volume can be equally apically constricted</li>
</ul></li>
</ol>
</div>
<div id="ACI-lat" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.2</span> Adaption for variation in lateral height</h6>
<p>To test different A.I. conditions, an apically constricted cell can be approximated by modeling a tetrahedron. For example, shrinking or enlarging a cell symmetrically should not affect the ACI. As described by Harding(2014)<span class="citation">(<em>56</em>)</span>, the <em>apical width</em> of a cell is measured first by manual 3-D object reconstruction, second manual re-orientation, and third by going 1 <span class="math inline">\(\mu\)</span>m from the apical tip into the cell (from now on referred to as <span class="math inline">\(\Delta\)</span>ap, <a href="2-mat-met.html#fig:ACICells">2.16</a>B). Finally, <em>apical width</em> is the total width of the 2-D object.</p>
<p>If <span class="math inline">\(\Delta\)</span>ap is a constant, the A.I. in a symmetrically enlarged cell increases from e.g. 15 to 23, since <em>apical width</em> stays the same but lateral height increases. On the contrary, if <span class="math inline">\(\Delta\)</span>ap is adjusted relative to a cells lateral height, e.g. by percentage, the A.I. in a symmetrically enlarged cell stays the same <a href="2-mat-met.html#fig:ACICells">2.16</a></p>
(ref:ACICells) A.I. model <strong>A-A’’</strong> A.I. Cell Models. A’ and A’’ show cells that are symmetrically increased versions of A. While in A’, constant delta was used, in A’’ delta was adjusted relative to the lateral height. <strong>B</strong> Illustrating delta ap. (left) apically constricted cells volume rendered in XY (top) and as a lateral cross-section in X-Z (bottom). (right) 2D area as seen at delta ap of 1 or 2.5 <span class="math inline">\(\mu\)</span>m.
<div class="figure" style="text-align: center"><span id="fig:ACICells"></span>
<img src="figures/summary/aci_fig-01.png" alt="(ref:ACICells)" width="75%" />
<p class="caption">
Figure 2.16: (ref:ACICells)
</p>
</div>
<p>Therefore the measurement for apical width has to be relative to lateral height.</p>
<p><span class="math display">\[\mathbf{ACI} = \frac{lateral\;height\;[\mu m]}{relative\;apical\;width\;[\mu m]}\]</span></p>
</div>
<div id="ACI-pol" class="section level6">
<h6><span class="header-section-number">2.2.3.3.1.3</span> Adaption for tissue polarization</h6>
<p>Organs develop in a 3-D space and are polarized along each axis. AC usually describes a 2-D morphogenetic movement towards a center along the X-Y axes. However, the contraction movements along X and Y might be independent of one another. This could mean that they happen at different speeds, or that one is absent. As a result, the tissue would look less radially <a href="2-mat-met.html#fig:cellpol">2.17</a> constricted, but more constricted against a line. In order to separate those two AC dimensions, the A.I. can be calculated for the <em>anterio-posterior</em> and for the <em>dorso-ventral</em> axis <a href="2-mat-met.html#fig:cellpol">2.17</a>.</p>
(ref:cellpol) Schematic AC along the A-P and D-V axis. <strong>A</strong> shows a A-P and D-V constricted cluster of cells. <strong>B</strong> shows a D-V constricted cluster of cells.
<div class="figure" style="text-align: center"><span id="fig:cellpol"></span>
<img src="figure/02-MaMo/ACI/ACI_Cells_pol.png" alt="(ref:cellpol)" width="75%" />
<p class="caption">
Figure 2.17: (ref:cellpol)
</p>
</div>
<p>By fitting an ellipsoid to the volume area taken at <span class="math inline">\(\mathrm{\Delta}\)</span>ap, one will obtain the following parameters.</p>
<ol style="list-style-type: decimal">
<li>Length of Major axis (figure <a href="2-mat-met.html#fig:ellipse">2.18</a>)
<ul>
<li>constituting <em>apical width</em></li>
</ul></li>
<li>Length of Minor axis (figure <a href="2-mat-met.html#fig:ellipse">2.18</a>)
<ul>
<li>constituting <em>apical height</em></li>
</ul></li>
<li>Angle of Major from 0<span class="math inline">\(^\circ\)</span> (figure <a href="2-mat-met.html#fig:ellipse">2.18</a>)
<ul>
<li>indicative for orientation of <em>lateral height</em></li>
</ul></li>
</ol>
(ref:ellipse) Scheme of ellipsoid measures. <strong>A</strong> shows the major axis as apical width and the minor axis as apical height. <strong>B</strong> shows the angular displacement from the horizon in steps of 30<span class="math inline">\(^\circ\)</span>.
<div class="figure" style="text-align: center"><span id="fig:ellipse"></span>
<img src="figure/02-MaMo/ACI/ellipse.png" alt="(ref:ellipse)" width="75%" />
<p class="caption">
Figure 2.18: (ref:ellipse)
</p>
</div>
<p>The two dimensions of AC indices can therefore be described as the following ratios…</p>
<p><span class="math display">\[\mathbf{A.I._{Major}} = \frac{lateral\;height\;[\mu m]}{Major\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{A.I._{Minor}} = \frac{lateral\;height\;[\mu m]}{Minor\;axis\;[\mu m]}\]</span>
<span class="math display">\[\mathbf{Angle_{Major}} = \measuredangle = \mathrm{\Delta}\;from\;horizont\;[0-90^\circ]\]</span></p>
</div>
</div>
<div id="ACI-Dis" class="section level5">
<h5><span class="header-section-number">2.2.3.3.2</span> Measurements</h5>
<div id="ACI-singlecell" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.1</span> Single Cell measurements</h6>
<p>Each geometric object has a centroid coordinate in X and Y (and Z) which is represented as the mean of all X or Y coordinates within the object. Here, the centroid coordinates in X and Y are used to plot the cells as points in the X-Y plane. Additionally, each point is colored for the A.I. value (high values are blue-violet, middle values are green, low values are yellow-red).</p>
(ref:ACIMajor) A.I.<sub>Major</sub> single cell measurements
<div class="figure" style="text-align: center"><span id="fig:ACIMajor"></span>
<img src="thesis_files/figure-html/ACIMajor-1.png" alt="(ref:ACIMajor)" width="85%" />
<p class="caption">
Figure 2.19: (ref:ACIMajor)
</p>
</div>
(ref:ACIMinor) A.I.<sub>Minor</sub> single cell measurements
<div class="figure" style="text-align: center"><span id="fig:ACIMinor"></span>
<img src="thesis_files/figure-html/ACIMinor-1.png" alt="(ref:ACIMinor)" width="85%" />
<p class="caption">
Figure 2.20: (ref:ACIMinor)
</p>
</div>
<!--

|         Obj          |  n  |
|:--------------------:|:---:|
| M_DMSO_P01_cells.csv | 157 |
| M_SU54_P17_cells.csv | 171 |
Table: (\#tab:cellcountaci) ACI Cellcount
-->
<p>Harding(2013)<span class="citation">(<em>56</em>)</span> were using a constant <span class="math inline">\(\mathrm{\Delta}\)</span>ap to measure the apical width, which we have shown to be incorrect in certain cases. In their study they found that certain mean A.I. values in the DMSO go as high as 15, which might be related to this <a href="2-mat-met.html#fig:ACICells">2.16</a>.
By measuring apical width at a relative <span class="math inline">\(\mathrm{\Delta}\)</span>ap the mean values for A.I.<sub>Major</sub> are at 4.7 for the DMSO control and 3.6 for the SU5402 treated condition, which seems to somewhat correlate with their findings<a href="2-mat-met.html#fig:HardingACI">2.21</a>. .</p>
<p>However overall our results indicate that the AC difference is not as strong, at least if all the cells of the PLLp are taken into account.</p>
(ref:HardingACI) <strong>E-G</strong> 3-D reconstructions of the highlighted cell. <strong>H</strong> A.I.s for embryos treated with DMSO, SU5402, PD0325901 or following induction of hsp70:dn-Ras. (n=180 cells / N = 6 embryos).
<div class="figure" style="text-align: center"><span id="fig:HardingACI"></span>
<img src="figure/02-MaMo/GrTr/HardingACI.png" alt="(ref:HardingACI)" width="60%" />
<p class="caption">
Figure 2.21: (ref:HardingACI)
</p>
</div>
</div>
<div id="ACI-Angledens" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.2</span> Angle densities</h6>
<p>To check whether there is a bias in orientation of the apical width, the angle measurements <a href="2-mat-met.html#fig:ellipse">2.18</a> can be shown as a function of density along X. Here, a binwidth of 2.5 and a grouping factor to color the two conditions ‘DMSO’ and ‘SU5402’ was used.</p>
<p>Interestingly the results indicate that there is less of a difference for the Major<sub>Angle</sub> at angles bigger than 15-20<span class="math inline">\(^\circ\)</span>. This would mean that the <em>apical width</em> in SU5402 treated embryos is more strongly oriented along the horizontal <em>antero-posterior</em> axis.</p>
</div>
<div id="ACI-mag" class="section level6">
<h6><span class="header-section-number">2.2.3.3.2.3</span> ACI magnitude at different Angles</h6>
<p>Now, to get an idea of the magnitude ACI<sub>Major</sub> and ACI<sub>Minor</sub> have at each angle, each can be shown as a function of the Major<sub>Angle</sub> over ACI (while Major<sub>Angle</sub> is the same for both ACIs). Again, the grouping factor ‘GT’ was used to color the two conditions ‘DMSO’ and ‘SU5402’.</p>
<p>Since AC is a 3-D morphogenetic process and since cells in a wild type PLLp are mostly radially organized, it does make sense to try to look at AC from more than just one perspective. Here we propose to separate the ACI into an <em>antero-posterior</em> and a <em>dorso-ventral</em> dimension.</p>
<ol style="list-style-type: decimal">
<li><p>Interestingly there does not seem to be much of a difference in ACI<sub>Minor</sub>, which can also be shown by the mean values which are at 2.6 for the DMSO control and at 2.1 for the SU5402 treated condition. This indicates that there is no significant difference in contraction along the <em>dorso-ventral</em> axis.</p></li>
<li><p>For the ACI<sub>Major</sub> the base constriction for both, DMSO and SU5402 is at around 3.6, however there is a peak at around 40-60° in the DMSO control where cells are most constricted having a maximum ACI at 15.8. This indicates that cells coming from 35.7° are most constricted in the <em>anterior-posterior</em> axis.</p></li>
</ol>
(ref:angletoACI) ACI<sub>Major</sub> / ACI<sub>Minor</sub> over Major<sub>Angle</sub>
<div class="figure" style="text-align: center"><span id="fig:angletoACI"></span>
<img src="thesis_files/figure-html/angletoACI-1.png" alt="(ref:angletoACI)" width="49%" /><img src="thesis_files/figure-html/angletoACI-2.png" alt="(ref:angletoACI)" width="49%" />
<p class="caption">
Figure 2.22: (ref:angletoACI)
</p>
</div>
</div>
</div>
</div>
<div id="timelapse-analysis" class="section level4">
<h4><span class="header-section-number">2.2.3.4</span> Timelapse analysis</h4>
<p>ImageJ Macro</p>
</div>
</div>
</div>
<div id="mat-datasets" class="section level2">
<h2><span class="header-section-number">2.3</span> Image Data Sets</h2>
<p>Summaries of Image datasets</p>






</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-Intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-res.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
