<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Results | Genetic and Mechanic Signaling in Organ Development</title>
  <meta name="description" content="Chapter 3 Results | Genetic and Mechanic Signaling in Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Results | Genetic and Mechanic Signaling in Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Results | Genetic and Mechanic Signaling in Organ Development" />
  
  
  

<meta name="author" content="David Simon Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-mat-met.html"/>
<link rel="next" href="4-discussion.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { font-weight: bold; } /* Alert */
code span.an { font-style: italic; } /* Annotation */
code span.cf { font-weight: bold; } /* ControlFlow */
code span.co { font-style: italic; } /* Comment */
code span.cv { font-style: italic; } /* CommentVar */
code span.do { font-style: italic; } /* Documentation */
code span.dt { text-decoration: underline; } /* DataType */
code span.er { font-weight: bold; } /* Error */
code span.in { font-style: italic; } /* Information */
code span.kw { font-weight: bold; } /* Keyword */
code span.pp { font-weight: bold; } /* Preprocessor */
code span.wa { font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#about"><i class="fa fa-check"></i>About</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#foreword"><i class="fa fa-check"></i>Foreword</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abbreviations"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a>
<ul>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#einleitung"><i class="fa fa-check"></i>Einleitung</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#zielsetzung"><i class="fa fa-check"></i>Zielsetzung</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html#ergebnisse"><i class="fa fa-check"></i>Ergebnisse</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#intro-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#cell-signaling"><i class="fa fa-check"></i><b>1.1.2</b> Cell Signaling</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#intro-morph"><i class="fa fa-check"></i><b>1.1.3</b> Morphogenesis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#open-questions-and-motivation"><i class="fa fa-check"></i><b>1.3</b> Open Questions and Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mos"><i class="fa fa-check"></i><b>2.1.9</b> Morpholinos</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.10</b> Hardware</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.11</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Data Strategy and Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.2</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.3</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.4</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.3</b> Ground Truth</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cluster-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Cluster Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#morphometric-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Morphometric analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.4</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#res-met"><i class="fa fa-check"></i><b>3.1</b> Quantitative imaging and image analysis</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#res-mount"><i class="fa fa-check"></i><b>3.1.1</b> A 3D-printed stamp to standardize sample mounting and semi-automatize imaging</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#mat-anallzr2d"><i class="fa fa-check"></i><b>3.1.2</b> anaLLzr2D - Automated 2D neuromast analysis and nuclei count</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#mat-anallzr2dt"><i class="fa fa-check"></i><b>3.1.3</b> anaLLzr2DT - Automated 2D pLLP migration and neuromast deposition analysis through time</a></li>
<li class="chapter" data-level="3.1.4" data-path="3-res.html"><a href="3-res.html#mat-anallzr3d"><i class="fa fa-check"></i><b>3.1.4</b> anaLLzr3D - Automated 3D single cell segmentation and A.I. analysis in the pLLP</a></li>
<li class="chapter" data-level="3.1.5" data-path="3-res.html"><a href="3-res.html#CNN"><i class="fa fa-check"></i><b>3.1.5</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#intro-phen"><i class="fa fa-check"></i><b>3.2.1</b> Phenotype description</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.3</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.4</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#apical-constriction-2"><i class="fa fa-check"></i><b>3.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.6</b> Haircell Specification</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-discussion.html"><a href="4-discussion.html"><i class="fa fa-check"></i><b>4</b> Discussion</a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-discussion.html"><a href="4-discussion.html#methods"><i class="fa fa-check"></i><b>4.1</b> Methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="4-discussion.html"><a href="4-discussion.html#standardized-mounting"><i class="fa fa-check"></i><b>4.1.1</b> Standardized mounting</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-discussion.html"><a href="4-discussion.html#automated-image-analysis"><i class="fa fa-check"></i><b>4.1.2</b> Automated image analysis</a></li>
<li class="chapter" data-level="4.1.3" data-path="4-discussion.html"><a href="4-discussion.html#rosette-detection-1"><i class="fa fa-check"></i><b>4.1.3</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-discussion.html"><a href="4-discussion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-discussion.html"><a href="4-discussion.html#lateral-line"><i class="fa fa-check"></i><b>4.2.1</b> Lateral Line</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-discussion.html"><a href="4-discussion.html#rosette-formation-1"><i class="fa fa-check"></i><b>4.2.2</b> Rosette Formation</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-discussion.html"><a href="4-discussion.html#apical-constriction-3"><i class="fa fa-check"></i><b>4.2.3</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-discussion.html"><a href="4-discussion.html#hair-cell-specification"><i class="fa fa-check"></i><b>4.2.4</b> Hair Cell specification</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-discussion.html"><a href="4-discussion.html#model-2"><i class="fa fa-check"></i><b>4.2.5</b> Model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="appendix-appendix.html"><a href="appendix-appendix.html"><i class="fa fa-check"></i>(Appendix) Appendix</a></li>
<li class="chapter" data-level="" data-path="supplement.html"><a href="supplement.html"><i class="fa fa-check"></i>Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Genetic and Mechanic Signaling in Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="res" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> Results</h1>
<div id="res-met" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Quantitative imaging and image analysis</h2>
<div id="res-mount" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> A 3D-printed stamp to standardize sample mounting and semi-automatize imaging</h3>
<p>Even though on a macroscopic scale development is a remarkably similar and synchronized process between zebrafish embryos, a single biological process on a <em>microscopic</em> scale even in sibling embryos can look drastically different. Given the noisy and variable character of biological systems, it is important to record a sufficient number of samples to obtain a quantitative and representative view of a biological process. Furthermore, to process biological samples of whole organisms in a high-content manner it is important to have a standardized way of sample mounting, data acquisition, data processing and analysis.</p>
<p>However, imaging a high number of samples and generating large datasets to date is still largely limited by the classical way developmental biologists mount embryos for imaging. A number of factors limiting the standardization are summarized in table <a href="#tab:resmountingtab"><strong>??</strong></a></p>
<p>Therefore, especially for 3D segmentation and 2D tracking experiments where an exact staging and orientation of the embryo is necessary, there is a need for methods to standardize sample mounting and image acquisition of multiple embryos.</p>
<p>The protocol we describe here was designed to be used with XY scanning universal sample holders that usually come with any motorized-stage inverted microscope. Similar to previous approaches <span class="citation">(<em>66</em>–<em>69</em>)</span>, it uses a 3D-printed stamping device to produce an Agarose imprint with a diameter of 20 mm on the cover glass of a 35 mm <span class="math inline">\(\mu\)</span>-dish. The imprint consists of 44 equally spaced <span class="math inline">\(\mu\)</span>-wells, which are designed to fit the average morphology of a zebrafish embryo between 24 and 96 hours-post-fertilization (hpf).</p>
<p>The aim was to develop a standardized mounting method allowing us to: (i) mount many samples in parallel in a 2D coordinate system of rows and columns, (ii) reduce the acquisition time and thus photo-bleaching and photo-toxicity during imaging, (iii) semi-automatize the acquisition, (iv) reduce the post-processing steps, and (v) facilitate subsequent processing such as genotyping due to a 1:1 correlation between image data and specimen arrangement sequence.</p>
<p>For mounting, an improved 3D specimen preparation and well-plate-like sample navigation for zebrafish larvae confocal microscopy was developed with which lateral line development can be recorded over more than 20 h, in up to 44 positions, in a confocal Z-stack of less than 120 <span class="math inline">\(\mu\)</span>m and a time interval of 5–10 min. (depending on the number of channels and exposure time). The stamp was designed to be used for embryos between 24 and 96 hpf. For a tailor-made well, embryos were fixed and imaged <em>in toto</em> to measure the dimensions in X, Y and Z of different (whole embryo, trunk, yolk) embryonic structures (figure <a href="3-res.html#fig:mountmicro">3.1</a>B-B’). Using Microsoft 3D-Builder a well was assembled from basic shapes like cube, sphere and wedge. After, the well was duplicated a couple of times and put in a grid-like arrangement to fit on a disc base 20 mm in diameter. Printing was performed on a Formlabs extrusion printer (figure <a href="3-res.html#fig:mountmicro">3.1</a>C).</p>

<div class="figure" style="text-align: center"><span id="fig:mountmicro"></span>
<img src="figures/results/00_methods/mounting/stamp_dims.png" alt="Stamp and \(\mu\)-well properties. A - A’ Mounting (A) without and with (A’) \(\mu\)-well. Legend to the right. B - B’ Dimensions of a single micro well in (B) X-Y and (B’) X-Z in mm C Elements and dimensions of the stamp wafer. D Assembled stamp with a screw mounted on the back of the stamp wafer." width="95%" />
<p class="caption">
Figure 3.1: Stamp and <span class="math inline">\(\mu\)</span>-well properties. <strong>A - A’</strong> Mounting (A) without and with (A’) <span class="math inline">\(\mu\)</span>-well. Legend to the right. <strong>B - B’</strong> Dimensions of a single micro well in (B) X-Y and (B’) X-Z in mm <strong>C</strong> Elements and dimensions of the stamp wafer. <em>D</em> Assembled stamp with a screw mounted on the back of the stamp wafer.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:stampresults"></span>
<img src="figures/results/00_methods/mounting/stamp_results.png" alt="Mounting Results. A - A’ Maximum intensity projection of a 50 hpf embryo mounted on its side in XY and XZ. (right) Color scale indicates depth encoding. C Multi-position (36), multi-channel (2) time-lapse recording (13 h duration; 15 min. interval). D Multi-channel (2) Extended Depth of Focus (EDF) projections from widefield Z-stacks (recorded with 20x Objective). Scale Bar = 1mm E Multipoint coordinates in X, Y and Z (recorded with 40x Objective). The offset describes the distance of each point from the mean of all points in X, Y and Z. Panel 1–3 (top to bottom) show dimensions X, Y and Z in comparison for the pLLP, the eye and the otic vesicle. The red line indicates the median, the blue line indicates zero offset, error bars indicate mean \(\pm\) s.d.. Numeric values indicate the variance in each group. F-F’’ Systematic retrieval for genotyping. F Mounted embryos in a 2-D coordinate system of rows (A-M) and columns (1–3). F’ Imaging Sequence in a snake-by-column fashion. In a time-lapse setting, it starts at point 1 (P01) again to initiate the next timepoint. F’’ After imaging, the embryos are retrieved in the same sequence as they were imaged (snake by column, left panel). F’’ Each genotyping result on the electrophoresis gel is easily correlated to one imaging dataset with defined X-Y coordinates." width="95%" />
<p class="caption">
Figure 3.2: Mounting Results. <strong>A - A’</strong> Maximum intensity projection of a 50 hpf embryo mounted on its side in XY and XZ. (right) Color scale indicates depth encoding. <strong>C</strong> Multi-position (36), multi-channel (2) time-lapse recording (13 h duration; 15 min. interval). <strong>D</strong> Multi-channel (2) Extended Depth of Focus (EDF) projections from widefield Z-stacks (recorded with 20x Objective). Scale Bar = 1mm <strong>E</strong> Multipoint coordinates in X, Y and Z (recorded with 40x Objective). The offset describes the distance of each point from the mean of all points in X, Y and Z. Panel 1–3 (top to bottom) show dimensions X, Y and Z in comparison for the pLLP, the eye and the otic vesicle. The red line indicates the median, the blue line indicates zero offset, error bars indicate mean <span class="math inline">\(\pm\)</span> s.d.. Numeric values indicate the variance in each group. <strong>F-F’’</strong> Systematic retrieval for genotyping. <strong>F</strong> Mounted embryos in a 2-D coordinate system of rows (A-M) and columns (1–3). <strong>F’</strong> Imaging Sequence in a snake-by-column fashion. In a time-lapse setting, it starts at point 1 (P01) again to initiate the next timepoint. <strong>F’’</strong> After imaging, the embryos are retrieved in the same sequence as they were imaged (snake by column, left panel). <strong>F’’</strong> Each genotyping result on the electrophoresis gel is easily correlated to one imaging dataset with defined X-Y coordinates.
</p>
</div>
<div id="procedure" class="section level4" number="3.1.1.1">
<h4><span class="header-section-number">3.1.1.1</span> Procedure</h4>
<div id="preparation-of-the-agarose-cast" class="section level5" number="3.1.1.1.1">
<h5><span class="header-section-number">3.1.1.1.1</span> Preparation of the agarose cast</h5>
<p>To prepare the agarose cast, the stamp is first cleaned from dust and remnants with tissue soaked in 70<span class="math inline">\(\%\)</span> Ethanol and pressured air (figure <a href="3-res.html#fig:mounting">3.3</a> IA). To prepare the casting medium a 1<span class="math inline">\(\%\)</span> Agarose (w/v) solution is prepared in an autoclaved 100 mL bluecap bottle by dissolving 200 mg of agarose in 20 mL of E3 using a microwave oven. From the ready solution 650 <span class="math inline">\(\mu\)</span>L are applied to the <span class="math inline">\(\varnothing\)</span> 20 mm coverslip of a <span class="math inline">\(\varnothing\)</span> 35 mm imaging dish (materials in table <a href="2-mat-met.html#tab:mat-mount">2.10</a>). Subsequently the clean stamp is gently placed onto the placed solution and adjusted to the center. The dish is then rotated to distribute excess agarose over the entire dish surface to stabilize the imprint once polymerized.</p>
<p>After about 30 min. the stamp is removed by first slipping a clean preparation needle between the stamp and the polymer and then lifting it from the cast (figure <a href="3-res.html#fig:mounting">3.3</a>IB). If necessary, air bubbles appearing between the cover glass and the polymer are eliminated by punctuation with a preparation needle. The mounting cast may be used immediately or stored at 4<span class="math inline">\(^\circ\)</span>C for several days (with lid closed).</p>
</div>
<div id="preparation-of-mounting-media" class="section level5" number="3.1.1.1.2">
<h5><span class="header-section-number">3.1.1.1.2</span> Preparation of mounting media</h5>
<p>Two solutions of low-melting point Agarose (LMPA) are prepared in autoclaved 100 mL bluecap bottles by dissolving 60 and 100 mg LMPA in 16 mL of E3 in a microwave oven - yielding 0.375 and 0.625 <span class="math inline">\(\%\)</span> respectively. Per stamped cast, 2 aliquots of 1.6 mL are prepared in 2 mL tubes for each LMPA concentration and placed in a heating block adjusted to 41<span class="math inline">\(^\circ\)</span>C.For live imaging, 400 <span class="math inline">\(\mu\)</span>L of 4.2 mg/mL Tricaine (25X) are added to keep the embryos anesthetized during imaging. Final concentrations of LMPA solution are therefore 0.3 and 0.5 <span class="math inline">\(\%\)</span>, respectively. LMPA solutions containing Tricaine were prepared fresh for each mounting session. The LMPA solution and the mounting cast have almost equal refractive indices. Therefore, when adding the LMPA solution the cast becomes invisible. To still be able to locate the <span class="math inline">\(\mu\)</span>-wells and to position the embryos accordingly, the illumination contrast and mirror angle of a transmitted light base are adjusted to make the <span class="math inline">\(\mu\)</span>-wells visible again (figure <a href="3-res.html#fig:mounting">3.3</a>IIA-C).</p>
</div>
<div id="mounting-procedure" class="section level5" number="3.1.1.1.3">
<h5><span class="header-section-number">3.1.1.1.3</span> Mounting procedure</h5>
<p>In case of live imaging the embryos are first anesthetized in a Petri dish with 4 to 5 drops of 4.2 mg/mL Tricaine (40 <span class="math inline">\(\mu\)</span>g/mL in E3) added 4 to 5 min. before usage.</p>
<p>For mounting, the cast is first gently filled from the border (figure <a href="3-res.html#fig:mounting">3.3</a>II A3) with 500 <span class="math inline">\(\mu\)</span>L of 0.3<span class="math inline">\(\%\)</span> LMPA solution. Then, 44 embryos (one for each well) are collected from their Petri dish with a glass Pasteur pipette. To minimize the amount of liquid added to the LMPA, the embryos are allowed to sink to the air – liquid interface and immediately added in one drop to the liquid LMPA solution in the stamped cast. Next, each embryo is moved to a separate <span class="math inline">\(\mu\)</span>-well with a preparation needle by positioning the yolk within the hemi-spherical structure of each well and the tail aligned horizontally with the shape of the <span class="math inline">\(\mu\)</span>-well (figure <a href="3-res.html#fig:mounting">3.3</a>II C-D). The LMPA was allowed to polymerize for about 40 min. For time-lapse recording longer than 1 h, 1 mL of 0.5<span class="math inline">\(\%\)</span> LMPA was added on top and allowed to polymerize for another 10 min. to construct an Agarose <em>sandwich</em> to stabilize the structure. Since the 0.3<span class="math inline">\(\%\)</span> LMPA will still be very fragile, the 0.5<span class="math inline">\(\%\)</span> LMPA should be added to the outer well first, carefully raising the level.</p>
<p>Since Agarose polymerization speed depends on temperature, for mounting the temperature of the room should not be less than 23<span class="math inline">\(^\circ\)</span>C to give sufficient time. For indefinite time of embryo orientation, a higher room temperature or a 5 V terrarium heating mat (at maximum temperature ca. 38<span class="math inline">\(^\circ\)</span>C) can be used. For the latter, a hole with the diameter of an imaging dish should first be cut in the middle of the heating mat. For mounting, the mat should then be placed and fixed on the stereo-microscope stage with the dish in the hole.</p>
</div>
<div id="imaging-setup" class="section level5" number="3.1.1.1.4">
<h5><span class="header-section-number">3.1.1.1.4</span> Imaging setup</h5>
<p>The dish is placed onto the sample holder of an inverted confocal spinning disc microscope so that the embryos are aligned to the Y axis of the microscope stage. The stage is then moved to place the embryo at Position 01 (P01, top-left position) right above the objective (figure <a href="3-res.html#fig:mounting">3.3</a>III A).</p>
<div id="define-embryo-positions" class="section level6" number="3.1.1.1.4.1">
<h6><span class="header-section-number">3.1.1.1.4.1</span> Define embryo positions</h6>
<p>Since the embryos are mounted in a 3-D grid with well defined dimensions, all positions can be defined <em>via</em> a pre-defined points list that is loaded into the microscope software. For our system we use the ‘Nikon Imaging Software’ where we move the stage to P01, define a multi-point list with distance X / distance Y = 3450 / 1280 <span class="math inline">\(\mu\)</span>m, bring P01 into focus and offset all points in Z. The list can also be saved for re-use in a future experiment. Alternatively one can also define a custom well plate and calibrate the stage.</p>
</div>
<div id="refine-positions" class="section level6" number="3.1.1.1.4.2">
<h6><span class="header-section-number">3.1.1.1.4.2</span> Refine Positions</h6>
<p>Even though the mounting method allows for a precise positioning, each embryo physiology is still a bit different resulting in differences between positions but same structures of up to 100 <span class="math inline">\(\mu\)</span>m in X, Y and Z (figure <a href="3-res.html#fig:stampresults">3.2</a>E). Therefore, before starting an experiment each position needs to be refined.</p>
</div>
</div>
<div id="retrieval" class="section level5" number="3.1.1.1.5">
<h5><span class="header-section-number">3.1.1.1.5</span> Retrieval</h5>
<p>For further experiments such as genotyping, the embryos are retrieved from the agarose in the same sequence as they were imaged (figure <a href="3-res.html#fig:stampresults">3.2</a>F’). To do so, a glass pipette is inserted into the agarose and directed to the head region of an embryo. By applying a gentle underpressure the embryo is then sucked into the glass pipette.</p>
<p>To lyse the embryos and extract the genomic DNA, each embryo is placed in a single tube of an 8-tube PCR strip. Since 8-tube PCR strips are designed to work with multichannel pipettes, the genotyping PCR is performed and analysed by gel electrophoresis using an 8x-multichannel pipette. When using a 34-well comb, the pipette tips will reach every second well of the agarose gel. Filling the wells staggered (offset by 1), one can load 4 × 8 wells in one row (figure <a href="3-res.html#fig:stampresults">3.2</a>F’‘). Since each embryo has a defined position, it is straightforward to associate each genotype to the corresponding image data (figure <a href="3-res.html#fig:stampresults">3.2</a>F’ - F’’’). Since a single mismatch would mess up the entire experiment by resulting in a frameshift of the one-to-one correspondence, this is a very important feature. The imaging dish can be reused several times. For cleaning, the agarose bed is removed from the dish using a small scoop or preparation needle and wiped gently with a lint-free tissue soaked in Ethanol.</p>

<div class="figure" style="text-align: center"><span id="fig:mounting"></span>
<img src="figures/results/00_methods/mounting/stamp_mounting.png" alt="Stamping procedure Agarose Cast (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. Mounting (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. Imaging (A) Positioning of the \(\mu\)-well (B) Alignment in Brightfield and (C) Definition of a custom well plate." width="95%"  />
<p class="caption">
Figure 3.3: Stamping procedure <strong>Agarose Cast</strong> (A) clean stamp surface (B) preparation of the stamp before lifting (C) ready-for-use agarose imprint. <strong>Mounting</strong> (A) without LMPA (B and C) with LMPA, while the latter shows the imprint with light coming from a different angle, making the chambers visible again. (D) Horizontal alignment of embryos. <strong>Imaging</strong> (A) Positioning of the <span class="math inline">\(\mu\)</span>-well (B) Alignment in Brightfield and (C) Definition of a custom well plate.
</p>
</div>
</div>
</div>
<div id="summary-1" class="section level4" number="3.1.1.2">
<h4><span class="header-section-number">3.1.1.2</span> Summary</h4>
<p>The major improvements introduced by this method are (1) using a low percentage LMPA, which extends the timespan for mounting which is necessary to align a higher number of embryos. It also gives the embryo more freedom to grow during longer time-lapse imaging and facilitates retrieval of afterwards. (2) using a stamped cast, which allows for standardized and reproducible positions of the embryo as shown for the lateral line primordium, the eye and the otic vesicle (figure <a href="3-res.html#fig:stampresults">3.2</a>E). A significant increase of number of embryos that can be imaged during a single experiment (figure <a href="3-res.html#fig:stampresults">3.2</a>C-D). A significant reduction of the Z-stack size and therefore of the illumination of the samples (figure <a href="3-res.html#fig:stampresults">3.2</a>A).</p>
<p>In comparison to existing methods we provide a solution that is easy and in-expensive even for non-specialized labs. Also, while similar methods are well suited for high throughput and lower resolutions, ours may also be used for long time-lapse and high resolution imaging.</p>
</div>
</div>
<div id="mat-anallzr2d" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> anaLLzr2D - Automated 2D neuromast analysis and nuclei count</h3>
<p>For LL analysis I developed a custom IJ macro script that segments individual cell clusters and the pLLP. From the opening dialog (figure <a href="3-res.html#fig:anallzr2ddialog">3.4</a>) the user can choose to <em>count nuclei</em> and / or <em>sort ROIs</em>. If nuclei count is chosen, the macro expects a dual-channel (Ch1: <em>cldnb:lyn-gfp</em>; Ch2: a nuclei label) <em>tiff</em>-file as input. If ROI sorting is selected, segmented CCs are numbered and sorted from left to right instead of top to bottom (IJ’s native sorting method).</p>
<ul>
<li><em>Membrane label blur</em> controls the detail of pLLP and CC segmentation, where lower values result in more detail</li>
<li><em>Closing filter</em> controls how harsh objects are separated from each other</li>
<li><em>Nuclei label blur</em> controls the details of single nuclei and is evaluated after ground truth data described in section <a href="2-mat-met.html#mat-GrTrDat">2.3</a></li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:anallzr2ddialog"></span>
<img src="figures/materials/macros/anallzr2D_macro.png" alt="anaLLzR2D opening dialog checkboxes Choose to count nuclei and whether ROIs sorting should be applied. sliders Choose filter and blurring levels." width="40%"  />
<p class="caption">
Figure 3.4: anaLLzR2D opening dialog <strong>checkboxes</strong> Choose to count nuclei and whether ROIs sorting should be applied. <strong>sliders</strong> Choose filter and blurring levels.
</p>
</div>
<div id="image-analysis" class="section level4" number="3.1.2.1">
<h4><span class="header-section-number">3.1.2.1</span> Image Analysis</h4>
<p>Using ROIs as masks, the nuclei within each ROI are counted with a 2-D maxima finder. However, in their unprocessed form the images are too noisy to get meaningful results. The images therefore have to be smoothened with a blurring filter. To detect the right amount of nuclei, it is necessary to evaluate the distance over which the blurring should be applied. A typical nucleus in the pLLP is about 5 <span class="math inline">\(\mu\)</span>m in diameter. To determine the right blurring value, a range of 4-6 <span class="math inline">\(\mu\)</span>m in steps of 0.5 was tested. Figure <a href="3-res.html#fig:maxllreg">3.5</a> shows a registered maximum Z projected lateral line used for ground truth evaluation.</p>

<div class="figure" style="text-align: center"><span id="fig:maxllreg"></span>
<img src="figures/materials/ground_truth/clusters/clusters.png" alt="Registration of 2D data. Cluster Segments Registered, MaxIP data with cell cluster segments of the lyn-GFP signal laid upon the DAPI signal. Cell Labels Magenta dots represent the maxima found within each ROI and hence the nuclei labels." width="95%"  />
<p class="caption">
Figure 3.5: Registration of 2D data. <strong>Cluster Segments</strong> Registered, MaxIP data with cell cluster segments of the lyn-GFP signal laid upon the DAPI signal. <strong>Cell Labels</strong> Magenta dots represent the maxima found within each ROI and hence the nuclei labels.
</p>
</div>
</div>
<div id="code-snippets" class="section level4" number="3.1.2.2">
<h4><span class="header-section-number">3.1.2.2</span> Code Snippets</h4>
<p>The <code>[...]</code> symbol indicates code re-use from an earlier instance.</p>
<div id="segmentation" class="section level5" number="3.1.2.2.1">
<h5><span class="header-section-number">3.1.2.2.1</span> Segmentation</h5>
<p>The membrane label image is segmented based on optimized filter parameters that were derived from trial and error. After, the macro halts for manual correction.</p>
<p></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb1-1"><a href="3-res.html#cb1-1"></a><span class="co"># Background subtraction</span></span>
<span id="cb1-2"><a href="3-res.html#cb1-2"></a>    <span class="fu">run</span>(<span class="st">&quot;Subtract Background...&quot;</span>, <span class="st">&quot;rolling = 50&quot;</span>);</span>
<span id="cb1-3"><a href="3-res.html#cb1-3"></a>    <span class="fu">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>,</span>
<span id="cb1-4"><a href="3-res.html#cb1-4"></a>        <span class="st">&quot;operation = Opening element = Disk radius = 20&quot;</span>);</span>
<span id="cb1-5"><a href="3-res.html#cb1-5"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 5 scaled&quot;</span>);</span>
<span id="cb1-6"><a href="3-res.html#cb1-6"></a></span>
<span id="cb1-7"><a href="3-res.html#cb1-7"></a><span class="co"># Thresholding</span></span>
<span id="cb1-8"><a href="3-res.html#cb1-8"></a>    <span class="fu">setAutoThreshold</span>(<span class="st">&quot;Moments dark&quot;</span>);</span>
<span id="cb1-9"><a href="3-res.html#cb1-9"></a>    <span class="fu">setOption</span>(<span class="st">&quot;BlackBackground&quot;</span>, false);</span>
<span id="cb1-10"><a href="3-res.html#cb1-10"></a>    <span class="fu">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb1-11"><a href="3-res.html#cb1-11"></a></span>
<span id="cb1-12"><a href="3-res.html#cb1-12"></a><span class="co"># Particle analysis</span></span>
<span id="cb1-13"><a href="3-res.html#cb1-13"></a>    <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size = 250-Infinity exclude add&quot;</span>);</span>
<span id="cb1-14"><a href="3-res.html#cb1-14"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;Show All without labels&quot;</span>);</span>
<span id="cb1-15"><a href="3-res.html#cb1-15"></a>    <span class="fu">run</span>(<span class="st">&quot;Enhance Contrast&quot;</span>, <span class="st">&quot;saturated = 0.35&quot;</span>);</span>
<span id="cb1-16"><a href="3-res.html#cb1-16"></a>    <span class="fu">waitForUser</span>(<span class="st">&quot;Check ROIs, correct if necessary&quot;</span>);</span>
<span id="cb1-17"><a href="3-res.html#cb1-17"></a>    <span class="cf">if</span> (sort) {</span>
<span id="cb1-18"><a href="3-res.html#cb1-18"></a>        <span class="fu">sortROIs</span>();</span>
<span id="cb1-19"><a href="3-res.html#cb1-19"></a>    }</span></code></pre></div>
<p></p>
</div>
<div id="sorting" class="section level5" number="3.1.2.2.2">
<h5><span class="header-section-number">3.1.2.2.2</span> Sorting</h5>
<p>To sort manually corrected ROIs from left to right, each ROIs position in calculated relatively to total image width.</p>
<p></p>
<div class="sourceCode" id="cb2"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb2-1"><a href="3-res.html#cb2-1"></a><span class="co"># Sort ROIs from left to right</span></span>
<span id="cb2-2"><a href="3-res.html#cb2-2"></a><span class="cf">function</span> <span class="fu">sortROIs</span>() {</span>
<span id="cb2-3"><a href="3-res.html#cb2-3"></a>    <span class="fu">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, <span class="st">&quot;centroid redirect = None decimal = 0&quot;</span>);</span>
<span id="cb2-4"><a href="3-res.html#cb2-4"></a>            <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb2-5"><a href="3-res.html#cb2-5"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-6"><a href="3-res.html#cb2-6"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;measure&quot;</span>);</span>
<span id="cb2-7"><a href="3-res.html#cb2-7"></a>                x <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;X&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb2-8"><a href="3-res.html#cb2-8"></a>                w <span class="ot">=</span> <span class="fu">getWidth</span>();</span>
<span id="cb2-9"><a href="3-res.html#cb2-9"></a>                a <span class="ot">=</span> x<span class="sc">/</span>w;</span>
<span id="cb2-10"><a href="3-res.html#cb2-10"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, a);</span>
<span id="cb2-11"><a href="3-res.html#cb2-11"></a>                <span class="fu">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-12"><a href="3-res.html#cb2-12"></a>                }</span>
<span id="cb2-13"><a href="3-res.html#cb2-13"></a>        <span class="fu">setBatchMode</span>(false);</span>
<span id="cb2-14"><a href="3-res.html#cb2-14"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;sort&quot;</span>); </span>
<span id="cb2-15"><a href="3-res.html#cb2-15"></a>            <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb2-16"><a href="3-res.html#cb2-16"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-17"><a href="3-res.html#cb2-17"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, j);</span>
<span id="cb2-18"><a href="3-res.html#cb2-18"></a>                <span class="fu">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-19"><a href="3-res.html#cb2-19"></a>                }   </span>
<span id="cb2-20"><a href="3-res.html#cb2-20"></a>            <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb2-21"><a href="3-res.html#cb2-21"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb2-22"><a href="3-res.html#cb2-22"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, j<span class="sc">+</span><span class="dv">1</span>);</span>
<span id="cb2-23"><a href="3-res.html#cb2-23"></a>                <span class="fu">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb2-24"><a href="3-res.html#cb2-24"></a>                }</span>
<span id="cb2-25"><a href="3-res.html#cb2-25"></a>}</span></code></pre></div>
<p></p>
</div>
<div id="count-nuclei" class="section level5" number="3.1.2.2.3">
<h5><span class="header-section-number">3.1.2.2.3</span> Count nuclei</h5>
<p>After smoothing the nuclei signal in the DAPI labeled channel, maxima are detected only within each ROI.</p>
<p></p>
<div class="sourceCode" id="cb3"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb3-1"><a href="3-res.html#cb3-1"></a><span class="co"># count nuclei within segmented cell clusters</span></span>
<span id="cb3-2"><a href="3-res.html#cb3-2"></a><span class="co"># steps performed on DAPI channel</span></span>
<span id="cb3-3"><a href="3-res.html#cb3-3"></a><span class="co"># count nuclei</span></span>
<span id="cb3-4"><a href="3-res.html#cb3-4"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 0.6 scaled&quot;</span>);</span>
<span id="cb3-5"><a href="3-res.html#cb3-5"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;open&quot;</span>, datdir <span class="sc">+</span> filename <span class="sc">+</span> <span class="st">&quot;_ROIset.zip&quot;</span>);</span>
<span id="cb3-6"><a href="3-res.html#cb3-6"></a>    rcount <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb3-7"><a href="3-res.html#cb3-7"></a></span>
<span id="cb3-8"><a href="3-res.html#cb3-8"></a><span class="co"># for each ROI</span></span>
<span id="cb3-9"><a href="3-res.html#cb3-9"></a>    <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> rcount; j<span class="sc">++</span>) {</span>
<span id="cb3-10"><a href="3-res.html#cb3-10"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;open&quot;</span>, datdir <span class="sc">+</span> filename <span class="sc">+</span> <span class="st">&quot;_ROIset.zip&quot;</span>);</span>
<span id="cb3-11"><a href="3-res.html#cb3-11"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb3-12"><a href="3-res.html#cb3-12"></a>            <span class="fu">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>);</span>
<span id="cb3-13"><a href="3-res.html#cb3-13"></a>            <span class="fu">run</span>(<span class="st">&quot;Enhance Contrast&quot;</span>, <span class="st">&quot;saturated = 0.35&quot;</span>);</span>
<span id="cb3-14"><a href="3-res.html#cb3-14"></a>            <span class="fu">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = 0 output = [Point Selection]&quot;</span>);</span>
<span id="cb3-15"><a href="3-res.html#cb3-15"></a>            <span class="fu">run</span>(<span class="st">&quot;Capture Image&quot;</span>);</span>
<span id="cb3-16"><a href="3-res.html#cb3-16"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb3-17"><a href="3-res.html#cb3-17"></a>        <span class="fu">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = 0 output = Count&quot;</span>);</span>
<span id="cb3-18"><a href="3-res.html#cb3-18"></a>        NC <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Count&quot;</span>);</span>
<span id="cb3-19"><a href="3-res.html#cb3-19"></a>        <span class="fu">setResult</span>(<span class="st">&quot;Nuclei&quot;</span>, j, NC);</span>
<span id="cb3-20"><a href="3-res.html#cb3-20"></a>        <span class="fu">setResult</span>(<span class="st">&quot;Pos&quot;</span>, j, j <span class="sc">+</span> <span class="dv">1</span>);</span>
<span id="cb3-21"><a href="3-res.html#cb3-21"></a>    }</span></code></pre></div>
<p></p>
</div>
</div>
<div id="data-analysis" class="section level4" number="3.1.2.3">
<h4><span class="header-section-number">3.1.2.3</span> Data Analysis</h4>
<p>Comparing the maxima counts of each Gaussian parameter with the Ground Truth gives an indication for false -positives resp. -negatives. In figure <a href="3-res.html#fig:GrTratioCC">3.6</a> the relative numbers for each blurring parameter can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). The red area represents false negatives, the green false positives.</p>

<div class="figure" style="text-align: center"><span id="fig:GrTratioCC"></span>
<img src="thesis_dsk_files/figure-html/GrTratioCC-1.png" alt="Relative difference of maxima counts" width="75%" />
<p class="caption">
Figure 3.6: Relative difference of maxima counts
</p>
</div>
<p>To estimate the quality of nuclei detection for each parameter, the ratio of automatically detected and ground truth objects count can be calculated and compared (table <a href="3-res.html#tab:nrcount">3.1</a>). The closer it is to 1, the better.</p>
<table class="table" style="font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:nrcount">Table 3.1: </span>Blurring parameter for nuclei count
</caption>
<thead>
<tr>
<th style="text-align:right;">
</th>
<th style="text-align:center;">
4.5
</th>
<th style="text-align:center;">
5.0
</th>
<th style="text-align:center;">
5.5
</th>
<th style="text-align:center;">
6.0
</th>
<th style="text-align:center;">
7.0
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
mean ratio
</td>
<td style="text-align:center;">
1.30
</td>
<td style="text-align:center;">
1.14
</td>
<td style="text-align:center;">
1.04
</td>
<td style="text-align:center;">
0.95
</td>
<td style="text-align:center;">
0.86
</td>
</tr>
<tr>
<td style="text-align:right;">
std.
</td>
<td style="text-align:center;">
0.23
</td>
<td style="text-align:center;">
0.17
</td>
<td style="text-align:center;">
0.15
</td>
<td style="text-align:center;">
0.14
</td>
<td style="text-align:center;">
0.14
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">column headers</span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> blurring parameters
</td>
</tr>
</tfoot>
</table>
<p>In summary, maximum performance is achieved at a scaled parameter of 6 <span class="math inline">\(\mu\)</span>m, with a ratio of 0.95 for the count objects and a standard deviation of 0.14.</p>
</div>
</div>
<div id="mat-anallzr2dt" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> anaLLzr2DT - Automated 2D pLLP migration and neuromast deposition analysis through time</h3>
<p>For the analysis of the migratory behavior (speed, acceleration) and shape (area, roundness) of the pLLP, as well as for the formation and deposition of the pro-Neuromasts through time, I developed a custom IJ macro script that segments the migrating pLLP and individual cell clusters on each frame of a timelapse. Upon macro execution the opening dialog is presented which is divided in three sections (figure <a href="3-res.html#fig:anallzr2dtdialog">3.7</a>A).</p>
<p>In the <strong>processing</strong> section the user may choose which modules are executed.</p>
<ul>
<li><em>Segmentation</em> controls whether the images are segmented before measurement. If de-seleted the user has to provide segmentation masks separately.</li>
<li><em>Include Cell Clusters</em> controls whether Cell Clusters should be included in the analysis. If de-selected, only the pLLP will be considered.</li>
<li><em>Registration</em> controls whether the pLLP should be captured in time and space and saved in a separate stack.</li>
<li><em>Multichannel</em> controls whether a second channel summary statistics from each ROI at each timepoint should be taken.
<ul>
<li>measurements taken are the mean, standard deviation, minimum and maximum intensity</li>
</ul></li>
</ul>
<p>In the <strong>options</strong> section the user may choose whether the macro should be run in <em>headless mode</em> (without showing every single action), whether the input images are timeseries and whether all other windows should be closed upon start of processing.</p>
<p>After confirmation, the user has to enter a date of experiment as an identifier in a second dialog. Furthermore the user is presented the images physical properties pre-filled where the idea here is just a review since this is a major source of mistakes (figure <a href="3-res.html#fig:anallzr2dtdialog">3.7</a>B). Finally, a third dialog is presented to the user giving an approximate duration and basic instructions (figure <a href="3-res.html#fig:anallzr2dtdialog">3.7</a>C).</p>

<div class="figure" style="text-align: center"><span id="fig:anallzr2dtdialog"></span>
<img src="figures/materials/macros/anallzr2DT_macro.png" alt="anaLLzR2DT opening dialog A Opening dialog: Main functionality B Opening Dialog: metadata setup options C Time approximation dialog and basic instructions." width="95%" />
<p class="caption">
Figure 3.7: anaLLzR2DT opening dialog <strong>A</strong> Opening dialog: Main functionality <strong>B</strong> Opening Dialog: metadata setup options <strong>C</strong> Time approximation dialog and basic instructions.
</p>
</div>
<p>Code-snippets describing the main functionality are described in the next couple of sub-sections. The <code>[...]</code> symbol indicates code re-use from an earlier instance.</p>
<div id="registration" class="section level6" number="3.1.3.0.0.1">
<h6><span class="header-section-number">3.1.3.0.0.1</span> Registration</h6>
<p>The first module of the macro is pLLP registration in X, Y and cropping in Z. For better segmentation results, first the SNR is enhanced by background subtraction. To rotate the image the pLLPs migrational path is approximated by the position of the first and the last segment, then the image is cropped to a fixed height.</p>
<p></p>
<div class="sourceCode" id="cb4"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb4-1"><a href="3-res.html#cb4-1"></a><span class="co">#   subtract background</span></span>
<span id="cb4-2"><a href="3-res.html#cb4-2"></a>        <span class="fu">run</span>(<span class="st">&quot;Z Project...&quot;</span>, <span class="st">&quot;projection = [Average Intensity]&quot;</span>);</span>
<span id="cb4-3"><a href="3-res.html#cb4-3"></a>        ZPAVG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-4"><a href="3-res.html#cb4-4"></a>        <span class="cf">if</span> (reg) {</span>
<span id="cb4-5"><a href="3-res.html#cb4-5"></a>            <span class="fu">print</span>(<span class="st">&quot;Calculating registration parameters...&quot;</span>);</span>
<span id="cb4-6"><a href="3-res.html#cb4-6"></a>            <span class="fu">setSlice</span>(n);</span>
<span id="cb4-7"><a href="3-res.html#cb4-7"></a>            <span class="fu">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>); </span>
<span id="cb4-8"><a href="3-res.html#cb4-8"></a>            DORG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-9"><a href="3-res.html#cb4-9"></a>            <span class="fu">imageCalculator</span>(<span class="st">&quot;Subtract create&quot;</span>, DORG, ZPAVG);</span>
<span id="cb4-10"><a href="3-res.html#cb4-10"></a>            <span class="fu">close</span>(DORG);</span>
<span id="cb4-11"><a href="3-res.html#cb4-11"></a>            <span class="fu">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>,</span>
<span id="cb4-12"><a href="3-res.html#cb4-12"></a>                <span class="st">&quot;operation = Closing element=Disk radius = 15&quot;</span>);</span>
<span id="cb4-13"><a href="3-res.html#cb4-13"></a>            REG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-14"><a href="3-res.html#cb4-14"></a>            <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 6 scaled&quot;</span>); </span>
<span id="cb4-15"><a href="3-res.html#cb4-15"></a>            <span class="fu">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot; &quot;</span>);</span>
<span id="cb4-16"><a href="3-res.html#cb4-16"></a>            <span class="fu">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>, <span class="st">&quot;saturated = 0.3 normalize&quot;</span>);</span>
<span id="cb4-17"><a href="3-res.html#cb4-17"></a>            <span class="fu">run</span>(<span class="st">&quot;8-bit&quot;</span>);</span>
<span id="cb4-18"><a href="3-res.html#cb4-18"></a>            <span class="fu">setAutoThreshold</span>(<span class="st">&quot;MaxEntropy  dark&quot;</span>);</span>
<span id="cb4-19"><a href="3-res.html#cb4-19"></a>            <span class="fu">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb4-20"><a href="3-res.html#cb4-20"></a></span>
<span id="cb4-21"><a href="3-res.html#cb4-21"></a>    <span class="co"># analyze segments</span></span>
<span id="cb4-22"><a href="3-res.html#cb4-22"></a>        <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size = 150-10000 include exclude add&quot;</span>);</span>
<span id="cb4-23"><a href="3-res.html#cb4-23"></a>        rmcount <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="sc">-</span><span class="dv">1</span>;</span>
<span id="cb4-24"><a href="3-res.html#cb4-24"></a>        <span class="fu">print</span>(<span class="st">&quot;rois: &quot;</span> <span class="sc">+</span> rmcount);</span>
<span id="cb4-25"><a href="3-res.html#cb4-25"></a>        </span>
<span id="cb4-26"><a href="3-res.html#cb4-26"></a>    <span class="co"># angle</span></span>
<span id="cb4-27"><a href="3-res.html#cb4-27"></a>        <span class="cf">if</span>(<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb4-28"><a href="3-res.html#cb4-28"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb4-29"><a href="3-res.html#cb4-29"></a>            List.setMeasurements;</span>
<span id="cb4-30"><a href="3-res.html#cb4-30"></a>            Angle <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;FeretAngle&quot;</span>);</span>
<span id="cb4-31"><a href="3-res.html#cb4-31"></a>            <span class="fu">print</span>(<span class="st">&quot;Angle: &quot;</span> <span class="sc">+</span> Angle);</span>
<span id="cb4-32"><a href="3-res.html#cb4-32"></a>            <span class="cf">if</span> (Angle <span class="sc">&lt;</span> <span class="dv">0</span>) {Angle <span class="ot">=</span> Angle <span class="sc">*</span> (<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb4-33"><a href="3-res.html#cb4-33"></a>            <span class="cf">if</span> (Angle <span class="sc">&gt;</span> <span class="dv">90</span>) {Angle <span class="ot">=</span> (<span class="dv">180</span><span class="sc">-</span>Angle) <span class="sc">*</span> (<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb4-34"><a href="3-res.html#cb4-34"></a>        } <span class="cf">else</span> {</span>
<span id="cb4-35"><a href="3-res.html#cb4-35"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb4-36"><a href="3-res.html#cb4-36"></a>            List.setMeasurements;</span>
<span id="cb4-37"><a href="3-res.html#cb4-37"></a>            X1Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-38"><a href="3-res.html#cb4-38"></a>            Y1Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-39"><a href="3-res.html#cb4-39"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb4-40"><a href="3-res.html#cb4-40"></a>            List.setMeasurements;</span>
<span id="cb4-41"><a href="3-res.html#cb4-41"></a>            X2Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-42"><a href="3-res.html#cb4-42"></a>            Y2Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-43"><a href="3-res.html#cb4-43"></a>            <span class="fu">makeLine</span>(X1Line, Y1Line, X2Line, Y2Line);</span>
<span id="cb4-44"><a href="3-res.html#cb4-44"></a>            List.setMeasurements;</span>
<span id="cb4-45"><a href="3-res.html#cb4-45"></a>            Angle <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb4-46"><a href="3-res.html#cb4-46"></a>            <span class="cf">if</span> (Angle <span class="sc">&lt;</span> <span class="dv">0</span>) {Angle <span class="ot">=</span> Angle<span class="sc">*</span>(<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb4-47"><a href="3-res.html#cb4-47"></a>            <span class="cf">if</span> (Angle <span class="sc">&gt;</span> <span class="dv">90</span>) {Angle <span class="ot">=</span> (<span class="dv">180</span><span class="sc">-</span>Angle)<span class="sc">*</span>(<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb4-48"><a href="3-res.html#cb4-48"></a>            }</span>
<span id="cb4-49"><a href="3-res.html#cb4-49"></a>            <span class="fu">print</span>(<span class="st">&quot;Angle: &quot;</span> <span class="sc">+</span> Angle);</span>
<span id="cb4-50"><a href="3-res.html#cb4-50"></a>            <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, </span>
<span id="cb4-51"><a href="3-res.html#cb4-51"></a>                <span class="st">&quot;angle = &quot;</span> <span class="sc">+</span> Angle <span class="sc">+</span> <span class="st">&quot; grid = 1 interpolation = Bilinear&quot;</span>);</span>
<span id="cb4-52"><a href="3-res.html#cb4-52"></a>            ZPAVG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-53"><a href="3-res.html#cb4-53"></a>            <span class="fu">selectWindow</span>(REG);</span>
<span id="cb4-54"><a href="3-res.html#cb4-54"></a>            <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, </span>
<span id="cb4-55"><a href="3-res.html#cb4-55"></a>                <span class="st">&quot;angle = &quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear&quot;</span>);</span>
<span id="cb4-56"><a href="3-res.html#cb4-56"></a>            <span class="fu">run</span>(<span class="st">&quot;Make Binary&quot;</span>);</span>
<span id="cb4-57"><a href="3-res.html#cb4-57"></a>            REG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-58"><a href="3-res.html#cb4-58"></a>        </span>
<span id="cb4-59"><a href="3-res.html#cb4-59"></a>        <span class="co"># cropping</span></span>
<span id="cb4-60"><a href="3-res.html#cb4-60"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb4-61"><a href="3-res.html#cb4-61"></a>            <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, </span>
<span id="cb4-62"><a href="3-res.html#cb4-62"></a>                <span class="st">&quot;size = 150 - 10000 include add&quot;</span>);</span>
<span id="cb4-63"><a href="3-res.html#cb4-63"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb4-64"><a href="3-res.html#cb4-64"></a>            List.setMeasurements;</span>
<span id="cb4-65"><a href="3-res.html#cb4-65"></a>            XRect <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb4-66"><a href="3-res.html#cb4-66"></a>            YRect <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb4-67"><a href="3-res.html#cb4-67"></a>            <span class="fu">selectWindow</span>(REG);</span>
<span id="cb4-68"><a href="3-res.html#cb4-68"></a>            <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb4-69"><a href="3-res.html#cb4-69"></a>            List.setMeasurements;</span>
<span id="cb4-70"><a href="3-res.html#cb4-70"></a>            height <span class="ot">=</span> <span class="dv">120</span> <span class="sc">/</span> sizeX; <span class="co"># change height of rect here</span></span>
<span id="cb4-71"><a href="3-res.html#cb4-71"></a>            <span class="fu">toUnscaled</span>(YRect);</span>
<span id="cb4-72"><a href="3-res.html#cb4-72"></a>            YRect <span class="ot">=</span> YRect <span class="sc">-</span> (height <span class="sc">/</span> <span class="dv">2</span>);</span>
<span id="cb4-73"><a href="3-res.html#cb4-73"></a>            <span class="fu">print</span>(<span class="st">&quot;YRectcor: &quot;</span> <span class="sc">+</span> YRect);</span>
<span id="cb4-74"><a href="3-res.html#cb4-74"></a>        } </span>
<span id="cb4-75"><a href="3-res.html#cb4-75"></a></span>
<span id="cb4-76"><a href="3-res.html#cb4-76"></a>    <span class="co"># register</span></span>
<span id="cb4-77"><a href="3-res.html#cb4-77"></a>        <span class="fu">resetMinAndMax</span>();</span>
<span id="cb4-78"><a href="3-res.html#cb4-78"></a>        <span class="cf">if</span> (dual) {</span>
<span id="cb4-79"><a href="3-res.html#cb4-79"></a>        <span class="co">#   C1</span></span>
<span id="cb4-80"><a href="3-res.html#cb4-80"></a>            <span class="fu">selectWindow</span>(ORG);</span>
<span id="cb4-81"><a href="3-res.html#cb4-81"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-82"><a href="3-res.html#cb4-82"></a>                <span class="fu">print</span>(<span class="st">&quot;  Registering &quot;</span> <span class="sc">+</span> embryoID <span class="sc">+</span><span class="st">&quot;...&quot;</span>);</span>
<span id="cb4-83"><a href="3-res.html#cb4-83"></a>                <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, </span>
<span id="cb4-84"><a href="3-res.html#cb4-84"></a>                    <span class="st">&quot;angle = &quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear stack&quot;</span>);</span>
<span id="cb4-85"><a href="3-res.html#cb4-85"></a>                <span class="fu">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-86"><a href="3-res.html#cb4-86"></a>                <span class="fu">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-87"><a href="3-res.html#cb4-87"></a>            }</span>
<span id="cb4-88"><a href="3-res.html#cb4-88"></a>        <span class="co"># C2</span></span>
<span id="cb4-89"><a href="3-res.html#cb4-89"></a>            <span class="fu">open</span>(dualdir <span class="sc">+</span> dualdirlist[q]);</span>
<span id="cb4-90"><a href="3-res.html#cb4-90"></a>            dualname <span class="ot">=</span> <span class="fu">replace</span>(dualdirlist[q], <span class="st">&quot;.tif&quot;</span>, <span class="st">&quot;&quot;</span>);</span>
<span id="cb4-91"><a href="3-res.html#cb4-91"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-92"><a href="3-res.html#cb4-92"></a>                <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>,</span>
<span id="cb4-93"><a href="3-res.html#cb4-93"></a>                    <span class="st">&quot;angle=&quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear stack&quot;</span>);</span>
<span id="cb4-94"><a href="3-res.html#cb4-94"></a>                <span class="fu">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-95"><a href="3-res.html#cb4-95"></a>                <span class="fu">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-96"><a href="3-res.html#cb4-96"></a>            }</span>
<span id="cb4-97"><a href="3-res.html#cb4-97"></a>            <span class="fu">close</span>();</span>
<span id="cb4-98"><a href="3-res.html#cb4-98"></a>        } <span class="cf">else</span> {</span>
<span id="cb4-99"><a href="3-res.html#cb4-99"></a>            <span class="fu">selectWindow</span>(ORG);</span>
<span id="cb4-100"><a href="3-res.html#cb4-100"></a>            <span class="cf">if</span> (reg) {</span>
<span id="cb4-101"><a href="3-res.html#cb4-101"></a>                <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>,</span>
<span id="cb4-102"><a href="3-res.html#cb4-102"></a>                    <span class="st">&quot;angle = &quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear stack&quot;</span>);</span>
<span id="cb4-103"><a href="3-res.html#cb4-103"></a>                <span class="fu">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-104"><a href="3-res.html#cb4-104"></a>                <span class="fu">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-105"><a href="3-res.html#cb4-105"></a>            } <span class="cf">else</span> {    </span>
<span id="cb4-106"><a href="3-res.html#cb4-106"></a>            }</span>
<span id="cb4-107"><a href="3-res.html#cb4-107"></a>        }</span>
<span id="cb4-108"><a href="3-res.html#cb4-108"></a>        ORG <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb4-109"><a href="3-res.html#cb4-109"></a>    <span class="co"># crop ZPAVG for image calc</span></span>
<span id="cb4-110"><a href="3-res.html#cb4-110"></a>        <span class="fu">selectWindow</span>(ZPAVG);</span>
<span id="cb4-111"><a href="3-res.html#cb4-111"></a>        <span class="cf">if</span> (reg) {</span>
<span id="cb4-112"><a href="3-res.html#cb4-112"></a>        <span class="fu">makeRectangle</span>(<span class="dv">0</span>, YRect, width, height);</span>
<span id="cb4-113"><a href="3-res.html#cb4-113"></a>        <span class="fu">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb4-114"><a href="3-res.html#cb4-114"></a>        }</span></code></pre></div>
<p></p>
</div>
<div id="segmentation-1" class="section level6" number="3.1.3.0.0.2">
<h6><span class="header-section-number">3.1.3.0.0.2</span> Segmentation</h6>
<p>After registration of the image follows segmentation. For this we again improve SNR by background correction, following by disconnecting loosely joint segments.</p>
<p></p>
<div class="sourceCode" id="cb5"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb5-1"><a href="3-res.html#cb5-1"></a><span class="co">#  background correction</span></span>
<span id="cb5-2"><a href="3-res.html#cb5-2"></a>    <span class="fu">print</span>(<span class="st">&quot;  Segmenting &quot;</span><span class="sc">+</span> embryoID <span class="sc">+</span><span class="st">&quot;_RC...&quot;</span>);</span>
<span id="cb5-3"><a href="3-res.html#cb5-3"></a>        <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb5-4"><a href="3-res.html#cb5-4"></a>        <span class="fu">imageCalculator</span>(<span class="st">&quot;Subtract create stack&quot;</span>, ORG, ZPAVG);</span>
<span id="cb5-5"><a href="3-res.html#cb5-5"></a>        IC <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb5-6"><a href="3-res.html#cb5-6"></a>    <span class="fu">selectWindow</span>(IC);</span>
<span id="cb5-7"><a href="3-res.html#cb5-7"></a>        <span class="fu">print</span>(<span class="st">&quot;Bleach correction...&quot;</span>);</span>
<span id="cb5-8"><a href="3-res.html#cb5-8"></a>        <span class="fu">run</span>(<span class="st">&quot;Bleach Correction&quot;</span>, </span>
<span id="cb5-9"><a href="3-res.html#cb5-9"></a>            <span class="st">&quot;correction = [Simple Ratio] background = 0&quot;</span>);</span>
<span id="cb5-10"><a href="3-res.html#cb5-10"></a>        nslbc <span class="ot">=</span> <span class="fu">nSlices</span>();</span>
<span id="cb5-11"><a href="3-res.html#cb5-11"></a>        <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">1</span>; j <span class="sc">&lt;</span> nslbc; j<span class="sc">++</span>) {</span>
<span id="cb5-12"><a href="3-res.html#cb5-12"></a>            <span class="fu">setSlice</span>(j);</span>
<span id="cb5-13"><a href="3-res.html#cb5-13"></a>            <span class="fu">run</span>(<span class="st">&quot;Morphological Filters&quot;</span>,</span>
<span id="cb5-14"><a href="3-res.html#cb5-14"></a>                <span class="st">&quot;operation = Closing element = Disk radius = 15&quot;</span>);</span>
<span id="cb5-15"><a href="3-res.html#cb5-15"></a>        }</span>
<span id="cb5-16"><a href="3-res.html#cb5-16"></a>        <span class="fu">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;name =&quot;</span><span class="sc">+</span> ORG <span class="sc">+</span><span class="st">&quot; title = [] use&quot;</span>);</span>
<span id="cb5-17"><a href="3-res.html#cb5-17"></a>        </span>
<span id="cb5-18"><a href="3-res.html#cb5-18"></a>    <span class="co">#   segmentation</span></span>
<span id="cb5-19"><a href="3-res.html#cb5-19"></a>        <span class="fu">selectWindow</span>(MC);</span>
<span id="cb5-20"><a href="3-res.html#cb5-20"></a>        <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 5.5 scaled stack&quot;</span>);</span>
<span id="cb5-21"><a href="3-res.html#cb5-21"></a>        </span>
<span id="cb5-22"><a href="3-res.html#cb5-22"></a>    <span class="co">#   disconnect segments</span></span>
<span id="cb5-23"><a href="3-res.html#cb5-23"></a>        <span class="fu">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>,</span>
<span id="cb5-24"><a href="3-res.html#cb5-24"></a>            <span class="st">&quot;saturated = 0.5 normalize process_all&quot;</span>);</span>
<span id="cb5-25"><a href="3-res.html#cb5-25"></a>        <span class="fu">setSlice</span>(n);</span>
<span id="cb5-26"><a href="3-res.html#cb5-26"></a>        <span class="fu">resetThreshold</span>();</span>
<span id="cb5-27"><a href="3-res.html#cb5-27"></a>        <span class="fu">setAutoThreshold</span>(<span class="st">&quot;MaxEntropy dark&quot;</span>);</span>
<span id="cb5-28"><a href="3-res.html#cb5-28"></a>        <span class="fu">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>,</span>
<span id="cb5-29"><a href="3-res.html#cb5-29"></a>            <span class="st">&quot;method = MaxEntropy background = Dark black&quot;</span>);</span>
<span id="cb5-30"><a href="3-res.html#cb5-30"></a>        <span class="fu">run</span>(<span class="st">&quot;Invert LUT&quot;</span>);</span>
<span id="cb5-31"><a href="3-res.html#cb5-31"></a>        <span class="fu">run</span>(<span class="st">&quot;Fill Holes&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb5-32"><a href="3-res.html#cb5-32"></a>        <span class="fu">run</span>(<span class="st">&quot;Options...&quot;</span>, </span>
<span id="cb5-33"><a href="3-res.html#cb5-33"></a>            <span class="st">&quot;iterations = 2 count = 1 pad do = Erode stack&quot;</span>);</span>
<span id="cb5-34"><a href="3-res.html#cb5-34"></a>        <span class="fu">run</span>(<span class="st">&quot;Options...&quot;</span>, </span>
<span id="cb5-35"><a href="3-res.html#cb5-35"></a>            <span class="st">&quot;iterations = 2 count = 1 pad do = Open stack&quot;</span>);</span>
<span id="cb5-36"><a href="3-res.html#cb5-36"></a>        <span class="fu">run</span>(<span class="st">&quot;Options...&quot;</span>, </span>
<span id="cb5-37"><a href="3-res.html#cb5-37"></a>            <span class="st">&quot;iterations = 1 count = 1 pad do = Dilate stack&quot;</span>);</span>
<span id="cb5-38"><a href="3-res.html#cb5-38"></a>    <span class="er">}</span></span>
<span id="cb5-39"><a href="3-res.html#cb5-39"></a>    <span class="fu">waitForUser</span>(<span class="st">&quot;Check Segmentations&quot;</span>);</span>
<span id="cb5-40"><a href="3-res.html#cb5-40"></a>    <span class="er">}</span></span></code></pre></div>
<p></p>
</div>
<div id="analysis" class="section level6" number="3.1.3.0.0.3">
<h6><span class="header-section-number">3.1.3.0.0.3</span> Analysis</h6>
<p>Finally we measure and save the results in the defined names and directories</p>
<p></p>
<div class="sourceCode" id="cb6"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb6-1"><a href="3-res.html#cb6-1"></a><span class="cf">for</span> (<span class="at">b =</span> <span class="dv">0</span>; b <span class="sc">&lt;</span> orgdirlist.length; b<span class="sc">++</span>) {</span>
<span id="cb6-2"><a href="3-res.html#cb6-2"></a><span class="co"># get genotypes and embryoIDs from arrays</span></span>
<span id="cb6-3"><a href="3-res.html#cb6-3"></a>    type <span class="ot">=</span> types[b];</span>
<span id="cb6-4"><a href="3-res.html#cb6-4"></a>    embryoID <span class="ot">=</span> embryoIDs[b];</span>
<span id="cb6-5"><a href="3-res.html#cb6-5"></a>    orgname <span class="ot">=</span> <span class="fu">replace</span>(orgdirlist[b], <span class="st">&quot;.tif&quot;</span>, <span class="st">&quot;&quot;</span>);</span>
<span id="cb6-6"><a href="3-res.html#cb6-6"></a>    embryodir <span class="ot">=</span> output <span class="sc">+</span> File.separator <span class="sc">+</span> orgname <span class="sc">+</span> File.separator;</span>
<span id="cb6-7"><a href="3-res.html#cb6-7"></a>    <span class="fu">File.makeDirectory</span>(embryodir);</span>
<span id="cb6-8"><a href="3-res.html#cb6-8"></a>    </span>
<span id="cb6-9"><a href="3-res.html#cb6-9"></a><span class="co"># open and define binary</span></span>
<span id="cb6-10"><a href="3-res.html#cb6-10"></a>    <span class="fu">open</span>(bindir<span class="sc">+</span>bindirlist[b]);</span>
<span id="cb6-11"><a href="3-res.html#cb6-11"></a>    BIN <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb6-12"><a href="3-res.html#cb6-12"></a>          </span>
<span id="cb6-13"><a href="3-res.html#cb6-13"></a><span class="co"># open and define orginal </span></span>
<span id="cb6-14"><a href="3-res.html#cb6-14"></a>    <span class="cf">if</span> (dual) {</span>
<span id="cb6-15"><a href="3-res.html#cb6-15"></a>        <span class="fu">open</span>(rcdirc1 <span class="sc">+</span> rcdirc1list[b]);</span>
<span id="cb6-16"><a href="3-res.html#cb6-16"></a>    } <span class="cf">else</span> {</span>
<span id="cb6-17"><a href="3-res.html#cb6-17"></a>        <span class="fu">open</span>(rcdir <span class="sc">+</span> rcdirlist[b]);</span>
<span id="cb6-18"><a href="3-res.html#cb6-18"></a>    }</span>
<span id="cb6-19"><a href="3-res.html#cb6-19"></a>    RC <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb6-20"><a href="3-res.html#cb6-20"></a>    dotIndex <span class="ot">=</span> <span class="fu">indexOf</span>(RC, <span class="st">&quot;.&quot;</span>);</span>
<span id="cb6-21"><a href="3-res.html#cb6-21"></a>    title <span class="ot">=</span> <span class="fu">substring</span>(RC, <span class="dv">0</span>, dotIndex);</span>
<span id="cb6-22"><a href="3-res.html#cb6-22"></a>    </span>
<span id="cb6-23"><a href="3-res.html#cb6-23"></a><span class="co"># enter 2nd loop to increment over each slice of the time-series</span></span>
<span id="cb6-24"><a href="3-res.html#cb6-24"></a>    <span class="fu">selectWindow</span>(BIN);   <span class="co"># select binary</span></span>
<span id="cb6-25"><a href="3-res.html#cb6-25"></a>    pangles <span class="ot">=</span> <span class="fu">newArray</span>(<span class="fu">nSlices</span>() <span class="sc">+</span> <span class="dv">1</span>);</span>
<span id="cb6-26"><a href="3-res.html#cb6-26"></a>    <span class="cf">for</span> (<span class="at">i =</span> <span class="dv">1</span> ; i <span class="sc">&lt;=</span> <span class="fu">nSlices</span>(); i<span class="sc">++</span>) {</span>
<span id="cb6-27"><a href="3-res.html#cb6-27"></a>      s <span class="ot">=</span> <span class="fu">nSlices</span>();</span>
<span id="cb6-28"><a href="3-res.html#cb6-28"></a>        <span class="fu">setSlice</span>(i);</span>
<span id="cb6-29"><a href="3-res.html#cb6-29"></a>        <span class="cf">if</span> (ccs) {</span>
<span id="cb6-30"><a href="3-res.html#cb6-30"></a>            <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>,</span>
<span id="cb6-31"><a href="3-res.html#cb6-31"></a>                <span class="st">&quot;size = 150-10000 include add&quot;</span>);</span>
<span id="cb6-32"><a href="3-res.html#cb6-32"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-33"><a href="3-res.html#cb6-33"></a>            <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>,</span>
<span id="cb6-34"><a href="3-res.html#cb6-34"></a>                <span class="st">&quot;size = 750-10000 include add&quot;</span>);</span>
<span id="cb6-35"><a href="3-res.html#cb6-35"></a>        }</span>
<span id="cb6-36"><a href="3-res.html#cb6-36"></a>    <span class="co"># loop though ROI List</span></span>
<span id="cb6-37"><a href="3-res.html#cb6-37"></a>        <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb6-38"><a href="3-res.html#cb6-38"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb6-39"><a href="3-res.html#cb6-39"></a>            <span class="fu">run</span>(<span class="st">&quot;Set Scale...&quot;</span>,</span>
<span id="cb6-40"><a href="3-res.html#cb6-40"></a>                <span class="st">&quot;distance = 1 known = 0.00005 pixel = 1 unit = micron&quot;</span>);</span>
<span id="cb6-41"><a href="3-res.html#cb6-41"></a>            List.setMeasurements;</span>
<span id="cb6-42"><a href="3-res.html#cb6-42"></a>            x <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb6-43"><a href="3-res.html#cb6-43"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, x);</span>
<span id="cb6-44"><a href="3-res.html#cb6-44"></a>        }</span>
<span id="cb6-45"><a href="3-res.html#cb6-45"></a>        <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>,</span>
<span id="cb6-46"><a href="3-res.html#cb6-46"></a>            <span class="st">&quot;channels = 1 slices = 1 frames = [s] unit = micron pixel_width = [xs]</span></span>
<span id="cb6-47"><a href="3-res.html#cb6-47"></a><span class="st">            pixel_height = [ys] voxel_depth = [zs] frame = [time] global&quot;</span>);</span>
<span id="cb6-48"><a href="3-res.html#cb6-48"></a>        </span>
<span id="cb6-49"><a href="3-res.html#cb6-49"></a>    <span class="co"># Sort ROIs and select last one</span></span>
<span id="cb6-50"><a href="3-res.html#cb6-50"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;Sort&quot;</span>);</span>
<span id="cb6-51"><a href="3-res.html#cb6-51"></a>        <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb6-52"><a href="3-res.html#cb6-52"></a>            ccn <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="sc">+</span>j;</span>
<span id="cb6-53"><a href="3-res.html#cb6-53"></a>            <span class="cf">if</span> (ccn <span class="sc">==</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>)) {</span>
<span id="cb6-54"><a href="3-res.html#cb6-54"></a>                ccn <span class="ot">=</span> <span class="st">&quot;prim&quot;</span>;</span>
<span id="cb6-55"><a href="3-res.html#cb6-55"></a>                roiselect <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="sc">-</span><span class="dv">1</span>;</span>
<span id="cb6-56"><a href="3-res.html#cb6-56"></a>            } <span class="cf">else</span> {</span>
<span id="cb6-57"><a href="3-res.html#cb6-57"></a>                ccn <span class="ot">=</span> <span class="st">&quot;CC&quot;</span><span class="sc">+</span>j;</span>
<span id="cb6-58"><a href="3-res.html#cb6-58"></a>                roiselect <span class="ot">=</span> j<span class="dv">-1</span>;</span>
<span id="cb6-59"><a href="3-res.html#cb6-59"></a>            }</span>
<span id="cb6-60"><a href="3-res.html#cb6-60"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, roiselect);</span>
<span id="cb6-61"><a href="3-res.html#cb6-61"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, ccn);</span>
<span id="cb6-62"><a href="3-res.html#cb6-62"></a>        }</span>
<span id="cb6-63"><a href="3-res.html#cb6-63"></a>        rmc <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb6-64"><a href="3-res.html#cb6-64"></a>        m <span class="ot">=</span> rmc<span class="dv">-1</span>;</span>
<span id="cb6-65"><a href="3-res.html#cb6-65"></a>        <span class="fu">selectWindow</span>(RC);</span>
<span id="cb6-66"><a href="3-res.html#cb6-66"></a>        </span>
<span id="cb6-67"><a href="3-res.html#cb6-67"></a>  <span class="co"># Prim registration</span></span>
<span id="cb6-68"><a href="3-res.html#cb6-68"></a>        <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-69"><a href="3-res.html#cb6-69"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-70"><a href="3-res.html#cb6-70"></a>        sln <span class="ot">=</span> <span class="fu">getSliceNumber</span>();</span>
<span id="cb6-71"><a href="3-res.html#cb6-71"></a>        <span class="fu">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge=6&quot;</span>);</span>
<span id="cb6-72"><a href="3-res.html#cb6-72"></a>        <span class="fu">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb6-73"><a href="3-res.html#cb6-73"></a>        <span class="fu">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot;use&quot;</span>);</span>
<span id="cb6-74"><a href="3-res.html#cb6-74"></a>        <span class="fu">rename</span>(sln);</span>
<span id="cb6-75"><a href="3-res.html#cb6-75"></a>        <span class="fu">resetMinAndMax</span>();</span>
<span id="cb6-76"><a href="3-res.html#cb6-76"></a>        </span>
<span id="cb6-77"><a href="3-res.html#cb6-77"></a>  <span class="co"># Rotate</span></span>
<span id="cb6-78"><a href="3-res.html#cb6-78"></a>        List.setMeasurements;</span>
<span id="cb6-79"><a href="3-res.html#cb6-79"></a>        A <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb6-80"><a href="3-res.html#cb6-80"></a>        <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-81"><a href="3-res.html#cb6-81"></a>        <span class="cf">if</span> (A <span class="sc">&lt;</span> <span class="dv">10</span>) {</span>
<span id="cb6-82"><a href="3-res.html#cb6-82"></a>            A <span class="ot">=</span> A;</span>
<span id="cb6-83"><a href="3-res.html#cb6-83"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-84"><a href="3-res.html#cb6-84"></a>            A <span class="ot">=</span> <span class="dv">180</span><span class="sc">-</span>A;</span>
<span id="cb6-85"><a href="3-res.html#cb6-85"></a>            A <span class="ot">=</span> A<span class="sc">*</span>(<span class="sc">-</span><span class="dv">1</span>);</span>
<span id="cb6-86"><a href="3-res.html#cb6-86"></a>        }</span>
<span id="cb6-87"><a href="3-res.html#cb6-87"></a>        pangles[i] <span class="ot">=</span> A;</span>
<span id="cb6-88"><a href="3-res.html#cb6-88"></a>        <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = [A] grid = 1 interpolation = Bilinear slice&quot;</span>);</span>
<span id="cb6-89"><a href="3-res.html#cb6-89"></a>        <span class="fu">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>);</span>
<span id="cb6-90"><a href="3-res.html#cb6-90"></a>            </span>
<span id="cb6-91"><a href="3-res.html#cb6-91"></a>  <span class="co"># Measure and save segmented Mask ROI</span></span>
<span id="cb6-92"><a href="3-res.html#cb6-92"></a>        pLLProis <span class="ot">=</span> embryodir <span class="sc">+</span> File.separator <span class="sc">+</span> <span class="st">&quot;ROIs&quot;</span> <span class="sc">+</span> File.separator;</span>
<span id="cb6-93"><a href="3-res.html#cb6-93"></a>        <span class="fu">File.makeDirectory</span>(pLLProis);</span>
<span id="cb6-94"><a href="3-res.html#cb6-94"></a>        pLLPxy <span class="ot">=</span> embryodir <span class="sc">+</span> File.separator <span class="sc">+</span> <span class="st">&quot;ROIsXY&quot;</span> <span class="sc">+</span> File.separator;</span>
<span id="cb6-95"><a href="3-res.html#cb6-95"></a>        <span class="fu">File.makeDirectory</span>(pLLPxy);</span>
<span id="cb6-96"><a href="3-res.html#cb6-96"></a>        <span class="fu">selectWindow</span>(BIN);</span>
<span id="cb6-97"><a href="3-res.html#cb6-97"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;show none&quot;</span>); <span class="co"># supress roimanager popping up</span></span>
<span id="cb6-98"><a href="3-res.html#cb6-98"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-99"><a href="3-res.html#cb6-99"></a>            </span>
<span id="cb6-100"><a href="3-res.html#cb6-100"></a>    <span class="co">#   Save ROIs and XY coordinates</span></span>
<span id="cb6-101"><a href="3-res.html#cb6-101"></a>        <span class="cf">if</span> (i <span class="sc">&lt;</span> <span class="dv">10</span>) {</span>
<span id="cb6-102"><a href="3-res.html#cb6-102"></a>            slice <span class="ot">=</span> <span class="fu">d2s</span>(<span class="dv">0</span>,<span class="dv">0</span>) <span class="sc">+</span> <span class="fu">d2s</span>(i,<span class="dv">0</span>);</span>
<span id="cb6-103"><a href="3-res.html#cb6-103"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;save&quot;</span>, pLLProis <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> slice <span class="sc">+</span> <span class="st">&quot;.zip&quot;</span>);</span>
<span id="cb6-104"><a href="3-res.html#cb6-104"></a>            <span class="fu">saveAs</span>(<span class="st">&quot;XY Coordinates&quot;</span>, pLLPxy <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> slice <span class="sc">+</span> <span class="st">&quot;.txt&quot;</span>);</span>
<span id="cb6-105"><a href="3-res.html#cb6-105"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-106"><a href="3-res.html#cb6-106"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;save&quot;</span>, pLLProis <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> i <span class="sc">+</span> <span class="st">&quot;.zip&quot;</span>);</span>
<span id="cb6-107"><a href="3-res.html#cb6-107"></a>            <span class="fu">saveAs</span>(<span class="st">&quot;XY Coordinates&quot;</span>, pLLPxy <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> i <span class="sc">+</span> <span class="st">&quot;.txt&quot;</span>);</span>
<span id="cb6-108"><a href="3-res.html#cb6-108"></a>        }</span>
<span id="cb6-109"><a href="3-res.html#cb6-109"></a>            </span>
<span id="cb6-110"><a href="3-res.html#cb6-110"></a>     <span class="co"># Measure</span></span>
<span id="cb6-111"><a href="3-res.html#cb6-111"></a>        <span class="fu">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, </span>
<span id="cb6-112"><a href="3-res.html#cb6-112"></a>            <span class="st">&quot;area centroid bounding fit shape feret&#39;s stack redirect = None decimal = 2&quot;</span>);</span>
<span id="cb6-113"><a href="3-res.html#cb6-113"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;measure&quot;</span>);</span>
<span id="cb6-114"><a href="3-res.html#cb6-114"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-115"><a href="3-res.html#cb6-115"></a>        <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-116"><a href="3-res.html#cb6-116"></a>    <span class="co"># Calculate additional variables based on measurements</span></span>
<span id="cb6-117"><a href="3-res.html#cb6-117"></a>        n <span class="ot">=</span> <span class="fu">nResults</span>();</span>
<span id="cb6-118"><a href="3-res.html#cb6-118"></a>        r <span class="ot">=</span> n<span class="dv">-1</span>;  <span class="co"># actual RowNumber</span></span>
<span id="cb6-119"><a href="3-res.html#cb6-119"></a>        r2 <span class="ot">=</span> n<span class="dv">-2</span>; <span class="co"># RowNumber -1</span></span>
<span id="cb6-120"><a href="3-res.html#cb6-120"></a>        <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {  <span class="co"># get X &amp; Y coordinates, keep X0 and Y0 for normalization</span></span>
<span id="cb6-121"><a href="3-res.html#cb6-121"></a>            X0 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb6-122"><a href="3-res.html#cb6-122"></a>            Y0 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb6-123"><a href="3-res.html#cb6-123"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-124"><a href="3-res.html#cb6-124"></a>            X1 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;X&quot;</span>, r2);</span>
<span id="cb6-125"><a href="3-res.html#cb6-125"></a>            X2 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;X&quot;</span>, r);</span>
<span id="cb6-126"><a href="3-res.html#cb6-126"></a>            Y1 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Y&quot;</span>, r2);</span>
<span id="cb6-127"><a href="3-res.html#cb6-127"></a>            Y2 <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Y&quot;</span>, r); </span>
<span id="cb6-128"><a href="3-res.html#cb6-128"></a>        }</span>
<span id="cb6-129"><a href="3-res.html#cb6-129"></a>    <span class="co"># Width of bounding rectangle</span></span>
<span id="cb6-130"><a href="3-res.html#cb6-130"></a>        W <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Width&quot;</span>);</span>
<span id="cb6-131"><a href="3-res.html#cb6-131"></a>    <span class="co"># Calculations (XN = normalized X; LE = Leading Edge)</span></span>
<span id="cb6-132"><a href="3-res.html#cb6-132"></a>    <span class="co"># Euclidian Distance of X + normalized to offspring &#39;zero&#39;</span></span>
<span id="cb6-133"><a href="3-res.html#cb6-133"></a>        <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb6-134"><a href="3-res.html#cb6-134"></a>            XED <span class="ot">=</span> <span class="dv">0</span>;</span>
<span id="cb6-135"><a href="3-res.html#cb6-135"></a>            XN <span class="ot">=</span> <span class="dv">0</span>;</span>
<span id="cb6-136"><a href="3-res.html#cb6-136"></a>        } <span class="cf">else</span> {</span>
<span id="cb6-137"><a href="3-res.html#cb6-137"></a>            XED <span class="ot">=</span> <span class="fu">sqrt</span>((X2<span class="sc">-</span>X1)<span class="sc">*</span>(X2<span class="sc">-</span>X1)<span class="sc">+</span>(Y2<span class="sc">-</span>Y1)<span class="sc">*</span>(Y2<span class="sc">-</span>Y1));</span>
<span id="cb6-138"><a href="3-res.html#cb6-138"></a>            XN <span class="ot">=</span> (X2 <span class="sc">-</span> X0) <span class="sc">+</span> XED;</span>
<span id="cb6-139"><a href="3-res.html#cb6-139"></a>        }</span>
<span id="cb6-140"><a href="3-res.html#cb6-140"></a>        LE <span class="ot">=</span> XN <span class="sc">+</span> (W<span class="sc">/</span><span class="dv">2</span>); <span class="co"># Leading Edge </span></span>
<span id="cb6-141"><a href="3-res.html#cb6-141"></a>        T <span class="ot">=</span> time <span class="sc">*</span> r; <span class="co"># Time interval</span></span>
<span id="cb6-142"><a href="3-res.html#cb6-142"></a>        <span class="fu">setResult</span>(<span class="st">&quot;embryo&quot;</span>, r, orgname); <span class="co"># set Results</span></span>
<span id="cb6-143"><a href="3-res.html#cb6-143"></a>        <span class="fu">setResult</span>(<span class="st">&quot;group&quot;</span>, r, type);</span>
<span id="cb6-144"><a href="3-res.html#cb6-144"></a>        <span class="fu">setResult</span>(<span class="st">&quot;time&quot;</span>, r, T);</span>
<span id="cb6-145"><a href="3-res.html#cb6-145"></a>        <span class="fu">setResult</span>(<span class="st">&quot;deg&quot;</span>, r, A);</span>
<span id="cb6-146"><a href="3-res.html#cb6-146"></a>        <span class="fu">setResult</span>(<span class="st">&quot;X_ED&quot;</span>, r, XED);</span>
<span id="cb6-147"><a href="3-res.html#cb6-147"></a>        <span class="fu">setResult</span>(<span class="st">&quot;X_N&quot;</span>, r, XN);</span>
<span id="cb6-148"><a href="3-res.html#cb6-148"></a>        <span class="fu">updateResults</span>();</span>
<span id="cb6-149"><a href="3-res.html#cb6-149"></a>        }</span>
<span id="cb6-150"><a href="3-res.html#cb6-150"></a>        <span class="fu">close</span>(BIN, RC);</span>
<span id="cb6-151"><a href="3-res.html#cb6-151"></a>        </span>
<span id="cb6-152"><a href="3-res.html#cb6-152"></a>    <span class="co">#   Merge registered prim timepoints</span></span>
<span id="cb6-153"><a href="3-res.html#cb6-153"></a>        <span class="fu">setBatchMode</span>(<span class="st">&quot;exit and display&quot;</span>);</span>
<span id="cb6-154"><a href="3-res.html#cb6-154"></a>        <span class="fu">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;method = [Copy (top-left)] name = Stack title = [] use&quot;</span>);</span>
<span id="cb6-155"><a href="3-res.html#cb6-155"></a>        <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>,</span>
<span id="cb6-156"><a href="3-res.html#cb6-156"></a>            <span class="st">&quot;channels = 1 slices = 1 frames = [s] unit = micron pixel_width = [xs]</span></span>
<span id="cb6-157"><a href="3-res.html#cb6-157"></a><span class="st">            pixel_height = [ys] voxel_depth = [zs] frame = [time] global&quot;</span>); </span>
<span id="cb6-158"><a href="3-res.html#cb6-158"></a>        <span class="fu">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb6-159"><a href="3-res.html#cb6-159"></a>        <span class="cf">if</span> (dual) {</span>
<span id="cb6-160"><a href="3-res.html#cb6-160"></a>      <span class="co"># save C1</span></span>
<span id="cb6-161"><a href="3-res.html#cb6-161"></a>          <span class="fu">saveAs</span>(<span class="st">&quot;Tiff&quot;</span>, pLLPdir <span class="sc">+</span> orgname <span class="sc">+</span> <span class="st">&quot;-C01.tif&quot;</span>);</span>
<span id="cb6-162"><a href="3-res.html#cb6-162"></a>          <span class="fu">close</span>();</span>
<span id="cb6-163"><a href="3-res.html#cb6-163"></a>      <span class="co"># open C2</span></span>
<span id="cb6-164"><a href="3-res.html#cb6-164"></a>          <span class="fu">open</span>(rcdirc2 <span class="sc">+</span> rcdirc2list[b]);</span>
<span id="cb6-165"><a href="3-res.html#cb6-165"></a>            <span class="fu">resetMinAndMax</span>();</span>
<span id="cb6-166"><a href="3-res.html#cb6-166"></a>            RC <span class="ot">=</span> <span class="fu">getTitle</span>();</span>
<span id="cb6-167"><a href="3-res.html#cb6-167"></a>            dotIndex <span class="ot">=</span> <span class="fu">indexOf</span>(RC, <span class="st">&quot;.&quot;</span>);</span>
<span id="cb6-168"><a href="3-res.html#cb6-168"></a>            title <span class="ot">=</span> <span class="fu">substring</span>(RC, <span class="dv">0</span>, dotIndex);</span>
<span id="cb6-169"><a href="3-res.html#cb6-169"></a>            <span class="cf">for</span> (<span class="at">i =</span> <span class="dv">1</span> ; i <span class="sc">&lt;=</span> <span class="fu">nSlices</span>(); i<span class="sc">++</span>) {</span>
<span id="cb6-170"><a href="3-res.html#cb6-170"></a>                s <span class="ot">=</span> i;</span>
<span id="cb6-171"><a href="3-res.html#cb6-171"></a>                <span class="fu">setSlice</span>(i);</span>
<span id="cb6-172"><a href="3-res.html#cb6-172"></a>            <span class="co">#   Prim registration</span></span>
<span id="cb6-173"><a href="3-res.html#cb6-173"></a>                <span class="cf">if</span> (i <span class="sc">&lt;</span> <span class="dv">10</span>) {</span>
<span id="cb6-174"><a href="3-res.html#cb6-174"></a>                    slice <span class="ot">=</span> <span class="fu">d2s</span>(<span class="dv">0</span>, <span class="dv">0</span>) <span class="sc">+</span> <span class="fu">d2s</span>(i, <span class="dv">0</span>);</span>
<span id="cb6-175"><a href="3-res.html#cb6-175"></a>                    <span class="fu">roiManager</span>(<span class="st">&quot;open&quot;</span>, pLLProis <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> slice <span class="sc">+</span> <span class="st">&quot;.zip&quot;</span>);</span>
<span id="cb6-176"><a href="3-res.html#cb6-176"></a>                } <span class="cf">else</span> {</span>
<span id="cb6-177"><a href="3-res.html#cb6-177"></a>                    <span class="fu">roiManager</span>(<span class="st">&quot;open&quot;</span>, pLLProis <span class="sc">+</span> <span class="st">&quot;s&quot;</span> <span class="sc">+</span> i <span class="sc">+</span> <span class="st">&quot;.zip&quot;</span>);</span>
<span id="cb6-178"><a href="3-res.html#cb6-178"></a>                }</span>
<span id="cb6-179"><a href="3-res.html#cb6-179"></a>                rmc <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>);</span>
<span id="cb6-180"><a href="3-res.html#cb6-180"></a>                m <span class="ot">=</span> rmc<span class="dv">-1</span>;</span>
<span id="cb6-181"><a href="3-res.html#cb6-181"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;Select&quot;</span>, m);</span>
<span id="cb6-182"><a href="3-res.html#cb6-182"></a>                <span class="fu">selectWindow</span>(RC);</span>
<span id="cb6-183"><a href="3-res.html#cb6-183"></a>                sln <span class="ot">=</span> <span class="fu">getSliceNumber</span>();</span>
<span id="cb6-184"><a href="3-res.html#cb6-184"></a>                <span class="fu">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge=6&quot;</span>);</span>
<span id="cb6-185"><a href="3-res.html#cb6-185"></a>                <span class="fu">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb6-186"><a href="3-res.html#cb6-186"></a>                <span class="fu">run</span>(<span class="st">&quot;Duplicate...&quot;</span>, <span class="st">&quot;use&quot;</span>);</span>
<span id="cb6-187"><a href="3-res.html#cb6-187"></a>                <span class="fu">rename</span>(sln);</span>
<span id="cb6-188"><a href="3-res.html#cb6-188"></a>            <span class="co">#   Rotate</span></span>
<span id="cb6-189"><a href="3-res.html#cb6-189"></a>                A <span class="ot">=</span> pangles[s];</span>
<span id="cb6-190"><a href="3-res.html#cb6-190"></a>                <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-191"><a href="3-res.html#cb6-191"></a>                <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = [A] grid = 1 interpolation = Bilinear slice&quot;</span>);</span>
<span id="cb6-192"><a href="3-res.html#cb6-192"></a>                <span class="fu">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>);</span>
<span id="cb6-193"><a href="3-res.html#cb6-193"></a>            <span class="co"># select &amp; deselect to remove selected ROIs</span></span>
<span id="cb6-194"><a href="3-res.html#cb6-194"></a>                <span class="fu">selectWindow</span>(RC);</span>
<span id="cb6-195"><a href="3-res.html#cb6-195"></a>                <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb6-196"><a href="3-res.html#cb6-196"></a>                <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-197"><a href="3-res.html#cb6-197"></a>            }</span>
<span id="cb6-198"><a href="3-res.html#cb6-198"></a>        <span class="co">#   close and merge individual pllp images into one stack</span></span>
<span id="cb6-199"><a href="3-res.html#cb6-199"></a>            <span class="fu">close</span>(RC);</span>
<span id="cb6-200"><a href="3-res.html#cb6-200"></a>            <span class="fu">run</span>(<span class="st">&quot;Images to Stack&quot;</span>, <span class="st">&quot;method = [Copy (top-left)] name = Stack title = [] use&quot;</span>);</span>
<span id="cb6-201"><a href="3-res.html#cb6-201"></a>            <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = 1 frames = [s] unit = micron</span></span>
<span id="cb6-202"><a href="3-res.html#cb6-202"></a><span class="st">                pixel_width = [xs] pixel_height = [ys] voxel_depth = [zs] frame = [time] global&quot;</span>);  </span>
<span id="cb6-203"><a href="3-res.html#cb6-203"></a>            <span class="fu">run</span>(<span class="st">&quot;Flip Horizontally&quot;</span>, <span class="st">&quot;stack&quot;</span>);</span>
<span id="cb6-204"><a href="3-res.html#cb6-204"></a>            <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb6-205"><a href="3-res.html#cb6-205"></a>                </span>
<span id="cb6-206"><a href="3-res.html#cb6-206"></a>     <span class="co">#  Save Results Table</span></span>
<span id="cb6-207"><a href="3-res.html#cb6-207"></a>      <span class="fu">run</span>(<span class="st">&quot;Input/Output...&quot;</span>,</span>
<span id="cb6-208"><a href="3-res.html#cb6-208"></a>          <span class="st">&quot;jpeg = 100 gif = -1 file = .txt use_file copy_column copy_row save_column&quot;</span>);</span>
<span id="cb6-209"><a href="3-res.html#cb6-209"></a>            <span class="fu">saveAs</span>(<span class="st">&quot;results&quot;</span>, embryodir <span class="sc">+</span> orgname <span class="sc">+</span> <span class="st">&quot;_Results&quot;</span> <span class="sc">+</span> <span class="st">&quot;.txt&quot;</span>);</span></code></pre></div>
<p></p>
</div>
</div>
<div id="mat-anallzr3d" class="section level3" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> anaLLzr3D - Automated 3D single cell segmentation and A.I. analysis in the pLLP</h3>
<p>For measurement of the apical index (section <a href="2-mat-met.html#ACI">2.2.4.3</a>) I developed a custom IJ macro script that segments and analyses each cell of a pLLP in 3D. Upon macro execution the opening dialog is presented which is divided in three sections (figure <a href="3-res.html#fig:anallzr3ddialog">3.8</a>A).</p>
<p>In the <strong>processing</strong> section the user has to define the input format as well as in which direction the Z-stack was recorded. Furthermore, the user may choose to have the pLLP registered, save intermediate steps for debugging and to have objects segmented without any restrictions and manual ROI correction.</p>
<p>In the <strong>thresholds</strong> section the user may fine tune segmentation and filter thresholds:</p>
<ul>
<li><em>Segmentation</em> controls the segmentation threshold at which the membrane signal is detected and therefore the cell volumes are separated from each other (section <a href="2-mat-met.html#mat-GrTrDat">2.3</a>)</li>
<li><em>Min. volume</em> controls the minimum volume, below which objects are discarded</li>
<li><em>Max. volume</em> controls the maximum volume, above which objects are discarded</li>
</ul>
<p>In the <strong>measurements</strong> section, the user may choose whether apical constriction measurement should be applied or not. In case apical constriction measurement is selected, the user may choose from a second dialog box whether A.I. should be measured at an absolute- or relative- distance from the tip. Furthermore, the user has the option to measure from a fit ellipsoid (as done for the A.I. measurement described in section <a href="2-mat-met.html#ACI">2.2.4.3</a>) or rectangle.</p>

<div class="figure" style="text-align: center"><span id="fig:anallzr3ddialog"></span>
<img src="figures/materials/macros/anallzr3D_macro.png" alt="anaLLzR3D opening dialog A Opening dialog: Main functionality B Opening Dialog: Apical Constriction options C Log window after startup" width="95%" />
<p class="caption">
Figure 3.8: anaLLzR3D opening dialog <strong>A</strong> Opening dialog: Main functionality <strong>B</strong> Opening Dialog: Apical Constriction options <strong>C</strong> Log window after startup
</p>
</div>
<div id="image-analysis-1" class="section level4" number="3.1.4.1">
<h4><span class="header-section-number">3.1.4.1</span> Image Analysis</h4>
<p>For pLLP analyses I developed a custom IJ macro script that recognizes cell boundaries <em>via</em> the fluorescence signal emitted by a membrane tethered eGFP which expression is controlled by the <em>claudinB</em> lateral line specific promotor <span class="citation">(<em>32</em>)</span>. The central IJ tool used to do this is the MorphoLibJ’s<span class="citation">(<em>70</em>)</span> <em>Morpholigical Segmentation</em> plugin. The plugin however requires to choose for a ‘segmentation threshold’ that determines the quality and the quantity of segmented objects. This parameter therefore plays an essential role in the reliability of the analysis results.</p>
<div id="registration-1" class="section level5" number="3.1.4.1.1">
<h5><span class="header-section-number">3.1.4.1.1</span> Registration</h5>
<p>The first module of the macro is the registration of the pLLP in X, Y and cropping in Z. This is accomplished by an initial maximum Z projection and blurring of the image, 2-D segmentation using a minimum threshold and lastly by rotating the segment through the angle formed by the long axis of the ellipsoid (see section <a href="2-mat-met.html#ACI-pol">2.2.4.3.1.3</a> for more information) and the horizon (at 0<span class="math inline">\(^{\circ}\)</span>). After rotation the image is cropped according to the obtained ROI, as described before. Additionally, the centers of the most constricting areas are detected <em>via</em> an intensity based dynamic threshold and highlighted as magenta circles in figure <a href="3-res.html#fig:maxraw">3.9</a>.</p>

<div class="figure" style="text-align: center"><span id="fig:maxraw"></span>
<img src="figures/materials/ground_truth/registration.png" alt="Registration of 3D data. (unregistered) location and orientation of the unregistered MaxIPs in XY. The red line indicates the angle in degrees from the horizontal midline. The blue oval indicates registration ROI as determined by the macro. (registered) pLLPs after XY transformation took place. red circles indicate rosette centers as detected by the macro based on maximum signal intensity." width="95%" />
<p class="caption">
Figure 3.9: Registration of 3D data. <strong>(unregistered)</strong> location and orientation of the unregistered MaxIPs in XY. The red line indicates the angle in degrees from the horizontal midline. The blue oval indicates registration ROI as determined by the macro. <strong>(registered)</strong> pLLPs after XY transformation took place. red circles indicate rosette centers as detected by the macro based on maximum signal intensity.
</p>
</div>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imgprop">Table 3.2: </span>3-D Ground Truth image scaling
</caption>
<tbody>
<tr>
<td style="text-align:right;">
XY resolution
</td>
<td style="text-align:left;">
0.1625 / 0.1625 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;">
Z resolution
</td>
<td style="text-align:left;">
0.4 <span class="math inline">\(\mu\)</span>m
</td>
</tr>
<tr>
<td style="text-align:right;">
Time difference between Z-planes in Z
</td>
<td style="text-align:left;">
0.5428 s
</td>
</tr>
</tbody>
</table>
</div>
<div id="image-data" class="section level5" number="3.1.4.1.2">
<h5><span class="header-section-number">3.1.4.1.2</span> Image data</h5>
<p>In figure <a href="3-res.html#fig:stackmem">3.10</a> the fluorescence signal of the three pLLPs used for the Ground-Truth is shown in a single central cross-section along the <em>dorso-ventral</em> and the <em>apico-basal</em> axis.</p>

<div class="figure" style="text-align: center"><span id="fig:stackmem"></span>
<img src="figures/materials/ground_truth/stackmem.png" alt="cldnb:lyn-gfp fluorescence signal in a cross-section of the pLLP (Obj.: 40X APO, scale bar = 100 \(\mu\)m)" width="70%" />
<p class="caption">
Figure 3.10: <em>cldnb:lyn-gfp</em> fluorescence signal in a cross-section of the pLLP (Obj.: 40X APO, scale bar = 100 <span class="math inline">\(\mu\)</span>m)
</p>
</div>
<p>To compare the results between the Ground Truth segments and the segments obtained from different threshold levels graphically, for a single pLLP the Ground Truth and threshold levels are shown as a composite color image in figure <a href="3-res.html#fig:anallzrvols">3.11</a>. By using a green lookup-table (LUT) for the Ground Truth and a magenta LUT for the threshold level[n], one can readily detect overlapping objects (white), over segmentation (magenta) and under segmentation (green). False <em>positive</em> segments are cells that are not part of the Ground Truth, including those cells that would distort our dataset. False <em>negative</em> segments are cells that are part of the Ground Truth but were not detected by the macro (green cells), excluding those cells impacts the cell count. As one can see, the green cells are randomly distributed, therefore, when averaging a variable for all cells, those cells are less likely to distort our dataset. Only if under segmentation becomes too high will it impact the distribution of values. At Treshold Level 2 (T02) we have the least green cells, but the most magenta cells. At Threshold Level 4 (T04) we have no magenta, but some more green cells. Therefore, to be on the safe side in terms of cell parameters and dataset integrity, for the definition of our Ground Truth T04 would be the best pick.</p>

<div class="figure" style="text-align: center"><span id="fig:anallzrvols"></span>
<img src="figures/materials/ground_truth/volumes.png" alt="Graphical comparison of the thresholds tested. Volume renderings have been done with IJ’s VolumeViewer" width="85%" />
<p class="caption">
Figure 3.11: Graphical comparison of the thresholds tested. Volume renderings have been done with IJ’s <a href="%22https://github.com/fiji/Volume_Viewer/releases/tag/Volume_Viewer-2.01.2%22">VolumeViewer</a>
</p>
</div>
<p></p>
</div>
</div>
<div id="code-snippets-1" class="section level4" number="3.1.4.2">
<h4><span class="header-section-number">3.1.4.2</span> Code Snippets</h4>
<p>The <code>[...]</code> symbol indicates code re-use from an earlier instance.</p>
<div id="registration-2" class="section level5" number="3.1.4.2.1">
<h5><span class="header-section-number">3.1.4.2.1</span> Registration</h5>
<p>First we need to get parameters angle and height for registration. All steps are performed on Z-projected data.</p>
<p></p>
<div class="sourceCode" id="cb7"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb7-1"><a href="3-res.html#cb7-1"></a><span class="co"># 2D segmentation mask</span></span>
<span id="cb7-2"><a href="3-res.html#cb7-2"></a>  <span class="fu">run</span>(<span class="st">&quot;Z Project...&quot;</span>, <span class="st">&quot;projection = [Max Intensity]&quot;</span>);</span>
<span id="cb7-3"><a href="3-res.html#cb7-3"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 8 scaled&quot;</span>);</span>
<span id="cb7-4"><a href="3-res.html#cb7-4"></a>    <span class="fu">setAutoThreshold</span>(<span class="st">&quot;Minimum dark&quot;</span>);</span>
<span id="cb7-5"><a href="3-res.html#cb7-5"></a>    <span class="fu">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb7-6"><a href="3-res.html#cb7-6"></a>    <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-7"><a href="3-res.html#cb7-7"></a></span>
<span id="cb7-8"><a href="3-res.html#cb7-8"></a><span class="co"># angle from horizontal midline</span></span>
<span id="cb7-9"><a href="3-res.html#cb7-9"></a>    <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;include add&quot;</span>);</span>
<span id="cb7-10"><a href="3-res.html#cb7-10"></a>    rmcount <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>)<span class="sc">-</span><span class="dv">1</span>;</span>
<span id="cb7-11"><a href="3-res.html#cb7-11"></a>    <span class="cf">if</span>(<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb7-12"><a href="3-res.html#cb7-12"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-13"><a href="3-res.html#cb7-13"></a>        <span class="fu">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb7-14"><a href="3-res.html#cb7-14"></a>        List.setMeasurements;</span>
<span id="cb7-15"><a href="3-res.html#cb7-15"></a>        Angle <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;FeretAngle&quot;</span>);</span>
<span id="cb7-16"><a href="3-res.html#cb7-16"></a>        <span class="cf">if</span> (Angle <span class="sc">&lt;</span> <span class="dv">0</span>) {Angle <span class="ot">=</span> Angle <span class="sc">*</span> (<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb7-17"><a href="3-res.html#cb7-17"></a>        <span class="cf">if</span> (Angle <span class="sc">&gt;</span> <span class="dv">90</span>) {Angle <span class="ot">=</span> (<span class="dv">180</span> <span class="sc">-</span> Angle) <span class="sc">*</span> (<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb7-18"><a href="3-res.html#cb7-18"></a>    } <span class="cf">else</span> {</span>
<span id="cb7-19"><a href="3-res.html#cb7-19"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-20"><a href="3-res.html#cb7-20"></a>        <span class="fu">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb7-21"><a href="3-res.html#cb7-21"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;update&quot;</span>);</span>
<span id="cb7-22"><a href="3-res.html#cb7-22"></a>        List.setMeasurements;</span>
<span id="cb7-23"><a href="3-res.html#cb7-23"></a>        X1Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-24"><a href="3-res.html#cb7-24"></a>        Y1Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-25"><a href="3-res.html#cb7-25"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb7-26"><a href="3-res.html#cb7-26"></a>        List.setMeasurements;</span>
<span id="cb7-27"><a href="3-res.html#cb7-27"></a>        X2Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-28"><a href="3-res.html#cb7-28"></a>        Y2Line <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-29"><a href="3-res.html#cb7-29"></a>        <span class="fu">makeLine</span>(X1Line, Y1Line, X2Line, Y2Line); </span>
<span id="cb7-30"><a href="3-res.html#cb7-30"></a>        List.setMeasurements;</span>
<span id="cb7-31"><a href="3-res.html#cb7-31"></a>        Angle <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Angle&quot;</span>);</span>
<span id="cb7-32"><a href="3-res.html#cb7-32"></a>        <span class="cf">if</span> (Angle <span class="sc">&lt;</span> <span class="dv">0</span>) {Angle <span class="ot">=</span> Angle<span class="sc">*</span>(<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb7-33"><a href="3-res.html#cb7-33"></a>        <span class="cf">if</span> (Angle <span class="sc">&gt;</span> <span class="dv">90</span>) {Angle <span class="ot">=</span> (<span class="dv">180</span><span class="sc">-</span>Angle)<span class="sc">*</span>(<span class="sc">-</span><span class="dv">1</span>);}</span>
<span id="cb7-34"><a href="3-res.html#cb7-34"></a>    }</span>
<span id="cb7-35"><a href="3-res.html#cb7-35"></a>    <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-36"><a href="3-res.html#cb7-36"></a>    <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = &quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear&quot;</span>);</span>
<span id="cb7-37"><a href="3-res.html#cb7-37"></a>    </span>
<span id="cb7-38"><a href="3-res.html#cb7-38"></a><span class="co"># height to crop image to</span></span>
<span id="cb7-39"><a href="3-res.html#cb7-39"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>); <span class="co"># the image was rotated, so we need to get the ROIs again</span></span>
<span id="cb7-40"><a href="3-res.html#cb7-40"></a>    <span class="fu">run</span>(<span class="st">&quot;Select None&quot;</span>);</span>
<span id="cb7-41"><a href="3-res.html#cb7-41"></a>    <span class="fu">run</span>(<span class="st">&quot;Make Binary&quot;</span>);</span>
<span id="cb7-42"><a href="3-res.html#cb7-42"></a>    <span class="fu">run</span>(<span class="st">&quot;Erode&quot;</span>);</span>
<span id="cb7-43"><a href="3-res.html#cb7-43"></a>    <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;size = 150-10000 include exclude add&quot;</span>);</span>
<span id="cb7-44"><a href="3-res.html#cb7-44"></a>    rmcount <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">-</span> <span class="dv">1</span>;</span>
<span id="cb7-45"><a href="3-res.html#cb7-45"></a>    <span class="cf">if</span>(<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb7-46"><a href="3-res.html#cb7-46"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb7-47"><a href="3-res.html#cb7-47"></a>    } <span class="cf">else</span> {</span>
<span id="cb7-48"><a href="3-res.html#cb7-48"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, rmcount);</span>
<span id="cb7-49"><a href="3-res.html#cb7-49"></a>    }</span>
<span id="cb7-50"><a href="3-res.html#cb7-50"></a>    List.setMeasurements;</span>
<span id="cb7-51"><a href="3-res.html#cb7-51"></a>    XRect <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb7-52"><a href="3-res.html#cb7-52"></a>    YRect <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Y&quot;</span>);</span>
<span id="cb7-53"><a href="3-res.html#cb7-53"></a>    <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb7-54"><a href="3-res.html#cb7-54"></a>    Regwidth <span class="ot">=</span> width;</span>
<span id="cb7-55"><a href="3-res.html#cb7-55"></a>    Regheight <span class="ot">=</span> <span class="dv">400</span>; <span class="co"># change height of rectangle here</span></span>
<span id="cb7-56"><a href="3-res.html#cb7-56"></a>    <span class="fu">toUnscaled</span>(YRect);</span>
<span id="cb7-57"><a href="3-res.html#cb7-57"></a>    YRect <span class="ot">=</span> YRect <span class="sc">-</span> (Regheight<span class="sc">/</span><span class="dv">2</span>);</span></code></pre></div>
<p></p>
</div>
<div id="transformation" class="section level5" number="3.1.4.2.2">
<h5><span class="header-section-number">3.1.4.2.2</span> Transformation</h5>
<p>Next we transform our 3D data based on the registration parameters derived from the previous step.</p>
<p></p>
<div class="sourceCode" id="cb8"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb8-1"><a href="3-res.html#cb8-1"></a><span class="co"># register pLLP</span></span>
<span id="cb8-2"><a href="3-res.html#cb8-2"></a>    <span class="fu">run</span>(<span class="st">&quot;Rotate... &quot;</span>, <span class="st">&quot;angle = &quot;</span><span class="sc">+</span> Angle <span class="sc">+</span><span class="st">&quot; grid = 1 interpolation = Bilinear stack&quot;</span>);</span>
<span id="cb8-3"><a href="3-res.html#cb8-3"></a>    <span class="fu">makeRectangle</span>(<span class="dv">0</span>, YRect, Regwidth, Regheight);</span>
<span id="cb8-4"><a href="3-res.html#cb8-4"></a>    <span class="fu">run</span>(<span class="st">&quot;Crop&quot;</span>);</span>
<span id="cb8-5"><a href="3-res.html#cb8-5"></a></span>
<span id="cb8-6"><a href="3-res.html#cb8-6"></a><span class="co"># create threshold mask to clear signals outside ROI</span></span>
<span id="cb8-7"><a href="3-res.html#cb8-7"></a>    <span class="fu">run</span>(<span class="st">&quot;Normalize Local Contrast&quot;</span>, </span>
<span id="cb8-8"><a href="3-res.html#cb8-8"></a>        <span class="st">&quot;block_radius_x = 300 block_radius_y = 20 standard_deviations = 4 stretch&quot;</span>);</span>
<span id="cb8-9"><a href="3-res.html#cb8-9"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 1 scaled&quot;</span>);</span>
<span id="cb8-10"><a href="3-res.html#cb8-10"></a>    <span class="fu">setAutoThreshold</span>(<span class="st">&quot;Otsu dark&quot;</span>);</span>
<span id="cb8-11"><a href="3-res.html#cb8-11"></a>    <span class="fu">run</span>(<span class="st">&quot;Convert to Mask&quot;</span>);</span>
<span id="cb8-12"><a href="3-res.html#cb8-12"></a></span>
<span id="cb8-13"><a href="3-res.html#cb8-13"></a><span class="co"># most right roi</span></span>
<span id="cb8-14"><a href="3-res.html#cb8-14"></a>    <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">0</span> ; j <span class="sc">&lt;</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>); j<span class="sc">++</span>) {</span>
<span id="cb8-15"><a href="3-res.html#cb8-15"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;select&quot;</span>, j);</span>
<span id="cb8-16"><a href="3-res.html#cb8-16"></a>        <span class="fu">run</span>(<span class="st">&quot;Set Scale...&quot;</span>, <span class="st">&quot;distance = 1 known = 0.00005 pixel = 1 unit = micron&quot;</span>);</span>
<span id="cb8-17"><a href="3-res.html#cb8-17"></a>        List.setMeasurements;</span>
<span id="cb8-18"><a href="3-res.html#cb8-18"></a>    x <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;X&quot;</span>);</span>
<span id="cb8-19"><a href="3-res.html#cb8-19"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;rename&quot;</span>, x);</span>
<span id="cb8-20"><a href="3-res.html#cb8-20"></a>    }</span>
<span id="cb8-21"><a href="3-res.html#cb8-21"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;Sort&quot;</span>);</span>
<span id="cb8-22"><a href="3-res.html#cb8-22"></a>    <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>, </span>
<span id="cb8-23"><a href="3-res.html#cb8-23"></a>        <span class="st">&quot;channels = 1 slices = 1 frames = 1 unit = micron pixel_width = [sizeX] </span></span>
<span id="cb8-24"><a href="3-res.html#cb8-24"></a><span class="st">        pixel_height = [sizeY] voxel_depth = [sizeZ]&quot;</span>);</span>
<span id="cb8-25"><a href="3-res.html#cb8-25"></a>    primroi <span class="ot">=</span> <span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">-</span> <span class="dv">1</span>;</span>
<span id="cb8-26"><a href="3-res.html#cb8-26"></a>    <span class="fu">roiManager</span>(<span class="st">&#39;select&#39;</span>, primroi);</span>
<span id="cb8-27"><a href="3-res.html#cb8-27"></a>    </span>
<span id="cb8-28"><a href="3-res.html#cb8-28"></a><span class="co"># enlarge rois to not miss anything</span></span>
<span id="cb8-29"><a href="3-res.html#cb8-29"></a>    <span class="fu">run</span>(<span class="st">&quot;Enlarge...&quot;</span>, <span class="st">&quot;enlarge = 10&quot;</span>);</span>
<span id="cb8-30"><a href="3-res.html#cb8-30"></a>    <span class="fu">run</span>(<span class="st">&quot;Fit Ellipse&quot;</span>);</span>
<span id="cb8-31"><a href="3-res.html#cb8-31"></a>    <span class="fu">roiManager</span>(<span class="st">&#39;update&#39;</span>);</span></code></pre></div>
<p></p>
</div>
<div id="rosette-detection" class="section level5" number="3.1.4.2.3">
<h5><span class="header-section-number">3.1.4.2.3</span> Rosette detection</h5>
<p>To analyze cells within rosettes resp. within a certain radius of rosettes we first need to know where the rosettes are. At rosette centers we observe an increase in signal intensity since here the membranes of many cells come together in a very small area. This effect we can use to utilize a maximum finder algorithm together with a threshold that is defined individually for each image.</p>
<p></p>
<div class="sourceCode" id="cb9"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb9-1"><a href="3-res.html#cb9-1"></a><span class="co">#  [...] registration</span></span>
<span id="cb9-2"><a href="3-res.html#cb9-2"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur...&quot;</span>, <span class="st">&quot;sigma = 4 scaled&quot;</span>);</span>
<span id="cb9-3"><a href="3-res.html#cb9-3"></a>    List.setMeasurements;</span>
<span id="cb9-4"><a href="3-res.html#cb9-4"></a>    mean <span class="ot">=</span> <span class="fu">List.getValue</span>(<span class="st">&quot;Mean&quot;</span>);</span>
<span id="cb9-5"><a href="3-res.html#cb9-5"></a>    pointthresh <span class="ot">=</span> mean<span class="sc">/</span><span class="fl">2.5</span>;</span>
<span id="cb9-6"><a href="3-res.html#cb9-6"></a>    pointthresh <span class="ot">=</span> <span class="fu">round</span>(pointthresh);</span>
<span id="cb9-7"><a href="3-res.html#cb9-7"></a>    <span class="fu">run</span>(<span class="st">&quot;Find Maxima...&quot;</span>, <span class="st">&quot;noise = &quot;</span><span class="sc">+</span> pointthresh <span class="sc">+</span><span class="st">&quot; output = [Point Selection]&quot;</span>);</span>
<span id="cb9-8"><a href="3-res.html#cb9-8"></a>    <span class="fu">run</span>(<span class="st">&quot;Point Tool...&quot;</span>, <span class="st">&quot;type = Dot color = Green size = [Extra Large] label counter = 0&quot;</span>);</span>
<span id="cb9-9"><a href="3-res.html#cb9-9"></a>    <span class="fu">getSelectionCoordinates</span>(xpoints, ypoints);</span>
<span id="cb9-10"><a href="3-res.html#cb9-10"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;Add&quot;</span>);</span>
<span id="cb9-11"><a href="3-res.html#cb9-11"></a></span>
<span id="cb9-12"><a href="3-res.html#cb9-12"></a><span class="co"># measure intensities along horizontal midline</span></span>
<span id="cb9-13"><a href="3-res.html#cb9-13"></a><span class="co">#   [...] registration</span></span>
<span id="cb9-14"><a href="3-res.html#cb9-14"></a>    Rlx <span class="ot">=</span> <span class="fu">lengthOf</span>(xpoints); <span class="co"># collect Arrays</span></span>
<span id="cb9-15"><a href="3-res.html#cb9-15"></a>    RX <span class="ot">=</span> <span class="fu">Array.sort</span>(xpoints); <span class="co"># put xpoints in right order</span></span>
<span id="cb9-16"><a href="3-res.html#cb9-16"></a>    RX <span class="ot">=</span> <span class="fu">Array.invert</span>(RX);</span>
<span id="cb9-17"><a href="3-res.html#cb9-17"></a></span>
<span id="cb9-18"><a href="3-res.html#cb9-18"></a><span class="co"># fill ypoints with mean values of all y coordinates</span></span>
<span id="cb9-19"><a href="3-res.html#cb9-19"></a>    <span class="fu">Array.getStatistics</span>(ypoints, min, max, mean, stdDev);</span>
<span id="cb9-20"><a href="3-res.html#cb9-20"></a>    meanline <span class="ot">=</span> mean;</span>
<span id="cb9-21"><a href="3-res.html#cb9-21"></a>    <span class="fu">Array.fill</span>(ypoints, meanline);</span>
<span id="cb9-22"><a href="3-res.html#cb9-22"></a>    RY <span class="ot">=</span> ypoints;</span>
<span id="cb9-23"><a href="3-res.html#cb9-23"></a>    <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb9-24"><a href="3-res.html#cb9-24"></a>    <span class="fu">makeLine</span>(<span class="dv">0</span>, meanline, width, meanline, <span class="dv">1</span>);</span>
<span id="cb9-25"><a href="3-res.html#cb9-25"></a>    <span class="fu">run</span>(<span class="st">&quot;Clear Results&quot;</span>);</span>
<span id="cb9-26"><a href="3-res.html#cb9-26"></a>    profile <span class="ot">=</span> <span class="fu">getProfile</span>();</span>
<span id="cb9-27"><a href="3-res.html#cb9-27"></a>    <span class="cf">for</span> (<span class="at">a =</span> <span class="dv">0</span>; a <span class="sc">&lt;</span> profile.length; a<span class="sc">++</span>) {</span>
<span id="cb9-28"><a href="3-res.html#cb9-28"></a>    <span class="fu">setResult</span>(<span class="st">&quot;Value&quot;</span>, a, profile[a]);</span>
<span id="cb9-29"><a href="3-res.html#cb9-29"></a>        <span class="fu">updateResults</span>();</span>
<span id="cb9-30"><a href="3-res.html#cb9-30"></a>    }</span></code></pre></div>
<p></p>
</div>
<div id="segmentation-2" class="section level5" number="3.1.4.2.4">
<h5><span class="header-section-number">3.1.4.2.4</span> Segmentation</h5>
<p>For image segmentation we use the MorphoLibJ’s<span class="citation">(<em>70</em>)</span> <em>Morpholigical Segmentation</em> plugin. Function calls and arguments are defined in the publication documentation.</p>
<p></p>
<div class="sourceCode" id="cb10"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb10-1"><a href="3-res.html#cb10-1"></a><span class="co"># 3D gaussian blur</span></span>
<span id="cb10-2"><a href="3-res.html#cb10-2"></a>    <span class="fu">run</span>(<span class="st">&quot;Gaussian Blur 3D...&quot;</span>, <span class="st">&quot;x=2 y=2 z=0.5&quot;</span>);</span>
<span id="cb10-3"><a href="3-res.html#cb10-3"></a>    <span class="fu">resetMinAndMax</span>();</span>
<span id="cb10-4"><a href="3-res.html#cb10-4"></a></span>
<span id="cb10-5"><a href="3-res.html#cb10-5"></a><span class="co"># run segmentation</span></span>
<span id="cb10-6"><a href="3-res.html#cb10-6"></a>    <span class="fu">run</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-7"><a href="3-res.html#cb10-7"></a>    <span class="fu">selectWindow</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-8"><a href="3-res.html#cb10-8"></a>    <span class="fu">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.setInputImageType&quot;</span>, <span class="st">&quot;border&quot;</span>);</span>
<span id="cb10-9"><a href="3-res.html#cb10-9"></a>    <span class="fu">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.segment&quot;</span>, <span class="st">&quot;tolerance = &quot;</span> <span class="sc">+</span> tol <span class="sc">+</span> <span class="st">&quot;&quot;</span>,</span>
<span id="cb10-10"><a href="3-res.html#cb10-10"></a>         <span class="st">&quot;calculateDams = true&quot;</span>, <span class="st">&quot;connectivity = 6&quot;</span>);</span>
<span id="cb10-11"><a href="3-res.html#cb10-11"></a></span>
<span id="cb10-12"><a href="3-res.html#cb10-12"></a><span class="co"># wait till segmentation is done</span></span>
<span id="cb10-13"><a href="3-res.html#cb10-13"></a>    initTime <span class="ot">=</span> <span class="fu">getTime</span>(); </span>
<span id="cb10-14"><a href="3-res.html#cb10-14"></a>  oldTime <span class="ot">=</span> initTime; </span>
<span id="cb10-15"><a href="3-res.html#cb10-15"></a>    <span class="cf">while</span> (<span class="fu">isOpen</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>)) {</span>
<span id="cb10-16"><a href="3-res.html#cb10-16"></a>      elapsedTime <span class="ot">=</span> <span class="fu">getTime</span>() <span class="sc">-</span> initTime; </span>
<span id="cb10-17"><a href="3-res.html#cb10-17"></a>        newTime <span class="ot">=</span> <span class="fu">getTime</span>() <span class="sc">-</span> oldTime;</span>
<span id="cb10-18"><a href="3-res.html#cb10-18"></a>        <span class="cf">if</span> (newTime <span class="sc">&gt;</span> <span class="dv">10000</span>) {</span>
<span id="cb10-19"><a href="3-res.html#cb10-19"></a>        oldTime <span class="ot">=</span> <span class="fu">getTime</span>(); </span>
<span id="cb10-20"><a href="3-res.html#cb10-20"></a>        newTime <span class="ot">=</span> <span class="dv">0</span>;</span>
<span id="cb10-21"><a href="3-res.html#cb10-21"></a>        loginfo <span class="ot">=</span> <span class="fu">getInfo</span>(<span class="st">&quot;log&quot;</span>);</span>
<span id="cb10-22"><a href="3-res.html#cb10-22"></a>            loginfo <span class="ot">=</span> <span class="fu">split</span>(loginfo, <span class="st">&quot; &quot;</span>);</span>
<span id="cb10-23"><a href="3-res.html#cb10-23"></a>            loginfo <span class="ot">=</span> <span class="fu">Array.reverse</span>(loginfo);</span>
<span id="cb10-24"><a href="3-res.html#cb10-24"></a>            loginfo <span class="ot">=</span> <span class="fu">Array.trim</span>(loginfo, <span class="dv">5</span>);</span>
<span id="cb10-25"><a href="3-res.html#cb10-25"></a>            loginfo <span class="ot">=</span> <span class="fu">Array.reverse</span>(loginfo);</span>
<span id="cb10-26"><a href="3-res.html#cb10-26"></a>            loginfo <span class="ot">=</span> <span class="fu">split</span>(loginfo[<span class="dv">0</span>], <span class="st">&quot;.&quot;</span>);</span>
<span id="cb10-27"><a href="3-res.html#cb10-27"></a>            loginfo <span class="ot">=</span> <span class="fu">Array.reverse</span>(loginfo);</span>
<span id="cb10-28"><a href="3-res.html#cb10-28"></a>            loginfo <span class="ot">=</span> loginfo[<span class="dv">0</span>];</span>
<span id="cb10-29"><a href="3-res.html#cb10-29"></a>            <span class="cf">if</span> (loginfo <span class="sc">==</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">Whole&quot;</span>) {</span>
<span id="cb10-30"><a href="3-res.html#cb10-30"></a>                <span class="fu">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.setDisplayFormat&quot;</span>, </span>
<span id="cb10-31"><a href="3-res.html#cb10-31"></a>                     <span class="st">&quot;Catchment basins&quot;</span>);</span>
<span id="cb10-32"><a href="3-res.html#cb10-32"></a>                <span class="fu">call</span>(<span class="st">&quot;inra.ijpb.plugins.MorphologicalSegmentation.createResultImage&quot;</span>);</span>
<span id="cb10-33"><a href="3-res.html#cb10-33"></a>                <span class="fu">run</span>(<span class="st">&quot;Grays&quot;</span>);</span>
<span id="cb10-34"><a href="3-res.html#cb10-34"></a>                <span class="fu">selectWindow</span>(<span class="st">&quot;Morphological Segmentation&quot;</span>);</span>
<span id="cb10-35"><a href="3-res.html#cb10-35"></a>                <span class="fu">close</span>();</span>
<span id="cb10-36"><a href="3-res.html#cb10-36"></a>            }</span>
<span id="cb10-37"><a href="3-res.html#cb10-37"></a>    }</span>
<span id="cb10-38"><a href="3-res.html#cb10-38"></a>    }</span>
<span id="cb10-39"><a href="3-res.html#cb10-39"></a>  <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = &quot;</span> <span class="sc">+</span> n <span class="sc">+</span> <span class="st">&quot; frames = 1 unit = </span></span>
<span id="cb10-40"><a href="3-res.html#cb10-40"></a><span class="st">      microns pixel_width = &quot;</span> <span class="sc">+</span> sizeX <span class="sc">+</span> <span class="st">&quot; pixel_height = &quot;</span> <span class="sc">+</span> sizeY <span class="sc">+</span> <span class="st">&quot;voxel_depth = &quot;</span><span class="sc">+</span> sizeZ);</span></code></pre></div>
<p></p>
</div>
<div id="filter-and-clearing" class="section level5" number="3.1.4.2.5">
<h5><span class="header-section-number">3.1.4.2.5</span> Filter and clearing</h5>
<p>Remove segments below a certain volume threshold defined in the startup dialog and clear blank slices in Z.</p>
<p></p>
<div class="sourceCode" id="cb11"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb11-1"><a href="3-res.html#cb11-1"></a><span class="co"># erase objects V &lt; vmin and V &gt; vmax</span></span>
<span id="cb11-2"><a href="3-res.html#cb11-2"></a>  <span class="fu">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments </span></span>
<span id="cb11-3"><a href="3-res.html#cb11-3"></a><span class="st">      feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) </span></span>
<span id="cb11-4"><a href="3-res.html#cb11-4"></a><span class="st">      bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy </span></span>
<span id="cb11-5"><a href="3-res.html#cb11-5"></a><span class="st">      sync distance_between_centers = 10 distance_max_contact = 1.80&quot;</span>);</span>
<span id="cb11-6"><a href="3-res.html#cb11-6"></a>    <span class="fu">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb11-7"><a href="3-res.html#cb11-7"></a>  <span class="fu">selectWindow</span>(OMap);</span>
<span id="cb11-8"><a href="3-res.html#cb11-8"></a>    <span class="fu">Ext.Manager3D_AddImage</span>();</span>
<span id="cb11-9"><a href="3-res.html#cb11-9"></a></span>
<span id="cb11-10"><a href="3-res.html#cb11-10"></a><span class="co"># get number of objects</span></span>
<span id="cb11-11"><a href="3-res.html#cb11-11"></a>    <span class="fu">Ext.Manager3D_Count</span>(nb);</span>
<span id="cb11-12"><a href="3-res.html#cb11-12"></a>    <span class="fu">Ext.Manager3D_MultiSelect</span>();</span>
<span id="cb11-13"><a href="3-res.html#cb11-13"></a></span>
<span id="cb11-14"><a href="3-res.html#cb11-14"></a><span class="co"># loop through all the objects and erase by filter settings</span></span>
<span id="cb11-15"><a href="3-res.html#cb11-15"></a>    <span class="cf">for</span>(<span class="at">k =</span> <span class="dv">0</span>; k <span class="sc">&lt;</span> nb; k<span class="sc">++</span>) {</span>
<span id="cb11-16"><a href="3-res.html#cb11-16"></a>        <span class="fu">showStatus</span>(<span class="st">&quot;Processing &quot;</span><span class="sc">+</span> k <span class="sc">+</span><span class="st">&quot;/&quot;</span><span class="sc">+</span> nb);</span>
<span id="cb11-17"><a href="3-res.html#cb11-17"></a>        <span class="fu">Ext.Manager3D_Measure3D</span>(k, <span class="st">&quot;Vol&quot;</span>,V);</span>
<span id="cb11-18"><a href="3-res.html#cb11-18"></a>        <span class="cf">if</span> (V <span class="sc">&lt;</span> vmin) {</span>
<span id="cb11-19"><a href="3-res.html#cb11-19"></a>            <span class="fu">Ext.Manager3D_Select</span>(k);</span>
<span id="cb11-20"><a href="3-res.html#cb11-20"></a>            <span class="fu">Ext.Manager3D_Erase</span>();</span>
<span id="cb11-21"><a href="3-res.html#cb11-21"></a>            <span class="cf">if</span> (V <span class="sc">&gt;</span> vmax) {</span>
<span id="cb11-22"><a href="3-res.html#cb11-22"></a>                <span class="fu">Ext.Manager3D_Select</span>(k);</span>
<span id="cb11-23"><a href="3-res.html#cb11-23"></a>                <span class="fu">Ext.Manager3D_Erase</span>();</span>
<span id="cb11-24"><a href="3-res.html#cb11-24"></a>            }</span>
<span id="cb11-25"><a href="3-res.html#cb11-25"></a>        }</span>
<span id="cb11-26"><a href="3-res.html#cb11-26"></a>    }</span>
<span id="cb11-27"><a href="3-res.html#cb11-27"></a></span>
<span id="cb11-28"><a href="3-res.html#cb11-28"></a><span class="co"># clean blank slices from bottom and top</span></span>
<span id="cb11-29"><a href="3-res.html#cb11-29"></a>    <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb11-30"><a href="3-res.html#cb11-30"></a>    var done <span class="ot">=</span> false; </span>
<span id="cb11-31"><a href="3-res.html#cb11-31"></a>    <span class="cf">for</span>(<span class="at">l =</span> <span class="dv">1</span>; l <span class="sc">&lt;</span> slices <span class="sc">&amp;&amp;!</span>done; l<span class="sc">++</span>) {</span>
<span id="cb11-32"><a href="3-res.html#cb11-32"></a>        <span class="fu">setSlice</span>(l);</span>
<span id="cb11-33"><a href="3-res.html#cb11-33"></a>        <span class="fu">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb11-34"><a href="3-res.html#cb11-34"></a>        <span class="cf">if</span>(max <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb11-35"><a href="3-res.html#cb11-35"></a>        amax <span class="ot">=</span> l<span class="dv">-1</span>;</span>
<span id="cb11-36"><a href="3-res.html#cb11-36"></a>            <span class="fu">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="sc">+</span> amax <span class="sc">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb11-37"><a href="3-res.html#cb11-37"></a>            <span class="fu">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb11-38"><a href="3-res.html#cb11-38"></a>            <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb11-39"><a href="3-res.html#cb11-39"></a>            <span class="cf">for</span>(<span class="at">l =</span> <span class="dv">1</span>; l <span class="sc">&lt;</span> slices <span class="sc">&amp;&amp;!</span>done; l<span class="sc">++</span>) {</span>
<span id="cb11-40"><a href="3-res.html#cb11-40"></a>                <span class="fu">setSlice</span>(l);</span>
<span id="cb11-41"><a href="3-res.html#cb11-41"></a>                <span class="fu">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb11-42"><a href="3-res.html#cb11-42"></a>                <span class="cf">if</span>(max <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb11-43"><a href="3-res.html#cb11-43"></a>                bmax <span class="ot">=</span> l<span class="dv">-1</span>;</span>
<span id="cb11-44"><a href="3-res.html#cb11-44"></a>                    <span class="fu">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="sc">+</span> bmax <span class="sc">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb11-45"><a href="3-res.html#cb11-45"></a>                    <span class="fu">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb11-46"><a href="3-res.html#cb11-46"></a>                    done <span class="ot">=</span> true;</span>
<span id="cb11-47"><a href="3-res.html#cb11-47"></a>                    }</span>
<span id="cb11-48"><a href="3-res.html#cb11-48"></a>            }</span>
<span id="cb11-49"><a href="3-res.html#cb11-49"></a>        }</span>
<span id="cb11-50"><a href="3-res.html#cb11-50"></a>    }</span></code></pre></div>
<p></p>
</div>
<div id="apical-constriction-1" class="section level5" number="3.1.4.2.6">
<h5><span class="header-section-number">3.1.4.2.6</span> Apical Constriction</h5>
<p>Next we navigate to the calculated relative distance from the apical site and take measurements.</p>
<p></p>
<div class="sourceCode" id="cb12"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb12-1"><a href="3-res.html#cb12-1"></a><span class="co"># define Z values</span></span>
<span id="cb12-2"><a href="3-res.html#cb12-2"></a>    <span class="cf">if</span> (rosAC) {</span>
<span id="cb12-3"><a href="3-res.html#cb12-3"></a>        cellum <span class="ot">=</span> rosum; <span class="co"># I copy pasted the code, so rosette um would be cell um</span></span>
<span id="cb12-4"><a href="3-res.html#cb12-4"></a>        Zslice <span class="ot">=</span> rosum<span class="sc">/</span>sizeZ;</span>
<span id="cb12-5"><a href="3-res.html#cb12-5"></a>        <span class="fu">round</span>(Zslice);</span>
<span id="cb12-6"><a href="3-res.html#cb12-6"></a>    }</span>
<span id="cb12-7"><a href="3-res.html#cb12-7"></a><span class="co"># cell Constriction</span></span>
<span id="cb12-8"><a href="3-res.html#cb12-8"></a></span>
<span id="cb12-9"><a href="3-res.html#cb12-9"></a>    <span class="cf">if</span> (cellAC) {</span>
<span id="cb12-10"><a href="3-res.html#cb12-10"></a>        aciza <span class="ot">=</span> cellum <span class="sc">/</span> sizeZ;</span>
<span id="cb12-11"><a href="3-res.html#cb12-11"></a>        Zslice <span class="ot">=</span> aciza;</span>
<span id="cb12-12"><a href="3-res.html#cb12-12"></a>        <span class="fu">round</span>(Zslice);</span>
<span id="cb12-13"><a href="3-res.html#cb12-13"></a>        <span class="fu">round</span>(aciza);</span>
<span id="cb12-14"><a href="3-res.html#cb12-14"></a>        <span class="cf">if</span> (fixAC<span class="sc">||</span>symAC) {</span>
<span id="cb12-15"><a href="3-res.html#cb12-15"></a>    <span class="co"># crop single objects and measure ----------------------</span></span>
<span id="cb12-16"><a href="3-res.html#cb12-16"></a>      <span class="fu">selectWindow</span>(OMap);</span>
<span id="cb12-17"><a href="3-res.html#cb12-17"></a>        <span class="fu">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments </span></span>
<span id="cb12-18"><a href="3-res.html#cb12-18"></a><span class="st">        feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) </span></span>
<span id="cb12-19"><a href="3-res.html#cb12-19"></a><span class="st">        bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy </span></span>
<span id="cb12-20"><a href="3-res.html#cb12-20"></a><span class="st">        sync distance_between_centers = 10 distance_max_contact = 1.80&quot;</span>);</span>
<span id="cb12-21"><a href="3-res.html#cb12-21"></a>        <span class="fu">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb12-22"><a href="3-res.html#cb12-22"></a>        <span class="fu">Ext.Manager3D_AddImage</span>();</span>
<span id="cb12-23"><a href="3-res.html#cb12-23"></a>    <span class="co"># get number of objects</span></span>
<span id="cb12-24"><a href="3-res.html#cb12-24"></a>        <span class="fu">Ext.Manager3D_Count</span>(nb);</span>
<span id="cb12-25"><a href="3-res.html#cb12-25"></a>    <span class="co"># create arrays to fill with measurements</span></span>
<span id="cb12-26"><a href="3-res.html#cb12-26"></a>        objlabelArray <span class="ot">=</span> <span class="fu">newArray</span>(nb);</span>
<span id="cb12-27"><a href="3-res.html#cb12-27"></a>        MajorAngle <span class="ot">=</span> <span class="fu">newArray</span>(nb);</span>
<span id="cb12-28"><a href="3-res.html#cb12-28"></a>        ACIMajor <span class="ot">=</span> <span class="fu">newArray</span>(nb);</span>
<span id="cb12-29"><a href="3-res.html#cb12-29"></a>        ACIMinor <span class="ot">=</span> <span class="fu">newArray</span>(nb);</span>
<span id="cb12-30"><a href="3-res.html#cb12-30"></a>        Dap <span class="ot">=</span> <span class="fu">newArray</span>(nb);</span>
<span id="cb12-31"><a href="3-res.html#cb12-31"></a>        <span class="fu">Ext.Manager3D_MultiSelect</span>();</span>
<span id="cb12-32"><a href="3-res.html#cb12-32"></a>        <span class="cf">for</span>(<span class="at">k =</span> <span class="dv">0</span>; k <span class="sc">&lt;</span> nb; k<span class="sc">++</span>) {</span>
<span id="cb12-33"><a href="3-res.html#cb12-33"></a>            <span class="cf">if</span> (k <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb12-34"><a href="3-res.html#cb12-34"></a>              <span class="fu">Ext.Manager3D_AddImage</span>();</span>
<span id="cb12-35"><a href="3-res.html#cb12-35"></a>            }</span>
<span id="cb12-36"><a href="3-res.html#cb12-36"></a>        <span class="fu">Ext.Manager3D_GetName</span>(k, obj);</span>
<span id="cb12-37"><a href="3-res.html#cb12-37"></a>        <span class="fu">Ext.Manager3D_Centroid3D</span>(k, cx, cy, cz);</span>
<span id="cb12-38"><a href="3-res.html#cb12-38"></a>    <span class="co"># measure feret</span></span>
<span id="cb12-39"><a href="3-res.html#cb12-39"></a>        <span class="cf">if</span> (symAC) {</span>
<span id="cb12-40"><a href="3-res.html#cb12-40"></a>          <span class="fu">Ext.Manager3D_Measure3D</span>(k, <span class="st">&quot;Feret&quot;</span>, ferr); </span>
<span id="cb12-41"><a href="3-res.html#cb12-41"></a>            da <span class="ot">=</span> ferr <span class="sc">*</span> cellum;</span>
<span id="cb12-42"><a href="3-res.html#cb12-42"></a>            da <span class="ot">=</span> <span class="fu">round</span>(da);</span>
<span id="cb12-43"><a href="3-res.html#cb12-43"></a>          <span class="cf">if</span> (da <span class="sc">==</span> <span class="dv">0</span>) {</span>
<span id="cb12-44"><a href="3-res.html#cb12-44"></a>              da <span class="ot">=</span> <span class="dv">1</span>;</span>
<span id="cb12-45"><a href="3-res.html#cb12-45"></a>          }</span>
<span id="cb12-46"><a href="3-res.html#cb12-46"></a>        Dap[k] <span class="ot">=</span> da;</span>
<span id="cb12-47"><a href="3-res.html#cb12-47"></a>        }</span>
<span id="cb12-48"><a href="3-res.html#cb12-48"></a>        <span class="fu">toString</span>(obj);</span>
<span id="cb12-49"><a href="3-res.html#cb12-49"></a>        objlabelArray[k] <span class="ot">=</span> obj;</span>
<span id="cb12-50"><a href="3-res.html#cb12-50"></a>    <span class="co"># erase all objects except current(k)</span></span>
<span id="cb12-51"><a href="3-res.html#cb12-51"></a>        <span class="fu">Ext.Manager3D_SelectAll</span>();</span>
<span id="cb12-52"><a href="3-res.html#cb12-52"></a>        <span class="fu">Ext.Manager3D_Select</span>(k);</span>
<span id="cb12-53"><a href="3-res.html#cb12-53"></a>        <span class="fu">Ext.Manager3D_Erase</span>();</span>
<span id="cb12-54"><a href="3-res.html#cb12-54"></a>        <span class="fu">run</span>(<span class="st">&quot;Enhance Contrast...&quot;</span>, <span class="st">&quot;saturated = 0.3 equalize process_all&quot;</span>);</span>
<span id="cb12-55"><a href="3-res.html#cb12-55"></a>        <span class="fu">run</span>(<span class="st">&quot;8-bit&quot;</span>);</span>
<span id="cb12-56"><a href="3-res.html#cb12-56"></a>        <span class="fu">run</span>(<span class="st">&quot;Crop Label&quot;</span>, <span class="st">&quot;label=255 border=5&quot;</span>);</span>
<span id="cb12-57"><a href="3-res.html#cb12-57"></a>    <span class="co"># clear cell stack in Z</span></span>
<span id="cb12-58"><a href="3-res.html#cb12-58"></a>        <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb12-59"><a href="3-res.html#cb12-59"></a>        var done <span class="ot">=</span> false; </span>
<span id="cb12-60"><a href="3-res.html#cb12-60"></a>        <span class="cf">for</span>(<span class="at">l =</span> <span class="dv">1</span>; l <span class="sc">&lt;</span> slices <span class="sc">&amp;&amp;!</span>done; l<span class="sc">++</span>) {</span>
<span id="cb12-61"><a href="3-res.html#cb12-61"></a>          <span class="fu">setSlice</span>(l);</span>
<span id="cb12-62"><a href="3-res.html#cb12-62"></a>            <span class="fu">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb12-63"><a href="3-res.html#cb12-63"></a>            <span class="cf">if</span>(max <span class="sc">&gt;</span> <span class="dv">0</span>) { <span class="sc">/</span><span class="er">/</span> from apical</span>
<span id="cb12-64"><a href="3-res.html#cb12-64"></a>            smax <span class="ot">=</span> l<span class="dv">-1</span>;</span>
<span id="cb12-65"><a href="3-res.html#cb12-65"></a>            <span class="fu">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first=1 last=&quot;</span><span class="sc">+</span>smax<span class="sc">+</span><span class="st">&quot; increment=1&quot;</span>);</span>
<span id="cb12-66"><a href="3-res.html#cb12-66"></a>            <span class="fu">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb12-67"><a href="3-res.html#cb12-67"></a>            <span class="fu">getDimensions</span>(width, height, channels, slices, frames);</span>
<span id="cb12-68"><a href="3-res.html#cb12-68"></a>            <span class="cf">for</span>(<span class="at">l =</span> <span class="dv">1</span>; l <span class="sc">&lt;</span> slices <span class="sc">&amp;&amp;!</span>done; l<span class="sc">++</span>) { <span class="sc">/</span><span class="er">/</span> from basal</span>
<span id="cb12-69"><a href="3-res.html#cb12-69"></a>                  <span class="fu">setSlice</span>(l);</span>
<span id="cb12-70"><a href="3-res.html#cb12-70"></a>                    <span class="fu">getStatistics</span>(area, mean, min, max, std, histogram);</span>
<span id="cb12-71"><a href="3-res.html#cb12-71"></a>                    <span class="cf">if</span>(max <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb12-72"><a href="3-res.html#cb12-72"></a>                      smax <span class="ot">=</span> l<span class="dv">-1</span>;</span>
<span id="cb12-73"><a href="3-res.html#cb12-73"></a>                        <span class="fu">run</span>(<span class="st">&quot;Slice Remover&quot;</span>, <span class="st">&quot;first = 1 last = &quot;</span><span class="sc">+</span> smax <span class="sc">+</span><span class="st">&quot; increment = 1&quot;</span>);</span>
<span id="cb12-74"><a href="3-res.html#cb12-74"></a>                        <span class="fu">run</span>(<span class="st">&quot;Reverse&quot;</span>);</span>
<span id="cb12-75"><a href="3-res.html#cb12-75"></a>                        done <span class="ot">=</span> true;</span>
<span id="cb12-76"><a href="3-res.html#cb12-76"></a>                    }</span>
<span id="cb12-77"><a href="3-res.html#cb12-77"></a>                }</span>
<span id="cb12-78"><a href="3-res.html#cb12-78"></a>            }</span>
<span id="cb12-79"><a href="3-res.html#cb12-79"></a>        }</span>
<span id="cb12-80"><a href="3-res.html#cb12-80"></a>        naci <span class="ot">=</span> <span class="fu">nSlices</span>();</span>
<span id="cb12-81"><a href="3-res.html#cb12-81"></a>        nacimax <span class="ot">=</span> naci<span class="sc">/</span><span class="dv">2</span>;</span>
<span id="cb12-82"><a href="3-res.html#cb12-82"></a>        <span class="fu">run</span>(<span class="st">&quot;Properties...&quot;</span>, <span class="st">&quot;channels = 1 slices = &quot;</span><span class="sc">+</span> naci <span class="sc">+</span><span class="st">&quot; frames = 1 unit = microns </span></span>
<span id="cb12-83"><a href="3-res.html#cb12-83"></a><span class="st">            pixel_width = &quot;</span><span class="sc">+</span> sizeX <span class="sc">+</span><span class="st">&quot; pixel_height = &quot;</span><span class="sc">+</span> sizeY <span class="sc">+</span><span class="st">&quot; voxel_depth = &quot;</span><span class="sc">+</span> sizeZ);</span>
<span id="cb12-84"><a href="3-res.html#cb12-84"></a>        <span class="cf">if</span> (symAC) {</span>
<span id="cb12-85"><a href="3-res.html#cb12-85"></a>            aciza <span class="ot">=</span> da <span class="sc">/</span> sizeZ;</span>
<span id="cb12-86"><a href="3-res.html#cb12-86"></a>            db <span class="ot">=</span> naci <span class="sc">-</span> da;</span>
<span id="cb12-87"><a href="3-res.html#cb12-87"></a>        }</span>
<span id="cb12-88"><a href="3-res.html#cb12-88"></a>    <span class="co"># if fixed ACI</span></span>
<span id="cb12-89"><a href="3-res.html#cb12-89"></a>        <span class="cf">if</span> (aciza <span class="sc">&lt;</span> (nacimax)) {</span>
<span id="cb12-90"><a href="3-res.html#cb12-90"></a>          acizb <span class="ot">=</span> naci <span class="sc">-</span> aciza;</span>
<span id="cb12-91"><a href="3-res.html#cb12-91"></a>    <span class="co"># measure apical</span></span>
<span id="cb12-92"><a href="3-res.html#cb12-92"></a>        <span class="fu">run</span>(<span class="st">&quot;Make Binary&quot;</span>, <span class="st">&quot;method = Default background=Default calculate black&quot;</span>);</span>
<span id="cb12-93"><a href="3-res.html#cb12-93"></a>        <span class="fu">setSlice</span>(aciza);</span>
<span id="cb12-94"><a href="3-res.html#cb12-94"></a>        <span class="fu">run</span>(<span class="st">&quot;Set Measurements...&quot;</span>, <span class="st">&quot;area centroid bounding fit feret&#39;s redirect = None decimal = 2&quot;</span>);</span>
<span id="cb12-95"><a href="3-res.html#cb12-95"></a>        <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;display slice&quot;</span>);</span>
<span id="cb12-96"><a href="3-res.html#cb12-96"></a>    <span class="co"># get angle</span></span>
<span id="cb12-97"><a href="3-res.html#cb12-97"></a>        MajorAngle[k] <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;FeretAngle&quot;</span>, <span class="dv">0</span>);</span>
<span id="cb12-98"><a href="3-res.html#cb12-98"></a>        <span class="cf">if</span> (MajorAngle[k] <span class="sc">&gt;</span> <span class="dv">90</span>) {</span>
<span id="cb12-99"><a href="3-res.html#cb12-99"></a>          MajorAngle[k] <span class="ot">=</span> <span class="dv">90</span> <span class="sc">-</span> (MajorAngle[k] <span class="sc">-</span> <span class="dv">90</span>);</span>
<span id="cb12-100"><a href="3-res.html#cb12-100"></a>        }</span>
<span id="cb12-101"><a href="3-res.html#cb12-101"></a>    <span class="co"># get min feret / major / minor</span></span>
<span id="cb12-102"><a href="3-res.html#cb12-102"></a>        resultsArray <span class="ot">=</span> <span class="fu">newArray</span>(<span class="fu">nResults</span>());</span>
<span id="cb12-103"><a href="3-res.html#cb12-103"></a>            <span class="cf">for</span>(<span class="at">p =</span> <span class="dv">0</span>; p <span class="sc">&lt;</span> <span class="fu">nResults</span>(); p<span class="sc">++</span>) { </span>
<span id="cb12-104"><a href="3-res.html#cb12-104"></a>            resultsArray[p] <span class="ot">=</span> <span class="fu">getResult</span>(<span class="st">&quot;Minor&quot;</span>, p); </span>
<span id="cb12-105"><a href="3-res.html#cb12-105"></a>            }</span>
<span id="cb12-106"><a href="3-res.html#cb12-106"></a>            total <span class="ot">=</span> <span class="dv">0</span>; </span>
<span id="cb12-107"><a href="3-res.html#cb12-107"></a>            <span class="cf">for</span>(<span class="at">p =</span> <span class="dv">0</span>; p <span class="sc">&lt;</span> <span class="fu">nResults</span>(); p<span class="sc">++</span>) { </span>
<span id="cb12-108"><a href="3-res.html#cb12-108"></a>                total <span class="ot">=</span> total <span class="sc">+</span> resultsArray[p]; </span>
<span id="cb12-109"><a href="3-res.html#cb12-109"></a>            }</span></code></pre></div>
<p></p>
</div>
<div id="single-cell-measurement" class="section level5" number="3.1.4.2.7">
<h5><span class="header-section-number">3.1.4.2.7</span> Single cell measurement</h5>
<p>Get 3D measurements from each object using the 3D Manager plugin</p>
<p></p>
<div class="sourceCode" id="cb13"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb13-1"><a href="3-res.html#cb13-1"></a><span class="fu">run</span>(<span class="st">&quot;3D Manager Options&quot;</span>, <span class="st">&quot;volume surface compactness fit_ellipse 3d_moments </span></span>
<span id="cb13-2"><a href="3-res.html#cb13-2"></a><span class="st">    feret centroid_(pix) centroid_(unit) distance_to_surface centre_of_mass_(unit) </span></span>
<span id="cb13-3"><a href="3-res.html#cb13-3"></a><span class="st">    bounding_box radial_distance surface_contact closest exclude_objects_on_edges_xy </span></span>
<span id="cb13-4"><a href="3-res.html#cb13-4"></a><span class="st">    sync distance_between_centers=10 distance_max_contact = 1.80&quot;</span>);</span>
<span id="cb13-5"><a href="3-res.html#cb13-5"></a><span class="fu">run</span>(<span class="st">&quot;3D Manager&quot;</span>);</span>
<span id="cb13-6"><a href="3-res.html#cb13-6"></a><span class="fu">Ext.Manager3D_AddImage</span>();</span>
<span id="cb13-7"><a href="3-res.html#cb13-7"></a><span class="fu">Ext.Manager3D_DeselectAll</span>();</span>
<span id="cb13-8"><a href="3-res.html#cb13-8"></a><span class="fu">Ext.Manager3D_Measure</span>();</span>
<span id="cb13-9"><a href="3-res.html#cb13-9"></a><span class="fu">Ext.Manager3D_SaveResult</span>(<span class="st">&quot;M&quot;</span>, datcelldir <span class="sc">+</span> name <span class="sc">+</span> <span class="st">&quot;.csv&quot;</span>);</span>
<span id="cb13-10"><a href="3-res.html#cb13-10"></a><span class="fu">Ext.Manager3D_CloseResult</span>(<span class="st">&quot;M&quot;</span>);</span>
<span id="cb13-11"><a href="3-res.html#cb13-11"></a><span class="fu">Ext.Manager3D_Reset</span>();</span>
<span id="cb13-12"><a href="3-res.html#cb13-12"></a><span class="fu">Ext.Manager3D_Close</span>();</span></code></pre></div>
<p></p>
</div>
<div id="functions" class="section level5" number="3.1.4.2.8">
<h5><span class="header-section-number">3.1.4.2.8</span> Functions</h5>
<p></p>
<div class="sourceCode" id="cb14"><pre class="sourceCode numberSource r numberLines"><code class="sourceCode r"><span id="cb14-1"><a href="3-res.html#cb14-1"></a><span class="cf">function</span> <span class="fu">sliceclear</span>() {</span>
<span id="cb14-2"><a href="3-res.html#cb14-2"></a>  <span class="cf">for</span> (<span class="at">j =</span> <span class="dv">1</span>; j <span class="sc">&lt;=</span> n; j<span class="sc">++</span>) {</span>
<span id="cb14-3"><a href="3-res.html#cb14-3"></a>    <span class="fu">roiManager</span>(<span class="st">&quot;reset&quot;</span>);</span>
<span id="cb14-4"><a href="3-res.html#cb14-4"></a>    <span class="fu">setSlice</span>(j);</span>
<span id="cb14-5"><a href="3-res.html#cb14-5"></a>    <span class="fu">run</span>(<span class="st">&quot;Analyze Particles...&quot;</span>, <span class="st">&quot;include add slice&quot;</span>);</span>
<span id="cb14-6"><a href="3-res.html#cb14-6"></a>    <span class="cf">if</span> (<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">==</span> <span class="dv">0</span>) {</span>
<span id="cb14-7"><a href="3-res.html#cb14-7"></a>        <span class="fu">setSlice</span>(j);</span>
<span id="cb14-8"><a href="3-res.html#cb14-8"></a>        <span class="fu">makeRectangle</span>(<span class="dv">0</span>, <span class="dv">0</span>, width, height);</span>
<span id="cb14-9"><a href="3-res.html#cb14-9"></a>        <span class="fu">run</span>(<span class="st">&quot;Cut&quot;</span>);</span>
<span id="cb14-10"><a href="3-res.html#cb14-10"></a>    }</span>
<span id="cb14-11"><a href="3-res.html#cb14-11"></a>    <span class="cf">if</span> (<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb14-12"><a href="3-res.html#cb14-12"></a>        <span class="fu">setSlice</span>(j);</span>
<span id="cb14-13"><a href="3-res.html#cb14-13"></a>        <span class="fu">allroi</span>();</span>
<span id="cb14-14"><a href="3-res.html#cb14-14"></a>        <span class="fu">wait</span>(<span class="dv">200</span>);</span>
<span id="cb14-15"><a href="3-res.html#cb14-15"></a>        <span class="fu">run</span>(<span class="st">&quot;Clear Outside&quot;</span>, <span class="st">&quot;slice&quot;</span>);</span>
<span id="cb14-16"><a href="3-res.html#cb14-16"></a>    }</span>
<span id="cb14-17"><a href="3-res.html#cb14-17"></a>    <span class="cf">if</span> (<span class="fu">roiManager</span>(<span class="st">&quot;count&quot;</span>) <span class="sc">&gt;</span> <span class="dv">1</span>) {</span>
<span id="cb14-18"><a href="3-res.html#cb14-18"></a>        <span class="fu">setSlice</span>(j);</span>
<span id="cb14-19"><a href="3-res.html#cb14-19"></a>        <span class="fu">allroi</span>();</span>
<span id="cb14-20"><a href="3-res.html#cb14-20"></a>        <span class="fu">roiManager</span>(<span class="st">&quot;Combine&quot;</span>);</span>
<span id="cb14-21"><a href="3-res.html#cb14-21"></a>        <span class="fu">run</span>(<span class="st">&quot;Clear Outside&quot;</span>, <span class="st">&quot;slice&quot;</span>);</span>
<span id="cb14-22"><a href="3-res.html#cb14-22"></a>    }</span>
<span id="cb14-23"><a href="3-res.html#cb14-23"></a>  }</span>
<span id="cb14-24"><a href="3-res.html#cb14-24"></a>}</span></code></pre></div>
<p></p>
</div>
</div>
<div id="data-analysis-1" class="section level4" number="3.1.4.3">
<h4><span class="header-section-number">3.1.4.3</span> Data Analysis</h4>
<div id="cell-count" class="section level5" number="3.1.4.3.1">
<h5><span class="header-section-number">3.1.4.3.1</span> Cell Count</h5>
<p>In figure <a href="3-res.html#fig:gtratio">3.12</a> the relative numbers for each threshold level can be seen in percentage above or below the mean cell count of the ground truth (blue horizon). Analogous to the graphical inspection, the magenta area represents over segmentation, the green under segmentation.</p>

<div class="figure" style="text-align: center"><span id="fig:gtratio"></span>
<img src="thesis_dsk_files/figure-html/gtratio-1.png" alt="Relative difference of segment counts" width="75%" />
<p class="caption">
Figure 3.12: Relative difference of segment counts
</p>
</div>
</div>
<div id="mat-anallzr3d-ac" class="section level5" number="3.1.4.3.2">
<h5><span class="header-section-number">3.1.4.3.2</span> Apical Constriction</h5>
<p>For a proof of concept of segmentation results and concept around measuring Apical Constriction Index, analog to Harding(2013)<span class="citation">(<em>62</em>)</span>, 13 DMSO and 15 SU5402 treated embryos were imaged, segmented and processed. The results (figure <a href="3-res.html#fig:resacimeasure">3.13</a>) show a strong significant difference between DMSO und SU5402 treated embryos in both A.I.<sub>Major</sub> and A.I.<sub>Minor</sub>, validating the general concept.</p>

<div class="figure" style="text-align: center"><span id="fig:resacimeasure"></span>
<img src="thesis_dsk_files/figure-html/resacimeasure-1.png" alt="A.I. proof of concept. A and B Each dot represents the average of all cells of a whole pLLP (DMSO = 1769 cells / 13 pLLPs; SU5402 = 2066 cells / 15 pLLPs)." width="60%" />
<p class="caption">
Figure 3.13: A.I. proof of concept. <strong>A and B</strong> Each dot represents the average of all cells of a whole pLLP (DMSO = 1769 cells / 13 pLLPs; SU5402 = 2066 cells / 15 pLLPs).
</p>
</div>
</div>
<div id="cell-morphology" class="section level5" number="3.1.4.3.3">
<h5><span class="header-section-number">3.1.4.3.3</span> Cell Morphology</h5>
<blockquote>
<p><em>There is always a precision / recall trade-off in detection tasks, e.g., when we set a lower threshold [. . .], we can get a higher recall with a lower precision ([. . .], but meanwhile we also get more false-positives in the results).</em><span class="citation">(<em>51</em>)</span></p>
</blockquote>
<p>Inspecting the cell count unfortunately does not directly tell us how well the cell morphology is conserved at different threshold levels, since at higher threshold levels the cell boundaries are differently determined and eventually not even recognized as such anymore. The volume of a cell is a very robust morphological feature with respect to changes in pose and topology <span class="citation">(<em>19</em>)</span>. Therefore, if its volume does not differ significantly at a given threshold level we consider its morphology to be conserved.</p>
<p>In figure <a href="3-res.html#fig:volsecdffull">3.14</a> the cell volumes and their distribution across the different threshold levels can be inspected. The closer the slopes are to the Ground Truth, the stronger they are conserved. The upper graph of figure <a href="3-res.html#fig:volsecdffull">3.14</a> shows the full distribution, where the major differences seem to appear within the 0.4 quantile (red line). To get a more detailed impression, in the lower graph of figure <a href="3-res.html#fig:volsecdffull">3.14</a> only the values <em>within</em> the 0.4 quantile are shown.</p>

<div class="figure" style="text-align: center"><span id="fig:volsecdffull"></span>
<img src="thesis_dsk_files/figure-html/volsecdffull-1.png" alt="upper Cell volumes full distribution (red line = .4 Quantile) lower Cell volumes distribution within .4 Quantile" width="90%" /><img src="thesis_dsk_files/figure-html/volsecdffull-2.png" alt="upper Cell volumes full distribution (red line = .4 Quantile) lower Cell volumes distribution within .4 Quantile" width="90%" />
<p class="caption">
Figure 3.14: <strong>upper</strong> Cell volumes full distribution (red line = .4 Quantile) <strong>lower</strong> Cell volumes distribution within .4 Quantile
</p>
</div>
</div>
<div id="confidence-level" class="section level5" number="3.1.4.3.4">
<h5><span class="header-section-number">3.1.4.3.4</span> Confidence Level</h5>
<p>To statistically check how closely related each threshold level (section <a href="3-res.html#mat-anallzr3d-ac">3.1.4.3.2</a>) sample distribution is compared to the Ground Truth, a Kolmogorov-Smirnov-Test (ks.test) was performed. The ks.test is a nonparametric test whose null hypothesis is that the both groups compared were sampled from populations with identical attributes. Therefore, the closer the p-value is to 1, the more similar the tested sample distribution would be to the Ground Truth (figure <a href="3-res.html#fig:kstestvol">3.15</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:kstestvol"></span>
<img src="thesis_dsk_files/figure-html/kstestvol-1.png" alt="Volumetric conservation (red line = 5\(\%\))" width="70%"  />
<p class="caption">
Figure 3.15: Volumetric conservation (red line = 5<span class="math inline">\(\%\)</span>)
</p>
</div>
<p>In comparison to the Mann-Whitney test, the ks.test is more sensitive to detect changes in the shape of the distribution than to detect a shift of the median<span class="citation">(<em>71</em>)</span>. Since we have no reason to assume a significant change in the <em>average</em> cell volume, it would not make much sense to use a test that requires averages. Instead we want to test if there is a change in the shape or skew of the whole distribution, which allows to compare each group on the single cell level (figure <a href="3-res.html#fig:volsecdffull">3.14</a>). In summary, the test results suggest a larger deviance in shape of distribution between the Ground Truth and the automatically segmented cells for Threshold levels T02 and T10. For T03 we are already close to rejection of the <span class="math inline">\(H_0\)</span> hypothesis (assuming a 5<span class="math inline">\(\%\)</span> threshold). Only for T04 and T05 are well within the 5<span class="math inline">\(\%\)</span> rejection threshold. However, since we want to be as close to the Ground Truth as possible, we settled for T04.</p>
</div>
</div>
</div>
<div id="CNN" class="section level3" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Rosette Detection</h3>
<p>The method used for rosette detection is based on a convolutional neuronal network (CNN) and was modified from the “rosette detector” algorithm previously used in the lab and described in <span class="citation">(<em>1</em>, <em>51</em>)</span>. Since the former method was technically outdated and since we had new data in which we needed to detect rosettes, we updated the former method to a <em>state-of-the-art</em> CNN using Caffe<span class="citation">(<em>72</em>)</span> as a backend. Network configuration and training was done by our collaborators at the institute for Informatics, Albert-Ludwigs-University Freiburg.</p>
<p>The training dataset consitsts of 17 DMSO- and SU5402- treated embryos each. SU5402 is an inhibitor of FGF receptors and embryos treated with these inhibitors show a strongly reduced number of rosettes <span class="citation">(<em>38</em>)</span>. In order to train the network, the data had to be labeled manually, which was done in ImageJ by placing multipoint ROIs at the center of the rosettes. The data was then further permutated (rotated and changed in size) to artificially increase the amount of training data and make the detection more robust against different kinds of input. Further parameters about the training data is listed in Table <a href="3-res.html#tab:cnntraintab">3.3</a>. One example for each is shown in figure <a href="3-res.html#fig:cnntrain">3.16</a>.</p>

<div class="figure" style="text-align: center"><span id="fig:cnntrain"></span>
<img src="figures/materials/cnn/CNNtrain.png" alt="Example for rosette detection on the training data. left Maximum Z-projected input data. middle heatmap of scores. blue indicates a low score, red a high one. right score map projected onto the input data." width="85%" />
<p class="caption">
Figure 3.16: Example for rosette detection on the training data. <strong>left</strong> Maximum Z-projected input data. <strong>middle</strong> heatmap of scores. blue indicates a low score, red a high one. <strong>right</strong> score map projected onto the input data.
</p>
</div>
<p>The main advantages for using a neural network in a task like this are…</p>
<ol style="list-style-type: decimal">
<li>Objectivity
<ul>
<li>A computer model is not biased in a way that it prefers one outcome over the other. It evaluates based on what it was trained to.</li>
<li>Unlike the human brain, once a CNN is trained it is static and does not keep on learning. This promotes reproducibility.</li>
</ul></li>
<li>Degree of rosette registration
<ul>
<li>The output data are continuous rational numbers (<span class="math inline">\(\mathbb{Q}\)</span>) instead of integers (<span class="math inline">\(\mathbb{Z}\)</span>) which does not only tell if a rosette is there or not, but also for ‘how much’ (50-100<span class="math inline">\(\%\)</span>) it is there.</li>
</ul></li>
<li>Training is done relatively quick</li>
</ol>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:cnntraintab">Table 3.3: </span>CNN training data
</caption>
<thead>
<tr>
<th style="text-align:center;font-weight: bold;">
group
</th>
<th style="text-align:center;font-weight: bold;">
conc.
</th>
<th style="text-align:center;font-weight: bold;">
nm
</th>
<th style="text-align:center;font-weight: bold;">
intensity…
</th>
<th style="text-align:center;font-weight: bold;">
exposure
</th>
<th style="text-align:center;font-weight: bold;">
Z.planes
</th>
<th style="text-align:center;font-weight: bold;">
magnif.
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
DMSO
</td>
<td style="text-align:center;">
0.1<span class="math inline">\(\%\)</span>
</td>
<td style="text-align:center;">
488
</td>
<td style="text-align:center;">
100
</td>
<td style="text-align:center;">
100 ms
</td>
<td style="text-align:center;">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
<tr>
<td style="text-align:center;">
SU5402
</td>
<td style="text-align:center;">
10<span class="math inline">\(\mu\)</span>M
</td>
<td style="text-align:center;">
488
</td>
<td style="text-align:center;">
100
</td>
<td style="text-align:center;">
100 ms
</td>
<td style="text-align:center;">
~70
</td>
<td style="text-align:center;">
40X
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="shroom3" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Shroom3</h2>
<p>In the previous section a number of methods were presented I developed specifically to perform the analyses on the <em>shroom3</em> mutant phenotype which are described in the following section.</p>
<div id="intro-phen" class="section level3" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Phenotype description</h3>
<p>While birth rates follow a distribution of Mendelian inheritance (after genotyping at 3 months of age), <em>homozygous</em> mutant adults seem to be more sensitive to mechanical stress and have a shortened lifespan (~6-9 months). In body shape, <em>shroom3</em> mutants are not smaller or possess any other striking phenotype (figure <a href="3-res.html#fig:shrmmut">3.17</a>C), however their gill flaps seem to be increased in size, swollen, and not exactly streamlined with the body. This is also evident by an increased frequency of gill flap beating. When looking at the pLLP at different stages (figure <a href="3-res.html#fig:shrmmut">3.17</a>D), they exhibit the same phenotype as the MO injected embryos <span class="citation">(<em>1</em>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:shrmmut"></span>
<img src="figures/intro/shrm_mutants.png" alt="shroom3 mutant phenotype (A-A’) mutation strategy (A) Talen arms (black blocks) bordering a sequence within SD1 including a restriction site for NsiI (indicated back arrows and grey background) (A’) wildtype and mutant allele with 8 bp deletion (B-B’) Amino acid code and protein schematic with functional domains and stop codon (indicated by asterisk) (C-C’) Adult phenotype with closeup to gill flaps (D) pLLP phenotype for three different stages (columns) and different manifestations (rows) (40X WI objective). Arrows indicate epithelial rosettes. (E) LL phenotype at end of migration (10X air objective). Scale bars = 1 mm. MaxIPs for D-E." width="95%" />
<p class="caption">
Figure 3.17: <em>shroom3</em> mutant phenotype <strong>(A-A’)</strong> mutation strategy (A) Talen arms (black blocks) bordering a sequence within SD1 including a restriction site for NsiI (indicated back arrows and grey background) (A’) wildtype and mutant allele with 8 bp deletion <strong>(B-B’)</strong> Amino acid code and protein schematic with functional domains and stop codon (indicated by asterisk) <strong>(C-C’)</strong> Adult phenotype with closeup to gill flaps <strong>(D)</strong> pLLP phenotype for three different stages (columns) and different manifestations (rows) (40X WI objective). Arrows indicate epithelial rosettes. <strong>(E)</strong> LL phenotype at end of migration (10X air objective). Scale bars = 1 mm. MaxIPs for D-E.
</p>
</div>
<p><em>Shroom3</em> heterozygous zebrafish show no phenotypic abnormality. When incrossed, genotyping of two to five days post fertilization (dpf) embryos results in a Mendelian distribution. However, stock-fish that are regenerated and genotyped by finclip at about 3 months of age show only a rate of 5-10 <span class="math inline">\(\%\)</span> of <em>shroom3</em> homozygous mutants. After another 3 – 6 months those would usually decease too. Therefore, the loss of Shroom3 activity leads to increased mortality for reasons we have not uncovered yet.</p>
</div>
<div id="lateral-line-morphometrics" class="section level3" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> Lateral Line Morphometrics</h3>
<p>Since in the previous study performed on <em>shroom3</em> morphants we found a significantly reduced number of rosettes in the pLLP, a reduced number of CCs was expected to be deposited at the end of pLLP migration as well. Instead we found more CCs deposited (figure <a href="3-res.html#fig:shrmmut">3.17</a>E).</p>
<div id="dataset" class="section level4" number="3.2.2.1">
<h4><span class="header-section-number">3.2.2.1</span> Dataset</h4>
<p>To analyze this observation quantitatively a dataset was put together consisting of 83 zebrafish embryos fixed at the end of pLLP migration, derived from four different parent pairs. After genotyping, this gave us 26 wildtypes, 31 heterozygous and 26 homozygous mutants for statistical tests. In addition to the number and position of deposited NMs, we also measured the area and the number of nuclei in each deposited CC.</p>
</div>
<div id="res-ccounts" class="section level4" number="3.2.2.2">
<h4><span class="header-section-number">3.2.2.2</span> Number and Position of Cell Clusters</h4>
<p>Although the embryos were closely staged (<span class="math inline">\(\pm\)</span> 20 min.), there are always some individual variations in the distance migrated by the pLLP of each. To compare the number of deposited CCs independently of LL length the ratio of LL length over CC count (<a href="3-res.html#fig:llcounts">3.18</a>B) was calculated. Fixation with PFA may introduce a slight bending of the embryo and mounting introduces a different tilt for each embryo, to correct for uneven LL paths and irregularities in mounting, CC distances are calculated in Euclidean space rather than solely in dimension X.</p>
<p>While <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos deposit 6 <span class="math inline">\(\pm\)</span> 0.7 CCs, <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos deposit 8 <span class="math inline">\(\pm\)</span> 0.9 CCs (figure <a href="3-res.html#fig:llcounts">3.18</a>A). This difference stays true also when normalizing against length (figure <a href="3-res.html#fig:llcounts">3.18</a>B).</p>

<div class="figure" style="text-align: center"><span id="fig:llcounts"></span>
<img src="figures/results/01_morphometrics/ll_counts.png" alt="Cluster counts. A cluster count [\(n\)] B normalized to length (LL length [\(mm\)] / cluster count [\(n\)])" width="60%" />
<p class="caption">
Figure 3.18: Cluster counts. <strong>A</strong> cluster count [<span class="math inline">\(n\)</span>] <strong>B</strong> normalized to length (LL length [<span class="math inline">\(mm\)</span>] / cluster count [<span class="math inline">\(n\)</span>])
</p>
</div>
<p>Even though CC position in individual <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos seems more random, the position of the first deposited CC is mostly conserved (<a href="3-res.html#fig:llpos">3.19</a>A), <span class="math inline">\(p\)</span> for difference in position = 0.2). Similarly, the position of the pLLP isn’t significantly changed confirming that Shroom3 activity is not required for migration, as was already shown in <em>shroom3</em> morphants <span class="citation">(<em>1</em>)</span>. While for the remaining CCs an average lag of -50.4 <span class="math inline">\(\mu\)</span>m as compared to <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> CC positions is observed, it also seems to increase with later CC positions. For <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos CC position is mostly conserved through development, however it remains elusive if this is true for <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos too.
Figure <a href="3-res.html#fig:llpos">3.19</a>A’, shows the kernel density distribution for the CC position without grouping individual positions. At a binwidth of 50 <span class="math inline">\(\mu\)</span>m, the distribution curves show a high and narrow density at around 350 <span class="math inline">\(\mu\)</span>m, which is the average location of CC1 (<a href="3-res.html#fig:llpos">3.19</a> 2A). For the remaining CCs the kernel distribution does, neither for <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> nor for <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos, reveal a precise location. In contrast, if based on CC sequential identity the mean position and standard deviation is calculated, a more explicit pattern emerges which clearly shows the increased count and average frequency (<a href="3-res.html#fig:llpos">3.19</a> 2A).</p>
<p>This shows that in the absence of Shroom3 activity, more NMs form. This is particularly surprising since the lab had previously shown that <em>shroom3</em> knock-down leads to a defect in rosette formation and these rosettes prefigure the deposited NM.</p>

<div class="figure" style="text-align: center"><span id="fig:llpos"></span>
<img src="figures/results/01_morphometrics/ll_positions.png" alt="Cell Cluster positions. A Exemplary shroom3\(^{+/+}\) and shroom3\(^{-/-}\) embryos with CCs highlighted. CC0 marks the reference location to compare individual embryos. Scale bar = 1 mm A’ Kernel Density Estimate without (KDE) grouping (\(n\)++ = 162, \(n\)-- = 206) A’' Dots = mean positions, bars = standard deviation (\(n\)max for both ++ and -- = 26)." width="95%" />
<p class="caption">
Figure 3.19: Cell Cluster positions. <strong>A</strong> Exemplary <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos with CCs highlighted. CC0 marks the reference location to compare individual embryos. Scale bar = 1 mm <strong>A’</strong> Kernel Density Estimate without (KDE) grouping (<span class="math inline">\(n\)</span>++ = 162, <span class="math inline">\(n\)</span>-- = 206) <strong>A’'</strong> Dots = mean positions, bars = standard deviation (<span class="math inline">\(n\)</span>max for both ++ and -- = 26).
</p>
</div>
</div>
<div id="res-llmorph" class="section level4" number="3.2.2.3">
<h4><span class="header-section-number">3.2.2.3</span> Cell Count and Area of CCs</h4>
<p>The CC count and position analysis clearly shows that more CCs are deposited in <em>shroom3</em> mutant embryos. In order to determine whether these additional clusters are comprised of as many cells as their wild-type counterparts or less, I quantified the number of cells and the size of each clusters in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos, the cell count per CC was determined by counting DAPI stained cells within CC segments derived from the <em>cldnb:lyn-gfp</em> membrane signal (section <a href="2-mat-met.html#mat-GrTrDat">2.3</a>). Both the CC cell number and the CC area were reduced by about 6<span class="math inline">\(\%\)</span> in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos so that the density remains unchanged (<a href="3-res.html#fig:llclus">3.20</a>A). Interestingly, the total number of cells in CC per embryo is increased by 9<span class="math inline">\(\%\)</span> (<a href="3-res.html#fig:llclus">3.20</a>B). Both the sum of CC cells and total LL cells (CC+pLLP) are significantly increased in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos, while the count of only pLLP cells remains unchanged.</p>

<div class="figure" style="text-align: center"><span id="fig:llclus"></span>
<img src="figures/results/01_morphometrics/ll_clusters.png" alt="LL Morphometrics A individual CC statistics B Sums per embryo. (Bars = median, errorbars = 95% CI)" width="75%" />
<p class="caption">
Figure 3.20: LL Morphometrics <strong>A</strong> individual CC statistics <strong>B</strong> Sums per embryo. (Bars = median, errorbars = 95% CI)
</p>
</div>
</div>
<div id="res-tempresc" class="section level4" number="3.2.2.4">
<h4><span class="header-section-number">3.2.2.4</span> Temperature Rescue</h4>
<p>Increasing temperature usually leads to an acceleration in thermodynamics and embryonic development, while lower temperatures slow down development <span class="citation">(<em>73</em>)</span>. Slowing down development also leads to a reduction in speed of migration and therefore to a reduction in forces that act on cells, cell-cell junctions and organ structures.</p>
<p>For this experiment the hypothesis to test was that it should be possible to at least partially restore the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> number of CCs by giving rosettes a longer time to develop properly. Since embryos develop at different speeds, it is more challenging to exactly stage match them at end of migration. To account for this the CC count was normalized to LL length.</p>

<div class="figure" style="text-align: center"><span id="fig:resctemp"></span>
<img src="figures/results/06_rescues/temp/rescue_temp.png" alt="Temperature rescue A LL lengths and end of migration stage matching B groupwise comparison of the length normalized CC count per temperature." width="85%" />
<p class="caption">
Figure 3.21: Temperature rescue <strong>A</strong> LL lengths and end of migration stage matching <strong>B</strong> groupwise comparison of the length normalized CC count <em>per</em> temperature.
</p>
</div>
<p>The question was if we could restore wildtype CC counts in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos when incubating at lower temperatures. Therefore, we need to compare CC counts between T25-- and T30++ respectively p-values between the pairs T30-- <em>vs</em> for T30++ (the original) and T25-- <em>vs</em> T30++ (the actual hypothesis). Even though the CC count T25-- cannot be completely restored (~3.2 to ~2.8 in median normalized CC count), the difference between tested pairs shrunk from 0.00000 (T30-- <em>vs</em> for T30++) to 0.00210 (T25-- <em>vs</em> T30++) (figure <a href="supplement.html#fig:supppos">4.4</a>D and <a href="3-res.html#fig:resctemp">3.21</a>B), which is much closer to a rejection of <span class="math inline">\(H_0\)</span>. At this point, the question remains open but seems probable that at even lower temperatures the wildtype CC count could be rescued in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos.</p>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resctemptab">Table 3.4: </span>Temperature rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
treatment
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
treatment
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3++
</td>
<td style="text-align:center;">
25°C
</td>
<td style="text-align:center;border-right:1px solid;">
18
</td>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3-\-
</td>
<td style="text-align:center;">
25°C
</td>
<td style="text-align:center;">
19
</td>
</tr>
<tr>
<td style="text-align:center;">
30°C
</td>
<td style="text-align:center;border-right:1px solid;">
15
</td>
<td style="text-align:center;">
30°C
</td>
<td style="text-align:center;">
20
</td>
</tr>
</tbody>
</table>
<table class="table" style="font-size: 7px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resctempsignif">Table 3.5: </span>Temperature rescue statistics
</caption>
<thead>
<tr>
<th style="text-align:right;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
p.adj
</th>
<th style="text-align:center;">
p.signif
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;vertical-align: top !important;" rowspan="3">
T25 ++
</td>
<td style="text-align:left;">
T25 -\-
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
T30 ++
</td>
<td style="text-align:center;">
0.00747
</td>
<td style="text-align:center;">
0.00750
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;">
T30 -\-
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:right;vertical-align: top !important;" rowspan="2">
T25 -\-
</td>
<td style="text-align:left;">
T30 ++
</td>
<td style="text-align:center;">
0.00104
</td>
<td style="text-align:center;">
0.00210
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;">
T30 -\-
</td>
<td style="text-align:center;">
0.00015
</td>
<td style="text-align:center;">
0.00046
</td>
<td style="text-align:center;">
***
</td>
</tr>
<tr>
<td style="text-align:right;">
T30 ++
</td>
<td style="text-align:left;">
T30 -\-
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
</tbody>
</table>
</div>
<div id="summary-2" class="section level4" number="3.2.2.5">
<h4><span class="header-section-number">3.2.2.5</span> Summary</h4>
<p>Two additional CCs are deposited in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos. While each CC is comprised of less cells, in sum there are more cells in embryos deficient for Shroom3 leading to a net increase in cell count of ~9<span class="math inline">\(\%\)</span> (31 cells). Lowering the temperature by 5<span class="math inline">\(^\circ\)</span> C was sufficient to partially rescue the number of NMs suggesting that Shroom3 might be required to make the system more robust against stress conditions.</p>
</div>
</div>
<div id="proliferation" class="section level3" number="3.2.3">
<h3><span class="header-section-number">3.2.3</span> Proliferation</h3>
<p>LL morphometric analysis revealed that deposited CCs in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos were on average slightly smaller and had fewer cells incorporated. However, due to the additional two cell clusters deposited, the net count is ~9<span class="math inline">\(\%\)</span> increased at the end of migration.
To test whether this is due to higher proliferative activity a dataset of time-lapse movies (12 h / <span class="math inline">\(\Delta\)</span>T = 6 min.) was generated to count the number of mitoses in a <em>cxcr4b(BAC):H2BRFP</em> transgenic background similar to previous proliferation studies in the pLLP <span class="citation">(<em>34</em>, <em>74</em>)</span>.</p>
<div id="datasets" class="section level4" number="3.2.3.1">
<h4><span class="header-section-number">3.2.3.1</span> Datasets</h4>
<p>The live imaging dataset consists of 12 <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and 18 <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos with about 75 timepoints for each pLLP and 150 cell clusters. For counting, tracks were created for each proliferative cell on MaxIPs (section <a href="2-mat-met.html#prolif">2.2.4.2</a>). Figure <a href="3-res.html#fig:prolpLLP">3.22</a>A shows a <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and a <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> LL after 12 h of imaging where each mitotic track is individually colored and overlaid to the image data.</p>
<p>To support and validate this data two more datasets from fixed embryos were generated, one with a EdU (N = 3, n++ = 26, n-- = 17) and one with a phospho-Histone marker (N = 3, n++ = 22, n-- = 29).</p>
</div>
<div id="res-prolpLLP" class="section level4" number="3.2.3.2">
<h4><span class="header-section-number">3.2.3.2</span> Lateral Line Mitoses</h4>
<p>Tracking mitoses in the pLLP does not reveal any difference in proliferation from start (~28 hpf) till about mid of migration 12 hours later (<a href="3-res.html#fig:prolpLLP">3.22</a>B). For confirmation, this finding was also validated <em>via</em> two more additional methods. During the cell cycle genetic material is replicated in S-phase, while in metaphase of mitosis histones are found to be heavily phosphorylated <span class="citation">(<em>75</em>)</span>. With an 5-Ethynyl-2’-deoxyuridine (EdU) assay <span class="citation">(<em>76</em>)</span>, cells in S-phase can be detected (<a href="3-res.html#fig:prolpLLP">3.22</a>B’). Using a specific phospho-Histone (p-Hist.) antibody cells in meta-phase can be detected. When comparing the
EdU index (EdU positive cells over total pLLP cell number) or the mitotic index (p-Hist. positive cells over total pLLP cell number) I could confirm that there is no difference in proliferation in the migrating pLLP. Still, at the end of migration the LL system in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos does contain 9<span class="math inline">\(\%\)</span> more cells (figure <a href="3-res.html#fig:llclus">3.20</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:prolpLLP"></span>
<img src="figures/results/02_proliferation/prol-01.png" alt="pLLP proliferation A Tracks of Mitosis. (Image data) Last timepoint of Z-projected time-lapse movie. (Labels) each dots marks a mitotic events. Each color represents one mitosis. Each connection between two dots represents 6 min. B-B’' Mitoses in the pLLP. (B) Count of Mitoses through time (mean\(\pm\)sd) (B’) Proliferation Index at 36 hpf for EdU and phospho Histone labeled pLLPs (B’‘) Examples of EdU staining (scale bar = 100 \(\mu\)m) C Mitoses in CCs. B’ and C, the colored bar indicated the median, bars indicate the 95% CI." width="95%" />
<p class="caption">
Figure 3.22: pLLP proliferation <strong>A</strong> Tracks of Mitosis. (Image data) Last timepoint of Z-projected time-lapse movie. (Labels) each dots marks a mitotic events. Each color represents one mitosis. Each connection between two dots represents 6 min. <strong>B-B’'</strong> Mitoses in the pLLP. (B) Count of Mitoses through time (mean<span class="math inline">\(\pm\)</span>sd) (B’) Proliferation Index at 36 hpf for EdU and phospho Histone labeled pLLPs (B’‘) Examples of EdU staining (scale bar = 100 <span class="math inline">\(\mu\)</span>m) <strong>C</strong> Mitoses in CCs. B’ and C, the colored bar indicated the median, bars indicate the 95% CI.
</p>
</div>
<p>Since individual <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> CCs are smaller, we wondered if there could be compensation mechanisms activated that increases proliferation to restore wildtype CC size once they are deposited. To verify this, CC mitoses were tracked on the same data as before. Other than the pLLP, which can be observed throughout the time-lapse, CCs only begin to <em>exist</em> when they are deposited. To normalize for individual CC lifetimes, the number of mitoses per CC is divided by the duration of the total time-lapse minus the time span to when it first appeared.</p>

<div class="definition">
<span id="def:unnamed-chunk-10" class="definition"><strong>Definition 3.1  (Normalized mitotic rate)  </strong></span><span class="math display">\[\frac{mitoses\ per\ CC\ [n]}{total\ time- time\ of\ deposition\ [T]}\]</span>
</div>
<p>This clearly shows that the additional cells in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos are derived from CC proliferation, rather than pLLP proliferation (<a href="3-res.html#fig:prolpLLP">3.22</a>C).</p>
</div>
<div id="summary-3" class="section level4" number="3.2.3.3">
<h4><span class="header-section-number">3.2.3.3</span> Summary</h4>
<p>These results show that the additional cells at the end of migration observed in the LL analysis do not stem from an increase in proliferation in the pLLP, but instead from increased proliferation after CCs are deposited.</p>
</div>
</div>
<div id="rosette-formation-and-cluster-deposition" class="section level3" number="3.2.4">
<h3><span class="header-section-number">3.2.4</span> Rosette Formation and Cluster Deposition</h3>
<p>Having shown that the additional CC deposition in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos is not caused by an over-proliferation in the migrating pLLP, the next step was to have a closer look at the dynamics of rosette formation, in relation to pLLP morphometrics and CC deposition. To test dependencies between different observations and developmental dynamics, variables can be correlated. To do this and to have a coherent dataset to work with, the data of three different analyses on a single set of image data were joined.</p>
<p><strong>First</strong>, to get to know the exact timing of CC deposition, a manual tracking tool <span class="citation">(<em>59</em>)</span> was used (figure <a href="3-res.html#fig:rdt">3.23</a>, tracking). <strong>Second</strong>, to deduce pLLP morphometrics an IJ macro was developed (anaLLzr2DT, section <a href="3-res.html#mat-anallzr2dt">3.1.3</a>) for spatiotemporal registration of the pLLP and to yield information about its speed, area, roundness, etc. (<a href="3-res.html#fig:rdt">3.23</a>, Registration). <strong>Third</strong>, to detect rosettes and quantify their weights, a CNN <span class="citation">(<em>77</em>)</span> was used on the registered pLLP output from the before mentioned IJ macro (section <a href="3-res.html#CNN">3.1.5</a>; figure <a href="3-res.html#fig:rdt">3.23</a> - Detection).
Finally, all three datasets were joined by embryo id and timepoint.</p>
<div id="res-det-ds" class="section level4" number="3.2.4.1">
<h4><span class="header-section-number">3.2.4.1</span> Dataset</h4>
<p>The image data set analyzed consists of 20 time-lapse movies (11 <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>, 9 <em>shroom3</em><span class="math inline">\(^{+/+}\)</span>). Each time-lapse has a duration of ~20 h (~8 min. interval)<a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a>, summing up to ~1650 <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>, and ~1350 <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> timepoints.</p>

<div class="figure" style="text-align: center"><span id="fig:rdt"></span>
<img src="figures/results/03_rosettes/RDT-01.png" alt="Rosette Formation Joined Datasets. Tracking pLLP is marked as no.1. The rest of the CCs is numbered sequentially as they appear. Registration The black outline marks the region of interest (ROI) that is the pLLP as it is detected by the anaLLzr2DT. The red line highlights the pLLPs leading edge. Detection Each square highlights a detected rosette by the CNN, colors represent rosette weights." width="95%" />
<p class="caption">
Figure 3.23: Rosette Formation Joined Datasets. <strong>Tracking</strong> pLLP is marked as no.1. The rest of the CCs is numbered sequentially as they appear. <strong>Registration</strong> The black outline marks the region of interest (ROI) that is the pLLP as it is detected by the anaLLzr2DT. The red line highlights the pLLPs leading edge. <strong>Detection</strong> Each square highlights a detected rosette by the CNN, colors represent rosette weights.
</p>
</div>
</div>
<div id="cluster-deposition" class="section level4" number="3.2.4.2">
<h4><span class="header-section-number">3.2.4.2</span> Cluster Deposition</h4>
<p>Figure <a href="3-res.html#fig:rdtdepo">3.24</a>A shows a montage of a <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and a <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> scenario in cluster deposition (3.5 h / ~20 min. interval). For <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> the rosette structure seems tight and two depositions occur in a regular manner. In the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> on the other hand, rosette structure are more fragile with less pronounced rosette centers and four observed depositions. Interestingly, based on the <em>cldnb:lyn-gfp</em> signal the area of constriction seems to be less radially organized but more oriented towards the horizontal midline. Furthermore, the trailing rosettes are significantly smaller and do not seem to separate as clean from the migrating primordium. In addition, for L3 it first seems like two CCs could be deposited, until they merge again to a single CC about 1.5 h later.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtdepo"></span>
<img src="figures/results/03_rosettes/tracking-01.png" alt="Cluster Deposition. A shroom3\(^{+/+}\) and shroom3\(^{-/-}\) LL development in comparison. L1 - L4 are deposited CCs. Arrows indicate deposition events. Dotted lines are tracks of rosette to CC transition. B Statistics of deposited cluster counts C Change of CC position through time. Each line represents the locally weighted scatterplot smoothing (LOESS) of all CCn positions observed. See section 3.2.4.3 for a dataset description." width="95%" />
<p class="caption">
Figure 3.24: Cluster Deposition. <strong>A</strong> <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> LL development in comparison. L1 - L4 are deposited CCs. Arrows indicate deposition events. Dotted lines are tracks of rosette to CC transition. <strong>B</strong> Statistics of deposited cluster counts <strong>C</strong> Change of CC position through time. Each line represents the locally weighted scatterplot smoothing (LOESS) of all CCn positions observed. See section <a href="3-res.html#res-rosreg">3.2.4.3</a> for a dataset description.
</p>
</div>
<p>On average, as it was shown in section <a href="3-res.html#res-ccounts">3.2.2.2</a>, there’s a significant increase in clusters deposited (<a href="3-res.html#fig:rdtdepo">3.24</a>B). Also, neither <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> nor <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> CCs drastically change their position once they are deposited.</p>
</div>
<div id="res-rosreg" class="section level4" number="3.2.4.3">
<h4><span class="header-section-number">3.2.4.3</span> Registration</h4>
<p>As the pLLP migrates it moves through space and time. To measure velocity and acceleration, one needs to detect where in space (XY coordinates) the pLLP is located at multiple timepoints. When duplicating the pLLP based on a detection ROI from the whole image, the pLLP is registered in time and space (figure <a href="3-res.html#fig:rdtreg">3.25</a>A).</p>
<p>As shown in figure <a href="3-res.html#fig:rdtreg">3.25</a>B-B’ neither speed nor acceleration drastically differ throughout the complete course of migration. Speed drops from an initial ~75 <span class="math inline">\(\mu\)</span>m/h to about ~30 <span class="math inline">\(\mu\)</span>m/h for both <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> 17 h later (figure 3.3 A). Similarly, while there is a positive acceleration of almost ~2 <span class="math inline">\(\mu\)</span>m/h peaking after ~2-3 hours, the remaining two peaks progressively get smaller (<a href="3-res.html#fig:rdtreg">3.25</a>A’).
While for the area no difference can be detected (<a href="3-res.html#fig:rdtreg">3.25</a>C), interestingly the roundness is on average significantly reduced in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs (<a href="3-res.html#fig:rdtreg">3.25</a>D). This is also evident from the montage in figure <a href="3-res.html#fig:rdtdepo">3.24</a>A.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtreg"></span>
<img src="figures/results/03_rosettes/registration.png" alt="pLLP time-resolved morphometrics. A Registration procedure B-B’ Leading edge (l.e.) speed and acceleration in \(\mu\)m/h, displayed as LOESS curves with at a span of 0.5 C Area in square \(\mu\)m and D Roundness displayed as mean ± s.e. (standard error). See section 3.2.4.3 for a dataset description." width="95%" />
<p class="caption">
Figure 3.25: pLLP time-resolved morphometrics. <strong>A</strong> Registration procedure <strong>B-B’</strong> Leading edge (l.e.) speed and acceleration in <span class="math inline">\(\mu\)</span>m/h, displayed as LOESS curves with at a span of 0.5 <strong>C</strong> Area in square <span class="math inline">\(\mu\)</span>m and <strong>D</strong> Roundness displayed as mean ± s.e. (standard error). See section <a href="3-res.html#res-rosreg">3.2.4.3</a> for a dataset description.
</p>
</div>
</div>
<div id="res-rdtdet" class="section level4" number="3.2.4.4">
<h4><span class="header-section-number">3.2.4.4</span> Rosette Detection</h4>
<p>To quantify the maturity of rosettes within the pLLP a CNN was used which allows to detect objects within images (Region Proposal based CNN). Inherent to object detection and classification by a neural network is that the network not only tells us if an object was <em>detected</em> (of numbers room <span class="math inline">\(x\in[0, \infty]\subset\mathbb{N}\)</span>), but also how secure it is - the probability - that the object detected is the right one (of numbers room <span class="math inline">\(x\in[0, 1]\subset\mathbb{R}\)</span>) - the <em>weighted detection</em>. Furthermore, the weighted detection was capped a score <span class="math inline">\(\leq\)</span> 0.5, since below 0.5 the probability would be to insecure. In our case the network was trained to detect wildtype rosettes, while also presenting pLLPs of embryos treated with a compound that inhibits rosette formation (SU5402) to refine learning. Therefore, if the network detected 1 rosette with a probability of 0.99, it is very likely that this is a wildtype rosette. However, at different developmental stages and genetic background (e.g. mutants with impaired rosette formation) the pLLP exhibits a different count of rosettes. Therefore it would be wrong to compare just the sum of <em>detections</em> or <em>weighted detections</em> per pLLP, but it is necessary to normalize for the different count of rosettes to get a global figure of pLLP rosettization - the <em>rosettiness</em>.</p>

<div class="definition">
<span id="def:unnamed-chunk-11" class="definition"><strong>Definition 3.2  (rosettiness)  </strong></span><span class="math display">\[\frac{\sum_{} \textrm{Weighted Detections}}{\sum_{} \textrm{Detections}} = \textrm{Normalized Weight}\]</span>
</div>

<div class="figure" style="text-align: center"><span id="fig:rdtdetmdl"></span>
<img src="figures/results/03_rosettes/detection_model.png" alt="Quantification of rosettes. A Model of detection and weighted detection in shroom3\(^{+/+}\) and shroom3\(^{-/-}\) embryos B Actual detection in pLLPs of shroom3\(^{+/+}\) and shroom3\(^{-/-}\) embryos at six different timepoints and highlighted detection weight." width="95%" />
<p class="caption">
Figure 3.26: Quantification of rosettes. <strong>A</strong> Model of <em>detection</em> and <em>weighted detection</em> in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos <strong>B</strong> Actual detection in pLLPs of <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos at six different timepoints and highlighted detection weight.
</p>
</div>
<p>The kymographs<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> in figure <a href="3-res.html#fig:rdtdet">3.27</a>A-A’ were generated from pLLPs that were previously registered with the anaLLzr2DT IJ macro (section <a href="3-res.html#mat-anallzr2dt">3.1.3</a>). After registration, a line was drawn along the horizontal midline (<a href="3-res.html#fig:rdtreg">3.25</a>A). Finally, recorded kymographs were turned to false color (blue = low intensity, red = high intensity) for better visual interpretation. In <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> the higher intensities are highly concentrated to two to three distinct regions within the migrating pLLP, while in the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLP the higher intensities are more fragmented and overall reduced.</p>
<p>The numbers reveal two very interesting things. First, the medians show that rosette count (<a href="3-res.html#fig:rdtreg">3.25</a>B’) is only reduced for the first six hours, while rosettiness is reduced almost throughout the entire 20 hours. While the median rosettiness of <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> is at a constant level of about 0.9, <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> rosettiness starts out at about 0.5 but then linearly grows until it also reaches levels of about 0.9 at 16-20 hours of migration. These results, together with the finding that we could reduce the number of CCs deposited when lowering the temperature, suggests that there should be a interdependecy with speed of migration. As for rosette detection, variance is much higher.</p>
<p>Second, while for rosette counts comprised between 2 and 6, half or more of the pLLPs are <em>shroom3</em><span class="math inline">\(^{+/+}\)</span>, for counts below two and above six there are more <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs (<a href="3-res.html#fig:rdtreg">3.25</a>B). Furthermore figure <a href="3-res.html#fig:rdtdet">3.27</a>C shows that at a rosettiness of ~0.7 there are about as many <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> as there are <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs, marking this point as a threshold were pLLPs above are more like likely to be <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and below more likely to be <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>. At ~0.1 rosettiness there are almost only <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs, while at ~0.9 rosettiness there are almost only <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> pLLPs.
In addition, the data distributions shown on the right (<a href="3-res.html#fig:rdtdet">3.27</a>B’’ and C’’) reveals that for the rosettiness the ditribution did not actually shift to a lower number, but rather flattened and variance is increased.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtdet"></span>
<img src="figures/results/03_rosettes/detection.png" alt="Rosette counts and weights. A-A’ pLLP kymographs. Length in A’ = 100 \(\mu\)m. (A) shows the signal and line drawn. (A’) shows the signal through time. B-B’’ Rosette detections as whole numbers (\(\mathbb{Z}\)). (B) Relative shows the ratio of detections in shroom3\(^{+/+}\) and shroom3\(^{-/-}\) (x axis) at each count of detections per pLLP (y axis) throughout time. (B’) Medians shows the median \(\pm\) standard deviation of detections (y axis) as dots in time intervals of 2 h (x axis). (B’‘) KDE shows the density in the distribution of the data. C-C’’ Rosettiness as rational numbers (\(\mathbb{Q}\)). (B) Relative shows the ratio of rosettiness in shroom3\(^{+/+}\) and shroom3\(^{-/-}\) (x axis) at level of rosettiness per pLLP (y axis) throughout time. At ~70\(\%\) rosettiness the ratio of shroom3\(^{+/+}\) over shroom3\(^{-/-}\) is equally distributed (B’) Medians shows the median \(\pm\) standard deviation of rosettiness (y axis) as dots in time intervals of 2 h (x axis). (B’’) KDE shows the density in the distribution of the data. See section 3.2.4.3 for a dataset description." width="95%" />
<p class="caption">
Figure 3.27: Rosette counts and weights. <strong>A-A’</strong> pLLP kymographs. Length in A’ = 100 <span class="math inline">\(\mu\)</span>m. (A) shows the signal and line drawn. (A’) shows the signal through time. <strong>B-B’’</strong> Rosette detections as whole numbers (<span class="math inline">\(\mathbb{Z}\)</span>). (B) <em>Relative</em> shows the ratio of detections in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> (x axis) at each count of detections per pLLP (y axis) throughout time. (B’) <em>Medians</em> shows the median <span class="math inline">\(\pm\)</span> standard deviation of detections (y axis) as dots in time intervals of 2 h (x axis). (B’‘) <em>KDE</em> shows the density in the distribution of the data. <strong>C-C’’</strong> Rosettiness as rational numbers (<span class="math inline">\(\mathbb{Q}\)</span>). (B) <em>Relative</em> shows the ratio of rosettiness in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> (x axis) at level of rosettiness per pLLP (y axis) throughout time. At ~70<span class="math inline">\(\%\)</span> rosettiness the ratio of <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> over <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> is equally distributed (B’) <em>Medians</em> shows the median <span class="math inline">\(\pm\)</span> standard deviation of rosettiness (y axis) as dots in time intervals of 2 h (x axis). (B’’) <em>KDE</em> shows the density in the distribution of the data. See section <a href="3-res.html#res-rosreg">3.2.4.3</a> for a dataset description.
</p>
</div>
</div>
<div id="correlations" class="section level4" number="3.2.4.5">
<h4><span class="header-section-number">3.2.4.5</span> Correlations</h4>
<p>In the previous sections I have shown that in embryos deficient for Shroom3 more NMs are deposited and that this is not caused by an overproliferation in the pLLP. In this section my goal was to collect detailed information about LL morphometrics and rosette formation to find out what is causing the additional CCs to be deposited.</p>
<p>To do this I combined the results of three different analyses to be able to detect relationships between variables. For my question the most interesting variable to find correlations with is rosettiness. As mentioned in section <a href="3-res.html#intro-phen">3.2.1</a> when we had a first closer look at the pLLP phenotype we found that some of the clearly as <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> identified embyros still were reminiscent of the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> phenotype. However, what surprised us the most (due to to previous results indicating an important role during rosette assembly for Shroom3), was that a fraction had significantly more CCs deposited. To test the interdependence between rosettiness and CCs deposited quantitatively we set up two hypotheses.</p>
<p>First, when we had a closer look at timelapse movies of the migrating pLLP at a high temporal resolution we had the impression that while in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos the pLLP stalled for a moment during CC deposition, this was not the case in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos. Therefore the hypothesis to test was that higher velocity or acceleration lead to a lower degree of rosettiness and <em>vice versa</em> (figure <a href="3-res.html#fig:rdtcorr">3.28</a>A-B). While velocity is the rate at which an object moves in a certain direction, acceleration tells us the rate at which velocity changes. While most of the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> pLLPs nicely cluster in the top right corner (high rosettiness and velocity, green dots in figure <a href="3-res.html#fig:rdtcorr">3.28</a>A), for the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs there is more of a spread. However, there seems to be a weak relationship between velocity and rosettiness which is also reflected by the correlation coefficient. The relationship between acceleration and rosettiness on the other hand seems to be stronger. Again, <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> pLLPs nicely cluster in the lower right corner (high rosettiness and negative acceleration, green dots in figure <a href="3-res.html#fig:rdtcorr">3.28</a>B). Since over the course of migration the pLLP gets slower (<a href="3-res.html#fig:rdtreg">3.25</a>B), the average acceleration is negative. For the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs there is also more of a spread, but the trend is more clearly visible which is also reflected by the correlation coefficient.</p>
<p>Second, since we know Shroom3 plays a role in rosette assembly and that embryos deficient for Shroom3 have on average more CCs deposited, we wanted to test if there is a direct link between the two and have statistical certainty. Therefore, the hypothesis to test for figure <a href="3-res.html#fig:rdtcorr">3.28</a>B was if lower rosettiness leads to a higher number of CCs deposited and <em>vice versa</em>. Since here the end of migration cluster count is one-dimensional (one number per embryo), we had to correlate it to a <em>per embryos</em> summary statistic rosettiness. Here the correlation coefficient is rather high, suggesting there might be an interdependency between pLLP rosettiness and the final pattern of NM deposition. Additionally, the clusters determined are nicely separated from each other and the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs seem to form two independent groups.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtcorr"></span>
<img src="figures/results/03_rosettes/rdt_cor_velo_acc_depo.png" alt="Correlation between rosettiness, speed, acceleration and final pattern of NMs. Single dots in the scatterplots represent the variables scatter in \(x\) and \(y\), the red line shows a linear model through the point cloud of both groups shroom3\(^{+/+}\) and shroom3\(^{-/-}\), red digits indicate the correlation coefficient. Black spots with grey circles indicate the centers for clusters calculated on these two sets of data. Clustering was performed hierarchical A Correlation between pLLP rosettiness and speed of migration B Correlation between pLLP rosettiness and acceleration C Correlation between pLLP rosettiness and number of deposited clusters normalized to the length of the lateral line." width="100%" />
<p class="caption">
Figure 3.28: Correlation between rosettiness, speed, acceleration and final pattern of NMs. Single dots in the scatterplots represent the variables scatter in <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>, the red line shows a linear model through the point cloud of both groups <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>, red digits indicate the correlation coefficient. Black spots with grey circles indicate the centers for clusters calculated on these two sets of data. Clustering was performed hierarchical <strong>A</strong> Correlation between pLLP rosettiness and speed of migration <strong>B</strong> Correlation between pLLP rosettiness and acceleration <strong>C</strong> Correlation between pLLP rosettiness and number of deposited clusters normalized to the length of the lateral line.
</p>
</div>
</div>
<div id="res-rockresc" class="section level4" number="3.2.4.6">
<h4><span class="header-section-number">3.2.4.6</span> Mechanics</h4>
<p>It is very well known that Rho kinases are necessary for mechanic force transmission in various developmental processes <span class="citation">(<em>26</em>, <em>44</em>, <em>49</em>, <em>78</em>, <em>79</em>)</span>. Since Rock is also thought to be a key player of the Shroom force mediation pathway in the pLLP, the idea for a mechanic rescue was to titrate different concentrations of a Rock inhibitor (Rockout, section <a href="2-mat-met.html#mat-chem">2.1.1</a>) and a non-muscle myosin II (NMII) activator (Calyculin, section <a href="2-mat-met.html#mat-chem">2.1.1</a>) to see if this way can confirm the correlation between rosettiness and CC number in an independent manner.</p>
<div id="rock-inhibition" class="section level5" number="3.2.4.6.1">
<h5><span class="header-section-number">3.2.4.6.1</span> Rock inhibition</h5>
<p>Rock2a phosphorylates NMII which is necessary for contraction of the actin network and apical constriction (section <a href="1-Intro.html#intro-shroom">1.2.4.1</a>). Here we wanted to mimic the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> phenotype in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos by inhibiting Rock activity <em>via</em> small molecule Rockout drug treatment.</p>
<p>Embryos were incubated at distinct (figure <a href="3-res.html#fig:rescrock">3.29</a> legend) concentrations from 24 hpf till end of migration (about 60 hpf). At this stage embryos incubated at higher concentrations displayed a significant reduction in overall body length (<a href="3-res.html#fig:rescrock">3.29</a>A’). Therefore, to see if there is a specific effect on length, LL length was normalized to body length.</p>

<div class="figure" style="text-align: center"><span id="fig:rescrock"></span>
<img src="figures/results/06_rescues/rockout/rescue_rockout.png" alt="Titration of Rock inhibitor. A LL lengths are reduced with increased concentrations of Rock inhibitor and B groupwise comparison of the length normalized cluster count per concentration of Rock inhibitor." width="95%" />
<p class="caption">
Figure 3.29: Titration of Rock inhibitor. <strong>A</strong> LL lengths are reduced with increased concentrations of Rock inhibitor and <strong>B</strong> groupwise comparison of the length normalized cluster count per concentration of Rock inhibitor.
</p>
</div>
<p>Rockout treatment at 10 <span class="math inline">\(\mu\)</span>M does not lead to a significant reduction in LL length but does significantly increase the number of CCs deposited(figure <a href="supplement.html#fig:supppos">4.4</a>B and figure <a href="3-res.html#fig:rescrock">3.29</a>B), confirming a possible link between lower actomyosin contraction in the pLLP and increased number of deposited NMs.</p>
<table class="table" style="font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescrocktab">Table 3.6: </span>Rockout rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:right;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:left;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;vertical-align: top !important;" rowspan="4">
shroom3++
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:left;">
17
</td>
</tr>
<tr>
<td style="text-align:center;">
10
</td>
<td style="text-align:left;">
13
</td>
</tr>
<tr>
<td style="text-align:center;">
50
</td>
<td style="text-align:left;">
17
</td>
</tr>
<tr>
<td style="text-align:center;">
100
</td>
<td style="text-align:left;">
20
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">conc.</span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO solution
</td>
</tr>
</tfoot>
</table>
</div>
<div id="nmii-activation" class="section level5" number="3.2.4.6.2">
<h5><span class="header-section-number">3.2.4.6.2</span> NMII activation</h5>
<p>Downstream of Shroom3 NMII, as a motor protein, is necessary for contraction of the actin network and apical constriction (section <a href="1-Intro.html#intro-shroom">1.2.4.1</a>). The idea for this experiment was to rescue the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> phenotype by over-activation of NMII. Unfortunately, even when incubation at 100 nM (the maximum solubility), no significant differences can be detected in LL length (figure <a href="3-res.html#fig:resccal">3.30</a>A), cluster count (figure <a href="3-res.html#fig:resccal">3.30</a>B) or cluster position (figure <a href="supplement.html#fig:supppos">4.4</a>A).</p>

<div class="figure" style="text-align: center"><span id="fig:resccal"></span>
<img src="figures/results/06_rescues/caliculyn/rescue_cali.png" alt="NMII activation. A LL lengths and B groupwise comparison of the length normalized cluster count." width="85%" />
<p class="caption">
Figure 3.30: NMII activation. <strong>A</strong> LL lengths and <strong>B</strong> groupwise comparison of the length normalized cluster count.
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resccaltab">Table 3.7: </span>NMII activation dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (nM)
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (nM)
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3++
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;border-right:1px solid;">
8
</td>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3-\-
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8
</td>
</tr>
<tr>
<td style="text-align:center;">
100
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;border-right:1px solid;">
100
</td>
<td style="text-align:center;">
12
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">conc.</span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO solution
</td>
</tr>
</tfoot>
</table>
</div>
</div>
<div id="summary-4" class="section level4" number="3.2.4.7">
<h4><span class="header-section-number">3.2.4.7</span> Summary</h4>
<p>I could (<strong>1</strong>) demonstrate, from fixed embryos (figure <a href="3-res.html#fig:llcounts">3.18</a>B) as well as here on live imaging data (figure <a href="3-res.html#fig:rdtdepo">3.24</a>B), that in <em>shroom3</em> heterozygous mutants significantly more CCs are deposited, gaining more insight in the deposition process which shows smaller and more fragmented CCs and a more chaotic deposition process overall (figure <a href="3-res.html#fig:rdtdepo">3.24</a>A) (<strong>2</strong>) show a possible interdependence with velocity and acceleration (figure <a href="3-res.html#fig:rdtcorr">3.28</a>A-B). Also the area of the pLLP is not affected, but roundness, confirming our impression that the pLLP in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos is more elongated (figure <a href="3-res.html#fig:rdtreg">3.25</a>C-D) (<strong>3</strong>) show that while the integer number of rosettes in the pLLP is only significantly increased during the first 6-8 hours of migration, the overall rosettiness (rosette integrity per pLLP) is significantly reduced over the whole path of migration (figure <a href="3-res.html#fig:rdtdet">3.27</a>C’).</p>
<p>Rosettiness is strongly correlated to the final number of CCs deposited (<a href="3-res.html#fig:rdtcorr">3.28</a>C), concluding that rosettiness is the key parameter for the future outcome of CC pattern. Interestingly however this conclusion contradicts the previous, and intuitive, assumption that embryos deficient for Shroom3 should not deposit CCs at all. However, in the light of this data Shroom3 is important to strengthen rosette integrity in order to have CCs of the right size deposited.</p>
<p>Still, the question is: What determines rosette weight? Velocity and rosettiness are correlated weaker than acceleration and rosettiness are which indicates that the rate at which velocity changes has more of an impact than velocity itself. This finding is also confirmed by an experiment where the CC count of <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> can be rescued when incubated at lower temperature (section <a href="3-res.html#res-tempresc">3.2.2.4</a>).</p>
</div>
</div>
<div id="apical-constriction-2" class="section level3" number="3.2.5">
<h3><span class="header-section-number">3.2.5</span> Apical Constriction</h3>
<p>In the previous analysis I have shown a direct relationship between rosette integrity and CCs deposited, where lower rosette integrity leads to a fragmented rosettes and therefore to an increase in CC deposition frequency. To preserve rosette integrity and for proper CC maturation cells need to apically constrict. To gain further insight into Shroom3 contribution to apical constriction, it is appropriate to have a closer look at rosette conformation on a cellular scale. To do so, first a set of high spatial resolution, volumetric image data was generated (XY = 0.164 <span class="math inline">\(\mu\)</span>m / pixel; Z = 0.4 <span class="math inline">\(\mu\)</span>m) that allowed for…</p>
<ol style="list-style-type: decimal">
<li>a more rigorous inspection of morphological differences, and</li>
<li>automated and unbiased single cell 3D reconstruction (section <a href="2-mat-met.html#ACI-singlecell">2.2.4.3.2.1</a>) using a newly developed IJ macro (section <a href="3-res.html#mat-anallzr3d">3.1.4</a>). Secondly, to investigate the contribution of various single cell morphometrics to rosette formation an app was developed<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> that allows for a convenient handling of large amounts of retrieved data.</li>
</ol>

<div class="figure" style="text-align: center"><span id="fig:acdata"></span>
<img src="figures/results/04_constriction/reconstriction_data.png" alt="High resolution, volumetric image data. Upper row shows the fluorescence signal at the membranes. Lower row shows the 3D reconstructed cells. Columns show the same pLLP from different angles." width="95%" />
<p class="caption">
Figure 3.31: High resolution, volumetric image data. Upper row shows the fluorescence signal at the membranes. Lower row shows the 3D reconstructed cells. Columns show the same pLLP from different angles.
</p>
</div>
<p>Inhibition of FGFR1 <em>via</em> drug treatment (SU5402, section <a href="2-mat-met.html#mat-chem">2.1.1</a>) results in a concentration dependent loss of morphogenesis and rosette assembly in the pLLP <span class="citation">(<em>38</em>, <em>40</em>, <em>61</em>)</span>. As a proof of principle and to validate previous studies, data of drug treated embryos (20 <span class="math inline">\(\mu\)</span>M) and appropriate DMSO (0.1<span class="math inline">\(\%\)</span> DMSO / E3<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a>) controls were included.</p>
<div id="dataset-1" class="section level4" number="3.2.5.1">
<h4><span class="header-section-number">3.2.5.1</span> Dataset</h4>
<p>The dataset generated consists of three pLLP stages (<a href="3-res.html#fig:acstages">3.32</a>), four groups (DMSO, SU5402, <em>shrm3</em>++ and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>), 267 pLLPs and 33.163 single cells.</p>

<div class="figure" style="text-align: center"><span id="fig:acstages"></span>
<img src="figures/results/04_constriction/Yolk_ext.png" alt="Scheme showing a part of the posterior part of a zebrafish embryo and recorded developmental stages for 3D reconstruction." width="60%" />
<p class="caption">
Figure 3.32: Scheme showing a part of the posterior part of a zebrafish embryo and recorded developmental stages for 3D reconstruction.
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:acdatatab">Table 3.8: </span>Apical Constriction dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
id labels
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
metrics
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
stage
</th>
<th style="text-align:left;">
group
</th>
<th style="text-align:center;">
pLLPs
</th>
<th style="text-align:center;">
cells
</th>
<th style="text-align:center;">
ratio
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="4">
32 hpf
</td>
<td style="text-align:left;">
DMSO
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
2511
</td>
<td style="text-align:center;">
125.5
</td>
</tr>
<tr>
<td style="text-align:left;">
SU5402
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
2215
</td>
<td style="text-align:center;">
116.6
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3++
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
3259
</td>
<td style="text-align:center;">
135.8
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3-\-
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
4281
</td>
<td style="text-align:center;">
125.9
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="4">
36 hpf
</td>
<td style="text-align:left;">
DMSO
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
1881
</td>
<td style="text-align:center;">
110.6
</td>
</tr>
<tr>
<td style="text-align:left;">
SU5402
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
1286
</td>
<td style="text-align:center;">
98.9
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3++
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
3144
</td>
<td style="text-align:center;">
131.0
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3-\-
</td>
<td style="text-align:center;">
31
</td>
<td style="text-align:center;">
3924
</td>
<td style="text-align:center;">
126.6
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="4">
40 hpf
</td>
<td style="text-align:left;">
DMSO
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
2237
</td>
<td style="text-align:center;">
124.3
</td>
</tr>
<tr>
<td style="text-align:left;">
SU5402
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
1788
</td>
<td style="text-align:center;">
137.5
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3++
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
2660
</td>
<td style="text-align:center;">
120.9
</td>
</tr>
<tr>
<td style="text-align:left;">
shroom3-\-
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
3977
</td>
<td style="text-align:center;">
124.3
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">ratio</span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> cells / pLLPs
</td>
</tr>
</tfoot>
</table>
</div>
<div id="shape-and-arrangement" class="section level4" number="3.2.5.2">
<h4><span class="header-section-number">3.2.5.2</span> Shape and Arrangement</h4>
<p>Figure <a href="3-res.html#fig:acstages">3.32</a> illustrates the three developmental stages used for this analysis. Figure <a href="3-res.html#fig:acshape">3.33</a> shows for each of the 3 stages a representative example of pLLP in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span>, <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> and SU5402-treated embryo.</p>
<p>For <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> pLLPs two (<a href="3-res.html#fig:acshape">3.33</a>A, 32 hpf) to four (<a href="3-res.html#fig:acshape">3.33</a>A, 40 hpf) areas of constricting regions can be observed (XZ pane), in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> its three (<a href="3-res.html#fig:acshape">3.33</a>A, 36 hpf) to seven (<a href="3-res.html#fig:acshape">3.33</a>A, 40 hpf), confirming the results of rosette detection (<a href="3-res.html#res-rdtdet">3.2.4.4</a>). While at 32 and 36 hpf the spread of those regions (indicated by bars in <a href="3-res.html#fig:acshape">3.33</a>, XZ panes) is wider in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>, at 40 hpf they are smaller but also increased in number. When looking at the central rosette along the width ((indicated by arrows in <a href="3-res.html#fig:acshape">3.33</a>), YZ panes), it seems like there is no considerable difference between <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> rosettes. In SU5402 treated embryos, neither in XZ nor in YZ constriction can be observed and the cells seem to be mostly columnar in shape. In addition, while constriction seems isotropic in the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> rosette center, constriction in the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> rosette center cells appears anisotropic.</p>

<div class="figure" style="text-align: center"><span id="fig:acshape"></span>
<img src="figures/results/04_constriction/Figure_5-1.png" alt="Apical Constriction in 3D. Stages are in columns, the groups are in rows. Each panel shows a MaxIP (top left), and orthogonal views along the length (XZ, bottom left) and the width (YZ, right). Arrow annotations in YZ panels indicate points of constriction. Bar annotations in XZ indicate the spread of region of constriction." width="95%" />
<p class="caption">
Figure 3.33: Apical Constriction in 3D. <strong>Stages are in columns</strong>, the <strong>groups are in rows</strong>. Each panel shows a MaxIP (top left), and orthogonal views along the length (XZ, bottom left) and the width (YZ, right). Arrow annotations in YZ panels indicate points of constriction. Bar annotations in XZ indicate the spread of region of constriction.
</p>
</div>
</div>
<div id="height" class="section level4" number="3.2.5.3">
<h4><span class="header-section-number">3.2.5.3</span> Height</h4>
<p>Earlier attempts for indexing AC include measuring the ‘apical index’ (A.I.) of bottle cells during <em>X.laevis</em> gastrulation <span class="citation">(<em>80</em>)</span> or for cells of the pLLP <span class="citation">(<em>61</em>)</span>. For both, the metric describes the ratio of lateral height over apical width which, depending on the situation, is a very coarse and inaccurate description. In the pLLP cells of different forms, volumes and heights can be found. When measuring apical width at a fix distance from the apical site cells with different height but otherwise equal degree of AC turn out to have a very different A.I. (figure <a href="2-mat-met.html#fig:ACICells">2.3</a>A and A’). If however apical width is measured at a distance relative to the cells height, cells with an equal degree of AC turn out to have the same A.I. (figure <a href="2-mat-met.html#fig:ACICells">2.3</a>A and A’’). In addition, it remains questionable exactly how to define <em>apical width</em> in a two-dimensional cellular cross-section. By introducing apical width measurement along both, the major and the minor axis (section <a href="2-mat-met.html#ACI-pol">2.2.4.3.1.3</a>), the improved model for AC and method for automated and unbiased cellular segmentation therefore represents a major advancement.</p>
<p>For another demonstration of the method, figure <a href="3-res.html#fig:dismap">3.34</a> shows a heatmap of cell height, how its distributed in the pLLP and under different conditions. At stages 32 and 36 hpf, height in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> cells is not as strongly reduced as in SU5402 treated ones and cells in the central region of the pLLP are higher than in the leading, trailing and lateral regions. Therefore this representation also gives an idea which cells are most affected.</p>

<div class="figure" style="text-align: center"><span id="fig:dismap"></span>
<img src="figures/summary/methodology_mapr.png" alt="pLLP terrain maps. To get a more detailed impression about how variables (Height in this case) are distributed within the cells of the pLLP, single cell measurements of pLLPs that are normalized in orientation can be superimposed in a two-dimensional map of organ length and width. Next, cell coordinates are grouped into bins of hexagons and the summary statistic is represented in color. (bottom right) link to the LLMapR web-app." width="95%" />
<p class="caption">
Figure 3.34: pLLP terrain maps. To get a more detailed impression about how variables (Height in this case) are distributed within the cells of the pLLP, single cell measurements of pLLPs that are normalized in orientation can be superimposed in a two-dimensional map of organ length and width. Next, cell coordinates are grouped into bins of hexagons and the summary statistic is represented in color. (bottom right) link to the LLMapR web-app.
</p>
</div>
</div>
<div id="res-ai" class="section level4" number="3.2.5.4">
<h4><span class="header-section-number">3.2.5.4</span> Apical Index</h4>
<p>To get a hold on AC anisotropy we measured the length of the minor and major axis of an ellipsoid fitted to the imaging Z-plane at a certain distance from the cell apex (as described in section <a href="2-mat-met.html#ACI-pol">2.2.4.3.1.3</a> and illustrated in figure <a href="3-res.html#fig:acaci">3.35</a>A). The angle formed by the major axis of the ellipsoid and the horizontal midline indicate the orientation of AC (figure <a href="3-res.html#fig:acaci">3.35</a>A’). If more cells are constricted along the DV axis, we expect angles closer to 0<span class="math inline">\(^\circ\)</span>. If more cells are constricted along the AP axis, we expect angles closer to 90<span class="math inline">\(^\circ\)</span>. For angle measurements, all angles are normalized to a 90<span class="math inline">\(^\circ\)</span> range, where 0<span class="math inline">\(^\circ\)</span> is along the horizontal midline.</p>
<p>When comparing the ratio of cells in intervals of 15<span class="math inline">\(^\circ\)</span> along a range of 90<span class="math inline">\(^\circ\)</span> it can be shown that there are more <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> than <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> cells within the 0-15<span class="math inline">\(^\circ\)</span> interval through all three stages (<a href="3-res.html#fig:acstages">3.32</a>). Even more, it shows that there are more <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> than <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> cells within the remaining intervals – confirming the model of anisotropic constriction along the DV axis.</p>
<p>While <em>reduction</em> in A.I. is stronger than the A.I. in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> cells (<a href="3-res.html#fig:acaci">3.35</a>C-C’), both are significantly reduced throughout time – again confirming the increase in constriction an-isotropy. For all three timepoints the A.I. for SU5402 treated embryos is not only the smallest, but also stays at the same level (~2 in the A.I., ~3.4 in the A.I.), therefore it can be seen as a lower threshold. Interestingly, the A.I.s for DMSO, <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> are progressively converging this threshold from earlier to later stages.</p>

<div class="figure" style="text-align: center"><span id="fig:acaci"></span>
<img src="figures/results/04_constriction/Figure_5-2.png" alt="Apical constriction metrics. A-A’ Measurements. (A) Oval (black outline) represents the fit ellipse. Red arrows represent major and minor axes which are representative for A-P and D-V respectively (A’) Angle measurements and intervals B Cellular orientation as bar chart. The y axis represents the (normalized) fraction of cells per cells in the whole organ. p-value levels (wilcox-test) are indicated as stars. C-C’ Apical Constriction measurements. Colored bars represent the median. Errorbars indicate 95\(\%\) CI." width="95%" />
<p class="caption">
Figure 3.35: Apical constriction metrics. <strong>A-A’</strong> Measurements. (A) Oval (black outline) represents the fit ellipse. Red arrows represent major and minor axes which are representative for A-P and D-V respectively (A’) Angle measurements and intervals <strong>B</strong> Cellular orientation as bar chart. The y axis represents the (normalized) fraction of cells per cells in the whole organ. p-value levels (wilcox-test) are indicated as stars. <strong>C-C’</strong> Apical Constriction measurements. Colored bars represent the median. Errorbars indicate 95<span class="math inline">\(\%\)</span> CI.
</p>
</div>
</div>
<div id="sum-ai" class="section level4" number="3.2.5.5">
<h4><span class="header-section-number">3.2.5.5</span> Summary</h4>
<p>In the previous chapter I provided strong indication that the main reason for the additional CCs deposited in Embryos deficient for Shroom3 is reduced rosettiness. Here I wanted to get a more detailed view on how rosette configuration and cell morphology compares between wildtype and mutant.</p>
<p>The method developed to quantify AC allows measurement along two orthogonal axes, the major (the longer axis) and a minor (the shorter axis), of an ellipse fit to a cellular cross-section at a distinct distance from the apical site. In addition, the cells orientation in degrees from the horizontal midline is obtained (<a href="3-res.html#fig:acaci">3.35</a>A-A’).</p>
<p>In summary, AC is significantly reduced in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos. The results show an increased axial an-isotropy, meaning that AC is more reduced along the major than along the minor axis. In conclusion this means that due to less tight rosette packing and apical constriction the migrational pulling forces are probably accounting for the an-isotropy. In terms of cell shape this means that rosette cells go from a pyramidal geometry, more to a prism geometry (figure <a href="3-res.html#fig:sumcells">3.36</a>). This is also confirmed by angular measurement which show that in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs there are more cells in the range of 0-15<span class="math inline">\(^\circ\)</span> while in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> pLLPs more cells are in the range of 30-45<span class="math inline">\(^\circ\)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:sumcells"></span>
<img src="figures/summary/cell-model.png" alt="Rosette and Cell Shape Model. Shapes reconstructed from real cells." width="90%" />
<p class="caption">
Figure 3.36: Rosette and Cell Shape Model. Shapes reconstructed from real cells.
</p>
</div>
<p>Zona occludens-1 (ZO-1) is a peripheral membrane, tight-junction associated protein. Within the apical region of pLLP rosettes FGF filled luminal structures are developing that are hubs for locally confined morphogen signaling. All cells of a rosette make up and are connected to a single lumen. When visualizing the lumen <em>via</em> an antibody for ZO-1, the lumen appears like a buckyball where each side is the connection to a single cell <span class="citation">(<em>40</em>)</span>. While in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> rosettes those structures appear circular in the trailing rosette, in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> rosettes they appear more oval (supplement figure <a href="supplement.html#fig:suppzo1">4.2</a>, which supports the theory of an-isotropy. Furthermore, in more leading regions the signal is more fragmented, which supports the theory of micro-rosette structures.</p>
</div>
</div>
<div id="res-hc" class="section level3" number="3.2.6">
<h3><span class="header-section-number">3.2.6</span> Haircell Specification</h3>
<p>In the analyses presented so far I have shown that embryos deficient for Shroom3 deposit more CCs, that this is caused by a fragmentation of rosettes (not by overproliferation) and that AC, which is important for CC maturation, is only partially diminished (more in A-P than in D-V axis). Since we also observe a shift in cell geometry from a pyramidal to a more prism geometry, we wanted to know if this had downstream consequences on hair-cell (HC) specification.</p>
<p>As described in section <a href="1-Intro.html#intro-morph">1.1.3</a>, the shape of a cell has direct impact on its ability to perceive and react to external cues. Here we were interested to test if the reduction in AC (change in cell shape, section <a href="2-mat-met.html#ACI-pol">2.2.4.3.1.3</a>) in the pLLP had an impact on HC specification and therefore if the LLs function as a sensory system would still be intact. Atoh1a (section <a href="1-Intro.html#intro-atoh">1.2.3.2</a>, a TF that gives cells the potential to become sensory HCs), in the pLLP is locally expressed between leader and follower cells and is finally restricted to a single center cell <em>via</em> lateral inhibition (section <a href="1-Intro.html#intro-notch">1.1.2.3</a>). Since till today it was not shown why the center cell becomes the HC - here our hypothesis was that rosette assembly (radial organization) respectively AC could play a role.</p>
<p>First, to check if <em>atoh1a</em> was still expressed in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> cells and if there were further implications in feedback loops with Notch signaling, an <em>In Situ Hybridization</em> (ISH, section <a href="2-mat-met.html#ISH-met">2.2.3.2</a>) experiment was conducted (probes in section <a href="2-mat-met.html#mat-probes">2.1.8</a>). For 36 and 40 hpf the count of <em>atoh1a</em> and <em>deltaD</em> expressing cells (<a href="3-res.html#fig:hcish">3.37</a>) seemed to be increased. Furthermore, especially at 40 hpf, the signal seems to be a lot more fragmented and fuzzier.</p>

<div class="figure" style="text-align: center"><span id="fig:hcish"></span>
<img src="figures/results/05_atoh/hc_ish.png" alt="Expression of deltaD and atoh1a in the pLLP. Recorded in greyscale at brightfield at 20X Magnification. Images show background subtracted and de-noised EDFs of original Z-stack data." width="85%" />
<p class="caption">
Figure 3.37: Expression of deltaD and atoh1a in the pLLP. Recorded in greyscale at brightfield at 20X Magnification. Images show background subtracted and de-noised EDFs of original Z-stack data.
</p>
</div>
<p>Traditional ISH allows for detection of RNA transcripts on a whole embryo scale <em>via</em> a hybridizing anti-sense probe. However, since embryos are fixed, it only allows to analyze single time points. Furthermore, since images are taken in brightfield, quantification of signal intensity is difficult and since what is visualized is a precipitate rather than a signal there is not a clear linear relationship to quantify the amount of expression.</p>
<p>To get access to developmental dynamics we made use of an additional, besides <em>cldnb:lyn-gfp</em>, transgenic construct where tdTomato <span class="citation">(<em>81</em>)</span> is expressed under direct control of the <em>atoh1a</em> promotor. By simultaneous observation of both fluorophores in a time-lapse setup (<a href="3-res.html#fig:hctl">3.38</a>B), the strategy was to follow up the dynamics of tdTomato to quantify the expression of <em>atoh1a</em> in cell count and intensity of expression.</p>
<div id="dataset-2" class="section level4" number="3.2.6.1">
<h4><span class="header-section-number">3.2.6.1</span> Dataset</h4>
<p>For each group, <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>, and timepoint the dataset was split into pLLP and CC (table <a href="3-res.html#tab:hcdatatab">3.9</a>). Each time-lapse has a duration of 18-20 hours, consists of about 54 timepoints and two channels (488 &amp; 561 nm).</p>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:hcdatatab">Table 3.9: </span>Hair-cell specification dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
id labels
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
metrics
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
genotype
</th>
<th style="text-align:left;">
structure
</th>
<th style="text-align:center;">
embryos
</th>
<th style="text-align:center;">
timepoints
</th>
<th style="text-align:center;">
per 3h
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="2">
shroom3++
</td>
<td style="text-align:left;">
pLLP
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
1566
</td>
<td style="text-align:center;">
261
</td>
</tr>
<tr>
<td style="text-align:left;">
CC
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
2499
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="2">
shroom3-\-
</td>
<td style="text-align:left;">
pLLP
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
1134
</td>
<td style="text-align:center;">
189
</td>
</tr>
<tr>
<td style="text-align:left;">
CC
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
3566
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
</tbody>
</table></li>
</ul></li>
</ul>
</div>
<div id="atoh1a-in-the-ll" class="section level4" number="3.2.6.2">
<h4><span class="header-section-number">3.2.6.2</span> Atoh1a in the LL</h4>
<p>While in both <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> tdTomato signal intensities are rising as soon as CCs are deposited (figure <a href="3-res.html#fig:hctl">3.38</a>A), in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs intensities are on the rise already in the first four hours of migration. Comparing signal intensities in deposited CCs for both, <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> and <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> show a similar pattern of increasing tdTomato signal intensity over the course of development but decreasing tdTomato signal intensity at the timepoint of deposition with progressive CC depositions (figure <a href="3-res.html#fig:hctl">3.38</a>B). Similarly, when looking at ungrouped CC signal intensity (not <em>per</em> CC, but as one group), no difference can be detected (figure <a href="3-res.html#fig:hctl">3.38</a>C).</p>
<p>While there is no difference in relative numbers of <em>atoh1a</em> expressing cells within the migrating pLLP throughout time (figure <a href="3-res.html#fig:hctl">3.38</a>D), when comparing mean <em>atoh1a</em> expressing cell counts at specific intervals (about the deposition frequency) the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> line is always on or above the <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> line (figure <a href="3-res.html#fig:hctl">3.38</a>D’), indicating significantly more <em>atoh1a</em> expressing cells in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLPs.</p>
<p>The extrusions to the left in the Kymographs in figure <a href="3-res.html#fig:hctl">3.38</a>D are representative to the elongating pLLP during the process of CC deposition. Magenta curves represent expression of <em>atoh1a</em>. For the <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> pLLP, premature expression of <em>atoh1a</em> can be detected. The diagonal lines in the kymographs in figure <a href="3-res.html#fig:hctl">3.38</a>D’ represent the moving pLLP along the LL. Vertical curves represent deposited CCs. As for the registered pLLP, Magenta curves represent expression of <em>atoh1a</em>.</p>

<div class="figure" style="text-align: center"><span id="fig:hctl"></span>
<img src="figures/results/05_atoh/ato-tom.png" alt="Hair-cell specification in the LL.A Sample of images analyzed. upper panel shows the cldnb:lyn-gfp transgene with segmentation masks highlighted in red. Each structure is labeled with the corresponding label in the tabular data. lower panel shows the atoh1a:Tom transgene with segmentation mask transferred from upper panel. B and C Maximum tdTomato intensities in the pLLP and CCs. All CC lifetimes were normalized to the timepoint of deposition (timepoint zero, highlighted by striped vertical line). pLLP intensities are shown to the right (black curve), CC intensities are shown on a negative scale to the left (colored curves). D Relative numbers of atoh1a expressing cells to number of timepoints and D’ mean counts of atoh1a expressing cells. D-D’ Kymographs along the horizontal midline showing nascent signal of Tom (in magenta) E two individual pLLPs that were registered in time and to the leading edge. And E’ Tom signal during LL development." width="95%" />
<p class="caption">
Figure 3.38: Hair-cell specification in the LL.<strong>A</strong> Sample of images analyzed. <em>upper panel</em> shows the <em>cldnb:lyn-gfp</em> transgene with segmentation masks highlighted in red. Each structure is labeled with the corresponding label in the tabular data. <em>lower panel</em> shows the <em>atoh1a:Tom</em> transgene with segmentation mask transferred from upper panel. <strong>B and C</strong> Maximum tdTomato intensities in the pLLP and CCs. All CC lifetimes were normalized to the timepoint of deposition (timepoint zero, highlighted by striped vertical line). pLLP intensities are shown to the right (black curve), CC intensities are shown on a negative scale to the left (colored curves). <strong>D</strong> Relative numbers of <em>atoh1a</em> expressing cells to number of timepoints and <strong>D’</strong> mean counts of <em>atoh1a</em> expressing cells. <strong>D-D’</strong> Kymographs along the horizontal midline showing nascent signal of Tom (in magenta) <strong>E</strong> two individual pLLPs that were registered in time and to the leading edge. And <strong>E’</strong> Tom signal during LL development.
</p>
</div>
<p>Deposition and position of the first CC is more independent of Shroom3 (section <a href="3-res.html#res-ccounts">3.2.2.2</a>) than for the remaining CCs. Interestingly during the first four hours, when the first CC is deposited, analysis of hair-cell specification shows that not only the promotor for <em>atoh1a</em> is more active in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos (figure <a href="3-res.html#fig:hctl">3.38</a> A and B), but also that more cells are differentiating to become HCs (figure <a href="3-res.html#fig:hctl">3.38</a>C and C’). Together with my previous results these results suggest that the reduction in AC and rosettiness in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos correlate with a premature activation of <em>atoh1a</em> expression in the pLLP which, in turn, suggests that changes in cell shape might impact cell fate decision in the pLLP.</p>
</div>
<div id="morphogen-rescue" class="section level4" number="3.2.6.3">
<h4><span class="header-section-number">3.2.6.3</span> Morphogen Rescue</h4>
<p>As stated in section <a href="1-Intro.html#intro-FGF">1.1.2.2</a> and <a href="1-Intro.html#intro-shroom">1.2.4.1</a>, FGF plays a key role during LL development and, additionally, more recent research shows that levels of FGF activity have direct impact on the number of CCs deposited <span class="citation">(<em>40</em>)</span>.</p>
<p>Since in embryos deficient for Shroom3 there are significantly more CCs deposited and since we observe a pre-mature activation of Atoh1a (which activates FGF signaling), we wanted to test whether the increase in CCs deposited can be explained by an increase in FGF activity. Inhibition of FGF also reduces LL length <span class="citation">(<em>38</em>, <em>40</em>)</span>. Therefore, a titration series experiment with an FGF inhibitor (SU5402) was planned to find a concentration that would not affect the length of the LL but still restore <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> levels of CC count.</p>

<div class="figure" style="text-align: center"><span id="fig:rescsu"></span>
<img src="figures/results/06_rescues/su54/rescue_su.png" alt="Titration of FGF inhibitor. A LL lengths are reduced with increased concentrations of FGF inhibitor and B groupwise comparison of the length normalized cluster count per concentration of FGF inhibitor." width="95%" />
<p class="caption">
Figure 3.39: Titration of FGF inhibitor. <strong>A</strong> LL lengths are reduced with increased concentrations of FGF inhibitor and <strong>B</strong> groupwise comparison of the length normalized cluster count per concentration of FGF inhibitor.
</p>
</div>
<p>Concentrations 0.10 and 0.20 <span class="math inline">\(\mu\)</span>M do not show a significant reduction in LL length (figure <a href="3-res.html#fig:rescsu">3.39</a>A), but they also do not exhibit any difference in CC count (figure <a href="supplement.html#fig:supppos">4.4</a>C and figure <a href="3-res.html#fig:rescsu">3.39</a>B). While the titration worked out well, we reach wildtype levels of CC count only at a concentration of 1 <span class="math inline">\(\mu\)</span>M of SU5402, however at this concentration the LL length is also reduced by ~ 1 mm. Since FGF plays a crucial role in many developmental processes and since SU5402 affects all of these, <span class="math inline">\(H_0\)</span> can at this point neither be rejected nor accepted. More precise methods need to be applied.</p>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescsutab">Table 3.10: </span>Morphogen rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;vertical-align: top !important;" rowspan="6">
shroom3++
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:center;border-right:1px solid;">
14
</td>
<td style="text-align:center;vertical-align: top !important;" rowspan="6">
shroom3-\-
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:center;">
8
</td>
</tr>
<tr>
<td style="text-align:center;">
0.10
</td>
<td style="text-align:center;border-right:1px solid;">
16
</td>
<td style="text-align:center;">
0.10
</td>
<td style="text-align:center;">
15
</td>
</tr>
<tr>
<td style="text-align:center;">
0.25
</td>
<td style="text-align:center;border-right:1px solid;">
24
</td>
<td style="text-align:center;">
0.25
</td>
<td style="text-align:center;">
21
</td>
</tr>
<tr>
<td style="text-align:center;">
0.40
</td>
<td style="text-align:center;border-right:1px solid;">
22
</td>
<td style="text-align:center;">
0.40
</td>
<td style="text-align:center;">
23
</td>
</tr>
<tr>
<td style="text-align:center;">
0.50
</td>
<td style="text-align:center;border-right:1px solid;">
13
</td>
<td style="text-align:center;">
0.50
</td>
<td style="text-align:center;">
4
</td>
</tr>
<tr>
<td style="text-align:center;">
1.00
</td>
<td style="text-align:center;border-right:1px solid;">
10
</td>
<td style="text-align:center;">
1.00
</td>
<td style="text-align:center;">
10
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">conc.</span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO solution
</td>
</tr>
</tfoot>
</table>
</div>
<div id="atoh1a-knockdown" class="section level4" number="3.2.6.4">
<h4><span class="header-section-number">3.2.6.4</span> <em>atoh1a</em> knockdown</h4>
<p>Since expression of Atoh1a depends on Fgf signling <span class="citation">(<em>40</em>)</span>, the hypothesis to test was if premature CC deposition in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos is determined by a premature rise in Atoh1a signaling and if Fgf was also sufficient for CC deposition.</p>
<p>To test this hypothesis, I injected <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos with an anti-sense morpholino (MO) that had previously been shown to inhibit <em>atoh1a</em> mRNA translation. Furthermore, since many MOs have been shown to induce unspecific, p53-mediated cell death <span class="citation">(<em>82</em>)</span>, in a second group I co-injected embryos with an MO for p53 (section <a href="2-mat-met.html#mat-mos">2.1.9</a> for details about MOs).</p>
<p>Interestingly injection of <em>atoh1a</em> MO lead to a loss of the first CC, usually deposited at around 400 <span class="math inline">\(\mu\)</span>m distance from the otic vesicle (figure <a href="3-res.html#fig:rescato">3.40</a>A). Furthermore, while median CC counts in <em>atoh1a</em> MO injected <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos are actually restored to <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> levels, CC counts in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos are significantly reduced as well.</p>

<div class="figure" style="text-align: center"><span id="fig:rescato"></span>
<img src="figures/results/06_rescues/atoh1a/rescue_atoh.png" alt="atoh1a knockdown. A CC mean positions and B groupwise comparison of the length normalized cluster count." width="95%" />
<p class="caption">
Figure 3.40: <em>atoh1a</em> knockdown. <strong>A</strong> CC mean positions and <strong>B</strong> groupwise comparison of the length normalized cluster count.
</p>
</div>
<p>In the expression analysis of <em>atoh1a</em> in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos I could show a premature expression of <em>atoh1a</em> in the pLLP. When trying to rescue the phenotype by knocking down <em>atoh1a</em>, neither in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> nor in <em>shroom3</em><span class="math inline">\(^{+/+}\)</span> embryos the first NM is deposited anymore. Both of these results would in theory argue for an Atoh1a dependent mechanism of CC deposition, however this result has not been reported before and would need further validation. Furthermore, even though an MO for p53 was co-injected, MO injection has been shown to potentially have many more unspecific effects due to the high concentrations usually injected <span class="citation">(<em>83</em>)</span>. For this reason double mutants ( <em>shroom3</em><span class="math inline">\(^{-/-}\)</span>; <em>atoh1a</em>--) were generated - which at this timepoints however are not analyzed yet.</p>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 9px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescatotab">Table 3.11: </span>Atoh rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
group
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
group
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3++
</td>
<td style="text-align:center;">
p53
</td>
<td style="text-align:center;border-right:1px solid;">
24
</td>
<td style="text-align:center;vertical-align: top !important;" rowspan="2">
shroom3-\-
</td>
<td style="text-align:center;">
p53
</td>
<td style="text-align:center;">
22
</td>
</tr>
<tr>
<td style="text-align:center;">
p53+atoh1a
</td>
<td style="text-align:center;border-right:1px solid;">
22
</td>
<td style="text-align:center;">
p53+atoh1a
</td>
<td style="text-align:center;">
27
</td>
</tr>
</tbody>
</table>
<table class="table" style="font-size: 7px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescatosignif">Table 3.12: </span>Atoh1a MO rescue statistics
</caption>
<thead>
<tr>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
p.adj
</th>
<th style="text-align:center;">
p.format
</th>
<th style="text-align:left;">
p.signif
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="3">
++ p53
</td>
<td style="text-align:left;">
++ p53-ato
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
-\- p53
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
-\- p53-ato
</td>
<td style="text-align:center;">
0.26803
</td>
<td style="text-align:center;">
0.27000
</td>
<td style="text-align:center;">
0.26800
</td>
<td style="text-align:left;">
ns
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="2">
++ p53-ato
</td>
<td style="text-align:left;">
-\- p53
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
-\- p53-ato
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
0.00002
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
-\- p53
</td>
<td style="text-align:left;">
-\- p53-ato
</td>
<td style="text-align:center;">
0.00466
</td>
<td style="text-align:center;">
0.00930
</td>
<td style="text-align:center;">
0.00470
</td>
<td style="text-align:left;">
**
</td>
</tr>
</tbody>
</table>
</div>
<div id="summary-5" class="section level4" number="3.2.6.5">
<h4><span class="header-section-number">3.2.6.5</span> Summary</h4>
<p>Spatiotemporal expression analysis of <em>atoh1a</em> in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> revealed that expression of <em>atoh1a</em> is initiated ealier in <em>shroom3</em><span class="math inline">\(^{-/-}\)</span> embryos (figure <a href="3-res.html#fig:hctl">3.38</a>B-C) and that the number of cells expressing <em>atoh1a</em> within the migrating pLLP is also increased (figure <a href="3-res.html#fig:hctl">3.38</a>D’). Reduction of <em>atoh1a</em> expression by inhibition of FGF did not work since inhibition of FGF also significantly affects LL length. Rescue by morpholino injection resulted in a complete absence of the location of the first neuromasts - a result which does not allow for concise conclusion at this point and needs to be repeated <em>in vivo</em> with <em>shroom3</em> ; <em>atoh1a</em> double mutants.</p>

</div>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="15">
<li id="fn15"><p>for further details about dataset acquisition see section <a href="2-mat-met.html#mat-datasets">2.4</a> - Proliferation dataset<a href="3-res.html#fnref15" class="footnote-back">↩︎</a></p></li>
<li id="fn16"><p>a kymograph is a tool to record position over time<a href="3-res.html#fnref16" class="footnote-back">↩︎</a></p></li>
<li id="fn17"><p>LLMapR - dskleinhans.shinyapps.io/LLmapR<a href="3-res.html#fnref17" class="footnote-back">↩︎</a></p></li>
<li id="fn18"><p>‘Embryo Medium 3,’ section <a href="2-mat-met.html#mat-buff">2.1.6</a><a href="3-res.html#fnref18" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-mat-met.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis_dsk.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
