<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Results | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="Chapter 3 Results | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.13.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Results | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Results | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-mat-met.html"/>
<link rel="next" href="4-summary-conclusion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html"><i class="fa fa-check"></i>Instructions</a><ul>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#print-version"><i class="fa fa-check"></i>Print version</a></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#electronic-version"><i class="fa fa-check"></i>Electronic version</a></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#repository"><i class="fa fa-check"></i>repository</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="list-of-abbreviations.html"><a href="list-of-abbreviations.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#morphogenesis"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#objective-and-open-questions"><i class="fa fa-check"></i><b>1.3</b> Objective and Open Questions</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-Intro.html"><a href="1-Intro.html#motivation"><i class="fa fa-check"></i><b>1.3.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-Intro.html"><a href="1-Intro.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>1.4</b> Data Strategy and Analysis</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-Intro.html"><a href="1-Intro.html#sample-preparation"><i class="fa fa-check"></i><b>1.4.1</b> Sample Preparation</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-Intro.html"><a href="1-Intro.html#imaging"><i class="fa fa-check"></i><b>1.4.2</b> Imaging</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-Intro.html"><a href="1-Intro.html#data-handling"><i class="fa fa-check"></i><b>1.4.3</b> Data handling</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-Intro.html"><a href="1-Intro.html#image-and-data-analysis"><i class="fa fa-check"></i><b>1.4.4</b> Image and Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.9</b> Hardware</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.10</b> Software</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.1.11</b> Ground Truth Data</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.1</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.2</b> Wetlab</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.3</b> Computational</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.3</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.1</b> Lateral Line Morphometrics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#dataset"><i class="fa fa-check"></i><b>3.1.1</b> Dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#res-ccounts"><i class="fa fa-check"></i><b>3.1.2</b> Number and Position of Cell Clusters</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#res-llmorph"><i class="fa fa-check"></i><b>3.1.3</b> Cell Count and Area of CCs</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2</b> Proliferation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#dataset-1"><i class="fa fa-check"></i><b>3.2.1</b> Dataset</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#res-prolpllp"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Mitoses</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.3</b> Rosette Formation and Cluster Deposition</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-res.html"><a href="3-res.html#dataset-2"><i class="fa fa-check"></i><b>3.3.1</b> Dataset</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-res.html"><a href="3-res.html#cluster-deposition"><i class="fa fa-check"></i><b>3.3.2</b> Cluster Deposition</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-res.html"><a href="3-res.html#registration-1"><i class="fa fa-check"></i><b>3.3.3</b> Registration</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-res.html"><a href="3-res.html#res-rdtdet"><i class="fa fa-check"></i><b>3.3.4</b> Rosette Detection</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-res.html"><a href="3-res.html#correlations"><i class="fa fa-check"></i><b>3.3.5</b> Correlations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-res.html"><a href="3-res.html#apical-constriction-1"><i class="fa fa-check"></i><b>3.4</b> Apical Constriction</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-res.html"><a href="3-res.html#dataset-3"><i class="fa fa-check"></i><b>3.4.1</b> Dataset</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-res.html"><a href="3-res.html#cell-shape-and-arrangement"><i class="fa fa-check"></i><b>3.4.2</b> Cell Shape and Arrangement</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-res.html"><a href="3-res.html#single-cell-metrics"><i class="fa fa-check"></i><b>3.4.3</b> Single Cell Metrics</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.5</b> Haircell Specification</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-res.html"><a href="3-res.html#dataset-4"><i class="fa fa-check"></i><b>3.5.1</b> Dataset</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-res.html"><a href="3-res.html#atoh1a-in-the-ll"><i class="fa fa-check"></i><b>3.5.2</b> Atoh1a in the LL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-res.html"><a href="3-res.html#rescue-experiments"><i class="fa fa-check"></i><b>3.6</b> Rescue Experiments</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-res.html"><a href="3-res.html#res-shrmresc"><i class="fa fa-check"></i><b>3.6.1</b> Shroom3 Ectopic Expression</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-res.html"><a href="3-res.html#morphogen-rescue"><i class="fa fa-check"></i><b>3.6.2</b> Morphogen Rescue</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-res.html"><a href="3-res.html#res-rockresc"><i class="fa fa-check"></i><b>3.6.3</b> Mechanic Rescue</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-res.html"><a href="3-res.html#res-tempresc"><i class="fa fa-check"></i><b>3.6.4</b> Thermodynamic Rescue</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-res.html"><a href="3-res.html#atoh1a-knockdown"><i class="fa fa-check"></i><b>3.6.5</b> atoh1a knockdown</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#adult-phenotype"><i class="fa fa-check"></i><b>4.1</b> Adult Phenotype</a></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#lateral-line"><i class="fa fa-check"></i><b>4.2</b> Lateral Line</a></li>
<li class="chapter" data-level="4.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.3</b> Rosettes</a></li>
<li class="chapter" data-level="4.4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#hair-cells"><i class="fa fa-check"></i><b>4.4</b> Hair Cells</a></li>
<li class="chapter" data-level="4.5" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-2"><i class="fa fa-check"></i><b>4.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.6" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.6</b> Model</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.1</b> Proliferation</a></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection"><i class="fa fa-check"></i><b>5.2</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.3" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification-1"><i class="fa fa-check"></i><b>5.3</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.4" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.4</b> Single cell reconstruction and AC measurement</a></li>
<li class="chapter" data-level="5.5" data-path="5-discussion.html"><a href="5-discussion.html#future-perspectives"><i class="fa fa-check"></i><b>5.5</b> Future perspectives</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="res" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Results</h1>
<div id="lateral-line-morphometrics" class="section level2">
<h2><span class="header-section-number">3.1</span> Lateral Line Morphometrics</h2>
<p>Based on the working model and due to the morphological similarity to <em>fgf3/10</em><span class="math inline">\(--\)</span> and drug treated Fgfr1 inhibited pLLPs <span class="citation">(<em>12</em>, <em>24</em>, <em>27</em>, <em>55</em>)</span>, a reduced number of CCs was expected to be deposited at the end of pLLP migration. Instead we found more CCs deposited <a href="1-Intro.html#fig:shrmmut">1.14</a>E.</p>
<div id="dataset" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Dataset</h3>
<p>To confirm these findings quantitatively a dataset was put together consisting of ~100 zebrafish embryos fixed at the end of PLLp migration, derived from four different parent pairs. After genotyping, this gave us ~33 wildtypes, ~66 heterozygous and ~33 homozygous mutants for statistical tests. Furthermore, since an increased number of deposited CCs could potentially indicate…</p>
<ul>
<li>that CCs are deposited prematurely and that CC area<a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a> would therefore be reduced</li>
<li>an increase in proliferation which would lead to an increase in cell number in the total system,</li>
</ul>
<p>we also measured the area each CC took up and counted the number of cells each CC was made up of.</p>
</div>
<div id="res-ccounts" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Number and Position of Cell Clusters</h3>
<p>To normalize the number of deposited CCs to individual LL lengths, for comparison the ratio of LL length over CC count is used (<a href="3-res.html#fig:llcounts">3.1</a>B). To correct for uneven LL paths<a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a> and irregularities in mounting<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a>, CC distances are calculated in Euclidian space rather than solely in dimension X. While <em>shrm3++</em> embryos deposit 6 <span class="math inline">\(\pm\)</span> 0.7 CCs, <em>shroom3</em><span class="math inline">\(--\)</span> embryos deposit 8 <span class="math inline">\(\pm\)</span> 0.9 CCs (<a href="3-res.html#fig:llcounts">3.1</a>A). This difference stays true also when normalizing against length (<a href="3-res.html#fig:llcounts">3.1</a>B).</p>

<div class="figure" style="text-align: center"><span id="fig:llcounts"></span>
<img src="figures/results/01_morphometrics/ll_counts.png" alt="Cluster counts A cluster count B normalized to length (LL length [µm] / cluster count [n])" width="0.6\textwidth" />
<p class="caption">
Figure 3.1: Cluster counts <strong>A</strong> cluster count <strong>B</strong> normalized to length (LL length [µm] / cluster count [n])
</p>
</div>
<p>Even though CC position in individual <em>shroom3</em><span class="math inline">\(--\)</span> embryos seems more random, the position of the first deposited CC is mostly conserved (<a href="3-res.html#fig:llpos">3.2</a>A), <span class="math inline">\(p\)</span> for difference in position = 0.2). Similarly, the position of the pLLP also isn’t significantly changed. While for the remaining CCs an average lag of -50.4 <span class="math inline">\(\mu\)</span>m as compared to <em>shroom3</em>++ CC positions is observed, it also seems to increase with later CC positions. For <em>shroom3</em>++ embryos CC position is mostly conserved through development, however it remains elusive if this is true for <em>shroom3</em><span class="math inline">\(--\)</span> embryos too.
Figure <a href="3-res.html#fig:llpos">3.2</a>A’, shows the kernel density distribution for the CC position without grouping individual positions. At a binwidth of 50 <span class="math inline">\(\mu\)</span>m, the distribution curves show a high and narrow density at around 350 <span class="math inline">\(\mu\)</span>m, which is the average location of CC1 (<a href="3-res.html#fig:llpos">3.2</a> 2A). For the remaining CCs the kernel distribution does, neither for <em>shroom3</em>++ nor for <em>shroom3</em><span class="math inline">\(--\)</span> embryos, reveal a precise location. In contrast, if based on CC sequential identity the mean position and standard deviation is calculated, a more explicit pattern emerges which clearly shows the increased count and average frequency (<a href="3-res.html#fig:llpos">3.2</a> 2A).</p>

<div class="figure" style="text-align: center"><span id="fig:llpos"></span>
<img src="figures/results/01_morphometrics/ll_positions.png" alt="Cell Cluster positions A Exemplary shroom3++ and \(--\) embryos with CCs hightlighted. CC0 marks the reference location to compare individual embryos. Scale bar = 1 mm A’ Kernel Density Estimate without (KDE) grouping (n++=162, n–=206) A’' Dots = mean positions, bars = standard deviation (nmax for both ++ and -- =26)." width="0.85\textwidth" />
<p class="caption">
Figure 3.2: Cell Cluster positions <strong>A</strong> Exemplary <em>shroom3</em>++ and <span class="math inline">\(--\)</span> embryos with CCs hightlighted. CC0 marks the reference location to compare individual embryos. Scale bar = 1 mm <strong>A’</strong> Kernel Density Estimate without (KDE) grouping (n++=162, n–=206) <strong>A’'</strong> Dots = mean positions, bars = standard deviation (nmax for both ++ and -- =26).
</p>
</div>
</div>
<div id="res-llmorph" class="section level3">
<h3><span class="header-section-number">3.1.3</span> Cell Count and Area of CCs</h3>
<p>To get an idea about whether facilitated proliferation contributes to the phenotype we observe in <em>shroom3</em><span class="math inline">\(--\)</span> embryos, the cell count per CC was determined by counting DAPI stained cells within CC segments derived from the <em>cldnb:lyn-gfp</em> membrane signal (section <a href="2-mat-met.html#mat-GrTrDat">2.1.11</a>). Both cells per CC and area per CC are shrunk by about 6% in <em>shroom3</em><span class="math inline">\(--\)</span> embryos, while the density of cells per area remains unchanged (<a href="3-res.html#fig:llclus">3.3</a>A). Interestingly, when comparing net cell counts per embryo an increase of 9% is observed (<a href="3-res.html#fig:llclus">3.3</a>B). Both the sum of CC cells and total LL cells (CC+pLLP) are significantly increased in <em>shroom3</em><span class="math inline">\(--\)</span> embryos, while the count of only pLLP cells remains unchanged.</p>

<div class="figure" style="text-align: center"><span id="fig:llclus"></span>
<img src="figures/results/01_morphometrics/ll_clusters.png" alt="LL Morphometrics A individual CC statistics B Sums per embryo. (Bars = median, errorbars = 95% CI)" width="0.7\textwidth" />
<p class="caption">
Figure 3.3: LL Morphometrics <strong>A</strong> individual CC statistics <strong>B</strong> Sums per embryo. (Bars = median, errorbars = 95% CI)
</p>
</div>
</div>
</div>
<div id="proliferation" class="section level2">
<h2><span class="header-section-number">3.2</span> Proliferation</h2>
<p>LL morphometric analysis revealed that deposited CCs in <em>shroom3</em><span class="math inline">\(--\)</span> embryos were on average slightly smaller and had fewer cells incorporated. However, due to the additional two cell clusters deposited, the net count is ~9<span class="math inline">\(\%\)</span> increased at the end of migration.
To test whether this is due to higher proliferative activity a dataset of time-lapse movies (12 h / <span class="math inline">\(\Delta\)</span>T = 6 min.) was generated to count the number of mitoses in a <em>cxcr4b(BAC):H2BRFP</em> transgenic background similar to previous proliferation studies in the pLLP <span class="citation">(<em>23</em>, <em>57</em>)</span>.</p>
<div id="dataset-1" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Dataset</h3>
<p>The dataset consists of 12 <em>shrm3</em>++ and 18 <em>shrm3</em><span class="math inline">\(--\)</span> embryos with about 75 timepoints for the pLLP and 150 for cell clusters. For counting, tracks were created for each proliferative cell on MaxIPs (section <a href="2-mat-met.html#prolif">2.2.3.2</a>). Figure <a href="3-res.html#fig:prolpllp">3.4</a>A shows a <em>shroom3</em>++ and a <em>shroom3</em><span class="math inline">\(--\)</span> LL after 12 h of imaging where each mitotic track is individually colored and overlaid to the image data.</p>
<p>To index the proliferative activity (minutes per mitosis) per embryo, the product of total number of timepoints and <span class="math inline">\(\Delta\)</span>T in minutes was divided by the total number of mitotic events. From this we get to know the average time [min.] during which one mitosis occurs.</p>

</div>
<div id="res-prolpllp" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Lateral Line Mitoses</h3>
<p>Tracking mitoses in the pLLP does not reveal any difference in proliferation from start (~28 hpf) till about mid of migration 12 hours later (<a href="3-res.html#fig:prolpllp">3.4</a>B). For confirmation, this finding was also validated via two more traditional methods. During mitosis genetic material is replicated in S-phase<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a>, while in meta-phase chromosomes are found to be heavily phosphorylated <span class="citation">(<em>58</em>)</span>. With an EdU<a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a> assay <span class="citation">(<em>59</em>)</span> cells in S-phase can be detected (<a href="3-res.html#fig:prolpllp">3.4</a>B’). Using a specific phospho-histone (p-Hist.) antibody cells in meta-phase can be detected. When comparing the relative numbers of EdU resp. p-Hist cells over total pLLP cells, again, no difference in proliferation can be detected. Still, at end of migration the LL system in <em>shroom3</em><span class="math inline">\(--\)</span> embryos does incorporate 9<span class="math inline">\(\%\)</span> more cells.</p>

<div class="figure" style="text-align: center"><span id="fig:prolpllp"></span>
<img src="figures/results/02_proliferation/prol-01.png" alt="pLLP proliferation A Tracks of Mitosis. (Image data) Last timepoint of Z-projected time-lapse movie. (Labels) each dots marks a mitotic events. Each color represents one mitosis. Each connection between two dots represents 6 min. B-B’' Mitoses in the pLLP. (B) Count of Mitoses through time (mean\(\pm\)sd) (B’) Proliferation Index at 36 hpf for EdU and phospho Histone labeled pLLPs (B’‘) Examples of EdU staining (scale bar = 100 \(\mu\)m) C Mitoses in Cell Clusters. B’ and C, the colored bar indicated the median, bars indicate the 95% CI." width="0.85\textwidth" />
<p class="caption">
Figure 3.4: pLLP proliferation <strong>A</strong> Tracks of Mitosis. (Image data) Last timepoint of Z-projected time-lapse movie. (Labels) each dots marks a mitotic events. Each color represents one mitosis. Each connection between two dots represents 6 min. <strong>B-B’'</strong> Mitoses in the pLLP. (B) Count of Mitoses through time (mean<span class="math inline">\(\pm\)</span>sd) (B’) Proliferation Index at 36 hpf for EdU and phospho Histone labeled pLLPs (B’‘) Examples of EdU staining (scale bar = 100 <span class="math inline">\(\mu\)</span>m) <strong>C</strong> Mitoses in Cell Clusters. B’ and C, the colored bar indicated the median, bars indicate the 95% CI.
</p>
</div>
<p>Due to their high regenerative ability, zebrafish are a popular model in regenerative research <span class="citation">(<em>19</em>, <em>22</em>, <em>60</em>)</span>. It is well known that zebrafish NMs have a very high regenerative capacity <span class="citation">(<em>61</em>–<em>63</em>)</span>. Since individual <em>shroom3</em><span class="math inline">\(--\)</span> CCs are smaller, we wondered if there could be compensation mechanisms activated that increase proliferation to restore wildtype CC size once they are deposited. To verify this, CC mitoses were tracked on the same data as before. Other than the pLLP, which can be observed throughout the time-lapse, CCs only begin to <em>exist</em> when they are deposited. To normalize for individual CC lifetimes, the number of mitoses per CC is divided by the duration of the total time-lapse minus the time span to when it first appeared. This way it becomes evident that the additional cells in <em>shroom3</em><span class="math inline">\(--\)</span> embryos are derived from CC proliferation, rather than pLLP proliferation (<a href="3-res.html#fig:prolpllp">3.4</a>C).</p>
</div>
</div>
<div id="rosette-formation-and-cluster-deposition" class="section level2">
<h2><span class="header-section-number">3.3</span> Rosette Formation and Cluster Deposition</h2>
<p>Having shown that the additional CC deposition in <em>shroom3</em><span class="math inline">\(--\)</span> embryos is not caused by an over-proliferation in the migrating pLLP, the next step was to have a closer look at the dynamics of rosette formation, in relation to pLLP morphometrics and CC deposition. To test dependencies between different observations and developmental dynamics, variables can be correlated. To do this and to have a coherent dataset to work with, the data of three different analyses on a single set of image data were merged.
First, to get to know the exact timing of CC deposition, a manual tracking tool <span class="citation">(<em>53</em>)</span> was used (<a href="3-res.html#fig:rdt">3.5</a>, Tracking). Second, to deduce pLLP morphometrics a self-made IJ macro <span class="citation">(<em>64</em>)</span> was developed<a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a> for spatiotemporal registration of the pLLP and to yield information about its speed, area, roundness, etc. (<a href="3-res.html#fig:rdt">3.5</a>, Registration). Third, to detect rosettes and quantify their weights<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a>, a CNN <span class="citation">(<em>65</em>)</span> was used on the registered pLLP output from the anaLLzr2DT (<a href="3-res.html#fig:rdt">3.5</a>, Detection). Finally, all three datasets were merged by a unique identifier for each datapoint by a unique identifier for each embryo and timepoint.</p>
<div id="dataset-2" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Dataset</h3>
<p>The image data set analyzed consists of 20 time-lapse movies (11 <em>shroom3</em><span class="math inline">\(--\)</span>, 9 <em>shroom3</em>++). Each time-lapse has a duration of ~20 h (~8 min. interval)<a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a>, summing up to ~1650 <em>shroom3</em><span class="math inline">\(--\)</span>, and ~1350 <em>shroom3</em>++ timepoints.</p>

<div class="figure" style="text-align: center"><span id="fig:rdt"></span>
<img src="figures/results/03_rosettes/RDT-01.png" alt="Merged Datasets. (Tracking) pLLP is marked as no.1. The rest of the CCs is numbered sequentially as they appear. (Registration) The black outline marks the region of interest (ROI) that is the pLLP as it is detected by the anaLLzr2DT. The red line highlights the pLLPs leading edge. (Detection) Each square highlights a detected rosette by the CNN, colors represent rosette weights." width="0.7\textwidth" />
<p class="caption">
Figure 3.5: Merged Datasets. (Tracking) pLLP is marked as no.1. The rest of the CCs is numbered sequentially as they appear. (Registration) The black outline marks the region of interest (ROI) that is the pLLP as it is detected by the anaLLzr2DT. The red line highlights the pLLPs leading edge. (Detection) Each square highlights a detected rosette by the CNN, colors represent rosette weights.
</p>
</div>
</div>
<div id="cluster-deposition" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Cluster Deposition</h3>
<p>Figure <a href="3-res.html#fig:rdtdepo">3.6</a>A shows a montage of a <em>shroom3</em>++ and a <em>shroom3</em><span class="math inline">\(--\)</span> scenario in cluster deposition (3.5 h / ~20 min. interval). For <em>shroom3</em>++ the rosette structure seems tight and two depositions occur in a regular manner. In the <em>shroom3</em><span class="math inline">\(--\)</span> on the other hand, rosette structure seems more fragile with less pronounced rosette centers and four observed depositions. Interestingly, based on the <em>cldnb:lyn-gfp</em> signal the area of constriction seems to be less radially organized but more oriented towards the horizontal midline. Furthermore, the trailing rosettes are significantly smaller and do not seem to delaminate as clean from the migrating primordium. In addition, for L3 it first seems like two CCs could be deposited, until they merge again to a single CC about 1.5 h later.
On average, as it was shown before (section <a href="3-res.html#res-ccounts">3.1.2</a>), there’s a significant increase in clusters deposited (<a href="3-res.html#fig:rdtdepo">3.6</a>B). Also, neither <em>shroom3</em>++ nor <em>shroom3</em><span class="math inline">\(--\)</span> CCs drastically change their position once they are deposited.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtdepo"></span>
<img src="figures/results/03_rosettes/tracking-01.png" alt="Cluster Deposition A shroom3++ and shroom3\(--\) LL development in comparison. L1 - L4 are deposited CCs. Arrows indicate deposition events. Dotted lines are tracks of rosette to CC transition. B Statistics of deposited cluster counts C Change of CC position through time. Each line represents the locally weighted scatterplot smoothing (LOESS) of all CCn positions observed."  />
<p class="caption">
Figure 3.6: Cluster Deposition <strong>A</strong> <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> LL development in comparison. L1 - L4 are deposited CCs. Arrows indicate deposition events. Dotted lines are tracks of rosette to CC transition. <strong>B</strong> Statistics of deposited cluster counts <strong>C</strong> Change of CC position through time. Each line represents the locally weighted scatterplot smoothing (LOESS) of all CCn positions observed.
</p>
</div>
</div>
<div id="registration-1" class="section level3">
<h3><span class="header-section-number">3.3.3</span> Registration</h3>
<p>As shown in figure<a href="3-res.html#fig:rdtreg">3.7</a>A-A’ neither speed not acceleration drastically differ throughout the complete course of migration. Speed drops from an initial ~75 <span class="math inline">\(\mu\)</span>m/h to about ~30 <span class="math inline">\(\mu\)</span>m/h for both <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> 17 h later (figure 3.3 A). Similarly, while there is a positive acceleration of almost ~2 <span class="math inline">\(\mu\)</span>m/h peaking after ~2-3 hours, the remaining two peaks progressively get smaller (<a href="3-res.html#fig:rdtreg">3.7</a>A’).
While for the area no difference can be detected (<a href="3-res.html#fig:rdtreg">3.7</a>B), interestingly the roundness is on average significantly reduced in <em>shroom3</em><span class="math inline">\(--\)</span> pLLPs (<a href="3-res.html#fig:rdtreg">3.7</a>C). This is also evident from the montage in <a href="3-res.html#fig:rdtdepo">3.6</a>A.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtreg"></span>
<img src="figures/results/03_rosettes/registration.png" alt="pLLP time-resolved morphometrics A-A’ Leading edge (l.e.) speed and acceleration in \(\mu\)m/h, displayed as LOESS curves with at a span of 0.5 B Area in square \(\mu\)m and C Roundness displayed as mean±se (standard error)."  />
<p class="caption">
Figure 3.7: pLLP time-resolved morphometrics <strong>A-A’</strong> Leading edge (l.e.) speed and acceleration in <span class="math inline">\(\mu\)</span>m/h, displayed as LOESS curves with at a span of 0.5 <strong>B</strong> Area in square <span class="math inline">\(\mu\)</span>m and <strong>C</strong> Roundness displayed as mean±se (standard error).
</p>
</div>
</div>
<div id="res-rdtdet" class="section level3">
<h3><span class="header-section-number">3.3.4</span> Rosette Detection</h3>
<p>The kymographs<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a> in figure <a href="3-res.html#fig:rdtdet">3.8</a>A-A’ were generated on pLLPs that were registered in space and time. After registration, a line was drawn along the horizontal midline (<a href="3-res.html#fig:rdtreg">3.7</a>A). Finally, recorded kymographs were turned to false color<a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a> for better visual display. In <em>shroom3</em>++ the higher intensities are highly concentrated to two to three distinct regions within the migrating pLLP, while in the <em>shroom3</em><span class="math inline">\(--\)</span> pLLP the higher intensities are more fragmented and overall reduced. To analyze this statistically all pLLPs were quantified in terms of rosette count and average rosette weight or ‘rosettiness’ (see section <a href="2-mat-met.html#CNN">2.2.3.1</a>).
The numbers reveal two very interesting things. First, the medians show that rosette count (<a href="3-res.html#fig:rdtreg">3.7</a>B’) is only reduced for the first six hours, while rosettiness is reduced almost throughout the entire 20 hours. Second, while for rosette counts two to six indeed half or more of the pLLPs are <em>shroom3</em>++, for counts below two and above six there are more <em>shroom3</em><span class="math inline">\(--\)</span> pLLPs (<a href="3-res.html#fig:rdtreg">3.7</a>B). Furthermore figure @(fig:rdtreg)C shows that at a rosettiness of ~0.7 there are about as many <em>shroom3</em>++ as there are <em>shroom3</em><span class="math inline">\(--\)</span> pLLPs, marking this point as a threshold were pLLPs above are more like likely to be <em>shroom3</em>++ and below more likely to be <em>shroom3</em><span class="math inline">\(--\)</span>. At ~0.1 rosettiness there are almost only <em>shroom3</em><span class="math inline">\(--\)</span> pLLPs, while at ~0.9 rosettiness there are almost only <em>shroom3</em>++ pLLPs.
In addition, the data distributions attached to the sides (<a href="3-res.html#fig:rdtreg">3.7</a>B’’ and C’’) reveal that for the rosettiness the KDE did not actually shifted to a lower number, but rather flattened and variance is increased.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtdet"></span>
<img src="figures/results/03_rosettes/detection.png" alt="Rosette counts and weights. A-A’ pLLP kymographs. Length = 100 \(\mu\)m. (A) shows the signal and line drawn. (A’) shows the signal through time B-B’’ and C-C’’ Different graphic representation of rosette count data. (B) Filled histogram (B’) Median\(\pm\)standard deviation (B’’) KDE."  />
<p class="caption">
Figure 3.8: Rosette counts and weights. <strong>A-A’</strong> pLLP kymographs. Length = 100 <span class="math inline">\(\mu\)</span>m. (A) shows the signal and line drawn. (A’) shows the signal through time <strong>B-B’’ and C-C’’</strong> Different graphic representation of rosette count data. (B) Filled histogram (B’) Median<span class="math inline">\(\pm\)</span>standard deviation (B’’) KDE.
</p>
</div>
</div>
<div id="correlations" class="section level3">
<h3><span class="header-section-number">3.3.5</span> Correlations</h3>
<p>The intent behind generating this combined dataset was to be able to efficiently detect interdependencies between different variables in time and space. The single dots in the scatterplots below represent the variables scatter in <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>, the red line shows a linear model through the point cloud, the red digits are the correlation coefficient. Black spots with grey circles indicate the centers for clusters calculated on these two sets of data. Clustering was performed hierarchical.
For my questioning the most interesting variable to find dependencies with is rosettiness. In figure@ref(fig.rdtcorr)A and B, the <span class="math inline">\(x\)</span> axis is occupied with the median rosettiness of all timepoints. As mentioned in section <a href="1-Intro.html#intro-phen">1.2.5.1</a> when we had a first closer look at the pLLP phenotype we found that a fraction of the clearly as <em>shroom3</em><span class="math inline">\(--\)</span> identified ones still were reminiscent of the <em>shroom3</em>++ phenotype. The hypothesis to test for figure <a href="3-res.html#fig:rdtcorr">3.9</a>A was that higher speeds lead to a lower degree of rosettiness and <em>vice versa</em>. The correlation coefficient and model indicate a rather low interdependence. However, both also showed (<a href="3-res.html#fig:rdtdet">3.8</a>C’ and <a href="3-res.html#fig:rdtreg">3.7</a>A) a high timely dependence, an effect that is lost when reducing the time dimension to the median. Interestingly, however, the line that goes through the centers determined by clustering shows is much steeper and potentially a much higher interdependence.
The hypothesis to test for figure <a href="3-res.html#fig:rdtcorr">3.9</a>B was if lower rosettiness leads to a higher number of CCs deposited and <em>vice versa</em>. Since here the end of migration cluster count is one-dimensional, there is no other way than correlating to a summary statistic rosettiness. Here the correlation coefficient is rather high, the interdependence seems to be true. Additionally, the clusters determined are nicely separated from each other and the <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> pLLPs seem to form two independent groups in this space.</p>

<div class="figure" style="text-align: center"><span id="fig:rdtcorr"></span>
<img src="figures/results/03_rosettes/rdt_corr_speed-ros.png" alt="Developmental interdependencies. Digits = pearson coefficient A Pearson correlation coefficient between rosettiness and speed of migration through time B Correlation between shrm3++ and shrm3\(--\) at end of migration."  />
<p class="caption">
Figure 3.9: Developmental interdependencies. Digits = pearson coefficient <strong>A</strong> Pearson correlation coefficient between rosettiness and speed of migration through time <strong>B</strong> Correlation between shrm3++ and shrm3<span class="math inline">\(--\)</span> at end of migration.
</p>
</div>
</div>
</div>
<div id="apical-constriction-1" class="section level2">
<h2><span class="header-section-number">3.4</span> Apical Constriction</h2>
<p>To gain further insight into rosette assembly and the contribution of apical constriction, it is appropriate to have a closer look at rosette conformation on a cellular scale. To do so, first a set of high spatial resolution, volumetric image data was generated (XY = 0.164 <span class="math inline">\(\mu\)</span>m / pixel; Z = 0.4 <span class="math inline">\(\mu\)</span>m) that allowed for</p>
<ol style="list-style-type: decimal">
<li>a more rigorous inspection of morphological differences, and</li>
<li>automated and unbiased single cell 3D reconstruction (<a href="2-mat-met.html#ACI-singlecell">2.2.3.3.2.1</a>) using a newly developed IJ macro<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a>. Secondly, to investigate the contribution of various single cell morphometrics to rosette formation an app was developed<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a> that allows for a convenient handling of large amounts of retrieved data.</li>
</ol>

<div class="figure" style="text-align: center">
<img src="figures/results/04_constriction/reconstriction_data.png" alt="High resolution, volumetric image data. Upper row shows the fluorescence signal at the membranes. Lower row shows the 3D reconstructed cells. Columns show the same pLLP from different angles."  />
<p class="caption">
(#fig:acdata,)High resolution, volumetric image data. Upper row shows the fluorescence signal at the membranes. Lower row shows the 3D reconstructed cells. Columns show the same pLLP from different angles.
</p>
</div>
<p>Inhibition of Fgfr1 <em>via</em> drug treatment<a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a> results in a concentration dependent loss of morphogenesis and rosette assembly in the pLLP <span class="citation">(<em>12</em>, <em>27</em>, <em>55</em>)</span>. As a proof of principle and to validate previous studies, data of drug treated embryos (20 <span class="math inline">\(\mu\)</span>M) and appropriate DMSO (0.1<span class="math inline">\(\%\)</span> DMSO / E3<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a>) controls were included.</p>
<div id="dataset-3" class="section level3">
<h3><span class="header-section-number">3.4.1</span> Dataset</h3>
<p>The dataset generated consists of three pLLP stages (<a href="3-res.html#fig:acstages">3.10</a>), four groups (DMSO, SU5402, <em>shrm3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span>), 267 pLLPs and 33.163 single cells.</p>

<div class="figure" style="text-align: center"><span id="fig:acstages"></span>
<img src="figures/results/04_constriction/Yolk_ext.png" alt="Scheme showing a part of the posterior part of a zebrafish embryo and recorded developmental stages for 3D reconstruction" width="0.60\textwidth" />
<p class="caption">
Figure 3.10: Scheme showing a part of the posterior part of a zebrafish embryo and recorded developmental stages for 3D reconstruction
</p>
</div>
<p>Use the sample data in the <a href="https://dskleinhans.shinyapps.io/LLmapR/">LLMapR</a> web-app to explore the dataset.</p>
</div>
<div id="cell-shape-and-arrangement" class="section level3">
<h3><span class="header-section-number">3.4.2</span> Cell Shape and Arrangement</h3>
<p>Figure <a href="3-res.html#fig:acshape">3.11</a>B’ illustrates the three developmental stages used here. For <em>shrm3</em>++ pLLPs two (<a href="3-res.html#fig:acshape">3.11</a>A, 32 hpf) to four (<a href="3-res.html#fig:acshape">3.11</a>A, 40 hpf) areas of constricting regions can be observed (XZ pane), in <em>shroom3</em><span class="math inline">\(--\)</span> its three (<a href="3-res.html#fig:acshape">3.11</a>A, 36 hpf) to seven (<a href="3-res.html#fig:acshape">3.11</a>A, 40 hpf), confirming the results of rosette detection (<a href="3-res.html#res-rdtdet">3.3.4</a>). While at 32 and 36 hpf the spread of those regions is wider in <em>shroom3</em><span class="math inline">\(--\)</span>, at 40 hpf they are smaller but also increased in number. When looking at the central rosette along the width (YZ pane), it seems like there is no considerable difference between <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> rosettes. In SU5402 treated embryos, neither in XZ nor in YZ constriction can be observed and the cells seem to be mostly retained in their mesenchymal character and columnar shape. In addition, while constriction seems isotropic in the shroom3++ rosette center, constriction in the <em>shroom3</em><span class="math inline">\(--\)</span> rosette center cells appears anisotropic.</p>

<div class="figure" style="text-align: center"><span id="fig:acshape"></span>
<img src="figures/results/04_constriction/Figure_5-1.png" alt="Apical Constriction in 3D. The stages are in columns, the groups are in rows. Each panel shows a MaxIP (top left), and orthogonal views along the length (XZ, bottom left) and along the width (YZ, right). Arrow annotations in YZ panels indicate points of constriction, bar annotations in XZ indicate the spread of region of constriction."  />
<p class="caption">
Figure 3.11: Apical Constriction in 3D. The stages are in columns, the groups are in rows. Each panel shows a MaxIP (top left), and orthogonal views along the length (XZ, bottom left) and along the width (YZ, right). Arrow annotations in YZ panels indicate points of constriction, bar annotations in XZ indicate the spread of region of constriction.
</p>
</div>
</div>
<div id="single-cell-metrics" class="section level3">
<h3><span class="header-section-number">3.4.3</span> Single Cell Metrics</h3>
<p>To put the above mentioned in numbers and to gain a more objective view on cell geometry, apical constriction and other metrics such as height were quantified . To get a hold on AC anisotropy we measured the length of the minor and major axis of a fit oval (<a href="3-res.html#fig:acaci">3.12</a>A). If AC was anisotropic and more cells are oriented towards the horizontal midline instead of being radially oriented, this could be shown by degrees of divergence (<span class="math inline">\(\measuredangle\)</span>) from the horizontal midline (<a href="3-res.html#fig:acshape">3.11</a>A’).
For angle measurements, all angles are normalized to a 90<span class="math inline">\(^\circ\)</span> range, where 0<span class="math inline">\(^\circ\)</span> is along the horizontal midline. When comparing the ratio of cells in intervals of 15<span class="math inline">\(^\circ\)</span> along a range of 90<span class="math inline">\(^\circ\)</span> it can be shown that there are more <em>shroom3</em><span class="math inline">\(--\)</span> than <em>shroom3</em>++ cells within the 0-15<span class="math inline">\(^\circ\)</span> interval through all three stages (<a href="3-res.html#fig:acstages">3.10</a>). Even more, it shows that there are more <em>shroom3</em>++ than <em>shroom3</em><span class="math inline">\(--\)</span> cells within the remaining intervals – confirming the model of constriction anisotropy.
While reduction in ACI Major is stronger than the ACI Minor in shroom3<span class="math inline">\(--\)</span> cells (<a href="3-res.html#fig:acaci">3.12</a>C-C’), both are significantly reduced throughout time – again confirming the increase in constriction anisotropy. The ACI for SU5402 treated embryos depicts the base value for both Major and Minor. For all three timepoints, this value stays at the same level. Interestingly, the ACI measurements for all other measurements are progressively approximating this level.</p>

<div class="figure" style="text-align: center"><span id="fig:acaci"></span>
<img src="figures/results/04_constriction/Figure_5-2.png" alt="3D metrics and pLLP maps. A-A’ Measurements. (A) Oval (black outline) represents the fit ellipse. Red arrows represent major and minor axes which are representative for A-P and D-V respectively (A’) Angle measurements and intervals B Cellular orientation as bar chart. Statistics are indicated as stars. C-C’ Apical Constriction measurements. Colored bars represent the median. Errorbars indicate 95% CI."  />
<p class="caption">
Figure 3.12: 3D metrics and pLLP maps. <strong>A-A’</strong> Measurements. (A) Oval (black outline) represents the fit ellipse. Red arrows represent major and minor axes which are representative for A-P and D-V respectively (A’) Angle measurements and intervals <strong>B</strong> Cellular orientation as bar chart. Statistics are indicated as stars. <strong>C-C’</strong> Apical Constriction measurements. Colored bars represent the median. Errorbars indicate 95% CI.
</p>
</div>
</div>
</div>
<div id="res-hc" class="section level2">
<h2><span class="header-section-number">3.5</span> Haircell Specification</h2>
<p>Due to the rather strong cellular phenotype and overall smaller CCs we were interested if hair-cell specification would still take place and therefore if the LLs function as a sensory system would still be intact. First, to check if <em>atoh1a</em> was still expressed in <em>shroom3</em><span class="math inline">\(--\)</span> cells and if there were further implications in feedback loops with Notch signaling, an <em>In Situ Hybridization</em> (ISH) experiment was conducted (probes can be found at section <a href="2-mat-met.html#mat-probes">2.1.8</a>). For 36 and 40 hpf the count of <em>atoh1a</em> and <em>deltaD</em> expressing cells (<a href="3-res.html#fig:hcish">3.13</a>) seemed to be increased. Furthermore, especially at 40 hpf, the signal seems to be a lot more fragmented and fuzzier.
Traditional ISH allows for extremely sensitive detection of RNA transcripts on a whole embryo scale via a hybridizing anti-sense probe (section <a href="2-mat-met.html#ISH-met">2.2.2.1</a>). However, since embryos are fixed, it only allows to analyze single time points. Furthermore, since images are taken in brightfield, there is a problem of background signal and quantification of intensity.</p>

<div class="figure" style="text-align: center"><span id="fig:hcish"></span>
<img src="figures/results/05_atoh/hc_ish.png" alt="Expression of deltaD and atoh1a in the pLLP. Recorded in greyscale at brightfield at 20X Magnification. Images show background subtracted and de-noised EDFs of original Z-stack data."  />
<p class="caption">
Figure 3.13: Expression of <em>deltaD</em> and <em>atoh1a</em> in the pLLP. Recorded in greyscale at brightfield at 20X Magnification. Images show background subtracted and de-noised EDFs of original Z-stack data.
</p>
</div>
<p>To get access to developmental dynamics we made use of an additional, besides <em>cldnb:lyn-gfp</em>, transgenic construct where tdTomato<a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a> <span class="citation">(<em>66</em>)</span> is expressed under direct control of the <em>atoh1a</em> promotor. By simultaneous observation of both fluorophores in a time-lapse setup (<a href="3-res.html#fig:hctl">3.14</a>B), the strategy was to follow up the dynamics of tom expression to count quantities and to measure the strength of expression by measuring signal intensity.</p>
<div id="dataset-4" class="section level3">
<h3><span class="header-section-number">3.5.1</span> Dataset</h3>
<p>The dataset generated consists of two groups, <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span>, where the segmented lateral line images are split into pLLP and CC (section <a href="#mat-ato-set"><strong>??</strong></a>). Each time-lapse movie has a total duration of 18-20 hours, consists of about 54 timepoints and two channels (488 &amp; 561 nm).</p>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:hcdatatab">Table 3.1: </span>Haircell specification dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
id labels
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
metrics
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
genotype
</th>
<th style="text-align:left;">
sub-structure
</th>
<th style="text-align:center;">
embryos
</th>
<th style="text-align:center;">
timepoints
</th>
<th style="text-align:center;">
per 3h
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
shroom3++
</td>
<td style="text-align:left;">
pLLP
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
1566
</td>
<td style="text-align:center;">
261
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
2499
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
shroom3
</td>
<td style="text-align:left;">
pLLP
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
1134
</td>
<td style="text-align:center;">
189
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
3566
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
</tbody>
</table></li>
</ul></li>
</ul>
</div>
<div id="atoh1a-in-the-ll" class="section level3">
<h3><span class="header-section-number">3.5.2</span> Atoh1a in the LL</h3>
<p>While in both <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> Tom signal intensities are rising as soon as CCs are deposited (<a href="3-res.html#fig:hctl">3.14</a>A), in <em>shroom3</em><span class="math inline">\(--\)</span> embryos intensities are on the rise in the pLLP already in the first four hours of migration. When comparing signal intensities in deposited CCs for both, <em>shroom3</em>++ and <em>shroom3</em><span class="math inline">\(--\)</span> show a similar pattern of decreasing Tom signal intensity with progressive depositions. Similarly, when looking ungrouped overall development in CC signal intensity (<a href="3-res.html#fig:hctl">3.14</a>B), no difference can be detected.
While there is no difference in relative numbers of <em>atoh1a</em> expressing cells within the migrating pLLP throughout time (<a href="3-res.html#fig:hctl">3.14</a>C), when comparing mean cell counts at specific intervals the numbers reveal a significant difference from 0 – 3 and 12 – 15 hours (<a href="3-res.html#fig:hctl">3.14</a>C’). The extrusions to the left in the kymographs in figure <a href="3-res.html#fig:hctl">3.14</a>D are representative to the elongating pLLP during the process of CC deposition. Magenta curves represent expression of <em>atoh1a</em>. For the <em>shroom3</em><span class="math inline">\(--\)</span> pLLP, premature expression of <em>atoh1a</em> can be detected. The diagonal lines in the kymographs in figure <a href="3-res.html#fig:hctl">3.14</a>D’ represent the moving pLLP along the LL. Vertical curves represent deposited CCs. As for the registered pLLP, Magenta curves represent expression of <em>atoh1a</em>.</p>

<div class="figure" style="text-align: center"><span id="fig:hctl"></span>
<img src="figures/results/05_atoh/Figure_7-white-01.png" alt="Haircell specification in the LL. A and B maximum Tom intensities in the pLLP and CCs. All CC lifetimes were normalized to the timepoint of deposition (timepoint zero, highlighted by striped vertical line). pLLP intensities are shows to the right, CC intensities are shows on a negative scale to the left. C Relative numbers of atoh1a expressing cells and C’ mean counts of atoh1a expressing cells. D-D’ Kymographs along the horizontal midline showing nascent signal of Tom (in magenta) D two individual pLLPs that were registered in time and to the leading edge. And D’ Tom signal during LL development."  />
<p class="caption">
Figure 3.14: Haircell specification in the LL. <strong>A and B</strong> maximum Tom intensities in the pLLP and CCs. All CC lifetimes were normalized to the timepoint of deposition (timepoint zero, highlighted by striped vertical line). pLLP intensities are shows to the right, CC intensities are shows on a negative scale to the left. <strong>C</strong> Relative numbers of <em>atoh1a</em> expressing cells and <strong>C’</strong> mean counts of <em>atoh1a</em> expressing cells. <strong>D-D’</strong> Kymographs along the horizontal midline showing nascent signal of Tom (in magenta) <strong>D</strong> two individual pLLPs that were registered in time and to the leading edge. And <strong>D’</strong> Tom signal during LL development.
</p>
</div>
</div>
</div>
<div id="rescue-experiments" class="section level2">
<h2><span class="header-section-number">3.6</span> Rescue Experiments</h2>
<p>Rescue experiments are focused on LL CC count and pLLP rosettiness. Since Shroom3 affects development not linearly as e.g. a simple signaling molecule, but by providing a scaffold where different dimensions can act together, we tried rescuing by manipulating those single dimensions to find out which one had the most impact.</p>
<div id="res-shrmresc" class="section level3">
<h3><span class="header-section-number">3.6.1</span> Shroom3 Ectopic Expression</h3>
<p>To find out if the LL phenotype can be rescued by ectopic expression of Shroom3, a transgene was used where <em>shroom3v1</em> is fused to RFP and expressed under control of a heatshock protein<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a>. While this construct was used for rescue before <span class="citation">(<em>29</em>)</span> (see section <a href="1-Intro.html#intro-shroom">1.2.4.1</a>), here it was used to test for CC count and rosettiness too. Rescuing the <em>shroom3</em><span class="math inline">\(--\)</span> phenotype with Shroom3 itself is interesting to control for LL unspecific effects that might have happened earlier in development and to find out about Shroom3s morphogenic potential.
For this experiment, embryos were imaged two times. First at around 36 hpf, and a second time at end of migration after about 24 hours of incubation at 28.5<span class="math inline">\(^\circ\)</span>. This way it is possible to test for correlations between rosettiness earlier in development and CC count at end of migration.
While CC count is restored in heat-shocked, <em>shroom3</em><span class="math inline">\(--\)</span>, transgene carrying embryos, it also seems to reduce the number of CCs deposited in <em>shroom3</em>++ (<a href="3-res.html#fig:rescshrm">3.15</a>A). For the rosettiness however, we still get a significant difference (<a href="3-res.html#fig:rescshrm">3.15</a>B). Correlation between CC count and rosettiness for heat-shocked embryos without <em>hs:shr3v1</em> transgene (<a href="3-res.html#fig:rescshrm">3.15</a>C, <em>hs:shr3v1</em> 00) is about the same as for embryos not heat-shocked (<a href="3-res.html#fig:rescshrm">3.15</a>C, <em>hs:shr3v1</em> ??), but inverse to heat-shocked embryos with transgene (<a href="3-res.html#fig:rescshrm">3.15</a>C, <em>hs:shr3v1</em> +?). Importantly, correlation is lost between heat-shocked <em>shroom3</em><span class="math inline">\(--\)</span> embryos with transgene and not heat-shocked <em>shroom3</em>++ embryos (<a href="3-res.html#fig:rescshrm">3.15</a>C, right panel). Table 3 summarizes the number of datapoints.</p>

<div class="figure" style="text-align: center"><span id="fig:rescshrm"></span>
<img src="figures/results/06_rescues/shrm3/rescue_temp.png" alt="shroom3 ectopic expression A Normalized cluster count and statistics B Rosettiness and statistics C Correlations between beginning of migration rosettiness and end of migration cluster count"  />
<p class="caption">
Figure 3.15: <em>shroom3</em> ectopic expression <strong>A</strong> Normalized cluster count and statistics <strong>B</strong> Rosettiness and statistics <strong>C</strong> Correlations between beginning of migration rosettiness and end of migration cluster count
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescshrmtab">Table 3.2: </span>Heatshock rescue dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
id labels
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
metrics
</div>
</th>
</tr>
<tr>
<th style="text-align:right;">
treatment
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
transgene
</th>
<th style="text-align:center;">
CCs
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;vertical-align: top !important;" rowspan="4">
hs
</td>
<td style="text-align:center;">
++ / +-
</td>
<td style="text-align:center;">
Yes
</td>
<td style="text-align:center;">
232
</td>
<td style="text-align:center;">
153
</td>
</tr>
<tr>
<td style="text-align:center;">
++ / +-
</td>
<td style="text-align:center;">
No
</td>
<td style="text-align:center;">
320
</td>
<td style="text-align:center;">
</td>
</tr>
<tr>
<td style="text-align:center;">
–
</td>
<td style="text-align:center;">
Yes
</td>
<td style="text-align:center;">
175
</td>
<td style="text-align:center;">
</td>
</tr>
<tr>
<td style="text-align:center;">
–
</td>
<td style="text-align:center;">
No
</td>
<td style="text-align:center;">
202
</td>
<td style="text-align:center;">
</td>
</tr>
<tr>
<td style="text-align:right;vertical-align: top !important;" rowspan="2">
nhs
</td>
<td style="text-align:center;">
++ / +-
</td>
<td style="text-align:center;">
Unknown
</td>
<td style="text-align:center;">
543
</td>
<td style="text-align:center;">
127
</td>
</tr>
<tr>
<td style="text-align:center;">
–
</td>
<td style="text-align:center;">
Unknown
</td>
<td style="text-align:center;">
354
</td>
<td style="text-align:center;">
</td>
</tr>
</tbody>
</table>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescshrmsignif">Table 3.3: </span>Heatshock statistics
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
id labels
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
cluster counts
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
rosettiness
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
adj.p
</th>
<th style="text-align:center;">
level
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
adj.p
</th>
<th style="text-align:center;">
level
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="5">
hs; –; tg+?
</td>
<td style="text-align:left;width: 2cm; ">
hs; –; tg00
</td>
<td style="text-align:center;">
0.01870
</td>
<td style="text-align:center;">
0.09300
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
0.00012
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
hs; +?; tg+?
</td>
<td style="text-align:center;width: 2cm; ">
0.00470
</td>
<td style="text-align:center;">
0.03300
</td>
<td style="text-align:center;">
**
</td>
<td style="text-align:center;">
0.36735
</td>
<td style="text-align:center;">
0.73000
</td>
<td style="text-align:center;">
ns
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
hs; +?; tg00
</td>
<td style="text-align:center;width: 2cm; ">
0.51503
</td>
<td style="text-align:center;">
0.60000
</td>
<td style="text-align:center;">
ns
</td>
<td style="text-align:center;">
0.00441
</td>
<td style="text-align:center;">
0.04000
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; –; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.00000
</td>
<td style="text-align:center;">
0.00002
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.00673
</td>
<td style="text-align:center;">
0.04700
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; +?; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.05087
</td>
<td style="text-align:center;">
0.15000
</td>
<td style="text-align:center;">
ns
</td>
<td style="text-align:center;">
0.02962
</td>
<td style="text-align:center;">
0.18000
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="4">
hs; –; tg00
</td>
<td style="text-align:left;width: 2cm; ">
hs; +?; tg+?
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr></li>
</ul>
<td style="text-align:left;width: 2cm; ">
hs; +?; tg00
</td>
<td style="text-align:center;width: 2cm; ">
0.00208
</td>
<td style="text-align:center;">
0.01900
</td>
<td style="text-align:center;">
**
</td>
<td style="text-align:center;">
0.00584
</td>
<td style="text-align:center;">
0.04700
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; –; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.02068
</td>
<td style="text-align:center;">
0.09300
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
0.03663
</td>
<td style="text-align:center;">
0.18000
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr></li>
</ul></li>
</ul>
<td style="text-align:left;width: 2cm; ">
nhs; +?; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.30244
</td>
<td style="text-align:center;">
0.60000
</td>
<td style="text-align:center;">
ns
</td>
<td style="text-align:center;">
0.00002
</td>
<td style="text-align:center;">
0.00023
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="3">
hs; +?; tg+?
</td>
<td style="text-align:left;width: 2cm; ">
hs; +?; tg00
</td>
<td style="text-align:center;">
0.01018
</td>
<td style="text-align:center;">
0.06100
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
0.00008
</td>
<td style="text-align:center;">
0.00092
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr></li>
</ul>
<td style="text-align:left;width: 2cm; ">
nhs; –; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.00014
</td>
<td style="text-align:center;">
0.00160
</td>
<td style="text-align:center;">
***
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; +?; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.00086
</td>
<td style="text-align:center;">
0.00860
</td>
<td style="text-align:center;">
***
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;width: 2cm; " rowspan="2">
hs; +?; tg00
</td>
<td style="text-align:left;width: 2cm; ">
nhs; –; tg??
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.66817
</td>
<td style="text-align:center;">
0.73000
</td>
<td style="text-align:center;">
ns
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; +?; tg??
</td>
<td style="text-align:center;width: 2cm; ">
0.00206
</td>
<td style="text-align:center;">
0.01900
</td>
<td style="text-align:center;">
**
</td>
<td style="text-align:center;">
0.08040
</td>
<td style="text-align:center;">
0.24000
</td>
<td style="text-align:center;">
ns
</td>
</tr>
<tr>
<td style="text-align:left;width: 2cm; ">
nhs; –; tg??
</td>
<td style="text-align:left;width: 2cm; ">
nhs; +?; tg??
</td>
<td style="text-align:center;">
0.00002
</td>
<td style="text-align:center;">
0.00024
</td>
<td style="text-align:center;">
****
</td>
<td style="text-align:center;">
0.05789
</td>
<td style="text-align:center;">
0.23000
</td>
<td style="text-align:center;">
ns
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="morphogen-rescue" class="section level3">
<h3><span class="header-section-number">3.6.2</span> Morphogen Rescue</h3>
<p>It is known that increasing concentrations of SU5402 inhibit pLLP migration and rosette formation, leading to a reduced length of the LL system and the number of CCs deposited <span class="citation">(<em>12</em>, <em>27</em>)</span>.
The idea for this experiment was to rescue the <em>shroom3</em> phenotype by combining it with the phenotype of impaired Fgf signaling. Since inhibition of Fgf also reduces LL length, a titration experiment was planned to find a concentration that would not affect the length of the LL but still restore <em>shroom3</em>++ levels of CC count. While concentrations 0.10 and 0.20 <span class="math inline">\(\mu\)</span>M do not show a significant reduction in LL length, they also do not exhibit any difference in cell count.</p>

<div class="figure" style="text-align: center"><span id="fig:rescsu"></span>
<img src="figures/results/06_rescues/su54/rescue_su.png" alt="Titration of Fgf inhibitor A LL lengths are reduced with increased concentrations of Fgf inhibitor B Groupwise comparison of the length normalized cluster count per concentration of Fgf inhibitor."  />
<p class="caption">
Figure 3.16: Titration of Fgf inhibitor <strong>A</strong> LL lengths are reduced with increased concentrations of Fgf inhibitor <strong>B</strong> Groupwise comparison of the length normalized cluster count per concentration of Fgf inhibitor.
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescsutab">Table 3.4: </span>Morphogen rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;vertical-align: top !important;" rowspan="6">
++
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;vertical-align: top !important;" rowspan="6">

</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:center;">
8
</td>
</tr>
<tr>
<td style="text-align:center;">
0.10
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
15
</td>
</tr>
<tr>
<td style="text-align:center;">
0.25
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
21
</td>
</tr>
<tr>
<td style="text-align:center;">
0.40
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
23
</td>
</tr>
<tr>
<td style="text-align:center;">
0.50
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
4
</td>
</tr>
<tr>
<td style="text-align:center;">
1.00
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
10
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">Note: </span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO
</td>
</tr>
</tfoot>
</table>
</div>
<div id="res-rockresc" class="section level3">
<h3><span class="header-section-number">3.6.3</span> Mechanic Rescue</h3>
<p>It is very well known that Rho kinases are necessary for mechanic force transmission in various developmental processes <span class="citation">(<em>9</em>, <em>33</em>, <em>38</em>, <em>67</em>, <em>68</em>)</span>. Since Rock is also thought to be a key player of the Shroom force mediation pathway in the pLLP, the idea for a mechanic rescue was to titrate different concentrations of a Rock inhibitor<a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a> and an inductor<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a> to see if this way the <em>shroom3</em><span class="math inline">\(--\)</span> phenotype can be mimicked in <em>shroom</em>++ embryos resp. restored in <em>shroom3</em><span class="math inline">\(--\)</span>.</p>
<div id="rock-inhibition" class="section level4">
<h4><span class="header-section-number">3.6.3.1</span> Rock inhibition</h4>
<p>Embryos were incubated at distinct (figure <a href="3-res.html#fig:rescrock">3.17</a> legend) concentrations from 24 hpf till end of migration (about 60 hpf). At this stage embryos incubated at higher concentrations displayed a significant reduction in overall body length (<a href="3-res.html#fig:rescrock">3.17</a>A’). Therefore, to see if there is a lateral line specific effect on LL length, LL length was normalized to body length. Rockout treatment at 10 <span class="math inline">\(\mu\)</span>M does not lead to a significant reduction in LL length but does significantly increase the number of cell clusters deposited.</p>

<div class="figure" style="text-align: center"><span id="fig:rescrock"></span>
<img src="figures/results/06_rescues/rockout/rescue_rockout.png" alt="Titration of Rock inhibitor A LL lengths are reduced with increased concentrations of Rock inhibitor B Groupwise comparison of the length normalized cluster count per concentration of Fgf inhibitor."  />
<p class="caption">
Figure 3.17: Titration of Rock inhibitor <strong>A</strong> LL lengths are reduced with increased concentrations of Rock inhibitor <strong>B</strong> Groupwise comparison of the length normalized cluster count per concentration of Fgf inhibitor.
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescrocktab">Table 3.5: </span>Rockout rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:right;">
genotype
</th>
<th style="text-align:center;">
conc. (<span class="math inline">\(\mu\)</span>M)
</th>
<th style="text-align:left;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
++
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:left;">
17
</td>
</tr>
<tr>
<td style="text-align:right;">
++
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:left;">
13
</td>
</tr>
<tr>
<td style="text-align:right;">
++
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:left;">
17
</td>
</tr>
<tr>
<td style="text-align:right;">
++
</td>
<td style="text-align:center;">
100
</td>
<td style="text-align:left;">
20
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">Note: </span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO
</td>
</tr>
</tfoot>
</table>
</div>
<div id="rock-induction" class="section level4">
<h4><span class="header-section-number">3.6.3.2</span> Rock induction</h4>
<p>The idea for this experiment was to rescue <em>shroom3</em><span class="math inline">\(--\)</span> embryos by enhancing the effects of Rock. Incubation at 100 nM, the maximum solubility, does not lead any significant differences.</p>

<div class="figure" style="text-align: center"><span id="fig:resccal"></span>
<img src="figures/results/06_rescues/caliculyn/rescue_cali.png" alt="Calyculin treatment (A) (B)" width="0.85\textwidth" />
<p class="caption">
Figure 3.18: Calyculin treatment (A) (B)
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resccaltab">Table 3.6: </span>Calyculin rescue dataset summary
</caption>
<thead>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (nM)
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
conc. (nM)
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
++
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8
</td>
</tr>
<tr>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
100
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
100
</td>
<td style="text-align:center;">
12
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">Note: </span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> concentrations in 0.1% DMSO
</td>
</tr>
</tfoot>
</table>
</div>
</div>
<div id="res-tempresc" class="section level3">
<h3><span class="header-section-number">3.6.4</span> Thermodynamic Rescue</h3>
<p>Increasing temperature usually leads to an acceleration in thermodynamics and embryonic development, while lower temperatures decelerate <span class="citation">(<em>69</em>)</span>. Decelerated development also leads to a reduction in speed of migration and therefore to a reduction in forces that act on cells, cell-cell junctions and organ structures. In this experiment the hypothesis was that it should be possible to at least partially restore the <em>shroom3</em>++ number of CCs since reduced forces would stabilize rosettes and therefore rosettes would have longer time to develop properly. Since embryos develop at different speeds, it is more challenging to exactly stage match them at end of migration. To account for this the CC count was normalized to LL length.
Even though the CC count in <em>shroom3</em><span class="math inline">\(--\)</span> embryos incubated at 25<span class="math inline">\(^\circ\)</span>C cannot be completely restored, they are hardly more different than levels of <em>shroom3</em>++ embryos incubated at 30<span class="math inline">\(^\circ\)</span>C.</p>

<div class="figure" style="text-align: center"><span id="fig:resctemp"></span>
<img src="figures/results/06_rescues/temp/rescue_temp.png" alt="Thermodynamic rescue (A) (B)" width="0.75\textwidth" />
<p class="caption">
Figure 3.19: Thermodynamic rescue (A) (B)
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resctemptab">Table 3.7: </span>Thermodynamic rescue dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
shroom3++
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
shroom3–
</div>
</th>
</tr>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
treatment
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
treatment
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
++
</td>
<td style="text-align:center;">
25°C
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
25°C
</td>
<td style="text-align:center;">
19
</td>
</tr>
<tr>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
30°C
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
30°C
</td>
<td style="text-align:center;">
20
</td>
</tr>
</tbody>
</table>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:resctempsignif">Table 3.8: </span>Thermodynamic rescue statistics
</caption>
<thead>
<tr>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
p.adj
</th>
<th style="text-align:center;">
p.signif
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="3">
T25_wt
</td>
<td style="text-align:left;">
T25_hom
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
T30_wt
</td>
<td style="text-align:center;">
0.00747
</td>
<td style="text-align:center;">
0.00750
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;">
T30_hom
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="2">
T25_hom
</td>
<td style="text-align:left;">
T30_wt
</td>
<td style="text-align:center;">
0.00104
</td>
<td style="text-align:center;">
0.00210
</td>
<td style="text-align:center;">
**
</td>
</tr>
<tr>
<td style="text-align:left;">
T30_hom
</td>
<td style="text-align:center;">
0.00015
</td>
<td style="text-align:center;">
0.00046
</td>
<td style="text-align:center;">
***
</td>
</tr>
<tr>
<td style="text-align:left;">
T30_wt
</td>
<td style="text-align:left;">
T30_hom
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
****
</td>
</tr>
</tbody>
</table>
</div>
<div id="atoh1a-knockdown" class="section level3">
<h3><span class="header-section-number">3.6.5</span> atoh1a knockdown</h3>
<p>Referring back to the results we got from the hair cell analysis in section <a href="3-res.html#res-hc">3.5</a>, the hypothesis to test was if premature CC deposition in <em>shroom3</em><span class="math inline">\(--\)</span> embryos is determined by a premature rise in Atoh1a signaling. To inhibit Atoh1a, a morpholino (MO) for <em>atoh1a</em> mRNA was injected at 0.4 ng/mL, a concentration that had already been shown to be sufficient to deplete Atoh1a signaling <span class="citation">(<em>27</em>)</span>. Furthermore, since many MOs have been shown to induce cell death <span class="citation">(<em>70</em>)</span>, in a second group we co-injected embryos with an MO for p53<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a>. Interestingly injection of <em>atoh1a</em> MO leads to a loss of the first CC deposited usually deposited at around 400 <span class="math inline">\(\mu\)</span>m distance (<a href="3-res.html#fig:rescato">3.20</a>A). Furthermore, while median CC counts in <em>atoh1a</em> MO injected <em>shroom3</em><span class="math inline">\(--\)</span> embryos are actually restored to <em>shroom3</em>++ levels, CC counts in <em>shroom3</em>++ embryos are reduced as well.</p>

<div class="figure" style="text-align: center"><span id="fig:rescato"></span>
<img src="figures/results/06_rescues/atoh1a/rescue_atoh.png" alt="atoh1a knockdown (A) (B)"  />
<p class="caption">
Figure 3.20: atoh1a knockdown (A) (B)
</p>
</div>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 11px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescatotab">Table 3.9: </span>Atoh rescue dataset summary
</caption>
<thead>
<tr>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
shroom3++
</div>
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; font-weight: bold; " colspan="3">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
shroom3–
</div>
</th>
</tr>
<tr>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
group
</th>
<th style="text-align:center;">
LLs
</th>
<th style="text-align:center;">
genotype
</th>
<th style="text-align:center;">
group
</th>
<th style="text-align:center;">
LLs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
++
</td>
<td style="text-align:center;">
p53
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">

</td>
<td style="text-align:center;">
p53
</td>
<td style="text-align:center;">
22
</td>
</tr>
<tr>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
p53+atoh1a
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
</td>
<td style="text-align:center;">
p53+atoh1a
</td>
<td style="text-align:center;">
27
</td>
</tr>
</tbody>
</table>
<table class="table table" style="margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:rescatosignif">Table 3.10: </span>Atoh1a MO rescue statistics
</caption>
<thead>
<tr>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:center;">
p
</th>
<th style="text-align:center;">
p.adj
</th>
<th style="text-align:center;">
p.format
</th>
<th style="text-align:left;">
p.signif
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="3">
wt_p53
</td>
<td style="text-align:left;">
wt_p53-ato
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
hom_p53
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
hom_p53-ato
</td>
<td style="text-align:center;">
0.26803
</td>
<td style="text-align:center;">
0.27000
</td>
<td style="text-align:center;">
0.26800
</td>
<td style="text-align:left;">
ns
</td>
</tr>
<tr>
<td style="text-align:left;vertical-align: top !important;" rowspan="2">
wt_p53-ato
</td>
<td style="text-align:left;">
hom_p53
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:center;">
0.00000
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
hom_p53-ato
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:center;">
0.00002
</td>
<td style="text-align:center;">
0.00001
</td>
<td style="text-align:left;">
****
</td>
</tr>
<tr>
<td style="text-align:left;">
hom_p53
</td>
<td style="text-align:left;">
hom_p53-ato
</td>
<td style="text-align:center;">
0.00466
</td>
<td style="text-align:center;">
0.00930
</td>
<td style="text-align:center;">
0.00470
</td>
<td style="text-align:left;">
**
</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="19">
<li id="fn19"><p>in Maximum intensity Z projections (MaxIP)<a href="3-res.html#fnref19" class="footnote-back">↩</a></p></li>
<li id="fn20"><p>fixation with PFA may introduce a slight bending of the embryo<a href="3-res.html#fnref20" class="footnote-back">↩</a></p></li>
<li id="fn21"><p>When mounted, all embryos have a different tilt<a href="3-res.html#fnref21" class="footnote-back">↩</a></p></li>
<li id="fn22"><p>‘S‘ like ‚Synthesis‘<a href="3-res.html#fnref22" class="footnote-back">↩</a></p></li>
<li id="fn23"><p>5-Ethynyl-2’-deoxyuridine<a href="3-res.html#fnref23" class="footnote-back">↩</a></p></li>
<li id="fn24"><p>anaLLzr2DT - <a href="https://github.com/KleinhansDa/anaLLzr2DT" class="uri">https://github.com/KleinhansDa/anaLLzr2DT</a><a href="3-res.html#fnref24" class="footnote-back">↩</a></p></li>
<li id="fn25"><p>as a decimal from 0-1 (1 being a wild-type rosette). For further info see section <a href="2-mat-met.html#CNN">2.2.3.1</a><a href="3-res.html#fnref25" class="footnote-back">↩</a></p></li>
<li id="fn26"><p>for further details about dataset acquisition see section @(refdetect-data)<a href="3-res.html#fnref26" class="footnote-back">↩</a></p></li>
<li id="fn27"><p>a kymograph is a tool to record position over time. Here, its position of fluorescence signal intensities.<a href="3-res.html#fnref27" class="footnote-back">↩</a></p></li>
<li id="fn28"><p>blue = low intensity, red = high intensity<a href="3-res.html#fnref28" class="footnote-back">↩</a></p></li>
<li id="fn29"><p>anaLLzr3D - <a href="https://github.com/KleinhansDa/anaLLzr3D" class="uri">https://github.com/KleinhansDa/anaLLzr3D</a><a href="3-res.html#fnref29" class="footnote-back">↩</a></p></li>
<li id="fn30"><p>LLMapR - <a href="https://dskleinhans.shinyapps.io/LLmapR/" class="uri">https://dskleinhans.shinyapps.io/LLmapR/</a><a href="3-res.html#fnref30" class="footnote-back">↩</a></p></li>
<li id="fn31"><p>Drug name = SU-5402 (section <a href="2-mat-met.html#mat-chem">2.1.1</a>)<a href="3-res.html#fnref31" class="footnote-back">↩</a></p></li>
<li id="fn32"><p>‘Embryo Medium 3’, standard zebrafish embryo incubation medium<a href="3-res.html#fnref32" class="footnote-back">↩</a></p></li>
<li id="fn33"><p>Short ‚Tom‘<a href="3-res.html#fnref33" class="footnote-back">↩</a></p></li>
<li id="fn34"><p>In this case ‚hsp70‘<a href="3-res.html#fnref34" class="footnote-back">↩</a></p></li>
<li id="fn35"><p>Rockout (section <a href="2-mat-met.html#mat-chem">2.1.1</a>)<a href="3-res.html#fnref35" class="footnote-back">↩</a></p></li>
<li id="fn36"><p>Calyculin (section <a href="2-mat-met.html#mat-chem">2.1.1</a>)<a href="3-res.html#fnref36" class="footnote-back">↩</a></p></li>
<li id="fn37"><p>p53 is a gene upregulated in cancer cells, which would then self-destruct<a href="3-res.html#fnref37" class="footnote-back">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-mat-met.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-summary-conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
