<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Mechano – Chemical Feedback Loops During Vertebrate Organ Development</title>
  <meta name="description" content="References | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta name="generator" content="bookdown 0.13.2 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Mechano – Chemical Feedback Loops During Vertebrate Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="statutory-declaration.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html"><i class="fa fa-check"></i>Instructions</a><ul>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#print-version"><i class="fa fa-check"></i>Print version</a></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#electronic-version"><i class="fa fa-check"></i>Electronic version</a></li>
<li class="chapter" data-level="" data-path="instructions.html"><a href="instructions.html#repository"><i class="fa fa-check"></i>repository</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="list-of-abbreviations.html"><a href="list-of-abbreviations.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#cell-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#morphogenesis"><i class="fa fa-check"></i><b>1.1.2</b> Morphogenesis</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#cell-communication"><i class="fa fa-check"></i><b>1.1.3</b> Cell communication</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#objective-and-open-questions"><i class="fa fa-check"></i><b>1.3</b> Objective and Open Questions</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-Intro.html"><a href="1-Intro.html#motivation"><i class="fa fa-check"></i><b>1.3.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-Intro.html"><a href="1-Intro.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>1.4</b> Data Strategy and Analysis</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-Intro.html"><a href="1-Intro.html#sample-preparation"><i class="fa fa-check"></i><b>1.4.1</b> Sample Preparation</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-Intro.html"><a href="1-Intro.html#imaging"><i class="fa fa-check"></i><b>1.4.2</b> Imaging</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-Intro.html"><a href="1-Intro.html#data-handling"><i class="fa fa-check"></i><b>1.4.3</b> Data handling</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-Intro.html"><a href="1-Intro.html#image-and-data-analysis"><i class="fa fa-check"></i><b>1.4.4</b> Image and Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.9</b> Hardware</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.10</b> Software</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.1.11</b> Ground Truth Data</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.1</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.2</b> Wetlab</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.3</b> Computational</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.3</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.1</b> Lateral Line Morphometrics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#dataset"><i class="fa fa-check"></i><b>3.1.1</b> Dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#res-ccounts"><i class="fa fa-check"></i><b>3.1.2</b> Number and Position of Cell Clusters</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#res-llmorph"><i class="fa fa-check"></i><b>3.1.3</b> Cell Count and Area of CCs</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2</b> Proliferation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#dataset-1"><i class="fa fa-check"></i><b>3.2.1</b> Dataset</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#res-prolpllp"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Mitoses</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.3</b> Rosette Formation and Cluster Deposition</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-res.html"><a href="3-res.html#dataset-2"><i class="fa fa-check"></i><b>3.3.1</b> Dataset</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-res.html"><a href="3-res.html#cluster-deposition"><i class="fa fa-check"></i><b>3.3.2</b> Cluster Deposition</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-res.html"><a href="3-res.html#registration-1"><i class="fa fa-check"></i><b>3.3.3</b> Registration</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-res.html"><a href="3-res.html#res-rdtdet"><i class="fa fa-check"></i><b>3.3.4</b> Rosette Detection</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-res.html"><a href="3-res.html#correlations"><i class="fa fa-check"></i><b>3.3.5</b> Correlations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-res.html"><a href="3-res.html#apical-constriction-1"><i class="fa fa-check"></i><b>3.4</b> Apical Constriction</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-res.html"><a href="3-res.html#dataset-3"><i class="fa fa-check"></i><b>3.4.1</b> Dataset</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-res.html"><a href="3-res.html#cell-shape-and-arrangement"><i class="fa fa-check"></i><b>3.4.2</b> Cell Shape and Arrangement</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-res.html"><a href="3-res.html#single-cell-metrics"><i class="fa fa-check"></i><b>3.4.3</b> Single Cell Metrics</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.5</b> Haircell Specification</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-res.html"><a href="3-res.html#dataset-4"><i class="fa fa-check"></i><b>3.5.1</b> Dataset</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-res.html"><a href="3-res.html#atoh1a-in-the-ll"><i class="fa fa-check"></i><b>3.5.2</b> Atoh1a in the LL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-res.html"><a href="3-res.html#rescue-experiments"><i class="fa fa-check"></i><b>3.6</b> Rescue Experiments</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-res.html"><a href="3-res.html#res-shrmresc"><i class="fa fa-check"></i><b>3.6.1</b> Shroom3 Ectopic Expression</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-res.html"><a href="3-res.html#morphogen-rescue"><i class="fa fa-check"></i><b>3.6.2</b> Morphogen Rescue</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-res.html"><a href="3-res.html#res-rockresc"><i class="fa fa-check"></i><b>3.6.3</b> Mechanic Rescue</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-res.html"><a href="3-res.html#res-tempresc"><i class="fa fa-check"></i><b>3.6.4</b> Thermodynamic Rescue</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-res.html"><a href="3-res.html#atoh1a-knockdown"><i class="fa fa-check"></i><b>3.6.5</b> atoh1a knockdown</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html"><i class="fa fa-check"></i><b>4</b> Summary &amp; Conclusion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#adult-phenotype"><i class="fa fa-check"></i><b>4.1</b> Adult Phenotype</a></li>
<li class="chapter" data-level="4.2" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#lateral-line"><i class="fa fa-check"></i><b>4.2</b> Lateral Line</a></li>
<li class="chapter" data-level="4.3" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#rosettes"><i class="fa fa-check"></i><b>4.3</b> Rosettes</a></li>
<li class="chapter" data-level="4.4" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#hair-cells"><i class="fa fa-check"></i><b>4.4</b> Hair Cells</a></li>
<li class="chapter" data-level="4.5" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#apical-constriction-2"><i class="fa fa-check"></i><b>4.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.6" data-path="4-summary-conclusion.html"><a href="4-summary-conclusion.html#model-2"><i class="fa fa-check"></i><b>4.6</b> Model</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-discussion.html"><a href="5-discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a><ul>
<li class="chapter" data-level="5.1" data-path="5-discussion.html"><a href="5-discussion.html#proliferation-1"><i class="fa fa-check"></i><b>5.1</b> Proliferation</a></li>
<li class="chapter" data-level="5.2" data-path="5-discussion.html"><a href="5-discussion.html#rosette-detection"><i class="fa fa-check"></i><b>5.2</b> Rosette Detection</a></li>
<li class="chapter" data-level="5.3" data-path="5-discussion.html"><a href="5-discussion.html#hair-cell-specification-1"><i class="fa fa-check"></i><b>5.3</b> Hair Cell specification</a></li>
<li class="chapter" data-level="5.4" data-path="5-discussion.html"><a href="5-discussion.html#single-cell-reconstruction-and-ac-measurement"><i class="fa fa-check"></i><b>5.4</b> Single cell reconstruction and AC measurement</a></li>
<li class="chapter" data-level="5.5" data-path="5-discussion.html"><a href="5-discussion.html#future-perspectives"><i class="fa fa-check"></i><b>5.5</b> Future perspectives</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplement.html"><a href="A-supplement.html"><i class="fa fa-check"></i><b>A</b> Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Mechano – Chemical Feedback Loops During Vertebrate Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->

<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->

<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>1. E. H. P. A. Haeckel, <em>Generelle morphologie der organismen. Allgemeine grundzüge der organischen formen-wissenschaft, mechanisch begründet durch die von Charles Darwin reformirte descendenztheorie, von Ernst Haeckel.</em> (G. Reimer, Berlin, 1866; <a href="http://www.biodiversitylibrary.org/bibliography/3953">http://www.biodiversitylibrary.org/bibliography/3953</a>).</p>
</div>
<div>
<p>2. W. Garstang, The Theory of Recapitulation: A Critical Re-statement of the Biogenetic Law. <em>J. Linn. Soc. London, Zool.</em> <strong>35</strong>, 81–101 (1922).</p>
</div>
<div>
<p>3. M. Davies, The Embodiment of the Concept of Organic Expression: Frank Lloyd Wright. <em>Archit. Hist.</em> <strong>25</strong>, 120 (1982).</p>
</div>
<div>
<p>4. O. Shaya, U. Binshtok, M. Hersch, D. Rivkin, S. Weinreb, L. Amir-Zilberstein, B. Khamaisi, O. Oppenheim, R. A. Desai, R. J. Goodyear, G. P. Richardson, C. S. Chen, D. Sprinzak, Cell-Cell Contact Area Affects Notch Signaling and Notch-Dependent Patterning. <em>Dev. Cell</em>. <strong>40</strong>, 505–511.e6 (2017).</p>
</div>
<div>
<p>5. M. Galli, D. O. Morgan, Cell Size Determines the Strength of the Spindle Assembly Checkpoint during Embryonic Development. <em>Dev. Cell</em>. <strong>36</strong>, 344–352 (2016).</p>
</div>
<div>
<p>6. C. J. Miller, L. A. Davidson, The interplay between cell signalling and mechanics in developmental processes. <em>Nat. Rev. Genet.</em> <strong>14</strong>, 733–744 (2013).</p>
</div>
<div>
<p>7. M. Théry, A. Jiménez-Dalmaroni, V. Racine, M. Bornens, F. Jülicher, Experimental and theoretical study of mitotic spindle orientation. <em>Nature</em>. <strong>447</strong>, 493–496 (2007).</p>
</div>
<div>
<p>8. D. St Johnston, B. Sanson, Epithelial polarity and morphogenesis. <em>Curr. Opin. Cell Biol.</em> <strong>23</strong>, 540–546 (2011).</p>
</div>
<div>
<p>9. M. J. Harding, H. F. McGraw, A. Nechiporuk, The roles and regulation of multicellular rosette structures during morphogenesis. <em>Development</em>. <strong>141</strong>, 2549–2558 (2014).</p>
</div>
<div>
<p>10. C. Niehrs, The complex world of WNT receptor signalling. <em>Nat. Rev. Mol. Cell Biol.</em> <strong>13</strong>, 767–779 (2012).</p>
</div>
<div>
<p>11. N. Turner, R. Grose, Fibroblast growth factor signalling: From development to cancer. <em>Nat. Rev. Cancer</em>. <strong>10</strong>, 116–129 (2010).</p>
</div>
<div>
<p>12. S. Durdu, M. Iskar, C. Revenu, N. Schieber, A. Kunze, P. Bork, Y. Schwab, D. Gilmour, Luminal signalling links cell communication to tissue architecture during organogenesis. <em>Nature</em>. <strong>515</strong>, 120–124 (2014).</p>
</div>
<div>
<p>13. S. J. Bray, Notch signalling in context. <em>Nat. Rev. Mol. Cell Biol.</em> <strong>17</strong>, 722–735 (2016).</p>
</div>
<div>
<p>14. N. Guisoni, R. Martinez-Corral, J. Garcia-Ojalvo, J. de Navascués, Diversity of fate outcomes in cell pairs under lateral inhibition. <em>Dev.</em> <strong>144</strong>, 1177–1186 (2017).</p>
</div>
<div>
<p>15. G. L. Hunter, L. He, N. Perrimon, G. Charras, E. Giniger, B. Baum, A role for actomyosin contractility in Notch signaling. <em>BMC Biol.</em> <strong>17</strong>, 1–15 (2019).</p>
</div>
<div>
<p>16. I. Khait, Y. Orsher, O. Golan, U. Binshtok, N. Gordon-Bar, L. Amir-Zilberstein, D. Sprinzak, Quantitative Analysis of Delta-like 1 Membrane Dynamics Elucidates the Role of Contact Geometry on Notch Signaling. <em>Cell Rep.</em> <strong>14</strong>, 225–233 (2016).</p>
</div>
<div>
<p>17. K. Howe, M. D. Clark, C. F. Torroja, J. Torrance, C. Berthelot, M. Muffato, J. E. Collins, S. Humphray, K. McLaren, L. Matthews, S. McLaren, I. Sealy, M. Caccamo, C. Churcher, C. Scott, J. C. Barrett, R. Koch, G.-J. Rauch, S. White, W. Chow, B. Kilian, L. T. Quintais, J. a Guerra-Assunção, Y. Zhou, Y. Gu, J. Yen, J.-H. Vogel, T. Eyre, S. Redmond, R. Banerjee, J. Chi, B. Fu, E. Langley, S. F. Maguire, G. K. Laird, D. Lloyd, E. Kenyon, S. Donaldson, H. Sehra, J. Almeida-King, J. Loveland, S. Trevanion, M. Jones, M. Quail, D. Willey, A. Hunt, J. Burton, S. Sims, K. McLay, B. Plumb, J. Davis, C. Clee, K. Oliver, R. Clark, C. Riddle, D. Elliot, D. Eliott, G. Threadgold, G. Harden, D. Ware, S. Begum, B. Mortimore, B. Mortimer, G. Kerry, P. Heath, B. Phillimore, A. Tracey, N. Corby, M. Dunn, C. Johnson, J. Wood, S. Clark, S. Pelan, G. Griffiths, M. Smith, R. Glithero, P. Howden, N. Barker, C. Lloyd, C. Stevens, J. Harley, K. Holt, G. Panagiotidis, J. Lovell, H. Beasley, C. Henderson, D. Gordon, K. Auger, D. Wright, J. Collins, C. Raisen, L. Dyer, K. Leung, L. Robertson, K. Ambridge, D. Leongamornlert, S. McGuire, R. Gilderthorp, C. Griffiths, D. Manthravadi, S. Nichol, G. Barker, S. Whitehead, M. Kay, J. Brown, C. Murnane, E. Gray, M. Humphries, N. Sycamore, D. Barker, D. Saunders, J. Wallis, A. Babbage, S. Hammond, M. Mashreghi-Mohammadi, L. Barr, S. Martin, P. Wray, A. Ellington, N. Matthews, M. Ellwood, R. Woodmansey, G. Clark, J. D. Cooper, J. Cooper, A. Tromans, D. Grafham, C. Skuce, R. Pandian, R. Andrews, E. Harrison, A. Kimberley, J. Garnett, N. Fosker, R. Hall, P. Garner, D. Kelly, C. Bird, S. Palmer, I. Gehring, A. Berger, C. M. Dooley, Z. Ersan-Ürün, C. Eser, H. Geiger, M. Geisler, L. Karotki, A. Kirn, J. Konantz, M. Konantz, M. Oberländer, S. Rudolph-Geiger, M. Teucke, C. Lanz, G. Raddatz, K. Osoegawa, B. Zhu, A. Rapp, S. Widaa, C. Langford, F. Yang, S. C. Schuster, N. P. Carter, J. Harrow, Z. Ning, J. Herrero, S. M. J. Searle, A. Enright, R. Geisler, R. H. a Plasterk, C. Lee, M. Westerfield, P. J. de Jong, L. I. Zon, J. H. Postlethwait, C. Nüsslein-Volhard, T. J. P. Hubbard, H. Roest Crollius, J. Rogers, D. L. Stemple, The zebrafish reference genome sequence and its relationship to the human genome. <em>Nature</em>. <strong>496</strong>, 498–503 (2013).</p>
</div>
<div>
<p>18. C. B. Kimmel, W. W. Ballard, S. R. Kimmel, B. Ullmann, T. F. Schilling, Stages of embryonic development of the zebrafish. <em>Dev. Dyn.</em> <strong>203</strong>, 253–310 (1995).</p>
</div>
<div>
<p>19. A. B. Chitnis, D. Dalle Nogare, M. Matsuda, Building the posterior lateral line system in zebrafish. <em>Dev. Neurobiol.</em> <strong>72</strong>, 234–255 (2012).</p>
</div>
<div>
<p>20. A. Ghysen, C. Dambly-Chaudiere, The lateral line microcosmos. <em>Genes &amp; Dev.</em> <strong>21</strong>, 2118–2130 (2007).</p>
</div>
<div>
<p>21. P. Haas, D. Gilmour, Chemokine Signaling Mediates Self-Organizing Tissue Migration in the Zebrafish Lateral Line. <em>Dev. Cell</em>. <strong>10</strong>, 673–680 (2006).</p>
</div>
<div>
<p>22. E. D. Thomas, I. A. Cruz, D. W. Hailey, D. W. Raible, There and back again: development and regeneration of the zebrafish lateral line system. <em>Wiley Interdiscip. Rev. Dev. Biol.</em> <strong>4</strong>, 1–16 (2015).</p>
</div>
<div>
<p>23. L. Laguerre, A. Ghysen, C. Dambly-Chaudière, Mitotic patterns in the migrating lateral line cells of zebrafish embryos. <em>Dev. Dyn.</em> <strong>238</strong>, 1042–1051 (2009).</p>
</div>
<div>
<p>24. A. Nechiporuk, D. W. Raible, FGF-dependent mechanosensory organ patterning in zebrafish. <em>Science</em>. <strong>320</strong>, 1774–1777 (2008).</p>
</div>
<div>
<p>25. A. Aman, T. Piotrowski, Wnt/<span class="math inline">\(\beta\)</span>-Catenin and Fgf Signaling Control Collective Cell Migration by Restricting Chemokine Receptor Expression. <em>Dev. Cell</em>. <strong>15</strong>, 749–761 (2008).</p>
</div>
<div>
<p>26. D. Hava, U. Forster, M. Matsuda, S. Cui, B. a Link, J. Eichhorst, B. Wiesner, A. Chitnis, S. Abdelilah-Seyfried, Apical membrane maturation and cellular rosette formation during morphogenesis of the zebrafish lateral line. <em>J. Cell Sci.</em> <strong>122</strong>, 687–695 (2009).</p>
</div>
<div>
<p>27. V. Lecaudey, G. Cakan-Akdogan, W. H. J. Norton, D. Gilmour, Dynamic Fgf signaling couples morphogenesis and migration in the zebrafish lateral line primordium. <em>Development</em>. <strong>135</strong>, 2695–705 (2008).</p>
</div>
<div>
<p>28. M. Tsang, R. Friesel, T. Kudoh, I. B. Dawid, Identification of Sef, a novel modulator of FGF signalling. <em>Nat. Cell Biol.</em> <strong>4</strong>, 165–169 (2002).</p>
</div>
<div>
<p>29. S. Ernst, K. Liu, S. Agarwala, N. Moratscheck, M. E. Avci, D. D. Nogare, A. B. Chitnis, O. Ronneberger, V. Lecaudey, Shroom3 is required downstream of FGF signalling to mediate proneuromast assembly in zebrafish. <em>Development</em>. <strong>139</strong>, 4571–4581 (2012).</p>
</div>
<div>
<p>30. M. Matsuda, A. B. Chitnis, Atoh1a expression must be restricted by Notch signaling for effective morphogenesis of the posterior lateral line primordium in zebrafish. <em>Development</em>. <strong>137</strong>, 3477–3487 (2010).</p>
</div>
<div>
<p>31. I. Mirkovic, S. Pylawka, a J. Hudspeth, Rearrangements between differentiating hair cells coordinate planar polarity and the establishment of mirror symmetry in lateral-line neuromasts. <em>Biol. Open</em>. <strong>1</strong>, 498–505 (2012).</p>
</div>
<div>
<p>32. G. W. Rouse, J. O. Pickles, Paired development of hair cells in neuromasts of the teleost lateral line. <em>Proc. R. Soc. B Biol. Sci.</em> <strong>246</strong>, 123–128 (1991).</p>
</div>
<div>
<p>33. D. Das, J. K. Zalewski, S. Mohan, T. F. Plageman, A. P. VanDemark, J. D. Hildebrand, The interaction between Shroom3 and Rho-kinase is required for neural tube morphogenesis in mice. <em>Biol. Open</em>. <strong>3</strong>, 850–60 (2014).</p>
</div>
<div>
<p>34. J. D. Hildebrand, Shroom regulates epithelial cell shape via the apical positioning of an actomyosin network. <em>J. Cell Sci.</em> <strong>118</strong>, 5191–203 (2005).</p>
</div>
<div>
<p>35. J. D. Hildebrand, P. Soriano, Shroom, a PDZ Domain–Containing Actin-Binding Protein, Is Required for Neural Tube Morphogenesis in Mice. <em>Cell</em>. <strong>99</strong>, 485–497 (1999).</p>
</div>
<div>
<p>36. C. Lee, H. M. Scherr, J. B. Wallingford, Shroom family proteins regulate gamma-tubulin distribution and microtubule architecture during epithelial cell shape change. <em>Development</em>. <strong>134</strong>, 1431–1441 (2007).</p>
</div>
<div>
<p>37. T. Nishimura, M. Takeichi, Shroom3-mediated recruitment of Rho kinases to the apical cell junctions regulates epithelial and neuroepithelial planar remodeling. <em>Development</em>. <strong>135</strong>, 1493–1502 (2008).</p>
</div>
<div>
<p>38. T. F. Plageman, B. K. Chauhan, C. Yang, F. Jaudon, X. Shang, Y. Zheng, M. Lou, A. Debant, J. D. Hildebrand, R. a Lang, A Trio-RhoA-Shroom3 pathway is required for apical constriction and epithelial invagination. <em>Development</em>. <strong>138</strong>, 5177–88 (2011).</p>
</div>
<div>
<p>39. M. Tariq, J. W. Belmont, S. Lalani, T. Smolarek, S. M. Ware, SHROOM3 is a novel candidate for heterotaxy identified by whole exome sequencing. <em>Genome Biol.</em> <strong>12</strong>, R91 (2011).</p>
</div>
<div>
<p>40. J. K. Zalewski, J. H. Mo, S. Heber, A. Heroux, R. G. Gardner, J. D. Hildebrand, A. P. VanDemark, Structure of the Shroom-Rho kinase complex reveals a binding interface with monomeric shroom that regulates cell morphology and stimulates kinase activity. <em>J. Biol. Chem.</em> <strong>291</strong>, 25364–25374 (2016).</p>
</div>
<div>
<p>41. K. Liu, S. Ernst, V. Lecaudey, O. Ronneberger, Epithelial rosette detection in microscopic images, 1–8 (2010).</p>
</div>
<div>
<p>42. V. M. Bedell, Y. Wang, J. M. Campbell, T. L. Poshusta, C. G. Starker, R. G. Krug II, W. Tan, S. G. Penheiter, A. C. Ma, A. Y. H. Leung, S. C. Fahrenkrug, D. F. Carlson, D. F. Voytas, K. J. Clark, J. J. Essner, S. C. Ekker, In vivo genome editing using a high-efficiency TALEN system. <em>Nature</em>. <strong>490</strong>, 114–118 (2012).</p>
</div>
<div>
<p>43. P. J. Keller, E. H. K. Stelzer, Digital scanned laser light sheet fluorescence microscopy. <em>Cold Spring Harb. Protoc.</em> <strong>2010</strong>, pdb.top78 (2010).</p>
</div>
<div>
<p>44. R. Gräf, J. Rietdorf, T. Zimmermann, in <em>Microsc. Tech.</em> (Springer, 2005; <a href="http://link.springer.com/10.1007/b102210">http://link.springer.com/10.1007/b102210</a>), pp. 57–75.</p>
</div>
<div>
<p>45. C. Haddon, L. Smithers, S. Schneider-Maunoury, T. Coche, D. Henrique, J. Lewis, Multiple delta genes and lateral inhibition in zebrafish primary neurogenesis. <em>Development</em>. <strong>125</strong>, 359–370 (1998).</p>
</div>
<div>
<p>46. J. Schindelin, I. Arganda-Carreras, E. Frise, V. Kaynig, M. Longair, T. Pietzsch, S. Preibisch, C. Rueden, S. Saalfeld, B. Schmid, J.-Y. Tinevez, D. J. White, V. Hartenstein, K. Eliceiri, P. Tomancak, A. Cardona, Fiji: an open-source platform for biological-image analysis. <em>Nat. Methods</em>. <strong>9</strong>, 676–82 (2012).</p>
</div>
<div>
<p>47. K. S. Button, J. P. Ioannidis, C. Mokrysz, B. A. Nosek, J. Flint, E. S. Robinson, M. R. Munafò, Power failure: Why small sample size undermines the reliability of neuroscience. <em>Nat. Rev. Neurosci.</em> (2013), doi:<a href="https://doi.org/10.1038/nrn3475">10.1038/nrn3475</a>.</p>
</div>
<div>
<p>48. S. Krig, in <em>Comput. Vis. Metrics</em> (Apress, Berkeley, CA, 2014; <a href="http://link.springer.com/10.1007/978-1-4302-5930-5{\_}7">http://link.springer.com/10.1007/978-1-4302-5930-5{\_}7</a>), pp. 283–311.</p>
</div>
<div>
<p>49. D. Legland, I. Arganda-Carreras, P. Andrey, MorphoLibJ: Integrated library and plugins for mathematical morphology with ImageJ. <em>Bioinformatics</em> (2016), doi:<a href="https://doi.org/10.1093/bioinformatics/btw413">10.1093/bioinformatics/btw413</a>.</p>
</div>
<div>
<p>50. J. Ollion, J. Cochennec, F. Loll, C. Escudé, T. Boudier, TANGO: A generic tool for high-throughput 3D image analysis for studying nuclear organization. <em>Bioinformatics</em> (2013), doi:<a href="https://doi.org/10.1093/bioinformatics/btt276">10.1093/bioinformatics/btt276</a>.</p>
</div>
<div>
<p>51. E. L. Lehman, <em>Nonparametric Statistical Methods Based on Ranks</em> (1975).</p>
</div>
<div>
<p>52. Y. Jia, E. Shelhamer, J. Donahue, S. Karayev, J. Long, R. Girshick, S. Guadarrama, T. Darrell, Caffe: Convolutional Architecture for Fast Feature Embedding (2014), doi:<a href="https://doi.org/10.1145/2647868.2654889">10.1145/2647868.2654889</a>.</p>
</div>
<div>
<p>53. E. Meijering, O. Dzyubachyk, I. Smal, Methods for cell and particle tracking., doi:<a href="https://doi.org/10.1016/B978-0-12-391857-4.00009-4">10.1016/B978-0-12-391857-4.00009-4</a>.</p>
</div>
<div>
<p>54. J.-y. Lee, R. M. Harland, Actomyosin contractility and microtubules drive apical constriction in Xenopus bottle cells. <em>Dev. Biol.</em> <strong>311</strong>, 40–52 (2007).</p>
</div>
<div>
<p>55. M. J. Harding, A. V. Nechiporuk, Fgfr-Ras-MAPK signaling is required for apical constriction via apical positioning of Rho-associated kinase during mechanosensory organ formation. <em>Development</em>. <strong>139</strong>, 3467–3467 (2012).</p>
</div>
<div>
<p>56. M. J. Harding, Regulation of Cell Shape During Development of the Nervous System, 1–97 (2013).</p>
</div>
<div>
<p>57. L. Laguerre, F. Soubiran, A. Ghysen, N. König, C. Dambly-Chaudière, Cell proliferation in the developing lateral line system of zebrafish embryos. <em>Dev. Dyn.</em> <strong>233</strong>, 466–472 (2005).</p>
</div>
<div>
<p>58. F. Hans, S. Dimitrov, Histone H3 phosphorylation and cell division. <em>Oncogene</em>. <strong>20</strong>, 3021–3027 (2001).</p>
</div>
<div>
<p>59. T. Fischer, EdU Proliferation Kit (2019), (available at <a href="https://www.thermofisher.com/de/de/home/references/newsletters-and-journals/bioprobes-journal-of-cell-biology-applications/bioprobes-70/click-it-plus-edu-proliferation-kits.html">https://www.thermofisher.com/de/de/home/references/newsletters-and-journals/bioprobes-journal-of-cell-biology-applications/bioprobes-70/click-it-plus-edu-proliferation-kits.html</a>).</p>
</div>
<div>
<p>60. M. H. Mokalled, C. Patra, A. L. Dickson, T. Endo, D. Y. R. Stainier, K. D. Poss, Injury-induced ctgfa directs glial bridging and spinal cord regeneration in zebrafish. <strong>354</strong> (2016), doi:<a href="https://doi.org/10.1126/science.aaf2679">10.1126/science.aaf2679</a>.</p>
</div>
<div>
<p>61. I. A. Cruz, R. Kappedal, S. M. Mackenzie, D. W. Hailey, T. L. Hoffman, T. F. Schilling, D. W. Raible, Robust regeneration of adult zebrafish lateral line hair cells reflects continued precursor pool maintenance. <em>Dev. Biol.</em> <strong>402</strong>, 229–238 (2015).</p>
</div>
<div>
<p>62. F. Pinto-Teixeira, O. Viader-Llargués, E. Torres-Mejía, M. Turan, E. González-Gualda, L. Pola-Morell, H. López-Schier, Inexhaustible hair-cell regeneration in young and aged zebrafish. <em>Biol. Open</em>. <strong>4</strong>, 903–9 (2015).</p>
</div>
<div>
<p>63. H. Wada, A. Ghysen, K. Asakawa, G. Abe, T. Ishitani, K. Kawakami, Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. <em>Curr. Biol.</em> <strong>23</strong>, 1559–1565 (2013).</p>
</div>
<div>
<p>64. D. S. Kleinhans, anaLLzr2DT (2019), (available at <a href="https://github.com/KleinhansDa/anaLLzr2DT">https://github.com/KleinhansDa/anaLLzr2DT</a>).</p>
</div>
<div>
<p>65. T. Falk, D. Mai, R. Bensch, Ö. Çiçek, A. Abdulkadir, Y. Marrakchi, A. Böhm, J. Deubner, Z. Jäckel, K. Seiwald, A. Dovzhenko, O. Tietz, C. Dal Bosco, S. Walsh, D. Saltukoglu, T. L. Tay, M. Prinz, K. Palme, M. Simons, I. Diester, T. Brox, O. Ronneberger, U-Net: deep learning for cell counting, detection, and morphometry. <em>Nat. Methods</em>. <strong>16</strong>, 67–70 (2019).</p>
</div>
<div>
<p>66. N. C. Shaner, R. E. Campbell, P. A. Steinbach, B. N. Giepmans, A. E. Palmer, R. Y. Tsien, Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. red fluorescent protein. <em>Nat. Biotechnol.</em> <strong>22</strong>, 1567–1572 (2004).</p>
</div>
<div>
<p>67. A. J. Ridley, Rho GTPase signalling in cell migration. <em>Curr. Opin. Cell Biol.</em> <strong>36</strong>, 103–112 (2015).</p>
</div>
<div>
<p>68. G. Wang, A. B. Cadwallader, D. S. Jang, M. Tsang, H. J. Yost, J. D. Amack, The Rho kinase Rock2b establishes anteroposterior asymmetry of the ciliated Kupffer’s vesicle in zebrafish. <em>Development</em>. <strong>138</strong>, 45–54 (2011).</p>
</div>
<div>
<p>69. T. A. Dettlaff, in <em>Anim. Species dev. Stud.</em> (Springer US, Boston, MA, 1991; <a href="http://link.springer.com/10.1007/978-1-4615-3654-3{\_}1">http://link.springer.com/10.1007/978-1-4615-3654-3{\_}1</a>), pp. 1–14.</p>
</div>
<div>
<p>70. J. S. Eisen, J. C. Smith, Controlling morpholino experiments: don’t stop making antisense. <em>Development</em>. <strong>135</strong>, 1735–43 (2008).</p>
</div>
<div>
<p>71. A. Kozlovskaja-Gumbrienė, R. Yi, R. Alexander, A. Aman, R. Jiskra, D. Nagelberg, H. Knaut, M. McClain, T. Piotrowski, Proliferation-independent regulation of organ size by Fgf/Notch signaling. <em>Elife</em>. <strong>6</strong>, 1–31 (2017).</p>
</div>
<div>
<p>72. T. Ching, D. S. Himmelstein, B. K. Beaulieu-Jones, A. A. Kalinin, B. T. Do, G. P. Way, E. Ferrero, P. M. Agapow, M. Zietz, M. M. Hoffman, W. Xie, G. L. Rosen, B. J. Lengerich, J. Israeli, J. Lanchantin, S. Woloszynek, A. E. Carpenter, A. Shrikumar, J. Xu, E. M. Cofer, C. A. Lavender, S. C. Turaga, A. M. Alexandari, Z. Lu, D. J. Harris, D. Decaprio, Y. Qi, A. Kundaje, Y. Peng, L. K. Wiley, M. H. Segler, S. M. Boca, S. J. Swamidass, A. Huang, A. Gitter, C. S. Greene, Opportunities and obstacles for deep learning in biology and medicine. <em>J. R. Soc. Interface</em>. <strong>15</strong> (2018), doi:<a href="https://doi.org/10.1098/rsif.2017.0387">10.1098/rsif.2017.0387</a>.</p>
</div>
<div>
<p>73. S. Webb, Deep learning for biology. <em>Nature</em>. <strong>554</strong>, 555–557 (2018).</p>
</div>
<div>
<p>74. S. Schulte-Merker, D. Y. R. Stainier, Out with the old, in with the new: reassessing morpholino knockdowns in light of genome editing technology. <em>Development</em>. <strong>141</strong>, 3103–3104 (2014).</p>
</div>
<div>
<p>75. J. Y. Lee, R. M. Harland, Actomyosin contractility and microtubules drive apical constriction in Xenopus bottle cells. <em>Dev. Biol.</em> (2007), doi:<a href="https://doi.org/10.1016/j.ydbio.2007.08.010">10.1016/j.ydbio.2007.08.010</a>.</p>
</div>
</div>
</div>











































            </section>

          </div>
        </div>
      </div>
<a href="statutory-declaration.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
