<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Genetic and Mechanic Signaling in Organ Development</title>
  <meta name="description" content="References | Genetic and Mechanic Signaling in Organ Development" />
  <meta name="generator" content="bookdown 0.20.2 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Genetic and Mechanic Signaling in Organ Development" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Genetic and Mechanic Signaling in Organ Development" />
  
  
  

<meta name="author" content="David Kleinhans" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="statutory-declaration.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { font-weight: bold; } /* Alert */
code span.an { font-style: italic; } /* Annotation */
code span.cf { font-weight: bold; } /* ControlFlow */
code span.co { font-style: italic; } /* Comment */
code span.cv { font-style: italic; } /* CommentVar */
code span.do { font-style: italic; } /* Documentation */
code span.dt { text-decoration: underline; } /* DataType */
code span.er { font-weight: bold; } /* Error */
code span.in { font-style: italic; } /* Information */
code span.kw { font-weight: bold; } /* Keyword */
code span.pp { font-weight: bold; } /* Preprocessor */
code span.wa { font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html"><i class="fa fa-check"></i>Preliminary Content</a><ul>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#about"><i class="fa fa-check"></i>About</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#foreword"><i class="fa fa-check"></i>Foreword</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abbreviations"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="preliminary-content.html"><a href="preliminary-content.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="1-Intro.html"><a href="1-Intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-Intro.html"><a href="1-Intro.html#development"><i class="fa fa-check"></i><b>1.1</b> Development</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-Intro.html"><a href="1-Intro.html#intro-types"><i class="fa fa-check"></i><b>1.1.1</b> Cell Types</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-Intro.html"><a href="1-Intro.html#cell-signaling"><i class="fa fa-check"></i><b>1.1.2</b> Cell Signaling</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-Intro.html"><a href="1-Intro.html#intro-morph"><i class="fa fa-check"></i><b>1.1.3</b> Morphogenesis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-Intro.html"><a href="1-Intro.html#model-organism-and-system"><i class="fa fa-check"></i><b>1.2</b> Model Organism and System</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-Intro.html"><a href="1-Intro.html#developmental-stages"><i class="fa fa-check"></i><b>1.2.1</b> Developmental Stages</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-Intro.html"><a href="1-Intro.html#the-lateral-line-system"><i class="fa fa-check"></i><b>1.2.2</b> The Lateral Line System</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-Intro.html"><a href="1-Intro.html#posterior-lateral-line-primordium"><i class="fa fa-check"></i><b>1.2.3</b> Posterior Lateral Line Primordium</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-in-the-pllp"><i class="fa fa-check"></i><b>1.2.4</b> Shroom3 in the pLLP</a></li>
<li class="chapter" data-level="1.2.5" data-path="1-Intro.html"><a href="1-Intro.html#shroom3-mutants"><i class="fa fa-check"></i><b>1.2.5</b> <em>shroom3</em> mutants</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-Intro.html"><a href="1-Intro.html#open-questions-and-motivation"><i class="fa fa-check"></i><b>1.3</b> Open Questions and Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-mat-met.html"><a href="2-mat-met.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat"><i class="fa fa-check"></i><b>2.1</b> Materials</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-chem"><i class="fa fa-check"></i><b>2.1.1</b> Chemicals</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sol"><i class="fa fa-check"></i><b>2.1.2</b> Solutions</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-anitb"><i class="fa fa-check"></i><b>2.1.3</b> Antibodies</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-enz"><i class="fa fa-check"></i><b>2.1.4</b> Enzymes</a></li>
<li class="chapter" data-level="2.1.5" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mobikits"><i class="fa fa-check"></i><b>2.1.5</b> Molecular Biology Kits</a></li>
<li class="chapter" data-level="2.1.6" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-buff"><i class="fa fa-check"></i><b>2.1.6</b> Buffers</a></li>
<li class="chapter" data-level="2.1.7" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-lines"><i class="fa fa-check"></i><b>2.1.7</b> Zebrafish lines</a></li>
<li class="chapter" data-level="2.1.8" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-probes"><i class="fa fa-check"></i><b>2.1.8</b> ISH probes</a></li>
<li class="chapter" data-level="2.1.9" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-mos"><i class="fa fa-check"></i><b>2.1.9</b> Morpholinos</a></li>
<li class="chapter" data-level="2.1.10" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-hrdwr"><i class="fa fa-check"></i><b>2.1.10</b> Hardware</a></li>
<li class="chapter" data-level="2.1.11" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-sftwr"><i class="fa fa-check"></i><b>2.1.11</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#met"><i class="fa fa-check"></i><b>2.2</b> Methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-mat-met.html"><a href="2-mat-met.html#data-strategy-and-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Data Strategy and Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-mat-met.html"><a href="2-mat-met.html#Zeb-met"><i class="fa fa-check"></i><b>2.2.2</b> Zebrafish</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#Wet-met"><i class="fa fa-check"></i><b>2.2.3</b> Wet lab</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#comp-met"><i class="fa fa-check"></i><b>2.2.4</b> Dry lab</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-GrTrDat"><i class="fa fa-check"></i><b>2.3</b> Ground Truth</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-mat-met.html"><a href="2-mat-met.html#cluster-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Cluster Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-mat-met.html"><a href="2-mat-met.html#morphometric-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Morphometric analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-mat-met.html"><a href="2-mat-met.html#mat-datasets"><i class="fa fa-check"></i><b>2.4</b> Image Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-res.html"><a href="3-res.html"><i class="fa fa-check"></i><b>3</b> Results</a><ul>
<li class="chapter" data-level="3.1" data-path="3-res.html"><a href="3-res.html#res-met"><i class="fa fa-check"></i><b>3.1</b> Quantitative imaging and image analysis</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-res.html"><a href="3-res.html#res-mount"><i class="fa fa-check"></i><b>3.1.1</b> A 3D-printed stamp to standardize sample mounting and semi-automatize imaging</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-res.html"><a href="3-res.html#mat-anallzr2d"><i class="fa fa-check"></i><b>3.1.2</b> anaLLzr2D - Automated 2D neuromast analysis and nuclei count</a></li>
<li class="chapter" data-level="3.1.3" data-path="3-res.html"><a href="3-res.html#mat-anallzr2dt"><i class="fa fa-check"></i><b>3.1.3</b> anaLLzr2DT - Automated 2D neuromast analysis and nuclei count through time</a></li>
<li class="chapter" data-level="3.1.4" data-path="3-res.html"><a href="3-res.html#mat-anallzr3d"><i class="fa fa-check"></i><b>3.1.4</b> anaLLzr3D - Automated 3D single cell segmentation and A.I. analysis in the pLLP</a></li>
<li class="chapter" data-level="3.1.5" data-path="3-res.html"><a href="3-res.html#CNN"><i class="fa fa-check"></i><b>3.1.5</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-res.html"><a href="3-res.html#shroom3"><i class="fa fa-check"></i><b>3.2</b> Shroom3</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-res.html"><a href="3-res.html#intro-phen"><i class="fa fa-check"></i><b>3.2.1</b> Phenotype description</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-res.html"><a href="3-res.html#lateral-line-morphometrics"><i class="fa fa-check"></i><b>3.2.2</b> Lateral Line Morphometrics</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-res.html"><a href="3-res.html#proliferation"><i class="fa fa-check"></i><b>3.2.3</b> Proliferation</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-res.html"><a href="3-res.html#rosette-formation-and-cluster-deposition"><i class="fa fa-check"></i><b>3.2.4</b> Rosette Formation and Cluster Deposition</a></li>
<li class="chapter" data-level="3.2.5" data-path="3-res.html"><a href="3-res.html#apical-constriction-3"><i class="fa fa-check"></i><b>3.2.5</b> Apical Constriction</a></li>
<li class="chapter" data-level="3.2.6" data-path="3-res.html"><a href="3-res.html#res-hc"><i class="fa fa-check"></i><b>3.2.6</b> Haircell Specification</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-discussion.html"><a href="4-discussion.html"><i class="fa fa-check"></i><b>4</b> Discussion</a><ul>
<li class="chapter" data-level="4.1" data-path="4-discussion.html"><a href="4-discussion.html#methods"><i class="fa fa-check"></i><b>4.1</b> Methods</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-discussion.html"><a href="4-discussion.html#standardized-mounting"><i class="fa fa-check"></i><b>4.1.1</b> Standardized mounting</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-discussion.html"><a href="4-discussion.html#automated-image-analysis"><i class="fa fa-check"></i><b>4.1.2</b> Automated image analysis</a></li>
<li class="chapter" data-level="4.1.3" data-path="4-discussion.html"><a href="4-discussion.html#rosette-detection-1"><i class="fa fa-check"></i><b>4.1.3</b> Rosette Detection</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-discussion.html"><a href="4-discussion.html#shroom3-1"><i class="fa fa-check"></i><b>4.2</b> Shroom3</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-discussion.html"><a href="4-discussion.html#lateral-line"><i class="fa fa-check"></i><b>4.2.1</b> Lateral Line</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-discussion.html"><a href="4-discussion.html#rosette-formation-1"><i class="fa fa-check"></i><b>4.2.2</b> Rosette Formation</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-discussion.html"><a href="4-discussion.html#apical-constriction-4"><i class="fa fa-check"></i><b>4.2.3</b> Apical Constriction</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-discussion.html"><a href="4-discussion.html#hair-cell-specification"><i class="fa fa-check"></i><b>4.2.4</b> Hair Cell specification</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-discussion.html"><a href="4-discussion.html#model-2"><i class="fa fa-check"></i><b>4.2.5</b> Model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="appendix-appendix.html"><a href="appendix-appendix.html"><i class="fa fa-check"></i>(Appendix) Appendix</a></li>
<li class="chapter" data-level="" data-path="supplement.html"><a href="supplement.html"><i class="fa fa-check"></i>Supplement</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum Vitae</a></li>
<li><a href="statutory-declaration.html#statutory-declaration"><strong>Statutory Declaration</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>Genetic and Mechanic Signaling in Organ Development</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->

<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->

<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>1. S. Ernst, K. Liu, S. Agarwala, N. Moratscheck, M. E. Avci, D. D. Nogare, A. B. Chitnis, O. Ronneberger, V. Lecaudey, Shroom3 is required downstream of FGF signalling to mediate proneuromast assembly in zebrafish. <em>Development</em>. <strong>139</strong>, 4571–4581 (2012).</p>
</div>
<div>
<p>2. C. Niehrs, The complex world of WNT receptor signalling. <em>Nature Reviews Molecular Cell Biology</em>. <strong>13</strong>, 767–779 (2012).</p>
</div>
<div>
<p>3. N. Turner, R. Grose, Fibroblast growth factor signalling: From development to cancer. <em>Nature Reviews Cancer</em>. <strong>10</strong>, 116–129 (2010).</p>
</div>
<div>
<p>4. S. J. Bray, Notch signalling in context. <em>Nature Reviews Molecular Cell Biology</em>. <strong>17</strong>, 722–735 (2016).</p>
</div>
<div>
<p>5. N. Guisoni, R. Martinez-Corral, J. Garcia-Ojalvo, J. de Navascués, Diversity of fate outcomes in cell pairs under lateral inhibition. <em>Development (Cambridge)</em>. <strong>144</strong>, 1177–1186 (2017).</p>
</div>
<div>
<p>6. G. L. Hunter, L. He, N. Perrimon, G. Charras, E. Giniger, B. Baum, A role for actomyosin contractility in Notch signaling. <em>BMC Biology</em>. <strong>17</strong>, 1–15 (2019).</p>
</div>
<div>
<p>7. I. Khait, Y. Orsher, O. Golan, U. Binshtok, N. Gordon-Bar, L. Amir-Zilberstein, D. Sprinzak, Quantitative Analysis of Delta-like 1 Membrane Dynamics Elucidates the Role of Contact Geometry on Notch Signaling. <em>Cell Reports</em>. <strong>14</strong>, 225–233 (2016).</p>
</div>
<div>
<p>8. O. Shaya, U. Binshtok, M. Hersch, D. Rivkin, S. Weinreb, L. Amir-Zilberstein, B. Khamaisi, O. Oppenheim, R. A. Desai, R. J. Goodyear, G. P. Richardson, C. S. Chen, D. Sprinzak, Cell-Cell Contact Area Affects Notch Signaling and Notch-Dependent Patterning. <em>Developmental cell</em>. <strong>40</strong>, 505–511.e6 (2017).</p>
</div>
<div>
<p>9. M. Davies, The Embodiment of the Concept of Organic Expression: Frank Lloyd Wright. <em>Architectural History</em>. <strong>25</strong>, 120 (1982).</p>
</div>
<div>
<p>10. Y. Pan, I. Heemskerk, C. Ibar, B. I. Shraiman, K. D. Irvine, Differential growth triggers mechanical feedback that elevates Hippo signaling. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <strong>113</strong>, E6974–E6983 (2016).</p>
</div>
<div>
<p>11. W. T. Gibson, J. H. Veldhuis, B. Rubinstein, H. N. Cartwright, N. Perrimon, G. W. Brodland, R. Nagpal, M. C. Gibson, Control of the mitotic cleavage plane by local epithelial topology. <em>Cell</em>. <strong>144</strong>, 427–438 (2011).</p>
</div>
<div>
<p>12. B. I. Shraiman, Mechanical feedback as a possible regulator of tissue growth. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <strong>102</strong>, 3318–3323 (2005).</p>
</div>
<div>
<p>13. K. P. Landsberg, R. Farhadifar, J. Ranft, D. Umetsu, T. J. Widmann, T. Bittig, A. Said, F. Jülicher, C. Dahmann, Increased Cell Bond Tension Governs Cell Sorting at the Drosophila Anteroposterior Compartment Boundary. <em>Current Biology</em>. <strong>19</strong>, 1950–1955 (2009).</p>
</div>
<div>
<p>14. N. M. Prpic, N. Posnien, Size and shape—integration of morphometrics, mathematical modelling, developmental and evolutionary biology. <em>Development Genes and Evolution</em>. <strong>226</strong>, 109–112 (2016).</p>
</div>
<div>
<p>15. T. E. Saunders, P. W. Ingham, Open questions: How to get developmental biology into shape? <em>BMC Biology</em>. <strong>17</strong>, 10–12 (2019).</p>
</div>
<div>
<p>16. D. E. Ingber, N. Wang, D. Stamenović, Tensegrity, cellular biophysics, and the mechanics of living systems. <em>Reports on Progress in Physics</em>. <strong>77</strong> (2014), doi:<a href="https://doi.org/10.1088/0034-4885/77/4/046603">10.1088/0034-4885/77/4/046603</a>.</p>
</div>
<div>
<p>17. E. Coen, A. G. Rolland-Lagan, M. Matthews, J. A. Bangham, P. Prusinkiewicz, The genetics of geometry. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <strong>101</strong>, 4728–4735 (2004).</p>
</div>
<div>
<p>18. J.-c. Ro, N. M. Dempsey, E. Farge, Mechanotransductive cascade of Myo-II-dependent mesoderm and endoderm invaginations in embryo gastrulation (2017), doi:<a href="https://doi.org/10.1038/ncomms13883">10.1038/ncomms13883</a>.</p>
</div>
<div>
<p>19. A. Haupt, N. Minc, How cells sense their own shape - mechanisms to probe cell geometry and their implications in cellular organization and function. <em>Journal of Cell Science</em>. <strong>131</strong> (2018), doi:<a href="https://doi.org/10.1242/jcs.214015">10.1242/jcs.214015</a>.</p>
</div>
<div>
<p>20. F. Xiong, W. Ma, T. W. Hiscock, K. R. Mosaliganti, A. R. Tentner, K. A. Brakke, N. Rannou, A. Gelas, L. Souhait, I. A. Swinburne, N. D. Obholzer, S. G. Megason, Interplay of cell shape and division orientation promotes robust morphogenesis of developing epithelia. <em>Cell</em>. <strong>159</strong>, 415–427 (2014).</p>
</div>
<div>
<p>21. S. Rocha, J. Carvalho, P. Oliveira, M. Voglstaetter, D. Schvartz, A. R. Thomsen, N. Walter, R. Khanduri, J. C. Sanchez, A. Keller, C. Oliveira, I. Nazarenko, 3D Cellular Architecture Affects MicroRNA and Protein Cargo of Extracellular Vesicles. <em>Advanced Science</em>. <strong>6</strong> (2019), doi:<a href="https://doi.org/10.1002/advs.201800948">10.1002/advs.201800948</a>.</p>
</div>
<div>
<p>22. P. Gómez-Gálvez, P. Vicente-Munuera, S. Anbari, J. Buceta, L. M. Escudero, The complex three-dimensional organization of epithelial tissues. <em>Development</em>. <strong>148</strong>, dev195669 (2021).</p>
</div>
<div>
<p>23. D. A. Fletcher, R. D. Mullins, Cell mechanics and the cytoskeleton. <strong>463</strong>, 485–492 (2010).</p>
</div>
<div>
<p>24. D. St Johnston, B. Sanson, Epithelial polarity and morphogenesis. <em>Current Opinion in Cell Biology</em>. <strong>23</strong>, 540–546 (2011).</p>
</div>
<div>
<p>25. A. C. Martin, B. Goldstein, Apical constriction: themes and variations on a cellular mechanism driving morphogenesis. <em>Development</em>. <strong>141</strong>, 1987–1998 (2014).</p>
</div>
<div>
<p>26. M. J. Harding, H. F. McGraw, A. Nechiporuk, The roles and regulation of multicellular rosette structures during morphogenesis. <em>Development</em>. <strong>141</strong>, 2549–2558 (2014).</p>
</div>
<div>
<p>27. K. Howe, M. D. Clark, C. F. Torroja, J. Torrance, C. Berthelot, M. Muffato, J. E. Collins, S. Humphray, K. McLaren, L. Matthews, S. McLaren, I. Sealy, M. Caccamo, C. Churcher, C. Scott, J. C. Barrett, R. Koch, G.-J. Rauch, S. White, W. Chow, B. Kilian, L. T. Quintais, J. a Guerra-Assunção, Y. Zhou, Y. Gu, J. Yen, J.-H. Vogel, T. Eyre, S. Redmond, R. Banerjee, J. Chi, B. Fu, E. Langley, S. F. Maguire, G. K. Laird, D. Lloyd, E. Kenyon, S. Donaldson, H. Sehra, J. Almeida-King, J. Loveland, S. Trevanion, M. Jones, M. Quail, D. Willey, A. Hunt, J. Burton, S. Sims, K. McLay, B. Plumb, J. Davis, C. Clee, K. Oliver, R. Clark, C. Riddle, D. Elliot, D. Eliott, G. Threadgold, G. Harden, D. Ware, S. Begum, B. Mortimore, B. Mortimer, G. Kerry, P. Heath, B. Phillimore, A. Tracey, N. Corby, M. Dunn, C. Johnson, J. Wood, S. Clark, S. Pelan, G. Griffiths, M. Smith, R. Glithero, P. Howden, N. Barker, C. Lloyd, C. Stevens, J. Harley, K. Holt, G. Panagiotidis, J. Lovell, H. Beasley, C. Henderson, D. Gordon, K. Auger, D. Wright, J. Collins, C. Raisen, L. Dyer, K. Leung, L. Robertson, K. Ambridge, D. Leongamornlert, S. McGuire, R. Gilderthorp, C. Griffiths, D. Manthravadi, S. Nichol, G. Barker, S. Whitehead, M. Kay, J. Brown, C. Murnane, E. Gray, M. Humphries, N. Sycamore, D. Barker, D. Saunders, J. Wallis, A. Babbage, S. Hammond, M. Mashreghi-Mohammadi, L. Barr, S. Martin, P. Wray, A. Ellington, N. Matthews, M. Ellwood, R. Woodmansey, G. Clark, J. D. Cooper, J. Cooper, A. Tromans, D. Grafham, C. Skuce, R. Pandian, R. Andrews, E. Harrison, A. Kimberley, J. Garnett, N. Fosker, R. Hall, P. Garner, D. Kelly, C. Bird, S. Palmer, I. Gehring, A. Berger, C. M. Dooley, Z. Ersan-Ürün, C. Eser, H. Geiger, M. Geisler, L. Karotki, A. Kirn, J. Konantz, M. Konantz, M. Oberländer, S. Rudolph-Geiger, M. Teucke, C. Lanz, G. Raddatz, K. Osoegawa, B. Zhu, A. Rapp, S. Widaa, C. Langford, F. Yang, S. C. Schuster, N. P. Carter, J. Harrow, Z. Ning, J. Herrero, S. M. J. Searle, A. Enright, R. Geisler, R. H. a Plasterk, C. Lee, M. Westerfield, P. J. de Jong, L. I. Zon, J. H. Postlethwait, C. Nüsslein-Volhard, T. J. P. Hubbard, H. Roest Crollius, J. Rogers, D. L. Stemple, The zebrafish reference genome sequence and its relationship to the human genome. <em>Nature</em>. <strong>496</strong>, 498–503 (2013).</p>
</div>
<div>
<p>28. C. B. Kimmel, W. W. Ballard, S. R. Kimmel, B. Ullmann, T. F. Schilling, Stages of embryonic development of the zebrafish. <em>Developmental dynamics : an official publication of the American Association of Anatomists</em>. <strong>203</strong>, 253–310 (1995).</p>
</div>
<div>
<p>29. S. Washausen, W. Knabe, W. Knabe, Lateral line placodes of aquatic vertebrates are evolutionarily conserved in mammals (2018).</p>
</div>
<div>
<p>30. A. B. Chitnis, D. Dalle Nogare, M. Matsuda, Building the posterior lateral line system in zebrafish. <em>Developmental Neurobiology</em>. <strong>72</strong>, 234–255 (2012).</p>
</div>
<div>
<p>31. A. Ghysen, C. Dambly-Chaudiere, The lateral line microcosmos. <em>Genes &amp; Development</em>. <strong>21</strong>, 2118–2130 (2007).</p>
</div>
<div>
<p>32. P. Haas, D. Gilmour, Chemokine Signaling Mediates Self-Organizing Tissue Migration in the Zebrafish Lateral Line. <em>Developmental Cell</em>. <strong>10</strong>, 673–680 (2006).</p>
</div>
<div>
<p>33. E. D. Thomas, I. A. Cruz, D. W. Hailey, D. W. Raible, There and back again: development and regeneration of the zebrafish lateral line system. <em>Wiley Interdisciplinary Reviews: Developmental Biology</em>. <strong>4</strong>, 1–16 (2015).</p>
</div>
<div>
<p>34. L. Laguerre, A. Ghysen, C. Dambly-Chaudière, Mitotic patterns in the migrating lateral line cells of zebrafish embryos. <em>Developmental Dynamics</em>. <strong>238</strong>, 1042–1051 (2009).</p>
</div>
<div>
<p>35. A. Nechiporuk, D. W. Raible, FGF-dependent mechanosensory organ patterning in zebrafish. <em>Science (New York, N.Y.)</em>. <strong>320</strong>, 1774–1777 (2008).</p>
</div>
<div>
<p>36. A. Aman, T. Piotrowski, Wnt/<span class="math inline">\(\beta\)</span>-Catenin and Fgf Signaling Control Collective Cell Migration by Restricting Chemokine Receptor Expression. <em>Developmental Cell</em>. <strong>15</strong>, 749–761 (2008).</p>
</div>
<div>
<p>37. D. Hava, U. Forster, M. Matsuda, S. Cui, B. a Link, J. Eichhorst, B. Wiesner, A. Chitnis, S. Abdelilah-Seyfried, Apical membrane maturation and cellular rosette formation during morphogenesis of the zebrafish lateral line. <em>Journal of cell science</em>. <strong>122</strong>, 687–695 (2009).</p>
</div>
<div>
<p>38. V. Lecaudey, G. Cakan-Akdogan, W. H. J. Norton, D. Gilmour, Dynamic Fgf signaling couples morphogenesis and migration in the zebrafish lateral line primordium. <em>Development (Cambridge, England)</em>. <strong>135</strong>, 2695–705 (2008).</p>
</div>
<div>
<p>39. M. Tsang, R. Friesel, T. Kudoh, I. B. Dawid, Identification of Sef, a novel modulator of FGF signalling. <em>Nature Cell Biology</em>. <strong>4</strong>, 165–169 (2002).</p>
</div>
<div>
<p>40. S. Durdu, M. Iskar, C. Revenu, N. Schieber, A. Kunze, P. Bork, Y. Schwab, D. Gilmour, Luminal signalling links cell communication to tissue architecture during organogenesis. <em>Nature</em>. <strong>515</strong>, 120–124 (2014).</p>
</div>
<div>
<p>41. M. Matsuda, A. B. Chitnis, Atoh1a expression must be restricted by Notch signaling for effective morphogenesis of the posterior lateral line primordium in zebrafish. <em>Development</em>. <strong>137</strong>, 3477–3487 (2010).</p>
</div>
<div>
<p>42. I. Mirkovic, S. Pylawka, a J. Hudspeth, Rearrangements between differentiating hair cells coordinate planar polarity and the establishment of mirror symmetry in lateral-line neuromasts. <em>Biology open</em>. <strong>1</strong>, 498–505 (2012).</p>
</div>
<div>
<p>43. G. W. Rouse, J. O. Pickles, Paired development of hair cells in neuromasts of the teleost lateral line. <em>Proceedings of the Royal Society B: Biological Sciences</em>. <strong>246</strong>, 123–128 (1991).</p>
</div>
<div>
<p>44. D. Das, J. K. Zalewski, S. Mohan, T. F. Plageman, A. P. VanDemark, J. D. Hildebrand, The interaction between Shroom3 and Rho-kinase is required for neural tube morphogenesis in mice. <em>Biology open</em>. <strong>3</strong>, 850–60 (2014).</p>
</div>
<div>
<p>45. J. D. Hildebrand, Shroom regulates epithelial cell shape via the apical positioning of an actomyosin network. <em>Journal of cell science</em>. <strong>118</strong>, 5191–203 (2005).</p>
</div>
<div>
<p>46. J. D. Hildebrand, P. Soriano, Shroom, a PDZ Domain–Containing Actin-Binding Protein, Is Required for Neural Tube Morphogenesis in Mice. <em>Cell</em>. <strong>99</strong>, 485–497 (1999).</p>
</div>
<div>
<p>47. C. Lee, H. M. Scherr, J. B. Wallingford, Shroom family proteins regulate gamma-tubulin distribution and microtubule architecture during epithelial cell shape change. <em>Development (Cambridge, England)</em>. <strong>134</strong>, 1431–1441 (2007).</p>
</div>
<div>
<p>48. T. Nishimura, M. Takeichi, Shroom3-mediated recruitment of Rho kinases to the apical cell junctions regulates epithelial and neuroepithelial planar remodeling. <em>Development</em>. <strong>135</strong>, 1493–1502 (2008).</p>
</div>
<div>
<p>49. T. F. Plageman, B. K. Chauhan, C. Yang, F. Jaudon, X. Shang, Y. Zheng, M. Lou, A. Debant, J. D. Hildebrand, R. a Lang, A Trio-RhoA-Shroom3 pathway is required for apical constriction and epithelial invagination. <em>Development</em>. <strong>138</strong>, 5177–88 (2011).</p>
</div>
<div>
<p>50. J. K. Zalewski, J. H. Mo, S. Heber, A. Heroux, R. G. Gardner, J. D. Hildebrand, A. P. VanDemark, Structure of the Shroom-Rho kinase complex reveals a binding interface with monomeric shroom that regulates cell morphology and stimulates kinase activity. <em>Journal of Biological Chemistry</em>. <strong>291</strong>, 25364–25374 (2016).</p>
</div>
<div>
<p>51. K. Liu, S. Ernst, V. Lecaudey, O. Ronneberger, Epithelial rosette detection in microscopic images, 1–8 (2010).</p>
</div>
<div>
<p>52. M. Westerfield, <em>The zebrafish book</em> (2007; <a href="https://zfin.org/zf%7B\_%7Dinfo/zfbook/zfbk.html">https://zfin.org/zf{\_}info/zfbook/zfbk.html</a>).</p>
</div>
<div>
<p>53. C. Haddon, L. Smithers, S. Schneider-Maunoury, T. Coche, D. Henrique, J. Lewis, Multiple delta genes and lateral inhibition in zebrafish primary neurogenesis. <em>Development</em>. <strong>125</strong>, 359–370 (1998).</p>
</div>
<div>
<p>54. E. Y. Ma, D. W. Raible, Signaling Pathways Regulating Zebrafish Lateral Line Development. <em>Current Biology</em>. <strong>19</strong>, R381–R386 (2009).</p>
</div>
<div>
<p>55. M. E. Robu, J. D. Larson, A. Nasevicius, S. Beiraghi, C. Brenner, S. A. Farber, S. C. Ekker, P53 Activation By Knockdown Technologies. <em>PLoS Genetics</em>. <strong>3</strong>, 787–801 (2007).</p>
</div>
<div>
<p>56. D. S. Kleinhans, V. Lecaudey, Standardized mounting method of ( zebrafish ) embryos using a 3D-printed stamp for high-content , semi-automated confocal imaging, 1–10 (2019).</p>
</div>
<div>
<p>57. P. J. Keller, E. H. K. Stelzer, Digital scanned laser light sheet fluorescence microscopy. <em>Cold Spring Harbor protocols</em>. <strong>2010</strong>, pdb.top78 (2010).</p>
</div>
<div>
<p>58. R. Gräf, J. Rietdorf, T. Zimmermann, in <em>Microscopy techniques</em> (Springer, 2005; <a href="http://link.springer.com/10.1007/b102210">http://link.springer.com/10.1007/b102210</a>), pp. 57–75.</p>
</div>
<div>
<p>59. E. Meijering, O. Dzyubachyk, I. Smal, Methods for cell and particle tracking., doi:<a href="https://doi.org/10.1016/B978-0-12-391857-4.00009-4">10.1016/B978-0-12-391857-4.00009-4</a>.</p>
</div>
<div>
<p>60. J.-y. Lee, R. M. Harland, Actomyosin contractility and microtubules drive apical constriction in Xenopus bottle cells. <em>Developmental Biology</em>. <strong>311</strong>, 40–52 (2007).</p>
</div>
<div>
<p>61. M. J. Harding, A. V. Nechiporuk, Fgfr-Ras-MAPK signaling is required for apical constriction via apical positioning of Rho-associated kinase during mechanosensory organ formation. <em>Development</em>. <strong>139</strong>, 3467–3467 (2012).</p>
</div>
<div>
<p>62. M. J. Harding, Regulation of Cell Shape During Development of the Nervous System, 1–97 (2013).</p>
</div>
<div>
<p>63. J. Schindelin, I. Arganda-Carreras, E. Frise, V. Kaynig, M. Longair, T. Pietzsch, S. Preibisch, C. Rueden, S. Saalfeld, B. Schmid, J.-Y. Tinevez, D. J. White, V. Hartenstein, K. Eliceiri, P. Tomancak, A. Cardona, Fiji: an open-source platform for biological-image analysis. <em>Nature methods</em>. <strong>9</strong>, 676–82 (2012).</p>
</div>
<div>
<p>64. K. S. Button, J. P. A. Ioannidis, C. Mokrysz, B. A. Nosek, J. Flint, E. S. J. Robinson, M. R. Munafò, Power failure: Why small sample size undermines the reliability of neuroscience. <em>Nature Reviews Neuroscience</em> (2013), doi:<a href="https://doi.org/10.1038/nrn3475">10.1038/nrn3475</a>.</p>
</div>
<div>
<p>65. S. Krig, in <em>Computer vision metrics</em> (Apress, Berkeley, CA, 2014; <a href="http://link.springer.com/10.1007/978-1-4302-5930-5%7B\_%7D7">http://link.springer.com/10.1007/978-1-4302-5930-5{\_}7</a>), pp. 283–311.</p>
</div>
<div>
<p>66. P. Campinho, P. Lamperti, F. Boselli, J. Vermot, Three-dimensional microscopy and image analysis methodology for mapping and quantification of nuclear positions in tissues with approximate cylindrical geometry (2018).</p>
</div>
<div>
<p>67. S. Donoughe, C. Kim, C. G. Extavour, High-throughput live-imaging of embryos in microwell arrays using a modular specimen mounting system. <em>Biology Open</em>. <strong>7</strong>, bio031260 (2018).</p>
</div>
<div>
<p>68. J. N. Wittbrodt, U. Liebel, J. Gehrig, Generation of orientation tools for automated zebrafish screening assays using desktop 3D printing. <em>BMC Biotechnology</em>. <strong>14</strong>, 36 (2014).</p>
</div>
<div>
<p>69. T. Yu, Y. Jiang, S. Lin, A 3-dimensional (3D)-printed Template for High Throughput Zebrafish Embryo Arraying. <em>Journal of Visualized Experiments</em>, 3–7 (2018).</p>
</div>
<div>
<p>70. D. Legland, I. Arganda-Carreras, P. Andrey, MorphoLibJ: Integrated library and plugins for mathematical morphology with ImageJ. <em>Bioinformatics</em> (2016), doi:<a href="https://doi.org/10.1093/bioinformatics/btw413">10.1093/bioinformatics/btw413</a>.</p>
</div>
<div>
<p>71. E. L. Lehman, <em>Nonparametric Statistical Methods Based on Ranks</em> (1975).</p>
</div>
<div>
<p>72. Y. Jia, E. Shelhamer, J. Donahue, S. Karayev, J. Long, R. Girshick, S. Guadarrama, T. Darrell, Caffe: Convolutional Architecture for Fast Feature Embedding (2014), doi:<a href="https://doi.org/10.1145/2647868.2654889">10.1145/2647868.2654889</a>.</p>
</div>
<div>
<p>73. T. A. Dettlaff, in <em>Animal species for developmental studies</em> (Springer US, Boston, MA, 1991; <a href="http://link.springer.com/10.1007/978-1-4615-3654-3%7B\_%7D1">http://link.springer.com/10.1007/978-1-4615-3654-3{\_}1</a>), pp. 1–14.</p>
</div>
<div>
<p>74. L. Laguerre, F. Soubiran, A. Ghysen, N. König, C. Dambly-Chaudière, Cell proliferation in the developing lateral line system of zebrafish embryos. <em>Developmental Dynamics</em>. <strong>233</strong>, 466–472 (2005).</p>
</div>
<div>
<p>75. F. Hans, S. Dimitrov, Histone H3 phosphorylation and cell division. <em>Oncogene</em>. <strong>20</strong>, 3021–3027 (2001).</p>
</div>
<div>
<p>76. T. Fischer, EdU Proliferation Kit (2019), (available at <a href="https://www.thermofisher.com/de/de/home/references/newsletters-and-journals/bioprobes-journal-of-cell-biology-applications/bioprobes-70/click-it-plus-edu-proliferation-kits.html">https://www.thermofisher.com/de/de/home/references/newsletters-and-journals/bioprobes-journal-of-cell-biology-applications/bioprobes-70/click-it-plus-edu-proliferation-kits.html</a>).</p>
</div>
<div>
<p>77. T. Falk, D. Mai, R. Bensch, Ö. Çiçek, A. Abdulkadir, Y. Marrakchi, A. Böhm, J. Deubner, Z. Jäckel, K. Seiwald, A. Dovzhenko, O. Tietz, C. Dal Bosco, S. Walsh, D. Saltukoglu, T. L. Tay, M. Prinz, K. Palme, M. Simons, I. Diester, T. Brox, O. Ronneberger, U-Net: deep learning for cell counting, detection, and morphometry. <em>Nature Methods</em>. <strong>16</strong>, 67–70 (2019).</p>
</div>
<div>
<p>78. A. J. Ridley, Rho GTPase signalling in cell migration. <em>Current Opinion in Cell Biology</em>. <strong>36</strong>, 103–112 (2015).</p>
</div>
<div>
<p>79. G. Wang, A. B. Cadwallader, D. S. Jang, M. Tsang, H. J. Yost, J. D. Amack, The Rho kinase Rock2b establishes anteroposterior asymmetry of the ciliated Kupffer’s vesicle in zebrafish. <em>Development (Cambridge, England)</em>. <strong>138</strong>, 45–54 (2011).</p>
</div>
<div>
<p>80. J. Y. Lee, R. M. Harland, Actomyosin contractility and microtubules drive apical constriction in Xenopus bottle cells. <em>Developmental Biology</em> (2007), doi:<a href="https://doi.org/10.1016/j.ydbio.2007.08.010">10.1016/j.ydbio.2007.08.010</a>.</p>
</div>
<div>
<p>81. N. C. Shaner, R. E. Campbell, P. A. Steinbach, B. N. G. Giepmans, A. E. Palmer, R. Y. Tsien, Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. red fluorescent protein. <em>Nature Biotechnology</em>. <strong>22</strong>, 1567–1572 (2004).</p>
</div>
<div>
<p>82. J. S. Eisen, J. C. Smith, Controlling morpholino experiments: don’t stop making antisense. <em>Development (Cambridge, England)</em>. <strong>135</strong>, 1735–43 (2008).</p>
</div>
<div>
<p>83. S. Schulte-Merker, D. Y. R. Stainier, Out with the old, in with the new: reassessing morpholino knockdowns in light of genome editing technology. <em>Development</em>. <strong>141</strong>, 3103–3104 (2014).</p>
</div>
<div>
<p>84. R. Pulak, Tools for automating the imaging of zebrafish larvae. <em>Methods</em>. <strong>96</strong>, 118–126 (2016).</p>
</div>
<div>
<p>85. I. A. Swinburne, K. R. Mosaliganti, A. A. Green, S. G. Megason, Improved Long-Term Imaging of Embryos with Genetically Encoded <span class="math inline">\(\alpha\)</span>-Bungarotoxin. <em>PLOS ONE</em>. <strong>10</strong>, e0134005 (2015).</p>
</div>
<div>
<p>86. E. Hirsinger, B. Steventon, A Versatile Mounting Method for Long Term Imaging of Zebrafish Development. <em>Journal of Visualized Experiments</em> (2017), doi:<a href="https://doi.org/10.3791/55210">10.3791/55210</a>.</p>
</div>
<div>
<p>87. W. Masselink, J. C. Wong, B. Liu, J. Fu, P. D. Currie, Low-Cost Silicone Imaging Casts for Zebrafish Embryos and Larvae. <em>Zebrafish</em>. <strong>11</strong>, 26–31 (2014).</p>
</div>
<div>
<p>88. K. Alessandri, L. Andrique, M. Feyeux, A. Bikfalvi, P. Nassoy, G. Recher, All-in-one 3D printed microscopy chamber for multidimensional imaging, the UniverSlide. <em>Scientific Reports</em>. <strong>7</strong>, 42378 (2017).</p>
</div>
<div>
<p>89. M. P. Mattson, Superior pattern processing is the essence of the evolved human brain. <em>Frontiers in Neuroscience</em>. <strong>8</strong>, 1–17 (2014).</p>
</div>
<div>
<p>90. L. Vincent, L. Vincent, P. Soille, Watersheds in Digital Spaces: An Efficient Algorithm Based on Immersion Simulations. <em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em>. <strong>13</strong>, 583–598 (1991).</p>
</div>
<div>
<p>91. D. Ciresan, U. Meier, J. Schmidhuber, in <em>2012 ieee conference on computer vision and pattern recognition</em> (IEEE, 2012; <a href="http://ieeexplore.ieee.org/document/6248110/">http://ieeexplore.ieee.org/document/6248110/</a>), vol. 44, pp. 3642–3649.</p>
</div>
<div>
<p>92. J. Hartmann, M. Wong, E. Gallo, D. Gilmour, An image-based data-driven analysis of cellular architecture in a developing tissue. <em>eLife</em>. <strong>9</strong>, 1–33 (2020).</p>
</div>
<div>
<p>93. A. Ghorbani, A. Abid, J. Zou, Interpretation of neural networks is fragile. <em>33rd AAAI Conference on Artificial Intelligence, AAAI 2019, 31st Innovative Applications of Artificial Intelligence Conference, IAAI 2019 and the 9th AAAI Symposium on Educational Advances in Artificial Intelligence, EAAI 2019</em>, 3681–3688 (2019).</p>
</div>
<div>
<p>94. F. Pinto-Teixeira, O. Viader-Llargués, E. Torres-Mejía, M. Turan, E. González-Gualda, L. Pola-Morell, H. López-Schier, Inexhaustible hair-cell regeneration in young and aged zebrafish. <em>Biology open</em>. <strong>4</strong>, 903–9 (2015).</p>
</div>
<div>
<p>95. A. Romero-Carvajal, J. Navajas Acedo, L. Jiang, A. Kozlovskaja-Gumbrienė, R. Alexander, H. Li, T. Piotrowski, Regeneration of Sensory Hair Cells Requires Localized Interactions between the Notch and Wnt Pathways. <em>Developmental cell</em> (2015), doi:<a href="https://doi.org/10.1016/j.devcel.2015.05.025">10.1016/j.devcel.2015.05.025</a>.</p>
</div>
<div>
<p>96. J. R. Head, L. Gacioch, M. Pennisi, J. R. Meyers, Activation of canonical Wnt/??-catenin signaling stimulates proliferation in neuromasts in the zebrafish posterior lateral line. <em>Developmental Dynamics</em>. <strong>242</strong>, 832–846 (2013).</p>
</div>
<div>
<p>97. H. Wada, A. Ghysen, K. Asakawa, G. Abe, T. Ishitani, K. Kawakami, Wnt/Dkk negative feedback regulates sensory organ size in zebrafish. <em>Current Biology</em>. <strong>23</strong>, 1559–1565 (2013).</p>
</div>
<div>
<p>98. A. Kozlovskaja-Gumbrienė, R. Yi, R. Alexander, A. Aman, R. Jiskra, D. Nagelberg, H. Knaut, M. McClain, T. Piotrowski, Proliferation-independent regulation of organ size by Fgf/Notch signaling. <em>eLife</em>. <strong>6</strong>, 1–31 (2017).</p>
</div>
</div>
</div>



























            </section>

          </div>
        </div>
      </div>
<a href="statutory-declaration.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis_dsk.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
